# Derepression of the Her-2 uORF is mediated by a novel post-transcriptional control mechanism in cancer cells  

Anuradha Mehta, Christopher R. Trotta, and Stuart W. Peltz PTC Therapeutics, Inc., South Plainfield, New Jersey 07080, USA Transcripts harboring $5^{\prime}$ upstream open reading frames $\mathbf{\sigma_{[uORFs]}}$ are often found in genes controlling cell growth including receptors, oncogenes, or growth factors. uORFs can modulate translation or RNA stability and mediate inefficient translation of these potent proteins under normal conditions. In dysregulated cancer cells, where the gene product, for example Her-2 receptor, is overexpressed, post-transcriptional processes must exist that serve to override the inhibitory effects of the uORFs. The $5^{\prime}$ untranslated region (UTR) of Her-2 mRNA contains a short uORF that represses translation of the downstream coding region. We demonstrate that in Her-2 overexpressing breast cancer cells, the $3^{\prime}$ UTR of the Her-2 mRNA can override translational inhibition mediated by the Her-2 uORF. Within this $3^{\prime}$ UTR, a translational derepression element (TDE) that binds to a 38-kDa protein was identified. These results define a novel biological mechanism in which translational control of genes harboring a $5^{\prime}$ uORF can be modulated by elements in their $3^{\prime}$ UTRs.  

[Keywords: Her-2; uORF; $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs] Received October 27, 2005; revised version accepted February 21, 2006.  

Many studies have established a critical role for the Her-2 receptor in the biochemical pathways responsible for transduction of mitogenic signals from a variety of growth factor receptors (Olayioye et al. 2000; Yarden and Sliwkowski 2001). Overexpression of Her-2/neu protein is pro-oncogenic and has been implicated in ${\sim}30\%$ of solid tumors of the human breast, ovary, prostate, lung, bladder, stomach, esophagus, and pancreas (Hynes and Stern 1994; Holbro and Hynes 2004; Hynes and Lane 2005). Overexpression of Her-2 has been demonstrated to be the result of gene amplification as well as modulation of transcriptional or post-transcriptional control mechanisms (Ishii et al. 1987; Kraus et al. 1987; Bae et al. 2001; Vernimmen et al. 2003; Kuwada et al. 2005).  

Her-2 mRNA and protein are expressed at basal levels in normal cells, but are increased to ${>}50$ -fold in Her-2 positive tumor cells (Kraus et al. 1987; Vernimmen et al. 2003). The major $4.5~\mathrm{kb}$ Her-2 transcript is translated into a protein of $185\;\mathrm{kDa}$ and is flanked by a 178-nucleotide (nt) $5^{\prime}$ untranslated region (UTR) (Ishii et al. 1987; Tal et al. 1987; Hollywood and Hurst 1993) and $612{\-}{\mathrm{nt}}\,3^{\prime}$ UTR. The $5^{\prime}$ UTR is $78\%$ GC rich and contains a sixcodon upstream open reading frame (uORF) that precedes the Her-2 translation start codon by $5\,\textrm{n t}$ and is highly conserved among mammalian species. Although the 612-nt ${\mathit{3}}^{\prime}$ UTR has no recognizable regulatory sequence elements, the ${\mathit{3}}^{\prime}$ -UTR sequences have been reported to stabilize Her-2 mRNA in SKOV-3 ovarian cancer cells (Doherty et al. 1999).  

Two distinct translational mechanisms have been reported to control Her-2 protein expression. One is a celltype-independent mechanism that represses translation of Her-2 mRNA. This repression is mediated by the short uORF in the $5^{\prime}$ UTR of the Her-2 transcript. Mutagenesis experiments demonstrated that the translational repression was independent of the peptide coding sequence of the uORF, as well as the identity of the downstream cistron. This regulatory mechanism has been explored in detail and is believed to be the result of the inability of ribosomes to reinitiate at the downstream AUG codon due to short intercistronic spacing between the two ORFs (Child et al. 1999b).  

The second post-transcriptional control mechanism that is not as well understood involves an increased celltype-dependent Her-2 translational efficiency in transformed cells compared with that in primary cells. In primary human fibroblasts and human mammary epithelial cells, Her-2 mRNA is associated with monosomes and small polysome fractions (Child et al. 1999a). The Her-2 transcript cosediments with larger polysomes in Her-2 overexpressing cancer cells, indicating that it is translated with a higher efficiency in those cells. Importantly, no differences were observed in the sequence and size of the Her-2 mRNA in primary cells compared with that in transformed human cells.  

In this report, we have tested the hypothesis that posttranscriptional processes mediate the efficiency of Her-2 translation in cancer cells through interactions between the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs of Her-2 mRNA. We have identified a U-rich translational derepression element (TDE) in the ${\mathit{3}}^{\prime}$ UTR that can modulate the translational repression by the uORF in the Her-2 $5^{\prime}$ UTR. The protein factors that directly bind to the TDE were identified and were shown to form complexes with the Her-2 mRNA in breast cancer cells. These results elucidate a novel posttranscriptional mechanism involved in the modulation of the translation efficiency of transcripts harboring uORFs in certain cancer cell types.  

# Results  

# Her-2 UTRs modulate luciferase expression in a cell-type-dependent manner  

To investigate the role of the Her-2 UTRs in modulating translation efficiency, a series of gene constructs that contained the firefly luciferase (FLuc) reporter gene flanked by various combinations of either the Her-2 UTRs or control UTRs were made (Fig. 1A). Expression of the reporter gene constructs was studied by transfection into a panel of cell lines that express surface Her-2 protein at very low (MCF-10A, 293H), moderate (Hela, MCF-7), or very high (SKBR-3, BT-474, AU-565) levels. FLuc activity was normalized to the activity units from cotransfected ${\upbeta}$ -galactosidase or Renilla luciferase reporter constructs. Transfection of each construct into various cell lines demonstrated that, compared with the vector-derived UTRs, the Her- $\mathcal{Q}\ 5^{\prime}$ UTR represses translation of reporter transcripts in all of the cell lines studied (Fig. 1B). To compare the effects of the Her-2 $5^{\prime}$ UTR on luciferase expression, we denoted all subsequent activity as fold increase over the repression caused by the $5^{\prime}$ Her-2 construct.  

We also monitored the luciferase activity of the reporter that harbored both the Her- $.2\ 5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs. The results were striking since insertion of the ${\mathit{3}}^{\prime}$ Her-2 UTR into the construct above significantly abrogated the ability of the $5^{\prime}$ UTR to inhibit translation in a number of Her-2 expressing cell lines (Fig. 1B). The greatest modulation was observed in SKBR-3 (12-fold $\pm\ 3.5$ -fold) and BT-474 (10-fold $\pm~2.5$ -fold) cells. MCF-7 cells showed a significant (fourfold $\pm\ 1.5$ -fold) increase in reporter gene expression. These breast cancer cell lines also demonstrate the greatest overexpression of Her-2 protein. Conversely, reporter expression in 293H and MCF-10A cells was hardly affected (1.1-fold $\pm\ 0.86$ -fold) by the presence of Her-2 ${\mathit{3}}^{\prime}$ UTR. The fact that these cells have low endogenous Her-2 expression suggests a possible role for cellular factors in modulating Her-2 expression in Her-2 overexpressing cells.  

The translational derepression by the ${\mathit{3}}^{\prime}$ UTR was not due to changes in transcript stability or alternative ${\mathit{3}}^{\prime}$ -end processing. As determined by Northern blot analysis, there were similar levels of transiently expressed RNAs in SKBR-3 cells (Fig. 3B, below). Sequencing of the ${\mathit{3}}^{\prime}$ -RACE products of the transiently expressed mRNA confirmed that all transcripts used the vector encoded polyadenylation signal and mapped to the same ${\mathit{3}}^{\prime}$ end (data not shown).  

![](images/83e004bf3b61d03ca796750135a186869a9a254d65abd7a4ec97d161a95c4120.jpg)  
Figure 1. Translational regulation of Luc mRNA by the Her-2 UTRs in a panel of cell lines. $\left\vert A\right\vert$ Schematic of the firefly luciferase (Fluc) reporter constructs containing various combinations of $5^{\prime}$ and $3^{\prime}$ UTRs. $\left\vert B\right\vert$ The reporter constructs were expressed via transient transfection in a number of cell lines. The activity was measured $72\,\mathrm{{h}}$ post-transfection and normalized to either Renilla luciferase or to $\upbeta$ -galactosidase units. The results were expressed as fold increase over the $5^{\prime}$ UTR-Fluc units. (C) Reporter constructs containing $5^{\prime}$ Her-2 UTR in combination with $3^{\prime}$ UTR from either Her-2 or GAPDH were expressed transiently in SKBR-3 cells. Translational derepression specifically requires Her-2 ${\mathit{3}}^{\prime}$ UTR since the Luc constructs containing GAPDH $3^{\prime}$ UTR do not derepress translational inhibition by the Her-2 $5^{\prime}$ UTR in SKBR-3 cells.  

Taken together, these results suggest that the translation efficiency of the Her-2 transcript in Her-2 positive cells is regulated by interactions between the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs. These results are consistent with previous reports demonstrating an increased association of the Her-2 mRNA in larger polysome fractions of BT-474 breast cancer cells (Child et al. 1999a) and suggest a novel mechanism in which translational repression by the $5^{\prime}$ UTR can be modulated by the ${\mathit{3}}^{\prime}$ UTR of Her-2.  

# The ${\mathit{3}}^{\prime}$ UTR of Her-2 is required for translational derepression  

We next asked whether the Her-2 ${\mathit{3}}^{\prime}$ UTR was specifically required for the derepression. To test this, the ${\mathit{3}}^{\prime}$ UTR of the reporter gene harboring the Her-2 $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs was replaced with GAPDH ${\mathit{3}}^{\prime}$ UTR. This reporter was transfected into SKBR-3 cells as described previously and luciferase activity was quantified. The results demonstrated that in the presence of Her-2 $5^{\prime}$ UTR, the translation efficiency of the reporter harboring the ${\mathit{3}}^{\prime}$ UTR of GAPDH was greatly reduced compared with the reporter containing the Her-2 ${\mathit{3}}^{\prime}$ UTR (Fig. 1C). No differences in the RNA levels of the constructs were detected by real-time PCR analysis (data not shown). Thus, the GAPDH ${\mathit{3}}^{\prime}$ UTR failed to overcome translation repression caused by the Her- $\mathcal{Q}\:5^{\prime}$ UTR. In the absence of the Her-2 $5^{\prime}$ UTR, normal levels of translation were obtained with luciferase reporter under the control of the ${\mathit{3}}^{\prime}$ UTR of GAPDH (Fig. 1C), again demonstrating the inhibitory effect of the Her-2 $5^{\prime}$ UTR on translation.  

# Translation of a uORF is required for the ${\mathit{3}}^{\prime}$ UTR of Her-2 to mediate translation derepression  

We next examined whether the Her-2 uORF is utilized in cells overexpressing the Her-2 protein. The AUG start codon of the uORF is in an optimal Kozak sequence context and thus is expected to give rise to a six-amino-acid peptide upon initiation from this small ORF. Since the production of such a small peptide would be difficult to monitor, we mutated the uORF termination codon (UGA to GGA) and inserted one nucleotide so that the luciferase ORF would be in frame with the uORF in order to detect initiation at Her-2 uORF. Initiation at upstream AUG (uAUG) would result in the synthesis of an elongated luciferase protein. Transfection of this construct into SKBR-3 cells resulted in a 12-fold increase in activity in the reporter construct harboring the Her-2 uORF containing the GGA in-frame mutant compared with the reporter construct harboring the wild-type Her-2 uORF (Table 1). A further twofold increase $\left(24\pm3.5\right)$ was observed when the Her-2 ${\mathit{3}}^{\prime}$ UTR was present (Table 1). These results suggest that the uAUG is used efficiently by the translating ribosomes and that removal of the upstream termination codon significantly reduced the ${\mathit{3}}^{\prime}$ UTR-mediated derepression. Mutation of the luciferase translation initiation AUG codon in the uORF in-frame construct to AAG did not have an appreciable effect on the expression of luciferase, supporting the conclusion that translation initiated efficiently at the uAUG (Table 1). To determine if the repression by the $5^{\prime}$ UTR of Her-2 is affected by the stop codon sequence in the uORF, the UGA in the uORF was mutated to UAA or UAG and luciferase expression was monitored. The repression of translation was found to be independent of the stop codon sequence (Table 1).  

We next asked whether the interactions between the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs required the translation of the uORF. To address this question, the translation initiation site of the uORF was removed by point mutagenesis (the AUG of the uORF to AAG) and the construct was transiently expressed in SKBR-3 cells. The translational repression was removed in the presence of a dysfunctional uORF. However, in presence of the mutant uORF and the wildtype Her-2 ${\mathit{3}}^{\prime}$ UTR, there was no further synergistic increase in translational efficiency of the construct (Fig. 2A). This result suggests that the derepression of translation mediated by the ${\mathit{3}}^{\prime}$ UTR is dependent on the translation of the uORF. The mRNA levels of all of the transfected constructs reported in these studies were not significantly different as determined by real-time PCR (data not shown).  

We next asked if the increased translation efficiency observed was a consequence of internal translation initiation that was controlled by the Her-2 UTRs. Since the insertion of nucleotides predicted to result in strong secondary structures in the $5^{\prime}$ UTRs in mRNAs have previously been shown to inhibit cap-dependent translation (Kozak 1991a; Muhlrad et al. 1995), translational upregulation by Her-2 UTRs in the presence of a secondary structure would suggest a UTR-mediated internal ribosome entry site (IRES) mechanism. To test this possibility a stem-loop $(\mathrm{dG}=-56.9\;\mathrm{kcal/mol})$ was inserted close to the translation start site of the Her-2 $5^{\prime}$ UTR and its effect on translation of the luciferase constructs was examined. The stem-loop inhibited the cap-dependent translation by ${}_{>200}$ -fold in the vector construct compared with that in the no-stem-loop control. The IRES from the $5^{\prime}$ UTR of HIF- $1\upalpha$ served as a positive control and showed a 13-fold increase in translation in the presence of the stem-loop. However, there was no increase in translation of luciferase under the control of Her-2 $5^{\prime}$ UTR alone or with the $3^{\prime}\;\mathrm{UTR},$ thus indicating that capdependent translation was required for the observed synergy (Table 2).  

Table 1. Effect of mutations in the ${\mathit{5}}^{\prime}$ Her-2–FLUC constructs   

![](images/28b435ed26702c58690a4eff7a6568cc6b02ebb6a2c722376c25e010ce16e3d0.jpg)  
(WT) Wild type; (uORF) upstream open reading frame; (Luc) Luciferase cistron.  

![](images/97ff434fd22cafc0465524479c417f3c006a02d7d82f0524c4375ceedd4c17b4.jpg)  
Figure 2. $\left\vert A\right\vert$ Derepression by the $3^{\prime}$ UTR requires translation of uORF in Her-2 $5^{\prime}$ UTR. The constructs containing wild-type and mutated uORF (AUG to AAG) were transfected into SKBR-3 cells and the activity determined as described in Figure 1. Removal of the uAUG results in loss of the ${\mathit{3}}^{\prime}$ -UTR-mediated derepression of translation. (B) Schematic depicting the location of uORF in SHIP-2 mRNA and the Fluc constructs containing chimeric Her-2 and SHIP-2 UTRs. (C) The $3^{\prime}$ Her-2 UTR derepresses heterologous Ship-2 uORF-mediated inhibition of translation of the reporter in SKBR-3 cells. $\left\vert D\right\vert$ The table shows the relative levels of mRNA of transiently expressed constructs compared with that of $5^{\prime}$ Her-2 mRNA. The RT–PCR data show that the difference in the RNA levels of $5^{\,\prime}+3^{\,\prime}$ SHIP-2 and $5^{\prime}$ SHIP- $\mathcal{Q}+\mathcal{3}^{\prime}$ Her-2 constructs did not contribute significantly to the derepression of SHIP-2 uORF.  

# The ${\mathit{3}}^{\prime}$ UTR of Her-2 can promote translation derepression of other uORFs  

We next determined whether translational derepression by the ${\mathit{3}}^{\prime}$ UTR of Her-2 specifically required the Her-2 uORF or whether it could promote derepression of other known uORFs. To test this, reporters in which the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ Her-2 UTRs were replaced with the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs of the SHIP-2 gene were constructed. The $5^{\prime}$ UTR of SHIP-2 RNA is similar in size to the Her- $2\ 5\,^{\prime}$ UTR and harbors a 10-amino-acid uORF with similar spacing relative to the SHIP-2 protein coding region (Fig. 2B). The reporter constructs were transfected into SKBR-3 cells, and activity was monitored as described above. The results demonstrated that the $5^{\prime}$ UTR of SHIP-2 RNA also inhibited the translation of the reporter in SKBR-3 cells, and that this translational inhibition of the SHIP-2 uORF was not relieved when the SHIP-2 ${\mathit{3}}^{\prime}$ UTR was inserted into the reporter gene (Fig. 2C). The presence of the Her-2 ${\mathit{3}}^{\prime}$ UTR, however, demonstrated a $4.5{\mathrm{-fold}}\pm1.1$ -fold increase in translation efficiency compared with that of the reporter containing the SHIP-2 $5^{\prime}$ UTR. These results demonstrate that, although suboptimal compared with a reporter construct containing both the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs from Her- $2,$ the Her-2 ${\mathit{3}}^{\prime}$ UTR could function in the context of a heterologus uORF sequence to derepress translation of the reporter. Quantitative real-time PCR analysis showed that both $5^{\prime}+3^{\prime}$ SHIP-2 Fluc and $5^{\prime}$ SHIP$2+3^{\prime}$ Her-2 Fluc transcripts were expressed to similar levels and that the RNA levels did not account for the derepression of SHIP-2 uORF by Her- $\mathcal{Q}\,\mathcal{3}^{\prime}$ UTR (Fig. 2D).  

# Identification of sequences required for translational derepression  

We next defined more precisely the cis-acting elements in the ${\mathit{3}}^{\prime}$ UTR that are required for Her-2 uORF-mediated translational derepression. To accomplish this, deletions were made in the 612-nt ${\mathit{3}}^{\prime}$ UTR starting from the ${\mathit{3}}^{\prime}$ end of the reporter harboring the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs of Her-2. Each resulting reporter construct was transfected in SKBR-3 cells and luciferase activity was determined. The results demonstrated that removal of the last 75 nt (Del 537–612) did not affect the ability of ${\mathit{3}}^{\prime}$ UTR in overcoming the inhibition by $5^{\prime}$ UTR (Fig. 4A, below). However, removal of an additional 73 nt (Del 465–612) completely abolished the derepression function of the ${\mathit{3}}^{\prime}$ UTR. These results demonstrated that a 73-nt sequence element (TDE) located between nucleotides 465 and 537 of the ${\mathit{3}}^{\prime}$ UTR is required to override the inhibition of Her-2 translation by the uORF. Northern blot analysis of the transiently expressed RNAs showed that the effect was independent of RNA levels in SKBR-3 cells (Fig. 3B).  

To identify the nucleotides required for derepression, deletions were made within the 73-nt TDE. Deleting nucleotides 497–507 was found to decrease derepression activity of the ${\mathit{3}}^{\prime}$ UTR to $50\%$ of the wild-type construct. A greater loss of derepression activity was obtained when a stretch of $22~\mathrm{nt}$ was deleted from the Her-2 ${\mathit{3}}^{\prime}$ UTR (485–507) or when almost all (Del 463–519) (Fig. 3A) or the entire 73-nt element (Del 465–537, Fig. 4A) was deleted. Sequence analysis of the 73-nt TDE showed the presence of uridine-rich stretches (Fig. 3C). Intriguingly, the TDE is $80\%$ conserved at the nucleotide level across mammalian species.  

Table 2. Effect of stem-loop on translation of the Her-2 UTR FLUC constructs   

![](images/9c6d196e40cc1d7447d13391203efa89e93c80fa597c8a6f6046ff1d488c9059.jpg)  
(SL) Stem-loop.  

![](images/45cd775661ba3dbb005c214723a0321cb68ad90dcdc04e02e301a349f5124d21.jpg)  
Figure 3. Identification of a U-rich TDE in the $3^{\prime}$ UTR of Her-2 mRNA. $\left\langle{A}\right\rangle$ The firefly luciferase activity of the constructs containing ${\mathit{3}}^{\prime}$ -end deletions and internal deletions is shown. $\left\vert B\right\vert$ Northern analysis of the transiently expressed firefly luciferase RNA and the endogenous GAPDH RNA shows stable accumulation of all transcripts $72~\mathrm{h}$ post-transfection. The sequence alignment of the 73-nt U-rich TDE across mammalian species is shown in $C$ .  

To identify the minimal region required for derepression activity, deletions were made from the $5^{\prime}$ end of the ${\mathit{3}}^{\prime}$ UTR (data not shown). The removal of nucleotides 1–107, nucleotides 1–207, nucleotides 1–307, and nucleotides 1–465 from the $5^{\prime}$ end of the ${\mathit{3}}^{\prime}$ UTR completely abrogated derepression activity. In addition, removal of the first five nucleotides in a construct containing nucleotides 5–612 in the ${\mathit{3}}^{\prime}$ UTR resulted in loss of derepression activity, thus indicating that the regions in the $5^{\prime}$ end of the ${\mathit{3}}^{\prime}$ UTR are also required in addition to the TDE. Interestingly, the entire ${\mathit{3}}^{\prime}$ UTR was found to fold into a thermodynamically stable secondary structure with a $\Delta G$ of $-228\ \mathrm{kcal/mol}$ (M-fold analysis; data not shown).  

# Identification of factors that interact with the TDE  

We hypothesized that the TDE interacts with trans-acting factors to regulate translational derepression. To test this hypothesis, UV-cross-linking studies were performed to identify proteins that interact with the TDE.  

Several radiolabeled RNA transcripts were prepared that contained either the full-length Her-2 ${\mathit{3}}^{\prime}$ UTR or a deletion construct lacking the TDE or the Her- $\mathcal{Q}\:5^{\prime}$ UTR. The RNAs were incubated with cytoplasmic extracts from the SKBR-3 cells and after $^{1\mathrm{~h},}$ the protein–RNA complexes were cross-linked by UV irradiation. Following RNase $\mathrm{A}/\mathrm{T}1$ digestion, the radiolabeled RNA–protein complexes were resolved on SDS-PAGE and visualized by phosphorimager analysis. The results demonstrated that a predominant $38–\mathrm{kDa}\pm4$ -kDa protein cross-linked to both the 612-nt Her-2 ${\mathit{3}}^{\prime}$ UTR and to the 73-nt $\mathrm{TDE},$ but did not cross-link to the $5^{\prime}$ UTR (Fig. 4B,C). The 38-kDa protein also did not cross-link to the RNA with 485–507-nt TDE internal deletion (Fig. 4B). The crosslinking activity largely correlated with the luciferase activity of the deletion constructs, indicating the possible involvement of the trans-acting factors in the derepression of translation (Fig. 4A,B).  

![](images/0945d92d217a26c176faf704c6b36b8213607e5fb35acb5133af530daa878096.jpg)  
Figure 4. Derepression activity of TDE correlates with the UVcross-linking of $38{\mathrm{-kDa}}$ protein. $\left\vert A\right\vert$ The luciferase activity of the Fluc constructs containing $5^{\prime}$ Her-2 UTR and internal and $3^{\prime}$ -end deletions of Her-2 $3^{\prime}$ UTR. $\left\vert B\right\vert$ The ${\mathit{3}}^{\prime}$ Her-2 UTR UVcross-linked to a $38{\mathrm{-kDa}}$ protein from SKBR-3 extracts. Proteins UV-cross-linked to labeled RNA probes were resolved on $10\%-$ $14.5\%$ SDS-PAGE as described in Materials and Methods. Molecular size markers are indicated on the right. $|C|$ Competition of the UV-cross-linked band by 73-nt TDE but not by the sequences of the $5^{\prime}$ UTR of Her-2 mRNA. The T7 polymerasetranscribed $^{32}\mathrm{P}$ -labeled $3^{\prime}$ Her-2 UTR RNA was incubated with SKBR-3 cytoplasmic extracts in the absence of competitor (none) or in the presence of a 2.5-fold, fivefold, 20-fold, 50-fold, or 100-fold molar excess of unlabeled 465–537-nt RNA or with a 10-fold, 50-fold, 100-fold, or 500-fold molar excess of the 178- nt $5^{\prime}$ Her-2 UTR RNA.  

Since the results demonstrated that a $38{\mathrm{-}}{\mathrm{kDa}}$ protein UV-cross-linked to the full-length 612-nt Her2 ${\mathit{3}}^{\prime}$ UTR, competition experiments were performed to investigate the binding specificity of the $38{\mathrm{-}}{\mathrm{kDa}}$ protein (Fig. 4C). The cross-linking activity was effectively competed out by 50-fold molar excess of the cold-73-nt TDE (465–537) but not by a 500-fold molar excess of 178-nt Her-2 $5^{\prime}$ UTR or by region 1–307 of Her-2 ${\mathit{3}}^{\prime}$ UTR lacking the binding site (data not shown). These results show that the UV-cross-linking activity is specific to the 73-nt TDE in the Her-2 ${\mathit{3}}^{\prime}$ UTR.  

# Identification of proteins binding to Her-2 ${\mathit{3}}^{\prime}$ UTR in SKBR-3 cells  

We hypothesized that the TDE in the Her- $.2\,3^{\prime}$ UTR may function by recruiting protein factor(s) that facilitate interactions between the UTRs of Her-2 mRNA and the cellular translation machinery. To study the role of trans-acting factors in modulating interactions between the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs, we utilized RNA affinity capture with in vitro biotinylated ${\mathit{3}}^{\prime}$ UTR transcripts to purify and identify the protein(s) that bound to the Her-2 ${\mathit{3}}^{\prime}$ UTR TDE. SKBR-3 cytoplasmic extracts were precleared using RNA (1–407) lacking the TDE. Unbound proteins were subsequently incubated with the TDE-containing RNA (1–612) resin. After extensive washing, the bound proteins were eluted stepwise with $0.2~\mathrm{M},~0.5~\mathrm{M},$ and 1 $\mathrm{\bfM}\,\mathrm{\bfNaCl}$ containing buffer A. Almost all of the UV-crosslinking activity was associated with the $0.5~\mathrm{~M~}$ eluate. The proteins in the eluate were concentrated and resolved by SDS-PAGE. Protein bands were excised and their identity determined by mass-spectrometric analysis.  

The mass-spectrometric analysis identified HuR and hnRNP A1 as Her-2 ${\mathit{3}}^{\prime}$ UTR-binding proteins and suggested that multiple proteins may interact with ${\mathit{3}}^{\prime}$ UTR (data not shown). To determine which of the identified proteins directly bound to the TDE, in vitro binding affinity chromatography was again utilized with biotinylated RNA. The protein–RNA complexes were transferred to nitrocellulose, and the presence of the candidate proteins, HuR and hnRNP A1, as well as a number of other hnRNP proteins were examined by immunodetection with antibodies. The results demonstrated that hnRNP A1 bound not only to the full-length RNA (1– 612) and the TDE (465–537) but also to RNA (1–407) lacking the TDE-binding site, indicating the presence of multiple binding sites in Her-2 ${\mathit{3}}^{\prime}$ UTR (Fig. 5A). hnRNP ${\cal C}1/{\cal C}2$ and HuR bound specifically to the TDE and to the 1–612-nt RNA but not to the RNA (1–407) (Fig. 5A). Other hnRNP proteins $\scriptstyle(\mathrm{Q},\ \mathrm{K}/\mathrm{J}),$ transportin, and AUF (data not shown) were not detected. A very small proportion of hnRNP L (cf. the total input) was detected in the complex and could be the result of nonspecific binding under the conditions used. Overall, these results suggest that a large protein complex containing several known RNA-binding proteins assembles specifically on the Her- $.2\:3^{\prime}$ UTR TDE.  

We next determined which of the proteins detected in the Her- $.2\,3^{\prime}$ UTR RNA complex were also present in the UV-cross-linked band. The protein extracts were UVcross-linked to full-length ${\mathit{3}}^{\prime}$ UTR RNA (1–612) or the 73-nt TDE and after RNase treatment, the reactions were immunoprecipitated with various antibodies. Antibodies to the proteins HuR and ${\cal C}1/{\cal C}2$ immunoprecipitated the UV-cross-linked proteins whereas antibodies to hnRNP A1, L, Q, U, K/J, AUF, and transportin did not (Fig. 5B). These results confirm the presence of HuR and hnRNP C1/C2 as components of a RNP complex making direct contact with the Her-2 ${\mathit{3}}^{\prime}$ UTR.  

![](images/7d19290ce6d13a5885c2c414e62c1a757dee9ff2eb67f5290eae9fec09e6148c.jpg)  
Figure 5. $\left\langle{A}\right\rangle$ A specific protein complex assembles on the 73-nt TDE. Western blot analysis of the protein complexes formed on biotinylated RNA encoding fulllength $3^{\prime}$ UTR (1–612) or the 73-nt (465–537) or on RNA lacking the TDE (1–407). The complexes formed on RNA–streptavidin beads were resolved on SDS-PAGE and immunoblotted to nitrocellulose membrane. The blots were probed with antibodies to various proteins. $\left\vert B\right\vert$ The UV-cross-linked proteins were immunoprecipitated by anti-HuR and anti- $.C1/C2$ antibodies. The proteins cross-linked to the full-length ${\mathit{3}}^{\prime}$ UTR (1–612) or to the 73-nt binding site were incubated with various antibodies and the immunoprecipitated complexes resolved by SDS-PAGE and visualized by autoradiography.  

# Characterization of the native complexes in SKBR-3 cells  

To determine if HuR and ${\cal C}1/{\cal C}2$ exist as native complexes in vivo, extracts from SKBR-3 cells were subjected to immunoprecipitation by anti-HuR or by anti- $_{\mathrm{Cl/C2}}$ antibodies. The immunoprecipitates were resolved on SDS-PAGE and, after transfer to nylon membranes, were probed with various antibodies. The hnRNP A1 and ${\mathrm{C}}1/\,$ C2 proteins were found to be present in the complex immunoprecipitated by anti-HuR antibody. In addition, hnRNP A1 and HuR were pulled down by anti${\cal C}1/{\cal C}2$ antibody. However, the interaction between hnRNP ${\mathrm{C}}1/{\mathrm{C}}2$ and HuR was abrogated when the extract was treated with RNases before immunoprecipitation. This result suggests that RNA facilitates the interactions between factors and that there is no direct interaction between HuR and hnRNP ${\mathrm{C}}1/{\mathrm{C}}2$ (Fig. 6A). As seen with the in vitro assembled complexes, the in vivo complexes contained hnRNP A1 but did not contain AUF and other hnRNP proteins $\langle\mathrm{L},\mathrm{~Q},\mathrm{~U},\mathrm{~K}/\mathrm{J},$ transportin, or $\mathrm{A}_{\mathrm{o}}|$ . Thus, we conclude that $\operatorname{HuR},$ hnRNP ${\mathrm{C1/C2}},$ and A1 are present in the native complexes in SKBR-3 cells.  

Since the ${\mathit{3}}^{\prime}$ UTR of Her-2 mRNA was found to override the translational inhibition imposed by the uORF in the $5^{\prime}$ UTR, we were interested in determining whether there was association of the TDE complex with known translation factors. Thus, in addition to the proteins detected in the in vitro assembled complexes, the native immunoprecipitates were also examined for the presence of proteins known to mediate cap-dependent translation. While the poly(A)-binding protein (PABP) was detectable in the immunoprecipitates of both HuR and C1/ C2 (Fig. 6A), other factors involved in mRNA translation, such as eIF4E, eIF4G, $\mathrm{eIF}3\alpha.$ ,$\mathrm{eIF5}\alpha.$ , and $\mathrm{eIF2}\alpha,$ , were not detected (data not shown). The interaction of the PABP with the TDE complex was found to be RNA independent (Fig. 6A), indicating a direct association with both HuR and with hnRNP ${\mathrm{C}}1/{\mathrm{C}}2$ .  

# HuR and hnRNP C1/C2 interact with the ${\mathit{3}}^{\prime}$ Her-2 UTR in cells  

To study if the Her-2 mRNA interacts with the protein complexes in vivo, a ribonucleoprotein immunoprecipitation (RIP) assay was performed as described in Materials and Methods. Briefly, the in vivo SKBR3–RNP complexes were reversibly cross-linked with formaldehyde and the extracts immunoprecipitated with anti-HuR or anti- $_{\cdot\mathrm{Cl/C2}}$ -protein-G-Sepharose. Total RNA was isolated after the cross-links were reversed, DNase I treated, and the RNA was used for RT–PCR. As seen in Figure 6B, the Her- $2\:3^{\,\prime}$ UTR was detectable in the immunoprecipitates of anti-HuR and anti- $_{\mathrm{Cl/C2}}$ antibodies. As expected, no product was detected when reverse transcriptase was omitted. The absence of a PCR product corresponding to GAPDH mRNA in the immunoprecipitates demonstrated the specificity of the RIP assay. These results indicate that the Her-2 mRNA is in complex with HuR and ${\mathrm{C}}1/{\mathrm{C}}2$ proteins in vivo in SKBR-3 cells.  

![](images/fc7d7a9140c48cd9cab450dd32ea919588adb9619d42bb9e30dd5c52ec7c3bc2.jpg)  
Figure 6. $\left\vert A\right\vert$ Identification of native complexes assembled in SKBR-3 cells. The SKBR-3 cytoplasmic extracts were left untreated or treated with RNase cocktail on ice for $30\ \mathrm{min}$ . The extracts were immunoprecipitated with anti-HuR or anti-C1/ C2 antibodies and the complexes immobilized on nitrocellulose. The presence of PABP, AUF, HuR, C1/C2, A1, and other hnRNP proteins was examined by immunodetection with corresponding antibodies. $\left\vert B\right\vert$ Association of the endogenous Her-2 mRNA with the $\mathrm{HuR}/{\mathrm{C}}1/{\mathrm{C}}2$ complexes in SKBR-3 cells. The RNA–protein complexes in SKBR-3 cells were cross-linked by formaldehyde and the extracts were made as described in Materials and Methods. The extracts were immunoprecipitated with either anti-HuR or anti- $_{\cdot\mathrm{Cl/C2}}$ antibodies and the RNA– protein cross-links were reversed by heat treatment. The RNA associated with the complexes was extracted with Trizol and used for first-strand cDNA synthesis. The presence of Her-2 RNA or nonspecific GAPDH RNA was detected by PCR using gene-specific primers.  

Functional relevance of the protein complex in breast cancer cells  

Since the proteins associating with the TDE complex (HuR, hnRNP C1/C2, PABP, and hnRNP A1) are ubiquitously expressed, we wanted to establish that the TDE–protein complex is functionally relevant to Her-2 translational derepression. Since the small interfering RNA (siRNA)-mediated knockdown of HuR and hnRNP ${\cal C}1/{\cal C}2$ resulted in significant cytotoxicity to SKBR3 cells, we used UV-cross-linking analysis to study if the cross-linked complex was detectable in the cell lines expressing Her-2 protein to different levels. The UV-crosslinking experiments with extracts made from a number of cell lines showed that the relative abundance of proteins that cross-link to the TDE correlates directly not only with the level of Her-2 expression but also with the fold-derepression activity of the Her-2 ${\mathit{3}}^{\prime}$ UTR (Fig. 7A). The intensity of the UV-cross-linked complex was found to be stronger in cell lines that supported derepression by Her-2 ${\mathit{3}}^{\prime}$ UTR and expressed high Her-2 protein levels (SKBR3, AU565, BT474, MCF-7) when compared with cell lines in which derepression by Her- $\mathcal{Q}\ 3^{\prime}$ UTR is not observed and that have low Her-2 levels (293, HeLa). The results suggest that the expression levels of the protein complex may contribute to derepression activity and Her-2 expression in some cancer cells. Since these are cell extracts, however, we are unable to determine if the differences in cross-linking are directly due to relative abundance of the protein(s) or due to post-translational modifications of one or more of the cross-linked proteins that may influence affinity of interactions and/or intracellular distribution of the protein complex.  

![](images/e928c709cfee4502cf81159b1c3238e1288fdde2303b391e9f3849bc0a138083.jpg)  
Figure 7. Functional significance of the TDE–protein complex. UV-cross-linking studies were performed using extracts prepared from cell lines that express Her-2 to varying levels. A more intense cross-linked complex was detected in high Her-2 expressers (SKBR3, AU565, Calu3, MCF-7) compared with the low Her-2 expressers (293T, HeLa, HepG2). (B) Immunofluorescence microscopy of HuR in breast cancer cells. More cytoplasmic HuR was detected in SKBR-3 cells, a high Her-2 expresser compared with MCF-10A, a low Her-2 expressing cell line.  

To study the intracellular distribution of one or more protein(s) in the TDE complex, we performed an in situ immunofluorescence experiment. As seen in Figure 7B, one of the proteins in the TDE complex, $\operatorname{HuR},$ is differentially distributed between two breast cancer cell lines, SKBR3 and MCF10A. Although HuR is found predominantly in the nucleus of both cell lines, there is a significant portion of HuR within the cytoplasm of the high Her-2 expressing cell line SKBR3 compared with the primary breast cancer cell line, MCF10A. This observation is in accordance with the earlier reports on increased cytoplasmic HuR levels in higher grades of breast tumor samples (Denkert et al. 2004) and in colon cancer cells (Dixon et al. 2001).  

# Discussion  

The $5^{\prime}$ UTRs of $\sim\!5\,\%{-}10\%$ of eukaryotic mRNAs are known to harbor an uORF. Interestingly, a high percentage of RNAs containing uORFs encode oncogenes and growth factors (Kozak 1991b), and expression of these genes is highly regulated, as their protein products are important in cell growth and proliferation. Studies of a subset of RNAs harboring uORFs have shown that they can function by modulating translation efficiencies of downstream cistrons via a number of distinct mechanisms including translation termination and reinitiation as well as nascent-peptide-dependent ribosome stalling on the mRNA (for review, see Morris and Geballe 2000).  

Overexpression of the Her-2 receptor is a critical factor in oncogenic transformation of certain breast cancers. In all mammals studied so far, the $5^{\prime}$ UTR of Her-2 mRNA contains an uORF that is highly conserved in its position, length, and sequence, suggesting an important role for the uORF in translational regulation. The Her-2 uORF is translated and strongly inhibits the expression of the downstream coding region in a peptide-sequence-independent manner (Child et al. 1999b). In the absence of the Her-2 ${\mathit{3}}^{\prime}$ UTR, it has been reported that termination and reinitiation at the main Her-2 AUG occurs inefficiently due to short intercistronic spacing. Thus, post-transcriptional mechanisms must exist to repress the function of the uORF and allow for overexpression of Her-2 mRNA that contributes to oncogenic potential of tumor cells.  

# Tumor-specific Her-2 translational control requires the ${\mathit{3}}^{\prime}$ UTR  

To investigate the potential role for translation control of the Her-2 mRNA, we focused on the function of the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs. Utilizing reporter constructs where the Her-2 $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs flank the firefly luciferase, we demonstrated that the uORF of Her-2 is derepressed in the presence of the ${\mathit{3}}^{\prime}$ Her-2 UTR in a tumor-cell-linespecific manner. Derepression of the uORF-mediated translation specifically required the presence of the Her-2 ${\mathit{3}}^{\prime}$ UTR; a control reporter harboring the GAPDH ${\mathit{3}}^{\prime}$ UTR was unable to cause uORF derepression. In addition we demonstrated that derepression is independent of the nascent peptide, but requires translation of the uORF. We hypothesize that the translation of the uORF allows the ribosome to properly terminate at the uORF stop codon where derepression is facilitated by factors that interact with the Her-2 ${\mathit{3}}^{\prime}$ UTR.  

Deletion analysis of the ${\mathit{3}}^{\prime}$ UTR identified two important regions that are necessary to mediate derepression of the Her-2 uORF. The $5^{\prime}$ -proximal portion of the ${\mathit{3}}^{\prime}$ UTR $(1{-}465\,\mathrm{\nt})$ was required for translation derepression. This region has the potential to fold into thermodynamically stable strong secondary structures (M-fold) including IRES-like Y-type stem-loops (141–244 nt) (http://www.ba.itb.cnr.it/BIG/UTRScan; Le and Maizel 1997). A second sequence element, referred to as a TDE, located at 465–537 nt of the ${\mathit{3}}^{\prime}$ UTR was also shown to be necessary to mediate translation derepression of the uORF.  

# The Her-2 TDE binds a novel protein complex  

Several RNA-binding proteins were identified that associated specifically with the TDE. Among the proteins identified, HuR and hnRNP ${\cal C}1/{\cal C}2$ proteins were found to bind specifically and directly to a short sequence (nucleotides 497–507) in the TDE (Fig. 4B). In addition, the hnRNP A1 protein was found to be associated with the immunoprecipitated complex formed on the TDE both in vitro and in vivo.  

Both HuR and hnRNP C1/C2 bind to the U-rich sequences with high affinity (Gorlach et al. 1994; Lopez de Silanes et al. 2004). HuR is a highly conserved member of the Elav/Hu family of RNA-binding proteins involved in the modulation of RNA stability and translation (Good 1997). hnRNP C1/C2 proteins are nuclear proteins proposed to be involved in mRNA processing (Choi et al. 1986; Swanson et al. 1987). The hnRNP C1/C2 proteins have recently been implicated in translational control of the c-sis mRNA as well as poliovirus RNAs (Sella et al. 1999; Brunner et al. 2005). Post-translational modifications that mask the strong nuclear retention signal of hnRNP C1/C2 have been hypothesized to result in their retrafficking in the cytoplasm (Mayrand et al. 1993; Pinol-Roma and Dreyfuss 1993; Sella et al. 1999). Both HuR and hnRNP C1/C2 were shown to bind to a U-rich translational regulatory element in the $5^{\prime}$ UTR of ${\mathfrak{p}}27^{k i p}$ RNA in a cell-cycle-dependent manner (Millard et al. 2000). The hnRNP A1 protein has been shown to be associated with pre-mRNA processing (Burd and Dreyfuss 1994) and export (Michael et al. 1995) of RNAs from the nucleus. hnRNP A1 binds to UAGGGU/A sequences in the RNA (Burd and Dreyfuss 1994), and one such consensus binding site is present in Her-2 mRNA (nucleotides 207–247).  

In addition to proteins that directly cross-linked to the Her-2 TDE, the PABP was found to be associated with the native TDE complexes from SKBR-3 cells isolated by immunoprecipitation with anti-HuR and anti- $_{\cdot\mathrm{Cl/C2}}$ antibodies. The PABP is normally bound to the poly(A) tract of the ${\mathrm{mRNA}},$ although it has been implicated in binding to certain regions in $5^{\prime}$ (Yohn et al. 1998; Hornstein et al. 1999) and ${\mathit{3}}^{\prime}$ UTRs. PABP was shown to interact not only with components of translation initiation machinery (Sachs and Davis 1989; Tarun and Sachs 1996; Wells et al. 1998) but also with the translation termination release factor eRF3 (Hoshino et al. 1999). This has led to the notion that PABP most likely plays important roles in translation initation, termination, and recycling (Kahvejian et al. 2001, 2005; Wilkie et al. 2003; Kuhn and Wahle 2004). The observation that PABP interacts with TDE-binding proteins and key components of the translation initiation and termination process may suggest a role for PABP in regulating the translation derepression efficiency of the Her-2 transcript.  

An important question is what determines the celltype specificity that allows for translational derepression of the Her-2 transcript. HuR, hnRNP C1/C2, hnRNP A1 and PABP are ubiquitously and highly expressed proteins. It is conceivable that changes in localization or modification of these proteins, rather than absolute levels may be important. Post-translational modification of HuR (Li et al. 2002) and cell-environment-dependent nuclear–cytoplasmic shuttling of HuR (Yaman et al. 2002) has been reported earlier. Consistent with this idea, increased cytoplasmic HuR levels were observed in human breast carcinoma samples (Denkert et al. 2004). Increased cytoplasmic HuR levels were also shown to stabilize the COX-2 transcript in colon cancer cell lines (Dixon et al. 2001). Consistent with this observation, we found strikingly higher levels of HuR in the cytoplasm of SKBR-3 cells that overexpress Her-2 compared with MCF10A cells in which the derepression activity is not observed. This increase in cytoplasmic levels of HuR in cells that show high derepression activity, alone or in combination with post-translational modifications, altered affinities, and interactions of other proteins in the TDE complex, could account for the cell-type-specific derepression.  

# ${\mathit{3}}^{\prime}$ UTRs can modulate translation efficiency  

Derepression of the Her-2 uORF involves recruitment of a protein complex bound to the TDE of Her-2 ${\mathit{3}}^{\prime}$ UTR to modulate translation termination/reinitiation efficiency of the Her-2 uORF, underscoring the importance of the termination/reinitiation process in gene regulation. This represents the first demonstration of how disregulation of this mechanism can lead to cancer.  

The role of ${\mathit{3}}^{\prime}$ UTRs in regulating gene expression is well documented, and several lines of evidence indicate that the ${\mathit{3}}^{\prime}$ UTR can modulate the translation efficiency of an mRNA (Ostareck et al. 1997; Grskovic et al. 2003; Kuersten and Goodwin 2003; Hesketh 2004; Cho et al. 2005; de Moor et al. 2005; Espel 2005). The closed-loop model of eukaryotic mRNA structure is one conceptual framework that can explain how ${\mathit{3}}^{\prime}$ UTRs and their associated factors can modulate translation initiation efficiency (Tarun et al. 1997; Wells et al. 1998; Mangus et al. 2003). Interactions of the factors that bind to the $5^{\prime}$ cap, the ${\mathit{3}}^{\prime}$ poly(A) tract, and $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs promote contacts between the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ ends of eukaryotic mRNAs that regulate translation initiation efficiency (Wilkie et al. 2003). The intriguing observation that PABP interacts with eRF3 and the factors at the $5^{\prime}$ cap suggests a further modification of the closed-loop model. This interaction suggests that terminating ribosomes may influence the RNP structure of the closed loop that affects subsequent rounds of translation initiation (Welch et al. 2000). Hence, logically the closed-loop model may include a more pretzel-like structure in which the $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs are interacting at the terminating ribosome to modulate translation reinitiation.  

An important question to consider is how the Her- $.2\,3^{\,\prime}$ UTR can modulate its translation initiation efficiency. At present there are only a few examples illustrating the role of uORFs in regulating the initiation of translation. Perhaps the best example of uORF regulation is the control of expression of GCN4 in the yeast Saccharomyces cerevisiae. The GCN4 mRNA harbors four uORFs and translation initiation/reinitiation efficiency of the uORFs governs the quantity of GCN4 protein produced (for review, see Hinnebusch 2000). The GCN4 uORF1 is always translated, and reinitiation following translation termination of uORF1 is very efficient. Translation reinitiation occurs either at the fourth uORF or at the AUG of the GCN4 protein-coding region depending on the nutrient supply. Under amino acid sufficient conditions, the ribosomes reinitiate at uORF4, but following translation termination, the GCN4 coding region is not translated. Under amino acid deficient conditions, the ribosomes bypass the fourth uORF and reinitiate further downstream at the GCN4 translation initiation site. Starvation generates uncharged tRNA that induces phosphorylation and subsequent inactivation of eIF2. When eIF2 levels fall, the ribosomes take longer to reacquire Met-tRNA and do not translate the fourth uORF, but initiate translation at the GCN4 protein-coding region. Thus, regulation of GCN4 protein expression is an example of how modulation of translation reinitiation events of uORFs can control protein levels.  

The ${\mathit{3}}^{\prime}$ UTR has also been shown to affect other aspects of the translation process. The translational recoding of UGA as selenocysteine is directed by a SECIS (selenocysteine insertion sequence) element in the ${\mathit{3}}^{\prime}$ UTRs of eukaryotic selenoprotein mRNAs (Copeland 2003). Proteins bound to SECIS elements found in the ${\mathit{3}}^{\prime}$ UTR alter translation termination efficiency at these stop codons so that selenocysteine amino acids are inserted. Clearly, there are a number of examples that can help to explain how the Her- $.2\:3^{\prime}$ UTR modulates translation initiation efficiency.  

# Model of translational regulation by uORF and Her-2 ${\mathit{3}}^{\prime}$ UTR  

The results presented here suggest that the uORF in the Her-2 $5^{\prime}$ UTR is a translational repressor that allows inefficient translation reinitiation at downstream cistrons. Factors bound to the Her-2 ${\mathit{3}}^{\prime}$ UTR, however, can alter this process such that terminating ribosomes can reinitiate at the downstream Her-2 protein coding region (Fig. 8). Several lines of evidence are consistent with this hypothesis. In the primary human mammary epithelial cells, which express low levels of the Her-2 receptor, the majority of the Her-2 mRNA cosediments with small polysomes and suggests that 80S ribosomes may stall at the uUGA of the uORF. In contrast, the Her-2 mRNA is distributed throughout the polysome gradient in BT-474 breast cancer cells that overexpress Her-2 protein (Child et al. 1999a). This result suggests that the Her-2 transcript is efficiently translated and suggests that translation reinitiation occurs in the Her-2 protein-coding region.  

![](images/59db34282982e538074dfb526fb6ffffddc5aedb1cd943b5ada3ac54a931e27d.jpg)  
Figure 8. Model for translational derepression by Her-2 $3^{\prime}$ UTR. $\left\vert A\right\vert$ Based on earlier studies, the conceptual closed-loop model shows interactions of PABP with the cap complex at the $5^{\prime}$ end as well as its interaction with eRF3 at the stop codon near the ${\mathit{3}}^{\prime}$ end of the coding region. In the simplest scenario, the ribosomes translating the uORF remain stalled at the uUGA in cell lines where derepression is not significant. Under these conditions the translation of Her-2 protein is dependent on leaky scanning and/or inefficient reinitiation. $\left\vert B\right\vert$ In cells overexpressing Her-2 protein, the highly folded $3^{\prime}$ UTR along with the protein complex docked on TDE facilitates rapid termination at the uUGA via the PABP–eRF3 contacts. The 40S ribsome that remains attached retains certain initiation factors, reacquires eIF2-Met- $\mathrm{tRNA_{i}}$ -GTP, and is recycled to the main coding region with higher efficiency. The ribosomes terminating at the uUGA support formation of a pretzel-like mRNP structure as described in the Discussion. Such a structure would allow terminating ribosomes to recycle onto the same transcript through interactions with the ${\mathit{3}}^{\prime}$ -UTR protein complex.  

We hypothesize that the 80S ribosomes translating the short uORF can remain associated with the Her-2 mRNA (Fig. 8A). The interactions of the ribosomes with the Her-2 ${\mathit{3}}^{\prime}$ UTR–protein complex may impact termination efficiency at the stop codon of uORF and facilitate efficient recharging of the ribsome to allow translation reinitiation at the Her-2 translation start site. The TDEassociated complex that includes the PABP may allow rapid dissociation of the 80S complex through its interactions with the translation termination release factor, eRF3, at the stop codon of the uORF so that the 40S subunit remains associated with the mRNA. The eRF3 complexes with the eRF1 and contributes to efficient termination at the stop codon (Stansfield et al. 1995a,b). Since the activity of the release factors can be modulated by other factors such as Upf1 (Czaplinski et al. 1998; Welch et al. 2000), we hypothesize that protein factors within the TDE protein complex may modulate activity of the release factors and subsequent recycling of the ribosomal subunits (Fig. 8B). Because of the small size of the uORF, the 40S subunit may retain certain translation initiation factors and rapidly reacquire eIF2/Met-tRNAi/ GTP, so that they efficiently reinitiate at the Her-2 translation start site. It is also conceivable that the theoretical IRES-like structure within the Her-2 ${\mathit{3}}^{\prime}$ UTR may mimic the $5^{\prime}$ cap and, in concert with the TDE-bound protein complex, may facilitate recruitment of translation factors for rapid reinitiation at the Her-2 translation start site.  

The model described above is a modification of the closed-loop model of eukaryotic mRNA that describes how ribosomes are recycled following translation termination at the end of the protein-coding region (Fig. 8A; Kahvejian et al. 2001). In this scenario, terminating ribosomes proximal to the ${\mathit{3}}^{\prime}$ UTR can be recycled to the translation start site proximal to the $5^{\prime}$ UTR as a consequence of the interactions of factors associated with $5^{\prime}$ and ${\mathit{3}}^{\prime}$ UTRs, as well as factors associated with ribosomes. In the case of Her-2 uORF, we hypothesize that the factors associated with Her-2 ${\mathit{3}}^{\prime}$ UTR can promote recycling of the terminating ribosomes at the uORF to facilitate their subsequent interaction at the Her-2 translation start site $(^{\prime\prime}{\mathrm{pretzel-loop}}^{\prime\prime})$ ; Fig. 8B).  

Interestingly, this model suggests that the single Her-2 uORF functions like a combination of the GCN4 uORF1 and uORF4 uORFs. In the absence of other elements and factors, the Her-2 uORF is analogous to the GCN4 uORF4 in that ribosomes that translate this uORF cannot efficiently translate downstream cistrons. Factors bound to the Her- $\mathcal{Q}\,\mathcal{3}^{\prime}$ UTR, however, can alter the Her-2 uORF so that it functions more like the GCN4 uORF1, allowing efficient translation reinitiation to occur at downstream cistrons. The key difference is that, much like the SECIS elements that promote translation readthrough for synthesis of selenocyteine proteins, the Her-2 regulatory sequences that modulate this activity appear to be linearly distant from the site of action.  

The work reported here describes a novel translation mechanism in which the translation efficiency of genes harboring uORFs in their $5^{\prime}$ UTRs can be governed by sequences in their ${\mathit{3}}^{\prime}$ UTRs. The detailed mechanism of how the ${\mathit{3}}^{\prime}$ UTR facilitates increased translation efficiency at the translation initiation site of the Her-2 protein-coding region requires further elucidation. Further dissection of the mechanism will help gain insight into the post-transcriptional mechanisms modulating translation of many regulatory genes and will facilitate the identification of new molecular targets for drug discovery to treat cancer.  

# Materials and methods  

# Generation of Her-2 constructs  

The 178-nt $5^{\prime}$ UTR and the 612-nt $3^{\prime}$ UTR of Her-2 mRNA were amplified by PCR using SKBR-3 cDNA and the human genomic DNA (Promega), respectively, as template and then cloned into pcDNA3.1/FLUC vector, a pcDNA $3.1(+)$ (Invitrogen) plasmid containing a firefly luciferase (FLuc) expression cassette driven by a CMV promoter. Three different constructs were generated, including one in which the luciferase expression is controlled by the Her- $\mathcal{I}\:5^{\prime}$ UTR alone $(5^{\prime}$ Her-2), one controlled by only the Her-2 $3^{\prime}$ UTR $\left|3^{\prime}\right.$ Her-2), and the one in which both the $5^{\prime}$ and the $3^{\prime}$ UTRs from Her-2 controlled Luc expression $\left(5^{\prime}+3^{\prime}\right.$ Her2). The GAPDH $3^{\prime}$ UTR was also PCR amplified and cloned into the above vectors to provide a control for specificity. The sequences of all constructs were confirmed by the dideoxysequencing method.  

# Cell culture and preparation of lysates  

All cell lines were obtained from ATCC and all cell lines except SKBR-3 and MCF10A were cultured in Dulbecco’s Essential Modified Medium (DMEM) supplemented with $10\%$ fetal bovine serum (FBS; Invitrogen) under humidified atmosphere containing $5\,\%$ $\mathrm{CO}_{2}$ . SKBR-3 cells were grown in McCoy’s medium containing $10\%$ FBS, and MCF10A cells were grown in Mammary Epithelial Growth Medium $\langle{\cal M}{\cal E}{\cal G}{\cal M}_{\rangle}$ Cambrex, Inc.) supplemented with $10\%$ FBS and bovine pituitary extract. Whole-cell lysates for luciferase and ${\upbeta}$ -galactosidase assays were prepared by lysing cells in passive lysis buffer (Promega) after two washes with PBS. The cytoplasmic extracts were prepared by swelling the cells in hypotonic buffer $10\;\mathrm{mM}$ HEPES at $\mathrm{pH}$ 7.4, 15 mM KCl, $1.5\;\mathrm{mM}$ magnesium acetate, $0.5\;\mathrm{mM}$ Pefabloc [Roche], $2~\mathrm{mM}$ DTT) for $30\ \mathrm{min}$ followed by lysis by Dounce homogenizer. The lysates were clarified by centrifugation at $10,000~\mathrm{rpm}$ for $10\ \mathrm{min}$ at $4^{\circ}\mathrm{C}$ . The total protein content was quantitated by the BCA microtiter protein assay reagents (Pierce).  

# Mutagenesis  

Deletions in the $3^{\prime}$ UTR were made by amplifying regions of interest using polymerase chain reaction with appropriate primers. The primer sequences used for generating different deletions are listed in Table 3. Point mutations or internal deletions or insertions were made using the QuikChange mutagenesis kit (Stratagene) and all of the constructs were confirmed by dideoxy-chain termination DNA sequencing.  

Table 3. Sequences of oligonucleotides used for plasmid constructions   

![](images/1d40bf6a5e5946c739e3be5b524b66a53aff832f20a69e6db554def1a14b97c3.jpg)  
(S) Sense; (AS) antisense.  

# Transfection of cell-lines and reporter assays  

Transient transfections were performed using Fugene (Roche Diagnostics) according to the manufacturer’s instructions. Briefly, SKBR-3 cells were seeded at a density of $5\times10^{5}$ cells per well of a 6-well plate and grown to $80\%{-}90\%$ confluency. Plasmid DNA $|1.0{\mathrm{~}}\upmu\mathrm{g}|$ and CMV- $\cdot{\upbeta}$ -galactosidase DNA $|0.5~\upmu\mathrm{g}|$ or Renilla luciferase DNA $(0.125~\upmu\mathrm{g})$ were incubated with $7~\upmu\mathrm{L}$ Fugene (Roche) in $100~\upmu\mathrm{L}$ serum-free media for $45~\mathrm{min}$ and the DNA–Fugene complexes added drop-wise to $2~\mathrm{mL}$ serum-containing media. Fresh media was added after $12~\mathrm{{h}}$ and the cells grown for another $48~\mathrm{h}$ . For steady-state experiments, cellular protein and/or total RNA was extracted $48~\mathrm{h}$ post-transfection. The fold increase over the $5^{\prime}$ UTR was found to be a function of both the cell-passage number and the transfection efficiency. As a general guideline, most transfections were performed within passages 5–7 in culture. For most constructs, at least three independent transfection studies were performed. Firefly and $R e_{}$ -nilla luciferase reporter activity was determined using the Dual Luciferase Assay system (Promega) and the $\upbeta$ -galactosidase activity was determined using the ${\upbeta}$ -Gal assay kit (Invitrogen). Luciferase activity was normalized to $\upbeta$ -Gal activity or to $R e$ nilla luciferase activity in each experiment.  

# RNA analysis  

Total cellular RNA from transfected SKBR-3 cells was extracted using the RNAeasy extraction kit (Qiagen). The steady-state levels of RNA were determined by either Northern analysis or via real-time PCR. Northern blots were prepared and the blots hybridized using the Ultra-Hyb hybridization solution (Ambion). The gel-purified cDNA encoding firefly luciferase was labeled with $[\alpha^{-32}\mathrm{P}]$ -dCTP using Prime-it II random-primer labeling kit (Stratgene). The probes prepared from the GAPDH control were added along with luciferase for normalization. Radioactive signals were detected with a Storm PhosphorImager (Typhoon) and quantitated using the ImageQuant 2.2 software. Real-time PCR was performed using $5^{\prime}$ FAM– $-3^{\prime}$ TAMARA probes corresponding to the coding region of FLUC or $5^{\prime}$ VIC– ${\mathit{\Omega}}_{3}^{\prime}$ TAMARA probes for $\upbeta$ -Actin (Applied Biosystems). The RT– PCR efficiency for both FLUC and actin was determined across a range of concentrations of input RNAs, and the Comparative ${C_{\mathrm{T}}}$ analysis method (Jordan et al. 2004) was used to calculate the fold changes in expression of all constructs compared with that of the $5^{\prime}$ Her-2 Fluc RNA.  

# ${\mathit{3}}^{\prime}$ -end mapping  

Total RNA from transfected and control SKBR-3 cells was isolated using RNAeasy kits (Qiagen). First-strand cDNA was synthesized from total RNA $(1\;\upmu\mathrm{g})$ using the $3^{\prime}$ -RACE Adaptor (Ambion) and MMLV RT. The cDNA (1 µL) was subjected to PCR using $3^{\prime}$ -RACE primer and a gene-specific primer. The genespecific primers were complementary to the coding regions of either luciferase (for transfected RNA) or the Her-2 receptor. The PCR products were analyzed by agarose gels, and the PCR products were submitted for dideoxy chain termination sequencing.  

# UV-cross-linking (UVXL) of RNA–protein complexes  

RNA–protein binding reactions were performed using $3{-}5~\upmu\mathrm{g}$ of cytoplasmic extract, 10 fmol of $^{32}\mathrm{P}$ -UTP-labeled RNA, $2~\upmu\mathrm{g}$ tRNA, in a final volume of $20\,\upmu\mathrm{L}$ using Binding Buffer A $\mathrm{\left(20\,mM\right.}$ HEPES-KOH at ${\mathrm{pH}}\ 7.5,$ ,$2.5\,\mathrm{\mM}$ ${\mathrm{MgCl}}_{2},$ $100\,\mathrm{\mM}$ KCl, $20\%$ glycerol, $0.5~\mathrm{\mM}$ dithiothreitol, protease inhibitor tablets). Heparin was added to some reactions at a concentration of 0.25 $\upmu\mathrm{g/\upmuL}$ . Addition of heparin to $0.75\,\mathrm{\mg/mL}$ did not have any significant influence on the pattern of UV-cross-linking. Reaction mixtures were incubated for $1~\mathrm{h}$ at $37^{\circ}\mathrm{C}$ and UV irradiated at $254~\mathrm{nm}$ for $10\ \mathrm{min}$ (StrataLinker 1800) on ice. The reaction mixtures were then treated with $4~\upmu\mathrm{L}$ of RNase cocktail (Ambion) for $30\;\mathrm{min}$ at $37^{\circ}\mathrm{C}$ . The samples were analyzed by $15\,\%$ or $10\%{-}14\%$ Criterion gels (Bio-Rad) by SDS-PAGE and detected by phosphorimager analysis. For UVXL immunoprecipitation assays, UV-cross-linking was performed as described above, and the reactions were incubated for $2~\mathrm{{h}}$ with specific antibodies at $4^{\circ}\mathrm{C},$ followed by incubation with protein G beads (Amersham) for $1~\mathrm{h}$ at $4^{\circ}\mathrm{C}$ . After washing with immunoprecipitation Buffer B$50~\mathrm{mM}$ Tris at ${\mathrm{pH}}\ 7.5,$ 2 mM ${\mathrm{MgCl}}_{2},$ $1\,\%$ NP40, $100\,\mathrm{\mM}$ $\mathrm{NaCl},$ Complete [EDTA-free] protease inhibitor tablets [Roche], Pefabloc $\mathrm{[0.1\mM)]}.$ , the RNA–protein complexes were resolved by SDS-PAGE and detected by PhosphorImager.  

# Biotin-RNA affinity purification  

Biotinylated RNA was synthesized in vitro using a T7-Megascript RNA synthesis kit (Ambion, Inc.) in the presence of 14Biotin-UTP (Roche). The biotinylated RNA was bound to streptavidin-coated Dynabeads (Dynal, Inc.) according to the manufacturer’s recommendations. The cytoplasmic extracts from SKBR-3 were precleared by incubation with RNA-affinity resin containing RNA (1–407) in Binding Buffer A. The unbound fraction was incubated with RNA-affinity resin prepared with the RNA (1–612). After $2~\mathrm{{h}}$ of incubation at $4^{\circ}\mathrm{C},$ the unbound proteins were removed and after extensive washes with the binding buffer, the proteins were step-eluted with Binding Buffer A containing $0.25~\mathrm{mM}$ or $0.5\mathrm{~mM~NaCl}$ . The proteins in $0.5\;\mathrm{M}$ eluate were resolved on SDS-PAGE and the protein bands $(38\mathrm{-}40\ \mathrm{kDa})$ size were excised and submitted for mass-spectrometric analysis (City of Good Hope).  

$\upmu\mathrm{M}$ Pefabloc (Roche). Cytoplasmic extracts prepared from SKBR-3 cells were either left untreated or treated with RNase $\mathrm{A}/\mathrm{T}1$ cocktail (100 U) for $30\,\mathrm{\min}$ on ice. The extracts were precleared on protein $_\mathrm{G}$ sepharose for $1~\mathrm{{h}}$ at $4^{\circ}\mathrm{C}$ . The unbound proteins incubated with anti-HuR or anti-hnRNP C1/C2 protein G sepharose for $2\,\mathrm{{h}}$ at $4^{\circ}\mathrm{C}$ . After 5 washes (10 min per wash, $4^{\circ}\mathrm{C}]$ , the proteins bound to the resin were eluted into the SDSPAGE buffer by heating to $100^{\circ}\mathrm{C}$ for $10\ \mathrm{min}$ and resolved on SDS-PAGE and transferred to the nitrocellulose membranes. The anti-HuR and ${\cal C}1/{\cal C}2$ immunoprecipitates were screened with the following antibodies: HuR (Invitrogen); hnRNPs C1/ C2, L, Q, K, transportin, U, A1 (Sigma-Aldrich); eIF4E (Cell Signaling Technology); $\mathrm{eIF2}\alpha,$ eIF5', eIF3, PABP (SantaCruz); AUF-1 (Upstate Biotechnologies).  

# Ribonucleoprotein immunoprecipitation assay (RIA)  

The RIA was performed according to Niranjanakumari et al. (2002), with some modifications. SKBR-3 cells $(1.5\times10^{6})$ were washed twice in PBS after trypsinization and resuspended in 10 mL PBS containing $1\,\%$ formaldehyde $16\%$ stock; Polysciences, Inc.). After $40\;\mathrm{min}$ of cross-linking, the reaction was quenched with glycine $\ensuremath{\left(\mathrm{pH}\;7.0\right)}$ to $0.25~\mathrm{M}$ for $10\ \mathrm{min}$ . After two $50~\mathrm{mL}$ washes with cold-PBS, the fixed cells were lysed by sonication and the insoluble material removed by centrifugation. The cell lysate was precleared on protein-G-sepharose for $1\,\mathrm{{h}}$ at $4^{\circ}\mathrm{C},$ and the precleared lysate was incubated with Superase-in (Ambion) treated protein-G-sepharose coated with anti-HuR or anti-C1/ C2 antibodies. After $3~\mathrm{{h}}$ of incubation the beads were washed extensively in IP Buffer A and resuspended in $100\,\upmu\mathrm{L}$ of $50\;\mathrm{mM}$ Tris.Cl $\mathrm{\Delta}|\mathrm{pH}\,7.0|$ ,$5~\mathrm{mM}$ EDTA, $10\;\mathrm{mM}$ dithiothretol (DTT), and $1\%$ SDS. Samples were incubated at $70^{\circ}\mathrm{C}$ for $45~\mathrm{min}$ to reverse the cross-links. The total RNA bound to the beads was extracted with Trizol and precipitated with isopropanol. The samples were digested with DNase I (Ambion, Inc.) for $45~\mathrm{min}$ at $37^{\circ}\mathrm{C},$ followed by the removal with DNase-inactivating reagent (Ambion, Inc.). The RNA was used as template for cDNA synthesis, and the presence of specific targets was detected by PCR amplification using the gene-specific primers (Table 3).  

Immunofluorescence microscopy MCF-10A and SKBR-3 cells were grown on glass coverslips and were washed in PBS, fixed using $3.7\%$ formaldehyde/PBS for 30 ${\mathrm{min}},$ and permeablized with $0.5\,\%$ Triton X-100/PBS for $5\,\mathrm{min}$ at room temperature. Fixed cells were blocked in $3\,\%$ BSA/PBS for $^{\textrm{\scriptsize1h,}}$ and immunofluorescence staining was performed by incubating with anti-HuR antibody (Invitrogen) diluted in $3\,\%$ BSA/ PBS, followed by the anti-mouse $\mathrm{{IgG}}$ coupled to Alexafluor555 (Molecular Probes). Images were obtained using a Zeiss Axiovert 200 epi-fluorescence microscope and captured using IPLab version 3.6 for Windows software.  

# Acknowledgments  

We thank Sergey Paushkin and Ellen Welch for helpful discussions, Charles Romfo for the SHIP-2 UTR and SL-Fluc constructs, and Joseph Colacino, Thomas Davis, and Pannayiota Triffilis for critical reading of the manuscript.  

Native complex $I P$  

# References  

The SKBR-3 cells grown to $80\%$ confluency were washed twice with PBS and the cells scraped rapidly from the plates. The cells were suspended in IP-Buffer B containing $0.1\%$ NP40 and 0.2 Bae, C.D., Juhnn, Y.S., and Park, J.B. 2001. Post-transcriptional control of c-erb B-2 overexpression in stomach cancer cells. Exp. Mol. Med. 33: 15–19.  

Brunner, J.E., Nguyen, J.H., Roehl, H.H., Ho, T.V., Swiderek, K.M., and Semler, B.L. 2005. Functional interaction of heterogeneous nuclear ribonucleoprotein C with poliovirus RNA synthesis initiation complexes. J. Virol. 79: 3254–3266.   
Burd, C.G. and Dreyfuss, G. 1994. RNA binding specificity of hnRNP A1: Significance of hnRNP A1 high-affinity binding sites in pre-mRNA splicing. EMBO J. 13: 1197–1204.   
Child, S.J., Miller, M.K., and Geballe, A.P. 1999a. Cell typedependent and -independent control of HER-2/neu translation. Int. J. Biochem. Cell Biol. 31: 201–213. —. 1999b. Translational control by an upstream open reading frame in the HER-2/neu transcript. J. Biol. Chem. 274: 24335–24341.   
Cho, P.F., Poulin, F., Cho-Park, Y.A., Cho-Park, I.B., Chicoine, J.D., Lasko, P., and Sonenberg, N. 2005. A new paradigm for translational control: Inhibition via $5^{\,\prime}{-}3^{\,\prime}$ mRNA tethering by Bicoid and the eIF4E cognate 4EHP. Cell 121: 411–423.   
Choi, Y.D., Grabowski, P.J., Sharp, P.A., and Dreyfuss, G. 1986. Heterogeneous nuclear ribonucleoproteins: Role in RNA splicing. Science 231: 1534–1539.   
Copeland, P.R. 2003. Regulation of gene expression by stop codon recoding: Selenocysteine. Gene 312: 17–25.   
Czaplinski, K., Ruiz-Echevarria, M.J., Paushkin, S.V., Han, X., Weng, Y., Perlick, H.A., Dietz, H.C., Ter Avanesyan, M.D., and Peltz, S.W. 1998. The surveillance complex interacts with the translation release factors to enhance termination and degrade aberrant mRNAs. Genes & Dev. 12: 1665–1677.   
de Moor, C.H., Meijer, H., and Lissenden, S. 2005. Mechanisms of translational control by the ${\mathit{3}}^{\prime}$ UTR in development and differentiation. Semin. Cell Dev. Biol. 16: 49–58.   
Denkert, C., Weichert, W., Winzer, K.J., Muller, B.M., Noske, A., Niesporek, S., Kristiansen, G., Guski, H., Dietel, M., and Hauptmann, S. 2004. Expression of the ELAV-like protein HuR is associated with higher tumor grade and increased cyclooxygenase-2 expression in human breast carcinoma. Clin. Cancer Res. 10: 5580–5586.   
Dixon, D.A., Tolley, N.D., King, P.H., Nabors, L.B., McIntyre, T.M., Zimmerman, G.A., and Prescott, S.M. 2001. Altered expression of the mRNA stability factor HuR promotes cyclooxygenase-2 expression in colon cancer cells. J. Clin. Invest. 108: 1657–1665.   
Doherty, J.K., Bond, C.T., Hua, W., Adelman, J.P., and Clinton, G.M. 1999. An alternative HER-2/neu transcript of $8~\mathrm{kb}$ has an extended $3^{\prime}$ UTR and displays increased stability in SKOV-3 ovarian carcinoma cells. Gynecol. Oncol. 74: 408– 415.   
Espel, E. 2005. The role of the AU-rich elements of mRNAs in controlling translation. Semin. Cell Dev. Biol. 16: 59–67.   
Good, P.J. 1997. The role of elav-like genes, a conserved family encoding RNA-binding proteins, in growth and development. Semin. Cell Dev. Biol. 8: 577–584.   
Gorlach, M., Burd, C.G., and Dreyfuss, G. 1994. The determinants of RNA-binding specificity of the heterogeneous nuclear ribonucleoprotein C proteins. J. Biol. Chem. 269: 23074–23078.   
Grskovic, M., Hentze, M.W., and Gebauer, F. 2003. A co-repressor assembly nucleated by Sex-lethal in the ${\mathit{3}}^{\prime}$ UTR mediates translational control of Drosophila msl-2 mRNA. EMBO J. 22: 5571–5581.   
Hesketh, J. 2004. $3^{\prime}$ -Untranslated regions are important in mRNA localization and translation: Lessons from selenium and metallothionein. Biochem. Soc. Trans. 32: 990–993.   
Hinnebusch, A.G. (2000). Mechanism and regulation of initiator methionyl-tRNA binding to the ribosomes. In Translational control of gene expression (eds. N. Sonenberg et al.), pp. 185– 243. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.   
Holbro, T. and Hynes, N.E. 2004. ErbB receptors: Directing key signaling networks throughout life. Annu. Rev. Pharmacol. Toxicol. 44: 195–217.   
Hollywood, D.P. and Hurst, H.C. 1993. A novel transcription factor, OB2-1, is required for overexpression of the protooncogene c-erbB-2 in mammary tumour lines. EMBO J. 12: 2369–2375.   
Hornstein, E., Git, A., Braunstein, I., Avni, D., and Meyuhas, O. 1999. The expression of poly(A)-binding protein gene is translationally regulated in a growth-dependent fashion through a $5^{\prime}$ -terminal oligopyrimidine tract motif. J. Biol. Chem. 274: 1708–1714.   
Hoshino, S., Imai, M., Kobayashi, T., Uchida, N., and Katada, T. 1999. The eukaryotic polypeptide chain releasing factor (eRF3/GSPT) carrying the translation termination signal to the $3^{\prime}$ -Poly(A) tail of mRNA. Direct association of erf3/ GSPT with polyadenylate-binding protein. J. Biol. Chem. 274: 16677–16680.   
Hynes, N.E. and Lane, H.A. 2005. ERBB receptors and cancer: The complexity of targeted inhibitors. Nat. Rev. Cancer 5: 341–354.   
Hynes, N.E. and Stern, D.F. 1994. The biology of erbB-2/neu/ HER-2 and its role in cancer. Biochim. Biophys. Acta 1198: 165–184.   
Ishii, S., Imamoto, F., Yamanashi, Y., Toyoshima, K., and Yamamoto, T. 1987. Characterization of the promoter region of the human c-erbB-2 protooncogene. Proc. Natl. Acad. Sci. 84: 4374–4378.   
Jordan, R.C., Macabeo-Ong, M., Shiboski, C.H., Dekker, N., Ginzinger, D.G., Wong, D.T., and Schmidt, B.L. 2004. Overexpression of matrix metalloproteinase-1 and -9 mRNA is associated with progression of oral dysplasia to cancer. Clin. Cancer Res. 10: 6460–6465.   
Kahvejian, A., Roy, G., and Sonenberg, N. 2001. The mRNA closed-loop model: The function of PABP and PABP-interacting proteins in mRNA translation. Cold Spring Harb. Symp. Quant. Biol. 66: 293–300.   
Kahvejian, A., Svitkin, Y.V., Sukarieh, R., M’Boutchou, M.N., and Sonenberg, N. 2005. Mammalian poly(A)-binding protein is a eukaryotic translation initiation factor, which acts via multiple mechanisms. Genes & Dev. 19: 104–113.   
Kozak, M. 1991a. Structural features in eukaryotic mRNAs that modulate the initiation of translation. J. Biol. Chem. 266: 19867–19870. —. 1991b. An analysis of vertebrate mRNA sequences: Intimations of translational control. J. Cell Biol. 115: 887–903.   
Kraus, M.H., Popescu, N.C., Amsbaugh, S.C., and King, C.R. 1987. Overexpression of the EGF receptor-related proto-oncogene erbB-2 in human mammary tumor cell lines by different molecular mechanisms. EMBO J. 6: 605–610.   
Kuersten, S. and Goodwin, E.B. 2003. The power of the ${\mathit{3}}^{\prime}$ UTR: Translational control and development. Nat. Rev. Genet. 4: 626–637.   
Kuhn, U. and Wahle, E. 2004. Structure and function of poly(A) binding proteins. Biochim. Biophys. Acta 1678: 67–84.   
Kuwada, S.K., Kuang, J., and Li, X. 2005. Integrin $\upalpha5/\upbeta1$ expression mediates HER-2 down-regulation in colon cancer cells. J. Biol. Chem. 280: 19027–19035.   
Le, S.Y. and Maizel Jr., J.V. 1997. A common RNA structural motif involved in the internal initiation of translation of cellular mRNAs. Nucleic Acids Res. 25: 362–369.   
Li, H., Park, S., Kilburn, B., Jelinek, M.A., Henschen-Edman, A., Aswad, D.W., Stallcup, M.R., and Laird-Offringa, I.A. 2002. Lipopolysaccharide-induced methylation of HuR, an mRNA-stabilizing protein, by CARM1. Coactivator-associated arginine methyltransferase. J. Biol. Chem. 277: 44623– 44630.   
Lopez de Silanes, I., Zhan, M., Lal, A., Yang, X., and Gorospe, M. 2004. Identification of a target RNA motif for RNA-binding protein HuR. Proc. Natl. Acad. Sci. 101: 2987–2992.   
Mangus, D.A., Evans, M.C., and Jacobson, A. 2003. Poly(A)- binding proteins: Multifunctional scaffolds for the post-transcriptional control of gene expression. Genome Biol. 4: 223.   
Mayrand, S.H., Dwen, P., and Pederson, T. 1993. Serine/threonine phosphorylation regulates binding of C hnRNP proteins to pre-mRNA. Proc. Natl. Acad. Sci. 90: 7764–7768.   
Michael, W.M., Choi, M., and Dreyfuss, G. 1995. A nuclear export signal in hnRNP A1: A signal-mediated, temperaturedependent nuclear protein export pathway. Cell 83: 415–422.   
Millard, S.S., Vidal, A., Markus, M., and Koff, A. 2000. A U-rich element in the $5^{\prime}$ untranslated region is necessary for the translation of p27 mRNA. Mol. Cell. Biol. 20: 5947–5959.   
Morris, D.R. and Geballe, A.P. 2000. Upstream open reading frames as regulators of mRNA translation. Mol. Cell. Biol. 20: 8635–8642.   
Muhlrad, D., Decker, C.J., and Parker, R. 1995. Turnover mechanisms of the stable yeast PGK1 mRNA. Mol. Cell. Biol. 15: 2145–2156.   
Niranjanakumari, S., Lasda, E., Brazas, R., and Garcia-Blanco, M.A. 2002. Reversible cross-linking combined with immunoprecipitation to study RNA–protein interactions in vivo. Methods 26: 182–190.   
Olayioye, M.A., Neve, R.M., Lane, H.A., and Hynes, N.E. 2000. The ErbB signaling network: Receptor heterodimerization in development and cancer. EMBO J. 19: 3159–3167.   
Ostareck, D.H., Ostareck-Lederer, A., Wilm, M., Thiele, B.J., Mann, M., and Hentze, M.W. 1997. mRNA silencing in erythroid differentiation: hnRNP K and hnRNP E1 regulate 15- lipoxygenase translation from the $3^{\prime}$ end. Cell 89: 597–606.   
Pinol-Roma, S. and Dreyfuss, G. 1993. Cell cycle-regulated phosphorylation of the pre-mRNA-binding (heterogeneous nuclear ribonucleoprotein) C proteins. Mol. Cell. Biol. 13: 5762–5770.   
Sachs, A.B. and Davis, R.W. 1989. The poly(A) binding protein is required for poly(A) shortening and 60S ribosomal subunitdependent translation initiation. Cell 58: 857–867.   
Sella, O., Gerlitz, G., Le, S.Y., and Elroy-Stein, O. 1999. Differentiation-induced internal translation of c-sis mRNA: Analysis of the cis elements and their differentiation-linked binding to the hnRNP C protein. Mol. Cell. Biol. 19: 5429– 5440.   
Stansfield, I., Jones, K.M., Kushnirov, V.V., Dagkesamanskaya, A.R., Poznyakovski, A.I., Paushkin, S.V., Nierras, C.R., Cox, B.S., Ter Avanesyan, M.D., and Tuite, M.F. 1995a. The products of the SUP45 (eRF1) and SUP35 genes interact to mediate translation termination in Saccharomyces cerevisiae. EMBO J. 14: 4365–4373.   
Stansfield, I., Jones, K.M., and Tuite, M.F. 1995b. The end in sight: Terminating translation in eukaryotes. Trends Biochem. Sci. 20: 489–491.   
Swanson, M.S., Nakagawa, T.Y., LeVan, K., and Dreyfuss, G. 1987. Primary structure of human nuclear ribonucleoprotein particle C proteins: Conservation of sequence and domain structures in heterogeneous nuclear RNA, mRNA, and prerRNA-binding proteins. Mol. Cell. Biol. 7: 1731–1739.   
Tal, M., King, C.R., Kraus, M.H., Ullrich, A., Schlessinger, J., and Givol, D. 1987. Human HER2 (neu) promoter: Evidence for multiple mechanisms for transcriptional initiation. Mol. Cell. Biol. 7: 2597–2601.   
Tarun Jr., S.Z. and Sachs, A.B. 1996. Association of the yeast poly(A) tail binding protein with translation initiation factor eIF-4G. EMBO J. 15: 7168–7177.   
Tarun Jr., S.Z., Wells, S.E., Deardorff, J.A., and Sachs, A.B. 1997. Translation initiation factor eIF4G mediates in vitro poly(A) tail-dependent translation. Proc. Natl. Acad. Sci. 94: 9046– 9051.   
Vernimmen, D., Gueders, M., Pisvin, S., Delvenne, P., and Winkler, R. 2003. Different mechanisms are implicated in ERBB2 gene overexpression in breast and in other cancers. Br. J. Cancer 89: 899–906.   
Welch, E.M., Wong, W., and Peltz, S.W. (2000). Translation termination: It’s not the end of the story. In Translational control of gene expression (eds. N. Sonenberg et al.), pp. 467– 485. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.   
Wells, S.E., Hillner, P.E., Vale, R.D., and Sachs, A.B. 1998. Circularization of mRNA by eukaryotic translation initiation factors. Mol. Cell 2: 135–140.   
Wilkie, G.S., Dickson, K.S., and Gray, N.K. 2003. Regulation of mRNA translation by $5^{\prime}.$ - and $3^{\prime}$ -UTR-binding factors. Trends Biochem. Sci. 28: 182–188.   
Yaman, I., Fernandez, J., Sarkar, B., Schneider, R.J., Snider, M.D., Nagy, L.E., and Hatzoglou, M. 2002. Nutritional control of mRNA stability is mediated by a conserved AU-rich element that binds the cytoplasmic shuttling protein HuR. J. Biol. Chem. 277: 41539–41546.   
Yarden, Y. and Sliwkowski, M.X. 2001. Untangling the ErbB signalling network. Nat. Rev. Mol. Cell Biol. 2: 127–137.   
Yohn, C.B., Cohen, A., Danon, A., and Mayfield, S.P. 1998. A poly(A) binding protein functions in the chloroplast as a message-specific translation factor. Proc. Natl. Acad. Sci. 95: 2238–2243.  

# Derepression of the Her-2 uORF is mediated by a novel post-transcriptional control mechanism in cancer cells  

Anuradha Mehta, Christopher R. Trotta and Stuart W. Peltz  

Genes Dev. 2006, 20: Access the most recent version at doi:10.1101/gad.1388706  

# Related Content  

Downstream control of upstream open reading frames Matthew S. Sachs and Adam P. Geballe Genes Dev. UNKNOWN , 2006 20: 915-921  

# References  

This article cites 61 articles, 28 of which can be accessed free at: http://genesdev.cshlp.org/content/20/8/939.full.html#ref-list-1 Articles cited in: http://genesdev.cshlp.org/content/20/8/939.full.html#related-urls  

# License  

# Email Alerting Service  

Receive free email alerts when new articles cite this article - sign up in the box at the top right corner of the article or click here.  