# A uORF Represses the Transcription Factor AtHB1 in Aerial Tissues to Avoid a Deleterious Phenotype  

Pamela A. Ribone, Matías Capella, Agustín L. Arce, and Raquel L. Chan Instituto de Agrobiotecnología del Litoral, Universidad Nacional del Litoral, Consejo Nacional de Investigaciones Científicas y Técnicas, Centro Científico Tecnológico Consejo Nacional de Investigaciones Científicas y Técnicas Santa Fe, Paraje El Pozo, 3000 Santa Fe, Argentina  

ORCID IDs: 0000-0002-3447-8737 (P.A.R.); 0000-0003-4804-243X (M.C.); 0000-0003-3619-3898 (A.L.A.); 0000-0002-3264-0008 (R.L.C.).  

AtHB1 is an Arabidopsis (Arabidopsis thaliana) homeodomain-leucine zipper transcription factor that participates in hypocotyl elongation under short-day conditions. Here, we show that its expression is posttranscriptionally regulated by an upstream open reading frame (uORF) located in its $5^{\prime}$ untranslated region. This uORF encodes a highly conserved peptide $(\mathrm{CP}\bar{\mathrm{uORF}})$ that is present in varied monocot and dicot species. The Arabidopsis uORF and its maize (Zea mays) homolog repressed the translation of the main open reading frame in cis, independent of the sequence of the latter. Published ribosome footprinting results and the analysis of a frame-shifted uORF, in which the repression capability was lost, indicated that the uORF causes ribosome stalling. The regulation exerted by the $\mathrm{CPuORF}$ was tissue specific and did not act in the absence of light. Moreover, a photosynthetic signal is needed for the CPuORF action, since plants with uncoupled chloroplasts did not show uORF-dependent repression. Plants transformed with the native AtHB1 promoter driving AtHB1 expression did not show differential phenotypes, whereas those transformed with a construct in which the uORF was mutated exhibited serrated leaves, compact rosettes, and, most significantly, short nondehiscent anthers and siliques containing fewer or no seeds. Thus, we propose that the uncontrolled expression of AtHB1 is deleterious for the plant and, hence, finely repressed by a translational mechanism.  

Plants, as sessile organisms, have evolved complex traits to cope with the surrounding environment and show high resilience to external perturbations that are somehow buffered by the regulatory interaction of developmental networks. Transcription factors (TFs) play key roles in such networks by acting as mediators between the perception of environmental factors and the cellular responses.  

Six percent of plant genes encode TFs, which are classified in different families and subfamilies (for review, see Ribichich et al., 2014). This classification is based mainly on their DNA-binding domain structures. Among these families, the homeodomain-leucine zipper (HD-Zip) TF family has been assigned roles in the responses to biotic and abiotic stresses as well as in developmental processes (Capella et al., 2015a; Ribone et al., 2015a). The family has been divided into four subfamilies, denoted I to IV, based on structural and functional features. Members of subfamily I were identified in several plant species and related to different stress responses but also with processes such as leaf senescence and morphology (Vlad et al., 2014), stem elongation, hypocotyl elongation, venation patterning, and pollen hydration (Wang et al., 2003; Manavella et al., 2006; Ré et al., 2014; Capella et al., 2015b; Ribone et al., 2015b; Moreno Piovano et al., 2017). Besides the HD-Zip domain, HD-Zip I TFs contain conserved motifs in their C and N termini (Arce et al., 2011). In vitro and in vivo experiments in different plant species showed that the HD-Zip I C termini have key functional roles (Hofer et al., $200{\dot{9}}$ ; Arce et al., 2011; Sakuma et al., 2013). A motif similar to the AHA (aromatic and large hydrophobic residues in an acidic context) transactivation motif was identified at the end of the C termini and was functionally characterized for Arabidopsis (Arabidopsis thaliana) AtHB1, AtHB7, AtHB12, and AtHB13 members (Capella et al., 2014).  

Most Arabidopsis HD-Zip I proteins were resolved as pairs in phylogenetic trees. Some of these pairs exhibited cross regulation and overlapping functions in certain conditions (Ré et al., 2014; Ribone et al., 2015b). This was not the case for AtHB1, which belongs to clade III and does not have a paralog (Arce et al., 2011). This HD-Zip I TF was shown to interact with AtTBP2 both in yeast two-hybrid and in vitro pull-down assays (Capella et al., 2014). The expression of this gene was repressed in NaCl-treated plants and in plants subjected to low temperatures but was induced by darkness (Henriksson et al., 2005). In tobacco (Nicotiana tabacum) plants grown in absolute darkness, AtHB1 overexpression caused constitutive photomorphogenesis (Aoyama et al., 1995). More recently, it was demonstrated that AtHB1 expression is significant in hypocotyls and roots and that this expression is regulated by PIF1 (Phytochrome-Interacting Factor1) to promote hypocotyl elongation under a short-day regime (Capella et al., 2015b). The analysis of athb1 and pif1 mutants, as well as their double mutants, indicated that PIF1 and AtHB1 regulate genes involved in cell wall synthesis. Notably, AtHB1 overexpressor lines never exhibited expression levels higher than $5\times$ the endogenous levels, suggesting a posttranscriptional regulatory mechanism. Such a mechanism was evidenced when rdr6-12 mutant plants, which have nonfunctional small RNA-silencing machinery, were transformed with the same constructs as wild-type Columbia-0 (Col-0) plants. Those rdr6-12/AtHB1 plants exhibited high transcript levels and differential phenotypes (Romani et al., 2016), indicating that a silencing mechanism is taking place when AtHB1 is an overexpressed transgene.  

It is well known that the $5^{\prime}$ untranslated region (UTR) of mRNAs can contain different regulatory elements such as loops, protein-binding sites, internal segments for ribosome entry, and upstream open reading frames (uORFs; Somers et al., 2013). These uORFs are located upstream from the main open reading frame (mORF) and, following the model of Kozak (1987, 2002) for translation initiation, their first AUG codon starting from the CAP is recognized by the ribosome to commence translation. Hence, when a uORF exists, its AUG is the initiation codon, triggering a less efficient mORF translation in most cases.  

In eukaryotic organisms, about $20\%$ to $50\%$ of the transcripts have uORFs. However, those that encode conserved peptides occur in less than $1\%$ of transcripts. In these cases, the uORF is called CPuORF (for Conserved Peptide uORF; Jorgensen and Dorantes-Acosta, 2012). The analysis of Arabidopsis and rice (Oryza sativa) transcriptomes allowed the identification of 26 different CPuORFs (Hayden and Jorgensen, 2007), most of which are present in regulatory genes. Although the function of the CPuORFs is not yet well studied, a few reports have indicated that these sequences modulate the translational efficiency of the downstream mORF in combination with small signal molecules (Rahmani et al., 2009; Ivanov et al., 2010; Alatorre-Cobos et al., 2012; Guerrero-González et al., 2014; Laing et al., 2015). For example, the translation of the basic helix-loophelix TF SUPPRESSOR OF ACAULIS5 LIKE3 (SACL3) is blocked by a uORF in the absence of thermospermine (Katayama et al., 2015). Genes that do not encode TFs, like the Arabidopsis polyamine oxidase-2, also were shown to be regulated by a uORF, and in this case, the amino acid sequence was crucial for this regulation (Guerrero-González et al., 2016). Similarly, a noncanonical uORF represses GDP- $-\mathrm{L}$ -Gal phosphorylase, the major control enzyme of ascorbate biosynthesis, when ascorbate concentration is high (Laing et al., 2015).  

A uORF encoding a conserved peptide was identified previously in the $5^{\prime}$ UTR of AtHB1 and called CPuORF33 (At3G01472.1; Hayden and Jorgensen, 2007). Thus, it is conceivable that AtHB1 expression is regulated through mRNA translation. Indeed, by using in vitro translation approaches, it was shown recently that many CPuORFs, including CPuORF33, have the ability to cause ribosomal arrest (Hayashi et al., 2017). However, the physiological role of CPuORF33 and whether its mechanism of action also is functional in vivo remain unresolved.  

Here, we show that AtHB1 translation is repressed in vivo by a mechanism involving CPuORF33. Our results indicate that this element acts via a ribosome stalling mechanism, independently of the sequence of the mORF downstream of the uORF. The CPuORF33 exerts its repressive effect only in aerial tissues except in darkness. Moreover, the maize (Zea mays) CPuORF33 homolog showed a conserved function. Finally, we show that such a fine and sophisticated regulation is essential for the plant in order to avoid aberrant and lethal phenotypes caused by the uncontrolled expression of AtHB1.  

# RESULTS  

# AtHB1 Has a Conserved Open Reading Frame in Its ${\sf{5^{\prime}}}$ UTR  

Ten years ago, Hayden and Jorgensen (2007) revealed the presence of a conserved encoded peptide upstream from the main coding sequence of AtHB1, located in its $5^{\prime}$ UTR. To investigate if such a sequence/ peptide has a biological function, we carried out an in silico analysis of AtHB1 homologs from other plant species. Using the AtHB1 protein sequence as a query against the National Center for Biotechnology Information nonredundant protein sequences database, a search with BLASTP allowed the retrieval of 45 different nucleotide sequences encoding AtHB1 homologs belonging to 43 plant species, including monocots and dicots (Supplemental Table S1). These homologous sequences were assessed for the presence of ORFs upstream of the mORF; only ORFs starting with ATG and containing at least 24 bp were considered. This analysis led to the identification of 44 different uORFs, all of them belonging to the previously identified group 14 (Hayden and Jorgensen, 2007). An alignment of these sequences indicated a high degree of conservation and a difference in peptide length between monocots and dicots (Fig. 1). In monocot AtHB1 homologs, the peptide had 38 amino acids, whereas in dicots, the length varied between 29 and 30. Accordingly, a phylogenetic tree resolved two clades (Fig. 1B). Nucleotide sequences also were conserved, but to a lesser extent  

![](images/5ff6e071921969bfc4ffd9fce401065f354d6007401e56a78c145bf361131129.jpg)  
Figure 1. The predicted amino acid sequence of CPuORF33 is highly conserved between species. A, Schematic representation of the AtHB1 gene. In gray, exons; in light gray, the $5^{\prime}$ and $3^{\prime}$ UTRs; in green, the CPuORF33; the introns are shown as simple lines. B, At left, a phylogenetic tree constructed using the predicted uORF amino acid sequences of 44 AtHB1 homologs from different species, available in public databases. Two main clades can be distinguished: dicotyledonous and monocotyledonous plants. At right, an amino acid sequence alignment. C, Amino acid sequence logo of CPuORF33. The sequence logo resulted from the alignment of the uORF amino acid sequences of 44 AtHB1 homologs from different species. Letter height corresponds to the frequency in the alignment.  

# AM AsScd N141517181920112345672829333453637  

than the amino acid sequences (Supplemental Fig. S1). A BLAST analysis performed using either the monocot or the dicot consensus peptide did not find any other plant peptide or protein with sufficiently high similarity. The Kozak rule describes the optimal sequence around the initiator AUG for an efficient translation (Kozak, 1986) and has been verified by different studies (Zur and Tuller, 2013). Important positions include position $-3$ , with an A or G, and position $+4,$ , with a $G_{e}$ ,which can be summarized as (A/G)XXATGG. This rule is generally fit by the sequence context of AUGs from uORFs having a single initial AUG as well as those AUGs aligned to them but belonging to uORFs having two initial AUGs (Supplemental Fig. S2).  

A further analysis of AtHB1 (and its homologs) uORF sequences indicated that the length is another conserved trait, although other characteristics of these sequences also were interesting. For example, no overlap between the uORF and the mORF was observed in any case. Additionally, other properties of the sequence traits were assessed, but no remarkable features were found. Among the tested properties were the distances between the CAP and the uORF starting site and between the uORF stop codon and the mORF AUG as well as the phase of the uORF and the mORF (Supplemental Fig. S3). The high sequence similarity between species strongly suggested a regulatory role for the CPuORF33. However, no motifs or a strong indication of secondary structure were found for the encoded peptide (data not shown).  

# CPuORF33 Represses the Expression of AtHB1  

In light of the observations described above, we presumed that the CPuORF33 could play a regulatory role in the expression of AtHB1 and its homologs. To address this hypothesis, two genetic constructs were generated (Fig. 2A): in the first, the expression of the mORF of AtHB1 was controlled by the 1,415-bp upstream region from its ATG $(P r o m\bar{A}t H B1{:}A t H B1)$ , and in the second, two point mutations $(\mathrm{T}\mathrm{\rightarrow}\mathrm{C})$ deleting both ATGs at the beginning of CPuORF33 were introduced (PromAtHB1mut:AtHB1). These constructs were used to transform Col-0 Arabidopsis plants. T1 plants transformed with PromAtHB1:AtHB1 did not exhibit phenotypic differences with respect to the wild-type control. In contrast, those transformed with PromAtHB1mut:AtHB1 presented serrated leaves, short siliques with fewer or no seeds, and a notable delay in bolting and entry to the senescent stage. A similar phenotype was observed in plants expressing AtHB1 at high levels (Romani et al., 2016). Notably, the T2 generation of PromAtHB1mut:AtHB1 plants recovered the wild-type phenotype (Fig. 2B). To understand this observation, AtHB1 transcript levels were quantified in both generations (T1 and T2), with results high in T1 and clearly low in T2, even lower than in the wild type, indicating that a silencing mechanism was in action (Supplemental Fig. S4). For this analysis, 15 single-copy lines were used; these lines were selected on the basis of herbicide resistance segregation in the T1 generation. Notably, this silencing observed in T2 plants was independent of CPuORF33, since both genotypes transformed with either PromAtHB1:AtHB1 or PromAtHB1mut:AtHB1 exhibited lower transcript levels in T2 compared with T1.  

Silencing mediated by small RNAs, and triggered by the overexpression of the transgene, has already been described for the HD-Zip I-encoding genes AtHB1 and AtHB12 when driven by the constitutive 35S cauliflower mosaic virus (CaMV) promoter (Romani et al., 2016), but, to our knowledge, this is the first time this silencing has been observed using the endogenous promoter.  

To gain further insights into the molecular mechanism explaining the phenotypes regarding the CPuORF33, rdr6-12 mutant plants were transformed with the same constructs. These plants have a mutation in the gene encoding the RNA-dependent RNA polymerase6 (RDR6), which is absolutely necessary to display the small RNA-mediated silencing cascade. As shown in Figure 2, the phenotype of the rdr6-12 plants transformed with PromAtHB1:AtHB1 was indistinguishable from that of plants transformed with the empty vector, whereas those transformed with PromAtHB1mut:AtHB1 exhibited serrated leaves in both T1 and T2. Quantification of transcript levels in these new transgenic plants indicated high overexpression of the transgene in T1 and T2 (Supplemental Fig. S4). These results strongly indicated that the small RNA silencing mechanism is independent of the uORF. In addition, AtHB1 expression levels were similar comparing Col-0 and rdr6-12 plants without further transformation (Supplemental Fig. S5), indicating that the silencing mechanism is only displayed as a result of AtHB1 overexpression. Considering that the plants transformed with PromAtHB1mut:AtHB1 and those transformed with 35S:AtHB1 in the rdr6-12 background exhibited almost identical phenotypes, it can be concluded that those possessing the mutated version of the uORF are overexpressing AtHB1.  

# CPuORF33 Is Capable of Repressing the Translation of Different mORFs  

Considering the differential phenotypes shown by PromAtHB1mut:AtHB1 plants compared with those transformed with the construct bearing the native uORF, we found it reasonable to assume that the uORF is repressing AtHB1 at the translational level. Supporting this hypothesis, a recent report has shown that CPuORF33 can arrest ribosomes during mRNA translation in vitro (Hayashi et al., 2017). Moreover, we did not observe significant differences in AtHB1 transcript levels between plants transformed with the native or mutated CPuORF33 when the average levels from independent transgenic lines were calculated (Supplemental Fig. S4C). Unfortunately, AtHB1 protein levels were not detectable by western blots in Col-0 plants, despite using antibodies against two different tags (hemagglutinin [HA] or His).  

Upon discarding CPuORF33 action at the transcriptional level, we decided to assemble new genetic constructs in which the expression of the GUS reporter gene was driven by the AtHB1 promoter and the $5^{\circ}$ UTR with the native or mutated uORF (PromAtHB1:GUS and PromAtHB1mut:GUS). Arabidopsis Col-0 and rdr6-12 plants were transformed with these constructs, and several independent single-copy lines were obtained. Considering that the different insertion points for each independent line could lead to different expression levels, lines transformed with each of the constructs and showing similar GUS transcript levels (in 14-d-old plants) were selected and taken as pairs. These paired plants were analyzed by histochemistry, resulting in the detection of GUS activity in the same tissues (hypocotyls, vascular tissue of the roots, and leaves) for both constructs, but with a strong difference in the signal intensity (Fig. 3). Plants transformed with PromAtHB1:GUS had a weak expression, whereas those transformed with the construct in which the uORF was mutated exhibited a strong GUS color, especially in the leaf lamina (Fig. 3, B and C).  

![](images/4e33cdb4d723a9434d875f269b4877198d75cbc236f82546d37d64b7db289d87.jpg)  
Figure 2. AtHB1 overexpressor plants bearing a uORF mutated in the putative start codons present abnormal phenotypes. A, Schematic representation of the native AtHB1 promoter (PromAtHB1) and a mutated version (PromAtHB1mut), both including their $5^{\prime}$ UTRs. Two single nucleotides, located within the first two codons of the uORF, were mutated $(\mathsf{T}{\to}\mathsf{C})$ and are signaled in red. B, Illustrative photographs of 30-d-old Col-0 plants transformed with PromAtHB1:AtHB1 and PromAtHB1mut:AtHB1 compared with control plants transformed with an empty $p$ Cambia vector. C, Illustrative photographs of 30-d-old rdr6-12 mutant plants transformed with PromAtHB1:AtHB1 and PromAtHB1mut:AtHB1 compared with control plants transformed with an empty pCambia vector. Two independent transgenic lines for each genotype were analyzed. First (T1) and second (T2) generations are shown in the top and bottom rows, respectively.  

Consistent with the observations performed by histochemistry, the quantified GUS enzymatic activity was higher in the extracts obtained from PromAtHB1mut:GUS plants than in those from PromAtHB1:GUS plants (Fig. 3C). These results were independent of the genotype used (Col-0 or rdr6-12), indicating the action of a small RNA-independent mechanism for translational repression (Supplemental Fig. S6).  

# CPuORF33 Represses the Translation of the mORF by a Ribosome Stalling Mechanism  

There are several known mechanisms by which uORFs control translation. Among them, nonsense-mediated decay (NMD) and ribosome stalling are the most studied. We then considered which mechanism was taking place in the regulation exerted by CPuORF33 on AtHB1 translation.  

In view of the preceding findings, it was unlikely that NMD was taking place. To confirm this and discard the possibility of NMD occurrence, insertional mutant plants of UPF1 and UPF3 (upf1-5 and upf3-1, respectively), genes encoding key proteins for NMD, were grown under standard conditions. AtHB1 transcripts were evaluated in these mutants, and the levels were similar to those measured in Col-0 controls (Supplemental Fig. S7), indicating that CPuORF33 translational control was not mediated through NMD.  

To investigate whether CPuORF33 is capable of acting in trans at the transcriptional level, endogenous AtHB1 transcript levels were assessed in rdr6-12 mutant plants transformed either with PromAtHB1:AtHB1 or with PromAtHB1mut:AtHB1. To be sure that the quantified transcripts corresponded to the endogenous AtHB1, reverse transcription (RT)-quantitative realtime PCR (qPCR) assays were performed with primers annealing in the $3^{\breve{\prime}}$ UTR, which is absent in the constructs used to transform these plants. AtHB1 transcript levels were similar in both genotypes, suggesting that CPuORF33 is not able to act in trans at the transcriptional level (Supplemental Fig. S8).  

![](images/6a7808dc03b70cbbf509cab7e2f6b08cd33004f338dd062ac62da2284ae9bb82.jpg)  
Figure 3. Mutations in CPuORF33 enhance the translation of different downstream mORFs. A, Transcript levels of GUS in 14-d-old seedlings of Col-0 plants transformed with native PromAtHB1:GUS or PromAtHB1mut:GUS. Three independent lines of each genotype are shown, paired according to their transcript levels. Transcript levels in whole rosettes were measured by RT-qPCR, and the values were normalized with the smaller absolute value using the DDCt method. The y axis is shown in $\mathrm{log}_{2}$ scale. B, GUS expression analyzed by histochemical detection of GUS enzymatic activity in 14-d-old plants. C, GUS activity evaluated by fluorometry in wholerosette protein extracts from the same plants as in A. Error bars indicate SD of five biological replicates. Student’s t tests were performed, and $P<0.05$ is signaled with asterisks.  

Once NMD and trans-action were discarded as possible mechanisms exerted by CPuORF33, ribosome stalling was analyzed. Recently, ribosome footprinting analyses with Arabidopsis mRNAs were performed by three different research groups (Juntawong et al., 2014; Merchante et al., 2015; Hsu et al., 2016). These studies permit the identification of transcript regions protected by ribosomes from nuclease action (Ingolia et al., 2009). We used these data to inspect if CPuORF33 is, in fact, translated and whether it generates the stalling of the ribosomes. Among these ribosome footprinting experiments, those described by Juntawong et al. (2014) and Hsu et al. (2016) were the most informative for the case of AtHB1. Those authors used seedlings grown under long-day regimes, similar experimental conditions to those used in our experiments, whereas Merchante et al. (2015) used etiolated seedlings.  

The analyses of the data are shown in Supplemental Figure S9. The results indicated that the uORF had a higher occupation density compared with other $5^{\prime}$ UTRs and with the mORF. Weak peaks upstream from the CPuORF33 also were detected, and, considering the absence of AUGs in this region, they indicated that uORFs starting at non-AUG codons (Laing et al., 2015)  

should not be discarded. However, the differences in translation efficiency suggest that their relative importance with respect to CPuORF33 is much lower.  

Furthermore, the CPuORF33 did not exhibit a normal distribution but presented a peak at the end of the uORF, suggesting ribosome stalling. It is important to note that three different experiments resulted in similar results, showing clearly different ribosome footprinting profiles when compared with that of AtHB13, another TF from the same HD-Zip I family (Supplemental Fig. S9). A ribosome stalling process implies the interaction between the nascent peptide and the ribosome. Hence, we decided to test the importance of the CPuORF33 amino acid sequence by generating an additional genetic construct in which the frame of the uORF was shifted. In order to make the analysis independent of the transcriptional activity of the AtHB1 promoter, the native and mutated ${\dot{5}}^{\prime}$ UTRs of AtHB1 were cloned downstream of the 35S CaMV constitutive promoter driving the expression of the GUS reporter gene (35S:native-uORF:GUS and 35S: FS-uORF:GUS, respectively). A schematic representation of these constructs is shown in Figure 4A. In the 35S:FSuORF:GUS construct, the nucleotide sequence exhibits minimal changes, whereas the amino acid sequence is completely altered. Col-0 plants were transformed using these genetic constructs, and the GUS expression pattern was analyzed by histochemistry. As shown in Figure 4B, the uORF with the shifted frame was unable to repress GUS expression.  

To further test the importance of the amino acid sequence, different mutations were generated in the uORF sequence. They were cloned upstream from the GFP mORF, and yeast cells were transformed with these constructs. Protein extracts were analyzed by western blots, indicating reduced levels of GFP when the native uORF was used (Supplemental Fig. S10). In contrast, no repression was observed when the amino acid sequence of the uORF was altered significantly (from residues 3–25 or 3–29). Interestingly, changes on the N-terminal end of the CPuORF33 partially repressed GFP levels, indicating that certain amino acids are more important than others for translational repression (Supplemental Fig. S10). Altogether, these results supported the ribosome stalling mechanism and its dependence on the amino acid sequence encoded by the uORF.  

![](images/270731074680ec95fd63187f27b4867c5162aa7bcbf3897d71e5862004ebc9eb.jpg)  
Figure 4. Ribosome footprinting pattern and mutations in the uORF amino acid sequence suggest ribosome stalling at the CPuORF33. A, Schematic representation of the constructs used in Arabidopsis transformation, showing the nucleotide (top) and amino acid (bottom) sequences of the native and mutated uORF. FS, Frame shift; black, 35S CaMV promoter; white, AtHB1 $5^{\prime}$ UTR; light blue, CPuORF33; gray, GUS ORF; stars, single-base modification; arrowhead, insertion introduced. B, Illustrative photograph of 20-d-old leaves transformed with the indicated constructs and analyzed by GUS histochemisty.  

# The Activity of CPuORF33 Is Tissue Specific  

In view of the described observations, we wondered whether CPuORF33 activity depended on the tissue, developmental stage, or growth condition. To address this question, Col-0 plants transformed with 35S:nativeuORF:GUS or 35S:GUS were analyzed in detail by histochemistry (Fig. 5A). Cotyledons of 5-d-old seedlings, fully developed leaves, and inflorescences of plants transformed with 35S:native-uORF:GUS, grown under an LDP, clearly showed CPuORF33 repression (Fig. 5). However, leaf primordia and roots of the same plants exhibited the same GUS staining as those transformed with 35S:GUS. These observations indicated that the action exerted by CPuORF33 was tissue specific. Similar results were obtained when plants transformed with 35S:native-uORF:GUS were compared with plants transformed with 35S:FS-uORF:GUS (data not shown). Ribosome sequencing assays performed in roots and aerial tissue by Hsu et al. (2016) were consistent with our observations (Supplemental Fig. S11).  

This phenomenon could be the result of an alternative splicing event or the presence of a secondary transcription start site (TSS) that prevents the inclusion of the complete CPuORF in the mature mRNA. Indeed, the inspection of publicly available TSS results from whole Arabidopsis roots using the paired-end analysis of TSS protocol indicated that the AtHB1 locus presented a second TSS with a weak peak pattern between locus positions 194 and 403 (Peak_40644; Supplemental Fig. S12A; Morton et al., 2014). This TSS would exclude from the mRNA the CPuORF start codon, which is at position 163. To further investigate this hypothesis, we used published RNA sequencing data to compare the mRNA profiles between shoots and roots and found no strong indication of alternative splicing and disparate results for a secondary TSS (Supplemental Fig. S12, B–E). To test the secondary TSS hypothesis in our conditions, the tissue differential presence of a shorter AtHB1 transcript excluding the CPuORF start codon was tested by RT-qPCR in roots and shoots using two sets of oligonucleotides (Supplemental Fig. S12A). A within-tissue ratio of amplification products was calculated and compared between shoots and roots, but the results indicated no significant differences (data not shown). In consequence, although there is some evidence supporting the existence of a secondary TSS, this would not be the key mechanism explaining the tissue specificity observed for the activity of the CPuORF.  

![](images/2baf6a47e958b6853c94cb535a2ab5a6a482983f1cb7cba483e6275b523044ba.jpg)  
Figure 5. $\mathsf{C P u O R F33}$ repression action depends on the tissue and environmental conditions. Illustrative photographs show organs/tissues of plants revealed by GUS histochemisty. A, Five-day-old seedling transformed with 35S:GUS grown under long-day photoperiod (LDP). Bar $=2~\mathsf{m m}$ . B, Five-day-old seedling transformed with 35S:native-uORF:GUS grown under LDP. Bar $=2~\mathsf{m m}$ . C, Hypocotyl detail of B. $3\mathrm{ar}=500\,\mu\mathrm{m}$ . D, Ten-day-old seedling transformed with 35S:GUS grown under LDP. Bar $=2~\mathsf{m m}$ . E, Tenday-old seedling transformed with 35S:native-uORF:GUS grown under LDP. Bar $=2\;\mathrm{mm}$ . F, Hypocotyl detail of E. $\mathsf{B a r}=500\,\mu\mathsf{m}$ . G, Root detail of E. Bar $=200\,\mu\mathrm{m}$ . H, Inflorescence of a plant transformed with 35S:GUS. Bar $=5\ \mathrm{mm}$ . I, Inflorescence of a plant transformed with 35S:nativeuORF:GUS. Bar $=5\;\mathrm{mm}$ .  

# CPuORF33 Repression Activity Is Triggered by Light in Aerial Tissues  

As mentioned above, CPuORF33 was active in aerial tissues and inactive in roots (Fig. 5). Thus, an attractive hypothesis was that CPuORF action could be triggered somehow by light. In order to test this, 35S:nativeuORF:GUS transformed plants were grown during 6 d in complete darkness or under the LDP and GUS activity was evaluated by histochemistry. As shown in Figure 6A, cotyledons of seedlings grown in darkness were completely stained, indicating a lack of uORF repression under this condition, whereas the opposite scenario was obtained under the LDP. Moreover, the effect of illumination was not reverted in these plants after 2 d of darkness (Fig. 6B). Similar results were obtained using 15-d-old plants placed in darkness for an additional 5 d (Supplemental Fig. S13).  

To determine whether CPuORF33 repression activity was the result of the illumination quality, 6-dold seedlings transformed with 35S:native-uORF:GUS were grown under the LDP exposed to blue, red, or white light. All these treatments resulted in similar observations (i.e. CPuORF33 actively repressed GUS activity in aerial tissues; Supplemental Fig. S13). Additional treatments with abscisic acid, indole-3-acetic acid, and GAs also were carried out on dark-grown seedlings, indicating that none of these hormones was able to modify CPuORF33 repression (data not shown). Considering the hypothesis of a chloroplast signal as the switch to activate CPuORF33, 35S:native-uORF: GUS seedlings grown in complete darkness over 6 d were treated with dichlorophenyl dimethylurea (DCMU) and transferred to light conditions for an additional $24\,\mathrm{~h~}$ . Notably, the repression action of CPuORF33 was avoided, as GUS activity was clearly detected in cotyledons (Fig. 6C), indicating that a signal from coupled chloroplasts is responsible for initiating CPuORF33 activity.  

# The Homologous Maize CPuORF Also Functions as a Translational Repressor  

Monocot plants exhibit an insertion of seven amino acids in the C termini of the $C\mathrm{PuORF},$ , which makes them longer than those of dicot plants (Fig. 1A). To evaluate whether this longer peptide resulted in a different function, we decided to clone a monocot uORF and analyze its activity. To this end, the maize $5^{\prime}$ UTR of the AtHB1 homolog (ZmHB115) was cloned between the constitutive 35S CaMV promoter and the GUS reporter gene. Arabidopsis plants were then transformed and analyzed by GUS histochemistry (Fig. 7). The results indicated that the maize uORF represses GUS translation in the aerial portions of the plant, similar to the inhibition seen for the Arabidopsis CPuORF33. Notably, like its Arabidopsis homolog, the maize uORF also exhibited tissue-specific activity and did not function as a repressor in roots (Fig. 7).  

Aiming to elucidate if this uORF is active in maize, data obtained from ribosome footprinting analyses performed with samples of 14-d-old maize seedlings were examined (Supplemental Fig. S14; Lei et al., 2015). As expected, translation seemed to be stalled in the uORF region, and fewer ribosomes were detected in the mORF (Supplemental Fig. S14), supporting both the proposed ribosome stalling mechanism and the conservation between species of the uORF’s function.  

# The Absence of a Tightly Regulated Expression of AtHB1 Causes Severe Deleterious Effects  

It was surprising to discover such a sophisticated mechanism repressing AtHB1 expression, especially because we were not able to detect strong differential phenotypes in athb1 mutants and, certainly, no lethality in Col-0 plants transformed with 35S:AtHB1 (Capella et al., 2015b).  

To understand such phenomena, we decided to further analyze rdr6-12 plants transformed with PromAtHB1mut:AtHB1, in which neither ribosome stalling nor small RNA silencing was possible. As controls, we used rdr6-12 plants transformed with an empty vector or with PromAtHB1:AtHB1.  

![](images/a8c87156d3504bcee2ce2fc910e72613d2998d158289fde74e7098d8772d82cd.jpg)  
Figure 6. Light-dependent CPuORF33 repression action in cotyledons is not reverted in darkness but avoided by DCMU. Illustrative photographs show seedlings transformed with 35S:GUS or 35S:native-uORF:GUS and revealed by GUS histochemistry. A, Left, 6-d-old seedling transformed with 35S:nativeuORF:GUS grown under LDP. Center, 6-d-old seedling transformed with 35S: GUS grown in darkness. Right, 6-d-old seedling transformed with 35S:nativeuORF:GUS grown in darkness. Bars $=$ $2\,\mathsf{\ m m}$ . B, Left, 8-d-old 35S:nativeuORF:GUS seedling grown under LDP. Right, 6-d-old 35S:native-uORF:GUS seedling grown under LDP and then transferred for an additional $2\,\mathrm{~d~}$ to darkness. Bar $\v{r}=2\ \mathrm{mm}$ . C, Left, 6-dold 35S:native-uORF:GUS seedling grown in darkness and treated with ethanol (EtOH) during $24\,\mathrm{{h}}$ under LDP. Right, 6-d-old 35S:native-uORF:GUS seedling grown in darkness and treated during $24~\mathrm{h}$ with $50~\mu\mathrm{M}$ DCMU under LDP. Bars $=2~\mathsf{m m}$ . Insets show cotyledons in more detail.  

Hypocotyl length was analyzed in these plants, since this developmental trait is affected by AtHB1 (Capella et al., 2015b). As expected, rdr6-12 plants transformed with PromAtHB1mut:AtHB1 showed longer hypocotyls than the other transformed plants (Fig. 8A). In parallel, plants from the three genotypes were grown on soil under standard conditions. PromAtHB1mut:AtHB1 plants exhibited compact rosettes, a delay in bolting, and, more importantly, a strongly altered flower morphology. Pistils were reduced, anthers were extremely short and nondehiscent, and siliques were small and had fewer or no seeds. In several lines, the analysis of a second generation was not possible because T1 plants were sterile (Fig. 8B). Altogether, these results could explain why such a sophisticated mechanism acts to repress the overexpression of this TF (i.e. unregulated increased expression of AtHB1 led to an infertile, delayed, and aberrant phenotype).  

# DISCUSSION  

The adaptation of plants throughout evolution involved the loss and the acquisition of genome DNA sequences including the conservation of key elements. Many such conserved regulatory elements must be fundamental for plant development, reproduction, and/or survival. Here, we demonstrated that a highly conserved genetic element, the CPuORF33, is important to avoid plant sterility.  

![](images/23462b8a33f5cfe9f28403cc9845d26166eb8790f481506149b76deca25f7627.jpg)  
Figure 7. The uORF of the maize AtHB1 homolog functions as a translational repressor. Illustrative photographs show 14-d-old plants revealed by GUS histochemisty. At left, Arabidopsis plants transformed with 35S:GUS grown under LDP. At right, three independent lines of Arabidopsis plants transformed with 35S:maize59UTR:GUS grown under LDP. Bars $=500\,\mu\mathrm{m}$ .  

Although uORFs are present in a considerable number of mRNAs (Hayden and Bosco, 2008; Nagalakshmi et al., 2008; Calvo et al., 2009), it is remarkable that only a small portion of these varied genetic elements has been conserved between species, and only a few members within this group were assigned a function. Here, we show that CPuORF33 negatively regulates AtHB1 translation during plant development and that this regulatory mechanism is conserved in maize. Hence, it is tempting to suggest that a similar scenario occurs in other species with AtHB1 homologs. Notably, in all the available DNA sequences encoding AtHB1 homologs that have known $5^{\prime}$ UTRs, either from monocot or dicot species, CPuORF33 was identified. Considering that the number of sequences increases continuously, it would be interesting to repeat this analysis in the near future.  

Besides the high conservation of the nucleotide sequence, it is important to note that the amino acid sequence is even more conserved, indicating that the peptide, and not the RNA, is the active element. This suggestion also was supported by the absence of overlap between CPuORF33 and the mORF. This characteristic is relevant to allow ribosome reinitiation and the translation of the mORF. Overlap of the CPuORF and the mORF was described as being linked to NMD regulating the abundance of many gene transcripts involved in plant development, including TFs, RNA processing factors, and stress response genes (Kalyna et al., 2012).  

According to previous reports, the regulation exerted by eukaryotic uORFs on transcript levels of the mORF occurs through several different mechanisms of action, but most uORFs exert their effects in a sequenceindependent manner (Calvo et al., 2009). In contrast, certain uORFs control translation of the mORF in a peptide sequence-dependent manner (Ito and Chiba, 2013; von Arnim et al., 2014). Among the possible repression mechanisms exerted by uORFs, NMD is the one acting in the regulation of AdoMetDC1, which causes polyamine-responsive ribosomal arrest, the SAC51 gene encoding a basic helix-loop-helix TF, and AtMHX, which encodes a vacuolar magnesium-zinc/proton exchanger (Bender and Fink, 1998; Imai et al., 2006; Combier et al., 2008; Saul et al., 2009; Uchiyama-Kadokura et al., 2014).  

On the other hand, other mechanisms displayed by uORFs regulate RNA translation; among them are ribosome stalling (Wiese et al., 2004; Rahmani et al., 2009; Alatorre-Cobos et al., 2012), ribosome reinitiation (Wang and Wessler, 1998), a combination of both (Hanfrey et al., 2005), and others not well understood (Kwak and Lee, 2001). Among the possible mechanisms of CPuORF33 action, we were able to show that this particular element repressed AtHB1 translation in cis.  

During translation, ribosome movement along the mRNA molecule can be stopped either by a stable secondary structure in the mRNA or by the nascent translated peptide. Both cases can be observed in ribosome footprinting experiments as an evident increase in the number of reads in a particular region. The ribosome footprinting analyses for the $5^{\prime}$ UTR of $A t H B1$ showed a clear coverage peak ${\sim}40$ bp upstream of the uORF stop codon, suggesting the stalling of ribosomes in this region. This peak should not be confused with the one caused by the deceleration of ribosome movement during translational termination, which appears ${\sim}16$ pb upstream of the stop codon (Juntawong et al., 2014; Hou et al., 2016). Taking into account the high conservation observed in the amino acid sequences of CPuORF33 homologs, not so evident at the nucleotide level, it is tempting to speculate that the ribosome stalling is caused by the nascent peptide and not by a secondary structure in the mRNA. Supporting this conclusion, the repressive activity of the uORF was lost when frame-shift mutations were introduced, in both plants and yeast, even though there are only minor changes in the RNA sequence.  

In this work, we demonstrated the importance of the peptide structure of the CPuORF33 for in vivo translational repression, most likely by ribosome stalling. In this mechanism, the interaction between the polypeptide being synthesized and the ribosome tunnel can regulate the translation rate. The tunnel allows the formation of secondary structures like $\alpha$ -helix or zinc finger motifs (Nilsson et al., 2015). A recent report by Ebina and coworkers (2015) identified 16 novel uORFs in which the amino acids located at the C termini were crucial in determining their repressive action. In order to test the functionality of the CPuORF33 C terminus, its peptide sequence was changed by two point mutations: one located at amino acid 24 $\scriptstyle({\mathrm{H}}\to\Omega)$ and the second one deleting the stop codon, which adds 30 additional amino acids $(m u t{\bar{5}}^{\prime}U T R)$ . However, no differences were observed in plants transformed with 35S:mut59UTR:GUS compared with those transformed with the native $5^{\prime}$ UTR (data not shown). This indicates that CPuORF33 is more likely a class II uORF in which, according to Takahashi et al. (2012), the C terminus is not relevant for its action.  

![](images/216f1a3648edce2b64f255bbe802316769856ea0d8798e3e96ad923d8b0647e8.jpg)  
Figure 8. Expression levels of AtHB1 are fine-tuned due to the detrimental effects for plant reproduction and survival generated by its overexpression. A, Hypocotyl lengths of plants grown under a short photoperiod for $5\textup{d}$ . Control plants of Col-0 and rdr6-12 backgrounds are shown as well as the same plants transformed with PromAtHB1:AtHB1 (PromAtHB1: At1) and PromAtHB1mut:AtHB1 (PromAtHB1mut:At1). Two independent lines for each genotype are shown. An ANOVA test was performed, and pairwise differences were evaluated with Tukey’s posthoc test; different groups are marked with letters at the 0.05 significance level. B, Illustrative photographs of rdr6-12 plants transformed with PromAtHB1:AtHB1 and PromAtHB1mut:AtHB1. Rows are as follows: I, 25-d-old rosette leaves of control and transformed plants (four independent lines are shown for each genotype); II, front photographs of the same plants as in I; III, flowers of 40-dold plants; IV, siliques of the same plants.  

In contrast with our results, using an in vitro system, Hayashi et al. (2017) showed that CPuORF33 (called At3g01470 by the authors) arrests ribosomes in a peptide sequence-independent manner. A plausible explanation for this discrepancy could be that the sequence-dependent ribosome arrest activity needs a certain biomolecule to be absent in the in vitro system. An alternative explanation could be that, as in the in vitro assay, an N-terminal GST fusion protein was used, such that the 3D structure of the CPuORF33 could have been affected.  

There are some reports showing that the ribosome stalling mechanism involves small molecules like ascorbate, boron (as $\mathrm{H}_{3}\mathrm{BO}_{3}$ in solution), phosphocholine, or Suc (Rahmani et al., 2009; Alatorre-Cobos et al., 2012; Laing et al., 2015; Tanaka et al., 2016). Further studies will be necessary to reveal whether those or other molecules are necessary for CPuORF33 action. Nonetheless, even when unidentified molecules were necessary for AtHB1 repression, such molecules are normally present in plant leaves and flowers, since we observed the repression exerted by CPuORF33 in these tissues, especially when using stably transformed plants with the 35S:native-uORF:GUS and 35S:FS-uORF:GUS constructs, which make the analysis independent of transcriptional regulation. Moreover, and in view of the tissue-specific action of CPuORF33, one could speculate that such molecules, which also could be proteins, are not present in roots and apical meristems.  

Considering the differences between the roots and aerial parts of the plant, and also those between darkness and light, chloroplast functionality was assessed for its capacity to regulate CPuORF33 using DCMU (a known photosynthesis uncoupler). This treatment was able to inhibit CPuORF repressor action (Fig. 6). Additionally, several molecules related to photosynthesis, including sugars and hormones, were tested with negative results. Further investigation will be needed to reveal which is the chloroplast signal responsible for this effect.  

Another mechanism that might be potentially responsible for the tissue-specific action of CPuORF33 was considered. A secondary TSS in AtHB1 was found by Morton et al. (2014) and was defined as a region located downstream of the uORF start codon (Supplemental Fig. S12A). Transcripts starting at this region, therefore, would lack the full uORF, preventing the stalling of ribosomes and allowing the uninhibited expression of the mORF. However, the comparison of RNA transcript profiles between roots and shoots using publicly available data was not conclusive, and our qPCR assays comparing these tissues did not support this mechanism. In consequence, the presence of a secondary TSS would not be able to explain the tissue specificity of uORF activity. Nonetheless, we cannot rule out that this TSS could be functional in bypassing the repressive action of the uORF under certain conditions, as reported for other uORFs (Pumplin et al., 2016).  

The second mechanism repressing AtHB1 expression described in this work is mediated by small RNAs, although this might function only when it is expressed as a transgene. It is already known that, above a certain threshold, the expression of several transgenes, like GUS, $G F P$ , or Streptomycin Phosphotransferase, is silenced by such a mechanism, being threshold dependent on the gene (Schubert et al., 2004; Rajeevkumar et al., 2015). Two different mechanisms acting to silence transgenes have been described: transcriptional gene silencing and posttranscriptional gene silencing. In the first, a DNA segment encoding a certain mRNA is methylated, inhibiting transcription, whereas in the second, the mRNA is degraded by small interfering RNA after the formation of the mRNA (Matzke et al., 2001). Posttranscriptional gene silencing occurs during plant development and after meiosis initiation, whereas transcriptional gene silencing occurs during meiosis and is heritable (Vaucheret and Fagard, 2001). Since the repression of AtHB1 in transgenic plants takes place in the second generation, transcriptional gene silencing is likely the silencing mechanism. However, it is unusual that such silencing was displayed when the overexpression was controlled by a native promoter, since this scenario has been observed only with constitutive promoters like the 35S CaMV. These observations indicate that AtHB1 overexpression is tightly regulated to avoid expression above the threshold and that this threshold is very close to endogenous transcript levels. Moreover, AtHB1 was not silenced when the construct used to perform the transformation did not have the AtHB1 coding sequence (data not shown), indicating that the silencing is caused by AtHB1 transcript or protein levels but not by the promoter itself.  

We can conclude that the CPuORF33 present in the $5^{\prime}$ UTR of the Arabidopsis HD-Zip TF AtHB1 and its homologs in at least 43 other species exerts a strong tissue- and condition-specific regulation at the translational level by ribosome stalling in order to avoid an aberrant phenotype.  

# MATERIALS AND METHODS  

# Plant Material and Growth Conditions  

Arabidopsis (Arabidopsis thaliana) plants were grown directly on soil in a growth chamber at $22^{\circ}\mathbf{C}$ to $24^{\circ}\mathrm{C}$ under LDP $^\mathrm{16}\,\mathrm{h}$ light), at an intensity of approximately $120\ \mu\mathrm{mol\m}^{-2}\ \mathrm{s}^{-1}.$ , in $8\mathrm{-}\,\times\,7\mathrm{-cm}$ pots. Short-photoperiod conditions were used only to evaluate hypocotyl length as indicated in the corresponding figure legend.  

Arabidopsis ecotype Col-0, and the mutants athb1-1 (SALK_123216C), upf3-1 (SALK_025175), and upf1-5 (SALK_112922), all in the Col-0 ecotype background, were obtained from the Arabidopsis Biological Resource Center (http://www. arabidopsis.org). Mutant rdr6-12 seeds (Peragine et al., 2004) were kindly provided by Dr. Pablo Manavella from the Instituto de Agrobiotecnología del Litoral. $p K G W F S7$ PromAtHB1:GUS plants were described previously (Capella et al., 2015b). Homozygous lines were selected after two complete growth cycles.  

# DCMU Treatments  

Seeds were surface sterilized and then plated in petri dishes with $0.5\times$ Murashige and Skoog medium supplemented with vitamins (PhytoTechnology Laboratories). Plates were placed at $4^{\circ}\mathrm{C}$ during $2\textup{d}$ and transferred to the growth chamber $(22^{\circ}C{-}24^{\circ}C$ under LDP) for the periods indicated in the corresponding figure legends. Another group of plates was transferred to the same chamber but inside a dark box. For DCMU treatments, these plants were vacuum infiltrated with $50\ \mu\mathrm{M}$ DCMU solution; then, the liquid reagent was discarded and plants were placed under LDP for an additional $24\,\mathrm{{h}}$ .  

# Genetic Constructs  

pCambia HA-AtHB1 was described previously (Capella et al., 2015b).  

# PromAtHB1mut:GUS  

ThisconstructwasassembledbyPCRamplificationandoverlappingwiththe oligonucleotides (Higuchi et al., 1988) listed in Supplemental Table S2 using as probe the pKGWFS7 PromAtHB1:GUS construct. The amplification PCR product was cloned in pBluescript $\mathrm{SK-}$ . This last construct was restricted with BglII and HindIII and finally inserted in pKGWFS7 PromAtHB1:GUS, replacing the wild-type sequence. The correct insertion was verified by sequencing.  

# PromAtHB1:AtHB1 and PromAtHB1mut:AtHB1  

The native and mutated versions of the AtHB1 promoter were amplified using specific oligonucleotides (Supplemental Table S2) and pKGWFS7 PromAtHB1:GUS and pKGWFS7 PromAtHB1mut:GUS as templates, respectively. The PCR products were cloned into the SalI and XbaI sites of $p M T L22.$ , and then the obtained clones were digested with BamHI and XbaI. Finally, these products were cloned into the BglII and XbaI sites of pCambia HA-AtHB1, replacing the 35S CaMV.  

# 35S:Native-uORF:GUS  

The AtHB1 $5^{\prime}$ UTR was amplified by PCR using as template the pCambia PromAtHB1:AtHB1 clone and specific oligonucleotides (Supplemental Table S2). The amplification product was then cloned into the XbaI and BamHI sites of $p B I121$ .  

# 35S:FS-uORF:GUS  

The indicated mutations were introduced by PCR amplification and overlapping with the oligonucleotides listed in Supplemental Table S2 and using as probe the pCambia PromAtHB1:At1 clone. The PCR product was cloned into the XbaI and BamHI sites of $p B I121$ .  

# 35S:ZmHB115-59UTR:GUS  

The ZmHB115 $5^{\prime}$ UTR was amplified by PCR using genomic DNA as template and specific oligonucleotides (Supplemental Table S2). The amplification product was then cloned into the XbaI and BamHI sites of $p B I I21$ .  

# pADH::yeGFP and pADH::NLS::yeGFP  

The yeast enhanced GFP (yeGFP), with or without the SV40 nuclear localization signal (NLS), was amplified by PCR using as template the $p Y M25$ vector and specific oligonucleotides (Supplemental Table S2). The amplification products were then cloned into the BamHI and SalI sites of YCplac22 pADH.  

$p A D H{::}u O R F{::}y e G F P$ and pADH::FS-uORF::yeGFP mutant constructs, native-uORF and FS-uORF, were amplified using specific oligonucleotides (Supplemental Table S2) and the 35S:native-uORF:GUS and 35S:FS-uORF:GUS clones as probes. By Gibson cloning (New England Biolabs), the PCR products were cloned into $p A D H{::}y e G F P$ , previously restricted with BamHI. Finally, the indicated mutations (FS1, FS2, and FS3) were introduced by QuikChange II Site-Directed Mutagenesis (Agilent), using $p A D H{\mathrel{:}}{\mathrel{:}}F S-$ -uORF::yeGFP as template.  

# Stable Arabidopsis Plant Transformation  

Transformed Agrobacterium tumefaciens strain LBA4404 was used to obtain transgenic Arabidopsis plants by the floral dip procedure (Clough and Bent, 1998). Transformed plants were selected on the basis of their specific resistance in petri dishes with $0.5\times$ Murashige and Skoog medium supplemented with vitamins (PhytoTechnology Laboratories) and the appropriate selector chemical $(50\mathrm{~mg~L~}^{-1}$ kanamycin or $25\mathrm{~mg~L^{-1}}$ hygromycin). The seeds were surface sterilized, plated, and, after $2\textup{d}$ of incubation at $4{}^{\circ}\mathrm{C},$ placed in a growth chamber at $22^{\circ}\mathbf{C}$ to $24^{\circ}\mathbf{C}$ .  

The insertion of each transgene was checked by PCR using genomic DNA as template with specific oligonucleotides listed in Supplemental Table S2. Three or four positive independent lines for each construct were further reproduced, and homozygous T3 and T4 plants were used in order to analyze the expression levels of the specific transgene and plant phenotypes. T1 plants were used in a specific experiment as indicated in the corresponding figure legend.  

# RNA Extraction and Analysis  

Total RNA for transcript level evaluation by RT-qPCR was isolated from Arabidopsis leaves using the Trizol reagent (Invitrogen) according to the manufacturer’s instructions. One microgram of RNA was reverse transcribed using oligo $(\mathrm{dT})_{18}$ and Moloney murine leukemia virus reverse transcriptase II (Promega). For the alternative TSS assay, a different oligonucleotide (AtHB1qPCRR) was used for RT. qPCR was performed with the $\mathrm{Mx}3000\mathrm{P}$ Multiplex qPCR system (Stratagene) in a $20{-}\mu\mathrm{L}$ final volume containing $2\,\mu\mathrm{L}$ of SYBR Green $(4\!\times\!),8$ pmol of each primer, $2\,\mathrm{mM}\,\mathrm{MgCl}_{2},10\,\mu\mathrm{L}$ of a 1:15 dilution of the RT reaction, and $0.1~\mu\mathrm{L}$ of Taq Platinum (Invitrogen). Fluorescence was measured at $72{}^{\circ}\mathbf{C}$ during 40 cycles. Specific primers were designed (Supplemental Table S2). Quantification of mRNA levels was performed by normalization with the ACTIN transcript levels (ACTIN2 and ACTIN8) according to the DDCt method (Pfaffl, 2001). All the reactions were performed with, at least, three replicates. For a better visualization of the results, the $y$ axes of the figures containing transcript evaluation are represented on a logarithmic scale.  

# Histochemical GUS Staining  

GUS staining was performed as described by Jefferson et al. (1987). Plants were immersed in GUS staining buffer $\mathrm{1\,mM}\,5$ -bromo-4-chloro-3-indolyl-GlcA in $100\,\mathrm{\mM}$ sodium phosphate, pH 7, $0.1\%$ $(\mathrm{v/v}$ ) Triton X-100, and $100\,\mathrm{\mM}$ potassium ferrocyanide), vacuum was applied for $5\,\mathrm{{min}}$ , and then plants were incubated at $37^{\circ}\mathbf{C}$ for $12~\mathrm{h}$ . Chlorophyll was cleared from the plant tissues by immersion in $70\%$ ethanol.  

# Phenotype Analyses  

Plants were grown as described above and photographed using a Panasonic DMC-FH4 camera. Flowers and siliques were detached and photographed with a Nikon SMZ800 stereomicroscope. Hypocotyl length measurements were carried out as described (Capella et al., 2015b).  

# In Silico Sequence Analysis  

To retrieve nucleotide sequences, initially a BLASTP search was conducted with the full-length sequence of the AtHB1 TF against the National Center for Biotechnology Information nonredundant protein sequence database (default parameters were used; January 18, 2016; Altschul et al., 1990). Sequence redundancy was checked using the skipredundant program of the EMBOSS package (Rice et al., 2000), and the results were manually inspected and curated. After this filtering, full mRNA-containing hits were selected for further analysis.  

The amino acid sequence of CPuORF33 was analyzed for the prediction of secondary structure using Jpred 4 (http://www.compbio.dundee.ac.uk/ jpred4; Drozdetskiy et al., 2015) and for known motifs using hmmscan (https://www.ebi.ac.uk/Tools/hmmer/search/hmmscan; Finn et al., 2015), including all HMM databases (Pfam, TIGRFAM, Gene3D, Superfamily, and PIRSF).  

The uORF nucleotide and amino acid sequences were aligned with ClustalW (Larkin et al., 2007; Goujon et al., 2010) using Multiple Alignment Mode, iterated in each step, and the following parameters: gap extension, 0; gap opening, 15; negative matrix, off; DNA transition weight, 0.5; delay divergent seq, 30; protein weight matrix, Gonnet series; DNA weight matrix, IUB. Identity and IUB quality were used for protein and DNA analysis, respectively.  

A maximum likelihood phylogenetic tree was constructed using the ClustalW alignments, the $\mathrm{JTT}\mathrm{+}\mathrm{I}{+}\mathrm{G}$ model (Jones et al., 1992; Reeves, 1992; Yang, 1993), and 100 bootstrap repeats.  

# Ribosome Footprint Analysis  

Ribosome footprint sequence reads were obtained from Juntawong et al. (2014; SRX345243, SRX345250, SRX345242, and SRX345246), Merchante et al. (2015; SRX976546, SRX976568, SRX976713, and SRX976714), Lei et al. (2015; SRX845439, SRX845455, SRX847137, and SRX847138), and Hsu et al. (2016; SRX1756756, SRX1756757, SRX1756758, SRX1756759, SRX1756760, SRX1756761, SRX1756762, SRX1756763, SRX1756764, SRX1756765, SRX1756766, and SRX1756767). Reads of nonstressed plants were used in this analysis. The coverage was computed for the entire read in RNA sequencing samples and for nucleotide 13 in each read for ribosome sequencing samples. Translation effi- ciency was calculated as the relationship between the read count in the ribosome sequencing sample and the read count in the total RNA sample for each ORF.  

# RNA Profile Analysis  

The raw reads from the studies analyzed were retrieved from the Gene Expression Omnibus repository. The corresponding accession numbers are GSE68560 (Mancini et al., 2016), GSE61545 (Liu et al., 2016), and GSE87760 (W. Schmidt and L. Grilletes, unpublished data). Reads were first processed to remove adapters and low-quality bases using Trimmomatic version 0.36 (Bolger et al., 2014) with the suggested options: LEADING:3 TRAILING:3, SLIDINGWINDOW:4:15 MINLEN:36, with MAXINFO:90:0.4 and removing Illumina adapter sequences using the ILLUMINACLIP option. The quality of reads before and after trimming was evaluated with FastQC (http://www. bioinformatics.babraham.ac.uk/projects/fastqc/).  

Processed reads were mapped to the Arabidopsis genome (TAIR10; Lamesch et al., 2012) using Tophat2 version 2.1.1 (Kim et al., 2013) with the default settings. Duplicate reads were removed with MarkDuplicates from the picard toolkit version 2.7.0 (http://picarDsourceforge.net/). The results were inspected graphically with IGV (Thorvaldsdóttir et al., 2013), which was also used to obtain the Sashimi plots. The comparison of RNA profiles was carried out with RNAprof software version 1.2.6 (Tran et al., 2016) only for the AtHB1 locus.  

# Yeast Cell Culture, Transformation, and Immunoblotting  

Saccharomyces cerevisiae DF5 MATa cells were grown and transformed as described (Capella et al., 2014). Cells were cultured to exponential growth in synthetic minimal medium lacking Trp; one $\mathrm{OD}_{600}$ was collected, and total cell protein extracts were prepared by TCA precipitation. Proteins were resolved on NuPAGE $12\%$ gels (Invitrogen) and analyzed by standard immunoblotting techniques using mouse monoclonal antibodies against GFP (B-2; Santa Cruz Biotechnology) and Pgk1 (22C5; Invitrogen) and horseradish peroxidase-rabbit anti-mouse antibody (Invitrogen).  

# Accession Numbers  

Accession numbers are as follows: AtHB1 (At3G01472.1) and ZmHB115 (GRMZM2G021339).  

# Supplemental Data  

The following supplemental materials are available.   
Supplemental Figure S1. Nucleotide sequence alignment of the coding sequences of the peptides listed in Figure 1B.   
Supplemental Figure S2. The context of the second ATG codon fits the Kozak rule better.   
Supplemental Figure S3. The length of the uORF is the most conserved feature, differing only between monocotyledonous and dicotyledonous plants.   
Supplemental Figure S4. AtHB1 overexpression is impaired by a mechanism involving small interfering RNA.   
Supplemental Figure S5. AtHB1 expression is only impaired while AtHB1 is overexpressed.   
Supplemental Figure S6. The negative regulation exerted by CPuORF33 in not dependent on small RNAs.   
Supplemental Figure S7. CPuORF33 action is not mediated by NMD.   
Supplemental Figure S8. Transgenic expression of CPuORF33 does not affect the expression of endogenous AtHB1.   
Supplemental Figure S9. Comparative ribosome footprinting profile of AtHB1 and AtHB13 transcripts.   
Supplemental Figure S10. CPuORF33 represses translation in a sequencedependent manner in the heterologous yeast system.   
Supplemental Figure S11. Comparative ribosome footprinting profile of AtHB1 transcripts in root versus shoots.   
Supplemental Figure S12. AtHB1 potentially has a secondary TSS that could explain the activity of the differential CPuORF in certain cases.   
Supplemental Figure S13. Histochemical detection of GUS in 35S:nativeuORF:GUS plants grown under different light regimes.   
Supplemental Figure S14. Ribosome footprinting profile of the transcripts of the maize AtHB1 homolog ZmHB115.   
Supplemental Table S1. Species used in the bioinformatic analysis, and the accession numbers of the corresponding sequences.   
Supplemental Table S2. Oligonucleotides used in this work.  

# ACKNOWLEDGMENTS  

We thank Dr. Federico Ariel for critical reading of the article.  

Received July 31, 2017; accepted September 25, 2017; published September 27, 2017.  

# LITERATURE CITED  

Alatorre-Cobos $\mathbf{F},$ Cruz-Ramírez A, Hayden CA, Pérez-Torres CA, Chauvin $\mathbf{AL},$ Ibarra-Laclette E, Alva-Cortés E, Jorgensen RA, HerreraEstrella L (2012) Translational regulation of Arabidopsis XIPOTL1 is modulated by phosphocholine levels via the phylogenetically conserved upstream open reading frame 30. J Exp Bot 63: 5203–5221   
Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol 215: 403–410   
Aoyama T, Dong CH, Wu Y, Carabelli M, Sessa G, Ruberti I, Morelli G, Chua NH (1995) Ectopic expression of the Arabidopsis transcriptional activator Athb-1 alters leaf cell fate in tobacco. Plant Cell 7: 1773–1785   
Arce $\mathbf{AL},$ Raineri J, Capella M, Cabello JV, Chan RL (2011) Uncharacterized conserved motifs outside the HD-Zip domain in HD-Zip subfamily I transcription factors: a potential source of functional diversity. BMC Plant Biol 11: 42   
Bender J, Fink GR (1998) A Myb homologue, ATR1, activates tryptophan gene expression in Arabidopsis. Proc Natl Acad Sci USA 95: 5655–5660   
Bolger AM, Lohse M, Usadel B (2014) Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30: 2114–2120   
Calvo SE, Pagliarini DJ, Mootha VK (2009) Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc Natl Acad Sci USA 106: 7507–7512   
Capella M, Ré DA, Arce AL, Chan RL (2014) Plant homeodomain-leucine zipper I transcription factors exhibit different functional AHA motifs that selectively interact with TBP or/and TFIIB. Plant Cell Rep 33: 955– 967   
Capella M, Ribone PA, Arce AL, Chan RL (2015a) Homeodomain–leucine zipper transcription factors: structural features of these proteins, unique to plants. In DH González, ed, Plant Transcription Factors: Evolutionary, Structural and Functional Aspects. Elsevier, Amsterdam, pp 113– 126   
Capella M, Ribone PA, Arce $\mathbf{AL},$ Chan RL (2015b) Arabidopsis thaliana HomeoBox 1 (AtHB1), a Homedomain-Leucine Zipper I (HD-Zip I) transcription factor, is regulated by PHYTOCHROME-INTERACTING FACTOR 1 to promote hypocotyl elongation. New Phytol 207: 669–682   
Clough SJ, Bent AF (1998) Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana. Plant J 16: 735–743  
Combier JP, de Billy F, Gamas P, Niebel A, Rivas S (2008) Transregulation of the expression of the transcription factor MtHAP2-1 by a uORF controls root nodule development. Genes Dev 22: 1549–1559   
Drozdetskiy A, Cole C, Procter J, Barton GJ (2015) JPred4: a protein secondary structure prediction server. Nucleic Acids Res 43: W389–W394   
Ebina I, Takemoto-Tsutsumi M, Watanabe S, Koyama H, Endo Y, Kimata K, Igarashi T, Murakami K, Kudo R, Ohsumi A, et al (2015) Identifi- cation of novel Arabidopsis thaliana upstream open reading frames that control expression of the main coding sequences in a peptide sequencedependent manner. Nucleic Acids Res 43: 1562–1576   
Finn RD, Clements J, Arndt W, Miller BL, Wheeler TJ, Schreiber F, Bateman A, Eddy SR (2015) HMMER web server: 2015 update. Nucleic Acids Res 43: W30–W38   
Goujon M, McWilliam H, Li W, Valentin F, Squizzato S, Paern J, Lopez R (2010) A new bioinformatics analysis tools framework at EMBL-EBI. Nucleic Acids Res 38: W695–W699   
Guerrero-González $\mathbf{M}\mathbf{L},$ Ortega-Amaro MA, Juárez-Montiel M, JiménezBremont JF (2016) Arabidopsis polyamine oxidase-2 uORF is required for downstream translational regulation. Plant Physiol Biochem 108: 381–390   
Guerrero-González ML, Rodríguez-Kessler M, Jiménez-Bremont JF (2014) uORF, a regulatory mechanism of the Arabidopsis polyamine oxidase 2. Mol Biol Rep 41: 2427–2443   
Hanfrey C, Elliott KA, Franceschetti M, Mayer MJ, Illingworth C, Michael AJ (2005) A dual upstream open reading frame-based autoregulatory circuit controlling polyamine-responsive translation. J Biol Chem 280: 39229–39237   
Hayashi N, Sasaki S, Takahashi H, Yamashita $\mathbf{Y,}$ Naito S, Onouchi H (2017) Identification of Arabidopsis thaliana upstream open reading frames encoding peptide sequences that cause ribosomal arrest. Nucleic Acids Res 45: 8844–8858   
Hayden CA, Bosco G (2008) Comparative genomic analysis of novel conserved peptide upstream open reading frames in Drosophila melanogaster and other dipteran species. BMC Genomics 9: 61   
Hayden CA, Jorgensen RA (2007) Identification of novel conserved peptide uORF homology groups in Arabidopsis and rice reveals ancient eukaryotic origin of select groups and preferential association with transcription factor-encoding genes. BMC Biol 5: 32   
Henriksson E, Olsson ASB, Johannesson H, Johansson H, Hanson J, Engström P, Söderman E (2005) Homeodomain leucine zipper class I genes in Arabidopsis: expression patterns and phylogenetic relationships. Plant Physiol 139: 509–518   
Higuchi $\mathbf{R},$ Krummel B, Saiki RK (1988) A general method of in vitro preparation and specific mutagenesis of DNA fragments: study of protein and DNA interactions. Nucleic Acids Res 16: 7351–7367   
Hofer J, Turner L, Moreau C, Ambrose M, Isaac P, Butcher S, Weller J, Dupin A, Dalmais M, Le Signor $\mathbf{C},$ et al (2009) Tendril-less regulates tendril formation in pea leaves. Plant Cell 21: 420–428   
Hou CY, Lee WC, Chou HC, Chen AP, Chou SJ, Chen HM (2016) Global analysis of truncated RNA ends reveals new insights into ribosome stalling in plants. Plant Cell 28: 2398–2416   
Hsu PY, Calviello L, Wu HL, Li FW, Rothfels CJ, Ohler U, Benfey PN (2016) Super-resolution ribosome profiling reveals unannotated translation events in Arabidopsis. Proc Natl Acad Sci USA 113: E7126–E7135   
Imai A, Hanzawa Y, Komura M, Yamamoto KT, Komeda Y, Takahashi T (2006) The dwarf phenotype of the Arabidopsis acl5 mutant is suppressed by a mutation in an upstream ORF of a bHLH gene. Development 133: 3575–3585   
Ingolia NT, Ghaemmaghami S, Newman JRS, Weissman JS (2009) Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. Science 324: 218–223   
Ito K, Chiba S (2013) Arrest peptides: cis-acting modulators of translation. Annu Rev Biochem 82: 171–202   
Ivanov IP, Atkins JF, Michael AJ (2010) A profusion of upstream open reading frame mechanisms in polyamine-responsive translational regulation. Nucleic Acids Res 38: 353–359   
Jefferson RA, Kavanagh TA, Bevan MW (1987) GUS fusions: betaglucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO J 6: 3901–3907   
Jones DT, Taylor WR, Thornton JM (1992) The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci 8: 275– 282   
Jorgensen RA, Dorantes-Acosta AE (2012) Conserved peptide upstream open reading frames are associated with regulatory genes in angiosperms. Front Plant Sci 3: 191   
Juntawong P, Girke T, Bazin J, Bailey-Serres J (2014) Translational dynamics revealed by genome-wide profiling of ribosome footprints in Arabidopsis. Proc Natl Acad Sci USA 111: E203–E212   
Kalyna M, Simpson CG, Syed NH, Lewandowska D, Marquez Y, Kusenda B, Marshall J, Fuller J, Cardle $\mathbf{L},$ McNicol J, et al (2012) Alternative splicing and nonsense-mediated decay modulate expression of important regulatory genes in Arabidopsis. Nucleic Acids Res 40: 2454–2469   
Katayama $\mathbf{H},$ Iwamoto $\mathbf{K},$ Kariya $\mathbf{Y,}$ Asakawa T, Kan T, Fukuda H, Ohashi-Ito K (2015) A negative feedback loop controlling bHLH complexes is involved in vascular cell division and differentiation in the root apical meristem. Curr Biol 25: 3144–3150   
Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL (2013) TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol 14: R36   
Kozak M (1986) Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes. Cell 44: 283–292   
Kozak M (1987) Effects of intercistronic length on the efficiency of reinitiation by eucaryotic ribosomes. Mol Cell Biol 7: 3438–3445   
Kozak M (2002) Pushing the limits of the scanning mechanism for initiation of translation. Gene 299: 1–34   
Kwak SH, Lee SH (2001) The regulation of ornithine decarboxylase gene expression by sucrose and small upstream open reading frame in tomato (Lycopersicon esculentum Mill). Plant Cell Physiol 42: 314–323   
Laing WA, Martínez-Sánchez M, Wright MA, Bulley SM, Brewster D, Dare AP, Rassam M, Wang D, Storey R, Macknight RC, et al (2015) An upstream open reading frame is essential for feedback regulation of ascorbate biosynthesis in Arabidopsis. Plant Cell 27: 772–786   
Lamesch P, Berardini TZ, Li D, Swarbreck D, Wilks C, Sasidharan $\mathbf{R},$ Muller R, Dreher $\mathbf{K},$ Alexander DL, Garcia-Hernandez M, et al (2012) The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools. Nucleic Acids Res 40: D1202–D1210   
Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, McWilliam H, Valentin F, Wallace IM, Wilm A, Lopez R, et al (2007) Clustal W and Clustal X version 2.0. Bioinformatics 23: 2947–2948   
Lei $\mathbf{L},$ Shi J, Chen J, Zhang M, Sun S, Xie S, Li X, Zeng B, Peng L, Hauck A, et al (2015) Ribosome profiling reveals dynamic translational landscape in maize seedlings under drought stress. Plant J 84: 1206–1218   
Liu J, Deng S, Wang H, Ye J, Wu HW, Sun HX, Chua NH (2016) CURLY LEAF regulates gene sets coordinating seed size and lipid biosynthesis. Plant Physiol 171: 424–436   
Manavella PA, Arce AL, Dezar CA, Bitton F, Renou JP, Crespi M, Chan RL (2006) Cross-talk between ethylene and drought signalling pathways is mediated by the sunflower Hahb-4 transcription factor. Plant J 48: 125–137   
Mancini $\mathbf{E},$ Sanchez SE, Romanowski A, Schlaen RG, Sanchez-Lamas M, Cerdán PD, Yanovsky MJ (2016) Acute effects of light on alternative splicing in light-grown plants. Photochem Photobiol 92: 126–133   
Matzke MA, Matzke AJM, Pruss GJ, Vance VB (2001) RNA-based silencing strategies in plants. Curr Opin Genet Dev 11: 221–227   
Merchante C, Brumos J, Yun J, Hu Q, Spencer KR, Enríquez P, Binder BM, Heber S, Stepanova AN, Alonso JM (2015) Gene-specific translation regulation mediated by the hormone-signaling molecule EIN2. Cell 163: 684– 697   
Moreno Piovano GS, Moreno JE, Cabello JV, Arce AL, Otegui ME, Chan RL (2017) A role for LAX2 in regulating xylem development and lateralvein symmetry in the leaf. Ann Bot (Lond), in press. doi.org/10.1093/ aob/mcx091   
Morton T, Petricka J, Corcoran DL, Li S, Winter CM, Carda A, Benfey PN, Ohler U, Megraw M (2014) Paired-end analysis of transcription start sites in Arabidopsis reveals plant-specific promoter signatures. Plant Cell 26: 2746–2760   
Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, Gerstein M, Snyder M (2008) The transcriptional landscape of the yeast genome defined by RNA sequencing. Science 320: 1344–1349   
Nilsson OB, Hedman R, Marino J, Wickles S, Bischoff $\mathbf{L},$ Johansson M, Müller-Lucks A, Trovato F, Puglisi JD, O’Brien EP, et al (2015) Cotranslational protein folding inside the ribosome exit tunnel. Cell Rep 12: 1533–1540   
Peragine A, Yoshikawa M, Wu G, Albrecht HL, Poethig RS (2004) SGS3 and SGS2/SDE1/RDR6 are required for juvenile development and the production of trans-acting siRNAs in Arabidopsis. Genes Dev 18: 2368– 2379   
PfafflMW (2001) A new mathematical model for relative quantification in real-time RT-PCR. Nucleic Acids Res 29: e45   
Pumplin N, Sarazin A, Jullien PE, Bologna NG, Oberlin S, Voinnet O (2016) DNA methylation influences the expression of DICER-LIKE4 isoforms, which encode proteins of alternative localization and function. Plant Cell 28: 2786–2804   
Rahmani F, Hummel M, Schuurmans J, Wiese-Klinkenberg A, Smeekens S, Hanson J (2009) Sucrose control of translation mediated by an upstream open reading frame-encoded peptide. Plant Physiol 150: 1356– 1367   
Rajeevkumar S, Anunanthini $\mathbf{P},$ Sathishkumar R (2015) Epigenetic silencing in transgenic plants. Front Plant Sci 6: 693   
Ré DA, Capella M, Bonaventure G, Chan RL (2014) Arabidopsis AtHB7 and AtHB12 evolved divergently to fine tune processes associated with growth and responses to water stress. BMC Plant Biol 14: 150   
Reeves JH (1992) Heterogeneity in the substitution process of amino acid sites of proteins coded for by mitochondrial DNA. J Mol Evol 35: 17–31   
Ribichich KF, Arce AL, Chan RL (2014) Coping with drought and salinity stresses: role of transcription factors in crop improvement. In N Tuteja, SS Gill, eds, Climate Change and Plant Abiotic Stress Tolerance. WileyVCH Verlag, Weinheim, Germany, pp 641–684   
Ribone PA, Capella M, Arce AL, Chan RL (2015a) What do we know about homeodomain-leucine zipper I transcription factors? Functional and biotechnological considerations. In DH González, ed, Plant Transcription Factors: Evolutionary, Structural and Functional Aspects. Elsevier, Amsterdam, pp 343–356   
Ribone PA, Capella M, Chan RL (2015b) Functional characterization of the homeodomain leucine zipper I transcription factor AtHB13 reveals a crucial role in Arabidopsis development. J Exp Bot 66: 5929–5943   
Rice $\mathbf{P},$ Longden I, Bleasby A (2000) EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet 16: 276–277   
Romani F, Ribone PA, Capella M, Miguel VN, Chan RL (2016) A matter of quantity: common features in the drought response of transgenic plants overexpressing HD-Zip I transcription factors. Plant Sci 251: 139–154   
Sakuma S, Pourkheirandish M, Hensel G, Kumlehn J, Stein N, Tagiri A, Yamaji N, Ma JF, Sassa H, Koba T, et al (2013) Divergence of expression pattern contributed to neofunctionalization of duplicated HD-Zip I transcription factor in barley. New Phytol 197: 939–948   
Saul $\mathbf{H},$ Elharrar $\mathbf{E},$ Gaash R, Eliaz D, Valenci M, Akua T, Avramov M, Frankel N, Berezin I, Gottlieb D, et al (2009) The upstream open reading frame of the Arabidopsis AtMHX gene has a strong impact on transcript accumulation through the nonsense-mediated mRNA decay pathway. Plant J 60: 1031–1042   
Schubert D, Lechtenberg B, Forsbach A, Gils M, Bahadur S, Schmidt R (2004) Silencing in Arabidopsis T-DNA transformants: the predominant role of a gene-specific RNA sensing mechanism versus position effects. Plant Cell 16: 2561–2572   
Somers J, Pöyry T, Willis AE (2013) A perspective on mammalian upstream open reading frame function. Int J Biochem Cell Biol 45: 1690– 1700   
Takahashi H, Takahashi A, Naito S, Onouchi H (2012) BAIUCAS: a novel BLAST-based algorithm for the identification of upstream open reading frames with conserved amino acid sequences and its application to the Arabidopsis thaliana genome. Bioinformatics 28: 2231–2241   
Tanaka M, Sotta N, Yamazumi Y, Yamashita Y, Miwa K, Murota K, Chiba ${\boldsymbol{\Upsilon}},$ Hirai MY, Akiyama T, Onouchi $\mathbf{H},$ et al (2016) The minimum open reading frame, AUG-stop, induces boron-dependent ribosome stalling and mRNA degradation. Plant Cell 28: 2830–2849   
Thorvaldsdóttir $\mathbf{H},$ Robinson JT, Mesirov JP (2013) Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform 14: 178–192   
Tran VdT, Souiai O, Romero-Barrios N, Crespi M, Gautheret D (2016) Detection of generic differential RNA processing events from RNA-seq data. RNA Biol 13: 59–67   
Uchiyama-Kadokura N, Murakami K, Takemoto M, Koyanagi N, Murota K, Naito S, Onouchi H (2014) Polyamine-responsive ribosomal arrest at the stop codon of an upstream open reading frame of the AdoMetDC1 gene triggers nonsense-mediated mRNA decay in Arabidopsis thaliana. Plant Cell Physiol 55: 1556–1567   
Vaucheret $\mathbf{H},$ Fagard M (2001) Transcriptional gene silencing in plants: targets, inducers and regulators. Trends Genet 17: 29–35   
Vlad D, Kierzkowski D, Rast MI, Vuolo F, Dello Ioio R, Galinha C, Gan X, Hajheidari M, Hay A, Smith RS, et al (2014) Leaf shape evolution through duplication, regulatory diversification, and loss of a homeobox gene. Science 343: 780–783   
von Arnim AG, Jia Q, Vaughn JN (2014) Regulation of plant translation by upstream open reading frames. Plant Sci 214: 1–12   
Wang L, Wessler SR (1998) Inefficient reinitiation is responsible for upstream open reading frame-mediated translational repression of the maize R gene. Plant Cell 10: 1733–1746   
Wang W, Vinocur B, Altman A (2003) Plant responses to drought, salinity and extreme temperatures: towards genetic engineering for stress tolerance. Planta 218: 1–14   
Wiese A, Elzinga N, Wobbes B, Smeekens S (2004) A conserved upstream open reading frame mediates sucrose-induced repression of translation. Plant Cell 16: 1717–1729   
Yang Z (1993) Maximum-likelihood estimation of phylogeny from DNA sequences when substitution rates differ over sites. Mol Biol Evol 10: 1396–1401   
Zur H, Tuller T (2013) New universal rules of eukaryotic translation initiation fidelity. PLOS Comput Biol 9: e1003136  