# An upstream open reading frame (uORF) signals for cellular localization of the virulence factor implicated in pregnancy associated malaria  

Yair Fastman, Shany Assaraf, Miriam Rose, Elad Milrot, Katherine Basore, B. Sivanandam Arasu, Sanjay A. Desai, Michael Elbaum and Ron Dzikowski  

1Department of Microbiology and Molecular Genetics, The Institute for Medical Research Israel - Canada, The Kuvin Center for the Study of Infectious and Tropical Diseases, The Hebrew University-Hadassah Medical School, Jerusalem 91120, Israel, 2Electron Microscopy Unit, Weizmann Institute of Science, Rehovot 76100, Israel, 3The Laboratory of Malaria and Vector Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Rockville, MD, USA and Department of Materials and Interfaces, Weizmann Institute of Science, Rehovot 76100, Israel  

Received September 25, 2017; Revised February 27, 2018; Editorial Decision February 27, 2018; Accepted March 01, 2018  

# ABSTRACT  

Plasmodium falciparum, the causative agent of the deadliest form of human malaria, alternates expression of variable antigens, encoded by members of a multi-copy gene family named var. In var2csa, the var gene implicated in pregnancy-associated malaria, translational repression is regulated by a unique upstream open reading frame (uORF) found only in its $\mathsf{\pmb{5}^{\prime}}$ UTR. Here, we report that this translated uORF significantly alters both transcription and posttranslational protein trafficking. The parasite can alter a protein’s destination without any modifications to the protein itself, but instead by an element within the $\mathsf{\pmb{5}^{\prime}}$ UTR of the transcript. This uORF-dependent localization was confirmed by single molecule STORM imaging, followed by fusion of the uORF to a reporter gene which changes its cellular localization from cytoplasmic to ER-associated. These data point towards a novel regulatory role of uORF in protein trafficking, with important implications for the pathology of pregnancy-associated malaria.  

# INTRODUCTION  

Plasmodium falciparum, the protozoan parasite that causes human malaria remains one of the deadliest pathogens on earth (1). This parasite, transmitted by female Anopheline mosquitoes, causes malaria when it reaches the blood stream and proliferates within infected red blood cells (iRBC). In addition to anemia, the severe pathology caused by $P.$ falciparum is linked to its ability to modify the infected erythrocyte with proteins that cause the iRBC to adhere to neighboring uninfected RBCs as well as to several endothelial receptors (2). The main ligand responsible for the cytoadhesive properties of the iRBC are variable surface proteins named PfEMP1 encoded by a multi-copy gene family named var (3). To avoid exposure of its entire antigenic repertoire, the parasite expresses only a single var gene at a time while the rest of the var gene family is transcriptionally silent (4,5). Due to the constant switching of var gene expression, which occurs intrinsically at a very low rate, some parasites avoid the antibody-mediated response against the predominant PfEMP1 and continue the infection, expressing another PfEMP1 variant that is not recognized by the immune system (6). Thus, immune evasion by $P.$ falciparum is achieved both by antigenic switches between expression of different PfEMP1 variants and by avoiding clearance by the spleen through cytoadhesion to endothelial receptors and sequestration in deep tissues. It is well established that different PfEMP1 variants have different cytoadhesive properties since they bind different endothelial receptors such as CD36, ICAM1, Chondroitin Sulfate A (CSA) and EPCR (7,8). Thus, some of the severe disease manifestations, such as cerebral and pregnancy-associated malaria, are determined by the adhesion phenotype of the PfEMP1 variant expressed.  

CSA is an abundant glycosaminoglycan expressed in the placental intervillous space and is bound by a specific PfEMP1 variant named VAR2CSA (9). The only var gene expressed in a parasite culture selected for binding to CSA is var2csa (10), and parasites in which this gene has been deleted do not adhere to CSA (11). Interestingly, people living in endemic regions that were exposed to continuous $P.$ falciparum infections develop antibodies against most of the PfEMP1 repertoire. However, most studies indicate that antibodies against VAR2CSA can only be found in serum taken from women who were infected during pregnancy and that protection against placental sequestration is acquired over consecutive pregnancies (9,12). Although var genes undergo continuous transcription switching, the PfEMP1 expressed by var2csa is only translated in pregnant women. Similar to other var genes, var2csa transcription is tightly regulated in a mutually exclusive manner, but contains an additional unique mechanism that represses protein translation when expression of VAR2CSA isn’t needed. This mechanism is regulated by an upstream open reading frame (uORF) which is found uniquely in the $5^{\prime}$ UTR of var2csa (13). This uORF is 360bp long and is situated between the transcription start site (TSS) and the downstream open reading frame (dORF) that encodes VAR2CSA. Therefore, the var2csa transcript is bicistronic, with the uORF and the dORF separated by an inter-cistronic region (ICR), and it appears that var2csa mRNA contains both ORFs (13). The molecular mechanism that allows translation of the dORF in the presence of a placenta is unknown, but there is strong evidence that it is controlled by a switch in the efficiency of translation re-initiation of the dORF (14), and expression of an endogenous plasmodium translation-enhancing factor (15). Here, we show that in addition to its role in regulating translation of the var2csa-encoded PfEMP1, this uORF plays a key role in determining the cellular localization of the dORF-encoded protein.  

# MATERIALS AND METHODS  

# Parasite culture and parasitemia counts  

All parasites used were derivatives of the NF54 parasite line and were cultivated at $5\%$ hematocrit in RPMI 1640 medium, $0.5\%$ Albumax II (Invitrogen), $0.25\%$ sodium bicarbonate, and $0.1\ \mathrm{mg/ml}$ gentamicin. Parasites were incubated at $37^{\circ}\mathbf{C}$ in an atmosphere of $5\%$ oxygen, $5\%$ carbon dioxide and $90\%$ nitrogen. Parasite cultures were synchronized using percoll/sorbitol gradient centrifugation as previously described (16,17). Briefly, infected RBCs were layered on a step gradient of $40\%/70\%$ percoll containing $6\%$ sorbitol. The gradients were then centrifuged at 12 $000\mathrm{g}$ for $20\,\mathrm{min}$ at room temperature. Highly synchronized, late stage parasites were recovered from the $\bar{4}0\%/70\%$ interphase, washed twice with complete culture media and placed back in culture. The level of parasitemia of the resistance experiment (Supplementary Figure S3) was calculated by counting blood smears from two independent parasite cultures stained with Giemsa under a light microscope. The level of parasitemia of the KDEL experiment (Figure 3) was calculated by Flow Cytometry. For flow cytometry, aliquots of $50\upmu\mathrm{l}$ parasite cultures were washed in PBS and incubated $30\,\mathrm{min}$ with 1:10 000 SYBR Green I DNA stain (Life Technologies). The fluorescence profiles of infected erythrocytes were measured on CytoFLEX (Beckman Coulter) and analyzed by the CytExpert software.  

# Plasmid Construction  

The plasmids pVBh (18), pVBbIDh (19) and V2BbIDh (13) were previously described. V stands for a typical var promoter (var7b cloned previously from a Dd2 parasite line (20)) and V2 stands for var2csa promoter. The plasmid pV2Bh was obtained in two steps: First we inserted the AsiSI restriction site downstream of the KpnI restriction site by PCR amplification of var promoter region from the plasmid pVBh using $5^{\prime}$ -GGGGTACCGCGATCGCG TATAGACAAACCAGAC- $3^{\prime}$ and $5^{\prime}.$ -GGCATGTTAAC ATAGTCTACC- $3^{\prime}$ . This fragment was sub-cloned back into $\mathrm{pV}\mathrm{Bh}$ (18) by KpnI/HpaI restriction sites. As a second step, the $5^{\prime}$ UTR of var2csa was amplified from NF54 parasite line using $5^{\prime}$ -AAGCGATCGCGAACGCTTAA AGAAACAAGG- $3^{\prime}$ and $5^{\prime}$ -ATGTTAACTTTGTCCAAC CATTTACAAA- $3^{\prime}$ and cloned into the plasmid above by AsiSI/HpaI restriction sites. The plasmids pVB-myc and pV2B-myc were obtained by cloning sequence no. 1 (Supplementary Table S1) into the plasmids pVBbIDh (19) and V2BbIDh (13) respectively using HpaI/HindIII. The plasmid $\scriptstyle\mathrm{pVB+uORF}$ was obtained by cloning sequence no. 2 (Supplementary Table S1) into the plasmid pVBbIDh using EcoRI/HpaI. The plasmid pV2B'uORF was obtained by cutting out the uORF from the plasmid V2BduORF (14) using NcoI/ClaI, blunt ends were then generated using Klenow fragment (NEB M0210s) and the construct was finalized by self-ligation. The plasmids pV-GB and pV2-GB were obtained by cloning sequence no. 3 (Supplementary Table S1) into the plasmids pVBbIDh (19) and V2BbIDh (13) respectively using HpaI/HindIII. The plasmids pV-GB(KDEL) and pV2-GB(KDEL) were obtained by cloning sequence no. 4 (Supplementary Table S1) into the plasmids pVBbIDh (19) and V2BbIDh (13) respectively using HpaI/HindIII. The plasmid puORF-GFP was obtained by cloning sequence no. 5 (Supplementary Table S1) into the plasmid pV-GB using HpaI/PmiI. The plasmid pV2mycGB was obtained by cloning sequence no. 6 (Table S1) into the plasmid p2V-GB using KpnI/HpaI. The plasmid pDuORF-GFP was obtained by cloning sequence no. 7 (Supplementary Table S1) into the plasmid puORF-GFP using ClaI/PmiI.  

# Parasite transfection and selection  

Parasites were transfected as described (21). Briefly, 0.2 cm electroporation cuvettes were loaded with $0.175~\mathrm{ml}$ of erythrocytes and $50~\upmu\mathrm{g}$ of plasmid DNA in incomplete cytomix and electroporated using BIORAD GenePulser electroporator. Stable transfectants carrying plasmids with a hDHFR-selectable marker were initially selected on 4 nM WR99210. Stable transfectants carrying plasmids with BSD-selectable marker only (pVBh and pV2Bh) were initially selected on $2\ \upmu\mathrm{g/ml}$ blasticidin-S (Invitrogen). In order to obtain parasites carrying large plasmid copy numbers, these cultures were then subjected to elevated concentrations of $4{\mathrm{-}}20\ {\upmu}{\mathrm{g/ml}}$ blasticidin-S, depending on experimental design.  

# Solute permeability measurements  

Permeability assays of iRBCs were performed to exclude possible spontaneous resistance to blasticidin S. These assays were done as previously described (22). Briefly, the VB-myc and V2B-myc transfectants were cultivated to the trophozoite stage in standard media with $10\upmu\mathrm{g/mL}$ blasticidin, harvested and enriched to ${\tt>}95\%$ parasitemia using the percoll/sorbitol method, and resuspended in buffered saline $145\,\mathrm{mM}\,\mathrm{NaCl}$ ,$20\,\mathrm{mM}$ HEPES, $0.1\;\mathrm{mg/ml}$ BSA, pH 7.4 with NaOH). Solute uptake was initiated by addition of 20 volumes of $280\,\mathrm{mM}$ sorbitol or $145\,\mathrm{mM}$ phenyltrimethylammonium chloride (PhTMA-Cl) in buffer ( $20~\mathrm{mM}$ NaHEPES, $0.1~\mathrm{mg}$ BSA, $\mathrm{pH}\ 7.4)$ at $37^{\circ}\mathbf{C}$ with transport inhibitors as indicated (Supplementary Figure S3). Osmotic lysis, which results from net sorbitol or $\mathrm{PhTMA^{+}C l^{-}}$ uptake, was continuously recorded by measuring the transmittance of $700\ \mathrm{nm}$ light through the cell suspension (hematocrit, $0.125\%$ ; DU spectrophotometer with Peltier temperature control). The osmotic lysis halftime, the time required for half-maximal cell lysis, is inversely proportional to solute permeability and was determined by linear interpolation.  

# Genomic DNA extraction, RNA extraction and cDNA synthesis  

Genomic DNA was extracted as described (18). Briefly, iRBCs from a $20\;\mathrm{ml}$ cultures were lysed with saponin and the parasite pellet was washed with PBS and taken up in $200\;\upmu\mathrm{l}$ TSE buffer $\mathrm{100\,mM\,NaCl}$ ,$50\,\mathrm{mM}$ EDTA, $20\,\mathrm{mM}$ Tris, $\mathrm{pH}$ 8), $40\upmu\mathrm{l}$ of $10\%$ SDS and $20\ \upmu\mathrm{l\GM\NaClO_{4}}$ . This suspension was rocked for twelve hours and genomic DNA was extracted with phenol/chloroform. The resulting DNA was taken up in $100\,\upmu\mathrm{l}\,\mathrm{d}\mathrm{H}_{2}\mathrm{O}.$ . RNA was extracted from synchronized parasite cultures at $20{-}24\,\mathrm{h}$ after percoll/sorbitol gradient centrifugation (16,17). RNA was extracted with the TRIZOL LS Reagent $\textsuperscript{\textregistered}$ as described (23) and purified on PureLink column (Invitrogen) according to manufacturer’s protocol. Isolated RNA was then treated with DNase I (TaKaRa) to degrade contaminating gDNA. cDNA synthesis was performed from $500~\mathrm{ng}$ total RNA with PrimeScript™RT Reagent Kit (TaKaRa) as described by the manufacturer.  

# Real-time RT-qPCR  

For RT-qPCR reactions to detect transcription from all var genes present in the 3D7 genome, we used the primer sets published in (10) with few modifications (24). Transcript copy numbers were determined using the formula 2−''CT as described in the Applied Biosystems User Bulletin 2 using $\mathrm{NF54\,gDNA}$ as the calibrator. Specifically, relative copy number was calculated as 2 exponential negative ( $\upepsilon{}_{\mathrm{t}}$ target gene in cDNA – Ct reference gene in cDNA)-(Ct target gene in gDNA – Ct target gene in gDNA)). To quantify bsd transcription levels we used the primers of bsd that were previously published (19). To quantify clag3.1 and clag3.2 transcription levels we used the primers published elsewhere (25). Copy numbers of episomal promoters were calculated using RT-qPCR of gDNA and comparing the $\Delta{\bf C}{\bf T}$ of bsd to that of the single copy housekeeping gene arginine-tRNA ligase (PF3D7 1218600).  

# mRNA stability assay  

Parasite cultures were synchronized to late ring stages (20 h post percoll/sorbitol gradient centrifugation). The same amount of parasites were grown in the presence of $10\,\upmu\mathrm{g/ml}$ blasticidin in 12-well plates, $1.5\,\mathrm{\ml}$ culture in each well. $20\upmu\mathrm{g/ml}$ actinomycin D (ActD, TOKU-E) was added to cultures to inhibit transcription and samples were collected immediately and at 5, 15, 30, 60 and $120\ \mathrm{min}$ post addition of ActD. Cultures were collected by washing in PBS. Cell pellets were immediately dissolved in $750~\upmu\mathrm{l}$ TRIZOL LS Reagent $\circledast$ and followed by RNA extraction and cDNA synthesis as described above.  

# Western blot  

To collect parasite proteins, infected RBCs were lysed with saponin, then parasites were washed twice with PBS and collected in Laemmli sample buffer (Sigma). The same protein amounts of denatured samples were subjected to SDS-PAGE (gradient $4–20\%$ , Bio-Rad) and electroblotted to a nitrocellulose membrane. Immunodetection was carried out using rabbit anti-BSD (EMCbiosciences, 1:1000), mouse anti-c-Myc antibody (Santa-Cruz, 1:300), mouse anti-GFP (Roche, 1:1000) or rabbit polyclonal anti-aldolase (1:2000) (26). The secondary antibodies used were antibodies conjugated to Horseradish Peroxidase (HRP), goat anti-rabbit and goat anti-mouse (Jackson ImmunoResearch Laboratories, 1:10 000). The immunoblots were developed in EZ/ECL solution (Israel Biological Industries).  

# Cell fractionation of infected erythrocytes  

Parasite cultures were lysed on ice by saponin for $20\ \mathrm{min}$ and centrifuged at 5000 rpm for $5\;\mathrm{min}$ . The parasite pellet was washed three times with ice-cold PBS containing protease inhibitors, resuspended in $500~\upmu\mathrm{l}$ $\mathrm{H}_{2}\mathrm{O}$ and lysed by three ‘thaw and freeze’ cycles at $-80^{\circ}\mathrm{C}$ . Samples were centrifuged at $14\,000\,\mathrm{rpm}$ for $15\,\mathrm{min}$ at $4^{\circ}\mathrm{C}$ and then separated into soluble and pellet fractions. The soluble fraction was centrifuged again at $4^{\circ}\mathrm{C}$ to discard any membrane residuals and concentrated using Amicon Centrifugal filters (Amicon Ultra $0.5\,\mathrm{ml}$ centrifugal filters, 3K, Millipore) at $14\,000\,\mathrm{rpm}$ for $30\,\mathrm{{min}}$ at $4^{\circ}\mathbf{C}.$ . The pellet fraction was washed three times with ice-cold PBS containing protease inhibitors (Roche). Both fractions were mixed to the same volume with sample buffer and subjected to western blot analysis  

# Immunofluorescence assay and live cell imaging  

IFA was performed as described before (26) with few modifications. Briefly, $1\;\mathrm{ml}$ of parasite culture at $5\%$ parasitemia was washed with PBS and re-suspended in a fresh fixative solution ( $4\%$ Paraformaldehyde (EMS) and $0.0075\%$ glutaraldehyde (EMS) in PBS). Fixed parasites were treated with $0.1\%$ Triton-X 100 (Sigma) in PBS, then blocked with $3\%$ BSA (Sigma) in PBS. Cells were then incubated with primary antibodies used at the following dilutions: rabbit antiBSD (EMCbiosciences, 1:1000), mouse anti- $P f39$ (MR4, 1:1000), rabbit anti-GFP (Life Technologies, 1:300). Samples were washed and incubated with Alexa Fluor 555 goat anti-rabbit (Life Technologies, 1:500), Alexa Fluor 568 goat anti-Mouse (1:500, Life Technologies) or Alexa Fluor 488 goat anti-rabbit (Life Technologies, 1:500) secondary antibodies. Samples were washed and laid on ‘PTFE’ printed slides (EMS) and mounted in ProLong Gold antifade reagent with DAPI (Molecular Probes). For live cell imaging, $200~\upmu\mathrm{l}$ of the culture were pelleted, diluted with PBS and incubated $30\;\mathrm{min}$ with Hoechst 33342 fluorescent nuclear stain (Thermo Scientific). Then samples were laid on a Teflon coated slide (EMS), covered with cover slides and visualized immediately. Fluorescent images were obtained using a Plan Apo ' $100\times$ oil $\mathrm{NA}=1.5$ ,$\bar{\mathrm{WD}}=130\ \upmu\mathrm{m}$ lens on a Nikon Eclipse Ti-E microscope equipped with a CoolSNAP Myo CCD camera. Images were processed using the NIS-Elements AR (4.40 version) software.  

# Stochastic Optical Reconstruction Microscopy (STORM) imaging and analysis  

Parasite cultures were saponin lysed and washed three times with fresh PBS. Parasites were then re-suspended on ice with fresh fixative solution $4\%$ paraformaldehyde (EMS) in PBS) for $1\,\mathrm{~h~}$ . Fixed parasites were allowed to air dry in an eight well-chambered cover glass system $1.5\mathrm{H}$ (In Vitro Scientific) for $2{-}3\;\mathrm{h}$ at room temperature and covered with glycine ( $30\;\mathrm{mM}$ glycine in PBS) to quench unreacted aldehyde groups. Samples were washed three times with PBS and permeabilized with $0.1\%$ Triton-x 100 (PBS) for 10 min at room temperature, washed with PBS and blocked with $10\%$ horse serum (Biological Industries) containing $3\%$ BSA (IgG free, Sigma) and $0.025\%$ Tx-100 in PBS for $1\,\mathrm{~h~}$ at room temperature. Primary antibodies, rabbit antiGFP (1:300, life technologies) and mouse anti- $\cdot P f39$ (MR4, 1:750) were diluted in $5\%$ horse serum containing $0.025\%$ ${\mathrm{T}}{\mathrm{X}}{-}100$ and applied for $1\ \mathrm{h}$ at room temperature, followed by five washes in PBS containing $0.025\%$ Tx-100. Secondary antibodies, Donkey anti-Mouse Alexa647 (1:300, Jackson ImmunoResearch) and Goat anti-Rabbit Alexa568 (1:300, life technologies) were applied for $1\,\mathrm{~h~}$ in $5\%$ horse serum containing $0.025\%$ ${\mathrm{T}}{\mathrm{X}}{-}100$ in PBS at room temperature, followed by five washes with PBS containing $0.0{\bar{2}}5\%$ Tx100. Parasite nuclei were labelled with YOYO-1 (1:300, Life Technologies) for $20\;\mathrm{min}$ at room temperature, followed by 3 washes with PBS. STORM was performed by a Nikon Eclipse Ti-E microscope with a CFI Apo TIRF $\times~100$ DIC N2 oil objective (NA 1.49, WD $0.12\,\mathrm{mm}$ ). Multi-channel calibration was performed prior data acquisition using fluorescent TetraSpeckTM microspheres $0.1~{\upmu\mathrm{m}}$ in diameter (Life-Technologies, Molecular Probes). Stained cells were placed in Glox-MEA imaging buffer containing $100\mathrm{mM}$ MEA, $10\%$ glucose $\left(\mathbf{D}_{2}\mathbf{O}\right)$ , Glox $11.2\;\mathrm{mg/ml}$ glucose oxidase (Sigma) and $1.8\,\mathrm{mg/ml}$ catalase (Sigma, C30-500MG)) in dilution buffer (50 mM NaCl, $200\ \mathrm{mM}$ Tris in ${\bf D}_{2}{\bf O}_{\alpha}$ .Samples were illuminated by $561\,\mathrm{\nm}\,\left(80\,\mathrm{\mW}\right)$ and 647 nm $125~\mathrm{mW}$ excitation lasers in changing intensity over the duration of the imaging sequence (typically, using $50-$ $100\%$ ). $488\mathrm{nm}$ laser was used at $0.5\%$ power $\mathrm{80\;mW})$ to visualized nuclei by TIRF. For each acquisition $10\,000$ frames were recorded onto a $256~\times~256$ pixel region (pixel size $160\ \mathrm{nm})$ ) of an Andor iXon-897 EMCCD camera. Superresolution images were reconstructed from a series of at least 5000 images per channel using the N-STORM analysis module, version 1.1.21 of NIS Elements AR v. 4.40 (Laboratory imaging s.r.o.). One way co-localization analysis was performed using an ImageJ plugin ‘Interaction Factor package’ recently published as a method for quantifying molecular interactions from data obtained with STORM imaging (27). As a first step, four series of separate acquisitions (each of $10\ 000$ frames of $256\,\times\,256$ pixel region as described above) of the two channels (568 and $647~\mathrm{nm}$ STORM images were analyzed by using the ImageJ plugin ThunderSTORM with default settings (28). Then, both channels were merged into RGB color images where the BSD-GFP fusion protein was colored in green and the ER marker Pf39 was colored in red. Several ‘regions of interest’ (ROIs) surrounding parasites were randomly chosen from each RGB image. We then calculated the ‘interaction factor’ as one way green to red interaction using the default setting of the ‘Interaction Factor package’ plugin as described (27). The calculated ‘Interaction Factor’ scores between 0 and 1, where 0 represents no interaction and 1 represents complete overlap.  

# Immuno-electron microscopy  

Sample preparation for Immuno-electron microscopy was done as described (29) with few modifications. Samples were immersed in 1-Hexadecane and then high-pressure frozen in a high pressure freezing machine (HPM010, BAL-TEC). Samples were then freeze-substituted in dry acetone containing $0.1\%$ uranyl acetate and $0.1\%$ glutaraldehyde for $30~\mathrm{h}$ at $-90^{\circ}\mathrm{C}$ in a freeze substitution machine (Leica EM AFS). Samples were slowly warmed to $-30^{\circ}\mathbf{C}_{\mathrm{{c}}}$ , washed in dry ethanol, and infiltrated with increasing concentrations of Lowicryl HM-20 resin. Samples were UV-polymerized at – $30^{\circ}\mathbf{C}$ , sectioned and deposited on formvar-coated nickel 200 mesh grids. Grids were treated with $0.5\%$ blocking solution $(0.5\%$ gelatin, $0.5\%$ BSA) for $20\ \mathrm{min}$ and then incubated with anti-GFP antibody in blocking solution (ab6556, Abcam) for $2~\mathrm{h}$ at RT. Grids were rinsed with $0.1\%$ glycine in PBS and incubated with $10\;\mathrm{nm}$ gold conjugated goat antimouse (EMS) for $30\;\mathrm{min}$ . Grids were washed with PBSX1 and DDW. Samples were visualized using an FEI Tecnai T12 (FEI Company, Eindhoven, the Netherlands).  

# RESULTS  

We have previously shown that it is possible to silence the entire var gene family and erase its epigenetic memory using promoter titration (18,30). In this technique, one can select for increasing numbers of active var promoters on episomes that outcompete activation factors needed for transcription of the endogenous var genes and effectively silence the entire gene family. Surprisingly, we were unable to completely silence and erase the epigenetic memory of a clonal population expressing var2csa using promoter titration with a typical var promoter on the competing pVBh episomes (Supplementary Figure S1). We therefore decided to perform promoter titration with plasmids that express the drug selectable marker bsd (which encodes the deaminase that confers resistance to blasticidin-S) by a var2csa promoter. We used two approaches to perform promoter titration: first, we used the plasmid pV2Bh to directly select for active episomal var2csa promoters using blasticidin; and second, we used the plasmid pV2BbIDh, in which the silencing effect of var intron was disabled (31), and therefore it is possible to select first for stable episomal transfection using hDHFR as selectable marker and activate the var2csa promoter only after the establishment of parasite populations that stably carry the episomes (see plasmid maps in Supplementary Figure S2). For comparison, we used parasites which were transfected with similar plasmids except that a typical UpsC var promoter was used instead of the var2csa promoter (in pVBh and pVBbIDh respectively). As a first step, we wanted to confirm that increasing drug concentrations would increase bsd expression as previously observed (18,32–35). As expected, in the parasites that expressed the typical var promoter, increasing drug concentration was associated with an increase in the mRNA levels from the activated episomal var promoter (Supplementary Figure S2A and B). However, the parasites expressing bsd from the var2csa promoter did not elevate mRNA levels in response to increase selection pressure (Supplementary Figure S2C and D). To exclude that the different response in mRNA levels between the two promotor types was due to differences in RNA stability we incubated both parasite lines, expressing the two different promoter types, with the transcription inhibitor Actinomycin D (ActD) and compared the rate of RNA decay. There was no difference in the rate of RNA decay indicating that RNA transcribed by the two promoter types is equally stable (Supplementary Figure S2E and F).  

We then tested whether the increase in drug pressure had forced the parasites to express more protein, assuming that the similar growth rate of both lines (Figure 1A) implies that they make more of blasticidin-S-deaminase (BSD) to de-toxify blasticidin S and overcome the increase in its concentration. To our surprise, while the increase in selection pressure for activation of typical var promoters resulted in both elevated levels of mRNA and a gradual increase in protein expression, no protein was detected by western blot in parasites that were selected for gradual activation of var2csa promoters throughout the parasites’ intra-erythrocytic development cycle (IDC) (Figure 1B–F). Similar results were obtained by Immuno-Fluorescent Assay (IFA): the protein expressed by a typical var promoter gave a very strong signal which was spread throughout the parasite’s cytosol (Figure 1G, upper panel). However, very low signal was obtained from the proteins expressed by the var2csa promoter. The signal was located in spots around the nucleus but was hard to differentiate from microscopic background (Figure 1G, lower panel). As a first step towards understanding these puzzling results, we wanted to exclude the possibility that we selected for a mutation that confers resistance to blasticidin independent of expression of BSD. We took advantage of the fact that carrying stable episomes in $P.$ falciparum requires drug selection without which these parasites shed the episomes within a couple of weeks. Two clonal NF54 populations, which were transfected either with pV2Bh (CSA selected line) or pV2BbIDh (G6 line) episomes and selected for stable transfection and activation of the var2csa promoter on $10~\upmu\mathrm{g/ml}$ blasticidin, were released from drug pressure for a period of one month to allow the parasite to shed all episomes. The cultures were split and half of each culture continued to grow on regular media while the other cultures were put under selection of $2~\upmu\mathrm{g/ml}$ blasticidin. We show that the parasites that shed the episomes were sensitive to the drug similar to the clonal population taken initially for transfection with the bsd expressing episomes (Supplementary Figure S3A, C, E, G). In addition, P. falciparum can become resistant without bsd expression by reducing blasticidin S uptake at the host red cell membrane via plasmodial surface anion channel (PSAC). To reduce uptake, the parasite may either silence or switch expression of the two clag3 genes that determine PSAC activity (36–39). Low concentrations of blasticidin $(<\!0.4\,\upmu\mathrm{g/ml)}$ selects for clag3.1 expression whereas resistance to higher concentrations of blasticidin (up to $2\ \upmu\mathrm{g/ml})$ results from silencing of both clag3 genes (38). However, here both the pre-transfected lines as well as the parasites carrying the active var2csa promoters continued to express clag3.1 gene even when selected with very high concentrations of 10 $\upmu\mathrm{g}/\mathrm{ml}$ blasticidin (Supplementary Figure S3B, D, F, H). We also performed PSAC permeability assays with two organic solutes, sorbitol and phenyltrimethylammonium (PhTMA) that use distinct routes through this channel (40). These measurements revealed preserved channel-mediated permeability in both the VB-myc and V2B-myc lines (Supplementary Figure S3 I-L, halftimes of $12.5\pm0.7$ in sorbitol $(n=$ 6 trials for each line) and $5.4\pm0.5$ min in PhTMA $n=5$ trials each); $P=0.7$ for comparisons between the two lines in each solute); these values contrast with markedly greater halftimes indicative of reduced permeability in parasites resistant to blasticidin without BSD expression (106 and 22 min for sorbitol and PhTMA, respectively (36)). Block by the potent PSAC inhibitor ISG-21 was also unchanged and indistinguishable in the two transfectants (Supplementary Figure S3M–N); ISG-21 was less effective against PhTMA uptake than sorbitol in both lines, as previously reported (40). These findings clearly demonstrate that both parasite populations that express bsd, under either the typical var promoter or the var2csa promoter, are equally resistant to osmotic lysis (Supplementary Figure S3I–N), and excludes altered host cell permeability as the mechanism of blasticidin resistance in parasites expressing bsd by the var2csa promoter.  

We hypothesized that the uORF found within the $5^{\prime}$ UTR of var2csa could account for the different RNA and protein phenotypes we observed, since this is the main regulatory element that is different between the upstream region of typical var genes and var2csa. To test this hypothesis, we created two additional constructs. In the first we inserted the uORF into the $5^{\prime}$ UTR of a regular var promoter (Figure 2A, left), and in the second we have deleted the uORF from the $5^{\prime}$ UTR of var2csa (Figure 2A, right). We then subjected the transfected parasites to increasing concentrations of blasticidin, and measured steady-state RNA as well as BSD protein levels. In parasites carrying the episomes in which the uORF was removed from var2csa $5^{\prime}$ UTR, we observed an increase in steady state RNA levels that was associated with elevated protein levels detected by Western blot as we increased drug concentration (Figure 2C, E). In addition, the protein was easily observed in the parasite cytoplasm (Figure 2F, upper panel). On the other hand, in parasites that carried the episomes in which the uORF was inserted into a typical var $5^{\prime}$ UTR, at a similar location where it resides in the original var2csa promoter, mRNA levels did not elevate in response to the increase in drug pressure and more strikingly, the protein was undetectable by Western blot and IFA (Figure 2B, D and F, lower panel). These results are reciprocal to the original transfections with intact var and var2csa promoters, indicating that the uORF found in var2csa $5^{\prime}$ UTR sequence is responsible for the differences in both RNA and protein phenotypes.  

![](images/1704a35deaa1fc40adcd2bfc25e0a579faa91b572f7ad59a29a1d9ad7f3faf29.jpg)  
Figure 1. The protein expressed by an activated var2csa promoter is not detected. (A) Growth curves of parasites expressing bsd by typical var and var2csa promoters show that they grow at similar rates on increasing blasticidin pressure. (B) Western blot analyses of BSD expressed by either var or var2csa promoters and selected using increasing levels of blasticidin. (C) Western blot analyses of BSD expression by a var2csa promoter or a var promoter $\mathbf{\eta}(\mathbf{D})$ at different time points in the parasite’s life cycle. (E) Quantification of BSD expression by a var2csa promoter or a var promoter at different time points in the parasite’s life cycle using densitometry analysis of three independent experiments. $^{**}P<0.01$ . (F) Western blot analyses of BSD-myc fusion expressed by either var or var2csa promoters and selected using increasing levels of blasticidin. hpi - hours post invasion; NC: Negative control (untransfected); PC: positive control. Anti-Aldolase antibody was used as a loading control. (G) Immunofluorescence microscopy of BSD expressing parasites under the control of a var (upper panel) or var2csa promoter (lower panel). BSD was labeled with an anti-BSD antibody (Red). Nuclei were stained with DAPI (blue). Scale bars: $1~{\upmu\mathrm{m}}$ . (Total of 120 pVBbIDh expressing parasites all showed diffused cytoplasmic localization; Total of 45 pV2BbIDh expressing parasites all showed either no signal or faint foci around the nucleus).  

![](images/e66540a93d0edfec07f3b2972d8a35eb0b57f9a138290bf71fae6df16afd1b80.jpg)  
Figure 2. mRNA and protein levels expressed by the var2csa promoter are determined by the uORF. (A) Schematic of plasmids. Left: $\mathrm{pVB+ORF}$ bsd is expressed with a typical var promoter in which the var2csa uORF was inserted into the $5^{\prime}$ UTR. Right: $\mathrm{pV}2\mathrm{B}\Delta\mathrm{uORF}_{:}$ , bsd is expressed with var2csa promoter in which the uORF was deleted. (B and D) Steady-state transcript levels of bsd and western blot analyses of BSD expression from a typical var promoter which contains the uORF of the var2csa promoter $(\mathrm{pVB+ORF})$ ) selected with increasing blasticidin concentrations. ns: not significant ( $P>0.5$ : two-tailed, unpaired, Student’s $t$ -test). (C and E) Steady-state transcript levels of bsd and western blot analyses of BSD expression, from an uORF deleted var2csa promoter (pV2B'uORF) grown in the presence of increasing blasticidin concentrations. Significance is marked with $^{**}$ ($P\,<0.01$ : two-tailed, unpaired, Student’s $t$ -test). mRNA levels are presented as relative copy number to the housekeeping gene arginine-tRNA ligase (PF3D7 1218600). Aldolase was used as a loading control. (F) Immunofluorescence microscopy of BSD expressing parasites. Upper panel: expression of BSD by an uORF-deleted var2csa promoter (pV2B'uORF). Lower panel: expression of BSD from a var promoter which contains the uORF of the var2csa promoter $(\mathrm{pVB+uORF})$ . BSD was labeled with an anti-BSD antibody (red). Nuclei were stained with DAPI (blue). Scale bars: $2\ \upmu\mathrm{m}$ (39/47 pV2B'uORF expressing parasites showed diffused cytoplasmic localization; total of 39 pVB+ORF expressing parasites were counted and showed no signal).  

Struck by the observation that the parasites expressing BSD by the intact var2csa promoters (in pV2Bh, pV2B-myc and pV2BbIDh) displayed resistance to blasticidin despite virtually undetectable levels of BSD (Figure 1), we reasoned that it is likely that in these circumstances, the parasite expresses very small, undetectable amounts of the BSD that are sufficient to confer resistance. One way to achieve resistance in spite of very low levels of deaminase is if the deaminase is expressed at the location where blasticidin S blocks translation. This might create a subcellular niche that enables enough protein translation for the parasite to survive. We applied Stochastic Optical Reconstruction Microscopy (STORM), which is a technique sensitive enough to detect photons from single molecules, and were able to detect the deaminase expressed by var2csa in small complexes localized mainly to the ER (Figure 3A–C), while the protein expressed by a typical var promoter was distributed throughout the parasite cytoplasm (Figure 3D). Further, we decided to validate that the protein expressed by the var2csa promoter enters the ER-Golgi system by fusion of an ER retention signal to the expressed protein. We hypothesized that such a signal would retain more of the deaminase expressed by var2csa in the ER and would help the parasite to resist increasing blasticidin concentrations. We further predicted that addition of this signal would have little effect on cytoplasmically located BSD expressed from a typical var promoter. The four amino acid motif KDEL, which has been previously shown to function as an ER retention signal that retrieves proteins from the Golgi back to the ER by COPI vesicles (41) was fused at the $C^{\bullet}$ terminus of the gfp-bsd fusion protein expressed either by var2csa or var promoters. We then transfected parasites with these expression vectors as well as with versions of the plasmids that did not contain the KDEL signal for each of the promoter types (Figure 3E). Parasites were first selected using WR99210 for stable episomal transfection and then with $2\;\upmu\mathrm{g/ml}$ blasticidin for activation of either var2csa or var promoters. After the different parasite populations carrying each of the plasmids were established, we increased blasticidin concentration to $6\,\upmu\mathrm{g/ml}$ and followed their growth over time. We found that cultures of parasites expressing bsd by a typical var promoter grew at a normal growth rate regardless whether the bsd was fused with the KDEL signal or not, as expected for a cytosolic protein. However, in those parasites expressing the var2csa promoter there was a significant improvement in growth rate of parasites in which the bsd was fused to the KDEL motif compared to those that did not contain it (Figure 3F), even though the protein is still not detected by Western blot (Supplementary Figure S4). Very similar results were obtained when we increased blasticidin-S concentration further from 6 to $10~\upmu\mathrm{g/ml}$ , indicating that only for the var2csa promoters, the fusion of the ER retention signal fosters the parasite’s ability to resist increasing drug concentrations (Figure 3G). Altogether, these data suggest that protein expressed by var2csa promoters find their way to the ER-Golgi trafficking machinery.  

These data, together with the results of the reciprocal placement of the var2csa-uORF at a var $5^{\prime}$ UTR, suggest that the uORF could be responsible for diverting the protein expressed by the var2csa promoter to be associated with the ER. To test this assumption, and surpass the translational repressive effect of the uORF, we created a plasmid in which the uORF sequence is fused with a bsd-gfp reporter, which is expressed by a typical var promoter. We compared the cellular localization of the fused reporter with the localization of the non–fused reporter expressed by the same var promoter. Strikingly, while the protein which was not fused to the uORF is cytosolic (Figure 4A), the fusion of the uORF sequence significantly changes its subcellular location. The protein can be visualized around the nucleus as distinct foci associated with the ER (Figure 4B). A closer look on the cellular localization of the fused protein at the ultrastructural level by immuno-Electron Microscopy indicated that the fused protein is primarily associated with membranes surrounding the nuclear envelope (Supplementary Figure S5A–C) corresponding with ER location. To further validate these findings, we performed cellular fractionation followed by western blot analysis and found that the protein fused to the uORF sequence is targeted to the membrane fraction while the one expressed without the uORF-fusion is cytosolic (Figure 4D). This set of experiments shows that the var2csa-uORF contains a signal for expression at a specific cellular localization associated with the ER.  

The signal for specific cellular localization could be at the RNA transcript of the uORF or the peptide itself. To differentiate between these two mechanisms, we made a similar construct to the one described above, in which the uORF sequence is fused with a bsd-gfp reporter expressed by a typical var promoter, only in the current plasmid the uORFcoding sequence was replaced with a different DNA sequence, encoding for the same polypeptide (14). We found that bsd-gfp reporter fused to the uORF polypeptide encoded by the alternative sequence localized to a distinct spot similar to the fusion with the endogenous uORF sequence (Figure 4C), indicating that the cellular localization signal is determined by the polypeptide rather than the RNA transcript. In order to demonstrate that the uORF itself is translated into a polypeptide, we constructed a plasmid in which the gfp-bsd fusion protein is expressed by var2csa promoter where the uORF is fused with a Myc epitope tag that allows detection of the polypeptide (Figure 5A). We first confirmed that the uORF functions as a translational repressor and that GFP isn’t detectable (data not shown). Then we showed that the polypeptide encoded by the var2csa uORF is expressed and concentrates mainly around the nucleus at ring and trophozoite stages but is hardly expressed in schizonts (Figure 5B and C). This data shows that in its natural context in the $5^{\prime}$ of var2csa the uORF is translated into a polypeptide.  

# DISCUSSION  

Upstream open reading frames (uORFs) are important regulators of eukaryotic gene expression. uORFs may impact expression of a downstream protein coding region by triggering nonsense-mediated decay (NMD) or by controlling protein translation (42). Computational analysis of the $P.$  

![](images/498f4f3006ad1ed0619ffc3efe37bae2caec06b483f450eb0e16a7994a8b2d2e.jpg)  
Figure 3. The protein expressed by the var2csa promoter is associated with the ER. (A) Super-resolution STORM imaging of parasites expressing BSDGFP fusion by a var2csa promoter (Green; pV2-GB) that were co-stained with the ER marker $P f39$ (red). (B) High magnification of the region indicated in  

falciparum genome had identified a large number of possible uORFs. However, since translation start and stop codons are AT-rich, and the $P.$ falciparum genome is considered one of the most AT-rich among eukaryotes, composed of ${\sim}80\%$ AT, it is possible that the number of functional uORFs in P. falciparum is much lower. Nonetheless, recent ribosome footprint analysis provides experimental support to the prediction that uORFs are quite abundant in $P.$ falciparum $5^{\prime}$ UTRs (43). To the best of our knowledge, the only uORF in Plasmodium, to which a defined regulatory role had been demonstrated, is the uORF of var2csa that regulates expression of the dORF at the level of translation re-initiation (13,14).  

Here, we demonstrate that when forcing activation of the var2csa promoter using a drug selectable marker, the parasites express very low levels of protein while transcribing high levels of mRNA. It is possible that under these experimental conditions the strong translational repression effect of the uORF is maintained and that there was no switch that enhanced the efficiency of translation re-initiation of the dORF. Nonetheless, parasites expressed enough protein that enabled them to overcome drug pressure. The sensitivity of super-resolution STORM imaging, which is based on the ability to visualize single fluorescent molecules, allowed us to detect these low levels of the protein expressed by var2csa promoters and to associate it with the ER. Our data clearly indicate that a regulatory element found at the 5′UTR of var2csa influenced the localization of the protein encoded by the dORF.  

Blasticidin S is a fungal toxin that kills most prokaryotic and eukaryotic cells by inhibiting mRNA-directed translation on ribosomes (44). The specific cellular location of the deaminase expressed by the var2csa promoter, at foci associated with the ER (contrary to the cytosolic localization of the deaminase expressed by a var promoter) which is a primary translation site, might explain how the parasites overcome drug pressure with expression of only very small amounts of the deaminase. One possible explanation is that blasticidin enter the parasite through a retrograde transport, similar to shiga, cholera and ricin toxins, which enter the cell by retrograde transport through endosomes to the Golgi and ER (45). If blasticidin gets into the parasite in a similar pathway, localizing the deaminase to the ER, directly where blasticidin must pass, would greatly increase its efficiency in detoxifying the molecule. Nonetheless, the exact mechanism by which the parasite overcomes blasticidin pressure at very low levels of deaminase expression requires further investigation.  

Sequence analysis of the $5^{\prime}$ UTRs of the entire var gene family had indicated they could be classified into several subtypes named UpsA, UpsB and UpsC, while the $5^{\prime}$ UTR of var2csa shows very little similarity to other var subtypes and was classified as UpsE (46). Interestingly, the UpsE promoter could not be completely silenced by promoter titration, and was not responsive to selection by increasing the mRNA levels as observed with other var promoters. This suggests that while var2csa is part of the var gene family for mutually exclusive expression and antigenic switches, it has additional unique regulatory mechanisms. The only regulatory element characterized which is unique to the $5^{\prime}$ UTRs of var2csa is the uORF, and therefore we reasoned that it might be responsible for the different phenotypes of the two promoters. Remarkably, we were able to switch between these two phenotypes by removing the uORF from the var2csa promoter and inserting it into a typical var promoter, indicating that the presence of the uORF at the $\bar{5}^{\prime}\mathrm{UTR}$ determines the fate of the protein expressed by these promoters. The fusion of the uORF sequence with reporter genes overcame the translational repressive effect of the uORF and enabled us to determine that it contains a signal that controls cellular localization. This fusion showed that not only does the uORF sequence change the protein location from the cytoplasm to the ER, but it also diverts it from being a soluble protein to a membrane-associated protein. Interestingly, the uORF does not seem to have a defined hydrophobic region. Nevertheless, both reporter genes used in our experiments contained a predicted hydrophobic domain which could contribute to its association with the ER.  

Most of the proteins that are trafficked from the parasite to the red blood cell surface contain a signal peptide and a transmembrane domain that directs them to the ER. In addition, they have a PEXEL/HT motif positioned ${\sim}20$ amino acids downstream of the signal peptide (47,48), which is cleaved by plasmepsin V (49,50) and allows them to translocate through a translocon called PTEX, which enables them to enter the parasitophorous vacuole membrane (PVM) (51). However, PfEMP1 does not have a defined signal peptide, its transmembrane domain is located internally, near the C-terminus of the protein (at the $3^{\prime}$ end of exon 1), and its PEXEL-like motif isn’t cleaved (52). Therefore, PfEMP1 is classified as PNEP (PEXEL-negative exported protein). Using a PfEMP1 mini-gene system, Melcher et al., have demonstrated that the information for PfEMP1 export to the RBC surface is found within its semi-conserved head structure (N terminal segment––NTS) in combination with the predicted transmembrane domain and its cytoplasmic tail (53). This would explain why an unrelated protein that contains no PfEMP1 coding segments expressed from a var promoter will be localized to the parasite cytosol. Our data indicate that in addition to the export information found in PfEMP1 coding regions, the uORF of var2csa contains a unique signal that associates the expressed downstream protein with the ER. Moreover, sequence analysis of ${\sim}400$ var genes from different parasite isolates showed that PfEMP1 NTS is conserved among all var genes and could be classified into two subtypes (54). However, var2csa is exempt from this sequence conservation and its NTS shows no sequence similarity to the NTS of other var genes, which could be explained by the presence of a cellular localization signal at the uORF of var2csa.  

![](images/a8c8d34dcb0a8ec2e58c86576d74b3cbb5f2b115bb6a769d34ddac7a617b0c4a.jpg)  
Figure 4. The var2csa-uORF directs protein expression to a specific cellular localization associated with the ER. (A and B) Immunofluorescence microscopy of parasites expressing the constructs $\mathrm{pV-GB}$ and puORF-GFP (schematically shown above each panel). BSD was labeled with an anti-GFP antibody (green) and the ER with a specific antibody against $P f39$ (red). Nuclei were stained with DAPI (blue). Scale bars: $2\;\upmu\mathrm{m}$ (total of 158 GFP positive, pV-GB expressing parasites all showed diffused cytoplasmic localization; 37/41 GFP positive, puORF-GFP expressing parasites showed distinct focus associated with the ER). (C) in vivo imaging of parasites expressing the construct pDuORF-GFP, which is similar to the puORF-GFP plasmid only that the uORF sequence was replace with alternative sequence that encodes for the same peptide. Nuclei were stained with Hoechst (blue). Scale bars: $2~{\upmu\mathrm{m}}$ (total of 56 GFP positive, pDuORF-GFP expressing parasites all showed distinct focus at the periphery of the nucleus). (D) Western blot analyses of cellular fractionation of parasites expressing V-GB and uORF-GFP separated into soluble supernatant (S) and pellet (P) fractions. Aldolase was used as a marker of a parasite’s cytoplasmic protein.  

![](images/a283a083da07b4a4856a5e16c23d54035d0ee217930f98f97e0b74fcaf4cf7b7.jpg)  
Figure 5. The var2csa-uORF is translated into a polypeptide. (A) Schematic of the pV2mycGB plasmid used to express the var2csa uORF fused with a myc epitope tag in its natural context at the $5^{\prime}$ UTR. (B) Western blot analysis showing that the uORF polypepide is detected in parasites selected for stable transfection of the pV2mycGB plasmid with either blasticidin or WR99210. (C) Immunofluorescence microscopy of parasites expressing pV2mycGB at different stages during IDC. The uORF was visualized with an anti-c-Myc antibody (green) nuclei were stained with DAPI (blue). Scale bars: $2\,\upmu\mathrm{m}$ . (87/120 counted parasites were anti-c-Myc positive and showed weak diffuse pattern with a distinct focus at the periphery of the DAPI staining).  

The mechanisms by which the uORF of var2csa influences the cellular localization of the dORF encoded protein are yet to be discovered. One could hypothesize that a specific mRNA splicing event could fuse the uORF to the dORF and create one ORF similar to the one we have created with the reporter gene that associates it with the ER. However, we and others (13) were unable to verify that such a splicing event of the pre-mRNA exists in the native uORF context in the $5^{\prime}$ UTR. Another reasonable hypothesis is that the peptide encoded by the uORF could play a role in targeting the protein encoded by the dORF to its destination. Similar phenomena had been observed in a polyprotein system which were encoded by the same mRNA but separated by the 2A skip peptide (55). When the upstream protein was fused with an $_\mathrm{N}$ terminal signal sequence the downstream protein, lacking any signal sequence was also translocated into the ER despite the discontinuity from the peptide backbone of the upstream protein. Possible interpretation of these results is that the second protein, lacking any signal, ‘slipstreamed’ through the ribosome–translocon complex formed by the first protein containing the localization signal. uORFs are probably the best studied subclass of regulatory small open reading frames (sORFs) found in eukaryotic genomes. Over the past few years it has become evident that many of the sORFs could encode small proteins involved in regulation of protein-protein interactions, enzymatic activity, signaling and cell-cell communication (reviewed in (56)). It was also demonstrated that the Drosophila polar granule component (pgc), which is a short protein of 71 amino acids, could regulate recruitment and localization of the kinase P-TEFb to transcription sites (57,58). While the main regulatory mechanism by which uORFs function as repressive genetic elements is by the act of translation per se, one cannot preclude that the short proteins they encode also have a regulatory function. Very little is known about the expression of the putative short protein encoded by the var2csa uORF. Nonetheless, our data strongly supports the presence of a cellular localization signal that determines the fate of the dORF, however, how it functions and regulates these processes requires further investigation.  

# SUPPLEMENTARY DATA  

Supplementary Data are available at NAR online.  

# ACKNOWLEDGEMENTS  

R.D. is also supported by the Dr Louis M. Leland and Ruth M. Leland Chair in Infectious Diseases. Y.F. was supported by Dr Emanuel & Mrs Olga Lourie PhD Scholarships in Tropical Medicine.  

# FUNDING  

Israeli Academy for Science, Israel Science Foundation (ISF) Grant [141/13] (in part); European Research Council (erc.europa.eu) Consolidator Grant [615412 to R.D.] (in part). Funding for open access charge: ERC grant. Conflict of interest statement. None declared.  

# REFERENCES  

1. WHO (2010) World Malaria Report 2010. World Health Organization, Geneva.   
2. Pasternak,N.D. and Dzikowski,R. (2008) PfEMP1: an antigen that plays a key role in the pathogenicity and immune evasion of the malaria parasite Plasmodium falciparum. Int. J. Biochem. Cell Biol., 41, 1463–1466.   
3. Su,X.Z., Heatwole,V.M., Wertheimer,S.P., Guinet,F., Herrfeldt,J.A., Peterson,D.S., Ravetch,J.A. and Wellems,T.E. (1995) The large diverse gene family var encodes proteins involved in cytoadherence and antigenic variation of Plasmodium falciparum-infected erythrocytes. Cell, 82, 89–100.   
4. Scherf,A., Hernandez-Rivas,R., Buffet,P., Bottius,E., Benatar,C., Pouvelle,B., Gysin,J. and Lanzer,M. (1998) Antigenic variation in malaria: in situ switching, relaxed and mutually exclusive transcription of var genes during intra-erythrocytic development in Plasmodium falciparum. EMBO J., 17, 5418–5426.   
5. Chen,Q., Fernandez,V., Sundstrom,A., Schlichtherle,M., Datta,S., Hagblom,P. and Wahlgren,M. (1998) Developmental selection of var gene expression in Plasmodium falciparum. Nature, 394, 392–395.   
6. Smith,J.D., Chitnis,C.E., Craig,A.G., Roberts,D.J., Hudson-Taylor,D.E., Peterson,D.S., Pinches,R., Newbold,C.I. and Miller,L.H. (1995) Switches in expression of Plasmodium falciparum var genes correlate with changes in antigenic and cytoadherent phenotypes of infected erythrocytes. Cell, 82, 101–110.   
7. Smith,J.D., Rowe,J.A., Higgins,M.K. and Lavstsen,T. (2013) Malaria’s deadly grip: cytoadhesion of Plasmodium falciparum-infected erythrocytes. Cell Microbiol., 15, 1976–1983.   
8. Turner,L., Lavstsen,T., Berger,S.S., Wang,C.W., Petersen,J.E., Avril,M., Brazier,A.J., Freeth,J., Jespersen,J.S., Nielsen,M.A. et al. (2013) Severe malaria is associated with parasite binding to endothelial protein C receptor. Nature, 498, 502–505.   
9. Hviid,L. and Salanti,A. (2007) VAR2CSA and protective immunity against pregnancy-associated Plasmodium falciparum malaria. Parasitology, 134, 1871–1876.   
10. Salanti,A., Staalsoe,T., Lavstsen,T., Jensen,A.T., Sowa,M.P., Arnot,D.E., Hviid,L. and Theander,T.G. (2003) Selective upregulation of a single distinctly structured var gene in chondroitin sulphate A-adhering Plasmodium falciparum involved in pregnancy-associated malaria. Mol. Microbiol., 49, 179–191.   
11. Viebig,N.K., Gamain,B., Scheidig,C., Lepolard,C., Przyborski,J., Lanzer,M., Gysin,J. and Scherf,A. (2005) A single member of the Plasmodium falciparum var multigene family determines cytoadhesion to the placental receptor chondroitin sulphate A. EMBO Rep., 6, 775–781.   
12. Fried,M. and Duffy,P.E. (2015) Designing a VAR2CSA-based vaccine to prevent placental malaria. Vaccine, 33, 7483–7488.   
13. Amulic,B., Salanti,A., Lavstsen,T., Nielsen,M.A. and Deitsch,K.W. (2009) An upstream open reading frame controls translation of var2csa, a gene implicated in placental malaria. PLoS Pathog., 5, e1000256.   
14. Bancells,C. and Deitsch,K.W. (2013) A molecular switch in the efficiency of translation reinitiation controls expression of var2csa, a gene implicated in pregnancy-associated malaria. Mol. Microbiol., 90, 472–488.   
15. Chan,S., Frasch,A., Mandava,C.S., Ch’ng,J.H., Quintana,M.D.P., Vesterlund,M., Ghorbal,M., Joannin,N., Franzen,O., Lopez-Rubio,J.J. et al. (2017) Regulation of PfEMP1-VAR2CSA translation by a Plasmodium translation-enhancing factor. Nat. Microbiol., 2, 17068.   
16. Aley,S.B., Sherwood,J.A. and Howard,R.J. (1984) Knob-positive and knob-negative Plasmodium falciparum differ in expression of a strain-specific malarial antigen on the surface of infected erythrocytes. J. Exp. Med., 160, 1585–1590.   
17. Calderwood,M.S., Gannoun-Zaki,L., Wellems,T.E. and Deitsch,K.W. (2003) Plasmodium falciparum var genes are regulated by two regions with separate promoters, one upstream of the coding region and a second within the intron. J. Biol. Chem., 278, 34125–34132.   
18. Dzikowski,R. and Deitsch,K.W. (2008) Active transcription is required for maintenance of epigenetic memory in the malaria parasite Plasmodium falciparum. J. Mol. Biol., 382, 288–297.   
19. Dzikowski,R., Frank,M. and Deitsch,K. (2006) Mutually exclusive expression of virulence genes by malaria parasites is regulated independently of antigen production. PLoS Pathog, 2, e22.   
20. Deitsch,K.W., del Pinal,A. and Wellems,T.E. (1999) Intra-cluster recombination and var transcription switches in the antigenic variation of Plasmodium falciparum. Mol. Biochem. Parasitol., 101, 107–116.   
21. Deitsch,K., Driskill,C. and Wellems,T. (2001) Transformation of malaria parasites by the spontaneous uptake and expression of DNA from human erythrocytes. Nucleic Acids Res., 29, 850–853.   
22. Pillai,A.D., Nguitragool,W., Lyko,B., Dolinta,K., Butler,M.M., Nguyen,S.T., Peet,N.P., Bowlin,T.L. and Desai,S.A. (2012) Solute restriction reveals an essential role for clag3-associated channels in malaria parasite nutrient acquisition. Mol. Pharmacol., 82, 1104–1114.   
23. Kyes,S., Pinches,R. and Newbold,C. (2000) A simple RNA analysis method shows var and rif multigene family expression patterns in Plasmodium falciparum. Mol. Biochem. Parasitol., 105, 311–315.   
24. Frank,M., Dzikowski,R., Costantini,D., Amulic,B., Berdougo,E. and Deitsch,K. (2006) Strict pairing of var promoters and introns is required for var gene silencing in the malaria parasite Plasmodium falciparum. J. Biol. Chem., 281, 9942–9952.   
25. Crowley,V.M., Rovira-Graells,N., de Pouplana,L.R. and Cort´es,A. (2011) Heterochromatin formation in bistable chromatin domains controls the epigenetic repression of clonally variant Plasmodium falciparum genes linked to erythrocyte invasion. Mol. Microbiol., 80, 391–406.   
26. Dahan-Pasternak,N., Nasereddin,A., Kolevzon,N., Pe’er,M., Wong,W., Shinder,V., Turnbull,L., Whitchurch,C.B., Elbaum,M. and Gilberger,T.W. (2013) PfSec13 is an unusual chromatin-associated nucleoporin of Plasmodium falciparum that is essential for parasite proliferation in human erythrocytes. J. Cell Sci., 126, 3055–3069.   
27. Bermudez-Hernandez,K., Keegan,S., Whelan,D.R., Reid,D.A., Zagelbaum,J., Yin,Y., Ma,S., Rothenberg,E. and Fenyo,D. (2017) A method for quantifying molecular interactions using stochastic modelling and Super-Resolution microscopy. Scientific Rep., 7, 14882.   
28. Ovesny,M., Krizek,P., Borkovec,J., Svindrych,Z. and Hagen,G.M. (2014) ThunderSTORM: a comprehensive ImageJ plug-in for PALM and STORM data analysis and super-resolution imaging. Bioinformatics, 30, 2389–2390.   
29. Fannon,A.M., Sherman,D.L., Ilyina-Gragerova,G., Brophy,P.J., Friedrich,V.L. Jr and Colman,D.R. (1995) Novel E-cadherin-mediated adhesion in peripheral nerve: Schwann cell architecture is stabilized by autotypic adherens junctions. J. Cell Biol., 129, 189–202.   
30. Fastman,Y., Noble,R., Recker,M. and Dzikowski,R. (2012) Erasing the epigenetic memory and beginning to switch–the onset of antigenic switching of var genes in Plasmodium falciparum. PLoS One, 7, e34168.   
31. Avraham,I., Schreier,J. and Dzikowski,R. (2012) Insulator-like pairing elements regulate silencing and mutually exclusive expression in the malaria parasite Plasmodium falciparum. Proc. Natl. Acad. Sci. U.S.A., 109, E3678–E3686.   
32. Epp,C., Raskolnikov,D. and Deitsch,K.W. (2008) A regulatable transgene expression system for cultured Plasmodium falciparum parasites. Malar. J., 7, 86.   
33. Eshar,S., Allemand,E., Sebag,A., Glaser,F., Muchardt,C., Mandel-Gutfreund,Y., Karni,R. and Dzikowski,R. (2012) A novel Plasmodium falciparum SR protein is an alternative splicing factor required for the parasites’ proliferation in human erythrocytes. Nucleic Acids Res., 40, 9903–9916.   
34. Ukaegbu,U.E., Kishore,S.P., Kwiatkowski,D.L., Pandarinath,C., Dahan-Pasternak,N., Dzikowski,R. and Deitsch,K.W. (2014) Recruitment of PfSET2 by RNA polymerase II to variant antigen encoding loci contributes to antigenic variation in P. falciparum. PLoS Pathog., 10, e1003854.   
35. Heinberg,A., Siu,E., Stern,C., Lawrence,E.A., Ferdig,M.T., Deitsch,K.W. and Kirkman,L.A. (2013) Direct evidence for the adaptive role of copy number variation on antifolate susceptibility in Plasmodium falciparum. Mol. Microbiol., 88, 702–712.   
36. Hill,D.A., Pillai,A.D., Nawaz,F., Hayton,K., Doan,L., Lisk,G. and Desai,S.A. (2007) A blasticidin S-resistant Plasmodium falciparum mutant with a defective plasmodial surface anion channel. Proc. Natl. Acad. Sci. U.S.A., 104, 1063–1068.   
37. Nguitragool,W., Bokhari,A.A., Pillai,A.D., Rayavara,K., Sharma,P., Turpin,B., Aravind,L. and Desai,S.A. (2011) Malaria parasite clag3 genes determine channel-mediated nutrient uptake by infected red blood cells. Cell, 145, 665–677.   
38. Mira-Martinez,S., Rovira-Graells,N., Crowley,V.M., Altenhofen,L.M., Llinas,M. and Cortes,A. (2013) Epigenetic switches in clag3 genes mediate blasticidin S resistance in malaria parasites. Cell Microbiol., 15, 1913–1923.   
39. Sharma,P., Wollenberg,K., Sellers,M., Zainabadi,K., Galinsky,K., Moss,E., Nguitragool,W., Neafsey,D. and Desai,S.A. (2013) An epigenetic antimalarial resistance mechanism involving parasite genes linked to nutrient uptake. J. Biol. Chem., 288, 19429–19440.   
40. Bokhari,A.A., Solomon,T. and Desai,S.A. (2008) Two distinct mechanisms of transport through the plasmodial surface anion channel. J. Membr. Biol., 226, 27–34.   
41. Kulzer,S., Gehde,N. and Przyborski,J.M. (2009) Return to sender: use of Plasmodium ER retrieval sequences to study protein transport in the infected erythrocyte and predict putative ER protein families. Parasitol. Res., 104, 1535–1541.   
42. Barbosa,C., Peixeiro,I. and Romao,L. (2013) Gene expression regulation by upstream open reading frames and human disease. PLoS Genet., 9, e1003529.   
43. Caro,F., Ahyong,V., Betegon,M. and DeRisi,J.L. (2014) Genome-wide regulatory dynamics of translation in the Plasmodium falciparum asexual blood stages. eLife, 3, 04106.   
44. Yamaguchi,H., Yamamoto,C. and Tanaka,N. (1965) Inhibition of protein synthesis by blasticidin S. I. Studies with cell-free systems from bacterial and mammalian cells. J. Biochem., 57, 667–677.   
45. Falnes,P.O. and Sandvig,K. (2000) Penetration of protein toxins into cells. Curr. Opin. Cell Biol., 12, 407–413.   
46. Lavstsen,T., Salanti,A., Jensen,A.T.R., Arnot,D.E. and Theander,T.G. (2003) Sub-grouping of Plasmodium falciparum 3D7 var genes based on sequence analysis of coding and non-coding regions. Malaria J., 2, 27.   
47. Marti,M., Good,R.T., Rug,M., Knuepfer,E. and Cowman,A.F. (2004) Targeting malaria virulence and remodeling proteins to the host erythrocyte. Science, 306, 1930–1933.   
48. Hiller,N.L., Bhattacharjee,S., van Ooij,C., Liolios,K., Harrison,T., Lopez-Estrano,C. and Haldar,K. (2004) A host-targeting signal in virulence proteins reveals a secretome in malarial infection. Science, 306, 1934–1937.   
49. Boddey,J.A., Hodder,A.N., Gunther,S., Gilson,P.R., Patsiouras,H., Kapp,E.A., Pearce,J.A., de Koning-Ward,T.F., Simpson,R.J., Crabb,B.S. et al. (2010) An aspartyl protease directs malaria effector proteins to the host cell. Nature, 463, 627–631.   
50. Russo,I., Babbitt,S., Muralidharan,V., Butler,T., Oksman,A. and Goldberg,D.E. (2010) Plasmepsin V licenses Plasmodium proteins for export into the host erythrocyte. Nature, 463, 632–636.   
51. de Koning-Ward,T.F., Gilson,P.R., Boddey,J.A., Rug,M., Smith,B.J., Papenfuss,A.T., Sanders,P.R., Lundie,R.J., Maier,A.G.,Cowman,A.F. et al. (2009) A newly discovered protein export machine in malaria parasites. Nature, 459, 945–949.   
52. Spielmann,T. and Gilberger,T.W. (2015) Critical steps in protein export of Plasmodium falciparum blood stages. Trends Parasitol., 31, 514–525.   
53. Melcher,M., Muhle,R.A., Henrich,P.P., Kraemer,S.M., Avril,M., Vigan-Womas,I., Mercereau-Puijalon,O., Smith,J.D. and Fidock,D.A. (2010) Identification of a role for the PfEMP1 semi-conserved head structure in protein trafficking to the surface of Plasmodium falciparum infected red blood cells. Cell Microbiol., 12, 1446–1462.   
54. Rask,T.S., Hansen,D.A., Theander,T.G., Gorm Pedersen,A. and Lavstsen,T. (2010) Plasmodium falciparum erythrocyte membrane protein 1 diversity in seven genomes–divide and conquer. PLoS Comput Biol, 6, e1000933.   
55. de Felipe,P. and Ryan,M.D. (2004) Targeting of proteins derived from self-processing polyproteins containing multiple signal sequences. Traffic, 5, 616–626.   
56. Cabrera-Quio,L.E., Herberg,S. and Pauli,A. (2016) Decoding sORF translation - from small proteins to gene regulation. RNA Biol., 13, 1051–1059.   
57. Timinszky,G., Bortfeld,M. and Ladurner,A.G. (2008) Repression of RNA polymerase II transcription by a Drosophila oligopeptide. PLoS One, 3, e2506.   
58. Hanyu-Nakamura,K., Sonobe-Nojima,H., Tanigawa,A., Lasko,P. and Nakamura,A. (2008) Drosophila Pgc protein inhibits P-TEFb recruitment to chromatin in primordial germ cells. Nature, 451, 730–733.  