# DDX3 Activates CBC-eIF3–Mediated Translation of uORF-Containing Oncogenic mRNAs to Promote Metastasis in HNSCC  

Hung-Hsi Chen, Hsin-I $\mathsf{Y u}^{1}$ , Muh-Hwa Yang, and Woan-Yuh Tarn  

# Abstract  

Mutated or dysregulated DDX3 participates in the progression and metastasis of cancer via its multiple roles in regulating gene expression and cellular signaling. Here, we show that the high expression levels of DDX3 in head and neck squamous cell carcinoma (HNSCC) correlate with lymph node metastasis and poor prognosis and demonstrate that DDX3 is essential for the proliferation, invasion, and metastasis of oral squamous cell carcinoma (OSCC) cells. Microarray analyses revealed that DDX3 is required for the expression of a set of prometastatic genes, including ATF4-modulated genes in an aggressive OSCC cell line. DDX3 activated translation of ATF4 and a set of its downstream targets, all of which contain upstream open reading frames (uORF). DDX3 promoted translation of these targets, likely by skipping the inhibitory uORF. DDX3 specifi- cally enhanced the association of the cap-binding complex (CBC) with uORF-containing mRNAs and facilitated recruitment of the eukaryotic initiation factor 3 (eIF3). CBC and certain eIF3 subunits contributed to the expression of metastatic-related gene expression. Taken together, our results indicate a role for the novel DDX3–CBC–eIF3 translational complex in promoting metastasis.  

![](images/33300cff6989d9aaf4af77b8a13192bfcdb0a58cc2331a7976f1275d94270bd2.jpg)  

Significance: The discovery of DDX3-mediated expression of oncogenic uORF-containing genes expands knowledge on translational control mechanisms and provides potential targets for cancer therapy.  

Graphical Abstract: http://cancerres.aacrjournals.org/content/canres/78/16/4512/F1.large.jpg Cancer Res; 78(16); 4512–23. 2018 AACR.  

# Introduction  

Head and neck squamous cell carcinoma (HNSCC) is one of the most common cancers developed from mucosal linings of the upper aerodigestive tract (1). Oral squamous cell carcinoma (OSCC) is an aggressive HNSCC and is the fifth leading cause of cancer-related death in Taiwan. The poor prognosis and a low survival rate of HNSCC result from frequent local metastasis to cervical lymph nodes (2). Studies have revealed multiple molecular pathways involved in the pathogenesis of HNSCC and identified gene expression or genetic alteration signatures for metastatic HNSCC (3–6), yet the detailed mechanisms of HNSCC metastasis remain unclear. Unraveling the pathogenesis and mechanisms of HNSCC would provide new insights into therapeutic strategies.  

DDX3 is a member of the DEAD-box RNA helicase family involved in multiple steps of RNA metabolism from transcription to translation control. DDX3 can also directly participate in cellular signaling process, including the innate immune response and Wnt signaling (7). Human DDX3 may facilitate translation via its helicase activity and its interaction with factors of the translation initiation complex such as eIF4F (8). We and others previously reported that DDX3 facilitates the translation of mRNAs containing a complex $5^{\prime}$ untranslated region (UTR) perhaps by resolving $5^{\prime}$ UTR structures or remodeling ribonucleoprotein complexes (8–12). Nonetheless, the effects of DDX3 on global translation are controversial probably due to cell types (13, 14). Overexpression of DDX3 has been observed in several cancers although its functions and prognostic effects seem diversified in different cancers (15). A recent study showed that the nonsteroidal anti-inflammatory drug Ketorolac can inhibit 4-nitroquinoline-1-oxide–induced OSCC in mice by inhibiting DDX3 activity (16). Doxorubicin also acts as a DDX3 inhibitor (17) and exerts potent anticancer activity against EGFR inhibitorinsensitive HNSCC (18). Still, the exact function of DDX3 in NHSCC awaits more detailed investigation.  

In this study, we observed upregulated DDX3 expression in HNSCC and the correlation between DDX3 level and survival rate. We therefore investigated the function of DDX3 in OSCC cell lines. We identified the transcription factor ATF4 as a potential translational target of DDX3. ATF4 plays roles in metabolic adaptation and cancer cell progression (19, 20), and its expression can be induced under conditions of endoplasmic reticulum (ER) stress. Notably, ATF4 mRNA contains three upstream openreading frames (uORF), and uORF-mediated translation is one of the mechanisms underlying translational reprogramming during environmental stresses (21). Thus, our preliminary observation prompted us to explore the potential role of DDX3 in regulating uORF-mediated translation and its underlying mechanism and biological significance. This study reveals a novel translation program enhanced by DDX3 in HNSCC.  

accordance with ethical guidelines of Declaration of Helsinki. Tumor samples of head and neck squamous cell carcinoma (HNSCC) were collected from 109 Taiwan residents receiving treatment at Taipei Veterans General Hospital between 2007 and 2014. The inclusion criteria for patients were pathologically diagnosed HNSCC. The histology of each collected specimen was interpreted by certificated pathologists, and the tumor staging was determined according to the American Joint Committee on Cancer staging manual $7^{\mathrm{th}}$ edition. Clinical parameters were obtained by chartreview and confirmed by surgeons. Themedian follow-up time was 1001 days (ranging from 26 to 2743 days) and the follow-up period ended February 2015.  

# Immunohistochemistry  

Immunohistochemistry was performed as previously described except that anti-DDX3 (1:800; Santa Cruz Biotechnology, sc-365768) was used here (22). The intensity of DDX3 staining in cancer cells were classified as 0 (negative), 1 (low), 2 (moderate), and 3 (high) by the consensus of two observers (H.-H. Chen and H.-I. Yu). The samples exhibited more than $50\%$ of moderate and/or $20\%$ high DDX3 were classified as high DDX3 staining and the remaining was as low DDX3 staining.  

# Cell culture  

The human oral cancer cell lines SAS and OECM1 were obtained from Cheng-Chi Chang (National Taiwan University, Taipei, Taiwan) and Kuo-Wei Chang (National Yang-Ming University, Taipei, Taiwan), respectively, in 2011. The cell line authentication and cell culture were as described (22). The SAS/Luc2 stable cell lines were established as described (23) by using lentivirus kindly provided by Dr. Pei-Wen Hsiao (Academia Sinica, Taipei, Taiwan). The SAS/Luc2 cells were transfected with Tet-pLKO-puro (Addgene plasmid #21915) carrying a control or DDX3 shRNA and selected with puromycin to develop tetracycline-inducible knockdown cell lines (Tet-shC and Tet-shD). Transfection was performed with Lipofectamine 2000. No Mycoplasma contamination was detected by using the EZ-PCR Mycoplasma test kit (Biological Industries) every 6 months. All cell lines used for experiments were not cultured for more than 30 passages.  

# Materials and Methods  

# Constructs and siRNAs  

To generate the knockdown vectors, gene-specific shRNA sequences were constructed into the pAAVEMBL-CB-EGFP vector (12). For tetracycline-inducible knockdown cell lines, the TetpLKO-puro vector (Addgene) was adopted. The sense sequences of shRNA and siRNA are list in Supplementary Table S1. To generate the overexpression vectors, the coding sequences were constructed in frame with the sequence encoding FLAG epitope, the porcine teschovirus-1 2A peptide and EGFP in pcDNA3.1 (ThermoFisher). The in vivo translation reporters contained an inframe fusion of the main ORF initiation codon and the humanized Renilla luciferase in psiCHEK-2 (Promega). The humanized firefly luciferase in the same psiCHEK-2 vector was used for transfection reference. The luciferase activity was measured using the Dual-Luciferase Reporter Assay System (Promega).  

# Ethics approvals and study subjects  

The Institutional Review Board of the Taipei Veterans General Hospital (Taipei, Taiwan, ROC) approved the study (No. TVGHIRB-2014-008-005BC). The study was conducted in  

# Cell growth assay  

To calculate cumulative growth curves, cells transfected with siRNA for 48 hours were suspended and seeded in 96-well plates with 3,000 cells per well. Cell numbers were measured every 24 hours by using Cell Counting Kit-8 (Enzo Life Sciences). For relative cell growth rate, transfected cells were seeded in a 24-well plate with $2^{\stackrel{.}{\times}10^{4}}$ cells per well, and cell numbers were counted 72 hours after seeding using the Luna Automated Cell Counter (Logos Biosystems). The quantitative data were analyzed by the Student t test.  

# Boyden chamber assay  

SAS cells transfected with siRNA or overexpression plasmid for 48 hours or knockdown plasmid for 72 hours were seeded into 24-well Millicell inserts ( $8{\cdot}\upmu\mathrm{m}$ -pore, Merck) with $2\,\times\,10^{4}$ cells per insert. Cells were cultured for further 72 hours in DMEM containing $1\%$ FBS to suppress cell growth. Cells in the lower side of the member was stained and determined as described (11). The quantitative data were analyzed by the Student t test.  

# 3D tumor spheroid invasion assay  

SAS or OECM1 cells were dispensed into 96-well Clear Round Bottom Ultra Low Attachment Microplates (Corning) with 3,000 or 10,000 cells, respectively, in $50~\upmu\mathrm{L}$ of growth medium per well. The plates were centrifuged at $200\,\times\,g$ for 3 minutes in a swinging bucket rotor and then kept in cell culture incubators for 72 hours for tumor spheroid formation. The plates were cooled at $4^{\circ}\mathrm{C}$ for 15 minutes, and then $50\,\textrm{\textmu L}$ ice-cold Matrigel Growth Factor Reduced (Corning) was carefully dispensed into wells to ensure that spheroids remained in the center of wells. The plates were placed in cell culture incubators for 1 hours to solidify Matrigel, and then $100\,\mathrm{\AL}$ of fresh growth medium was added on top of the Matrigel. Images of tumor spheroids were acquired under a microscope for phase and/or EGFP. The plates were then kept in cell culture incubators to assess spheroid invasion, and images of tumor spheroids were acquired under the same microscope. The invasion activity was measured as the areas of invasive cells out of the spheroid. The EGFP intensity of tumor spheroids and invasion area was quantified with ImageJ. The quantitative data were analyzed by the Student $t$ test.  

# Orthotopic metastasis assay  

Academia Sinica Institutional Animal Care and Use Committee approved the experiments. SAS-derived stable cells were suspended in PBS, and ${10}^{5}$ cells in $10~\upmu\mathrm{L}$ were then orthotopically injected into 8-week-old BALB/c nude mice (National Laboratory Animal Center, Taipei, Taiwan, ROC) anesthetized with $0.2~\mathrm{mg}$ Zoletil and $4~\upmu\mathrm{g}$ Dexdomitor. To visualize the size of primary tumors in tongue and metastatic tumors in cervical lymph nodes, mice were intraperitoneally injected with 150 $\mathrm{mg/kg}$ VivoGlow Luciferin (Promega) and then monitored every 7 days with the Spectrum in vivo imaging system (PerkinElmer). For tetracycline-inducible cells, $50\,\upmu\mathrm{g/mL}$ doxycycline was supplied in drinking water. For AAV-mediated DDX3 knockdown, mice pre-injected with SAS/Luc2 cells for 1 week were equally divided into two groups according to luciferase activity. The mice were then intratumorally injected with $10^{10}$ vector genomes of AAV-shC or AAV-shD. Bioluminescence of tongue and cervical lymph nodes was quantified as total flux $\mathrm{(photon/s)}$ ). The quantitative data were analyzed by the Student $t$ test.  

# Microarray analysis  

The microarray analysis was performed with two pairs of independently prepared RNA samples as follows. SAS cells transfected with siC or $\mathrm{siD}\#1$ for 48 hours were collected for RNA extraction. The microarray experiments using the SurePrint G3 Human Gene Expression v3 array kit (Agilent) were performed by Welgene Biotech (Taipei, Taiwan). Fluorescence signals were normalized by the rank consistent LOWESS, and values generated from the microarray scanning were first filtered for those higher than background. Ratios of gene expression between siC- and $\mathrm{siD}\#1$ -transfected cells were converted to $\mathrm{log}_{2}$ values, and probes with $\mathrm{log}_{2}$ values larger than 0.7 or smaller than $-0.7$ were selected. The $\log_{2}$ values from two independent microarray data were then averaged; average values smaller than $-1$ or larger than 1 were defined as downregulated or upregulated, respectively. The microarray data were deposited in the NCBI GEO database (GSE113183).  

# Polysome fractionation, immunoprecipitation, RNA immunoprecipitation and RT-qPCR  

Polysome fractionation, immunoprecipitation, RNA immunoprecipitation (RIP) and RT-qPCR were performed as previously described except SAS cells were used here (10, 11). Antibodies (Santa Cruz Biotechnology) against DDX3 (sc-365768), CBP20 (sc-137123), eIF4E (sc-9976), or normal mouse IgG (sc-2025, as control) were used for IP. The RT-qPCR values of target mRNAs were first normalized to that of GAPDH mRNA and then the control (siC, control vectors or mock treatments) so that the control values are 1 for those bar charts.  

# Results  

# Oncogenic role for DDX3 in promoting tumorigenesis of HNSCC  

To gain insights into the role of DDX3 in HNSCC, we performed immunohistochemistry staining for DDX3 in 21 pairs of HNSCC sections and adjacent normal epidermis and 79 nonpaired HNSCC sections. DDX3 staining was generally stronger (15/21) in the HNSCC tissues than in the paired normal tissues (Fig. 1A). Moreover, DDX3 was detected in the nucleus of normal cells $(100\%)$ but was predominantly in the cytosol of most HNSCC samples ( $88\%;$ Fig. 1A). High level of DDX3 is associated with poor patient survival (Fig. 1B and C). Furthermore, analysis of clinicopathologic characteristics of patients with HNSCC revealed a positive correlation between DDX3 protein level and lymph node metastasis (N value) as well as stage, suggesting the oncogenic potential of DDX3 in HNSCC (Table 1). The multivariable analysis indicated that DDX3 is an independent predictor of survival (Supplementary Table S2).  

Next, we evaluated the functions of DDX3 in two OSCC cell lines, SAS and OECM1. SAS cells have greater metastatic potential than OECM1 cells (24). DDX3 level in SAS was 6-fold higher than in OECM1 (Supplementary Fig. S1A). Knockdown of DDX3 using siRNAs (siD) decreased the proliferation and migration rates of both lines (Fig. 2A for SAS, and Supplementary Fig. S1B for OECM1). DDX3 overexpression increased cell migration but had no significant effect on cell growth (Fig. 2B; Supplementary Fig. S1C). 3D invasion assays revealed the invasion potential of SAS but not OECM1 (Supplementary Fig. S1D). Knockdown of DDX3 largely disrupted the invasion activity of SAS cells (Fig. 2C). Transient expression of a DDX3-targeting short hairpin RNA shDDX3 in SAS cells restricted cells at the center of the tumorspheres and prevented cell invasion into Matrigel (Fig. 2D). In contrast, DDX3-overexpressing SAS cells tended to migrate to the periphery of the tumorspheres and had greater invasion activity than control cells (Fig. 2E). Thus, DDX3 likely affects to cell–cell and/or cell–matrix contacts and contributes to cell invasion.  

To evaluate the role of DDX3 in OSCC using an orthotopic mouse model, we established an SAS line that stably expressed the firefly luciferase Luc2 (SAS/Luc2). SAS/Luc2 cells were submucosally injected into the tongue of nude mice. Using in vivo imaging, we observed that cervical node metastasis occurred in all injected mice at 3 weeks (Supplementary Fig. S1E) and that metastatic SAS cells isolated from cervical nodes expressed higher levels of DDX3 (Supplementary Fig. S1F). Next, to conditionally attenuate DDX3 expression, we established SAS/Luc2 cells containing tetracycline-inducible shRNAs (Supplementary Fig. S1G). As compared with the control, SAS cells expressing DDX3 shRNA exhibited decreased tumor growth and metastatic ability in mice (Fig. 2F). To explore the therapeutic potential of targeting DDX3 in OSCC, the mice that had been injected with OSCC for 1 week were intratumorally injected with adeno-associated virus (AAV) carrying an shRNA-expressing vector (Supplementary Fig. S1H). The results revealed that depletion of DDX3 in colonized OSCC tumors (Supplementary Fig. S1I) prevented primary tumor growth as well as cervical node metastasis (Fig. 2G). Together, we demonstrated a role for DDX3 in OSCC metastasis and targeting DDX3 has therapeutic potential.  

![](images/0744fc4d955338f8488f9892afa63f96e793f41e31a47e6f1ba28b0bd7ead9af.jpg)  
Figure 1. DDX3 expression correlates with poor prognosis in HNSCC. A, Representative images of DDX3 staining in normal and tumor samples from the same patient. Enlarged images of insets are shown at right bottom corners. B, Representative images of tumor samples with low or high DDX3 staining. C, Survival curves of patients with HNSCC with low or high DDX3 protein expression. The $P$ value was determined by log-rank test.  

DDX3 modulates the transcriptional networks of ATF4 in OSCC To explore the mechanisms by which DDX3 promotes OSCC invasion and metastasis, we performed genome-wide analysis of mRNA expression in DDX3-depleted SAS. Two independent microarrays showed a high correlation of fold changes in gene expression (Supplementary Fig. S2A). Of 24,391 detected genes in SAS cells, 281 and 389 annotated RNAs (247 and 300 proteincodingmRNAs),respectively,showinga2-foldincreaseordecrease were subjected to further analysis (Supplementary Table S3). Gene ontology (GO) analysis using DAVID (25) revealed that the identified downregulated genes are largely involved in cell movement and DNA metabolism, whereas the upregulated genes have diverse functions with higher $\boldsymbol{p}$ -values (Supplementary Fig. S2B; Supplementary Table S4). Ingenuity Pathways Analysis (IPA) indicated that DDX3 depletion-downregulated genes were associated with decreased cell migration and invasion (Supplementary Fig. S2C). This result coincided with the observed phenotypes of DDX3-depleted cells. Moreover, IPA also revealed potential transcription factors involved in the regulation of the DDX3 target genes (Fig. 3A; Supplementary Table S5). ATF4 was the top downregulated transcription factor upon DDX3 knockdown (Fig. 3A).  

Next, we assessed whether DDX3 regulates the expression of ATF4. We observed that knockdown of DDX3 by siRNAs moderately reduced the mRNA level of ATF4 but greatly diminished its protein level in SAS cells (Fig. 3B). Downregulation of ATF4 was also observed by using DDX3-targeting shRNA (Supplementary Fig. S1G and S1H). Moreover, DDX3 and ATF4 levels were positively correlated in OECM1, SAS and metastatic SAS (Supplementary Fig. S1A and S1F). All these observations indicated that DDX3 positively regulates ATF4 expression. Using RT-qPCR, we confirmed that the expression of known ATF4 targets, namely ATF5, DDIT3, and ASNS, was downregulated in DDX3-depleted OSCC cells (Fig. 3C; Supplementary Fig. S2D), suggesting that DDX3 modulates an ATF4-regulated transcriptional network. DDX3 was also required for ATF4 expression and migration in seven other HNSCC cell lines as well as HeLa cells, suggesting that DDX3 generally activates ATF4 translation regardless of HPV infection or TP53 mutation (Supplementary Fig. S2E and S2F).  

Table 1. Comparison of clinicopathologic characteristics between patients with HNSCC with low and high DDX3 staining   

![](images/16d809fa782a7c1bc84b423fea1eb17cc2b161c949b84c51bdd698157a426b46.jpg)  
$^{\mathrm{a}}P<0.05$ by using the $\chi^{2}$ test.  

![](images/dbc895652e800f307c3be50b728289b64fa5d1f338cc6a39c1ce8aa276e11ba0.jpg)  
Figure 2.   
DDX3 contributes to the growth, migration, invasion, and metastasis of SAS cells. A, Cells transfected with control (siC) or DDX3 (siD) siRNA for 48 hours were subjected to cell growth (left) and Boyden chamber (top and right) assays. B, SAS cells transfected with the control (Con) or DDX3 overexpression vector were subjected to cell growth (left) and Boyden chamber (top and right) assays. C, Cells transfected with siRNA for 48 hours were subjected to the 3D invasion assay. Representative images (top) and quantitative data (bottom) are shown. D, Cells transfected with a plasmid expressing both shRNA and EGFP for 72 hours were subjected to the 3D invasion assay. Images and bar graphs show EGFP signal distribution in tumor spheres (left) and areas of invasion (right). shC and shD indicate control and DDX3 shRNA, respectively. E, SAS cells transfected with a plasmid expressing EGFP alone or EGFP plus DDX3 were subjected to the 3D invasion assay. Data are shown as in D. F, SAS cells stably expressing the luciferase (Luc2) and tetracycline-inducible shRNA were injected into mouse tongue. Bioluminescence signals of primary tumors (tongue) and metastases (lymph node) were quantified with an in vivo imaging system. G, Orthotopic SAS/Luc2 tumors intratumorally injected with AAV carrying the control or DDX3 knockdown shRNA were subjected to bioluminescence analysis as in F. Error bars, F and G, mean $\pm$ SEM; other, mean $\pm\ S\ D$ .  

The DDX3–ATF4 axis contributes to cell migration and invasion Next, we assessed whether the DDX3-ATF4 regulatory axis is responsible for cell migration and invasion of OSCC. ATF4 depletion significantly reduced SAS cell migration and invasion (Fig. 3D, shATF4 $+\mathrm{V}$ and Supplementary Fig. S3A) and also abolished DDX3-promoted cell migration (Fig. 3D, shATF4 DDX3), indicating that the DDX3-ATF4 axis is essential for OSCC cell migration and invasion. Our result was in line with previous reports, indicating that ATF4 is critical for the epithelial– mesenchymal transition (EMT) in avian neural crest and cancer cells (26, 27). Using microarray data, we filtered for mesenchymal and epithelial markers having a greater than 0.5 or less than $-0.5\log_{2}$ fold change and obtained five decreased mesenchymal markers, including ACTA2 ( $\upalpha$ -smooth muscle actin), CDH2 (N-cadherin), FAP (fibroblast activation protein a), SNAI2 (Snail2/slug) and VIM (vimentin) and one increased epithelial marker CDH1 (E-cadherin). RT-qPCR and immunoblotting confirmed that knockdown of DDX3 or ATF4 significantly increased CDH1 and decreased all mesenchymal markers examined (Fig. 3E and F; Supplementary Fig. S3B). Immunofluorescence staining further demonstrated increased E-cadherin and decreased vimentin expression as well as altered cell morphology in DDX3 knockdown SAS (Supplementary Fig. S3C). Moreover, ATF4 knockdown abolished the effects of DDX3 overexpression on EMT-related gene expression (Fig. 3F), indicating that DDX3- activated ATF4 expression is required for the intrinsic EMT program in SAS cells.  

![](images/65ed49700e02a3bd53ed4de33c2ae824c2c5f62691b1729a822a82f47b0f9945.jpg)  
Figure 3. The DDX3–ATF4 axis is involved in the expression of EMT-related genes in SAS cells. A, Upstream transcriptional regulators predicted by IPA. Activation $Z$ -score indicates factors negatively $(<-2)$ or positively $^{(>2)}$ affected by DDX3 knockdown. B, Right, immunoblotting analysis of siRNA-transfected cells. Left, bar graph showing relative levels of ATF4 mRNA (RT-qPCR) and protein (immunoblotting). C, Microarray (left) and RT-qPCR (right) analysis of known ATF4 target genes in siRNA-transfected cells. D, Cells transfected with the indicated shRNA vectors for 72 hours as well as the indicated overexpression vectors (V, control vector) for 48 hours were subjected to the Boyden chamber assay. Shown are representative images (top), immunoblots (bottom left), and quantitative data for migrated cells (bottom right). E, RT-qPCR of EMT-related mRNAs was analyzed in cells transfected with the indicated siRNAs for 48 hours. F, Cells transfected as in D were subjected to RT-qPCR analysis of EMT-related genes. Data in all bar graphs represent the mean $\pm\,S\mathsf{D}$ .  

# DDX3 promotes ATF4 mRNA translation  

We attempted to decipher how DDX3 regulates ATF4 protein expression. First, we observed that DDX3 depletion did not affect ATF4 protein stability (Supplementary Fig. S4). Next, sucrose gradient sedimentation revealed that ATF4 mRNA was shifted from heavier to lighter polysome fractions upon DDX3 knockdown in SAS cells (Fig. 4A), indicating a role for DDX3 in ATF4 mRNA translation. However, DDX3 knockdown neither affected eIF2a phosphorylation nor disrupted thapsigargin-induced ATF4 mRNA translation (Fig. 3B; Supplementary Fig. S5A and S5B), indicating that DDX3-mediated ATF4 mRNA translation control is independent of cellular stress responses. Because DDX3 knockdown moderately reduced the ATF4 mRNA level (Fig. 3B), we  

# Figure 4.  

![](images/b011449fc4c0545b921c717a9f123e30a3e692434b4ecded97f8f57150b3ca2e.jpg)  
DDX3 selectively modulates the translation of uORF-containing mRNAs in SAS cells. A, RT-qPCR analysis of ACTB (middle) and ATF4 (right) mRNAs from polysome fractions (left) of siRNA-transfected cells. B, RT-qPCR (top) and immunoblotting (bottom) of the indicated mRNAs and proteins, respectively, in siRNA-transfected cells. C, Organization of the uORF and protein-coding regions of ATF4 mRNA and reporters. Du0, Du1, and $\Delta u2$ indicate start codon mutations that, respectively, disrupt uORF0, uORF1 and uORF2. ext, stop codon mutations resulting in extended uORFs. D, Cells were transfected with the indicated vectors and control or ATF4 reporter for 48 hours and then subjected to the luciferase assay. E, Bar graph showing percentages of downregulated, upregulated, and nontarget genes identified from DDX3 knockdown microarray overlapping with uTIS- and non-uTIS-containing genes retrieved from the TIS database. F, In vivo translation assay of the reporters was performed in shRNA-transfected SAS cells. pCH, the control reporter. G–I, In vivo translation assays of cells transfected with wild-type (wt) or uORF mutants with indicated shRNAs. Du1u2 indicates mutations of initiation codons of both uORF1 and uORF2. J, In vivo translation assay of the wild-type (wt) and uORF-extended (ext) reporters illustrated in C was performed in shRNA-transfected cells. All assays were performed three times, and data in all bar graphs represent the mean $\pm\ S\ D$ .  

suspected that DDX3 depletion downregulates ATF4 mRNA translation and hence causes nonsense mediated decay (NMD) of ATF4 mRNA (28). Depletion of the NMD factor Upf1 restored the level of ATF4 mRNA in both mock and DDX3-depleted cells (Fig. 4B), as expected, but failed to restore ATF4 protein level in DDX3-knockdown cells (Fig. 4B, $\scriptstyle\sin\#1+\sin\Pi\mathrm{PF}1]$ ), indicating a critical role for DDX3 in ATF4 protein expression.  

The above result implied that DDX3 regulates ATF4 mRNA translation via uORFs. Human ATF4 harbors two conserved uORFs, namely uORF1 and uORF2, and a nonconserved uORF0 encoding only one methionine (Fig. 4C). uORF2 negatively regulates ATF4 mRNA translation (29). We generated a Renilla luciferase (RL) reporter containing the $5^{\prime}$ UTR of ATF4 (Fig. 4C, ATF4-RL). Treatment of SAS cells with thapsigargin activated ATF4-RL expression (Supplementary Fig. S5B), indicating that the ATF4 $5^{\prime}$ UTR was faithfully responsible for stress signalmediated control. Knockdown or overexpression of DDX3 decreased or increased, respectively, ATF4 reporter expression (Fig. 4D), suggesting that DDX3 modulates ATF4 mRNA translation via its uORF-containing $5^{\prime}$ UTR. GST-DDX3 specifically activated in vitro synthesized ATF4 reporter mRNA in the in vitro translation assay, indicating that DDX3 directly activates ATF4 translation (Supplementary Fig. S6A).  

Next, we investigated whether DDX3 modulates the translation of additional uORF-containing transcripts. On the basis of the assumption that inefficient translation of uORF-containing mRNAs triggers their degradation by NMD, we compared siDDX3-downregulated genes with uORF-containing genes in a database for alternative translation initiation (TISdb) in mammalian cells (Supplementary Table S6; ref. 30). The result revealed that $20\%$ of siDDX3-downregulated genes possessed upstream translation initiation sites (uTIS), whereas fewer siDDX3-upregulated genes $(8\%)$ and nontarget genes ${\left(11\%\right)}$ contained uTIS (Fig. 4E). Nevertheless, non-uTIS genes were distributed almost equally in these three categories of genes (Fig. 4E). This analysis indicated that DDX3 may regulate the expression of a set of uORF-containing genes.  

# A positive role for DDX3 in translation of uORF-containing mRNAs  

We selected four cancer-related uORF-containing genes, $A T F5,$ CEBPA, DDIT3, and ETV1 for mechanistic analysis (Fig. 4C). Using the in vivo translation assay, we observed that DDX3 knockdown decreased the translation of the ATF5 and DDIT3 reporters but not two other reporters (Fig. 4F). Similar results were obtained by transfection of in vitro transcribed mRNAs in DDX3 knockdown cells (Supplementary Fig. S6B and S6C), indicating the major effect of DDX3 in protein translation. To evaluate the effect of each uORF in DDX3-responsive $5^{\prime}$ UTRs (i.e., ATF4, ATF5 and DDIT3), we individually mutated their start codons to AGG. The uORF2 mutation $(\Delta u2)$ completely abolished the effect of DDX3 knockdown on ATF4 reporter translation, whereas the uORF0 or uORF1 mutation (Du0 or $\Delta u1$ ) had no effect (Fig. 4G). A similar result was observed with ATF5 (Fig. 4H). Therefore, uORF2 may serve as the critical cis-element for DDX3- regulated ATF4 and ATF5 translation. For DDIT3, mutations of both uORFs $\left(\Delta u l\Delta u2\right)$ necessitated complete inactivation of DDX3-activated translation, suggesting the redundant role of uORF1 and uORF2 in response to DDX3 (Fig. 4I). We deduced from these results that DDX3 counteracts repressive long uORFs by promoting leaky ribosomal scanning and hence enhances the translation of selective uORF-containing mRNAs. To test this hypothesis, we extended the CEBPA uORF1 and ETV1 uORF2 by disrupting their stop codon (Fig. 4C, ext). Knockdown of DDX3 reduced the expression of the ext reporters (Fig.4J), suggesting that the extended uORFs confer DDX3 responsiveness. Nevertheless, the mRNA level of the reporters was neither affected by uORF nor by cotransfected plasmids (Supplementary Fig. S7A–S7G). In conclusion, DDX3 may selectively activate translation of mRNAs containing repressive long uORFs.  

# DDX3 coordinates with the CBC–eIF3 complexes in the translation of uORF-containing mRNAs  

Next, we explored the mechanism of DDX3-regulated uORF-containing mRNA translation. Immunoprecipitation of endogenous DDX3 revealed its interaction with the nuclear cap-binding complex (CBC) and eIF3 complexes. In contrast with the CBC, only a minor fraction of eIF4E coprecipitated with DDX3 (Fig. 5A). The discrimination between the CBC and eIF4E by DDX3 suggested a previously unrecognized mechanism for DDX3-mediated translation control. Using RIP and RT-qPCR analysis, we confirmed that all the transcripts examined, i.e., ATF4, ATF5 and DDIT3, were co-precipitated with DDX3 and CBP20 but minimally with eIF4E (Fig. 5B). RIP of CBP20 or eIF4E in DDX3-depleted cells revealed that DDX3 knockdown abrogated the binding of CBP20 to examined mRNAs but enhanced eIF4E association with these transcripts (Fig. 5C). These results reiterated the intriguing role of the CBC in DDX3-mediated translation of uORF-containing mRNAs.  

To learn more about the potential of the CBC and eIF3 complexes in DDX3-mediated translation, we retrieved mRNAs containing crosslinking and immunoprecipitation (CLIP) tags of DDX3, CBP20 or $\mathrm{eIF3a/b/d/g}$ in the $5^{\prime}$ UTR from the AURA database (31). For comparison, we analyzed CLIP tags of the RNA helicase eIF4A3, which is also associated with CBC-bound messenger ribonucleoproteins (32). Our analysis revealed that $70\%$ to $80\%$ of the CBP20 and eIF3 targets overlapped with those of DDX3, whereas only approximately $50\%$ of them overlapped with targets of eIF4A3 (Supplementary Table S7), indicating that the CBC and eIF3 complexes preferentially associate with DDX3-bound $5^{\prime}$ UTRs. DDX3-specific $5^{\prime}$ UTRs were 2.35-fold and 2.96-fold enriched in CBP20- and eIF3-bound $5^{\prime}$ UTRs, respectively, compared to those of eIF4A3 (Supplementary Fig. S8A). Moreover, DDX3-, CBP20-, or eIF3-associated $5^{\prime}$ UTRs were significantly enriched in the category of uTIS-containing genes (Supplementary Table S8); such enrichment was enhanced when two factors bound to a $5^{\prime}$ UTR (Supplementary Fig. S8B).  

Next, we determined whether the CBC and eIF3 are essential for the translation of uORF-containing mRNAs. Among the 13 eIF3 components, the evolutionarily conserved $\mathrm{eIF3g}$ and eIF3i have been implicated in uORF-regulated translation in yeast (33). The in vivo translation assay revealed that knockdown of CBP20, CBP80, eIF3g or eIF3i diminished the translation of ATF4, ATF5, and DDIT3 reporters, whereas knockdown of eIF4E had no significant effect (Fig. 5D; Supplementary Fig. S8C). This result was confirmed by immunoblotting of endogenous ATF4 in SAS cell lysates (Fig. 5E; Supplementary Fig. S8D). We evaluated additional eIF3 subunits and found that certain of them (eIF3a and eIF3h) but not all were required for ATF4 mRNA translation (Supplementary Fig. S8E and S8F). Overexpression of CBP80 or eIF3i increased ATF4 reporter translation, and this increase was  

# Figure 5.  

![](images/200dcead532b7fed7567ce3e008a64ea97996dac1815b69b7090fb5a70d73b22.jpg)  
DDX3 cooperates with the CBC and eIF3 complexes to activate the translation of uORF-containing mRNAs in SAS cells. A, DDX3 was immunoprecipitated from cell lysates. Coprecipitated initiation factors were examined by immunoblotting. IgG, control mouse IgG. B and C, RIP was performed with mock (B) or siRNAtransfected (C) cell lysates using antibody against DDX3, CBP20, or eIF4E. Bar chart shows RT-qPCR data for the indicated transcripts. D and E, Invivo translation (D) and immunobloting (E) of cells transfected with indicated shRNA. Companion data are shown in Supplementary Fig. S6C. F, Invivo translation assaywas performed in cells that were initially transfected with an shRNA vector (shC or shD) for 48 hours, followed by an overexpression vector (control, CBP80, or eIF3i) and the ATF4 reporter for further 24 hours. G, Immunoprecipitation was performed using anti-eIF3b in siRNA-transfected cells. The bar chart shows results of the quantitative analysis of the indicated proteins. All assays were performed three times, and all bar graphs represent the mean $\pm\,S\ D$ . H, Model depicting DDX3-activated translation of uORF-containing mRNAs through the CBC and eIF3 complexes.  

abolished by DDX3 knockdown (Fig. 5F), indicating that DDX3 is required for CBC- or eIF3-activated ATF4 mRNA translation. We also observed that CBP80 knockdown diminished DDX3- or abolished eIF3-activated ATF4 mRNA translation and also that eIF3i knockdown partially compromised the effect of DDX3 and CBP80 on ATF4 mRNA translation (Supplementary Fig. S8G). Finally, we found that DDX3 knockdown attenuated the interaction between the CBC and the eIF3 complex (Fig. 5G). These results emphasized the interdependency of DDX3, CBC and eIF3 in the translation of uORF-containing mRNAs.  

According to our results, we deduced that DDX3 selects the capbinding protein for translation of different types of mRNAs and that CBC retention on uORF-containing transcripts may promote the translation of the main ORF (Fig. 5H).  

# Figure 6.  

![](images/f97476d390a5cd469bc8040b123c3e4eacb7f6cc1d9ee871861e72fa0510a7eb.jpg)  
The CBC and eIF3 complexes contribute to migration and invasion of SAS cells. A, Cells transfected with indicated shRNA vectors were subjected to cell migration and 3D invasion assays. B, Immunoblotting of the lysates obtained from cells that were transfected with an indicated overexpression vector for 48 hours. C, Experiments were performed as in A for cells transfected with the control, CBP80, or eIF3i expression vector. D, Transfection was as in A and C, followed by RT-qPCR analysis of EMT-related genes. All bar graphs show the mean $\pm\,S\mathsf{D}$ from three experiments. E, Model shows that DDX3 activates the translation of ATF4 and perhaps other uORF-containing mRNAs that upregulate the expression of metastasis-related genes and hence promotes cell metastasis.  

# CBC and eIF3 exhibit proinvasive activities  

Because the CBC and $\mathrm{eIF3g/i}$ were essential for the translation of ATF4 and likely other uORF-containing mRNAs (Fig. 5D and E), we next evaluated their effect on cell migration. Knockdown of a CBC subunit or eIF3i or $\mathrm{eIF3g}$ decreased cell migration in Boyden chambers, and invasion in 3D cultures (Fig. 6A). Overexpression of CBP80 or eIF3i increased the level of ATF4 protein (Fig. 6B) as well as cell migration and invasion (Fig. 6C). Accordingly, knockdown of a CBC or eIF3 component decreased the expression of mesenchymal-related genes but increased CDH1 expression (Fig. 6D, left); the reduction of vimentin was also confirmed by immunoblotting (Fig. 5E, VIM). Instead, overexpression of CBP80 or eIF3i, respectively, increased and decreased the expression of mesenchymal-related genes and CDH1 (Fig. 6D, right).  

Our results indicated that DDX3 acts coordinately with CBC/eIF3 to enhance the translation of ATF4 and other uORFcontaining mRNAs that together modulate EMT programs, and hence promote metastasis (Fig. 6E).  

# Discussion  

We provide evidence that the cellular level of DDX3 is significantly increased in HNSCC samples and its level negatively correlated with patient survival (Fig. 1), supporting a role for DDX3 in HNSCC oncogenesis. DDX3 promoted OSCC cell migration and invasion in vitro and was essential for metastasis in vivo (Fig. 2). A microarray screen revealed that DDX3 knockdown primarily impacted cell migration and proliferation related genes, and particularly the ATF4 pathway. We demonstrated that DDX3 modulates ATF4 irrespective of cell stress and selectively activates the translation of mRNAs bearing inhibitory uORFs via a previously unrecognized mechanism involving the CBC and eIF3 complexes.  

The cancer microenvironment-induced ER stress and integrated stress response enhance the expression of stress proteins for cell survival and even cancer invasion (34). A set of the stress-induced genes, including ATF4, contain uORFs. Under normal conditions, uORFs render inefficient translation and NMD of the transcripts, thereby limiting the expression of key regulators of the stress response and EMT (35). Our results indicated that ATF4 mRNA is indeed susceptible to NMD and that its translation depends on DDX3. DDX3-activated translation of ATF4 mRNA outcompetes NMD so that high levels of DDX3 in OSCC augment ATF4 expression at two different points during gene expression. Furthermore, DDX3 depletion reduced the expression of transcriptional targets of ATF4 (Fig. 3). Some of uORF-containing genes, such as ATF5 and DDIT3, were also subject to DDX3-mediated translation control. ATF5 knockdown indeed disrupted cell invasion in diverse cancer cell lines (36). Thus, DDX3 is able to multiply its effect on activating a specific axis of ATF4 signaling to promote cell invasion (Fig. 6, for the model).  

A recent report described that DDX3 participates in ER stressinduced ATF4 expression in hepatocarcinoma cell lines (37). Nevertheless, our study emphasizes the effect of high DDX3 levels in OSCC cells on sustained ATF4 expression without extrinsic stress. Thus, DDX3 may confer translation control of uORFcontaining mRNAs via different mechanisms under different cellular circumstances. DDX3 upregulation also provides a strategy to activate the translation of uORF-containing stress proteinencoding mRNAs.  

Elevated eIF4F activity is crucial for the tumorigenicity of various types of cancers, including HNSCC (38). The eIF4E/4EBP ratio or mTOR signaling, which can liberate eIF4E from 4E-BP, may determine the eIF4F activity and sensitivity of cancer cells to mTOR inhibition (39). Our present study uncovered DDX3- potentiated CBC activity in promoting the translation of oncogenic uORF-containing mRNAs, including ATF4, in cancer cells. It is noteworthy that mTOR signaling-induced ATF4 mRNA translation can occur in a stress-independent manner (40, 41) and that mTOR-activated S6 kinase can phosphorylate CBP80 to enhance the binding of the CBC to the mRNA cap and hence increase translation efficiency (42). Thus, certain cancer cells may exploit CBC-dependent translation to prevent recognition of uORF initiation codons by eIF4E-dependent translation so that leaky scanning would allow translation of the main ORF (Fig. 5). However, whether mTOR signaling may control the translation of growth factor-activated uORF-containing mRNAs in conjunction with DDX3–CBC remains as an interesting topic for investigation.  

It has been shown that the eIF3 subunits $\mathrm{\bfg}/\mathrm{\bfi}$ can facilitate the translation of uORF-containing mRNAs in yeast (33). The CBCdependent translation initiation factor (CTIF) directly interacts with eIF3g during the pioneer round of translation of an mRNA (43). Our study indicates that DDX3-mediated translation of uORF-containing mRNAs involves specific eIF3 subunits (Fig. 5; Supplementary Fig. S8E). It is possible that DDX3 via specific eIF3 components or eIF3 subcomplexes (44) assists leaky scanning in the $5^{\prime}$ UTR of CBC-bound mRNAs, and thereby overcomes the suppressive effect of the long and main ORF-overlapping uORF on translation.  

This study is the first to report that the CBC and eIF3 subunits may play oncogenic roles in HNSCC. Upregulation of eIF3 subunits has been observed in many other cancers (45, 46), whereas NCBP2 was amplified or upregulated in more than $30\%$ of patients with HNSCC (47), supporting our conclusion. Moreover, we provided evidence demonstrating an oncogenic role for the CBC complex. In conclusion, high levels of DDX3 in OSCC promote cell migration and invasion via its cooperative action with the eIF3 and CBC. Thus, CBC/eIF3-mediated translation is la potential therapeutic target in HNSCC and other cancers.  

# Disclosure of Potential Conflicts of Interest  

No potential conflicts of interest were disclosed.  

# Authors' Contributions  

Conception and design: H.-H. Chen, W.-Y. Tarn   
Development of methodology: H.-H. Chen, H.-I. Yu   
Acquisition of data (provided animals, acquired and managed patients,   
provided facilities, etc.): H.-H. Chen, H.-I. Yu, M.-H. Yang   
Analysis and interpretation of data (e.g., statistical analysis, biostatistics,   
computational analysis): H.-H. Chen, H.-I. Yu   
Writing, review, and/or revision of the manuscript: H.-H. Chen, W.-Y. Tarn   
Administrative, technical, or material support (i.e., reporting or organizing   
data, constructing databases): H.-H. Chen, H.-I. Yu   
Study supervision: W.-Y. Tarn  

# Acknowledgments  

We thank the Core Facilities of the Institute of Biomedical Sciences, Academia Sinica, for their technical assistance and bioinformatics analysis, especially the statistical advice from C.C. Chang. We also thank Dr. Te-Chang Lee (Institute of Biomedical Sciences, Academia Sinica) for providing $\mathrm{Ca9}–22.$ , CAL27, $\mathrm{FaD}\mathbf{u},$ HSC-3, TW2.6, and SCC-4 cell lines and Dr. Cheng-Chia Yu (School of Dentistry, Chung Shan Medical University) for providing GNM cell line. Funding was provided by grant IBMS-CRC104-P01 from the Institute of Biomedical Sciences, Academia Sinica.  

The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked advertisement in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.  

Received January 29, 2018; revised April 26, 2018; accepted June 14, 2018; published first June 19, 2018.  

# References  

1. Vigneswaran N, Williams MD. Epidemiologic trends in head and neck cancer and aids in diagnosis. Oral Maxillofac Surg Clin North Am 2014;26:123–41.   
2. Leemans CR, Braakhuis BJ, Brakenhoff RH. The molecular biology of head and neck cancer. Nat Rev Cancer 2011;11:9–22.   
3. Hedberg $\mathrm{ML},$ Goh G, Chiosea SI, Bauman JE, Freilino $\mathrm{ML},$ Zeng Y, et al. Genetic landscape of metastatic and recurrent head and neck squamous cell carcinoma. J Clin Invest 2016;126:169–80.   
4. Tabatabaeifar S, Thomassen M, Larsen MJ, Larsen SR, Kruse TA, Sorensen JA. The subclonal structure and genomic evolution of oral squamous cell carcinoma revealed by ultra-deep sequencing. Oncotarget 2017;8: 16571–80.   
5. Tonella $\mathrm{L},$ Giannoccaro M, Alfieri S, Canevari S, De Cecco L. Gene expression signatures for head and neck cancer patient stratification: are results ready for clinical application? Curr Treat Options Oncol 2017;18:32.   
6. Sepiashvili $\mathrm{L},$ Bruce JP, Huang SH, O'Sullivan B, Liu FF, Kislinger T. Novel insights into head and neck cancer using next-generation "omic" technologies. Cancer Res 2015;75:480–6.   
7. Ariumi Y. Multiple functions of DDX3 RNA helicase in gene regulation, tumorigenesis, and viral infection. Front Genet 2014;5:423.   
8. Soto-Rifo R, Rubilar PS, Limousin T, de Breyne S, Decimo D, Ohlmann T. DEAD-box protein DDX3 associates with eIF4F to promote translation of selected mRNAs. EMBO J 2012;31:3745–56.   
9. Lai MC, Lee YH, Tarn WY. The DEAD-box RNA helicase DDX3 associates with export messenger ribonucleoproteins as well as tip-associated protein and participates in translational control. Mol Biol Cell 2008;19:3847–58.   
10. Lai MC, Chang WC, Shieh SY, Tarn WY. DDX3 regulates cell growth through translational control of cyclin E1. Mol Cell Biol 2010;30:5444–53.   
11. Chen HH, Yu HI, Cho WC, Tarn WY. DDX3 modulates cell adhesion and motility and cancer cell metastasis via Rac1-mediated signaling pathway. Oncogene 2015;34:2790–800.   
12. Chen HH, Yu HI, Tarn WY. DDX3 modulates neurite development via translationally activating an RNA regulon involved in Rac1 activation. J Neurosci 2016;36:9792–804.   
13. Lee CS, Dias AP, Jedrychowski M, Patel AH, $\mathrm{Hsu\,IL},$ Reed R. Human DDX3 functions in translation and interacts with the translation initiation factor eIF3. Nucleic Acids Res 2008;36:4708–18.   
14. Shih JW, Tsai TY, Chao CH, Wu Lee YH. Candidate tumor suppressor DDX3 RNA helicase specifically represses cap-dependent translation by acting as an eIF4E inhibitory protein. Oncogene 2008;27:700–14.   
15. Bol GM, Xie M, Raman V. DDX3, a potential target for cancer treatment. Mol Cancer 2015;14:188.   
16. Samal SK, Routray S, Veeramachaneni GK, Dash R,Botlagunta M.Ketorolac salt is a newly discovered DDX3 inhibitor to treat oral cancer. Sci Rep 2015;5:9982.   
17. Botlagunta M, Kollapalli B, Kakarla L, Gajarla SP, Gade SP, Dadi CL, et al. In vitro anti-cancer activity of doxorubicin against human RNA helicase, DDX3. Bioinformation 2016;12:347–53.   
18. Bossi P, Bergamini C, Siano M, Cossu Rocca M, Sponghini AP, Favales F, et al. Functional genomics uncover the biology behind the responsiveness of head and neck squamous cell cancer patients to cetuximab. Clin Cancer Res 2016;22:3961–70.   
19. Ameri K, Harris AL. Activating transcription factor 4. Int J Biochem Cell Biol 2008;40:14–21.   
20. Horiguchi M, Koyanagi S, Okamoto A, Suzuki SO, Matsunaga N, Ohdo S. Stress-regulated transcription factor ATF4 promotes neoplastic transformation by suppressing expression of the INK4a/ARF cell senescence factors. Cancer Res 2012;72:395–401.   
21. Robichaud N, Sonenberg N. Translational control and the cancer cell response to stress. Curr Opin Cell Biol 2017;45:102–9.   
22. Hsu DS, Hwang WL, Yuh CH, Chu CH, Ho YH, Chen PB, et al. Lymphotoxin-beta interacts with methylated EGFR to mediate acquired resistance to cetuximab in head and neck cancer. Clin Cancer Res 2017;23:4388–401.   
23. Tsai CH, Tzeng SF, Chao TK, Tsai CY, Yang YC, Lee MT, et al. Metastatic progression of prostate cancer is mediated by autonomous binding of Galectin-4-O-Glycan to cancer cells. Cancer Res 2016;76:5756–67.   
24. Chiou SH, Yu CC, Huang CY, Lin SC, Liu CJ, Tsai TH, et al. Positive correlations of Oct-4 and Nanog in oral cancer stem-like cells and highgrade oral squamous cell carcinoma. Clin Cancer Res 2008;14:4085–95.   
25. Huangda W, Sherman BT, Lempicki RA. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat Protoc 2009;4:44–57.   
26. Suzuki T, Osumi N, Wakamatsu Y. Stabilization of ATF4 protein is required for the regulation of epithelial-mesenchymal transition of the avian neural crest. Dev Biol 2010;344:658–68.   
27. Feng YX, Sokol ES, Del Vecchio CA, Sanduja S, Claessen JH, Proia TA, et al. Epithelial-to-mesenchymal transition activates PERK-eIF2alpha and sensitizes cells to endoplasmic reticulum stress. Cancer Discov 2014;4:702–15.   
28. Lykke-Andersen S, Jensen TH. Nonsense-mediated mRNA decay: an intricate machinery that shapes transcriptomes. Nat Rev Mol Cell Biol 2015;16:665–77.   
29. Lu PD, Harding HP, Ron D. Translation reinitiation at alternative open reading frames regulates gene expression in an integrated stress response. J Cell Biol 2004;167:27–33.   
30. Wan J, Qian SB. TISdb: a database for alternative translation initiation in mammalian cells. Nucleic Acids Res 2014;42:D845–50.   
31. Dassi E, Re A, Leo S, Tebaldi T, Pasini $\mathrm{L},$ Peroni D, et al. AURA 2: Empowering discovery of post-transcriptional networks. Translation 2014;2:e27738.   
32. Choe J, Ryu I, Park OH, Park J, Cho H, Yoo JS, et al. eIF4AIII enhances translation of nuclear cap-binding complex-bound mRNAs by promoting disruption of secondary structures in 50UTR. Proc Natl Acad Sci U S A 2014; 111:E4577–86.   
33. Cuchalova L, Kouba T, Herrmannova A, Danyi I, Chiu WL, Valasek L. The RNA recognition motif of eukaryotic translation initiation factor 3g (eIF3g) is required for resumption of scanning of posttermination ribosomes for reinitiation on GCN4 and together with eIF3i stimulates linear scanning. Mol Cell Biol 2010;30:4671–86.   
34. Clarke HJ, Chambers JE, Liniker E, Marciniak SJ. Endoplasmic reticulum stress in malignancy. Cancer Cell 2014;25:563–73.   
35. Liu B, Qian SB. Translational reprogramming in cellular stress response. Wiley Interdiscip Rev RNA 2014;5:301–15.   
36. Nukuda A, Endoh H, Yasuda M, Mizutani T, Kawabata K, Haga H. Role of ATF5 in the invasive potential of diverse human cancer cell lines. Biochem Biophys Res Commun 2016;474:509–14.   
37. Adjibade P, Grenier St-Sauveur V, Bergeman J, Huot ME, Khandjian EW, Mazroui R. DDX3 regulates endoplasmic reticulum stress-induced ATF4 expression. Sci Rep 2017;7:13832.   
38. Pelletier J, Graff J, Ruggero D, Sonenberg N. Targeting the eIF4F translation initiation complex: a critical nexus for cancer development. Cancer Res 2015;75:250–63.   
39. Alain T, Morita M, Fonseca BD, Yanagiya A, Siddiqui N, Bhat M, et al. eIF4E/4E-BP ratio predicts the efficacy of mTOR targeted therapies. Cancer Res 2012;72:6468–76.   
40. Park Y, Reyna-Neyra A, Philippe L, Thoreen CC. mTORC1 balances cellular amino acid supply with demand for protein synthesis through posttranscriptional control of ATF4. Cell Rep 2017;19:1083–90.   
41. Ben-Sahra I, Hoxhaj G, Ricoult SJH, Asara JM, Manning BD. mTORC1 induces purine synthesis through control of the mitochondrial tetrahydrofolate cycle. Science 2016;351:728–33.   
42. Gonatopoulos-Pournatzis T, Cowling VH. Cap-binding complex (CBC). Biochem J 2014;457:231–42.   
43. Choe J, Oh N, Park S, Lee YK, Song OK, Locker N, et al. Translation initiation on mRNAs bound by nuclear cap-binding protein complex CBP80/20 requires interaction between CBP80/20-dependent translation initiation factor and eukaryotic translation initiation factor 3g. J Biol Chem 2012;287:18500–9.   
44. Valasek LS, Zeman J, Wagner S, Beznoskova P, Pavlikova $Z_{r}$ Mohammad MP, et al. Embraced by eIF3: structural and functional insights into the roles of eIF3 across the translation cycle. Nucleic Acids Res 2017;45:10948–68.   
45. Gomes-Duarte A, Lacerda R, Menezes J, Romao L. eIF3: a factor for human health and disease. RNA Biol 2017;15:26–34.   
46. Grzmil M, Hemmings BA. Translation regulation as a therapeutic target in cancer. Cancer Res 2012;72:3891–900.   
47. Reddy RB, Bhat AR, James $\mathrm{BL},$ Govindan SV, Mathew R, Ravindra DR, et al. Meta-analyses of microarray datasets identifies ANO1 and FADD as prognostic markers of head and neck cancer. PLoS ONE 2016;11:e0147409.  