# Charting DENR-dependent translation reinitiation uncovers predictive uORF features and links to circadian timekeeping via Clock  

Violeta Castelo-Szekely 1, Mara De Matos1, Marina Tusup2,3, Steve Pascolo2,3, Jernej $\mathsf{\mathbf{U}I e}^{4,5}$ and David Gatfield  

1Center for Integrative Genomics, University of Lausanne, Genopode, 1015 Lausanne, Switzerland, 2Department of Dermatology, University Hospital of Zurich, Gloriastrasse 31, 8091 Zurich, Switzerland, 3Faculty of Medicine, University of Zurich, 8091 Zurich, Switzerland, 4Department of Neuromuscular Diseases, UCL Institute of Neurology, Queen Square, London WC1N 3BG, UK and 5The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK  

Received August 28, 2018; Revised March 25, 2019; Editorial Decision March 30, 2019; Accepted April 08, 2019  

# ABSTRACT  

The non-canonical initiation factor DENR promotes translation reinitiation on mRNAs harbouring upstream open reading frames (uORFs). Moreover, DENR depletion shortens circadian period in mouse fibroblasts, suggesting involvement of uORF usage and reinitiation in clock regulation. To identify DENRregulated translation events transcriptome-wide and, in particular, specific core clock transcripts affected by this mechanism, we have used ribosome profiling in DENR-deficient NIH3T3 cells. We uncovered 240 transcripts with altered translation rate, and used linear regression analysis to extract $\mathsf{\pmb{5}^{\prime}}$ UTR features predictive of DENR dependence. Among core clock genes, we identified Clock as a DENR target. Using Clock $\mathsf{\pmb{5}^{\prime}}$ UTR mutants, we mapped the specific uORF through which DENR acts to regulate CLOCK protein biosynthesis. Notably, these experiments revealed an alternative downstream start codon, likely representing the bona fide CLOCK N-terminus. Our findings provide insights into uORF-mediated translational regulation that can regulate the mammalian circadian clock and gene expression at large.  

# INTRODUCTION  

Initiation is considered the rate-limiting step for eukaryotic translation (1) and can be regulated by multiple mechanisms including through upstream open reading frames (uORFs), which are short translated sequences defined by a start codon within the $5^{\prime}$ untranslated region $5^{\prime}$ UTR) of the transcript and an in-frame stop codon. Canonical translation initiation involves association of the small ribosomal subunit with an mRNA’s $5^{\prime}$ cap structure and scanning of the $5^{\prime}$ UTR until a start codon is recognized. In cases when this translation event is on a uORF, the ribosome can resume scanning after termination, allowing reinitiation at the downstream protein coding sequence (CDS). Alternatively, a scanning ribosome might skip certain uORFs, in a process known as leaky scanning. Translation reinitiation and leaky scanning are poorly understood mechanisms that may involve canonical and non-canonical initiation factors and that are likely regulated by specific features of the uORF and within the transcript (reviewed in (2,3)).  

Density-regulated protein (DENR) and Malignant Tcell amplified sequence 1 (MCTS1) form a heterodimeric non-canonical initiation factor that has been described to bind to the small ribosomal subunit and to promote the eIF2-independent recruitment of the initiator tRNA to viral mRNAs, as well as to participate in the recycling of post-termination 40S subunits (4,5). In Drosophila, DENRMCTS1 was identified as a reinitiation factor required for efficient CDS translation on transcripts containing short and strong uORFs (6). Furthermore, based on reporter assays with synthetic constructs containing uORFs of varying lengths and Kozak strengths, endogenous DENR targets have recently been predicted in human cells (7). However, an unbiased identification of DENR targets in vivo and a transcriptome-wide quantification of the effects exerted by DENR on translational regulation are still lacking.  

uORF usage is pervasive in the mammalian transcriptome, with up to $40\%$ of transcripts predicted to contain at least one uORF (8). It generally reduces CDS translation efficiency (TE) because scanning ribosomes are diverted from the main ORF and unlikely all reinitiate (3). We have previously reported that several transcripts encoding components of the circadian clock machinery contained translated uORFs (9). Moreover, DENR loss-offunction––as a means of globally interfering with uORF regulation––shortened the period of free-running circadian oscillations in mouse fibroblasts by up to $1.5\,\mathrm{h}$ (9). However, it has remained elusive on which circadian clock transcripts and through which uORFs DENR can act.  

Here, we have used ribosome profiling in mouse NIH3T3 fibroblasts to identify and quantify DENR-mediated translational regulation transcriptome-wide, and in particular within core clock genes. We have used comprehensive regression analysis to characterize $5^{\prime}$ UTR features associated with DENR-dependent translational regulation and validate targets identified from our translatome-wide approaches. Finally, we show that DENR regulates efficient CLOCK biosynthesis involving a strong uORF and a sofar unannotated CDS initiation codon.  

# MATERIALS AND METHODS  

# Cell culture  

NIH3T3 and HEK293FT cells were cultured under standard conditions (DMEM; $10\%$ FCS, $1\%$ penicillin/streptomycin, all from Invitrogen; $37^{\circ}\mathbf{C}$ ;$5\%$ $\mathbf{CO}_{2}$ ).  

Lentiviral particle production in HEK293FT cells using envelope pMD2.G and packaging psPAX2 plasmids, and viral transduction of NIH3T3 cells, were performed following published protocols (10), with puromycin selection at 5 $\upmu\mathrm{g/ml}$ for 4 days.  

# Cloning and plasmids  

For the generation of lentiviral shRNA expression vectors, two different sequences targeting Denr were cloned into pLKO.1puro backbone vector (Addgene no. 10878 (11)): shRNA1: GTACCACAGAAGGTCACGATA, corresponding to clone TRCN0000308443 of the TRC shRNA Library from the Broad Institute; and shRNA2: GTGCCAAGTTAGATGCGGATT, corresponding to clone TRCN0000098826. pLKO.1puro constructs containing $G\/p$ , and Scramble (Addgene no. 1864) shRNAs served as controls.  

For the generation of dual luciferase (Firefly/Renilla) reporter plasmids, fragments containing the $5^{\prime}$ UTR and first 5-14 codons of the selected DENR targets/controls were amplified by polymerase chain reaction (PCR) from genomic DNA or cDNA, cloned into the BamHI site of the prLV1 dual luciferase reporter plasmid (12), and validated by sequencing. The following primers were used for PCR: Etaa1, forward (for): aaaggatccGACTTGCAAAGATGGCGCTGCAC, reverse (rev): tttggatccactagtGTCCTTCAGCTGCAT TCACATTT; Map2k5, for: aaaggatccGGTTCCGGA GTAACAGCGGTCTAAC, rev: tttggatccactagtGGC CAGCCACAGCATTACAGGTTAA; Lrrc28, for: aaaggatccGCCGCTGAGTGCCGGTCAGCGGGC, rev: tttggatccactagtGATTTCGGATGCCATGACTGAACA; Klhdc8a, for: aaaggatccGTCTCCGACCCTGTAGACA CTGCAG, rev: tttggatccactagtATTGGGCACTTCCAT GGCAGCCCGG; Clock, for: aaaggatccGGGGAGGAG CGCGGCGGTAGCGGTG, rev: tttggatccactagtAACAA TTGAGCTCATTTTACTACAGCTTACGG; Fth1, for: aaaaggatcCAGACGTTCTCGCCCAGAGTCGC, rev: tttggatcccatGGTGGCGGCGGGGCGAGGCGC; Rpl30, for: aaggatcCAGCGGCGGGTGGACAGAGATGCC, rev: tttggatccacgCGTCTTCTTTGCGGCCACCA TCTTC; Deaf1, for: aaggatccGGGCGTCAGA CTGCCTCTTCGGTC, rev: tttggatccacgcgtGGC AGCGGCCGCCACAGCTGCCGC; Cry1, for: aaaaagatctactagtCACCCGGGCAGCCTCGGGACCG, rev: tttagatctTCGGAACCAGTGCACGGCGTTCAC. $5^{\prime}$ UTR mutants were generated by site-directed mutagenesis of the corresponding plasmids, using the following primers (mutated nucleotides in small case with mutated start/stop codon underlined in the forward sequence): Klhdc8a-Mut, forward (for): GGCAGGGAGCCCGTGCAGCGCGgtaccGAGG CTGAGAGAGGGGACGCGCC, reverse (rev): GGCGCGTCCCCTCTCTCAGCCTCggtacCGCG CTGCACGGGCTCCCTGCC; Map2k5-Mut, for: CTCTTACTCACAAGGACTACAGgctagcTTGTGGGT TCTGTTTTGTCCAC, rev: GTGGACAAAACAGAAC CCACAAgctagcCTGTAGTCCTTGTGAGTAAGAG; Etaa1-Mut, for: cgaaggatcCGACTTGCAAAGgtacc GCTGCACGGTGGCGCGCGGGCTC, rev: GAGCCCGCGCGCCACCGTGCAGCggtacCTTT GCAAGTCGgatccttcg; Clock-uORF-Stop-Mut, for: GT TATGGTGTTTACCGTcAGCTGTAGTAAAATGAG, rev: CTCATTTTACTACAGCTgACGGTAAACACC ATAAC; Klhdc8a-uORF-Stop-Mut, for: GCCGACCG GCGCCGTGcacCCCGCGGGCCCCGGGGCT, rev: AGCCCCGGGGCCCGCGGGgtgCACGGCGCCGG TCGGC; Etaa1-uORF-Stop-Mut (small case correspond to an insertion that changes uORF frame), for:TGCT CCGCTGTAGCAgcAAATGTGAATGCAGC, rev: GCTGCATTCACATTTgcTGCTACAGCGGAGCA; Clock-uORF1-Mut, for: CCCAAAATCACCAGCAAG AGTTgctagcGTCAGTCACACAGAAGACGGCC, rev: GGCCGTCTTCTGTGTGACTGACgctagcAACTCT TGCTGGTGATTTTGGG; Clock-uORF2-Mut, for: AAAGTGAAAGAGGAGAAGTACAggtacCTACCAC AAGACGAAAACATAAT, rev: ATTATGTTTTCGTC TTGTGGTAGgtaccTGTACTTCTCCTCTTTCACTTT; Clock-AnnotCDS-Mut, for: CAAGACGAAAACAT AATGTGTTtcGGTGTTTACCGTAAGCTGTAG, rev: CTACAGCTTACGGTAAACACCgaAACACAT TATGTTTTCGTCTTG; Clock-AlterCDS-Mut, for: GTTTACCGTAAGCTGTAGTAAAcaGAGCTCAAT TGTTactagtg, rev: cactagtAACAATTGAGCTCtgTTTA CTACAGCTTACGGTAAAC.  

# iCLIP experiments and analyses  

For iCLIP experiments, we used either NIH3T3 cells without overexpression to immunoprecipitate the endogenous DENR or NIH3T3 cells transiently transfected with Denr cloned into a FLAG-Venus-pLenti6 vector (Addgene 36948). Cells were grown to confluency in a $15{\mathrm{-cm}}$ dish. One dish was used for overexpression, and six for nontransfected experiments.  

iCLIP was performed as described (13,14). RNA– protein complexes were crosslinked with UV-C ( $254\;\mathrm{nm})$ at $\mathbf{\hat{l}}\,50\;\mathrm{mJ}/\mathrm{cm}^{2}$ in a Stratalinker 2400. After cell lysis ( $\mathrm{50\;mM}$ Tris–HCl, $\mathrm{pH}\nobreakspace7.4\$ ;$100\,\mathrm{\mM}$ NaCl, $1\%$ Igepal (Sigma), $0.1\%$ sodium dodecyl sulphate (SDS), $0.5\%$ Na deoxycholate, ethylenediaminetetra acetic acid (EDTA)-free protease inhibitors), RNA was partially digested with $0.2\mathrm{~U~}$ RNase I (Thermo Fisher). DENR–RNA complexes were pulled down with anti-DENR antibody (Abcam 108221), pre-immobilized on protein G-coated magnetic beads (Dynal). An IgG immunoprecipitation was used as control. Immuno-complexes were washed twice with High Salt Buffer $50\,\mathrm{mM}$ Tris–HCl pH 7.4, 1 M NaCl, $1\;\mathrm{mM}$ EDTA, $1\%$ Igepal, $0.1\%$ SDS, $0.5\%$ sodium deoxycholate) and twice with PNK Wash Buffer ( $20\;\mathrm{mM}$ Tris–HCl pH 7.4, $10\;\mathrm{mM}$ $\mathrm{MgCl}_{2}$ ,$0.2\%$ Tween-20). An infrared adaptor was ligated to RNA $3^{\prime}$ ends, and protein–RNA complexes were visualized in a LiCor CLx Imager (Odyssey), after denaturing PAGE and nitrocellulose membrane transfer. About $60{-}\bar{1}30\ \mathrm{kDa}$ DENR–RNA complexes were excised from the membrane and liberated by proteinase K (Roche) digestion. This size range likely reflects DENR–MCTS1 complexes (26 and $22\;\mathrm{kDa}$ , respectively) bound to RNA, rather than DENR– RNA alone, indicating that the DENR–MCTS1 interaction is strong. During library preparation, cDNAs were ligated on their $5^{\prime}$ end to a 5-nt sample barcode to allow demultiplexing. The barcode is preceded by a 4-nt and followed by a 3-nt random sequence that serves as unique molecular identifier (UMI) to mark individual cDNA molecules. cDNA libraries were then PCR amplified (24 cycles) and subjected to high-throughput sequencing (Illumina HiSeq2500 platform). Experiments were done in triplicate for DENRoverexpression, and in duplicate for non-transfected and IgG IP conditions.  

iCLIP data were processed and analysed using the iCount platform (https://github.com/tomazc/iCount), as described (15). Briefly, the sample barcodes were removed, and the UMI was excised and added to the read ID prior to mapping with Bowtie 0.12.7 to the mouse genome (mm10), allowing two mismatches. Sequences with the same UMI and starting at the same genomic position––likely representing PCR duplicates––were excluded from subsequent analyses. High-confidence crosslink sites were calculated with the iCount function peaks, with options RegionsAsOne and 15-nt flank.  

# Ribosome profiling  

One $15{\mathrm{-cm}}$ dish of confluent NIH3T3 cells was used per replicate. Ribosome profiling and parallel RNA-seq were performed in triplicate for Denr shRNA1, Denr shRNA2 and scr shRNA, and in duplicate for $G\/p$ shRNA, following the ART-Seq protocol by Epicentre-Illumina (Cat.no. RPHMR12126) with minor modifications as described below. Cells were treated with cycloheximide $\left(100\ \upmu\mathrm{g/ml}\right)$ ) for $2\ \mathrm{min}$ at $37^{\circ}\mathbf{C}$ to arrest translation elongation, and immediately harvested, followed by lysate preparation. For the generation of ribosome-protected fragments, cell lysates were treated with 5 units RNase I (ART-Seq) per OD260, and monosomes were purified in MicroSpin S-400 columns (GE Healthcare). For both RPF- and RNA-seq, $5~\upmu\mathrm{g}$ of RNA was used for rRNA depletion following the RiboZero Magnetic Kit instructions from Epicentre-Illumina (Cat.no. MRZH11124). Finally, libraries were amplified using 12 PCR cycles, and sequenced (HiSeq2500 platform).  

# Sequencing pre-processing, alignment and quantification  

Initial processing, mapping and quantification of mRNA and footprint abundance were performed as previously (16). Briefly, adaptor sequence was trimmed with Cutadapt (17) and trimmed sequences were filtered using a custom python script to keep 26–35 nt and 21–60 nt reads for RPF- and RNA-seq, respectively. Reads were then sequentially aligned to mouse rRNA, tRNA and cDNA (Ensembl mouse release 84) using Bowtie2 (18) and mouse genome using Tophat v2.0.11 (19). Trimmed and filtered sequences from RNA-seq were also directly mapped against the mouse genome with Tophat to estimate expressed transcript isoforms (using Cufflinks v2.2.1 (20)). Transcriptome-mapping reads from the sequential alignment were counted towards their location on $5^{\prime}$ UTR, CDS or $3^{\prime}$ UTR per gene, using only the set of expressed isoforms as estimated above. The location of the putative A-site of the footprint was used for RPF-seq counting (position $+15$ from the $5^{\prime}$ for reads $\leq\!30$ nt and position $+16$ for reads ${>}30\ \mathrm{nt})$ ), and the $5^{\prime}$ end was used for RNA-seq reads. For genes with multiple expressed isoforms, reads that did not map unambiguously to a single feature were assigned to one in preference order: CDS, $5^{\prime}$ UTR, $3^{\prime}$ UTR.  

RPF- and RNA-seq read counts were normalized separately with the upper quartile normalization from edgeR (21) and then transformed to RPKM values as read count per gene kilobase per normalized library size. Genes with $<\!10$ reads in 2/3 of samples and mitochondrially encoded genes were removed before normalization.  

Translation efficiencies were then calculated per sample as the ratio of RPF RPKM to RNA RPKM of the region of interest ( $5^{\prime}$ UTR, CDS or uORF), log2 transformed and averaged over all replicates.  

# Differential translation efficiency analysis  

Significant changes in TE between knockdown and control cells were assessed with several available algorithms: Xtail (22) with options $b i n s{=}I O O O O$ , RiboDiff (23) with options -d $I\cdot p\;I$ , Riborex (24) with options engine $\mathbf{\Sigma}^{=}\mathbf{\Sigma}^{\!\scriptscriptstyle D E S e q2}\mathbf{\Sigma}^{,*}$ and Babel (25) with options nrep $v a c{=}I O O O O O$ .  

Briefly, all four methods, with the options implemented in our study for maximal consistency among them, model raw read counts by following a negative binomial distribution and use global normalization methods for sequencing depth that would not account for global changes in TE if existing. RiboDiff and Riborex use DESeq2 to estimate a single mean and dispersion for both RPF- and RNA-seq, and detect differentially translated genes by fitting RPFand RNA-seq read counts to a single GLM, whereas Xtail uses DESeq2 for mean and dispersion estimation separately for RPF- and RNA-seq and fits two independent GLMs. Xtail then derives probability distributions for fold changes of RNA or RPF, or for RPF-to-RNA ratios, and evaluates statistical significance between conditions from one of the two (the most conservative) pipelines. Babel relies on edgeR for parameter estimation and uses an errors-in-variables regression to test for significant changes in translation within and between conditions.  

Differential TE was set at FDR $<0.1$ for all four methods and performance for false positive discovery rate was further assessed by randomly label-shuffling knockdown and control datasets, which should not yield differential TE genes.  

# uORF annotation and translation analyses  

For uORF detection, a set of transcripts was used for which reads can be unambiguously assigned to the $5^{\prime}$ UTR, i.e. transcripts that are the only protein-coding isoform expressed ( $N=4637\$ ). In order to include as many transcripts as possible for the annotation, genes with multiple protein-coding isoforms were also considered if (i) one isoform represented ${>}65\%$ of all expressed (as derived from our transcript expression estimation, see ‘Sequencing preprocessing, alignment and quantification’, $N=3211$ ) or (ii) all expressed isoforms had the same CDS start, in which case the transcript with the highest expression was selected $N=1826)$ ). For the selected transcripts, uORFs were annotated and considered as translated with the following criteria: (i) started with AUG, CUG, GUG or UUG, (ii) had an in-frame stop codon within the $5^{\prime}$ UTR or overlapping/within the CDS, (iii) were at least 9 nt long (including stop codon), (iv) had a coverage ${>}25\%$ , (v) showed a frame preference and (vi) the preferred frame was the one corresponding to the uORF $5^{\prime}$ end/start codon. Importantly, we considered only uORFs coding for at least two amino acids (i.e. 9 nt, including the stop codon), as for shorter uORFs frame bias assessment would be highly prone to false positives, and Kozak context scoring would be compromised (for 6 nt uORFs, the position $+4$ would always be the $\mathrm{T}$ of the stop codon, and thus cannot be a $_\mathrm{G}$ of the strongest Kozak context).  

# Multiple regression analysis  

For the modelling, all transcripts containing translated uORFs in Denr knockdown or control cells were included $N=4755$ ) and the following $5^{\prime}$ UTR features were used as predictors of the fold change in CDS TE across conditions: (i) the number of uORFs, (ii) the uORFs length, using the median length in transcripts with several uORFs, (iii) the proportion of AUG uORFs, (iv) the uORF Kozak context, scored relative to the consensus GccA/Gcc###G, where upper-case letters denote highly conserved bases (scored $+3$ ), lower-case letters indicate the most common nucleotides (scored $+1$ ) and ### represents the start codon (not scored). The median Kozak score was used when several uORFs were present on a transcript. (v) The CDS Kozak context, scored in the same manner, (vi) the distance of the first uORF start codon from the $5^{\prime}$ cap, (vii) the distance of the last uORF stop codon to the CDS start, (viii) the uORF GC content, using the median for transcripts with several uORFs, (ix) the presence of overlapping uORFs in the transcript (binary predictor). Predictors with skewed distributions (number of uORFs, uORF length, distance from the cap and distance to the CDS start) were log2 transformed. Partial regression plots were generated with the R package ‘visreg’. Absence of multicollinearity between predictors was assessed with the variance inflation factor using R function vif (car package).  

# Functional enrichment analysis  

Functional enrichment analysis was performed using g:Profiler (26), including Gene Ontology Biological Process and Molecular Function terms and biological pathway information from Reactome. Functional categories of more than 2 and less than 1500 members were used and $P\cdot$ -values were corrected by Benjamini–Hochberg FDR. Categories with FDR $_{<0.05}$ were considered as significant and were used for visualization with Cytoscape v.3.5.1 EnrichmentMap tool (27).  

# Dual luciferase assay  

NIH3T3 cells were first lentivirally transduced with the dual luciferase reporters containing the $5^{\prime}$ UTR construct of interest and then transduced with Denr-shRNA/scrambleshRNA. After puromycin selection, cells were lysed in passive lysis buffer and luciferase activities were quantified using DualGlo luciferase assay system and a GloMax 96 Microplate luminometer (all from Promega). Firefly/Renilla luciferase (FL/RL) activities were first normalized to that of the empty vector in scramble shRNAtransduced cells. For DENR target experiments, FL/RL signals were also normalized to the scramble shRNAtransduced, non-mutated construct. For Clock mutational experiments, FL/RL signals were secondly normalized to the scramble shRNA-transduced Clock-WT construct.  

# In vivo bioluminescence imaging  

mRNAs were produced using in vitro transcription at the ‘ivt mRNA production and formulation platform’ in Zurich (http://www.cancer.uzh.ch/en/Research/mRNAPlatform.html). The $5^{\prime}$ end consisted of a CleanCapTM (Trilink) followed by the WT or mutated Klhdc8a $5^{\prime}$ UTR and by a codon optimized firefly luciferase open reading frame. The $3^{\prime}$ end consisted of a tandem repeat of the mouse $\upbeta$ -globin $3^{\prime}$ UTR and a poly-A tail (28). Uridine residues were fully substituted by 1-methyl pseudouridine residues (this modification can prevent immunostimulation, avoiding production of type I interferon and consecutive inhibition of translation in vivo (28)).  

Animal procedures were approved by the Veterinary Office and its research ethics review committee of the University of Zurich (Kanton Zurich, Health Direction, Veterinary Office, Zollstrasse 20; 8090 Zurich; licence number ZH215/17). Animals were purchased from Envigo (Netherlands). Four- to eight-week-old mice were injected intravenously with $1~\upmu\mathrm{g}$ of mRNA formulated with TransIT transfection reagent following the manufacturer’s suggestions (Mirus). The procedure for one mouse was as follows: the nucleic acid was diluted in $38~\upmu\mathrm{l}$ of Opti-MEM (Gibco) before the addition of $0.72\,\upmu\mathrm{l}$ of the ‘mRNA Boost Reagent’. After mixing, $1.12\,\upmu\mathrm{l}$ of the TransIT reagent was added. The solution was thoroughly homogenated and injected immediately: $40~\upmu\mathrm{l}$ per mouse. At different time points after mRNA injection, in vivo bioluminescence imaging was performed on an IVIS Lumina instrument (PerkinElmer). Immediately before each measurement, Dluciferin (Synchem) dissolved in phosphate-buffered saline $(15\,\mathrm{mg/ml}$ stock) and sterile filtered was injected ( $150\,\upmu\mathrm{g/g}$ intraperitoneally). Emitted photons from live animals were quantified $10{-}20\,\mathrm{min}$ post luciferin injections, with an exposure time of 3 min. Regions of interest (ROI) were quantified for average radiance (photons s-1 cm-2 sr-1) (IVIS Living Image 3.2).  

# Western blotting  

Total protein extracts from NIH3T3 cells were prepared according to the NUN procedure (29). Antibodies used were rabbit anti-CLOCK 1:3000 (ab3517, abcam), mouse anti-U2AF65 1:5000 (Sigma), rabbit anti-DENR 1:4000 (ab108221, abcam) and anti-rabbit-HRP or anti-mouseHRP 1:10000 (Promega). Blots were visualized with the Fusion Fx7 system and quantified with ImageJ $1.48\mathrm{k}$ .  

# RESULTS  

# Redistribution of ribosomes from CDS to ${\sf5^{\prime}}$ UTRs upon Denr knockdown  

We hypothesized that if DENR is required for efficient protein production from the main CDS of mRNAs that undergo reinitiation after uORF translation, we would be able to identify DENR-regulated transcripts by comparing ribosome occupancies transcriptome-wide in DENR loss-offunction versus control cells by ribosome profiling. Conceivably, DENR target mRNAs should exhibit decreased translation efficiencies (TEs) on their CDS when DENR is absent (Figure 1A). We therefore performed shRNAmediated knockdown of Denr in mouse NIH3T3 fibroblasts, a cell line in which we had previously observed that Denr depletion led to a shortening of the period of circadian oscillations (9). Cells were lentivirally transduced with two different Denr-targeting shRNAs (shRNA1, shRNA2; in triplicate) or control RNAs (scramble shRNA, in triplicate; $G\/p$ shRNA in duplicate), grown to confluency and harvested in the presence of cycloheximide (CHX) to stabilize elongating ribosomes. After preparation and sequencing of ribosome-protected mRNA fragment (RPF-seq) and total RNA libraries (RNA-seq) as previously described (9), we obtained ribosome footprints that were predominantly $29{-}30\,\mathrm{nt}$ in length (Supplementary Figure S1A), which preferentially mapped to protein coding sequences (Supplementary Figure S1B) and that showed the characteristic triplet periodicity expected from high-quality footprint data (Supplementary Figure S1C). Across replicates, CDS-mapping normalised reads were highly correlated ( $R^{2}\,>\,0.97$ , Supplementary Figure S1D), indicating good reproducibility. The RNA-seq and RPF-seq data confirmed the efficiency of Denr knockdown $77\!-\!90\%$ reduction in Denr transcript levels), whereas the abundance of Mcts1 mRNA, encoding the DENR heterodimerization partner, or of $E i f2d$ , encoding a structurally and functionally related homologue of the DENR-MCTS1 dimer (4), remained unchanged (Figure 1B,C and Supplementary Figure S1E).  

A reduction in Denr levels that alters reinitiation efficiencies should engender differences in ribosome occupancies on the CDS and, possibly, on uORFs. First, we compared CDS footprint levels for all expressed genes ( $\bar{N}=10\ \bar{5}16$ ,Figure 1D, grey), and then selected the subset of transcripts that also displayed footprint coverage within the $5^{\prime}$ UTR $N=5148$ , Figure 1D, blue). Of note, the quantification of footprints from the full $5^{\prime}$ UTR can serve as a useful proxy for uORF translation (30,31), given that uORFs are considered the major source of $5^{\prime}$ UTR-mapping reads. This group of transcripts, thus presumably enriched for uORFcontaining genes, showed an overall similar distribution as all genes (Figure 1D). We concluded that the effect of DENR deficiency on this gene set did not markedly differ from the effect seen genome-wide, consistent with previous reports indicating that acute Denr loss did not lead to major global effects on translational output (6). Next, we calculated the ratio of $5^{\prime}$ UTR-mapping to CDS-mapping footprints per gene, and determined how this measure of ‘relative $5^{\prime}$ UTR translation’ was affected in Denr-depleted versus control cells. Our analysis revealed a striking increase in relative translation on $5^{\prime}$ UTRs in Denr-deficient cells, indicative of a redistribution of translating ribosomes within transcripts (Figure 1E and Supplementary Figure S1F). This effect was absent from the RNA-seq data (Supplementary Figure S1G). Conceivably, these changes on the translational landscape may reflect complex alterations in initiation efficiency and uORF usage, and are in line with a role of DENR in translation reinitiation (6).  

Translational remodelling that affects a sizeable proportion of the transcriptome is most consistent with a scenario in which DENR interacts with ribosomes at mRNAs with little, or no, sequence specificity. To address this question, we examined the direct interactions of DENR with RNAs in vivo by performing individual nucleotide resolution crosslinking and immunoprecipitation (iCLIP) (15) of DENR (Supplementary Figure S2A). Over $85\%$ of iCLIP reads mapped to ribosomal RNA, in particular to 18S rRNA (Supplementary Figure S2B and C), with the five most prominent binding sites on helices 24-26a (Supplementary Figure S2D and E). This outcome is in line with structural data that places the DENR–MCTS1 complex in proximity to the mRNA channel within the small ribosomal subunit and that identifies helix 24 as a potential interface of MCTS1 with 18S rRNA (32,33). Moreover, helix 26 has previously been reported to make contacts with sequences upstream of restarting AUG codons on viral RNAs (34). In contrast, only $10.5\%$ of iCLIP reads mapped to mRNAs, which was not sufficient to identify specific mRNA-binding sites. In summary, while we cannot exclude transient direct interactions of DENR–MCTS1 with mRNAs, our iCLIP data are consistent with structural studies and support a model in which the complex binds close to the mRNA channel of initiating ribosomes at a location that could facilitate reinitiation when uORFs are present, but travels indiscriminately along mRNAs.  

# Decreased CDS translation efficiency, uORF enrichment and pathway analysis of DENR-regulated transcripts  

From the shift in the distribution of $5^{\prime}$ UTR versus CDS footprints that were affecting a significant proportion of the transcriptome in Denr-deficient cells (Figure 1E), we wished to narrow down to the most-DENR-sensitive transcripts that were potentially direct and functionally relevant targets. We reasoned that changes in CDS translation efficiency (TE; defined as the ratio of CDS-mapping RPF-seq to RNA-seq reads) would represent an appropriate measure to identify such targets, as TE differences would alter the amount of protein produced from the transcript. We therefore calculated transcriptome-wide TEs and, given that there is currently no established consensus on the most suitable analytical tool for differential translation analysis, we then applied several available algorithms to detect significant TE changes. Briefly, the four methods we employed, Xtail (22), RiboDiff (23), Riborex (24) and Babel (25), are all conceived for the quantification of translational changes that deviate from possible variations already occurring at the level of mRNA abundances. They differ in parameter estimation and modelling approaches, as well as in sensitivity, false discovery rates and computational speed. When applied using settings that maximize comparability of results (see ‘Materials and Methods’ section), we identified 240, 78, 41 and 30 genes that had a significant change in TE between Denr-deficient and control cells with Xtail, RiboDiff, Riborex and Babel, respectively ( $\mathrm{FDR}<0.1$ for all four algorithms). Importantly, differential TE genes detected by Riborex, RiboDiff and Xtail completely overlapped, while only about half of the genes detected by Babel were also found by any of the other methods (Supplementary Figure S3A). Moreover, most genes detected by Xtail, RiboDiff and Riborex showed decreased TE in Denr-depleted cells, as expected for the knockdown of a reinitiation factor, and they spanned the full range of expression levels (Figure 2A and Supplementary Figure S3B,C). By contrast, Babel was biased towards low abundance transcripts and identified similar numbers of up- and down-regulated TE genes (Supplementary Figure S3B and C). We concluded that Xtail likely represented the most sensitive method to identify DENR targets, and further estimated the false discovery rate of the algorithm by randomly shuffling the dataset labels, which eliminated nearly all differential TE genes (Supplementary Figure S3D). Please note that, for simplicity, we shall further use the term ‘DENR targets’ for the 240 differential TE genes identified by Xtail, although in the absence of information on direct DENR–mRNA interactions, the term ‘most DENR-sensitive transcripts’ would be more precise.  

![](images/ca8fc6c2f13bf1be62ef8e85420b3d078362e91be8752758d6e00337038d399a.jpg)  
Figure 1. Denr knockdown engenders ribosome redistribution from CDS to $5^{\prime}$ UTR. (A) Experimental approach of the study. Ribosome profiling (RPFseq) and parallel RNA-seq were performed on NIH3T3 mouse fibroblasts depleted of Denr via lentivirally expressed shRNAs, with scramble (Scr) and $G\/\/p$ shRNAs as control conditions. Profiling was carried out in biological triplicates for the two Denr shRNAs (shRNA1, shRNA2) and for Scr shRNA, and in duplicate for $G\/\/p$ shRNA. We hypothesized that if DENR is required for efficient reinitiation after uORF translation, knocking down Denr would lead to reduced translation efficiency on the main coding sequence of DENR targets. (B) Distribution of normalized RNA-seq read counts along the Denr transcript shows high Denr knockdown efficiency. Different orange color shadings correspond to the different replicates. (C) RPKM values for CDSmapping RNA-seq read counts for Denr (left), Mcts1 (middle) and $E i f2d$ (right) indicate specific and high knockdown efficiency. Each dot corresponds to a sample and replicate and error bars indicate mean±sd. (D) Distribution of CDS translation (from RPF-seq) differences between Denr knockdown $(K\mathrm{d})$ and control cells. In grey, all detected genes $N=10516)$ ; in blue, genes with sufficient coverage in $5^{\prime}$ UTRs (i.e. a minimum of 20 RPF-seq reads in the $5^{\prime}$ UTR in 3/4 of samples; $N=5148)$ . Note that both distributions are centred around zero, indicating that Denr knockdown does not strongly affect global protein synthesis per se. (E) Distribution of $5^{\prime}$ UTR-to-CDS translation differences between Denr Kd and control cells, for genes with sufficient $5^{\prime}$ UTR coverage $N=5148.$ ). Relative $5^{\prime}$ UTR translation was significantly higher in Denr Kd cells (median $=0.32$ ,$P<2.2\mathrm{e}{-16}$ , Wilcoxon signed rank test), suggesting a ribosome footprint redistribution to $5^{\prime}$ UTRs upon Denr knockdown.  

Among the 240 differential TE genes identified by Xtail (see Supplementary File S1) were several for which the presence of regulatory uORFs has been described previously, such as Atf5 (35), Rnaseh1 (36,37), Jak2 and Map2k6 (38). In order to investigate whether the TE decrease among DENR-sensitive transcripts was associated with uORF translation, we annotated uORFs transcriptomewide. Briefly, we included AUG and non-canonical, nearAUG (i.e., CUG, GUG, UUG) initiation codons located in $5^{\prime}$ UTRs, and assessed the active translation of the corresponding ORFs by their footprint coverage, frame preference and triplet codon periodicity (Supplementary Figure S4A). Using this approach, we detected translated uORFs in nearly half of all expressed genes ( $N\!=\!4755$ genes, Figure 2B), consistent with previous predictions (8). For the majority of genes, the uORFs were fully contained in the $5^{\prime}$ UTRs $(51.8\%$ ,$N=2464$ genes), but genes with uORFs overlapping the CDS ( $19.5\%$ ,$N=926$ genes), or that had both overlapping and non-overlapping uORFs $(28.7\%$ ,$N\!=1365$ genes) were frequent, too (Supplementary Figure S4B). Notably, DENR targets were highly enriched for genes with translated uORFs $(72.7\%$ ,$P=5.73\mathrm{e}{-4}$ , Fisher test, Figure 2B) that were non-overlapping uORFs (Supplementary Figure S4B). The depletion of overlapping uORFs (Supplementary Figure S4B)––indicating that they are insensitive to DENR regulation––is consistent with the idea that after terminating on an overlapping uORF, a ribosome would presumably not be able to backtrack to reinitiate on an upstream CDS start codon. Moreover, relative $5^{\prime}$ UTR translation in Denr-depleted versus. control cells calculated over the full $5^{\prime}$ UTR––as a proxy of uORF translation––was significantly higher for DENR targets than for the global set of uORF-containing genes (Supplementary Figure S4C and D). In summary, these observations were fully consistent with an involvement of DENR in reinitiation and in efficient CDS translation of a subset of uORF-containing mRNAs.  

We next analysed whether specific functional pathways were over-represented among the 240 differential TE genes. GO term analysis revealed enrichment of categories associated with translation and mRNA stability, as well as cellular functions related to protein phosphorylation and catabolism, kinase binding and cell cycle (Figure 2C and Supplementary Figure S4E), in line with previous observations in Drosophila S2 cells (6) and indicating evolutionary conservation of DENR functions.  

Among the DENR-regulated transcripts were several relevant to cancer, such as Klhdc8a, Etaa1, Map2k5, Slc20a1 or Vegfd (39–43). Individual inspection confirmed their reduced CDS translation efficiency (Figure 3A), as well as a shift of ribosome occupancy from the CDS to the $5^{\prime}$ UTR in Denr-depleted cells (Figure 3B). This signature was characteristic of many differential TE genes (see Supplementary Figure S5A and B), and correlated with the presence of translated uORFs (Supplementary Figure S5C). Among the strongly DENR-regulated cases, we also noted $A t\phi$ (Supplementary File S1), whose translation (similar to that of $A t f4)$ is known to respond to cellular stress signals via two uORFs, the first of which is reinitiation-permissive whereas the second overlaps the main CDS (35,44). In Denr-depleted cells, translation of the first uORF (on Atf5 and, to lesser extent, on $A t f\!\!\!/\;\!\!/\;\!\!\!/\;\!\!\!\!/\;\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\;\!\!\!\!\!/\!\!\!\!\!/\!\!\!\!\!/\!\!\!\!\!\!/\!\!\!\!\!\!/\!\!\!\!\!\!\!/\!\!\!\!\!\!\!\!/\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!$ ) remained unchanged compared to control cells, while translation of the second uORF and the CDS were decreased (Supplementary Figure S5D and E). These findings validate that our approach indeed delivered reinitiation-prone transcripts.  

In order to confirm the DENR- and uORF-dependence of CDS translation for selected presumed targets, we cloned the $5^{\prime}$ UTR and immediate start codon context of several transcripts of interest in frame and upstream of Firefly luciferase (FL) in a lentiviral vector that also expresses Renilla luciferase (RL, for internal normalization) from the same bidirectional promoter (Figure 3C, upper panel). Denr depletion led to a $18–44\%$ reduction in FL/RL signals for five different control reporters that contained either no cloned $5^{\prime}$ UTR or $5^{\prime}$ UTRs of genes that were not among the differential TE genes (of these, Cry1 and Fth1 harbour translated uORFs as well, whereas Deaf1 and Rpl30 do not) (Figure 3C, lower panel). By contrast, we observed a stronger reduction in FL/RL signals––of 49– $90\%$ ––for four $5^{\prime}$ UTR reporters of presumed DENR target genes (Etaa1, Klhdc8a, Map2k5 and Lrrc28), indicating that DENR was indeed involved in their regulation. Mutating uORF start codons increased reporter signal and abolished most DENR-dependence (Figure 3C, bottom), indicating that uORF translation was necessary for DENR dependance. Furthermore, in line with the view of DENR as a reinitiation factor, rather than a leaky scanning factor, the presence of a functional stop codon was required for DENR regulation (Supplementary Figure S5F). uORF-dependent regulation of Klhdc8a translation was particularly striking, with a ${>}4$ -fold translation upregulation in WT cells with a uORF-less construct (Figure 3C, bottom). We next wished to investigate whether uORF-mediated translational regulation could also be recapitulated in vivo and selected $K l.$ -hdc8a for this experiment, as this mRNA contained the strongest, reinitiation-dependent uORF in the cellular assay. We injected mice intravenously with in vitro-synthesized mRNAs comprising the Klhdc8a $5^{\prime}$ UTR (WT or uORF start codon-mutated) fused to Firefly luciferase CDS and carrying a synthetic polyA tail. We then monitored expression by bioluminescence live imaging in mice. Bioluminescence signal was primarily detected in the livers of the animals, and was 2–4.3 times higher in the uORF-less compared to the WT constructs (Supplementary Figure S6A and B), demonstrating the strong inhibitory activity of the Klhdc8a uORF also in vivo.  

![](images/4a8ab5e1443666ab2d2060f309030a13d515cddd1ee3a2bbaa8dbbd3362dda19.jpg)  
Figure 2. Characteristics of DENR targets: decreased CDS TE; enrichment for uORFs; association with functional categories. (A) Scatter plot of Denr $K\mathrm{d}$ -to-control ratio of mRNA abundance versus CDS translation efficiency (TE) for all expressed genes ( $N\!=\!10$ 516). Highlighted in orange are genes with differential TE (detected by Xtail (22), FDR $<0.1$ ,$N\!=\!240$ ), the majority of which showed decreased TE in Denr Kd cells, in line with a role of DENR in regulating translation reinitiation. (B) uORF enrichment in DENR targets. Proportion of genes with translated uORFs transcriptome-wide compared to that among the differential TE genes detected in panel (A). DENR targets showed enrichment for translated uORF-containing genes ( $P=5.73\mathrm{e}{-4}$ , Fisher test). (C) Enrichment map for GO-Biological process, GO-Metabolic function and Reactome functional categories of genes with differential TE. Only categories with an FDR $<0.05$ in the GO analysis were considered. Node size is proportional to the number of genes associated with the GO category, shading corresponds to the statistical significance, and edge width is proportional to the number of genes shared between functional categories.  

Taken together, we concluded that DENR was involved in the regulation of protein biosynthetic output, and that a set of $>\!200$ transcripts containing uORFs was particularly sensitive to loss of this reinitiation factor.  

![](images/095b7a6f6a005c80018798f277919ca396f422cee94f2956fd9be34f84255e62.jpg)  
Figure 3. Mutation of uORF start codons relieves DENR dependence. (A) Representative examples of genes with differential TE. Panels show average CDS translation efficiency in control (grey) and Denr Kd (red) cells. (B) Denr Kd-to-control translation on the $5^{\prime}$ UTR (brown) and on the CDS (grey) for the same examples as in (A). DENR targets showed decreased CDS TE (A) and increased $5^{\prime}$ UTR translation concomitant with reduced CDS translation in Denr-deficient cells. (C) (Upper) Schematic representation of the dual luciferase reporter construct, where the $5^{\prime}$ UTR of the gene of interest––with the WT or mutated uORF start––and first few codons of the CDS were cloned upstream of firefly luciferase (FL). (Lower) Results of dual luciferase assays, where the influence of the $5^{\prime}$ UTR (WT and uORF mutant) on the expression of FL was measured in control and Denr knockdown cells, and internally normalized to RL (expressed from the same bidirectional promoter). Note that for all targets tested, depletion of Denr reduced the FL/RL luciferase signal, indicating DENR-dependence of reporter protein expression. Mutation of uORF start codons relieved the inhibition and partially abolished DENR-dependence. For each $5^{\bar{\prime}}$ UTR assayed, FL/RL signals were normalized to the Scr-transduced non-mutated construct. The area of grey shading marks the non-specific effect of Denr loss-of-function in this assay, as judged from the reduction in signal seen for negative control reporters that contained no cloned $5^{\prime}$ UTR (-) or irrelevant $5^{\prime}$ UTRs selected from non-targets.  

# Linear regression analysis correlates DENR-dependence with uORF strength and with intercistronic distance  

uORFs can operate through a bewildering diversity of mechanisms that act on CDS translation (by affecting start site selection/leaky scanning/re-initiation/ribosome shunting), transcript abundance (by eliciting nonsense-mediated mRNA decay) or in some cases through uORF-encoded bioactive peptides (reviewed in (3)). However, rules to predict specific mechanisms from specific uORF features have not yet been established. Our datasets could potentially enable the identification of signatures that render certain uORF-containing transcripts DENR-dependent. Previous studies have suggested that, at least in the context of synthetic reporters in Drosophila S2 (6) and human HeLa (7) cells, very short uORFs (coding for 1–2 amino acids) with favourable Kozak contexts were most strongly dependent on DENR for efficient reinitiation at a downstream reporter CDS.  

We used multiple regression analysis to determine whether DENR-dependent translation events were associated with specific transcript features. We modelled the CDS TE change observed upon Denr knockdown using nine relevant transcript features, namely the number of uORFs, uORF start codon identity (AUG versus near-cognate) and context, the length, GC content and location of the uORF within the $5^{\prime}$ UTR, the presence of overlapping uORFs and CDS start codon context (Figure 4A). The model was applied to all translated uORF-containing genes $N\!=$ 4741) and yielded several features with significant predictive value for DENR-sensitivity of TE (Figure 4B–J; adjusted $R^{2}\,=\,0.057$ ,$F$ -statistic $=33.21\$ on 4731 DF, $P<$ 2.2e-16). Three of the identified features reflected the expected positive relationship between efficient uORF translation and a greater need for reinitiation to subsequently translate the CDS as well. These features were: the number of uORFs present on a transcript (Figure 4B, $b\,=$ $-0.019$ ,$P<2\mathrm{e}{-16}$ ), the presence of AUG, rather than noncanonical uORF start codons (Figure 4C, $b=-0.05$ ,$P<$ 1.15e-9), and the strength of Kozak contexts around uORF start codons (Figure 4D, $b\,=\,-0.0026$ ,$P\,=\,0.013$ ). Relatedly, DENR targets were enriched for AUG uORFs as compared to the ensemble of translated uORF-containing transcripts $\textstyle P=0.0044$ , Fisher test, Supplementary Figure S7A), and genes harboring exclusively AUG uORFs showed the strongest effect on CDS TE in Denr-deficient cells (Supplementary Figure S7B). By contrast, the Kozak context around the CDS start codon bore no predictive value (Figure 4E, $b=-0.0006$ ,$P=0.38)$ ), suggesting no DENR involvement in its recognition. Moreover, the presence of CDS-overlapping uORFs was not associated with DENR-dependent TE change (Figure 4F, $b=0.005$ ,$P=$ 0.44), as expected for a reinitiation factor, given that overlapping uORFs would be reinitiation-incompetent unless the transcripts contained a downstream alternative CDS start codon.  

Two noteworthy outcomes from the model were that uORF length lacked a relationship (Figure 4G, $b=0.0006$ ,$P=0.81$ ), whereas the uORF–CDS intercistronic distance had predictive value for DENR-sensitivity of CDS TE (Figure $4\mathrm{H}$ ,$b=-0.0101$ ,$P=7.89\mathrm{e}{-7}$ ). Briefly, already early studies on a model transcript had suggested that the location of a uORF with respect to the main ORF could dictate the efficiency of reinitiation, as longer spacing would generally allow the reacquisition of initiation factors to the scanning ribosome (45). Our model provides an unbiased and transcriptome-wide validation of this notion for DENRregulated transcripts, where larger distances from the stop codon of the $3^{\prime}$ -most uORF to the start codon of the main ORF were associated with downregulation of TE (Figure 4H); uORFs located further upstream from the CDS were thus more likely to reinitiate in a DENR-dependent fashion. Importantly, this effect of uORF–CDS distance was not caused by longer $5^{\prime}$ UTRs per se, given that the distance from the $5^{\prime}$ cap to the first uORF was not associated with DENR regulation (Figure 4I, $b=-0.0018$ ,$P=0.18)$ ). Of note, the uORF–CDS distance was only weakly correlated with $5^{\prime}$ UTR length (Spearman rho $=0.16$ ), whereas the correlation of the $5^{\prime}$ cap-first uORF distance with $5^{\prime}$ UTR length was more prominent (Spearman rho $=0.27$ , Supplementary Figure S7C and D).  

No linear relationship was found between uORF length and CDS TE (Figure 4G, $b=0.0006$ ,$P=0.81$ ), in agreement with previous results in human cells using synthetic uORFs encoding ${>}2$ amino acids $(9\,\mathrm{nt})$ (7). Of note, a length dependence for extremely short uORFs of 1–2 amino acids, as reported by Schleich et al. (7), would go unnoticed in our analyses, as we excluded them due to the uncertainties associated with their annotation (see ‘Materials and Methods’ section).  

Finally, the uORF GC content showed a significant relationship with TE fold change (Figure 4J, $b=0.058$ ,$P=$ 0.005), suggesting that potential weaker secondary structures (low GC) favoured reinitiation involving DENR.  

For the above model, we used the CDS TE fold change as response, which has the benefit of accounting for DENR regulation in a continuous manner. Of note, however, also a logistic regression model based on a binary response (i.e. DENR target as defined in Figure 2 versus non-target) recapitulated the main findings (Supplementary Figure S7E), further supporting our predictions for the strongest cases of DENR translational control. In summary, although the model explained only a relatively modest proportion of the total variance (see ‘Discussion’ section), our results indicated that DENR was important to enable efficient protein synthesis when the transcripts contained uORFs initiating with AUG in strong Kozak contexts (i.e. uORFs expected to be highly translated), and that the distance between the most downstream uORF and the CDS start was an important parameter for efficient reinitiation.  

# DENR-mediated reinitiation regulates CLOCK protein biosynthesis  

We recently found that uORF translation was pervasive within circadian core clock transcripts, and that the depletion of Denr in NIH3T3 cells engendered robust circadian period shortening by $1{-}1.5\,\mathrm{h}$ (9). We therefore sought to uncover which core clock transcripts were DENR-regulated, and to map the relevant uORFs. Conceivably, these would represent likely contributors to the observed circadian phenotype.  

The expression of several core clock genes was altered in Denr-depleted versus control cells. In most cases $(P e r I/2/3$ ,Cry1/2, Nr1d1, Rorα), the observed changes occurred already at the level of mRNA abundance and subsequently led to comparable changes at the footprint level as well (Figure 5A and Supplementary Figure S8A). None of these transcripts (nor Bmal1, whose lower RNA abundance was partially compensated at the footprint level) was associated with marked TE reduction (Figure 5B and Supplementary Figure S8B), and we thus considered them unlikely candidates for DENR targets. For Clock, however, the reduction in expression was mostly translational, resulting in a ${\sim}20\%$ decrease in protein biosynthesis, as quantified from Clock RPFs (Figure 5A,B and Supplementary Figure S8A,B). In our initial analysis of differential TE (Figure 2A), Clock had, however, not passed the significance threshold (Supplementary File S1). The reason was that the applied algorithm, Xtail, evaluates statistical significance of TE differences through two separate tests––one regarding differences between the RNA and the RPF fold-change distributions, and a second one evaluating differences in TE distributions directly––and then derives the final $P$ -value from the most conservative of the two tests (22). While the additional test might prevent false positives, it can discard differential TE genes close to the statistical threshold. Importantly, for one of the tests, the TE fold change for Clock was significant ( $P$ $=0.025$ in test 1; $P=0.19$ for test 2; Supplementary File S1); Clock thus may represent a DENR target whose detection is at the borderline of statistical significance.  

![](images/8a25a6aa220830967c7bf1919e65f87a176e16c0e9c322b32cd7e0ef67d3acbb.jpg)  
Figure 4. $5^{\prime}$ UTR features predictive of translation efficiency regulation by DENR. (A) Schematic of the $5^{\prime}$ UTR features used as predictors for the modeling of CDS TE change. The number of $\mathrm{{uORFs}}$ , the start codon identity and context, the presence of overlapping uORFs, the uORF length and GC content and the relative location within the $5^{\prime}$ UTR were considered. Distance to the $5^{\prime}$ cap was computed for the first uORF, and distance to the CDS start was quantified for the last uORF. (B–J) Partial regression plots for the modelling of translation efficiency fold change (Denr Kd versus Ctrl) on each uORF feature. Indicated in blue are the slope coefficients for the partial regression line and their significance levels ( $\scriptstyle{t}$ -test). The number of uORFs (B), the proportion of AUG uORFs (C) and the distance of the last uORF to the CDS start (H) were the strongest predictors.  

We annotated two translated AUG-initiated uORFs in Clock (Figure 5C and Supplementary Figure S8C–E). The second uORF overlapped with the main CDS in a different frame––a configuration that would render the two translation events (uORF-2 and annotated CDS) mutually exclusive. However, downstream of the uORF-2 stop codon and 10 amino acids into the annotated Clock CDS, we noted an additional AUG in the Clock coding frame. We thus hypothesized that, when uORF-2 was translated, initiation at this alternative AUG could still allow for the biosynthesis of a CLOCK protein variant that was N-terminally truncated by nine amino acids (Figure 5C). In Denr-deficient cells, translation was increased on both uORFs (Figure 5C– D) and, in particular, uORF-2 showed ${\sim}23\%$ higher translation rate upon Denr depletion (Figure 5E), suggesting that DENR-mediated reinitiation after uORF-2 was crucial for the translation of the Clock CDS.  

![](images/46c7fb9774748176fc4c80414809a7929873728c8f639b9de050b53ac0d47700.jpg)  
Figure 5. DENR-mediated reinitiation regulates CLOCK protein abundance and N-terminus. (A) Bar graph of the average RPKM ratio (Denr Kd-tocontrol) for the main circadian core clock genes, at the level of mRNA abundance (dark shadings) and footprint abundance (light shadings). (B) CDS translation efficiency ratio between Denr Kd and control cells showed that Clock had a reduced TE upon Denr depletion. (C) Distribution of normalized RPF-seq (blue) and RNA-seq (orange) reads along the $5^{\prime}$ UTR and first $100\;\mathrm{nt}$ of Clock CDS, in control (top) and Denr Kd (bottom) cells. Green shaded areas indicate the two AUG-initiated uORFs; grey area corresponds to the annotated CDS; and brown area to the alternative CDS, in frame with the annotated one. (D) Quantification of relative translation ratio (Denr Kd-to-control) on both uORFs, and on the CDS, showing increased uORF translation and reduced CDS translation in Denr-deficient cells. (E) Quantification of relative translation efficiency (Denr Kd-to-control) on both uORFs, and on the CDS showed higher TE on the overlapping uORF (uORF2). (F) (Top) Schematic representation of the dual luciferase reporter construct, where Clock $5^{\prime}$ UTR and first 14 codons of the CDS (containing both the annotated and alternative start codons) were cloned upstream of FL. (Bottom) Dual luciferase assay results for Clock WT and mutants. Empty vector contained only the vector-encoded $5^{\prime}$ UTR. FL signal was internally normalized to RL signal; all constructs and conditions were first normalized to the value of condition ‘empty vector, scr shRNA’; all Clock mutant constructs were then additionally normalized to the value from condition ‘Clock WT, scr shRNA’. Signal from all Denr shRNA-transduced Clock constructs were also normalized to the signal obtained for ‘empty vector, Denr-shRNA’, to remove DENR effects that were not specific to the $5^{\prime}$ UTR. (G) Schematic of how uORFs and reinitiation regulate the translation of the transcript encoding the circadian core clock component Clock. Clock transcript contains two translated AUG uORFs, the second of which shows high ribosome occupancy and overlaps with the annotated CDS. An additional in-frame AUG is located nine amino acids downstream of the annotated AUG; usage of the downstream AUG would give rise to a so far unannotated CLOCK protein variant. Our data suggest that this second AUG is the true start codon for CLOCK biosynthesis in NIH3T3 cells, and that DENR is necessary for translation reinitiation on the alternative CDS after translation of the second uORF.  

Using GWIPS-viz, an online genome browser for the visualization of ribosome profiling data from published studies (46), we looked for further evidence of uORF-2 and alternative CDS translation across mammalian datasets (of note, both annotated and alternative methionine residues are conserved across mammals, see Supplementary Figure S9). Interestingly, initiating ribosomes can be detected on both uORF-2 and alternative CDS start codons (but not the annotated Clock start codon) across human and mouse datasets, and elongating ribosome profiling studies showed good footprint coverage on uORF-2 and the characteristic accumulation of ribosomes at the start and stop codons (Supplementary Figure S9). Together, the published and our new data thus suggested abundant translation of uORF-2 and of Clock CDS from the alternative AUG, indicating that reinitiation has an important role in ensuring CLOCK protein biosynthesis.  

In order to investigate uORF usage, effect, and DENRdependence for CLOCK biosynthesis from the annotated and alternative CDS, we cloned the Clock $5^{\prime}$ UTR and first 14 codons (containing both the annotated and alternative AUGs) in frame and upstream of Firefly luciferase in our dual luciferase vector (Figure 5F, upper panel). We then systematically mutated the uORF-1, uORF-2 and CDS annotated/alternative start codons, or combinations thereof (Supplementary Figure S8F), and compared the effects on luciferase output in Denr-depleted versus control cells. Of note––and in contrast to Figure 3C––we also adjusted for the effect of Denr depletion on the control reporter (whose levels were set to 1) to display the isolated effects of the Clock variants.  

Denr depletion resulted in an almost $30\%$ reduction in expression for the Clock WT construct (Figure 5F, lower panel), in line with the decrease in relative translation quantified for the endogenous transcript (Figure 5D) and with reduced levels of endogenous CLOCK protein (Supplementary Figure S10). Surprisingly, mutation of the annotated CDS initiation codon did not substantially affect reporter levels and DENR dependence when compared to Clock WT. However, mutation of the alternative AUG decreased luciferase signal by ${\sim}95\%$ and abolished DENR dependence (Figure 5F, lower panel), suggesting that most CLOCK biosynthesis in NIH3T3 cells involves the alternative AUG. Mutation of uORF-1 led to a $19\%$ increase in luciferase signal and fully retained DENR dependence. By contrast, uORF-2 mutation led to a ${\sim}50\%$ increase in protein synthesis and eliminated DENR regulation. These findings indicated that uORF-2 translation acts to significantly reduce CLOCK biosynthesis and to render it DENRdependent. This model was further supported by the combinatorial mutations. Mutation of both uORF- $1\!+\!2$ start codons thus resulted in an additive, ${\tt>}90\%$ increase in protein synthesis, and in a loss of DENR-dependence. In line with minor levels of synthesis of the long CLOCK isoform, mutation of the annotated CDS start codon in combination with uORF mutations led to similar results as mutation of the uORFs alone. Furthermore, mutation of the alternative AUG in combination with uORF mutations led to strongly decreased luciferase signals, consistent with the predominant synthesis of the short CLOCK isoform. In particular, the case where both the alternative CDS and uORF-2 (or uORF- $1+2$ ) start codons were mutated––a scenario that would allow unhindered CLOCK synthesis from the annotated AUG––resulted in a ${>}80\%$ reduction in CLOCK protein production (Figure 5F, lower panel). Taken together, we concluded that Clock required DENR for efficient protein synthesis from a CDS that is 9 codons shorter than the annotated one, and that uORF-2 plays a key role in coordinating the complex interactions that occur at the translational level on the Clock transcript (Figure 5G). Moreover, we propose that the CLOCK N-terminus is likely wrongly annotated not only in NIH3T3 cells, but based on our analysis of published ribosome profiling data, also in other cell types and in tissues (Supplementary Figure S9).  

# DISCUSSION  

The annotation of translated uORFs has remained a particular challenge, because they are short, frequently initiate from non-AUG codons, and do not obey the same rules of evolutionary conservation as most protein coding sequences (3). This has changed with the availability of ribosome profiling data, which has made it possible to systematically identify translated uORFs from footprints. The resulting burst in interest in uORFs and the rising appreciation of how prevalent their usage is in mammals, is, however, not yet matched by an understanding at the mechanistic level. In this study, we have investigated one particularly intriguing mechanism that is implicated in the regulation through uORFs, namely translation reinitiation. As an entry point, we have used the loss-of-function of DENR, a non-canonical initiation factor recently associated with reinitiation both in Drosophila (6) and in HeLa cells (7), with the aim of identifying rules that explain why certain uORFs trigger DENR-dependent reinitiation while others do not. The conclusions we were able to draw from experiments in mouse NIH3T3 fibroblasts showed similarities and striking differences to the previous studies in fly and in human cells (6,7). Differences in experimental and computational approaches are likely to have contributed to the observed discrepancies. Moreover, species-specificity in uORF biology may underlie some of the disparities as well. In this context it is, for example, noteworthy that recent data from yeast suggest that reinitiation occurs as a consequence of a lack in DENR/MCTS-1 and EIF2D orthologues (the study uses the double knockouts) (5) , whereas studies in metazoans (6,7,47,48) suggest that DENR:MCTS1 promotes reinitiation––a conclusion we would draw from our experiments as well. By contrast, the ribosome accumulation that we observe on $5^{\prime}$ UTRs in Denr-deficient cells is fully consistent with the yeast study that highlights the importance of EIF2D/DENR in ribosome recycling (5).  

We view our quantitative modelling approach of DENRdependent changes in CDS TE as particularly unbiased as it relies on translational changes detected on endogenous mRNAs transcriptome-wide. Several features carry predictive value for the magnitude of DENR-mediated CDS TE regulation. Of these, three (number of uORFs, start codon identity, start codon sequence context) likely reflect a straightforward logic between a stronger requirement for reinitiation when the usage of uORFs is efficient. This outcome is expected and fully consistent with human (7) and fly data (6) based on reporter assays. Two other outcomes from the model are noteworthy. First, uORFs located further away from the main ORF are more prone to trigger DENRdependent reinitiation. Such intercistronic distance effects were reported early on in the uORF field, typically using distinct uORF-containing model transcripts (45,49,50). They had generally been assumed to reflect the kinetics of association and dissociation of initiation and termination factors. Our data validate this idea in a transcriptome-wide fashion and are in line with the hypothesis that longer distances allow for extended re-scanning times, which would be favourable for the loss of termination factors and/or the re-acquisition of the initiation machinery, on time for CDS translation. Second, it is noteworthy that uORF length lacks predictive value in our model. Although it is tempting to speculate that increased uORF length is therefore not linked to impaired reinitiation efficiency (at least for DENR-regulated reinitiation events), it should be kept in mind that the nature of our approach (transcriptome-wide from endogenous transcripts) did not allow to confidently annotate uORFs down to only 1–2 amino acids in length. We may thus be missing DENR effects that have recently been observed in reporters carrying such ultra-short synthetic uORFs (7). We can evidently also not exclude that the time required for uORF translation in determining scanning resumption is relevant nevertheless, as linear length per se may simply not be a good proxy for translation time (2,8,51).  

Overall, our model explains a relatively modest proportion of the total variance (adjusted $R^{2}$ of 0.057). This is reminiscent of analogous models that have been developed to predict, for example, miRNA target sites, whose adjusted $\mathbf{\dot{\}}R^{2}\mathbf{s}$ range from 5 to $13\%$ , but that have nevertheless proven biologically useful (52). In our case, the presence of multiple uORFs on a transcript, often overlapping each other, and each with its own, distinct features (start codon, length, distance to the CDS, etc), and the resulting combinatorial effects, clearly complicated their scoring and likely masked some of the origins of translational regulation. We deliberately wished to keep our model simple, aiming at identifying the most significant predictors, but at the expense of overall explanatory power. In the future, the use of other regression types and models that include interaction terms could potentially explain higher biological variability, although at the cost of becoming considerably more difficult to interpret. Overall, our study constitutes a first attempt to analyse signatures associated with reinitiation through DENR. It was already able to recapitulate previous observations (6,7), and as we continue to identify more molecular features that can regulate non-canonical translation initiation and that can be integrated into the model in the future, a larger proportion of biological variance will be explained.  

We have identified 240 transcripts as particularly DENRsensitive. Although we lack formal proof that they are direct DENR targets, the observations that (i) most of them show TE downregulation and few upregulation in the absence of DENR, that (ii) they are depleted for CDS-overlapping and enriched for non-overlapping uORFs, and that (iii) a DENR-dependent inhibitory effect of their $5^{\prime}$ UTRs can be rescued by uORF start codon mutations in reporter assays, all make a direct activity of DENR on these transcripts very likely. Moreover, these observations are all compatible with a role of DENR in reinitiation, but not in leaky scanning (which could have been an alternative interpretation of the redistribution of ribosomes from CDS to $5^{\prime}$ UTRs seen in Figure 1E). In particular, we would not expect CDSoverlapping uORFs to be depleted (Supplementary Figure S4B), if DENR acted to regulate targets via leaky scanning. Moreover, mutating the uORF stop codon should not have had an influence on the DENR-effect in our reporter assays, if the activity of DENR consisted in regulating the efficiency of uORF start codon selection (Supplementary Figure S5F).  

Many identified DENR targets are involved in proliferation and associated with cancer. Of note, DENR-MCTS1 have themselves been identified as oncogenes (53) that are upregulated, for example, in prostate cancer (54). Moreover, translational reprogramming, especially with regard to upstream initiation sites, appears to represent an important cancer driver (30), and understanding how the translation of cancer-relevant DENR targets is regulated may thus lead to important insights of diagnostic or therapeutic value. We found, for example, very strong DENR requirement for the translation of Kelch Domain Containing 8A (KLHDC8A). It has been reported that KLHDC8A overexpression in human gliomas that become resistant to epidermal growth factor receptor (EGFR) silencing (a commonly used anticancer therapeutic approach), allows tumours to maintain aggressiveness through an unknown EGFR-independent pathway (39). It is conceivable that upon loss of EGFR sensitivity, KLHDC8A levels increase through DENR-mediated translational control. Similarly, other candidates from our DENR target list (Supplementary File S1) are upregulated in (and possibly causally contributing to) different cancers, such as Vascular endothelial growth factor D (VEGF-D) or Mitogen-Activated Protein Kinase 5 (MAP2K5) (41,43). By understanding how DENR regulates its targets, it may thus be possible to reduce reinitiation activity and thus the oncogenic levels of the modulated proteins in proliferating, but not in quiescent cells.  

A comparison of DENR targets found in our and in previous studies returned little overlap. Of the eight targets validated in HeLa cells (7), only one (Tmem60) was expressed in NIH3T3 cells. Tmem60 was predicted to undergo a $25\%$ translational downregulation upon Denr loss-of-function and indeed showed a decrease in that range in reporter assays in HeLa cells (7), whereas in our system a ${\sim}8\%$ decrease in CDS TE of endogenous Tmem60 was quantified. Moreover, for the ensemble of 104 predicted targets in human cells (7), ${\tt>}90\%$ were not expressed in NIH3T3 cells or lacked a mouse orthologue; others (e.g. Lca5, Mllt11 or Dsel) were indeed downregulated upon Denr knockdown, though to a lesser extent. Conceivably, DENR targets are species- and cell type-specific; moreover, it is likely that the differences in the magnitude of translational regulation observed across studies are the result of different experimental and analytical approaches.  

How do DENR and circadian rhythms mechanistically connect? The DENR target list did not reveal core clock genes that would explain the circadian clock phenotype, i.e. the previously reported $\mathord{\sim}1.5\mathrm{-h}$ period shortening of free-running oscillations (9). Individual inspection of core clock gene transcripts, however, suggested that Clock was a DENR target just below the significance threshold. Followup experiments confirmed Clock regulation through DENR and allowed mapping the responsible uORF. These findings establish uORF translation and re-initiation as a novel, potentially regulated mechanism acting on core clock gene expression, which opens several intriguing questions. First, is it possible that reduced biosynthesis of Clock would be responsible for the observed phenotype? Loss-of-function of a general gene expression machinery component typically goes along with sizeable distortions in global RNA and protein profiles, and Denr knockdown is no exception in this respect. Any phenotype thus likely represents a composite effect, rather than the result of a single misregulated gene expression event. We therefore do not view reduced CLOCK biosynthesis as a satisfactory hypothesis that alone would be able to explain the substantial period shortening. Nevertheless, it is reassuring that $C l o c\bar{k}^{-/-}$ fibroblasts have recently been shown to have shorter periods as well (55), which suggests that downregulation of Clock translation upon Denr depletion may be a factor contributing to the phenotype.  

A second intriguing question relates to the alternative initiation site that would generate an N-terminally truncated variant of CLOCK. We propose that the 9 amino acid shorter CLOCK is the main variant produced in NIH3T3 cells, and likely in other cell types and species as well. What are the functional implications? The bHLH domain responsible for DNA binding of CLOCK starts at amino acid 26, and the very N-terminus has not yet been associated with any function of the protein and is also not included in the crystal structure of the CLOCK-BMAL1 complex (56). However, it is noteworthy that for several myogenic bHLH transcription factors, N-terminal motifs are critical for cooperative DNA binding with cofactors (57). Much of our current knowledge on CLOCK stems from in vivo and in vitro experiments that have used Clock cDNA constructs devoid of the endogenous uORF regulation, and thus defining CLOCK’s N-terminus to the annotated one. It will be exciting to re-evaluate selected DNA-binding and activity assays using both CLOCK isoforms side by side.  

Finally, a speculative question to ask is whether DENRmediated control is a regulated, rather than constitutive step in CLOCK biosynthesis, serving for example to uncouple CLOCK protein abundance from mRNA levels in specific cell types or under certain physiological conditions. Of note, there is precedence from Neurospora crassa for uORF translation that mediates temperature-dependent regulation of biosynthesis of the core clock component FRQ (58). In the case of DENR, it is interesting that this protein was originally identified because its expression increased in cultured cells at high density, but not during growth arrest per se, which led to its name Density Regulated Protein (59). First links between cell density and circadian rhythmicity have indeed been described in recent years (60,61), and it would be interesting to assess whether CLOCK uORFs and DENR play roles in determining density-dependent differences of clock parameters.  

# DATA AVAILABILITY  

High-throughput sequencing data was deposited to GEO, under accession number: GSE124793 (SuperSeries composed of ribosome profiling data: GSE116221 and iCLIP data: GSE124790). Code for ribosome profiling preprocesssing, aligning and quantification, and for uORF annotation, is available online at http://doi.org/10.5281/ zenodo.521199.  

# SUPPLEMENTARY DATA  

Supplementary Data are available at NAR Online.  

# ACKNOWLEDGEMENTS  

We thank the Lausanne Genomics Technologies Facility for high-throughput sequencing infrastructure. We thank Angelica Liechti and Georgia Katsioudi who were involved in initial steps of reporter plasmid clonings. We are grateful to Igor Ruiz de los Mozos for assistance with the iClip analyses through iCount.  

# FUNDING  

Swiss National Science Foundation [157528, 179190 to D.G.]; National Centre of Competence in Research (NCCR) RNA & Disease [D.G., S.P.]; Olga Mayenfisch Stiftung [D.G.]; Fondation Herbette [D.G.]; University of Lausanne [D.G.]; University of Zurich (URPP ‘Translational Cancer Research’) (to S.P.). Funding for open access charge: Swiss National Science Foundation. Conflict of interest statement. None declared.  

# REFERENCES  

1. Jackson,R.J., Hellen,C.U.T. and Pestova,T.V. (2010) The mechanism of eukaryotic translation initiation and principles of its regulation. Nat. Rev. Mol. Cell Biol., 11, 113–127.   
2. Barbosa,C., Peixeiro,I. and Romao,L. (2013) Gene expression regulation by upstream open reading frames and human disease. PLOS Genet., 9, e1003529.   
3. Wethmar,K. (2014) The regulatory potential of upstream open reading frames in eukaryotic gene expression. Wiley Interdiscip. Rev. RNA, 5, 765–778.   
4. Skabkin,M.A., Skabkina,O.V., Dhote,V., Komar,A.A., Hellen,C.U.T. and Pestova,T.V. (2010) Activities of Ligatin and MCT-1/DENR in eukaryotic translation initiation and ribosomal recycling. Genes Dev., 24, 1787–1801.   
5. Young,D.J., Makeeva,D.S., Zhang,F., Anisimova,A.S., Stolboushkina,E.A., Ghobakhlou,F., Shatsky,I.N., Dmitriev,S.E., Hinnebusch,A.G. and Guydosh,N.R. (2018) Tma64/eIF2d, Tma20/MCT-1, and Tma22/DENR recycle Post-termination 40s subunits InVivo. Mol. Cell, 71, 761–774.   
6. Schleich,S., Strassburger,K., Janiesch,P.C., Koledachkina,T., Miller,K.K., Haneke,K., Cheng,Y.-S., Kuchler,K., Stoecklin,G., Duncan,K.E. et al. (2014) DENR-MCT-1 promotes translation re-initiation downstream of uORFs to control tissue growth. Nature, 512, 208–212.   
7. Schleich,S., Acevedo,J.M., Hohenberg,K.C.V. and Teleman,A.A. (2017) Identification of transcripts with short stuORFs as targets for DENR-MCTS1-dependent translation in human cells. Sci. Rep., 7, 3722.   
8. Calvo,S.E., Pagliarini,D.J. and Mootha,V.K. (2009) Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. U.S.A., 106, 7507–7512.   
9. Janich,P., Arpat,A.B., Castelo-Szekely,V., Lopes,M. and Gatfield,D. (2015) Ribosome profiling reveals the rhythmic liver translatome and circadian clock regulation by upstream open reading frames. Genome Res., 25, 1848–1859.   
10. Salmon,P. and Trono,D. (2007) Production and titration of lentiviral vectors. Curr. Protoc. Hum. Genet., 12.10.1–12.10.24.   
11. Moffat,J., Grueneberg,D.A., Yang,X., Kim,S.Y., Kloepfer,A.M., Hinkle,G., Piqani,B., Eisenhaure,T.M., Luo,B., Grenier,J.K. et al. (2006) A lentiviral RNAi library for human and mouse genes applied to an arrayed viral high-content screen. Cell, 124, 1283–1298.   
12. Du,N.-H., Arpat,A.B., De Matos,M. and Gatfield,D. (2014) MicroRNAs shape circadian hepatic gene expression on a transcriptome-wide scale. Elife, 3, e02510.   
13. Huppertz,I., Attig,J., D’Ambrogio,A., Easton,L.E., Sibley,C.R., Sugimoto,Y., Tajnik,M., Konig,J and Ule,J. (2014) iCLIP: Protein-RNA interactions at nucleotide resolution. Methods, 65, 274–287.   
14. Zarnegar,B.J., Flynn,R.A., Shen,Y, Do,B.T., Chang,H.Y. and Khavari,P.A. (2016) iCLIP platform for efficient characterization of protein-RNA interactions. Nat. Methods, 13, 489–492.   
15. Konig,J., Zarnack,K., Rot,G., Curk,T., Kayikci,M., Zupan,B., Turner,D.J., Luscombe,N.M. and Ule,J. (2010) iCLIP reveals the function of hnRNP particles in splicing at individual nucleotide resolution. Nat. Struct. Mol. Biol., 17, 909–915.   
16. Castelo-Szekely,V., Arpat,A.B., Janich,P. and Gatfield,D. (2017) Translational contributions to tissue specificity in rhythmic and constitutive gene expression. Genome Biol., 18, 116.   
17. Martin,M. (2011) Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet. journal, 17, 10–12.   
18. Langmead,B. and Salzberg,S.L. (2012) Fast gapped-read alignment with Bowtie 2. Nat. Methods, 9, 357–359.   
19. Trapnell,C., Pachter,L. and Salzberg,S.L. (2009) TopHat: discovering splice junctions with RNA-Seq. Bioinformatics, 25, 1105–1111.   
20. Trapnell,C., Williams,B.A., Pertea,G., Mortazavi,A., Kwan,G., Baren,M.J. van, Salzberg,S.L., Wold,B.J. and Pachter,L. (2010) Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat. Biotechnol., 28, 511–515.   
21. Robinson,M.D. and Oshlack,A. (2010) A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biol., 11, R25.   
22. Xiao,Z., Zou,Q., Liu,Y. and Yang,X. (2016) Genome-wide assessment of differential translations with ribosome profiling data. Nat. Commun., 7, 11194.   
23. Zhong,Y., Karaletsos,T., Drewe,P., Sreedharan,V.T., Kuo,D., Singh,K., Wendel,H.-G. and Ratsch,G. (2017) RiboDiff: detecting changes of mRNA translation efficiency from ribosome footprints. Bioinformatics, 33, 139–141.   
24. Li,W., Wang,W., Uren,P.J., Penalva,L.O.F. and Smith,A.D. (2017) Riborex: fast and flexible identification of differential translation from Ribo-seq data. Bioinformatics, 33, 1735–1737.   
25. Olshen,A.B., Hsieh,A.C., Stumpf,C.R., Olshen,R.A., Ruggero,D. and Taylor,B.S. (2013) Assessing gene-level translational control from ribosome profiling. Bioinformatics, 29, 2995–3002.   
26. Reimand,J., Arak,T., Adler,P., Kolberg,L., Reisberg,S., Peterson,H. and Vilo,J. (2016) g:Profiler-a web server for functional interpretation of gene lists (2016 update). Nucleic Acids Res., 44, W83–W89.   
27. Merico,D., Isserlin,R., Stueker,O., Emili,A. and Bader,G.D. (2010) Enrichment map: a network-based method for gene-set enrichment visualization and interpretation. PLoS ONE, 5, e13984.   
28. Tusup,M., French,L.E., Guenova,E., Kundig,T. and Pascolo,S. (2018) Optimizing the functionality of in vitro-Transcribed mRNA. BJSTR, 7, 001–006.   
29. Lavery,D.J. and Schibler,U. (1993) Circadian transcription of the cholesterol 7 alpha hydroxylase gene may involve the liver-enriched bZIP protein DBP. Genes Dev., 7, 1871–1884.   
30. Sendoel,A., Dunn,J.G., Rodriguez,E.H., Naik,S., Gomez,N.C., Hurwitz,B., Levorse,J., Dill,B.D., Schramek,D., Molina,H. et al. (2017) Translation from unconventional $5^{\prime}$ start sites drives tumour initiation. Nature, 541, 494–499.   
31. Blair,J.D., Hockemeyer,D., Doudna,J.A., Bateup,H.S. and Floor,S.N. (2017) Widespread translational remodeling during human neuronal differentiation. Cell Rep., 21, 2005–2016.   
32. Lomakin,I.B., Stolboushkina,E.A., Vaidya,A.T., Zhao,C., Garber,M.B., Dmitriev,S.E. and Steitz,T.A. (2017) Crystal structure of the human ribosome in complex with DENR-MCT-1. Cell Rep., 20, 521–528.   
33. Weisser,M., Schafer,T., Leibundgut,M., Bohringer,D., Aylett,C.H.S. and Ban,N. (2017) Structural and functional insights into human Re-initiation complexes. Mol. Cell, 67, 447–456.   
34. Zinoviev,A., Hellen,C.U.T. and Pestova,T.V. (2015) Multiple mechanisms of reinitiation on bicistronic calicivirus mRNAs. Mol. Cell, 57, 1059–1073.   
35. Watatani,Y., Ichikawa,K., Nakanishi,N., Fujimoto,M., Takeda,H., Kimura,N., Hirose,H., Takahashi,S. and Takahashi,Y. (2008) Stress-induced translation of ATF5 mRNA is regulated by the $5^{\prime}$ -Untranslated region. J. Biol. Chem., 283, 2543–2553.   
36. Suzuki,Y., Holmes,J.B., Cerritelli,S.M., Sakhuja,K., Minczuk,M., Holt,I.J. and Crouch,R.J. (2010) An upstream open reading frame and the context of the two AUG codons affect the abundance of mitochondrial and nuclear RNase H1. Mol. Cell. Biol., 30, 5123–5134.   
37. Liang,X.-h., Shen,W., Sun,H., Migawa,M.T., Vickers,T.A. and Crooke,S.T. (2016) Translation efficiency of mRNAs is increased by antisense oligonucleotides targeting upstream open reading frames. Nat. Biotechnol., 34, 875–880.   
38. Schulz,J., Mah,N., Neuenschwander,M., Kischka,T., Ratei,R., Schlag,P.M., Castanos-Velez,E., Fichtner,I., Tunn,P.-U. and Denkert,C. (2018) Loss-of-function uORF mutations in human malignancies. Sci. Rep., 8, 2395.   
39. Mukasa,A., Wykosky,J., Ligon,K.L., Chin,L., Cavenee,W.K. and Furnari,F. (2010) Mutant EGFR is required for maintenance of glioma growth in vivo, and its ablation leads to escape from receptor dependence. Proc. Natl. Acad. Sci. U.S.A., 107, 2616–2621.   
40. Childs,E.J., Mocci,E., Campa,D., Bracci,P.M., Gallinger,S., Goggins,M., Li,D., Neale,R.E., Olson,S.H., Scelo,G. et al. (2016) Common variation at 2p13.3, 3q29, 7p13 and 17q25.1 associated with susceptibility to pancreatic cancer. Nat. Genet., 47, 911–916.   
41. Diao,D., Wang,L., Wan,J., Chen,Z., Peng,J., Liu,H., Chen,X., Wang,W. and Zou,L. (2016) MEK5 overexpression is associated with the occurrence and development of colorectal cancer. BMC Cancer, 16, 302.   
42. Sato,K. and Akimoto,K. (2017) Expression levels of KMT2c and SLC20a1 identified by Information-theoretical analysis are powerful prognostic biomarkers in estrogen Receptor-positive breast cancer. Clin. Breast Cancer, 17, e135–e142.   
43. Thelen,A., Scholz,A., Benckert,C., von Marschall,Z., Schroder,M., Wiedenmann,B., Neuhaus,P., Rosewicz,S. and Jonas,S. (2008) VEGF-D promotes tumor growth and lymphatic spread in a mouse model of hepatocellular carcinoma. Int. J. Cancer, 122, 2471–2481.   
44. Vattem,K.M. and Wek,R.C. (2004) Reinitiation involving upstream ORFs regulates ATF4 mRNA translation in mammalian cells. Proc. Natl. Acad. Sci. U.S.A., 101, 11269–11274.   
45. Kozak,M. (1987) Effects of intercistronic length on the efficiency of reinitiation by eucaryotic ribosomes. Mol. Cell. Biol., 7, 3438–3445.   
46. Michel,A.M., Fox,G., A.,M.Kiran, De Bo,C., O’Connor,P.B.F., Heaphy,S.M., Mullan,J.P.A., Donohue,C.A., Higgins,D.G. and Baranov,P.V. (2014) GWIPS-viz: development of a ribo-seq genome browser. Nucleic Acids Res., 42, D859–D864.   
47. Haas,M., Ngo,L., Li,S., Schleich,S., Qu,Z., Vanyai,H., Cullen,H., Cardona-Alberich,A., Gladwyn-Ng,I., Pagnamenta,A. et al. (2016) De novo mutations in DENR disrupt neuronal development and link congenital neurological disorders to faulty mRNA translation Re-initiation. Cell Rep., 15, 2251–2265.   
48. Ahmed,Y.L., Schleich,S., Bohlen,J., Mandel,N., Simon,B., Sinning,I. and Teleman,A.A. (2018) DENR-MCTS1 heterodimerization and tRNA recruitment are required for translation reinitiation. PLOS Biol., 16, e2005160.   
49. Kozak,M. (2001) Constraints on reinitiation of translation in mammals. Nucleic Acids Res., 29, 5226–5232.   
50. Luukkonen,B.G., Tan,W. and Schwartz,S. (1995) Efficiency of reinitiation of translation on human immunodeficiency virus type 1 mRNAs is determined by the length of the upstream open reading frame and by intercistronic distance. J. Virol., 69, 4086–4094.   
51. Poyry,T.A.A., Kaminski,A. and Jackson,R.J. (2004) What determines whether mammalian ribosomes resume scanning after translation of a short upstream open reading frame? Genes Dev., 18, 62–75.   
52. Agarwal,V., Bell,G.W., Nam,J.-W. and Bartel,D.P. (2015) Predicting effective microRNA target sites in mammalian mRNAs. eLife Sci., 4, e05005.   
53. Reinert,L.S., Shi,B., Nandi,S., Mazan-Mamczarz,K., Vitolo,M., Bachman,K.E., He,H. and Gartenhaus,R.B. (2006) MCT-1 protein interacts with the cap complex and modulates messenger RNA translational profiles. Cancer Res., 66, 8994–9001.   
54. Cerami,E., Gao,J., Dogrusoz,U., Gross,B.E., Sumer,S.O., Aksoy,B.A., Jacobsen,A., Byrne,C.J., Heuer,M.L., Larsson,E. et al. (2012) The cBio cancer genomics Portal: An open platform for exploring multidimensional cancer genomics data. Cancer Discov., 2,   
401–404.   
55. Landgraf,D., Wang,L.L., Diemer,T. and Welsh,D.K. (2016) NPAS2 compensates for loss of CLOCK in peripheral circadian oscillators. PLOS Genet., 12, e1005882.   
56. Huang,N., Chelliah,Y., Shan,Y., Taylor,C.A., Yoo,S.-H., Partch,C., Green,C.B., Zhang,H. and Takahashi,J.S. (2012) Crystal structure of the heterodimeric CLOCK:BMAL1 transcriptional activator complex. Science, 337, 189–194.   
57. Knoepfler,P.S., Bergstrom,D.A., Uetsuki,T., Dac-Korytko,I., Sun,Y.H., Wright,W.E., Tapscott,S.J. and Kamps,M.P. (1999) A conserved motif N-terminal to the DNA-binding domains of myogenic bHLH transcription factors mediates cooperative DNA binding with pbx-Meis1/Prep1. Nucleic Acids Res., 27, 3752–3761.   
58. Diernfellner,A.C., Schafmeier,T., Merrow,M.W. and Brunner,M. (2005) Molecular mechanism of temperature sensing by the circadian clock of Neurospora crassa. Genes Dev., 19, 1968–1973.   
59. Deyo,J.E., Chiao,P.J. and Tainsky,M.A. (1998) drp, a novel protein expressed at high cell density but not during growth arrest. DNA Cell Biol., 17, 437–447.   
60. Noguchi,T., Wang,L.L. and Welsh,D.K. (2013) Fibroblast PER2 circadian rhythmicity depends on cell density. J. Biol. Rhythms, 28, 183–192.   
61. Guenthner,C.J., Luitje,M.E., Pyle,L.A., Molyneux,P.C., Yu,J.K., Li,A.S., Leise,T.L. and Harrington,M.E. (2014) Circadian rhythms of Per2::Luc in individual primary mouse hepatocytes and cultures. PLoS ONE, 9, e87573.  