# Cell Metabolism  

# An Evolutionarily Conserved uORF Regulates PGC1a and Oxidative Metabolism in Mice, Flies, and Bluefin Tuna  

# Authors  

![](images/e2a22caacc142aa2a8ef4e4d2237a5535e936c1c5919317880d2834f2edf0e16.jpg)  
Graphical Abstract  

Phillip A. Dumesic, Daniel F. Egan, Philipp Gut, ..., Shingo Kajimura, Samir M. Parikh, Bruce M. Spiegelman  

# Correspondence  

bruce_spiegelman@dfci.harvard.edu  

# In Brief  

Dumesic et al. identify a repressive upstream open reading frame (uORF) in the PPARGC1A mRNA. Loss of function of the uORF in mice raises PGC1 $\pmb{\alpha}$ function and is protective against kidney disease. Although conserved from human to fly, this uORF is absent in the highly metabolically active migratory Atlantic bluefin tuna.  

# Highlights  

An upstream open reading frame represses translation of the PPARGC1A mRNA uORF function is a conserved feature of human, fish, and fly PPARGC1A ${\mathfrak{s}}^{\prime}$ UTRs Atlantic bluefin tuna PPARGC1A ${\mathfrak{s}}^{\prime}$ UTR lacks a uORF and supports elevated translation Mouse PPARGC1A uORF ablation increases PGC1 $\pmb{\alpha}$ function and protects from kidney injury  

# An Evolutionarily Conserved uORF Regulates PGC1a and Oxidative Metabolism in Mice, Flies, and Bluefin Tuna  

Phillip A. Dumesic,1,2,11 Daniel F. Egan,1,2,11 Philipp Gut,1,2 Mei T. Tran,3,4 Alice Parisi,1,2 Nirmalya Chatterjee,5,6 Mark Jedrychowski,1,2 Margherita Paschini,7 Lawrence Kazak,8 Sarah E. Wilensky,1 Florence Dou,1 Dina Bogoslavski,1 Jeffrey A. Cartier,9 Norbert Perrimon,5,6 Shingo Kajimura,10 Samir M. Parikh,3,4 and Bruce M. Spiegelman1,2,12,\*  

1Dana-Farber Cancer Institute, Boston, MA 02115, USA   
2Department of Cell Biology, Harvard University Medical School, Boston, MA 02115, USA   
3Division of Nephrology and Department of Medicine, Beth Israel Deaconess Medical Center and Harvard Medical School, Boston,   
MA 02215, USA   
4Center for Vascular Biology Research, Beth Israel Deaconess Medical Center and Harvard Medical School, Boston, MA 02215, USA   
5Department of Genetics, Harvard University Medical School, Boston, MA 02115, USA   
6Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA   
7Boston Children’s Hospital, Boston, MA 02115, USA   
8Goodman Cancer Research Centre, Department of Biochemistry, McGill University, Montreal, Canada   
9Cartier and Company, LLC, Harwich, MA 02645, USA   
10Diabetes Center and Department of Cell and Tissue Biology, University of California, San Francisco, CA 94143, USA   
11These authors contributed equally   
12Lead Contact   
\*Correspondence: bruce_spiegelman@dfci.harvard.edu   
https://doi.org/10.1016/j.cmet.2019.04.013  

# SUMMARY  

Mitochondrial abundance and function are tightly controlled during metabolic adaptation but dysregulated in pathological states such as diabetes, neurodegeneration, cancer, and kidney disease. We show here that translation of $\mathsf{P G C1}\,\pmb{\alpha}$ , a key governor of mitochondrial biogenesis and oxidative metabolism, is negatively regulated by an upstream open reading frame (uORF) in the $\mathfrak{s}^{\prime}$ untranslated region of its gene (PPARGC1A). We find that uORF-mediated translational repression is a feature of PPARGC1A orthologs from human to fly. Strikingly, whereas multiple inhibitory uORFs are broadly present in fish PPARGC1A orthologs, they are completely absent in the Atlantic bluefin tuna, an animal with exceptionally high mitochondrial content. In mice, an engineered mutation disrupting the PPARGC1A uORF increases PGC1 $\pmb{\alpha}$ protein levels and oxidative metabolism and confers protection from acute kidney injury. These studies identify a translational regulatory element governing oxidative metabolism and highlight its potential contribution to the evolution of organismal mitochondrial function.  

# INTRODUCTION  

The transcriptional coactivator $\mathsf{P G C1}\,\mathsf{a}$ was discovered as a brown-fat enriched protein that bound $\mathsf{P P A R}_{\mathsf{Y}}$ and other transcription factors and activated mitochondrial and thermogenic gene expression (Puigserver et al., 1998). PGC1a has since been established as a master regulator of oxidative metabolism and mitochondrial biogenesis not only in brown fat but also in other tissues, such as liver, muscle, brain, and kidney (Fernandez-Marcos and Auwerx, 2011; Lin et al., 2002, 2004; Tran et al., 2011; Yoon et al., 2001). Its functions allow animals to adapt their energy metabolism to environmental and nutritional states.  

Post-transcriptional regulation of PGC1 $\pmb{\alpha}$ is likely to be crucial for adapting $\mathsf{P G C1}\,\mathsf{\alpha}$ levels and function to dynamic metabolic Mitochondria are the cell’s powerhouses. Their number and function are tightly controlled so that energy needs are carefully balanced, though this can go awry in diseases such as diabetes or kidney disease. $\mathsf{P G C1}\,\mathsf{\alpha}$ is key molecular regulator of mitochondria. Researchers at the Dana-Farber Cancer Institute identified a naturally occurring conserved genetic element that slows down production of the PGC1a protein. Genetic manipulation of this element in mice increased mitochondrial function and protected them from kidney injury. Although conserved from flies to humans, this PGC1 $\pmb{\alpha}$ element is absent in the highly migratory Atlantic bluefin tuna, which has high metabolic demands. This finding is reflective of evolutionary adaptations and opens new avenues to raise PGC1a levels to mitigate metabolic diseases.  

![](images/6326c8d0fc5360332cda6daa3b4f8b62d16f856685a76da59e9df869f04fa9c4.jpg)  
$\mathfrak{s}^{\prime}$   
(A) Ribosome profiling data indicating location of initiating (blue) or elongating (red) ribosomes on the PPARGC1A transcript in Jurkat cells. Initiating or elongating ribosomes were stalled using lactimidomycin or cycloheximide, respectively. Data were adapted from (Gawron et al., 2016). (B) Ribosome profiling data indicating location of elongating ribosomes on the PPARGC1A transcript in HeLa cells. Data were adapted from (Park et al., 2016; Wang et al., 2014; Zur et al., 2016). (C) Immunoblot (left) or RT-qPCR (right) of $\mathsf{P G C1}\,\mathsf{a}$ protein or mRNA following transfection of each indicated expression vector in 293E cells ${[\mathsf{n}=3}$ ; error bars represent the SE). The dash indicates the untransfected sample.  

demands on a rapid timescale. Consistent with this idea, $\mathsf{P G C1}\,\mathsf{a}$ protein has a short half-life $(<\!30\,\mathsf{m i n})$ and is heavily post-translationally modified. Furthermore, discrepancies have been noted between the amount of mRNA and the corresponding PGC1 $\alpha$ protein (Fisher et al., 2012; Pettersson-Klein et al., 2018; Tsukiyama-Kohara et al., 2001). Given the protective effects of elevated $\mathsf{P G C1}\,\mathsf{a}$ function in models of pathology, including neurodegenerative disease, muscular dystrophy, muscle atrophy, acute kidney ischemia, and ALS, an understanding of the post-transcriptional modes of quantitative $\mathsf{P G C1}\,\mathsf{a}$ modulation is of great interest (Da Cruz et al., 2012; Handschin et al., 2007; Sandri et al., 2006; Tran et al., 2016; Zheng et al., 2010).  

Upstream open reading frames (uORFs) reside in the $5^{\prime}$ untranslated regions (UTRs) of approximately half of human mRNAs (Calvo et al., 2009; McGillivray et al., 2018), where their function is context-dependent but is generally thought to suppress translation of downstream coding regions (Barbosa et al., 2013; Chew et al., 2016). Although fewer uORFs are present in the human genome than expected by chance, those that are present tend to be evolutionarily conserved, arguing for their adaptive function (Chew et al., 2016; Churbanov et al., 2005). Indeed, uORFs can play specialized roles, including regulation of downstream protein output during cell stress (Wethmar et al., 2010b).  

We show here that PPARGC1A is under translational control by a uORF. Inhibitory uORF action is functionally conserved in PPARGC1A orthologs from flies to humans. Interestingly, the Atlantic bluefin tuna, an animal with an exceptional oxidative metabolism capacity, completely lacks this regulatory element. A knock-in mutation in mice that prevents uORF translation elevates $\mathsf{P G C1}\,\mathsf{a}$ function and confers protection from ischemic kidney injury. These data identify a translational regulatory mechanism controlling $\mathsf{P G C1}\,\mathsf{a}$ and mitochondrial function in vivo.  

# RESULTS  

# The $\mathbf{\bar{s}}^{\prime}$ UTR of PPARGC1A Contains a uORF That Negatively Regulates PGC1a Translation  

Given the importance of post-transcriptional mechanisms in determining PGC1a1 (hereafter PGC1a) protein levels, we sought to identify regulatory elements controlling translation of the PPARGC1A mRNA. As a first approach, we analyzed ribosome profiling data to determine the distribution of ribosomes on the mRNA. Using published data from Jurkat cells treated with lactimidomycin (Gawron et al., 2016), an antibiotic that stalls initiating ribosomes, we noticed an accumulation of ribosome footprints at an ATG trinucleotide (AUG in RNA; notations hereafter based on DNA sequence) within the annotated $5^{\prime}$ UTR of PPARGC1A (Figure 1A). In the same cell type, footprints of elongating ribosomes were observed in the $5^{\prime}$ UTR downstream of this ATG. Ribosome profiling datasets collected in HeLa cells similarly demonstrated the presence of elongating ribosome footprints in the PPARGC1A $5^{\prime}$ UTR (Park et al., 2016; Wang et al., 2014; Zur et al., 2016), at levels comparable to those seen in the exon 1 coding region (Figure 1B). These findings suggest ribosomal engagement with an upstream start codon in the PPARGC1A $5^{\prime}$ UTR.  

We next sought to test whether ribosome engagement with this upstream start codon influences translation of $\mathsf{P G C1}\,\mathsf{a}$ .The upstream start codon initiates a 16-codon uORF that does not overlap with the PGC1 $\pmb{\alpha}$ coding region. We generated an expression vector in which a CMV promoter drives the $\mathsf{P G C1}\,\mathsf{a}$ coding region preceded by a truncated PPARGC1A $5^{\prime}$ UTR lacking the uORF (Short UTR, SU) (Figure 1C). We also generated a construct encoding the entire 139 nt PPARGC1A $5^{\prime}$ UTR (WT) and a construct in which the upstream ATG is mutated to TAA (TAA). Importantly, all constructs contain the same endogenous Kozak sequence surrounding the $\mathsf{P G C1}\,\mathsf{\alpha}$ start codon.  

Transfection of the SU-PGC1 $\alpha$ expression construct into HEK293E cells resulted in greater $\mathsf{P G C1}\,\mathsf{a}$ protein levels as compared to WT-PGC1a, suggesting that the PPARGC1A $5^{\prime}$ UTR negatively regulates translation (Figure 1C). The TAA$\mathsf{P G C1}\,\mathsf{a}$ construct also led to greater $\mathsf{P G C1}\,\mathsf{a}$ protein levels. To more directly test the idea that ribosome initiation at the upstream ATG is the cause of decreased translation of $\mathsf{P G C1}\,\mathsf{a}_{:}$ ,we also created an expression construct in which the endogenous weak Kozak sequence surrounding the upstream ATG is altered to an optimal Kozak environment (RCCATGG). Indeed, transfection of this construct (Strong Kozak) reduced $\mathsf{P G C1}\,\mathsf{a}$ protein levels below those observed with the WT $5^{\prime}$ UTR construct. The PPARGC1A transcript level was similar among all constructs (Figure 1C).  

To test whether similar regulation could be observed in vitro, we performed translation experiments using a rabbit reticulocyte system. As in intact cells, the full length PPARGC1A $5^{\prime}$ UTR inhibited $\mathsf{P G C1}\,\mathsf{a}$ translation as compared to the short $5^{\prime}$ UTR (Figure 2A). The $5^{\prime}$ UTR’s inhibitory effect could be enhanced by strengthening the uORF Kozak sequence and was mostly lost by mutating the uORF start codon to TAA.  

We next tested whether the PPARGC1A uORF can itself serve as a template for translation. We generated an expression construct in which both the uORF stop codon and the $\mathsf{P G C1}\,\mathsf{a}$ start codon were deleted (Figure 2B). Because the uORF is in frame with the $\mathsf{P G C1}\,\mathsf{a}$ ORF, translation initiating at the uORF start codon would in this construct be expected to generate an N-terminal extension of PGC1a. Indeed, $\mathsf{P G C1}\,\mathsf{a}$ protein produced from this construct migrates at a higher molecular weight upon resolution by SDS-PAGE.  

We then asked whether the PPARGC1A $5^{\prime}$ UTR could function outside the context of its specific mRNA. To this end, we first tested whether the inhibitory effect of the $5^{\prime}$ UTR on $\mathsf{P G C1}\,\mathsf{a}$ translation could occur in trans. The $5^{\prime}$ UTR did not affect translation of the PPARGC1A coding sequence when each element was encoded on a separate mRNA, nor did addition of a synthetic oligopeptide corresponding to the uORF sequence affect PGC1a translation in vitro (Figures S1A and S1B). Second, we asked whether the PPARGC1A $5^{\prime}$ UTR could function, in cis, on a novel downstream coding region. We fused the UTR sequences described above to a green fluorescent protein (GFP) coding sequence. Upon transfection into HEK293E cells, the TAA-GFP and SU-GFP constructs both elevated GFP production relative to the WTGFP construct, without altering levels of GFP mRNA (Figures 2C and 2D).  

Our finding that the PPARGC1A uORF can engage ribosomes led to a prediction that cellular signals acting to reduce ribosomal preinitiation complex (PIC) scanning fidelity might alter the uORF’s inhibitory potential by allowing the PIC to bypass the uORF in favor of the downstream PGC1a start codon (Barbosa et al., 2013). Consistent with this idea, arsenite treatment, which is expected to cause global suppression of translation and reduce PIC scanning fidelity (Andreev et al., 2015), dampened the PPARGC1A uORF’s quantitative effect on translation, as measured by luciferase assay (Figure S1C).  

# The PPARGC1A uORF Is Functionally Conserved from Human to Fly but Absent in Atlantic Bluefin Tuna  

To examine its evolutionary conservation, we assessed the presence of the murine PPARGC1A uORF throughout vertebrates. Since a uORF’s function does not necessarily depend on the polypeptide sequence it encodes, we identified relatives of the murine PPARGC1A uORF by asking which vertebrate genomes, when aligned with the mouse genome, encode a uORF with an ATG start codon at the precise equivalent position (Blanchette et al., 2004). We found that such PPARGC1A uORFs are broadly preserved from humans to lobe-finned fish, the clade of fish that gave rise to terrestrial tetrapods (Sarcopterygii—e.g., coelacanth) (Figure 2E). Supporting the accuracy of the uORF predictions, an alignment of polypeptides encoded by these uORFs revealed related sequences, with particular conservation of a set of serine residues (human positions 2, 5, and 7), valines (positions 3 and 6), and a cysteine (position 4) (Figure S1D). In contrast to the case of lobe-finned fish, a uORF related to the murine PPARGC1A uORF by predicted amino acid sequence or ATG position is not generally found in the more distantly related rayfinned fish (Actinopterygii), nor is it found in invertebrates.  

Importantly, a species that does not contain a direct relative of the murine PPARGC1A uORF may still encode an unrelated uORF in its PPARGC1A gene, possibly preserving uORF functionality. This possibility was assessed in Drosophila and zebrafish owing to the availability of experimental tools for these species. Spargel, the Drosophila PPARGC1A ortholog, encodes an eight-codon uORF. Its primary sequence and start codon location are distinct from those of the murine uORF. To test its function, we generated expression constructs in which the Drosophila Act promoter drives a GFP coding region fused to a $5^{\prime}$ UTR encoding the PPARGC1A or spargel uORF (Figure 3A). We also generated ‘‘TAA’’ versions of these constructs, in which their respective uORF start codons are mutated. When assessed in the Drosophila cell line ${}^{82\mathsf{R}+}$ , disruption of either the mouse or Drosophila uORF similarly elevated GFP levels. These findings suggest that the spargel uORF, though not evolutionarily related by sequence to the murine PPARGC1A uORF, performs a similar inhibitory function.  

The annotated zebrafish ppargc1a $5^{\prime}$ UTR contains two uORFs. Their primary sequences and start codon locations are distinct from the murine and fly uORFs. To test their function, we built DNA constructs in which the zebrafish $5^{\prime}$ UTR (with or without uORF start codon mutations) was fused to the reporter gene ZsGreen::NanoLuc. After in vitro transcription, capped mRNAs were injected into zebrafish zygotes, followed by in vivo fluorescence and bioluminescence measurements $48\;\mathsf{h}$ later. A separate DNA construct encoding the mCherry::Luc2 reporter was used as a control for injection efficiency. uORF mutation in this context significantly increased GFP expression $\scriptstyle\sim3$ -fold). Therefore, although zebrafish do not encode a ppargc1a uORF directly related to the murine uORF, they have preserved $5^{\prime}$ UTR repressive functionality via uORFs that evolved independently of the murine uORF (Figure 3B).  

Using the zebrafish ppargc1a $5^{\prime}$ UTR as a guide, we assessed a diverse set of ray-finned fish: medaka, turquoise killifish, and tilapia. Ppargc1a orthologs in each of these species preserved the second zebrafish uORF, all initiating from an equivalent start codon position (Figure 3C). These findings imply broad conservation of ppargc1a uORF function among ray-finned fish.  

![](images/1bbd57c08470e44e61c22589b35075f79c4ebd482e7bd5178e85e6d0745a7026.jpg)  
Figure 2. The PPARGC1A uORF Engages Ribosomes to Negatively Regulate Translation of PGC1a (A) Immunoblot of $\mathsf{P G C1}\,\alpha$ following translation of each expression construct in reticulocyte lysate. The dash indicates no vector added. (B) Top: schematic of expression constructs. The uORF stop codon and the PGC1a start codon are deleted in the DTAA DATG construct. Bottom: immunoblot of PGC1a following translation of each indicated construct in reticulocyte lysate.  

Another ray-finned fish, the Atlantic bluefin tuna (Thunnus thynnus), is known for its exceptionally high endurance, illustrated by its ability to swim thousands of miles across the Atlantic Ocean (Block et al., 2005). This can be explained, at least in part, by the high abundance of ‘‘red’’ oxidative muscle and mitochondria in this animal (Dalziel et al., 2005). The elevated red muscle content and oxidative capacity in Atlantic bluefin tuna is in stark contrast to the predominant white glycolytic muscle observed in most fish species (Katz, 2002; Korsmeyer and Dewar, 2001; Moyes, 2003). We hypothesized that the red fiber appearance in the Atlantic bluefin tuna, which is similar to the skeletal muscle observed in mice that overexpress PGC1 $\pmb{\alpha}$ (Lin et al., 2002), may be due to high activity of $\mathsf{P G C1}\,\mathsf{a}$ . This could, in principle, involve alterations in the uORF described here.  

The Thunnus thynnus genome has not been sequenced. To investigate its ppargc1a $5^{\prime}$ UTR, we caught a wild Atlantic bluefin tuna in the waters off Cape Cod, MA and performed $5^{\prime}$ RACE. Strikingly, the T. thynnus ppargc1a $5^{\prime}$ UTR is short and lacks any uORFs, including those that we observed in other ray-finned fish species (Figure 3C). To test its ability to support translation, DNA constructs were built encoding the $5^{\prime}$ UTRs of tuna or zebrafish fused to the ZsGreen::NanoLuc reporter gene. When assessed by microinjection, the T. thynnus $5^{\prime}$ UTR directed significantly greater NanoLuc translation than did the zebrafish $5^{\prime}$ UTR (Figure 3D). These findings are consistent with some of the remarkable features of this fish species.  

# Mutation of the PPARGC1A uORF in Mice Increases PGC1a Function in Multiple Tissues and Protects from Acute Kidney Injury  

To determine the functional significance of the mammalian PPARGC1A uORF in vivo, we generated uORF-deficient mice via knock-in mutation of the uORF start codon (Figures 4A and S2A and S2B). Mutant mice $({\mathsf{u O R F}}^{\mathsf{T A A}})$ ) were born at expected Mendelian ratios with no gross abnormalities, and their body weight was normal at 8 weeks of age (Figures S2C and S2D).  

Having obtained homozygous ${\mathsf{u O R F}}^{\mathsf{T A A}}$ mice, we investigated kidney function, as this tissue has among the highest levels of $\mathsf{P G C1}\,\mathsf{a}$ and mitochondria in the body in order to support to the energetic demands of solute transport (Pagliarini et al., 2008; Tran et al., 2011). We were particularly motivated to investigate the pathological setting of acute kidney injury (AKI), a common condition among hospitalized patients and a major contributor to kidney disease and kidney failure (Chawla et al., 2014). Previous work has established that PPARGC1A is downregulated in the context of chemical, genetic, obstructive, and ischemic kidney injury models (Han et al., 2017; Lynch et al., 2018). Importantly, genetic loss of PGC1a exacerbates experimental AKI, whereas increased expression of $\mathsf{P G C1}\,\mathsf{\alpha}$ in the kidney—even to a mild extent relative to endogenous PGC1a—protects from AKI (Tran et al., 2011, 2016). We therefore asked whether quantitative control of $\mathsf{P G C1}\,\mathsf{a}$ protein via the PPARGC1A uORF would be sufficient to protect the kidney from ischemia-reperfusion injury. Impairment of renal function could then be quantified by accumulation of serum creatinine, a waste product of endogenous metabolism normally cleared by healthy kidneys.  

In the baseline state prior to ischemic injury, uORFTAA mice exhibited normal gross kidney anatomy, urine albumin content, and serum creatinine levels, similar to those of wild-type mice (Figures S3A–S3F). Levels of PPARGC1A mRNA in kidney were equivalent between the two genotypes, but PGC1a protein was elevated in ${\mathsf{u O R F}}^{\mathsf{T A A}}$ mice (Figures S3G and S3H). A separate cohort of mice was subjected to 20 min of kidney ischemia and evaluated after $^{24\mathrm{~h~}}$ of reperfusion, a time point at which serum creatinine elevation is near maximal (Tran et al., 2016). At this time point, $\mathsf{P G C1}\,\mathsf{a}$ translation remained derepressed in ${\mathsf{u O R F}}^{\mathsf{T A A}}$ mice as compared to wild-type mice (Figures 4B and 4C).  

To assess the consequences of uORF ablation on AKI, we first used tandem mass spectrometry after isobaric peptide tagging to compare protein abundance in injured kidneys from ${\mathsf{u O R F}}^{\mathsf{T A A}}$ versus wild-type mice (Figure 4D; Table S1). Prior work has defined genes that are induced in the context of ischemic kidney injury (‘‘AKI-correlated’’); they are enriched for involvement in functions including leukocyte chemotaxis, cytokine signaling, and inflammatory response (Table S2) (Liu et al., 2017). Gene set enrichment analysis showed that the AKI-correlated gene set was significantly depressed in the uORFTAA kidney, implying a reduced extent of kidney injury in uORFTAA mice (Figure 4D). Conversely, proteins that were elevated in the uORFTAA kidney included components of multiple OXPHOS complexes, consistent with increased $\mathsf{P G C1}\,\mathsf{a}$ function and consequent augmentation of mitochondrial respiratory chain abundance (Figure 4E). Of note, when assessed in the absence of ischemic injury, the uORFTAA mutation’s effect on AKI-associated gene sets was more mild (Figure S3I, Table S1), and the mutation did not affect OXPHOS complex abundance (Figure S3J).  

We next assessed a more physiologic readout of kidney injury: serum creatinine levels. Mice carrying the uORF mutation exhibited lower creatinine levels $^{24\,\mathrm{~h~}}$ after injury as compared to wild-type littermates, demonstrating relative tolerance to AKI (Figure 4F). Histology confirmed that wild-type mice exhibited a larger zone of vascular congestion in the kidney medulla, a reflection of endothelial swelling in the context of ischemic injury (Figure 4G; dotted line). In the kidney cortex, the site of the most metabolically active tubular epithelium, epithelial damage and necrotic tubules were more pronounced in wild-type mice (Figure 4G, arrows). Together, these results indicate that disruption of the PPARGC1A uORF raises PGC1a levels and function to an extent that alters the course of ischemic kidney injury.  

![](images/8a3ba1ce5bad170bb2b0c4d32a23048760f2a4de17585e3e21e38818ebb136e9.jpg)  
Figure 3. Inhibitory PPARGC1A uORFs Are Functionally Conserved in Fly and Zebrafish but Not in Atlantic Bluefin Tuna  

(A) Top: schematic of GFP expression constructs that contain the mouse or fly PPARGC1A/spargel uORF. Bottom: constructs were transfected into Drosophila ${}^{82\mathsf{R}+}$ cells, and GFP was monitored by microscopy.   
(B) Expression construct mRNA was injected into zebrafish zygotes and analyzed by fluorescence (images) or bioluminescence (graph). mCherry-Luc2 served as control for injection efficiency. $\mathsf{N}=20$ ;$p<0.01$ .  
(C) Alignment of conserved uORF upstream of fish ppargc1a orthologs. The uORF start codon is highlighted in red, whereas the PGC1a start codon is highlighted in green.   
(D) Expression construct mRNA was injected into zebrafish zygotes and analyzed by fluorescence (images) or bioluminescence (graph). mCherry-Luc2 served as  

control for injection efficiency. $\mathsf{N}=20$ ;$p<0.01$ .  

![](images/11bc1cc0f03e87eef545c913ddb04fa85747717d96e87110f571d1e3a049787c.jpg)  
Figure 4. uORFTAA Mice Exhibit Elevated PGC1a Protein Levels and Are Protected from Ischemic Kidney Injury   
(A) Design of PPARGC1A uORFTAA mouse, in which uORF start codon is mutated. (B) RT-qPCR assessing PPARGC1A mRNA in injured uORFTAAmouse kidney $m=8-10$ ; error bars represent the SE). (C) Immunoblot for $\mathsf{P G C1}\,\mathsf{a}$ in injured kidney after immunoprecipitation from kidney lysates. TBP immunoblot of input is shown. (D) Top: protein lysates of uninjured kidneys were labeled with isobaric tags and subjected to tandem mass spectrometry $(\mathsf{n}=4)$ . Bottom: gene set enrichment plot. Proteins detected in MS were ranked based on the relative amount in uORFTAAkidney versus wild-type kidney. The first ranked protein is the most upregulated in uORFTAA kidney ( $|0\mathtt{g}_{2}$ scale). Hash marks represent positions in the ranked list corresponding to members of a given gene set. The normalized enrichment score (NES) indicates whether these members are enriched toward the upregulated end of this list (positive NES) or downregulated end of this list (negative NES) as compared to the chance expectation.  

(E) Injured kidney protein lysates blotted for protein components (right) of mitochondrial oxidative phosphorylation complexes $\mid-\lor$ (left).  

In addition to the kidney, preliminary characterization of skeletal muscle and brown adipose tissue (BAT) in uORFTAA mice further supports a role for the PPARGC1A uORF in regulating $\mathsf{P G C1}\,\mathsf{a}$ function and OXPHOS metabolism. In both quadriceps muscle and BAT, gene set enrichment analysis of RNA-seq data comparing uORFTAA versus wild-type mice revealed ‘‘oxidative phosphorylation’’ as the most significantly enriched gene set (Figures S4A–S4E). Skeletal muscle of uORFTAA mice also showed enrichment of a ‘‘PGC1a signature’’ gene set, as defined by genes that are upregulated in the muscle of MCK$\mathsf{P G C1}\,\mathsf{a}$ transgenic mice (Lin et al., 2002). In BAT, the gene expression changes in ${\mathsf{u O R F}}^{\mathsf{T A A}}$ mice were associated with elevated $\mathsf{P G C1}\,\mathsf{a}$ protein (but unchanged PPARGC1A mRNA) as well as increased tissue oxygen consumption (Figures S4F– S4H). Thus, our data establish the murine PPARGC1A uORF as a functional and physiologically impactful translation regulatory element in vivo.  

# DISCUSSION  

Mitochondrial biogenesis is a crucial process, allowing efficient use of fuels to produce ATP or heat. On the other hand, mitochondria are a source of potentially damaging reactive oxygen species and are energetically costly to produce, requiring assembly of over 1,500 proteins, from two distinct genomes, into specific subcellular structures (Scarpulla, 2011). As a consequence, mitochondrial biogenesis is tightly coupled to environmental cues. While other transcriptional components are also crucial for the expression of mitochondrial genes, it is typically $\mathsf{P G C1}\,\mathsf{a}$ that is quantitatively regulated in states of greater mitochondrial demand (Uldry et al., 2006). Our studies show that $\mathsf{P G C1}\,\mathsf{a}$ is under regulation by an evolutionarily conserved uORF in the PPARGC1A $5^{\prime}$ UTR.  

The extensive conservation of inhibitory uORF function in PPARGC1A orthologs, as well as our observation that multiple PPARGC1A uORFs have arisen independently in vertebrate evolution, highlight the idea that uORF alterations can contribute to the evolution of differences in mitochondrial density and distribution between species. In particular, the Atlantic bluefin tuna stands out as unusual because nearly its entire muscle bed is visibly red, a result of both increased mitochondrial-rich (‘‘slow-twitch’’) muscle as well as an unusually high oxidative capacity within its ‘‘fast-twitch’’ muscle (Korsmeyer and Dewar, 2001; Moyes, 2003). The tuna’s unique metabolic phenotype likely underlies the swimming endurance for which it is famed. Although tuna is not currently experimentally tractable, our findings using the cloned tuna ppargc1a $5^{\prime}$ UTR in zebrafish systems suggest that tuna, by virtue of uORF loss and likely other $5^{\prime}$ UTR alterations, encodes a ppargc1a mRNA that is more robustly translated, perhaps contributing at least in part to its unique physiology.  

Cap-dependent translation is a decision point at which regulation can allow complex biological control (Jackson et al.,  

2010). That uORFs might contribute to such regulation has gained appreciation as ribosome profiling studies have highlighted their prevalence (Ingolia et al., 2014; McGillivray et al., 2018). Indeed, studies in model systems have established roles for uORFs in coordinating dynamic gene expression programs, including in the context of cell differentiation or stress (Brar et al., 2012; Young and Wek, 2016). The conservation of a uORF in PPARGC1A, as well as our finding that its inhibitory effect can be modulated by pharmacological induction of cellular stress, raises the likelihood that this uORF contributes to dynamic regulation of $\mathsf{P G C1}\,\mathsf{a}$ . The uORF may also confer regulation by virtue of inclusion or exclusion in different transcript isoforms (Cheng et al., 2018); we note that it is absent in transcripts encoding PGC1a4 and PGC1 $\pmb{\alpha}$ -b (Martı´nez-Redondo et al., 2015).  

Given the generally modest effects of uORFs, many insights regarding their function have been obtained by genome-wide analysis. The extent to which an individual uORF can influence mammalian physiology by quantitative modulation of its downstream protein (as opposed to by effects on protein isoform choice) has been unclear (Wethmar et al., 2010a). Our studies demonstrate that preventing ribosome recognition of a single uORF element in the PPARGC1A gene is sufficient to alter $\mathsf{P G C1}\,\mathsf{a}$ functional output in multiple metabolically important tissues and to confer protection from AKI. The setting of AKI may be particularly sensitive to PGC1a levels, given that PGC1a function and oxidative phosphorylation systems decrease in the context of renal injury (Tran et al., 2016). Thus, it is possible that the PPARGC1A uORF mutation confers protection from AKI by preventing the AKI-associated drop in PGC1a function. It is also possible that the PGC1a elevation per se, present in baseline conditions, is responsible for the mutation’s therapeutic effect.  

Our findings provide a proof of principle that translational control of PGC1a operates within a physiologic window large enough to influence disease processes. Furthermore, our mouse model offers new opportunities to investigate $\mathsf{P G C1}\,\mathsf{a}$ function and translational control specifically at times and sites in which the PPARGC1A mRNA is normally expressed, a potential advantage over transgenic models that force heterologous expression.  

# Limitations of Study  

Our study identifies a translational regulatory element controlling $\mathsf{P G C1}\,\mathsf{a}$ expression and investigates its importance through several approaches: evolutionary conservation, functional effects on translation, and a knock-in mouse model. Evolutionary approaches are challenged by the relatively limited genomic annotations in non-model organisms and by our inability to perform experiments using tuna or their cells. Translational reporter assays lack the natural complexity of the PPARGC1A promoter, which expresses multiple transcript isoforms. For this reason, our knock-in mouse model is especially important for demonstrating the uORF’s impact on PPARGC1A translation, as well as on pathophysiology. It will be valuable in the future to examine these mice not only under challenges that cause decreased PGC1a expression (like AKI) but also under challenges that elevate $\mathsf{P G C1}\,\mathsf{a}$ (like exercise). Such studies may help reveal physiologic signals that act by modulating uORF function.  

# STAR+METHODS  

Detailed methods are provided in the online version of this paper and include the following:  

d KEY RESOURCES TABLE   
d CONTACT FOR REAGENT AND RESOURCE SHARING   
d EXPERIMENTAL MODEL AND SUBJECT DETAILS $\bigcirc$ Mice $\bigcirc$ Generation of uORF Deficient Animals B Zebrafish B Cell Culture   
d METHOD DETAILS $\bigcirc$ Plasmids $\bigcirc$ Gene Expression Analysis (RT-qPCR) B Immunoblotting $\bigcirc$ Immunoprecipitation B DNA Transfection B In Vitro Translation Assay $\bigcirc$ Luciferase Translation Assay B Cloning of Full-Length PPARGC1A cDNA from Bluefin Tuna Thunnus thynnus B mRNA Injections into Zebrafish $\bigcirc$ Mouse Ischemia-Reperfusion Injury Experiments B Histology B Tissue Respiration B RNA-Seq Library Preparation and Sequencing B RNA-seq and Gene Set Enrichment Analysis B Tissue Harvest, Protein Digest, and Peptide Isobaric Labeling B Basic pH Reversed-Phase Separation (BPRP) B Liquid Chromatography Separation and Tandem Mass Spectrometry (LC-MS/MS) B Peptide Quantification and Gene Set Enrichment Analysis   
d QUANTIFICATION AND STATISTICAL ANALYSIS B Experimental Replicates and Quantification B Statistical Analysis   
d DATA AND SOFTWARE AVAILABILITY  

# SUPPLEMENTAL INFORMATION  

Supplemental Information can be found online at https://doi.org/10.1016/j.   
cmet.2019.04.013.  

# ACKNOWLEDGMENTS  

This study was aided by the JPB Foundation, the National Institutes of Health (NIH DK061562 to B.M.S; 1F32DK112638-01A1 to D.E.), and the Damon Runyon Cancer Research Foundation (P.A.D.). We thank the Harvard histopathology core for histology. Work in S.M.P.’s laboratory is supported by R01- DK095072 and R35-HL139424. We thank members of the Spiegelman lab for helpful discussions.  

# AUTHOR CONTRIBUTIONS  

P.A.D. and D.F.E. designed the research, performed biochemical, cellular, and in vivo experiments, and analyzed data. P.G. and A.P. designed and performed zebrafish experiments. M.T.T. designed and performed ischemia-reperfusion experiments. N.C. designed and performed Drosophila experiments. M.P. assisted with biochemistry. L.K. aided in mouse studies. F.D., S.E.W., and D.B. provided technical assistance. J.A.C. caught Atlantic bluefin tuna and extracted RNA. S.K. sequenced tuna PPARGC1A locus. N.P. directed Drosophila experiments. S.M.P. directed kidney experiments. B.M.S. directed the research. P.A.D., D.F.E., and B.M.S. wrote the manuscript with assistance from all other authors.  

# DECLARATION OF INTERESTS  

BMS is a consultant for Calico, Inc.  

Received: November 27, 2018   
Revised: March 25, 2019   
Accepted: April 21, 2019   
Published: May 16, 2019  

# REFERENCES  

Andreev, D.E., O’Connor, P.B.F., Fahey, C., Kenny, E.M., Terenin, I.M., Dmitriev, S.E., Cormican, P., Morris, D.W., Shatsky, I.N., and Baranov, P.V. (2015). Translation of 50 leaders is pervasive in genes resistant to eIF2 repression. eLife 4, e03971.   
Barbosa, C., Peixeiro, I., and Roma˜ o, L. (2013). Gene expression regulation by upstream open reading frames and human disease. PLoS Genet. 9, e1003529. Blanchette, M., Kent, W.J., Riemer, C., Elnitski, L., Smit, A.F.A., Roskin, K.M., Baertsch, R., Rosenbloom, K., Clawson, H., Green, E.D., et al. (2004). Aligning multiple genomic sequences with the threaded blockset aligner. Genome Res. 14, 708–715.   
Block, B.A., Teo, S.L.H., Walli, A., Boustany, A., Stokesbury, M.J.W., Farwell, C.J., Weng, K.C., Dewar, H., and Williams, T.D. (2005). Electronic tagging and population structure of Atlantic bluefin tuna. Nature 434, 1121–1127.   
Brar, G.A., Yassour, M., Friedman, N., Regev, A., Ingolia, N.T., and Weissman, J.S. (2012). High-resolution view of the yeast meiotic program revealed by ribosome profiling. Science 335, 552–557.   
Calvo, S.E., Pagliarini, D.J., and Mootha, V.K. (2009). Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. USA 106, 7507–7512.   
Chawla, L.S., Eggers, P.W., Star, R.A., and Kimmel, P.L. (2014). Acute kidney injury and chronic kidney disease as interconnected syndromes. N. Engl. J. Med. 371, 58–66.   
Cheng, Z., Otto, G.M., Powers, E.N., Keskin, A., Mertins, P., Carr, S.A., Jovanovic, M., and Brar, G.A. (2018). Pervasive, coordinated protein-level changes driven by transcript isoform switching during meiosis. Cell 172, 910–923.   
Chew, G.-L., Pauli, A., and Schier, A.F. (2016). Conservation of uORF repressiveness and sequence features in mouse, human and zebrafish. Nat. Commun. 7, 11663.   
Churbanov, A., Rogozin, I.B., Babenko, V.N., Ali, H., and Koonin, E.V. (2005). Evolutionary conservation suggests a regulatory function of AUG triplets in 5’-UTRs of eukaryotic genes. Nucleic Acids Res. 33, 5512–5520.   
Da Cruz, S., Parone, P.A., Lopes, V.S., Lillo, C., McAlonis-Downes, M., Lee, S.K., Vetto, A.P., Petrosyan, S., Marsala, M., Murphy, A.N., et al. (2012). Elevated PGC-1a activity sustains mitochondrial biogenesis and muscle function without extending survival in a mouse model of inherited ALS. Cell Metab. 15, 778–786.  

Dalziel, A.C., Moore, S.E., and Moyes, C.D. (2005). Mitochondrial enzyme content in the muscles of high-performance fish: evolution and variation among fi- ber types. Am. J. Physiol. Regul. Integr. Comp. Physiol. 288, R163–R172.  

Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21.   
Elias, J.E., and Gygi, S.P. (2007). Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry. Nat. Methods 4, 207–214.   
Eng, J.K., McCormack, A.L., and Yates, J.R. (1994). An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. J. Am. Soc. Mass Spectrom. 5, 976–989.   
Fernandez-Marcos, P.J., and Auwerx, J. (2011). Regulation of PGC-1a, a nodal regulator of mitochondrial biogenesis. Am. J. Clin. Nutr. 93, 884S–890S. Fisher, F.M., Kleiner, S., Douris, N., Fox, E.C., Mepani, R.J., Verdeguer, F., Wu, J., Kharitonenkov, A., Flier, J.S., Maratos-Flier, E., and Spiegelman, B.M. (2012). FGF21 regulates PGC-1a and browning of white adipose tissues in adaptive thermogenesis. Genes Dev. 26, 271–281.   
Gawron, D., Ndah, E., Gevaert, K., and Van Damme, P. (2016). Positional proteomics reveals differences in N-terminal proteoform stability. Mol. Syst. Biol. 12, 858.   
Han, S.H., Wu, M.-Y., Nam, B.Y., Park, J.T., Yoo, T.-H., Kang, S.-W., Park, J., Chinga, F., Li, S.-Y., and Susztak, K. (2017). PGC-1a Protects from notchinduced kidney fibrosis development. J. Am. Soc. Nephrol. 28, 3312–3322. Handschin, C., Kobayashi, Y.M., Chin, S., Seale, P., Campbell, K.P., and Spiegelman, B.M. (2007). PGC-1alpha regulates the neuromuscular junction program and ameliorates Duchenne muscular dystrophy. Genes Dev. 21, 770–783.   
Huttlin, E.L., Jedrychowski, M.P., Elias, J.E., Goswami, T., Rad, R., Beausoleil, S.A., Ville´ n, J., Haas, W., Sowa, M.E., and Gygi, S.P. (2010). A tissue-specific atlas of mouse protein phosphorylation and expression. Cell 143, 1174–1189. Ingolia, N.T., Brar, G.A., Stern-Ginossar, N., Harris, M.S., Talhouarne, G.J.S., Jackson, S.E., Wills, M.R., and Weissman, J.S. (2014). Ribosome profiling reveals pervasive translation outside of annotated protein-coding genes. Cell Rep. 8, 1365–1379.   
Jackson, R.J., Hellen, C.U.T., and Pestova, T.V. (2010). The mechanism of eukaryotic translation initiation and principles of its regulation. Nat. Rev. Mol. Cell Biol. 11, 113–127.   
Katz, S.L. (2002). Design of heterothermic muscle in fish. J. Exp. Biol. 205, 2251–2266.   
Korsmeyer, K.E., and Dewar, H. (2001). Tuna metabolism and energetics. In Tuna: Physiology, Ecology, and Evolution, B. Block and E. Donald Stevens, eds. (Elsevier), pp. 35–78.   
Liberzon, A., Birger, C., Thorvaldsdo´ ttir, H., Ghandi, M., Mesirov, J.P., and Tamayo, P. (2015). The molecular signatures database (MSigDB) hallmark gene set collection. Cell Syst. 1, 417–425.   
Lin, J., Wu, H., Tarr, P.T., Zhang, C.-Y., Wu, Z., Boss, O., Michael, L.F., Puigserver, P., Isotani, E., Olson, E.N., et al. (2002). Transcriptional co-activator PGC-1 alpha drives the formation of slow-twitch muscle fibres. Nature 418, 797–801.   
Lin, J., Wu, P.-H., Tarr, P.T., Lindenberg, K.S., St-Pierre, J., Zhang, C.-Y., Mootha, V.K., Ja€ger, S., Vianna, C.R., Reznick, R.M., et al. (2004). Defects in adaptive energy metabolism with CNS-linked hyperactivity in PGC-1alpha null mice. Cell 119, 121–135.   
Liu, J., Kumar, S., Dolzhenko, E., Alvarado, G.F., Guo, J., Lu, C., Chen, Y., Li, M., Dessing, M.C., Parvez, R.K., et al. (2017). Molecular characterization of the transition from acute to chronic kidney injury following ischemia/reperfusion. JCI Insight 2, e94716.   
Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550. Lynch, M.R., Tran, M.T., and Parikh, S.M. (2018). PGC1a in the kidney. Am. J. Physiol. Renal Physiol. 314, F1–F8.   
Martı´nez-Redondo, V., Pettersson, A.T., and Ruas, J.L. (2015). The hitchhiker’s guide to PGC-1a isoform structure and biological functions. Diabetologia 58, 1969–1977.   
McAlister, G.C., Nusinow, D.P., Jedrychowski, M.P., W€uhr, M., Huttlin, E.L., Erickson, B.K., Rad, R., Haas, W., and Gygi, S.P. (2014). MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes. Anal. Chem. 86, 7150–7158.   
McGillivray, P., Ault, R., Pawashe, M., Kitchen, R., Balasubramanian, S., and Gerstein, M. (2018). A comprehensive catalog of predicted functional upstream open reading frames in humans. Nucleic Acids Res. 46, 3326–3338.   
Mootha, V.K., Lindgren, C.M., Eriksson, K.-F., Subramanian, A., Sihag, S., Lehar, J., Puigserver, P., Carlsson, E., Ridderstra˚ le, M., Laurila, E., et al. (2003). PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nat. Genet. 34, 267–273.   
Moyes, C.D. (2003). Controlling muscle mitochondrial content. J. Exp. Biol. 206, 4385–4391.   
Pagliarini, D.J., Calvo, S.E., Chang, B., Sheth, S.A., Vafai, S.B., Ong, S.-E., Walford, G.A., Sugiana, C., Boneh, A., Chen, W.K., et al. (2008). A mitochondrial protein compendium elucidates complex I disease biology. Cell 134, 112–123.   
Park, J.-E., Yi, H., Kim, Y., Chang, H., and Kim, V.N. (2016). Regulation of poly(A) tail and translation during the somatic cell cycle. Mol. Cell 62, 462–471.   
Pettersson-Klein, A.T., Izadi, M., Ferreira, D.M.S., Cervenka, I., Correia, J.C., Martinez-Redondo, V., Southern, M., Cameron, M., Kamenecka, T., Agudelo, L.Z., et al. (2018). Small molecule PGC-1a1 protein stabilizers induce adipocyte Ucp1 expression and uncoupled mitochondrial respiration. Mol. Metab. 9, 28–42.   
Puigserver, P., Wu, Z., Park, C.W., Graves, R., Wright, M., and Spiegelman, B.M. (1998). A cold-inducible coactivator of nuclear receptors linked to adaptive thermogenesis. Cell 92, 829–839.   
Ruas, J.L., White, J.P., Rao, R.R., Kleiner, S., Brannan, K.T., Harrison, B.C., Greene, N.P., Wu, J., Estall, J.L., Irving, B.A., et al. (2012). A PGC-1a isoform induced by resistance training regulates skeletal muscle hypertrophy. Cell 151, 1319–1331.   
Sandri, M., Lin, J., Handschin, C., Yang, W., Arany, Z.P., Lecker, S.H., Goldberg, A.L., and Spiegelman, B.M. (2006). PGC-1alpha protects skeletal muscle from atrophy by suppressing FoxO3 action and atrophy-specific gene transcription. Proc. Natl. Acad. Sci. USA 103, 16260–16265.   
Scarpulla, R.C. (2011). Metabolic control of mitochondrial biogenesis through the PGC-1 family regulatory network. Biochim. Biophys. Acta 1813, 1269– 1278.   
Subramanian, A., Tamayo, P., Mootha, V.K., Mukherjee, S., Ebert, B.L., Gillette, M.A., Paulovich, A., Pomeroy, S.L., Golub, T.R., Lander, E.S., and Mesirov, J.P. (2005). Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. USA 102, 15545–15550.   
Ting, L., Rad, R., Gygi, S.P., and Haas, W. (2011). MS3 eliminates ratio distortion in isobaric multiplexed quantitative proteomics. Nat. Methods 8, 937–940. Tran, M., Tam, D., Bardia, A., Bhasin, M., Rowe, G.C., Kher, A., Zsengeller, Z.K., Akhavan-Sharif, M.R., Khankin, E.V., Saintgeniez, M., et al. (2011). PGC-1a promotes recovery after acute kidney injury during systemic inflammation in mice. J. Clin. Invest. 121, 4003–4014.   
Tran, M.T., Zsengeller, Z.K., Berg, A.H., Khankin, E.V., Bhasin, M.K., Kim, W., Clish, C.B., Stillman, I.E., Karumanchi, S.A., Rhee, E.P., and Parikh, S.M. (2016). PGC1a drives NAD biosynthesis linking oxidative metabolism to renal protection. Nature 531, 528–532.   
Tsukiyama-Kohara, K., Poulin, F., Kohara, M., DeMaria, C.T., Cheng, A., Wu, Z., Gingras, A.C., Katsume, A., Elchebly, M., Spiegelman, B.M., et al. (2001). Adipose tissue reduction in mice lacking the translational inhibitor 4E-BP1. Nat. Med. 7, 1128–1132.   
Uldry, M., Yang, W., St-Pierre, J., Lin, J., Seale, P., and Spiegelman, B.M. (2006). Complementary action of the PGC-1 coactivators in mitochondrial biogenesis and brown fat differentiation. Cell Metab. 3, 333–341.   
Wang, X., Spandidos, A., Wang, H., and Seed, B. (2012). PrimerBank: a PCR primer database for quantitative gene expression analysis, 2012 update. Nucleic Acids Res. 40, D1144–D1149.  

Wang, X., Lu, Z., Gomez, A., Hon, G.C., Yue, Y., Han, D., Fu, Y., Parisien, M., Dai, Q., Jia, G., et al. (2014). N6-methyladenosine-dependent regulation of messenger RNA stability. Nature 505, 117–120.  

Wethmar, K., Be´ gay, V., Smink, J.J., Zaragoza, K., Wiesenthal, V., Do¨ rken, B., Calkhoven, C.F., and Leutz, A. (2010a). C/EBPbetaDeltauORF mice—a genetic model for uORF-mediated translational control in mammals. Genes Dev. 24, 15–20.  

Wethmar, K., Smink, J.J., and Leutz, A. (2010b). Upstream open reading frames: molecular switches in (patho)physiology. BioEssays 32, 885–893.  

Yoon, J.C., Puigserver, P., Chen, G., Donovan, J., Wu, Z., Rhee, J., Adelmant, G., Stafford, J., Kahn, C.R., Granner, D.K., et al. (2001). Control of hepatic gluconeogenesis through the transcriptional coactivator PGC-1. Nature 413,  

131–138.   
Young, S.K., and Wek, R.C. (2016). Upstream open reading frames differentially regulate gene-specific translation in the integrated stress response. J. Biol. Chem. 291, 16927–16935.   
Zheng, B., Liao, Z., Locascio, J.J., Lesniak, K.A., Roderick, S.S., Watt, M.L., Eklund, A.C., Zhang-James, Y., Kim, P.D., Hauser, M.A., et al. (2010). PGC1a, a potential therapeutic target for early intervention in Parkinson’s disease. Sci. Transl. Med. 2, 52ra73.  

Zur, H., Aviner, R., and Tuller, T. (2016). Complementary post transcriptional regulatory information is detected by PUNCH-P and ribosome profiling. Sci. Rep. 6, 21635.  

# STAR+METHODS  

KEY RESOURCES TABLE   

![](images/eeeabaa38a14707d4bba801b59e47e54d855a31361602e5acf654004f0571d87.jpg)  

![](images/2b6cc49b6e6fb86dae825aa8729a0645dd00b725a08c89cf416c2a4e8f12aedc.jpg)  

# CONTACT FOR REAGENT AND RESOURCE SHARING  

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Bruce Spiegelman (bruce_spiegelman@dfci.harvard.edu).  

# EXPERIMENTAL MODEL AND SUBJECT DETAILS  

# Mice  

Mice were housed at $^{23\circ}\mathrm{C}$ under a 12 hr light/dark cycle with free access to food and water. All experiments used 8-12 week old matched littermates. PPARGC1A uORFTAA mice and littermate uORFWT controls were generated by breeding heterozygous mice in a C57BL/6J background. Animal experiments were performed according to procedures approved by the Institutional Animal Care and Use Committee (IACUC) of the Beth Israel Deaconess Medical Center.  

# Generation of uORF Deficient Animals  

uORFTAAheterozygous mice were generated by Applied Stem Cell Inc.  

mPpargc1a.g9 is $5^{\prime}$ -CACACAGCACACACTCATGCAGG- ${\mathcal{B}}^{\prime}$ . Protospacer Adjacent Motif is underlined. mPpargc1a.g9-ssODN is: 50ctactttttaatagctttgtcatgtgactggggactgtagtaagacaggtgccttcagttcactctcagtaaggggctggtTgcctgctaaagtg tgtgctgtgtgtcagagtggattggagttgaaaaagcttgactggcgtcattcgggagctggatggcttggga- ${\mathcal{B}}^{\prime}$ . The resultant heterozygous mutant mouse was backcrossed with C57BL/6J for a minimum of three generations before carrying out experimental procedures.  

To detect the uORF mutation, PCR was performed with the primers DE144 $\Xi^{\prime}$ -ggagtttgtgcagcaagcttg- $3^{\prime}$ ) and DE384 $\mathrm{^{5^{\prime}}}$ -GAAAG AGTTTCTAACTGCTAC-30). The PCR product was sequenced using primer DE383 ( $5^{\prime}$ -GACTGGGGACTGTAGTAAG-30) by Eton Bio.  

# Zebrafish  

AB zebrafish were raised at $28^{\circ}\mathrm{C}$ under standard housing conditions. All experimental procedures were carried out according to Swiss and EU ethical guidelines and were carried out in an approved animal facility.  

# Cell Culture  

Human HEK293E cells and monkey COS cells (ATCC) were cultured in DMEM containing $10\%$ fetal bovine serum (HyClone) and penicillin/streptomycin at $37^{\circ}{\mathrm{C}}$ in $5\%$ $\mathsf{C O}_{2}$ . Drosophila ${}^{82\mathsf{R}+}$ cells were cultured in Schneider’s medium (Invitrogen) containing $10\%$ heat-inactivated FBS and penicillin/streptomycin at $25\mathrm{{^\circC}}$ .  

# METHOD DETAILS  

# Plasmids  

The cDNA for mouse PPARGC1A (Genbank: NM_008904.2) was cloned from a brown fat cDNA library and inserted into pDONR221 using BP clonase (Invitrogen). Site-directed mutagenesis was performed using QuikChange IIXL (Stratagene). For expression, plasmids were then cloned into pcDNA6.2V5dest(Invitrogen) using LR reaction.  

# Gene Expression Analysis (RT-qPCR)  

Total RNA was extracted from frozen tissue or cells using TRIzol (Invitrogen), purified with RNeasy Mini spin columns (QIAGEN) and reverse transcribed using a High-Capacity cDNA Reverse Transcription kit (Applied Biosystems). The resultant cDNA was analyzed by RT-qPCR using SYBR green fluorescent dye 2x qPCR master mix (Promega) in a ABI PRISM 7900HT real time PCR system (Applied Biosystems). Relative mRNA levels were normalized to TBP or GAPDH mRNA and calculated using the DDCt method. Primer sequences are shown in Table S3 (Wang et al., 2012).  

# Immunoblotting  

Tissue or cell culture samples were prepared in ice-cold Lysis Buffer $20\,\mathsf{m M}$ Tris, pH 7.4, 180 mM NaCl, 1mM EDTA, 1mM EGTA, $\mid\%$ Triton X-100, $2.5\;\mathrm{mM}$ pyrophosphate, 50mM NaF, 5mM glyerco-phosphoate, 50nM calyculin A, 1mM $\mathsf{N a}_{3}\mathsf{V O}_{4}$ and supplemented with Complete EDTA-free protease inhibitor tablet (Roche). Lysates were sonicated, then were centrifuged at $16{,}000\;\mathrm{g}$ for $15\;\mathsf{m i n}$ at $4\mathrm{{}^{\circ}C}$ , and the supernatants were used for subsequent analyses. Protein concentration was determined using the bicinchoninic acid assay (Pierce). Protein lysates were denatured in Laemmli buffer ${}^{\prime}60\,\mathsf{m M}$ Tris, $\mathsf{p H}\,6.8$ ,$2\%$ SDS, $10\%$ glycerol, $0.05\%$ bromophenol blue, 0.7 M $\upbeta$ -mercaptoethanol), resolved by $4\%{-}12\%$ NuPAGE Bis-Tris SDS–PAGE (Invitrogen) and transferred to a polyvinylidene difluoride (PVDF $0.45~{\upmu\mathrm{m}})$ membrane. Primary antibodies were diluted in TBS containing $0.05\%$ Tween (TBS- $\cdot\mathsf{T})$ ), $5\%$ BSA, and $0.02\%\;\mathsf{N a N}_{3}$ . Membranes were incubated overnight with primary antibodies at $4\mathrm{{}^{\circ}C}$ . For secondary antibody incubation, anti-rabbit or anti-mouse HRP (Promega) was diluted in TBS-T containing $5\%$ dry nonfat milk. Results were visualized with enhanced chemiluminescence (ECL) western blotting substrates (Pierce). Densitometry was performed using ImageJ $1.52\mathsf{k}$ . For each image, signal was normalized such that the average of all control samples equaled 1.0.  

# Immunoprecipitation  

$\mathsf{P G C1}\,\mathsf{a}$ detection in mouse tissue required immunoprecipitation prior to immunoblotting (Ruas et al., 2012). For BAT, one lobe of material was lysed by mechanical homogenization in Lysis Buffer (recipe above). Lysates were sonicated, centrifuged at $16{,}000\;\mathrm{g}$ for 15 min at $4\mathrm{{}^{\circ}C}$ , and the supernatants removed. Two mg total protein was used for each IP. To pre-clear the lysate, the input protein was incubated 2 hr $4\mathrm{{}^{\circ}C}$ with Protein A/G magnetic beads (Thermo) ( $10\,\upmu\mathrm{L}$ slurry, washed prior to addition). Beads were removed and then $4\,\upmu\mathfrak{g}$ anti- $P G C10.$ was added (D5, Santa Cruz), followed by overnight rotation at $4^{\circ}\mathrm{C}.$ . The next day, $15\,\upmu\mathrm{L}$ Protein $\mathsf{A}/\mathsf{G}$ magnetic bead slurry was washed and added to each IP, followed by rotation at $4\mathrm{{}^{\circ}C}$ for $\mathsf{3\,h r}$ . Beads were washed $3\times$ in lysis buffer and immunoprecipitated protein was eluted in Laemmli buffer. For kidney, one half kidney was lysed by mechanical homogenization in low-salt buffer ( $50\,\mathsf{m M}$ Tris pH 7.6, 50 mM KCl, $\mid\%$ NP-40, $20\%$ glycerol, $_{2\,\mathsf{m M}}$ EDTA, protease inhibitor). Lysate was centrifuged 12,000 g for $10\,\mathsf{m i n}$ at $4\mathrm{{^\circC}}$ , and supernatant was removed. The pellet was extracted with high-salt buffer ( $50\,\mathsf{m M}$ Tris pH 7.6, 400 mM KCl, $1\,\%$ NP40, $20\%$ glycerol, $^{2}\,\mathsf{m M}$ EDTA, protease inhibitor), sonicated, and centrifuged $16{,}000\,\mathfrak{g}$ for 15 min at $4\mathrm{{}^{\circ}C}$ . The supernatant was used for protein quantitation and subsequent IP following protocol above, with washes in medium-salt buffer $\mathsf{50\,m M}$ Tris pH 7.6, $200\,\mathsf{m M}$ KCl, $\mid\%$ NP-40, $20\%$ glycerol, $_{2\mathsf{\Pi}_{\mathsf{m M}}}$ EDTA, protease inhibitor),  

# DNA Transfection  

For transient expression of proteins, HEK293E cells were transfected with plasmids using Lipofectamine 2000 (Invitrogen) and lysed for immunoblotting $24\,\mathrm{hr}$ later. Drosophila ${}^{82\mathsf{R}+}$ cells were transfected with Effectene (QIAGEN).  

# In Vitro Translation Assay  

TNT $\mathbf{\widehat{\Phi}}(\mathbf{\widehat{R}})$ Quick Coupled Transcription/Translation Systems kit (Promega) was used for in vitro transcription and translation. One microgram of each plasmid was used per reaction, and $1~\upmu\mathrm{L}$ of each reaction was used for immunoblotting.  

# Luciferase Translation Assay  

The full-length PPARGC1A 50UTR was cloned upstream of the nanoLuc coding sequence (Promega), with preservation of the endogenous PPARGC1A Kozak sequence. This sequence was then amplified using a forward PCR primer containing a T7 promoter sequence, and the resulting DNA was purified. RNA was transcribed and polyadenylated from this template using the mMESSAGE mMACHINE T7 Ultra kit (Invitrogen), checked for integrity by denaturing gel electrophoresis, purified by phenol-chloroform extraction and precipitation, and quantified by $\mathsf{A}_{260}$ using a NanoDrop (Thermo Fisher).  

For translation assays, RNA was transfected into COS cells using Lipofectamine 2000 in a ratio of $2.5~\upmu\mathrm{L}$ Lipofectamine 2000 per $1~\upmu\mathfrak{g}$ RNA. COS cells were allowed to reach $90\%$ confluence in 24-well plates prior to transfection. Immediately prior to transfection, medium was replaced with serum-free DMEM with or without sodium arsenite (Sigma). Cells were harvested 3 hr after transfection and luciferase activity was assessed using Nano-Glo luciferase assay (Promega) and a FLUOstar Omega plate reader (BMG Labtech).  

# Cloning of Full-Length PPARGC1A cDNA from Bluefin Tuna Thunnus thynnus  

A wild Atlantic bluefin tuna (Thunnus thynnus) was caught off Cape Cod. Skeletal muscle tissues from the tail were dissected and immediately frozen on dry ice. Total RNA from the skeletal muscle tissue was extracted with TRIzol and purified using RNeasy spin columns (QIAGEN). Full-length cDNA library was constructed from purified tuna mRNAs using the SMART cDNA synthesis (TAKARA Bio). A partial cDNA coding tuna ppargc1a was amplified in the tuna cDNA library by using primers (Forward $5^{\prime}.$ - CACA GAACCATGCAGCCAGCA $-3^{\prime}$ and Reverse $5^{\prime}.$ - CCA GGT GCT CTT GGC TTC AC $-3^{\prime}$ ) that were designed for conserved regions of ppargc1a sequences among vertebrates. Subsequently, amplified cDNAs were cloned into a pCR4-TOPO vector (Invitrogen) for sequencing. $3^{\prime}$ RACE was performed by amplifying tuna PPARGC1A cDNAs using a tuna specific primer ( $5^{\prime}$ -GGC TTC ATA ACT TAC CGC TAC ACC-30) and adaptor primers for tuna cDNA library. Similarly, $5^{\prime}$ RACE was performed by using a tuna specific primer ($\Xi^{\prime}$ -TCT CTT GAC TGG CTT TGT GTG G-30) and adaptor primers specific to the $5^{\prime}$ linker regions. Full-length cDNA sequence was confirmed by reading more than five independent clones both in $3^{\prime}$ RACE and $5^{\prime}$ RACE.  

# mRNA Injections into Zebrafish  

In vitro transcription of plasmids 0.25ppargc1a:ZsGreen-Nanoluc- $\cdot{\mathsf{p C S}}2+$ ,0.25ppargc1a-0.225ATG>TAA,-0.126ATG>TAA:ZsGreen-Nanoluc- $\cdot{\mathsf{p C S}}2+$  

0.12tuna_ppargc1a:ZsGreen-Nanoluc-pCS2, and $_{-0.25}$ ppargc1a:mCherry-Luc2-pCS2 was carried out using an Ambion mMessage Sp6 in vitro transcription kit after linearization with NotI according to the manufacturer’s instructions. One nanoliter of $12.5\mathsf{n g}/\upmu\mathrm{l}$ capped mRNA was injected into 1-cell zygotes and visualized 48 hours post fertilization for fluorescence expression as a readout for translation efficiency using a Leica M165F stereoscope. 0.25ppargc1a:mCherry-Luc2 mRNA was co-injected to control for comparable mRNA levels among experiments. Fluorescence intensity was quantified using FIJI. Quantification of translation efficiency was performed using a Perkin Elmer EnVision plate reader. In brief, zebrafish zygotes were injected with 1nl of $1\ \mathsf{n g/\upmu l}$ of the above listed plasmids and luminescence emission values from Nanoluc luciferase were normalized to the Firefly Luciferase values of a co-injected  

0.25ppargc1a:mCherry-Luc2 control mRNA using a Promega Dual-Glo assay system according the manufacturer’s instructions.  

# Mouse Ischemia-Reperfusion Injury Experiments  

Bilateral renal ischemia-reperfusion injury (IRI) was performed as previously described (Tran et al., 2016). Blood and kidneys were harvested 24 hours after IRI. Serum creatinine measurements were analyzed by LC/MS-MS at the University of Alabama Birmingham O’Brien Core Center for Acute Kidney Injury Research.  

# Histology  

Kidneys were fixed in $4\%$ paraformaldehyde. Paraffin embedding and sectioning were done by the Dana-Farber/Harvard Cancer Center Research Pathology core facility and stained with hematoxylin and eosin.  

# Tissue Respiration  

Freshly isolated BAT was rapidly weighed and minced using scissors until homogeneous (3 min). The resulting homogenate was added to $1\ \mathsf{m L}$ tissue respiration buffer (10 mM ${\mathsf{N a}}_{2}{\mathsf{H P O}}_{4}$ , 2 mM $\mathsf{K H}_{2}\mathsf{P O}_{4}$ , 2.7 mM KCl, 137 mM NaCl, $2\%$ essentially fatty acid free BSA, 1 mM sodium pyruvate, $25\,\mathsf{m M}$ glucose, pH 7.4). Oxygen consumption rate was measured in a Clark-type oxygen electrode (Strathkelvin Instruments) at $37^{\circ}\mathrm{C}$ and normalized to tissue mass.  

# RNA-Seq Library Preparation and Sequencing  

Libraries were prepared using Kapa stranded mRNA Hyper Prep sample preparation kits from 500 ng of total RNA isolated from frozen mouse tissue using RNeasy columns (QIAGEN). The finished dsDNA libraries were quantified by Qubit fluorometer, Agilent TapeStation 2200, and RT-qPCR using the Kapa Biosystems library quantification kit according to manufacturer’s protocols. Uniquely indexed libraries were pooled in equimolar ratios and sequenced on an Illumina NextSeq500 with single-end 75bp reads by the Dana-Farber Cancer Institute Molecular Biology Core Facilities.  

# RNA-seq and Gene Set Enrichment Analysis  

Sequenced reads were aligned to the UCSC mm9 reference genome assembly and gene counts were quantified using STAR (v2.5.1b) (Dobin et al., 2013). Differential gene expression testing was performed by DESeq2 (v1.10.1) (Love et al., 2014), and resulting fold change values were used as a pre-ranked gene list for gene set enrichment analysis (Mootha et al., 2003; Subramanian et al., 2005). ‘Classic’ mode was used for enrichment statistic calculation. For unbiased discovery of enriched gene sets, the HALLMARK gene sets were used as queries (Liberzon et al., 2015). In a separate query, muscle gene expression data were examined using a custom gene set: $"P G C1a$ muscle gene set.’ This gene set contains all genes upregulated $>2$ -fold in the gastrocnemius muscle of MCK-PGC1 $\pmb{\alpha}$ transgenic mice (Lin et al., 2002) (Table S2).  

# Tissue Harvest, Protein Digest, and Peptide Isobaric Labeling  

Kidney tissues were lysed with a mechanical homogenizer using lysis buffer composed of $50\,\mathsf{m M}$ HEPES pH 8.5, 250 mM NaCl, and EDTA-free protease inhibitor cocktail (Roche), 5 mM TCEP in $2\%$ SDS. Kidney lysates were centrifuged at $10{,}000{\,\times\,}g$ for 10 min. Protein content was measured using a BCA assay (Thermo Scientific); disulfide bonds were reduced and cysteine residues alkylated with iodoacetamide $.14\,\mathsf{m M})$ essentially as previously described (Huttlin et al., 2010). Protein lysates were purified by methanol-chloroform precipitation and $200~\upmu\mathrm{g}$ was digested overnight with LysC (Wako, Japan) in a 1/200 enzyme/protein ratio in $^{2\textsf{M}}$ urea and 25 mM HEPES, $\mathsf{p H}\,8.5$ . Digests were acidified with $10\%$ acetic acid (AA) to a pH of ${\sim}2$ and subjected to C18 solid-phase extraction (50 mg SPE) (Sep-Pak, Waters).  

Isobaric labeling of the peptides was performed using 10-plex tandem mass tag (TMT) reagents (Thermo Fisher Scientific). Reagents, $5.0\;\mathrm{mg}$ , were dissolved in $252~\upmu\mathrm{l}$ acetonitrile (ACN) and 1/10 of the solution were added to $100~{\upmu\gimel}$ of peptides dissolved in $100\,\upmu|$ of $200\,\mathsf{m M}$ HEPES, $\mathsf{p H}\,8.5$ . After 1 hour (RT), the reaction was quenched by adding $3\,\upmu|$ of $5\%$ hydroxylamine. Labeled peptides were combined and acidified prior to C18 SPE on Sep-Pak cartridges $(50\;\mathsf{m g})$ .  

# Basic pH Reversed-Phase Separation (BPRP)  

TMT labeled peptides were solubilized in $500\,\upmu\mathrm{L}$ solution containing $\mathsf{\bar{3}\%A C N/10\ m M}$ ammonium bicarbonate, $\mathsf{p H}\,8.0$ and $300\,\upmu\mathfrak{g}$ of TMT labeled peptides was separated by an Agilent 300 Extend C18 column $(3.5\,\upmu\mathsf{m}$ particles, $4.6\:\mathsf{m m}$ ID and $250\;\mathrm{mm}$ in length). An Agilent 1260 binary pump coupled with a photodiode array (PDA) detector (Thermo Scientific) was used to separate the peptides. A 45 min linear gradient from $10\%$ to $40\%$ acetonitrile in $10\,\mathsf{m M}$ ammonium bicarbonate $\mathsf{p H}\,8$ (flow rate of $0.6\,\mathsf{m L/m i n}_{\phantom{0}}$ ) separated the peptide mixtures into a total of 96 fractions (36 s). A total of 96 fractions were consolidated into 24 samples in a checkerboard fashion, acidified with $20\,\upmu\mathrm{l}$ of $10\%$ formic acid and vacuum dried. Each sample was desalted via StageTips and re-dissolved in $12\,\upmu|\,5\%$ FA/ $5\%$ ACN, prior to LC-MS/MS analysis.  

# Liquid Chromatography Separation and Tandem Mass Spectrometry (LC-MS/MS)  

Data were collected using an Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) coupled with a Proxeon EASYnLC 1200 LC pump (Thermo Fisher Scientific). Peptides were separated on a $75~{\upmu\mathsf{m}}$ inner diameter microcapillary column packed with $35\;\mathsf{c m}$ of GP-18 resin $(2.6\ \upmu\mathsf{m}$ ,$200\mathrm{~\AA}.$ , Sepax). For each sample ${\sim}1.5~\upmu\g$ of peptides were separated using a 3 hr gradient of $6\%{-}27\%$ acetonitrile in $0.125\%$ formic acid with a flow rate of 400 nL/min.  

Each analysis used an $\mathsf{M}\mathsf{S}^{3}$ -based TMT method as described previously (McAlister et al., 2014; Ting et al., 2011). The data were acquired using a mass range of $m/z\,400-1400$ , resolution 120,000, AGC target $5\times10^{5}$ , maximum injection time $100\;\mathrm{ms}$ , dynamic exclusion of 120 s for the peptide measurements in the Orbitrap. Data dependent $\mathsf{M}\mathsf{S}^{2}$ spectra were acquired in the ion trap with a normalized collision energy (NCE) set at $35\%$ , AGC target set to $1.8\times10^{4}$ and a maximum injection time of 120 ms. $\mathsf{M S}^{3}$ scans were acquired in the Orbitrap with a HCD collision energy set to $55\%$ , AGC target set to $1.5\times10^{5}$ , maximum injection time of $150~\mathrm{ms}$ ,resolution at 50,000 and with a maximum synchronous precursor selection (SPS) precursors set to 10.  

# Peptide Quantification and Gene Set Enrichment Analysis  

A compendium of in-house developed software was used to convert mass spectrometric data (Raw file) to the mzXML format, as well as to correct monoisotopic $\mathsf{m}/\mathsf{z}$ measurements (Elias and Gygi, 2007). All experiments used the Mouse UniProt database (downloaded April 2018) where reversed protein sequences and known contaminants such as human keratins were appended. SEQUEST searches were performed using a 20 ppm precursor ion tolerance, while requiring peptide amino/carboxy (N/C) terminus to have trypsin protease specificity and allowing up to two missed cleavages (Eng et al., 1994). Ten-plex TMT tags on peptide N termini and lysine residues $(+\,229.162932\,\mathsf{D a})$ and carbamidomethylation of cysteine residues $(+57.02146\textsf{D a})$ were set as static modifications while methionine oxidation $\left(+\ 15.99492\ D\rightleftharpoons$ a) was set as variable modification. A $\mathsf{M S}^{2}$ spectra assignment false discovery rate (FDR) of less than $1\,\%$ was achieved by applying the target-decoy database search strategy (Elias and Gygi, 2007). Filtering was performed using an in-house linear discrimination analysis(LDA) method to create one combined filter parameter from the following peptide ion and $\mathsf{M}\mathsf{S}^{2}$ spectra metrics: SEQUEST parameters XCorr and $\Delta\mathsf{C n}$ , peptide ion mass accuracy and charge state, in-solution charge of peptide, peptide length and mis-cleavages. Linear discrimination scores were used to assign probabilities to each $\mathsf{M}\mathsf{S}^{2}$ spectrum for being assigned correctly and these probabilities were further used to filter the dataset with an $\mathsf{M}\mathsf{S}^{2}$ spectra assignment FDR of smaller than a $\mid\%$ at the protein level (Huttlin et al., 2010).  

For quantification, a $0.003\,m/z$ window centered on the theoretical $\mathsf{m}/\mathsf{z}$ value of each the six reporter ions and the intensity of the signal closest to the theoretical $\mathsf{m}/\mathsf{z}$ value was recorded. Reporter ion intensities were further de-normalized based on their ion accumulation time for each $\mathsf{M}\mathsf{S}^{2}$ or $\mathsf{M S}^{3}$ spectrum and adjusted based on the overlap of isotopic envelopes of all reporter ions (as determined by the manufacturer). The total signal intensity across all peptides quantified was summed for each TMT channel, and all intensity values were adjusted to account for potentially uneven TMT labeling and/or sample handling variance.  

Fold change values were used to generate a pre-ranked gene list for gene set enrichment analysis (Liberzon et al., 2015; Mootha et al., 2003; Subramanian et al., 2005). ‘Classic’ mode was used for enrichment statistic calculation. To query this gene list, a custom gene set was generated that corresponded to genes upregulated in the context of ischemic kidney injury: ‘AKI-correlated’ genes. Existing RNA-seq data examining kidney gene expression $24\,\mathrm{hr}$ after bilateral ischemic injury (as compared to sham surgery) was used to create this gene set (Liu et al., 2017). AKI-correlated genes were defined as those that were upregulated $>4$ -fold in the context of injury (Table S2).  

# QUANTIFICATION AND STATISTICAL ANALYSIS  

# Experimental Replicates and Quantification  

Replicate numbers are described in the figure legends. For cellular assays, n corresponds to the number of experimental replicates (e.g., independent transfections). For animal assays or tissue extracted from animals, n corresponds to the number of mice used per genotype or condition. All data are presented as mean and error bars represent standard error.  

# Statistical Analysis  

The two-tailed Student’s t test was used for single comparisons. Familywise-error rate (FWER) p values are displayed for gene set enrichment analysis to correct for multiple hypothesis testing.  

# DATA AND SOFTWARE AVAILABILITY  

The accession number for the RNA-seq data reported in this paper is GSE129687. Cloned Atlantic bluefin tuna PPARGC1A $5^{\prime}$ UTR sequence is presented in Figure 3C.  