# Review uORF Shuffling Fine-Tunes Gene Expression at a Deep Level of the Process  

Yukio Kurihara  

Synthetic Genomics Research Group, RIKEN Center for Sustainable Resource Science, 1-7-22 Suehirocho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan; yukio.kurihara@riken.jp  

Received: 8 April 2020; Accepted: 8 May 2020; Published: 11 May 2020 Abstract: Upstream open reading frames (uORFs) are present in the $5^{\prime}$ leader sequences (or $5^{\prime}$ untranslated regions) upstream of the protein-coding main ORFs (mORFs) in eukaryotic polycistronic mRNA. It is well known that a uORF negatively affects translation of the mORF. Emerging ribosome profiling approaches have revealed that uORFs themselves, as well as downstream mORFs, can be translated. However, it has also been revealed that plants can fine-tune gene expression by modulating uORF-mediated regulation in some situations. This article reviews several proposed mechanisms that enable genes to escape from uORF-mediated negative regulation and gives insight into the application of uORF-mediated regulation for precisely controlling gene expression.  

Keywords: uORF; translation; ribosome; NMD  

# 1. Introduction  

In the $5^{\prime}$ untranslated regions (UTRs) of mRNAs, there are a variety of cis-regulatory sequence elements that affect the subsequent fate (e.g., transportation, stability and translation efficiency), of the mRNAs [1]. One of the most characterized elements is an upstream open reading frame (uORF) that is distinct from the protein-coding main ORF (mORF) on the same polycistronic mRNA [2]. It is estimated that up to half of genes possess one or more uORFs in Arabidopsis thaliana [2]. Emerging ribosome profiling approaches have revealed that uORFs themselves can possibly be translated into short peptides [3–9] and that translation of some uORFs starts with non-AUG initiation codons [10,11]. Peptide lengths encoded by uORFs are variable but are typically between 1–100 amino acids [12]. Taking into consideration the actual translation of uORFs, it is better to rephrase $\prime\prime{}5^{\prime}\mathrm{UTR}^{\prime\prime}$ as $'5^{\prime}$ leader sequence” in this review.  

It is well known that uORFs negatively affect expression of downstream protein-coding mORFs [2,13,14]. In this context, uORFs act as repressors of translation of mORFs (Figure 1). The general process of eukaryotic translation is well described in previous review articles [6,15]. uORF translation precedes mORF translation, because ribosome scanning starts from the cap structure at the $5^{\prime}$ -distal end. It is proposed that uORF-mediated translational repression of mORFs occurs by releasing ribosomes after uORF translation or by stalling ribosomes in the uORF and then preventing them from accessing the mORF [2]. Some features, such as an increasing number of uORFs and a shorter distance between the uORF and mORF, may enhance efficiency of uORF-mediated repression of mORF translation [16]. A recent paper revealed that even minimum uORFs comprising AUG-STOP inhibit translation of mORFs by stalling ribosomes on the uORFs in Arabidopsis [17]. It is known that conservation in the sequence of uORF-encoded peptides among plant species is an important factor for the efficiency of uORF-mediated translational repression, because mutations reducing the conservation diminished the repressive efficiency of the uORFs [18,19]. Furthermore, some physiological roles of plant-conserved uORFs have been reported [20].  

In addition, uORFs can also act as triggers of nonsense-mediated mRNA decay (NMD), which is a seleIcnt iavded imtioRnN, Au OdRecFas yc amn eaclshoa naicst mas  (tFriiggguerres  1o)f  [n2o1,n2s2e]n. sIen- mtheidsi actaesde , mit RhNasA  bdeeecna ys p(eNcMulDat),e dw thhicaht  tish ea ssteolepc tciovde omn RofN tAh e duecOaRy F maencdh tahnei slomn (gF ipgruorvei s1i)o [n2a1l ,32’2U]. TIRn t—hiws hciacshe ,s tiat rhtsa sd obewenns tsrpeeacmul aotf etdh et huatO tRhFe —staorpe rceocdoognn iozfe dt haes  uNOMRDF  taanrgd ettsh e[ 2l3o] nagn dp rtohvei smioRnNal A3 i’sU tThRus— swelheicctihv estlay rtdse gdroawdendst. reTraamn socfr itphtse  wuiOthR lFo—ngareer rueOcoRgFnsi zaered  liaks elNy MtoD  btea rmgeotrse  [e23ffi] caienndt ltyh ee limmRinNaAte ids  tbhy uNs sMelDe citni vpellay ndtse [g2r4a]d. eUd.c hTiryaansmcari-pKtas dwoiktuh rlao entg aelr. suuOgRgFess t atrhe alti kineldyu tcot iboen  omf orNe MefDfi cbiye ntthlye  euliOmRiFn aotfe tdh eb yA dNoMMeDt iDnC p1l amntRs N[2A4 ]i.s  Uascshoicyiaatmead- Kwaidtho kriubroa seot male. sstuagllgiensgt  taht atth ien duuOcRtiFo ns toofp  NcoMdDo nb iy nt hAer aubiOdRopF siosf.  thAe  siAmdoilMare trDelCa1t iomnRshNiAp  ihsa sa sbsoeceina treedp owrittehd  riinb oyseoamste, sstuapllipnogr tiant gt hthe eu pOoRssFi bsitloitpy  ctohdato nr iibno sAormabei dsotpaslilsi.n gA  asti am iluaOr RreFl actoiounlsdh ibpe  hoanse  boef etnh er edpeotretremd iinna nytesa sotf, sNuMppDo r[t2i5n,g2 6t]h. eI np oostshiebri lictays tehs,a rt irbiobsoosomme es tsatlallilning g mait gah tu iOnRdFu cceo uoltdh ebr e RoNneA  odf etchaey  dliekteer nmoi-ngaon tds eocfa yN [M27D]. [T2h5,u2s6, ].u IOn RoFtsh erre cparseesss,  griebnoes oemxpe rsetsaslilionng  nmoit gohnt liyn daus cter aonthsleart iRonNaAl  rdeepcraeys sliokres  nbou-tg oa lsdoe caasy  t[r2i7g]g. eTrhs uosf, RuONRAF sd erceapyr.e  

![](images/d81ca07d0458dc8d19be2afea320f246a5443d3769bc849e2e2741fd38a3e7b7.jpg)  
FFiigguurree  11..  OOvveerrvviieeww  ooff  uuppssttrreeaamm  ooppeenn  rreeaaddiinngg  ffrraammee  ((uuOORRFF))--mmeeddiiaatteedd  rreepprreessssiioonn  ooff  ggeennee  eexxpprreessssiioonn.. ((AA))  TThhee  uuOORRFF  mmRRNNAA  iiss  ttrraannssllaatteedd  iinnttoo  pprrootteeiinn  bbyy  rriibboossoommeess..  ((BB))  OOnn  tthhee  ootthheerr  hhaanndd,,  uuOORRFF--mmeeddiiaatteedd rreepprreessssiioonn  ooff  tthhee  mmaaiinn  OORRFF  ((mmOORRFF))  ooccccuurrss  iinn  ppoollyycciissttrroonniicc $\scriptstyle\mathrm{{uORF+}}$ mmRRNNAA..  TTrraannssllaattiioonn  rreepprreessssiioonn  ooff an mORF occurs by ribosome releasing after uORF translation or ribosome stalling in the uORF. Some aarree  eelliimmiinnaatteedd  bbyy  RRNNAA  ddeeccaayy  ppaatthhwwaayyss  lliikkee  nnoonnsseennssee--mmeeddiiaatteedd  mmRRNNAA  ddeeccaayy  ((NNMMDD))..  

# 2. Mechanisms for Evading uORF-Mediated Regulation  

Some mechanisms for evading uORF-mediated negative regulation of gene expression have been proposed. This review explains not only well-known ones (leaky scanning and reinitiation) but also more recent ones, including those that have been experimentally proven and those that have yet to be. This includes alternative transcription start sites (TSSs) selection, splicing out of uORF initiation codons and initiation of cap-independent translation. In particular, alternative TSS and splicing can produce two mRNA variants, $\scriptstyle\mathrm{uORF+}$ and uORF–. Some plant stimuli result in shuffling the amount of variants by inducing a TSS positional shift or alternative splicing.  

# 2.1. Leaky Scanning and Reinitiation  

Viruses often use leaky scanning by ribosomes and translational reinitiation to produce more than one kind of protein from their polycistronic RNAs [28]. Eukaryotes, including plants, also use leaky scanning and reinitiation to evade uORF-mediated translational repression and then translate mORFs.  

Tdheepilre timoenc hbaenciasumsse  hleaavke yb escean nannianlgy zmeda yo foteccn uar.n dT rheuvsi, eewnevdi rboyn omtehnetrasl  [2c0h,a2n9g],e ss o stohmeye tairme eisn trreoidnfuocrecde tornalnys lbartiieoflnya lh ienreit.  

T4 ramnRslNatAio inn  gyeenaesrt ahllayv se taalrstso  bbye erne crwueiltli nexg e4m0Sp lrifiibeods o[1m5,e3 s2]u. bTuankiitns gt ot htihse $5^{\prime}$ dc aopt hsetrr uecvtiudreens coef  itnhteo amccRoNuAnts .[ 2I],n  tcraannsolnaitcioaln aslc arneinniintiga,t iuoOn RoFr  ilse afikrys ts creacnonginnigz emda yp rbioe rt thoe  tmheo stm cOoRmF.m Hono wmeevcehr,a innis lme afkoyr secvaandniinngg ,u tOheR F4-0Sm secdaiantneidn gt rrainbsolsaotimoanla ls urebpurneist siino n4 8oSf  pmreOinRitFisa.t ion complexes passes over the initiation codonT hoef rteh aer eu tOwRoF  tayvpoeisd oifn gu OuROFR: Fo tnrea inss laant iionnd e(Fpiegnudree n2t Au)O. RWFh tehna tt hdeo e4s0 Sn ostc aonvneirlnagp s uwibtuh ntiht er emacOhResF tahned i tnhitei aottihoenr  cios daon no ovfe trlhaep pmiOngR Fu, OthReF  6(0FSi gruibreo s2oAm). alL esaukbyu sncita insn riencgr ubiutte dn,o ft orreimniintiga taino n8 0cSa-nt roavnesrlcatoimnge roivbeorsloapmpei nwgi tuh Oa R4F0-S mseudbiuatneitd.  rOenpcree stshiios nh. appens, translation of the mORF starts.  

![](images/eb557fec575351fe3362b652aa236ec4b0cc91db911b4542266c83496bff2aa3.jpg)  
FFiigguurree  22..  SScchheemmaattiicc  rreepprreesseennttaattiioonn  ooff  lleeaakkyy  ssccaannnniinngg  ((AA))  aanndd  ttrraannssllaattiioonnaall  rreeiinniittiiaattiioonn  ((BB))..  TThheerree  aarree ttwwoo  ttyyppeess  ooff  uuOORRFFss..  OOnnee  iiss  aann  iinnddeeppeennddeenntt  uuOORRFF  tthhaatt  ddooeess  nnoott  oovveerrllaapp  wwiitthh  aann  mmOORRFF  aanndd  tthhee  ootthheerr iiss  aa  uuOORRFF  tthhaatt  ddooeess  oovveerrllaapp..  TTwwoo  mmeecchhaanniissmmss,,  lleeaakkyy  ssccaannnniinngg  aanndd  ttrraannssllaattiioonnaall  rreeiinniittiiaattiioonn,,  ccaann oovveerrccoommee  tthhee  uuOORRFF--mmeeddiiaatteedd  rreepprreessssiioonn  ooff  aann  mmOORRFF..  

SIenl eactniootnh oefr  Tcraasne,s carfitpetri otnr aSntsalratt iSoitne so f a uORF, the translating ribosome is disassembled and the 60S subunit is released (Figure 1B). In translational reinitiation, the remaining 40S subunit on the mRNA still continues scanning the downstream initiation codon and an 80S-translating ribosome is formed again at the initiation codon of the mORF (Figure 2B).  

Some events of leaky scanning or translational reinitiation have been reported in Arabidopsis [20]. For example, ribosomes are stalled at the stop codon of the second uORF of bZIP11 mRNAs in a peptide sequence-dependent manner under high sucrose conditions, resulting in low translation of a bZIP-encoding mORF [30,31]. This translational repression of an mORF is alleviated by sucrose depletion because leaky scanning may occur. Thus, environmental changes sometimes reinforce t ranslational initiation of mORFs. Leaky scanning of uORFs and reinitiation of the mORF on the GCN4 mRNA in yeast have also been well exemplified [15,32]. Taking this and other evidence into account [2], translational reinitiation or leaky scanning may be the most common mechanism for evading uORF-mediated translational repression of mORFs.  

There are two types of uORF: one is an independent uORF that does not overlap with the mORF and the other is an overlapping uORF (Figure 2A). Leaky scanning but not reinitiation can overcome overlapping uORF-mediated repression.  

# 2.2. Selection of Transcription Start Sites  

Approximately $75\%$ of genes are estimated to use multiple TSSs in Arabidopsis [33,34]. The TSS is one of the most striking determinants of the $5^{\prime}$ leader sequence. It is conceivable that the leader sequences of transcripts derived from more upstream TSSs possess more information than those from downstream TSSs. In this context, a few papers have reported that the selection of alternative TSSs affects the translation of genes [33,35].  

PTlraantns s20c2r0i, p9t, ix oFnO Rf rPoEEmR  RTESVSIEs Wu pstream of the uORF (uTSS) produces uORF $^+$ mRNAs, w4 hoifl 9e  that from downstream TSSs (dTSSs) generates uORF– mRNAs [33,36]. It is assumed that environmental differences may reinforce the occurrence of alternative TSS selection, which determines whether a gene dominantly produces $\scriptstyle\mathrm{{uORF+}}$ or uORF– mRNAs. In fact, our previous report showed that the main TSS position in more than two hundred light-responsive genes was shifted from upstream of the uORF to doAwranbsitdroepasims  soefe tdhlien tgrs agnrsolawtni oinn  idnaitrika twioerne  ceoxdpoons eodf  ttoh eb luueO lRigFh (t F[i3g3u]. reR i3bAos),o mweh eprno fyiloiungn ga nAarlaybsiisd opsis seedlidnegmso gnrstorawten di nt hdata trhke swe eTreS Se sxhpifotss ewd etroe  lbilkueel yl itgo hut p[r3e3g].u laRtieb torsaonsmlaet iporno efiflfiicnige nacny aolfy smisO dReFsm uopnostnr ated that tbhleusee  liTgShSt  sehxipftoss uwree,r ie nlidkicealtyi ntgo  tuhpart ebgluulea ltieg thrt aanllselvaitiatoend  euffiOcRieF-ncmye doifa tmedO tRraFns sulaptioonn abll uree plriegshsti oenx ipno sure, indicgateinnegs  tbhya tp obsliutieo lniagllhyt  salhlieftviinagt ethde  umOaRinF -TmSSe, drieastueldti tnrga inns ltahtei opnraold ruecptiroens soifo nm ionr eg eunOeRs Fb– y mpRoNsitAiso. nally shiftiFnugr tthheer mmoarien,  iTn SsSo, mree souf ltthien gg einne st hshe opwriondg uac tTiSoSn s hoiff t,m aocrceu muuOlaRtiFo–n  mof RuNORAFs.+  FmuRrtNhAesr wmaosr ed,e itenc tseod me of the genes showing a TSS shift, accumulation of $\scriptstyle\mathrm{{uORF+}}$ mRNAs was detected in upf1 NMD-deficient mutants compared to wild-type plants, indicating they are targeted by NMD [33]. Taken together, these TSS shifts from uTSS to dTSS not only avoid uORF-mediated translational repression of the mORFs but also NMD in some genes.  

![](images/c988d7aac2dafc7df0c0c5dac88a8384930b42c7fc598c2fc5ae7c66eb2b07d7.jpg)  
FigurFei g3u. rSe c3h. eScmhaetimca triec prreepsreensetanttiaotino no fo ft hthrreeee  nneewwllyy  pprrooppoosseed d mmecehcahnaisnimss mfos rf eovr aedvinagd iunOg RuF-OmReFd-iamteedd iated repression of gene expression. (A) Alternative transcription start site (TSS) selection. (B) Alternative splicing out of the uORF. (C) Cap-independent translational initiation mediated by an internal ribosomal entry site (IRES). The environment around plants may affect the occurrence of these mechanisms.  

Alternative TSS selection as described above can also overcome translational repression by uORFs overlapping with mORF initiation codons.  

# 2.3. Adletteerrnmatiinvae nStsl icoif ntgh eE x5c' lluedaidnegr  osre qMueondicfey. iInngd ueeOdR, iFts  

In adSdpiltiicoinn gt oo aultt eorf nianttirvoen sT ScSosn,t aailtneinrng atthive ei sniptliiactiionng  icso adlosno  oofn et hoef  tuhOe RmF oosrt  ia mppaortr toafn tt hdee tuerOmRiF nants of thes $5^{\prime}$ uleenacde efrr osemq tuhee n5c' el.e aIdnedr eseedq,u ietn chea so fb pereen- mreRpNoArtse pdr tohdautc easl temrRnNatAivs ew sitphliocuitn ug OtRhFats  oorc cmurRsN iAn st he $5^{\prime}$ leaderw isteh qturuennccaet eadls uo OaRffFesc,t sr etshpee cttriavnelsyl a(tFiiognu ree ffi3cBi)e, nwcyh iloef  ignetrnoens  rient epnltaionnt s p[r3o5d].u  

Splicing out of introns containing the initiation codon of the uORF or a part of the uORF sequence from the $5^{\prime}$ leader sequence of pre-mRNAs produces mRNAs without uORFs or mRNAs with truncated uORFs, respectively (Figure 3B), while intron retention produces mRNAs with full uORFs. Furthermore, the splicing out of an intron located between the uORF and the mORF reduces the distance between the two. As with alternative TSS selection, such splicing has the potential to avoid NMD and so it is possible that the gene evades or alleviates uORF-mediated repression by splicing out uORF sequences. In contrast, alternative splicing may produce a de novo initiation codon of a uORF at the splicing junction.  

However, at present, there have been a limited number of examples showing a possible link between splicing and uORFs [2,37–40]. For example, Pasentsis et al. showed that the PHY2 gene in Ceratodon purpureus produces two splicing variants harboring different uORF sequences in a light-dependent manner [39]. Combier et al. suggest that, in Medicago truncatula, alternative splicing of an intron containing uORFs in the HAP2-1 mRNA regulates root nodule development [40].  

Alternative splicing events frequently occur in response to environmental changes in plants [41]. It is easy to speculate that splicing of uORF-containing introns might be responsible for the plant’s physiological response through fine-tuning the translation of the gene. Therefore, some issues, such as how many genes undergo alternative splicing of uORFs, which stimuli induce them and whether splicing out of the uORF actually impacts translation or expression of genes, remain to be elucidated by future research.  

# 2.4. Cap-Independent Translation Initiation  

Cap-independent translation is commonly driven by internal ribosomal entry sites (IRESs) that form dynamic RNA secondary structures. IRESs are widely used by viruses for producing proteins from their polycistronic mRNAs [42]. In plant systems, a viral IRES has been used for expressing two genes from a polycistronic mRNA [43,44].  

A variety of endogenous IRESs or IRES-like structures have been discovered not only in viruses but also in eukaryotes including plants [45–48]. If an IRES is located between the initiation codon of a uORF and an mORF, internal entry of ribosomes should overcome uORF-mediated translational repression as proposed in Figure 3C.  

Although there have not been any examples illustrating such avoidance of translational repression (Figure 3C) at least in plants, some work has reported that environmental changes enhance the occurrence of IRES-dependent translation in plants [47,48]. For example, the IRES-mediated translation of the WUSCHEL mRNA in Arabidopsis is enhanced by environmental hazard stress [47]. In addition, recent reports have demonstrated that $N^{6}$ -methyladenosine $(\mathrm{m}^{6}\mathrm{A})$ methylation of $5^{\prime}$ leader sequences promotes cap-independent translation by IRESs in mammals [49,50]. Thus, it is speculated that environmental changes could induce hypermethylation of adenosine in $5^{\prime}$ leader sequences and then promote IRES activity.  

Regarding $\mathrm{m}^{6}\mathrm{A}$ modification, it has also been reported that amino acid starvation induces demethylation at the $\mathrm{m}^{6}\mathrm{A}$ site within overlapping uORF on the mammalian ATF4 mRNA and increases mORF translation [51]. However, in this case, translation of the $A T F4\;\mathrm{mORF}$ may be cap-dependent but not cap-independent.  

# 3. Application of uORF-Mediated Regulation  

Regulatory mechanisms of gene expression have often been modified and/or applied to expressing preferable genes under particular conditions. Application of transcriptional regulation using tissue-specific or stimulus-inducible promoters is the most popular. Recent papers have reported that translational fine-tuning by artificial modification of uORFs enables proper transgene expression to be achieved [14,52]. In particular, Zhang et al. showed that CRISPR/Cas9 editing of the uORF sequence of the GGP2 gene encoding a key enzyme for vitamin C biosynthesis results in an increased amount of vitamin C through enhanced mORF translation in lettuce [14].  

This review proposes that there are various ways this regulation can be applied. A dynamic range of spatial and temporal gene expression is controlled at the transcriptional level. In the case where an appropriate level of gene expression is required, fine-tuning by translational regulatory elements such as uORFs can be applied to elicit the required function of the gene. Adjustment of gene expression may also be achieved by combining the use of both uORFs and IRESs. In particular, the uORF is an attractive candidate for controlling gene expression at the translational level.  

# 4. Future Perspectives  

This review introduces several mechanisms that can overcome uORF-mediated translational repression of mORFs in polycistronic mRNAs. In particular, uORF shuffling by alternative TSS selection or alternative splicing may be a notable concept. It is expected that a growing body of future research will elucidate the unknown parts of the mechanisms. For this purpose, integrated analysis of multiple omics data derived from not only the transcriptome but also ribosome profiling and proteome will be required. As effectiveness of uORFs on mORF translation varies case by case and the action of IRESs is still obscure in plant biological processes, deeper understanding of the regulatory mechanisms of gene expression is necessary for their application in crops.  

Funding: This work was supported by a Grant-in-Aid for Scientific Research (C) (JP18K06304) from the Japanese Society for the Promotion of Science (JSPS) to Y.K.  

Conflicts of Interest: The author declares no conflict of interest.  

# References  

1. Vaughn, J.N.; Ellingson, S.R.; Mignone, F.; Arnim, A. Known and novel post-transcriptional regulatory sequences are conserved across plant families. RNA 2012, 18, 368–384. [CrossRef]   
2. von Arnim, A.G.; Jia, Q.; Vaughn, J.N. Regulation of plant translation by upstream open reading frames. Plant Sci. 2014, 214, 1–12. [CrossRef]   
3. Bazin, J.; Baerenfaller, K.; Gosai, S.J.; Gregory, B.D.; Crespi, M.; Bailey-Serres, J. Global analysis of ribosome-associated noncoding RNAs unveils new modes of translational regulation. Proc. Natl. Acad. Sci. USA 2017, 114, 10018–10027. [CrossRef] [PubMed]   
4. Juntawong, P.; Girke, T.; Bazin, J.; Bailey-Serres, J. Translational dynamics revealed by genome-wide profiling of ribosome footprints in Arabidopsis. Proc. Natl. Acad. Sci. USA 2014, 111, 203–212. [CrossRef] [PubMed]   
5. Merchante, C.; Brumos, J.; Yun, J.; Hu, Q.; Spencer, K.R.; Enríquez, P.; Binder, B.M.; Heber, S.; Stepanova, A.N.; Alonso, J.M. Gene-specific translation regulation mediated by the hormone-signaling molecule EIN2. Cell 2015, 163, 684–697. [CrossRef] [PubMed]   
6. Merchante, C.; Stepanova, A.N.; Alonso, J.M. Translation regulation in plants: An interesting past, an exciting present and a promising future. Plant J. 2017, 90, 628–653. [CrossRef] [PubMed]   
7. Fujita, T.; Kurihara, Y.; Iwasaki, S. The plant translatome surveyed by ribosome profiling. Plant Cell Physiol. 2019, 60, 1917–1926. [CrossRef] [PubMed]   
8. Ingolia, N.T.; Brar, G.A.; Stern-Ginossar, N.; Harris, M.S.; Talhouarne, G.J.; Jackson, S.E.; Wills, M.R.; Weissman, J.S. Ribosome profiling reveals pervasive translation outside of annotated protein-coding genes. Cell Rep. 2014, 8, 1365–1379. [CrossRef] [PubMed]   
9. Kurihara, Y.; Makita, Y.; Shimohira, H.; Fujita, T.; Iwasaki, S.; Matsui, M. Translational landscape of protein-coding and non-protein-coding RNAs upon light exposure in Arabidopsis. Plant Cell Physiol. 2020, 61, 536–545. [CrossRef]   
10. Hsu, P.Y.; Calviello, L.; Wu, H.L.; Li, F.W.; Rothfels, C.J.; Ohler, U.; Benfey, P.N. Super-resolution ribosome profiling reveals unannotated translation events in Arabidopsis. Proc. Natl. Acad. Sci. USA 2016. [CrossRef]   
11. van der Horst, S.; Snel, B.; Hanson, J.; Smeekens, S. Novel pipeline identifies new upstream ORFs and non-AUG initiating main ORFs with conserved amino acid sequences in the $5^{\prime}$ leader of mRNAs in Arabidopsis thaliana. RNA 2019, 25, 292–304. [CrossRef] [PubMed]   
12. Hsu, P.Y.; Benfey, P.N. Small but Mighty: Functional peptides encoded by small ORFs in plants. Proteomics 2018, 18, 1700038. [CrossRef] [PubMed]   
13. Calvo, S.E.; Pagliarini, D.J.; Mootha, V.K. Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. USA 2009, 106, 7507–7512. [CrossRef] [PubMed]   
14. Zhang, H.; Si, X.; Ji, X.; Fan, R.; Liu, J.; Chen, K.; Wang, D.; Gao, C. Genome editing of upstream open reading frames enables translational control in plants. Nat. Biotechnol. 2018, 36, 894–898. [CrossRef]   
15. Hinnebusch, A.G. The scanning mechanism of eukaryotic translation initiation. Annu. Rev. Biochem. 2014, 83, 779–812. [CrossRef]   
16. Chew, G.L.; Pauli, A.; Schier, A.F. Conservation of uORF repressiveness and sequence features in mouse, human and zebrafish. Nat. Commun. 2016, 7, 11663. [CrossRef]   
17. Tanaka, M.; Sotta, N.; Yamazumi, Y.; Yamashita, Y.; Miwa, K.; Murota, K.; Chiba, Y.; Hirai, M.Y.; Akiyama, T.; Onouchi, H.; et al. The minimum open reading frame, AUG-Stop, induces boron-dependent ribosome stalling and mRNA degradation. Plant Cell 2016, 28, 2830–2849. [CrossRef]   
18. Ebina, I.; Takemoto-Tsutsumi, M.; Watanabe, S.; Koyama, H.; Endo, Y.; Kimata, K.; Igarashi, T.; Murakami, K.; Kudo, R.; Ohsumi, A.; et al. Identification of novel Arabidopsis thaliana upstream open reading frames that control expression of the main coding sequences in a peptide sequence-dependent manner. Nucleic Acids Res. 2015, 43, 1562–1576. [CrossRef]   
19. Hayashi, N.; Sasaki, S.; Takahashi, H.; Yamashita, Y.; Naito, S.; Onouchi, H. Identification of Arabidopsis thaliana upstream open reading frames encoding peptide sequences that cause ribosomal arrest. Nucleic Acids Res. 2017, 45, 8844–8858. [CrossRef]   
20. van der Horst, S.; Filipovska, T.; Hanson, J.; Smeekens, S. Metabolite control of translation by conserved peptide uORFs: The ribosome as a metabolite multisensor. Plant Physiol. 2020, 182, 110–122. [CrossRef]   
21. Ohtani, M.; Wachter, A. NMD-based gene regulation-A strategy for fitness enhancement in plants? Plant Cell Physiol. 2019, 60, 1953–1960. [CrossRef] [PubMed]   
22. Kurihara, Y.; Matsui, A.; Hanada, K.; Kawashima, M.; Ishida, J.; Morosawa, T.; Tanaka, M.; Kaminuma, E.; Mochizuki, Y.; Matsushima, A.; et al. Genome-wide suppression of aberrant mRNA-like noncoding RNAs by NMD in Arabidopsis. Proc. Natl. Acad. Sci. USA 2009, 106, 2453–2458. [CrossRef] [PubMed]   
23. Hori, K.; Watanabe, Y. Context analysis of termination codons in mRNA that are recognized by plant NMD. Plant Cell Physiol. 2007, 48, 1072–1078. [CrossRef] [PubMed]   
24. Nyikó, T.; Sonkoly, B.; Mérai, Z.; Benkovics, A.H.; Silhavy, D. Plant upstream ORFs can trigger nonsense-mediated mRNA decay in a size-dependent manner. Plant Mol. Biol. 2009, 71, 367–378. [CrossRef]   
25. Uchiyama-Kadokura, N.; Murakami, K.; Takemoto, M.; Koyanagi, N.; Murota, K.; Naito, S.; Onouchi, H. Polyamine-responsive ribosomal arrest at the stop codon of an upstream open reading frame of the AdoMetDC1 gene triggers nonsense-mediated mRNA decay in Arabidopsis thaliana. Plant Cell Physiol. 2014, 55, 1556–1567. [CrossRef]   
26. Gaba, A.; Jacobson, A.; Sachs, M.S. Ribosome occupancy of the yeast CPA1 upstream open reading frame termination codon modulates nonsense-mediated mRNA decay. Mol. Cell 2005, 20, 449–460. [CrossRef]   
27. Doma, M.K.; Parker, R. Endonucleolytic cleavage of eukaryotic mRNAs with stalls in translation elongation. Nature 2006, 440, 561–564. [CrossRef]   
28. Pooggin, M.M.; Ryabova, L.A. Ribosome shunting, polycistronic translation, and evasion of antiviral defenses in plant pararetroviruses and beyond. Front. Microbiol. 2018, 9, 644. [CrossRef]   
29. Kozak, M. Effects of intercistronic length on the efficiency of reinitiation by eucaryotic ribosomes. Mol. Cell. Biol. 1987, 7, 3438–3445. [CrossRef]   
30. Rahmani, F.; Hummel, M.; Schuurmans, J.; Wiese-Klinkenberg, A.; Smeekens, S.; Hanson, J. Sucrose controlof translation mediated by an upstream open reading frame-encoded peptide. Plant Physiol. 2009, 150, 1356–1367. [CrossRef]   
31. Yamashita, Y.; Takamatsu, S.; Glasbrenner, M.; Becker, T.; Naito, S.; Beckmann, R. Sucrose sensing through nascent peptide-meditated ribosome stalling at the stop codon of Arabidopsis bZIP11 uORF2. FEBS Lett. 2017, 591, 1266–1277. [CrossRef] [PubMed]   
32. Dong, J.; Nanda, J.S.; Rahman, H.; Pruitt, M.R.; Shin, B.S.; Wong, C.M.; Lorsch, J.R.; Hinnebusch, A.G. Genetic identification of yeast 18S rRNA residues required for efficient recruitment of initiator tRNA(Met) and AUG selection. Genes. Dev. 2008, 22, 2242–2255. [CrossRef] [PubMed]   
33. Kurihara, Y.; Makita, Y.; Kawashima, M.; Fujita, T.; Iwasaki, S.; Matsui, M. Transcripts from downstream alternative transcription start sites evade uORF-mediated inhibition of gene expression in. Proc. Natl. Acad. Sci. USA 2018, 115, 7831–7836. [CrossRef] [PubMed]   
34. Tokizawa, M.; Kusunoki, K.; Koyama, H.; Kurotani, A.; Sakurai, T.; Suzuki, Y.; Sakamoto, T.; Kurata, T.; Yamamoto, Y.Y. Identification of Arabidopsis genic and non-genic promoters by paired-end sequencing of TSS tags. Plant J. 2017, 90, 587–605. [CrossRef] [PubMed]   
35. Srivastava, A.K.; Lu, Y.; Zinta, G.; Lang, Z.; Zhu, J.K. UTR-dependent control of gene expression in plants. Trends Plant Sci. 2018, 23, 248–259. [CrossRef]   
36. Ord, T.; Ord, D.; Kõivomägi, M.; Juhkam, K. Human TRB3 is upregulated in stressed cells by the induction of translationally efficient mRNA containing a truncated $5^{\prime}$ -UTR. Gene 2009, 444, 24–32. [CrossRef]   
37. Besançon, R.; Valsesia-Wittmann, S.; Locher, C.; Delloye-Bourgeois, C.; Furhman, L.; Tutrone, G.; Bertrand, C.; Jallas, A.C.; Garin, E.; Puisieux, A. Upstream ORF affects MYCN translation depending on exon 1b alternative splicing. BMC Cancer 2009, 9, 445. [CrossRef]   
38. Halterman, D.A.; Wise, R.P. Upstream open reading frames of the barley Mla13 powdery mildew resistance gene function co-operatively to down-regulate translation. Mol. Plant Pathol. 2006, 7, 167–176. [CrossRef]   
39. Pasentsis, K.; Paulo, N.; Algarra, P.; Dittrich, P.; Thümmler, F. Characterization and expression of the phytochrome gene family in the moss Ceratodon purpureus. Plant J. 1998, 13, 51–61. [CrossRef]   
40. Combier, J.P.; de Billy, F.; Gamas, P.; Niebel, A.; Rivas, S. Trans-regulation of the expression of the transcription factor MtHAP2-1 by a uORF controls root nodule development. Genes Dev. 2008, 22, 1549–1559. [CrossRef]   
41. Filichkin, S.A.; Priest, H.D.; Givan, S.A.; Shen, R.; Bryant, D.W.; Fox, S.E.; Wong, W.K.; Mockler, T.C. Genome-wide mapping of alternative splicing in Arabidopsis thaliana. Genome Res. 2010, 20, 45–58. [CrossRef] [PubMed]   
42. Lee, K.M.; Chen, C.J.; Shih, S.R. Regulation mechanisms of viral IRES-driven translation. Trends Microbiol. 2017, 25, 546–561. [CrossRef] [PubMed]   
43. Yamamoto, Y.Y.; Tsuhara, Y.; Gohda, K.; Suzuki, K.; Matsui, M. Gene trapping of the Arabidopsis genome with a firefly luciferase reporter. Plant J. 2003, 35, 273–283. [CrossRef] [PubMed]   
44. Kurihara, Y.; Okubo-Kurihara, E.; Matsui, M. Polycistronic expression of RNA silencing suppressor protects its own mRNA from RNA silencing. Plant Biotechnol. 2015, 32, 89–95. [CrossRef]   
45. Weingarten-Gabbay, S.; Elias-Kirma, S.; Nir, R.; Gritsenko, A.A.; Stern-Ginossar, N.; Yakhini, Z.; Weinberger, A.;Segal, E. Comparative genetics. Systematic discovery of cap-independent translation sequences in human and viral genomes. Science 2016, 351. [CrossRef]   
46. Hellen, C.U.; Sarnow, P. Internal ribosome entry sites in eukaryotic mRNA molecules. Genes Dev. 2001, 15, 1593–1612. [CrossRef]   
47. Cui, Y.; Rao, S.; Chang, B.; Wang, X.; Zhang, K.; Hou, X.; Zhu, X.; Wu, H.; Tian, Z.; Zhao, Z.; et al. AtLa1protein initiates IRES-dependent translation of WUSCHEL mRNA and regulates the stem cell homeostasis of Arabidopsis in response to environmental hazards. Plant Cell Environ. 2015, 38, 2098–2114. [CrossRef]   
48. Dinkova, T.D.; Zepeda, H.; Martínez-Salas, E.; Martínez, L.M.; Nieto-Sotelo, J.; de Jiménez, E.S. Cap-independent translation of maize Hsp101. Plant J. 2005, 41, 722–731. [CrossRef]   
49. Meyer, K.D.; Patil, D.P.; Zhou, J.; Zinoviev, A.; Skabkin, M.A.; Elemento, O.; Pestova, T.V.; Qian, S.B.; Jaffrey, S.R. $5^{\prime}$ UTR m(6)A promotes cap-independent translation. Cell 2015, 163, 999–1010. [CrossRef]   
50. Zhou, J.; Wan, J.; Gao, X.; Zhang, X.; Jaffrey, S.R.; Qian, S.B. Dynamic m(6)A mRNA methylation directs translational control of heat shock response. Nature 2015, 526, 591–594. [CrossRef]   
51. Zhou, J.; Wan, J.; Shu, X.E.; Mao, Y.; Liu, X.M.; Yuan, X.; Zhang, X.; Hess, M.E.; Brüning, J.C.; Qian, S.B.N6-Methyladenosine guides mRNA alternative translation during integrated stress response. Mol. Cell 2018, 69, 636–647. [CrossRef] [PubMed]   
52. Xu, G.; Yuan, M.; Ai, C.; Liu, L.; Zhuang, E.; Karapetyan, S.; Wang, S.; Dong, X. uORF-mediated translation allows engineered plant disease resistance without fitness costs. Nature 2017, 545, 491–494. [CrossRef] [PubMed]  