# High-Throughput Quantitation of Yeast uORF Regulatory Impacts Using FACS-uORF  

Gemma E. May, C. Joel McManus  

1Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA, USA   
2Computational Biology Department, Carnegie Mellon University, Pittsburgh, PA, USA  

# Abstract  

Eukaryotic upstream Open Reading Frames (uORFs) are short translated regions found in many transcript leaders (Barbosa et al. PLoS Genet 9:e1003529, 2013; Zhang et al. Trends Biochem Sci 44:782–794, 2019). Modern transcript annotations and ribosome profiling studies have found thousands of AUG-initiated uORFs, and many more uORFs initiated by near-cognate codons (CUG, GUG, UUG, etc.). Their translation generally decreases the expression of the main encoded protein by preventing ribosomes from reaching the main ORF of each gene, and by inducing nonsense mediated decay (NMD) through premature termination. Under many cellular stresses, uORF containing transcripts are de-repressed due to decreased translation initiation (Young et al. J Biol Chem 291:16927–16935, 2016). Traditional experimental evaluation of uORFs involves comparing expression from matched uORF-containing and start-codon mutated transcript leader reporter plasmids. This tedious process has precluded analysis of large numbers of uORFs. We recently used FACS-uORF to simultaneously assay thousands of yeast uORFs in order to evaluate the impact of codon usage on their functions (Lin et al. Nucleic Acids Res 2:1–10, 2019). Here, we provide a step-by-step protocol for this assay.  

# Keywords  

mRNA translation; uORFs; Massively parallel reporter assay  

# 1 Introduction  

Eukaryotic mRNA translation is regulated primarily at the initiation stage [5]. Most eukaryotic translation is cap-dependent under unstressed conditions. Translation initiation requires recruitment of the ribosomal subunit and numerous eukaryotic initiation factors (eIFs) to mRNA $5^{\prime}$ caps to form a preinitiation complex (PIC). The assembled PIC then scans directionally, from $5^{\prime}$ to $3^{\prime}$ , until a start codon is recognized, the complete ribosome is assembled, and protein synthesis is initiated. When translation initiates at uORFs, this typically precludes translation of the main protein encoded in the mRNA transcript. uORFs can also induce nonsense mediated decay, due to translation termination in the $5^{\prime}$ UTR [1]. Under stress conditions, phosphorylation of the alpha subunit of eIF2 decreases the rate  

of translation initiation. This allows stress-responsive mRNAs to be rapidly derepressed in response to numerous cellular stresses [3].  

Genomic analyses have revealed that uORFs are very common, occurring in ${\sim}15\%$ of yeast and ${\sim}50\%$ of mammalian transcripts [2, 6]. Despite their pervasive nature, the functions of most uORFs have not been tested. Traditional assays to test uORF functions typically involve comparing expression from reporter genes with wild-type $5^{\prime}$ transcript leaders to expression from reporters with transcript leaders in which the uORF start codon has been mutated. Typically, these reporters are individually cloned and assayed one at a time. This low throughput approach has precluded high-throughput analyses of uORF activity.  

Over the past decade, several groups have combined technological improvements in DNA synthesis and sequencing to create high-throughput reporter assays for transcription [7, 8], mRNA splicing [9–11], and translation [12–15]. These experimental systems are collectively known as massively parallel reporter assays, or MPRAs. Here we provide a detailed protocol for FACS-uORF, an MPRA system we generated to assay thousands of uORFs in parallel in yeast [4]. Our protocol covers all steps of the process, including cloning a designer library of reporter plasmids, FACS-sorting cells, and generating high-throughput sequencing libraries to evaluate uORF function (Fig. 1).  

# 2 Materials  

1. Pipettes with filter tips (P1000, P200, P20, P2).   
2. 1xTE, 10 mM Tris, 0.1 mM EDTA, pH 8.0.   
3. 10 pmole custom designed oligo pool (Agilent). Design an oligo library where each oligo has common $5^{\prime}$ and $3^{\prime}$ ends that allow for PCR amplification and plasmid integration. The $5^{\prime}$ end sequence is the PCR primer, ENO2-Lib-F1 plus the sequence, AACTAATA, and the $3^{\prime}$ end sequence is the complementary to the FACS-uORF-YFP-R primer (see Table 1, and Fig. 2)   
4. $1.5\;\mathrm{mL}$ microcentrifuge tubes.   
5. Vortexer.   
6. 0.2 mL PCR tube.   
7. DNase/RNase-free distilled water (nuclease free).   
8. Herculase II Fusion DNA polymerase with dNTPs Combo (Agilent).   
9. Custom DNA oligo primers (see Table 1).   
10. Plasmid vectors (pGM-ENO2-YFP-mCherry, pGM-YFP, pGM-mCherry).   
11. Thermocycler, any model.   
12. DNA Clean & Concentrator —5 (Zymo Research) or similar PCR column clean up kit.   
13. Tabletop centrifuge capable of accommodating $1.5\;\mathrm{mL}$ ultracentrifuge tubes.  

14. D1000 ScreenTape and sample buffer (Agilent).   
15. TapeStation Instrument (Agilent Technologies, model 4400 or similar).   
16. ${\mathrm{Q}}5^{\mathrm{(p)}}$ High-Fidelity DNA polymerase.   
17. dNTP mix, $10\;\mathrm{mM}$ each.   
18. NanoDrop  2000 Spectrophotometer or similar.   
19. FastDigest XmaJI (Cuts AvrII sites (isoschizomer)) (ThermoFisher Scientific).   
20. FastDigest BglII (ThermoFisher Scientific).   
21. Phenol–chloroform–isoamyl alcohol (25:24:1, v/v).   
22. Shrimp alkaline phosphatase (rSAP, New England Biolabs).   
23. AMPure XP beads (Beckman Coulter).   
24. Magnetic rack or plate for $0.2\;\mathrm{mL}$ tubes.   
25. Ethanol, absolute 200 proof.   
26. Rapid DNA Ligation Kit (ThermoFisher Scientific).   
27. ElectroMAX  DH10B Cells (ThermoFisher Scientific).   
28. 1.1 cm electroporation cuvette.   
29. Electroporator.   
30. S.O.C. medium.   
31. $16\,\mathrm{mL}$ culture tube.   
32. $30{-}37\ ^{\circ}\mathrm{C}$ shaking incubator with $16\;\mathrm{mL}$ culture tube holders.   
33. $100\;\mathrm{mL}$ and $1\,\mathrm{L}$ Erlenmeyer flask clamps.   
34. Luria Broth Agar (LB agar).   
35. Petri dishes $100\:\mathrm{mm}\times15\:\mathrm{mm}$ .  
36. Ampicillin.   
37. Sterile glass plating beads.   
38. $30{-}37\ ^{\circ}\mathrm{C}$ plate incubator.   
39. Luria Broth (LB), autoclaved and sterile.   
40. Laboratory wash bottle with a spout.   
41. $50\,\mathrm{mL}$ conical tubes.   
42. General use scale.   
43. Refrigerated centrifuge with $50\;\mathrm{mL}$ tube capacity rotor.   
44. Plasmid Maxi Prep Kit.   
45. Yeast extract-peptone-dextrose growth medium agar (YEPD agar) plates, sterile.   
46. Yeast extract-peptone-dextrose growth medium (YEPD broth), sterile.   
47.Frozen-EZ Yeast Transformation II Kit (Zymo Research).  
48. S. cerevisiae strain BY4741 (MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0).   
49. $100\;\mathrm{mL}$ and $1\,\mathrm{L}$ glass Erlenmeyer flasks.   
50. Autoclavable glass storage bottles (1 L).   
51. 1 mL spectrophotometer cuvette.   
52. Spectrophotometer with $1.0\;\mathrm{mL}$ cuvette capacity.   
53. SD-URA plates and broth (yeast nitrogen base without amino acids (Sigma  
Aldrich), yeast Synthetic drop-out medium supplements without uracil (Sigma  
Aldrich), dextrose, bacteriological agar).   
54. Sterile serological pipettes (5, 10, 25, 50 mL).   
55. Pipet-Aid or similar pipette controller.   
56. Bunsen burner and striker.   
57. Sterile cryovial $1.8\;\mathrm{mL}$ ) or similar cryogenic tubes.   
58. Sterile $30\%$ glycerol.   
59. $5\;\mathrm{mL}$ sterile culture tubes.   
60. FACSVantage Digital Cell Sorter, or similar.   
61. Tecan Microplate reader, or similar.   
62. White 96 well flat bottom plates compatible with plate reader.   
63. Zymoprep Yeast Plasmid Miniprep II kit (Zymo Research).   
64. NEBNext  dual index primers (New England Biolabs, E6440).   
65. Qubit  dsDNA HS Assay Kit (Invitrogen).   
66. $0.5\;\mathrm{mL}$ optically clear individual PCR tubes   
67. Qubit  Fluorometer, any model (Thermo Fisher Scientific).  

# 3 Methods  

# 3.1  Oligo Library PCR Amplification  

1. Resuspend 10 pmoles of the custom-designed oligo pool (item 3 in Subheading 2) by adding $100\,\upmu\mathrm{L}$ of TE to the lyophilized oligos. Vortex to mix.   
2. Set up the following reaction in a $1.5\;\mathrm{mL}$ ultracentrifuge tube. Herculase polymerase is specifically used to maintain library complexity when amplifying from a dilute oligo pool:  

![](images/26a95001cbb7cade03cd83f9b1a36053da48a1790e1116647c624a708f3f154a.jpg)  

3. Mix the reaction by setting the pipettor to $150\,\upmu\mathrm{L}$ and pipetting up and down several times.  

4. Aliquot the reaction into eight $0.2\;\mathrm{mL}$ PCR tubes with $50\,\upmu\mathrm{L}$ in each tube.  

5. Put the reaction in a thermocycler and run the program.  

a. $95\ ^{\circ}\mathbf{C}$ for $1\ \mathrm{min}$ .  
b. 10 cycles of $95\,^{\circ}\mathrm{C}$ for $20{\mathrm{~s}}.$ ,$55\ ^{\circ}\mathrm{C}$ for $20~\mathrm{s}$ , and $68\ ^{\circ}\mathrm{C}$ for $20~\mathrm{s}$ .  
c. $68\ ^{\circ}\mathrm{C}$ for $4\,\mathrm{min}$ .  

6. Combine the PCR into one $1.5\;\mathrm{mL}$ ultracentrifuge tube and purify over a DNA Clean & Concentrator  column (or similar PCR clean up column kit) following the manufacturer’s instructions. Elute the PCR product in $20\,\upmu\mathrm{L}$ of nuclease-free water. The PCR product can be saved at $-20~^{\circ}\mathrm{C}$ .  

7. Visualize $1\,\upmu\mathrm{L}$ of PCR product on a D1000 DNA ScreenTape on a TapeStation (or similar) to confirm that the correct sized products are present. The PCRs add 90 base pairs to the designed insert size.  

8. Set up the following reaction in a $1.5\;\mathrm{mL}$ ultracentrifuge tube.  

![](images/f49194cbda4a35dabae82b6839d7aa1f88352d9232ff5acb3eced38d287281e4.jpg)  

9. Aliquot $50\,\upmu\mathrm{L}$ of the reaction in to eight $0.2\;\mathrm{mL}$ PCR tubes. Place the tubes in a thermocycler and run the program.  

a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  
b. 10 cycles of $98\ ^{\circ}\mathbf{C}$ for $10\;\mathrm{s}.$ ,$55\ ^{\circ}\mathrm{C}$ for $20{\mathrm{~s}}.$ , and $72\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  

# c. $68\ ^{\circ}\mathrm{C}$ for $4\,\mathrm{min}$ .  

10. Combine the PCR into one $1.5\;\mathrm{mL}$ ultracentrifuge tube and purify over a DNA Clean & Concentrator  column following the manufacturer’s instructions. Elute the PCR product in $50\,\upmu\mathrm{L}$ of nuclease-free water.   
11. Quantify the PCR product using a Nanodrop. The typical yield is ${\sim}7.5~\upmu\mathrm{g}$ . The purified PCR product can be saved at $-20~^{\circ}\mathrm{C}$ .  

# 3.2  Restriction Digest—Digest the Oligo PCR Products and Vector  

1. Digest the PCR products and vector in preparation for ligation. Set up the following reactions in one $0.2\;\mathrm{mL}$ PCR tube for each reaction.  

![](images/2a1026f1d7881555dd3cdbf20bb55581ff474b3ea1266977db96a7319644c62b.jpg)  

2. Mix each restriction digest reaction thoroughly with a pipettor and incubate in a thermocycler for $1\,\mathrm{{h}}$ at $37\ ^{\circ}\mathbf{C}$ .  
3. Transfer the restriction digests to $1.5\;\mathrm{mL}$ microcentrifuge tubes. Add $150\,\upmu\mathrm{L}$ of TE to each restriction digest.   
4. In a fume hood, add $300\,\upmu\mathrm{L}$ of phenol–chloroform–isoamyl alcohol (25:24:1, v/v) to each reaction. Vortex for $30\;\mathrm{s}$ .  
5. Centrifuge the sample at top speed, or ${\sim}14\mathrm{,}000\times g$ in a tabletop microcentrifuge for $2\;\mathrm{min}$ at room temperature.   
6. In a fume hood, remove $250\,\upmu\mathrm{L}$ the supernatant to a fresh $1.5\;\mathrm{mL}$ tube, without disturbing the bottom phenol layer.   
7. Purify the supernatant over a DNA Clean & Concentrator  column following the manufacturer’s instructions and elute the cut PCR product and vector in 15 $\upmu\mathrm{L}$ and $43\,\upmu\mathrm{L}$ of nuclease-free water respectively.   
8. Quantify the digested PCR product using the NanoDrop. The typical yield for the PCR product is ${\sim}750\,\mathrm{ng}$ . Proceed immediately to Subheading 3.3, saving the PCR product on ice.  

# 3.3  Vector Phosphatase Treatment  

1. Set up the following reaction in a $0.2\;\mathrm{mL}$ PCR tube for the vector.  

![](images/8e708a55e18e6b3c3ffddd58421ceb654a45663d20afa16b1fc9ec3c194d492a.jpg)  

2. Place the above reaction in a thermocycler and incubate at $37\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{min}$ .Heat inactivate at $65\,^{\circ}\mathrm{C}$ for $5\;\mathrm{min}$ .  
3. Remove the phosphatase reaction from the thermocycler and add $50\,\upmu\mathrm{L}$ of nuclease-free water.   
4. Vortex the AMPure XP beads until they are resuspended and add $100\,\upmu\mathrm{L}$ of beads to the phosphatase reaction. Briefly vortex the reaction and incubate for 5 min at room temperature.   
5. Place the tube on the magnet and incubate for $5\;\mathrm{min}$ at room temperature.   
6. With the tube still on the magnet, carefully remove the supernatant and discard the supernatant.   
7. Wash the beads by adding $150\,\upmu\mathrm{L}$ of freshly prepared $80\%$ ethanol without disturbing the pellet. Incubate for $30\;\mathrm{s}$ and remove the ethanol.   
8. Let the pellet air-dry for around $3\;\mathrm{min}$ . Using a pipettor, pipet any remaining ethanol from the tube.   
9. Remove the tube from the magnet and add $20\,\upmu\mathrm{L}$ of nuclease-free water to the beads and resuspend by pipetting up and down. Incubate at room temperature for $2\;\mathrm{min}$ .  
10. Place the tube on the magnet and incubate for $2\;\mathrm{min}$ or until the supernatant is clear. Remove the supernatant to a fresh $1.5\;\mathrm{mL}$ microcentrifuge tube.   
11. Quantify the digested and phosphatased vector using a NanoDrop. The typical yield is $1.5\,\upmu\mathrm{g}$ . Proceed immediately to the ligation (Subheading 3.4).  

# 3.4  Ligation  

1. Set up the following reaction in a $0.2\;\mathrm{mL}$ PCR tube. For a negative control, set up a duplicate ligation reaction adding nuclease-free water instead of digested PCR product.  

![](images/1d08abb02276a1413d44f4129c7e1265f208e8cdb1fa47172338db1989d20cbf.jpg)  

![](images/f5476c7ad32b6439ae9f55696d7eec22cd4d4e20f3e46c5329f448321317fa88.jpg)  

2. Incubate both reactions at $22\ ^{\circ}\mathbf{C}$ for $10\;\mathrm{min}$ Proceed to the next step or save the reactions at $-20~^{\circ}\mathrm{C}$ overnight.   
3. Purify the ligation reactions over a DNA Clean & Concentrator™ column following the manufacturer’s instructions. Elute in $10\,\upmu\mathrm{L}$ of nuclease-free water. Proceed to Subheading 3.5 or save the ligations overnight at $-20~^{\circ}\mathrm{C}$ .  

# 3.5  E. coli Transformation  

Note: A small-scale initial transformation is performed to estimate the transformation efficiency and yield. Once these are established, a larger-scale transformation is performed to recover enough colonies to maintain library diversity.  

1. Prechill one $0.1\;\mathrm{cm}$ electroporation cuvette per reaction on ice.  

1   
2. In a cold $1.5\;\mathrm{mL}$ microcentrifuge tube, add $1.5\,\upmu\mathrm{L}$ of the ligation to $30\,\upmu\mathrm{L}$ of competent cells. Gently mix pipetting up and down 2–3 times with a pipettor set to $20\,\upmu\mathrm{L}$ .  
3. Transfer the cells and ligation mix to the electroporation cuvettes.   
4. Electroporate each reaction at $2.0\,\mathrm{kV},200\,\Omega$ , and $25\;\upmu\mathrm{F}.$ Immediately add $1.5\;\mathrm{mL}$ of room temperature S.O.C medium and transfer the cells and liquid to a $16\;\mathrm{mL}$ culture tube.   
5. Place the cells in a shaking incubator set to $37\ ^{\circ}\mathbf{C}$ and incubate shaking at 225 rpm for $^{1\,\mathrm{h}}$ . During this time, prewarm the necessary number of LB-AMP (LB agar medium $+\ 100\ \upmu\mathrm{g/mL}$ ampicillin) plates in a $37\ ^{\circ}\mathbf{C}$ plate incubator. See Note 1 for LB-AMP plate preparation.   
6. To determine the ligation and transformation efficiency of the reaction, use sterile glass plating beads to plate 10, 25, 50, 100, and $200\,\upmu\mathrm{L}$ of cells on 5 LB-AMP plates. Leave the glass plating beads in the dish lids and incubate overnight at $37\ ^{\circ}\mathbf{C}$ with the agar side up.   
7. Choose a plate that contains at least a few hundred colonies and count the colonies on the plate. Extrapolate the best volume for plating so that there will be about 2000–5000 colonies on a single plate. This volume can range from anywhere from 10 to $200\,\upmu\mathrm{L}$ of transformed $E_{\cdot}$ . coli per plate. The negative control may have a few colonies (less than 50) on the $200\,\upmu\mathrm{L}$ plates. These false  

positives are likely due to a very low rate of incomplete digestion or religation and does signify any technical issues of consequence.  

8. Once the ligation and transformation efficiencies are determined for the library construct, repeat steps 1–7 in this section until the desired number of colonies is reached (see Note 2). LB-AMP plates containing colonies can be saved at $4\,^{\circ}\mathbf{C}$ for a few days if multiple rounds of transformation are required to obtain enough colonies.  

# 3.6  Harvesting Colonies and Extracting Massively Parallel Reporter (MPR) Plasmid DNA  

1.   
2. Open the lids to at most 4 plates with 4000–5000 colonies each and squirt enough LB onto the plates so that the glass beads can freely roll on the plates. Be sure to not add too much LB so that liquid overflows out of the dish.   
3. Put the lids back on the plates and stack them on top of each other. Gently roll the beads across the colonies several times so that all the colonies on each plate are resuspended in the LB.   
4. Weigh a $50\,\mathrm{mL}$ conical (with the lid on) and record the weight. Open the $50\;\mathrm{mL}$ conical and place the lid to one side.   
5. For each plate, gently tilt the dish so that the liquid flows to one side. Using a 1 mL pipettor, remove the liquid from the dish and pipet it into the $50\;\mathrm{mL}$ conical. Repeat this step for each of the four dishes.   
6. Repeat steps 2–5 in this section as necessary so that all the colonies from all of the plates have been collected. Colonies are collected from four plates at a time because the liquid will soak into the LB agar after a few minutes. If the liquid has soaked into the plate before the colonies are collected, more liquid can be added to the dish prior to collection.   
7. Once all of the colonies have been collected, screw the lid back onto the $50\;\mathrm{mL}$ conical and centrifuge at $6000\times g$ for $15\;\mathrm{min}$ at $4\,^{\circ}\mathbf{C}$ .  
8. Carefully decant the clear liquid taking care to not disturb the cell pellet.   
9. Weigh the $50\;\mathrm{mL}$ conical again with the cell pellet to determine the weight of the cell pellet. If desired, the cell pellet can be stored at $-20\,^{\circ}\mathrm{C}$ at this step.   
10. Extract the MPR plasmid DNA using a QIAGEN Maxi prep kit according to the manufacturer’s instructions. Be sure to not overload the QIAGEN-tip 500 (see Note 3).   
11. Measure the concentration of the extracted MPR library using a NanoDrop. The typical yield is approximately $500\,\upmu\mathrm{g}$ . Store the library at $-20~^{\circ}\mathrm{C}$ .  
12. If desired, the sequence diversity of the MPR plasmid library can be verified. Continue with Subheading 3.11, steps 2–19 to create a high-throughput sequencing library from the plasmid library (see Note 4 for modifications to the protocol).  

# 3.7  Making Competent Yeast  

1. Streak BY4741 (see Note 5) on a YEPD agar plate. Incubate at $30\,^{\circ}\mathbf{C}$ for $48\,\mathrm{h}$ in a plate incubator.   
2. Pick a single colony and inoculate $5\;\mathrm{mL}$ of YEPD in a $16\;\mathrm{mL}$ culture tube. Incubate overnight shaking at $300\,\mathrm{rpm}$ at $30\,^{\circ}\mathbf{C}$ .  
3. In a sterile $100\;\mathrm{mL}$ Erlenmeyer flask, start a $50\;\mathrm{mL}$ culture in YEPD at an $\mathrm{OD}_{600}$ of 0.1 by diluting the overnight culture into fresh YEPD. Incubate the culture shaking at $300\;\mathrm{rpm}$ at $30\,^{\circ}\mathbf{C}$ for 4–6 h until mid-log phase ( $\mathrm{{[OD_{600}}}$ of 0.6–0.8).   
4. Pellet the cells by transferring the culture to two $50\,\mathrm{mL}$ conical tubes and centrifuging for $4\,\mathrm{min}$ at $500\times g$ at room temperature.   
5. Carefully decant the clear supernatant and resuspend the pellet in 10 mL EZ 1 solution (Frozen-EZ Yeast Transformation II Kit).   
6. Pellet the cells by centrifuging for $4\,\mathrm{min}$ at $500\times g$ at room temperature. Remove the supernatant by carefully decanting. Remove any excess liquid using a P200 pipettor.   
7. Add $500\,\upmu\mathrm{L}$ of EZ 2 solution to each of the pellets. Resuspend the cells and combine the cells into one $1.5\;\mathrm{mL}$ microcentrifuge tube.   
8. Aliquot $400\,\upmu\mathrm{L}$ of cells into two $1.5\;\mathrm{mL}$ microcentrifuge tubes. Aliquot the remaining $200\,\upmu\mathrm{L}$ as $50\,\upmu\mathrm{L}$ aliquots in among $1.5\;\mathrm{mL}$ microcentrifuge tubes.  

9. Slowly freeze the cells by placing them inside a Styrofoam box with a lid. Put the Styrofoam box in a $-80~^{\circ}\mathrm{C}$ freezer and allow the cells to freeze slowly overnight.  

# 3.8  Transforming Saccharomyces yeast with the pGM-YFP, and pGM-mCherry Controls  

1. Thaw two $50\,\upmu\mathrm{L}$ aliquots of cells on ice. Add $200\;\mathrm{ng}$ of $p G M\!-\!Y\!F\!P$ to one aliquot and ${200}\,\mathrm{ng}$ of $p G M\/$ -mCherry to the other aliquot. Add $500\,\upmu\mathrm{L}$ of EZ 3 solution and mix the transformations thoroughly. Incubate the cells shaking at $300\,\mathrm{rpm}$ at $30\,^{\circ}\mathbf{C}$ for $2\,\mathrm{{h}}$ .  
2. Plate $10\,\upmu\mathrm{L}$ of each transformation on SD-URA plates. Incubate for 2 days at 30 $^{\circ}\mathbf{C}.$ Discard the remaining transformation mixtures.   
3. Yeast transformed with the mCherry only vector should appear pink in color. The YFP only transformed yeast should not be pink. If desired to further check for positive transformants, examine the plates using a blue transilluminator with an amber filter. Positive colonies should appear to be glowing red (mCherry) and yellow (YFP).   
4. Prepare glycerol stocks by inoculating $5\;\mathrm{mL}$ of SD-URA-liquid media in a $16\,\mathrm{mL}$ culture tube from one positive colony from each transformation. Grow overnight shaking at $300\;\mathrm{rpm}$ at $30\,^{\circ}\mathbf{C}$ .  
5. Transfer each culture to a $15\;\mathrm{mL}$ conical tube. Centrifuge the conical tubes at $500\times g$ for $5\:\mathrm{min}$ at room temperature. Decant or pipet off the media leaving the cell pellet behind.   
6. Resuspend the pellets in $800\,\upmu\mathrm{L}$ of SD-URA media and add $800\,\upmu\mathrm{L}$ of $30\%$ glycerol. Transfer the cells to $1.8\;\mathrm{mL}$ cryovials and store the glycerol stocks at $-80~^{\circ}\mathrm{C}$ .  

# 3.9  Transforming Saccharomyces yeast with the MPR Library  

1. Thaw one $400\,\upmu\mathrm{L}$ aliquot of cells on ice. Add $2\,\upmu\mathrm{g}$ of MPR plasmid library and 4 $\mathrm{mL}$ of EZ 3 solution and mix thoroughly. Incubate the cells shaking at $30\,^{\circ}\mathbf{C}$ for $2\,\mathrm{{h}}$ .  
2. Add $40\;\mathrm{mL}$ SD-URA liquid media to a $100\;\mathrm{mL}$ Erlenmeyer flask. Add the $4\,\mathrm{mL}$ of MPR transformation mixture. Incubate shaking overnight at $30\,^{\circ}\mathbf{C}$ .  
3. To assess the transformation efficiency, plate $10\,\upmu\mathrm{L}$ of the MPR transformation on a SD-URA plate. Incubate the plate for 2 days in a plate incubator at $30\,^{\circ}\mathbf{C}$ .Typically, $10\,\upmu\mathrm{L}$ of transformation yields approximately 500 colonies.   
4. Add $200\;\mathrm{mL}$ of URA-media to a $1\,\mathrm{L}$ flask and add the previous night’s culture to the fresh media and incubate overnight shaking at $30\,^{\circ}\mathbf{C}$ .  
5. Prepare glycerol stocks. Aliquot the overnight culture into five $50\;\mathrm{mL}$ conical tubes with $49\;\mathrm{mL}$ of culture in each tube.   
6. Centrifuge the conical tubes at $500\times g$ for $5\;\mathrm{min}$ at room temperature. Decant or pipet off the media leaving the cell pellet behind.   
7. For each cell pellet, resuspend the pellet in $3\;\mathrm{mL}$ of SD-URA liquid media. Add $4\;\mathrm{mL}$ of $30\%$ glycerol and pipet up and down several times to mix. The volume of the pellet usually is around $1\ \mathrm{mL}$ , if this is not the case adjust the amount of glycerol so that the final concentration of glycerol is $15\%$ .  
8. Aliquot $1.6\;\mathrm{mL}$ of the yeast culture and glycerol mix to $1.8\;\mathrm{mL}$ cryovial (22–25 vials). Store the glycerol stocks at $-80~^{\circ}\mathrm{C}$ .  

# 3.10  FACS  

1. Thaw one aliquot of the yeast MPRA library (Subheading 3.9, step 8). Add the thawed yeast to $200\;\mathrm{mL}$ of liquid SD-URA media in a $^{1\,\mathrm{L}}$ Erlenmeyer flask. Incubate the culture overnight shaking at $300\;\mathrm{rpm}$ at $30\;^{\circ}\mathbf{C},$ . Go directly to step 2 without waiting for this overnight culture.   
2. Start cultures of the YFP and mCherry only controls (from glycerol stocks, Subheading 3.8, step 6) in $5\;\mathrm{mL}$ of SD-URA- liquid media in $16\;\mathrm{mL}$ culture tubes. In addition, start a culture of the untransformed BY4741 (“cells only” control) from a glycerol stock in $5\;\mathrm{mL}$ of YEPD liquid media in a $16\;\mathrm{mL}$ culture tube. Incubate the cultures overnight shaking at $300\,\mathrm{rpm}$ at $30\,^{\circ}\mathbf{C}$ .  
3. The next day, restart the MPR transformed yeast cells in $50\;\mathrm{mL}$ of SD-URA liquid media in an Erlenmeyer flask at an $\mathrm{OD}_{600}$ of 0.1. Grown shaking $30\,^{\circ}\mathbf{C}$ to an $\mathrm{OD}_{600}$ of 0.7. Restart the YFP only and mCherry only controls by pipetting $50\,\upmu\mathrm{L}$ of the overnight culture into a fresh aliquot of $5\;\mathrm{mL}$ of SD-URA liquid media in a $16\;\mathrm{mL}$ culture tube. Restart the cells only control by pipetting $50\,\upmu\mathrm{L}$ of the overnight culture into a fresh aliquot of $5\;\mathrm{mL}$ YEPD liquid media in a $16\,\mathrm{mL}$ culture tube. Grow all the cultures shaking at $300\;\mathrm{rpm}$ at $30\,^{\circ}\mathbf{C}$ until the MPR transformed yeast grow to an $\mathrm{OD}_{600}$ of 0.7 (approximately $5\,\mathrm{h}$ ).   
4. Save some unsorted “Bin $0^{\circ}$ cells for plasmid DNA extraction. Aliquot $1\ \mathrm{mL}$ of the MPR culture into a $1.5\;\mathrm{mL}$ microcentrifuge tube. Centrifuge the cells for 1 min at $12{,}000\times g$ at room temperature. Pipet off the supernatant and save the pellet at $-20~^{\circ}\mathrm{C}$ .  
5. Transfer the cells only, YFP only, and mCherry only controls into $5\;\mathrm{mL}$ culture tubes or tubes compatible with the FACS machine being used. Place the tubes on ice.   
6. Concentrate the MPR transformed yeast by transferring the culture to four 15 mL conical tubes and centrifuge them at $500\times g$ for 5 min in a refrigerated centrifuge. Pipet off the supernatant leaving around $2\;\mathrm{mL}$ at the bottom of the tube. Resuspend the pellets in the remaining liquid and transfer the remaining cultures into $5\;\mathrm{mL}$ culture tubes and place the tubes on ice.   
7. Create a gate using the nonfluorescent yeast control, then set fluorescence compensation levels using the YFP-only and mCherry-only controls.  

8. Place the MPR yeast on the sorter while in a chiller. If no chiller is available, a plastic bag filled with ice, wrapped around the tube and secured with tape is sufficient. Calculate the YFP-to-mCherry ratios using the available FACS software and divide the cells into nine bins (Bin 1–Bin 9), based on this ratio (see Fig. 3). Gate the cells that fall in the mid-range for mCherry values (excluding negative mCherry cells). Record how many events are counted for each of the gated bins. For each bin, deposit 100,000 cells into culture tubes containing $5\;\mathrm{mL}$ of SD-URA liquid media. Grow each bin’s culture overnight shaking at $30\,^{\circ}\mathbf{C}$ at $300\;\mathrm{rpm}$ .  

9. The next morning, measure the ratio of YFP to mCherry for each bin using the Tecan fluorometer (or other plate reader). Pipet $100\,\upmu\mathrm{L}$ of each culture into three separate wells $300\,\upmu\mathrm{L}$ total) into a white 96 well plate compatible with the plate reader being used. For a negative control, pipet 3 wells containing $100\,\upmu\mathrm{L}$ of SD-URA- media only.  

10. Measure the ratio of YFP to mCherry in the plate reader with the lid off at room temperature. For YFP, set the excitation to $512\pm5\;\mathrm{nm}$ and emission to $532\pm5$ nm. For mCherry, set the excitation to $586\pm5\:\mathrm{nm}$ and emission to $608\pm5~\mathrm{nm}$ .Before calculating the ratios, subtract the value for the negative control for YFP and mCherry from each of the corresponding values for YFP and mCherry for the bins. See Fig. 4 for typical replicates of YFP-to-mCherry ratios among bins.  

11. Harvest the cells from the bins by pipetting $1~\mathrm{mL}$ of cells from each bin into nine $1.5\;\mathrm{mL}$ microcentrifuge tubes. Centrifuge the cells in a microcentrifuge at room temperature for 1 min at top speed. Pipet off the supernatant. The cells can be frozen at $-20\,^{\circ}\mathrm{C}$ or the plasmid DNA can be extracted at this point (Subheading 3.11, step 1).  

# 3.11  Prepare Plasmid DNA Libraries  

1. Extract the plasmid DNA from bin 0 (Subheading 3.10, step 4) and each sorted bin (Subheading 3.10, step 11) using the Zymoprep  Yeast Plasmid Miniprep II kit according to the manufacturer’s instructions. Measure the concentration of DNA using a NanoDrop.  

2. Dilute each plasmid preparation to a concentration of $5~\mathrm{ng/\upmuL}$ . Set up the following PCR in $0.2\;\mathrm{mL}$ PCR tubes on ice for each plasmid prep. Include a negative control reaction where nuclease-free water is added to the reaction instead of plasmid template.  

![](images/1fc136457e8ba630556246c5aa6d11478397a9c49bb4d9ec32eef042a207569d.jpg)  

![](images/2ce71a134a613bdfea98583b88d6decd4fba6500b50f30edae7e1dd2f25f1929.jpg)  

3. Run the program.  

a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .b. 8 cycles of $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ ,$55\ ^{\circ}\mathrm{C}$ for $30\;\mathrm{s}$ , and $72\ ^{\circ}\mathbf{C}$ for $20\;\mathrm{s}$ .  
4. Purify each reaction (including the negative control) using AMPure XP beads. Vortex the beads until they are resuspended and add $37.5\,\upmu\mathrm{L}$ of beads to each reaction. Briefly vortex the reactions and incubate for $5\;\mathrm{min}$ at room temperature.   
5. Place the tubes on the magnet and incubate for $5\:\mathrm{min}$ at room temperature.   
6. With the tubes still on the magnet, carefully remove the supernatant and discard the supernatants.   
7. Wash the beads by adding $150\,\upmu\mathrm{L}$ of freshly prepared $80\%$ ethanol without disturbing the pellets. Incubate for $30\;\mathrm{s}$ and remove the ethanol.   
8. Let the pellets air-dry for around $3\;\mathrm{min}$ . Using a pipettor, pipet any remaining ethanol from the tube.   
9. Remove the tubes from the magnet and $10\,\upmu\mathrm{L}$ of nuclease-free water to the beads and resuspend by pipetting up and down. Incubate at room temperature for 2 min.   
10. Place the tubes on the magnet and incubate for $2\;\mathrm{min}$ or until the supernatants are clear. Remove the supernatants to a fresh $0.2\;\mathrm{mL}$ PCR tubes.   
11. Prepare a pool of primers such that all 7 forward primers (FuORF_2_DNA_N0- N7_F) and all 4 reverse primers $(\mathrm{FuORF\_2\_DNA\_NO},\mathrm{2},\mathrm{4},\mathrm{6\_R})$ are mixed together at a final concentration of $10\,\upmu\mathrm{M}$ in nuclease-free water.   
12. Set up the following PCR in $0.2\;\mathrm{mL}$ PCR tubes on ice for each purified PCR reaction from step 10. Include the purified negative control reaction from step 10 as the negative control reaction for this step.  

![](images/bec54ffe168d8a8f8095a17fb8fbe5f97a6193043fc72da975cc8fb4be489725.jpg)  

13. Run the program.  

a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  

6 cycles of $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ ,$55\ ^{\circ}\mathrm{C}$ for $30\;\mathrm{s}$ , and $72\ ^{\circ}\mathbf{C}$ for $20\;\mathrm{s}$ .  

14. Purify each reaction using AMPure XP beads (steps 4–10 in this section) with the following exceptions. For step 4, add $75\,\upmu\mathrm{L}$ of vortexed beads, and for step 9, add $50\,\upmu\mathrm{L}$ of nuclease-free water to elute the sample.   
15. Set up the following PCR in $0.2\;\mathrm{mL}$ PCR tubes on ice for each purified PCR reaction, and the purified negative control reaction from the previous step [14]. Use a different indexed primer pair for each sample. The remaining PCR products from step 14 can be saved at $20\,^{\circ}\mathbf{C}$ .  

![](images/31bf6c3088859b414c837a2b0eb80bc4d57e9c89b54c47be4a6e2fffd18acba9.jpg)  

16. Run the program.  

a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  

b. 15 cycles of $98\ ^{\circ}\mathbf{C}$ for $10\;\mathrm{s}.$ ,$64\,^{\circ}\!C$ for $30{\mathrm{~s}}$ , and $72\ ^{\circ}\mathbf{C}$ for $20~\mathrm{s}$ .  

17. Purify each reaction using AMPure XP beads (steps 4–10 in this section) with the following exception, for step 9, add $15\,\upmu\mathrm{L}$ of nuclease-free water to elute the sample.  

18. Assess the quality of the libraries on a DNA ScreenTape (D1000 or similar). Load $1\,\upmu\mathrm{L}$ of sample onto the TapeStation (see Fig. 5).  

19. Quantify the concentration using the Qubit  dsDNA HS Assay Kit according to the manufacturer’s instructions. Typically, $1{-}2\,\upmu\mathrm{L}$ of library is sufficient for accurate quantification. At this point, the libraries can be pooled and run on an Illumina sequencer.  

# References  

1. Barbosa C, Peixeiro I, Romaão L (2013) Gene expression regulation by upstream open reading frames and human disease. PLoS Genet 9:e1003529 [PubMed: 23950723] 2. Zhang H, Wang Y, Lu J (2019) Function and evolution of upstream ORFs in eukaryotes. Trends Biochem Sci 44:782–794 [PubMed: 31003826] 3. Young SK, Wek RC (2016) Upstream open Reading frames differentially regulate gene-specific translation in the integrated stress response. J Biol Chem 291:16927–16935 [PubMed: 27358398]  

4. Lin Y, May GE, Kready H et al. (2019) Impacts of uORF codon identity and position on translation regulation. Nucleic Acids Res 2:1–10   
5. Hinnebusch AG, Ivanov IP, Sonenberg N (2016) Translational control by $5^{\prime}$ -untranslated regions of eukaryotic mRNAs. Science 352:1413–1416 [PubMed: 27313038]   
6. McGillivray P, Ault R, Pawashe M et al. (2018) A comprehensive catalog of predicted functional upstream open reading frames in humans. Nucleic Acids Res 46:3326–3338 [PubMed: 29562350]   
7. Melnikov A, Murugan A, Zhang X et al. (2012) Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay. Nat Biotechnol 30:271–277 [PubMed: 22371084]   
8. Sharon E, Kalma Y, Sharp A et al. (2012) Inferring gene regulatory logic from high-throughput measurements of thousands of systematically designed promoters. Nat Biotechnol 30:521–530 [PubMed: 22609971]   
9. Soemedi R, Cygan KJ, Rhine CL et al. (2017) Pathogenic variants that alter protein code often disrupt splicing. Nat Genet 49:848–855 [PubMed: 28416821]   
10. Wong MS, Kinney JB, Krainer AR (2018) Quantitative activity profile and context dependence of all human $5^{\prime}$ splice sites. Mol Cell 71:1012–1026.e3 [PubMed: 30174293]   
11. Adamson SI, Zhan L, Graveley BR (2018) Vex-seq: high-throughput identification of the impact of genetic variation on pre-mRNA splicing efficiency. Genome Biol 19:1–12 [PubMed: 29301551]   
12. Dvir S, Velten L, Sharon E et al. (2013) Deciphering the rules by which $5^{\prime}$ -UTR sequences affect protein expression in yeast. Proc Natl Acad Sci U S A 110:E2792–E2801 [PubMed: 23832786]   
13. Weingarten-Gabbay S, Elias-Kirma S, Nir R et al. (2016) Comparative genetics: systematic discovery of cap-independent translation sequences in human and viral genomes. Science 351:aad4939 [PubMed: 26816383]   
14. Cuperus JT, Groves B, Kuchina A et al. (2017) Deep learning of the regulatory grammar of yeast $5^{\prime}$ untranslated regions from 500,000 random sequences. Genome Res 27:2015–2024 [PubMed: 29097404]   
15. Sample PJ, Wang B, Reid DW et al. (2019) Human $5^{\prime}$ UTR design and variant effect prediction from a massively parallel translation assay. Nat Biotechnol 37:803–809 [PubMed: 31267113]  

![](images/7ec26086198e8c546cad9a7eca9f5fae2c808ae90253c0d4430247b28a46c4a2.jpg)  

# Fig. 1.  

Overview of FACS-uORF. The FACS-uORF assay system involves PCR amplification of an oligo library containing designer 5′ UTR templates for wild-type (uORF containing) and start-codon mutant (e.g., AUG $\longrightarrow.$ AAG) sequence pairs. The resulting products are cut and pasted into a plasmid background to generate a Massively Parallel Reporter library. Yeast transformed with this library are fluorescently sorted using FACS. After overnight outgrowth, reporter plasmids are recovered from each expression bin and used to generate high-throughput sequencing libraries for further analysis  

![](images/11a6d933986bd80d5a2f05397ab23d8536aff1d1697e496c11f27345bb4d0bf4.jpg)  

# Fig. 2.  

MPR Assay Design. (a) Upstream open reading frames (orange) are present in their native leader sequence (black). The $5^{\prime}$ end of the oligo contains the $3^{\prime}$ end of the ENO2 promotor sequence (blue), which initiates transcription at the bold and underlined “A” (red). The $3^{\prime}$ end of the oligo contains the $5^{\prime}$ end of the yellow fluorescent protein (YFP) sequence (yellow). PCR primers ENO2-lib-F1 and FACS-uORF-YFP-R anneal to the oligo library and contain AvrII and BglII restriction sites. (b) The vector, pGM01 contains an AvrII site in the ENO2 promotor (blue) and a $\mathrm{BglII}$ site in the downstream YFP gene (yellow). The digested oligo library is cloned into the vector using the AvrII and BglII sites so that the effect of the transcript leader containing the uORF on translation is measured by the levels of YFP. (c) The start codons of the uORFs are mutated to a nonfunctional start codon (AAG or AGG) for the mutant uORF, such that each uORF containing construct has a matched nonfunctional mutant uORF construct  

# MPR transformed S. cerevisiae FACS scatter plot  

![](images/b1618488447eb131c0877d374f6f8e2e2e2ba24df744b56aa64b76f49a37fbbc.jpg)  

#  

Fig. 3. FACS of S. cerevisiae transformed with the FACS-uORF MPRA library (YFP/mCherry sorting). The ratios of YFP to mCherry are calculated and divided into nine bins ranging from high YFP to mCherry values (pink mask) to low (no mask). The cells are also gated restricting the range of mCherry values (white box), avoiding large cells with high mCherry values and cells which may have lost the vector (low or negative mCherry values)  

# FACS-uORF replicates  

![](images/792a959c28de876c43bceee3d5672fefc009ebcffb67077ca4a70b051b1b43e2.jpg)  

# Fig. 4.  

Typical YFP–mCherry ratios from replicate FACS-uORF sorted cell bins after overnight growth. The ratios were read with a Tecan Fluorometer. These measurements are used to verify consistent sorting across replicates and can be used for further normalization and data analysis after high-throughput sequencing  

![](images/d7e44a3592d558cf8f7efcaaa6fe7fa025ebf9e0bde7157e29bcbfad87cd5c6e.jpg)  
Fig. 5. TapeStation DNA ScreenTape gel image showing FACS-uORF libraries made from bin 0 (unsorted), and bins 1–9. The DNA ladder (leftmost lane) ranges from 25 bp to 1500 bp. Two control markers (in green and purple) at 25 and 1500 bp in every lane are used as internal sizing controls. A PCR negative (rightmost lane) carried through during the entire library preparation procedure (see methods) and does not contain amplified product. The typical library size shown ranges from approximately 200 bp to 350 bp. Occasionally, a minimal amount of small sized PCR products are present due to the amplification of primer dimers (bin 6 at ${\sim}40\,\mathrm{bp}$ ). These small products do not interfere with library sequencing  

# Table 1  

Custom DNA oligo primers   

![](images/775f712aab3a6db633ae42b3e0c02f9698f8166e2325b6eb5bee2c2a4306da48.jpg)  