# Ribosomal leaky scanning through a translated uORF requires eIF4G2  

Victoria V. Smirnova 1,†, Ekaterina D. Shestakova $\mathbb{Q}_{2,\dag}$ , Daria S. Nogina 2, Polina A. Mishchenko 2, Tatiana A. Prikazchikova3, Timofei S. Zatsepin 3,4, Ivan V. Kulakovskiy 5,6,7, Ivan N. Shatsky 1,\* and Ilya M. Terenin 1,8,\*  

1Belozersky Institute of Physico-Chemical Biology, Lomonosov Moscow State University, Moscow 119234, Russia, 2Faculty of Bioengineering and Bioinformatics, Lomonosov Moscow State University, Moscow 119234, Russia, 3Skolkovo Institute of Science and Technology, Skolkovo, Moscow 121205, Russia, 4Chemistry Department, Lomonosov Moscow State University, Moscow 119991, Russia, Center for Precision Genome Editing and Genetic Technologies for Biomedicine, Engelhardt Institute of Molecular Biology, Russian Academy of Sciences, Moscow 119991, Russia, Institute of Protein Research, Russian Academy of Sciences, Pushchino 142290, Russia, Vavilov Institute of General Genetics, Russian Academy of Sciences, Moscow 119991, Russia and Sirius University of Science and Technology, Sochi, Olimpiyskiy ave. b.1, 354349, Russia  

Received July 05, 2021; Revised December 07, 2021; Editorial Decision December 15, 2021; Accepted December 18, 2021  

# ABSTRACT  

eIF4G2 (DAP5 or Nat1) is a homologue of the canonical translation initiation factor eIF4G1 in higher eukaryotes but its function remains poorly understood. Unlike eIF4G1, eIF4G2 does not interact with the cap-binding protein eIF4E and is believed to drive translation under stress when eIF4E activity is impaired. Here, we show that eIF4G2 operates under normal conditions as well and promotes scanning downstream of the eIF4G1-mediated 40S recruitment and cap-proximal scanning. Specifically, eIF4G2 facilitates leaky scanning for a subset of mRNAs. Apparently, eIF4G2 replaces eIF4G1 during scanning of $\mathsf{\pmb{5}^{\prime}}$ UTR and the necessity for eIF4G2 only arises when eIF4G1 dissociates from the scanning complex. In particular, this event can occur when the leaky scanning complexes interfere with initiating or elongating 80S ribosomes within a translated uORF. This mechanism is therefore crucial for higher eukaryotes which are known to have long $\mathsf{\pmb{5}^{\prime}}$ UTRs with highly frequent uORFs. We suggest that uORFs are not the only obstacle on the way of scanning complexes towards the main start codon, because certain eIF4G2 mRNA targets lack uORF(s). Thus, higher eukaryotes possess two distinct scanning complexes: the principal one that binds mRNA and initiates scanning, and the accessory one that rescues scanning when the former fails.  

# INTRODUCTION  

As the penultimate step of gene expression, translation plays an important role in providing the qualitative and quantitative diversity of the proteome. At this stage, a cell determines the nature and amount of proteins needed at the current moment of cell cycle based on signals from multiple regulatory pathways, which preferentially affect the translation initiation step. In eukaryotes, all cytoplasmic mRNAs have the $\mathrm{m^{7}G}$ -cap on their $5^{\prime}$ -ends and use the $5^{\prime}$ -end dependent scanning mechanism proposed by Kozak (1–3) to locate their start codons. According to this model, the $\mathrm{m^{7}G}$ -cap is recognized by the trimeric factor eIF4F, consisting of eIF4E (cap-binding subunit), the scaffold protein eIF4G1 and the helicase eIF4A. eIF4G1, via its interaction with eIF3, attracts the 43S preinitiation complex, and thereby forms a ribosomal scanning complex. Then the 43S complex moves along the $5^{\prime}$ UTR of mRNA and scans for an AUG (or near-AUG) start codon. The context of the AUG determines if the ribosome initiates translation or bypasses the unfavourable start codon and continues scanning. Whether cellular mRNAs can also use the mechanism of internal initiation is still a matter of debate: despite the copious literature on cellular IRESs, none of them have been appropriately validated in the way genuine IRESes of viral origin were dissected (4,5).  

While a ribosome can scan an mRNA by itself (6–8), its intrinsic mRNA unwinding capacity is very limited. Accordingly, numerous dedicated helicases are implicated in translation initiation (9). The major scanning helicase eIF4A is not processive when unassisted, but its activity is strongly stimulated by eIF4G1 (10,11). Higher eukaryotes contain numerous homologues of eIF4G1. The closest among them is eIF4G2 (also known as DAP5, Nat1 and p97), which was discovered simultaneously in four laboratories back in 1997 (12–15). Unlike eIF4G1, eIF4G2 binds eIF2 (16–19), bears only one eIF4A-binding site (while eIF4G1 bears two), and does not interact with eIF4E or PABP (12–15). This protein is one of the most evolutionary conserved initiation factors in multicellular organisms. It is apparently not crucial for bulk translation, but the eIF4G2-deficient embryonic cells fail to differentiate properly (18,20,21). Several attempts to identify eIF4G2 mRNA targets genome-wide have been made (17,18,21). However, none of them has given a clue to the mechanics of eIF4G2. The majority of reports claim that the protein drives internal translation initiation (16,21–29), while a few others point to a specific cap-dependent translation initiation mechanism when the cap-recognizing factor eIF4E is dispensable (17,30). However, the protein has also been suggested to participate in conventional cap-dependent translation (19,31). Thus, it is still unclear whether the only role of eIF4G2 is to functionally replace eIF4G1 under specific conditions or whether it can also assist the latter.  

Here, we analysed various ribosome profiling data from the eIF4G2-deficient cells and arbitrarily chose several mRNAs that exhibited decreased ribosome coverage in the knockout cells for further analysis. All selected mRNAs were directly validated for their eIF4G2-dependencies. We confirm that, on these particular mRNAs, it is eIF4F that promotes initial ribosome binding and cap-proximal scanning, while eIF4G2 only comes into play at downstream sequences. It is noteworthy that many mRNAs that require eIF4G2 for their translation bear upstream open reading frames (uORFs), which are arguably the major controller of ribosomal scanning (32,33) and therefore, we focused on this particular element of $5^{\prime}$ UTRs.  

The data presented here show that, on several studied mRNAs, eIF4G2 is required for the 43S scanning complexes that pass through uORFs, i.e., eIF4G2 promotes leaky scanning. We demonstrate that eIF4G2 maintains ribosome scanning competence by assuming the role of eIF4G1 when the latter presumably dissociates from scanning complexes. On the studied mRNAs, this occurs when the scanning 40S ribosomes interfere with the initiating or elongating 80S ribosomes within an uORF.  

# MATERIALS AND METHODS  

# Antibodies  

Antibodies against eIF4G1 (A300-502A), eIF4G2 (A302- 249A), eIF3b/EIF3S9 (A301-761A), eIF3d/EIF3S7 (A301-758A), GAPDH (A300-639A) were purchased from Bethyl laboratories. Anti-phospho-eIF2' (ab32157) were from Abcam. Anti-eIF4E-BP1 (9452) and antiphospho-eIF4E-BP1 (Thr37/46) (2855) were from Cell Signaling Technology. HRP-conjugated secondary antibodies were from Invitrogen (anti-rabbit 31460, anti-mouse 31431).  

# Plasmids  

The human ${\upbeta}$ -actin, CCNI, eIF4G2, murine Map3k3, Pcbp2, human $\mathrm{TNF}\alpha$ , TP53 (31), APAF1 (34), ATF4, IFRD1, UCP2 (35), leaderless (36), rabbit ${\upbeta}$ -globin, HIV1 (37), L1JK (38) reporter plasmids were described previously. The human AKT2, ARRDC4, ATF3, ATF5, BCL2, CDK1, CES2, eIF5, EPAS1, HERC1, hnRNPK (both variants), HSPA2, c-jun, PCBP1, PKR, PPFIA4, SMAD1, $\mathrm{TGF}\upbeta1$ , TNRC6C, UHMK1 $5^{\prime}$ UTRs were amplified from cDNA obtained from $293\mathrm{T}$ cells. The PCPB1, AKT2 and SMAD1 $5^{\prime}$ UTR amplification required a partial dGTP substitution for 7-deaza-dGTP (dGTP:7-deaza-dGTP 3:1) (NEB, N0445). The murine Maf1 and Stard7 were amplified from Hepa1-6 cells-derived cDNA. The BCL2 $5^{\prime}$ UTR variants derived from the P1 and M promotors were accidentally amplified in two variants each, with or without the alternatively spliced intron. The exact $5^{\prime}$ boundaries of the $5^{\prime}$ UTRs were chosen on the basis of CAGE data available from Zenbu genome browser (39) and are provided in Supplementary Table S1C, along with all other $5^{\prime}$ UTRs used throughout the study. The human CFTR1, MDM2 and TUBA1B plasmids (40) were a kind gift from S. Dmitriev and K. Lashkevich (Lomonosov Moscow State University, Russia). The plasmid bearing the SARS-CoV2 $5^{\prime}$ UTR sequence was a kind gift from A. Anisenko (Lomonosov Moscow State University, Russia). All studied $5^{\prime}$ UTRs were cloned upstream of firefly luciferase (Fluc) into pGL3 vector (Promega), except CCNI, PPFIA4 and UHMK1 $5^{\prime}$ UTRs that were cloned into pNL1.1 vector (Promega) and thus code for NanoLuc (Nluc). All plasmids were created via conventional cloning strategies and the results have been confirmed by sequencing (Evrogen, Moscow, Russia). The pTE4396 plasmid (41) used for the AsCpf1-mediated eIF4G3 knockout was a gift from Ervin Welker (Addgene plasmid # 74041). Oligonucleotides used for cloning are disclosed (Supplemental Table S1A).  

# Western blotting  

Nitrocellulose membranes were blocked in $3\%$ ECL™ Blocking Agent (GE Healthcare, RPN418) in TBST at room temperature for $1\,\mathrm{~h~}$ , then probed with antibodies against eIF4G1 (1:5000), eIF3b (1:5000), eIF3d (1:5000), GADPH (1:5000), eIF4E-BP1 (1:1000), phosphorylated $\mathrm{eIF}2\upalpha$ (1:500) and detected by chemiluminescence using corresponding anti-rabbit or anti-mouse antibodies at 1:25000 dilution. Incubation with primary and secondary antibodies was also performed in $3\%$ blocking reagent in TBST under the same conditions. Antibodies bound to eIF4G2 and GADPH were visualized with an enhanced chemiluminescence detection kit $\mathbf{\langleECL^{\intercalM}}$ Prime Western Blotting System, GE Healthcare, RPN2232). eIF4G1, eIF3b, eIF4E-BP1, phosphorylated 4E-BP1 and phoshorylated $\mathrm{eIF}2\upalpha$ were detected with SuperSignal™West Femto Maximum Sensitivity Substrate (Thermo Fisher Scientific, 34095). Note that for phospho-eIF4E-BP1 detection, the membrane was blocked in $5\%$ BSA in TBST at room temperature for $1\,\mathrm{~h~}$ , incubation with primary antibodies (1:2000) was performed overnight at $4^{\circ}\mathrm{C}$ in $5\%$ BSA/TBST, corresponding secondary antibodies (1:25 000) were also diluted in $5\%$ BSA/TBST and visualization was performed with $\mathrm{ECL^{\intercalM}}$ Prime Western Blotting System (GE Healthcare). The images were captured using ChemiDoc ${\mathrm{XRS+}}$ with Image Lab™3.0 software for image processing and quantification (Bio-Rad).  

# In vitro transcription  

The matrices for transcription were prepared by PCR using primers RV3L (forward) and FLA50 or GL3r (reverse) for polyadenylated (50 nucleotide-long poly(A) tail) or nonpolyadenylated mRNAs, respectively (see Supplementary Table S1A). Amplification of the PCBP1 matrix required supplementation with 7-deaza-dGTP (dGTP:7- deaza-dGTP 3:1) (NEB, N0445). The PCR products were purified using Wizard SV Gel and PCR Clean-Up System (Promega) or Monarch  PCR & DNA Cleanup Kit $(5\,\upmu\mathrm{g})$ (NEB), additionally deproteinized via phenol–chloroform extraction, precipitated with ethanol and sodium acetate, washed with $70\%$ ethanol, dried and then dissolved in nuclease-free water (Promega, P1193). The transcription was performed in $0.5\;\mathrm{ml}$ DNA LoBind tubes (Eppendorf) using T7 RiboMAX™Large Scale RNA Production System (Promega, P1300) as suggested by the manufacturer. $15\,\upmu\mathrm{l}$ mixture contained homemade buffer (final concentrations were $80\;\mathrm{mM}$ HEPES-KOH pH 7.5, $24\;\mathrm{mM}$ $\mathrm{MgCl}_{2}$ , 2 $\mathrm{mM}$ spermidine, $40\;\mathrm{mM}$ DTT), $7.5\;\mathrm{mM}$ of ATP, CTP and UTP, $0.75~\mathrm{mM}$ GTP, $3~\mathrm{mM}$ ARCA (NEB, S1411) or Acap (NEB, S1406) analogues, $1.5\;\upmu\mathrm{l}$ T7 Enzyme mix and 1 $\upmu\mathrm{g}$ DNA template. The reactions were conducted for $2\,\mathrm{{h}}$ at $37^{\circ}\mathbf{C}$ . Then $35~\upmu\mathrm{l}$ nuclease-free water and $30~\upmu\mathrm{l}$ RNA precipitation solution ( $7.5\,\mathrm{M}$ lithium chloride, $50\,\mathrm{mM}$ EDTA) were added and tubes were left on ice for $1-2\,\mathrm{{h}}$ . RNAs were pelleted by centrifugation at $20~000~\mathrm{g}$ for $15~\mathrm{min}$ at $4^{\circ}\mathrm{C}$ washed with $70\%$ ethanol, dried, dissolved in nuclease-free water and stored at $-80^{\circ}\mathrm{C}$ . Concentration was determined by measuring absorbance at 260 and $280\ \mathrm{nm}$ (Eppendorf BioSpectrometer $\circledast$ basic).  

# Transfections  

All cells were cultured under standard conditions in DMEM (PanEco) supplemented with $10\%$ FBS (HyClone, SV30160.03). The wild type and $\mathrm{eIF4G}2^{(-)-)}$ NIH/3T3 cells (31) were plated in 48-well plate at density $20\times\mathrm{10^{3}\ c e l l s}/\mathrm{cm}^{2}$ for ${\sim}20\,\mathrm{h}$ prior to mRNA transfection. This quantification was necessary because the knockout and wild type cells show slightly but noticeably different doubling times. The ratios of translation efficiencies do not vary with cells plated at densities from $5\ \times\ 10^{3}$ to $40~\times~1\dot{0}^{3}$ cells per $\dot{\mathrm{cm}}^{2}$ (data not shown). The eIF4G1, eIF4G2 or eIF3d knockdowns in 293T, Huh7 or RKO $\mathrm{eIF4}\mathrm{G}3^{(-/-)}$ cells were performed as follows. On day 1, cells were plated in a 4-, 12- or 6-well plate (depending on the number of experimental points) at ${\sim}20\%$ density simultaneously with the first round of siRNA transfection. The siRNAs were transfected using either Lipofectamine™ RNAiMAX (Invitrogen, 13778075) as suggested by the manufacturer or homemade lipid particles (see below) with similar results, yet the use of the latter resulted in a noticeably more consistent data. The concentration of siRNAs was $10\,\mathrm{\nM}$ in culture medium. On day 3, the cells were replated to a 48-well plate at density ${\sim}30\%$ simultaneously with the second round of siRNA transfection. On day 4 $\sim\!72\mathrm{~h~}$ after the first siRNA application), mRNA transfection was performed. For a well of 48-well plate, $50\,\mathrm{~ng~}$ of reporter mRNA were mixed with $5\,\mathrm{\ng}$ of reference mRNA (in vitro transcribed $\mathrm{m^{7}G}$ -capped and polyadenylated ${\upbeta}$ -globin-Fluc or $\upbeta$ -globin-Nluc, as appropriate) in $20~\upmu\mathrm{l}$ Opti-MEM™(Gibco, 31985062). 0.4 $\upmu\mathrm{l}$ Lipofectamine 3000 reagent (Invitrogen) were diluted in $10~\upmu\mathrm{l}$ Opti-MEM™. The volumes were multiplied in accordance with the required number of experimental points. Then mRNA mixture was added to the transfection reagent solution, incubated for $5\:\mathrm{min}$ at room temperature and then applied to the cells. Where indicated, cells were treated with $1~\upmu\mathrm{M}$ PP242 (Tocris Bioscience, 4257) or $40~\upmu\mathrm{M}$ sodium arsenite (Sigma-Aldrich) $10\ \mathrm{min}$ prior to transfection. Three (two in case of the PP242 or sodium arsenite treatments) hours later, the cells were lysed using the Luciferase Cell Culture Lysis Reagent (Promega, E1531) according to the manufacturer’s protocol and luciferases’ activities were measured manually using the Dual-Luciferase® Reporter Assay System (Promega, E1980) with Modulus luminometer (Turner Biosystems).  

For the AsCpf1-mediated eIF4G3 knockout RKO cells were transfected with pTE4396 plasmid, the cells were grown for four days in the presence of $500~\upmu\mathrm{g/ml}$ G418, then cloned and analysed. The frame-shift deletion was confirmed by sequencing (Supplementary Figure S8G). The resulting protein consists of the $212~\mathrm{N}$ -terminal aminoacids of the eIF4G3 and 53 out-of-frame aminoacids.  

# siRNAs  

The duplexes for the eIF4G2 knockdown have been described (31), see Supplementary Table S1B (capital letters stand for the unmodified ribonucleotides and lowercase denote the $2^{\prime}–O$ -Me protected ribonucleotides). Chemically modified siRNAs were designed as 21-nucleotide dsRNA molecules with $3^{\prime}$ -overhangs of two dT nucleotides. Sequences of siRNAs targeting human eIF4G1 and eIF3d mRNAs were selected as previously described (42). Briefly, the mRNA sequence was cut into 19-mer sequences with one-nucleotide shifts. All 19-mer candidate sequences were evaluated by two criteria: high on-target potential (the siRNA efficacy) and low off-target potential (the siRNA specificity). To avoid off-target activity 19-mer candidates were aligned against the RefSeq mRNA database and their off-target binding capabilities were estimated based on the number of mismatches in the seed region, in the nonseed region and in the cleavage site position. The candidates with low off-target activity were further filtered according to the potential efficacy calculated based on several known criteria (43–45). The obtained sequences were additionally filtered to avoid known miRNA motifs and immune stimulatory sequence motifs. Finally, the $2^{\prime}$ -O-Me modification of pyrimidine nucleotides and single $3^{\prime}$ -internucleotide phosphorothioates linkage were introduced into sequences of the most preferable candidates to further reduce immune response, off-target effects and to increase stability against nucleases (46). All synthesized siRNAs were then directly assayed for their efficiencies by qPCR and western blotting and selected for further use based on their knockdown efficiencies (not shown). The knockdown efficiencies were estimated via western blotting and found to be 10–20 for eIF4G1 and eIF4G2, and $3-$ 4 for eIF3d. All oligoribonucleotides were synthesized inhouse using standard protocols for MerMade-12 synthesizer (BioAutomation Technologies) with $2^{\prime}$ -TBDMS/ $\mathbf{\mathcal{Z}}^{\bullet}$ -OMe phosphoramidites, purified by ion-exchange HPLC and verified by ESI-MS. For the formation of the siRNA duplexes equal amounts of the complementary oligoribonucleotides (5 nmol in $100\,\upmu\mathrm{l}$ of $10\,\mathrm{mM}$ TE buffer) were mixed together, heated to $90^{\circ}\mathbf{C}$ , cooled to room temperature and stored at $4^{\circ}\mathbf{C}.$ The lipid particles for RNA interference were prepared as described (47). The solutions of siRNA duplex in water and of lipids in ethanol (C12-200, 1,2-distearoyl$\boldsymbol{s n}$ -glycero-3-phosphocholine (DSPC, Avanti Polar Lipids), cholesterol (Sigma), C14 PEG 2000 (Avanti Polar Lipids) at a 50:10:38.5:1.5 molar ratio) were mixed in a microfluidic chip device. Then the LNPs were dialyzed overnight against PBS. The LNP dimensions were measured via dynamic light scattering (ZetaSizer, Malvern Instruments). The estimated diameter of the particles was 120–150 nm.  

# Statistical analysis  

The data are plotted as boxes with Tukey-style whiskers for all mRNA transfection experiments except those in Supplementary Figures S10A and S10B, where mean $\pm\ S\mathrm{E}$ is shown for the sake of readability. All the transfections have been replicated at least five times. The statistical significance is determined by two-tailed Mann–Whitney $U$ test as indicated in the article. All analyses have been performed using GraphPad Prism 7. Outliers were excluded from the plots (but not from analyses), except Supplementary Figure S4.  

# Ribosome profiling  

The wild type and $(\mathrm{eIF4G}2^{-/-1}$ ) NIH/3T3 cells were grown in DMEM (PanEco) supplemented with $10\%$ FBS (HyClone, SV30160.03). Twelve $150\;\mathrm{mm}$ cell culture dishes per experimental point were grown (i.e. 12 dishes of the wild type cells and 12 dishes of the cells with eIF4G2 knocked out). The medium was aspirated, the plates were put on ice, cells were washed with ice-cold PBS supplemented with $100~\upmu\mathrm{g/ml}$ cycloheximide (PBS-C), scraped in $0.5\;\mathrm{ml}$ PBSC per plate, combined, collected by centrifugation $50\;\mathrm{g},\,5^{\prime}$ ,$4^{\circ}\mathrm{C})$ ), resuspended in $1.5\,\mathrm{\ml}$ lysis buffer $20\,\mathrm{\mM}$ Tris– HCl $\mathrm{pH}\,7.5$ ,$250\;\mathrm{mM}$ NaCl, $1,5\;\mathrm{mM}\;\mathrm{MgCl}_{2}$ ,$1~\mathrm{mM}$ DTT, $0{,}5\%$ Triton X-100, $100\ \upmu\mathrm{g/ml}$ cycloheximide and $20\;\mathrm{U/ml}$ TURBO™DNase (Invitrogen, AM2238)), and incubated $10\ \mathrm{min}$ on ice. Debris was removed by centrifugation (20 $000\;\mathrm{g},\,10^{\circ},4^{\circ}\mathrm{C})$ . The supernatant was split as follows: 1/11 for polysomes validation, $4/11$ for total RNA isolation and $6/11$ for ribosome footprints generation.  

100U of RNase I (Ambion, AM2294) per $3.14\,\mathrm{AU}_{260}$ of the lysates were added and the mixtures were incubated at $23^{\circ}\mathbf{C}$ for $50\ \mathrm{min}$ in a thermo shaker $(400\ \mathrm{rpm})$ . Then, 2 $\upmu\mathrm{l}$ Superase In (Ambion, AM2694) per $1~{\upmu\mathrm{l}}$ of RNase I were added, the samples were centrifuged (3 min, $20\;000\mathrm{g}$ ,$4^{\circ}\mathrm{C}_{\prime}$ ) the supernatants were loaded onto $10–50\%$ sucrose gradient in SW41 tubes (contained $20~\mathrm{mM}$ Tris–HCl pH 7.5, $250~\mathrm{mM}$ NaCl, $15\ \mathrm{mM\MgCl}_{2}$ ,$1~\mathrm{mM}$ DTT and 100 $\upmu\mathrm{g}/\mathrm{ml}$ cycloheximide). The centrifugations were run for $3\,\mathrm{~h~}$ at $35\ 000\ \mathrm{rpm}$ at $4^{\circ}\mathrm{C}$ (SW41 rotor, Beckman Coulter). Then the gradients were manually fractionated and the $A_{260}$ of the fractions was measured in 96-well UVtransparent flat bottom plates (Corning, 3635) using Infinite F200 Pro plate reader (Tecan). The four fractions that represented 80S peak were combined, supplemented with $10\;\mathrm{{mM}}$ EDTA, and deproteinized twice using homemade acid phenol/chloroform/isopentanol $\mathrm{pH}\,4.5$ extraction. The RNA was precipitated overnight at $-30^{\circ}\mathbf{C}$ using $1/10\ \mathrm{v/v}$ 3M NaOAc $\mathrm{pH}\,5.1)$ and $1\ \mathrm{v/v}$ isopropanol.  

The lysates for total RNA extraction were split into two $1.5\;\mathrm{ml}$ tubes, $1200~\upmu\mathrm{l}$ Trizol-LS (Invitrogen, 10296028) per tube were added, gently mixed and left on a bench for $5\,\mathrm{{min}}$ .Then $320~\upmu\mathrm{l}$ chloroform per tube were added, shaken vigorously for $15\mathrm{~s~}$ by hand and left for $15~\mathrm{min}$ on a bench. The phases were separated via centrifugation ( $12\,000\,\mathrm{g},15^{\prime}$ ,$4^{\circ}\mathrm{C})$ ). An equal volume of isopropanol was added, mixed well, left on a bench for $15\;\mathrm{min}$ . The RNA pellet was collected via centrifugation at $20\ 000\ \mathrm{g}$ for $15~\mathrm{min}$ at room temperature. The pellet was washed with $80\%$ ethanol. The dried RNA pellet was dissolved in $100~\upmu\mathrm{l}$ nuclease-free water (Promega). $250~\upmu\mathrm{g}$ total RNA were further purified using Oligotex suspension (QIAGEN, 79000) as suggested by the manufacturer. However, to achieve a higher purity, another binding-elution cycle was performed (using the same resin that was used in the first purification cycle). Approximately $4~\upmu\mathrm{g}$ of poly(A)-containing mRNAs were obtained at this step. The RNA was fragmented for $1\,\mathrm{~h~}$ at $95\mathrm{{^C}C}$ using an alkaline buffer $\mathrm{\Delta50~mM~Na_{2}C O_{3}/N a H C O_{3}~p H}$ 9.2, 1 mM EDTA) then precipitated overnight with sodium acetate and isopropanol in a freezer at $-20^{\circ}\mathbf{C}.$ . The samples were then dissolved in nuclease-free water and run through a $15\%$ polyacrylamide gel in TBE buffer. The $28-$ 34 nucleotide long RNA fragments were excised, sliced and eluted overnight $0.1\%$ SDS, $1~\mathrm{mM}$ EDTA, 0.3 M NaOAc, $\mathrm{pH}\,5.1\$ .  

The RNA fragments were dephosphorylated with T4 PNK (NEB, M0201S), precipitated with isopropanol, ligated to preadenylated Universal miRNA cloning linker (NEB, S1315S) using truncated T4 RNA ligase 2 (NEB, M0242S). The ligated RNA was precipitated with isopropanol, purified from $15\%$ PAGE and subjected to a reverse transcription using SuperScript™III reverse transcriptase (Invitrogen, 18080093). The $20~\upmu\mathrm{l}$ reactions were performed for $30\ \mathrm{min}$ at $48^{\circ}\mathbf{C}$ The remaining RNA was hydrolysed by addition of $2.2~\upmu\mathrm{l}$ 1M $\mathrm{NaOH}$ and subsequent incubation for $20\ \mathrm{min}$ at $98^{\circ}\mathbf{C}$ . The reaction mixture was then diluted up to $300\,\upmu\mathrm{l}$ and the cDNA was precipitated using isopropanol followed by $7.5\%$ PAGE. The excised DNA was eluted, precipitated and dissolved in $16.5~\upmu\mathrm{l}$ nucleasefree water (Promega). Circularisation was executed for 2 h at $60^{\circ}\mathbf{C}$ using CircLigase™II (Epicentre, CL9021K) in a $20~\upmu\mathrm{l}$ reaction that contained CircLigase™II reaction buffer, $2.5\;\mathrm{mM}\;\mathrm{MnCl}_{2}$ and $100\;\mathrm{U}$ CircLigase™II. rRNAderived contaminations were removed using complementary biotinylated oligonucleotides (48). The resulting libraries were amplificated using Phusion® High-Fidelity DNA polymerase (NEB, M0530L) and NEBNext  Multiplex Oligos for Illumina $\circledast$ (NEB), and the sequencing procedure was accomplished on an Illumina HiSeq 2000 NGS platform following standard Illumina protocols.  

# Analysis of ribosome profiling data  

Basic quality control of the reads was performed with FastQC v0.11.8 (http://www.bioinformatics.babraham.ac. uk/projects/fastqc). Adapters and low-quality bases were removed with cutadapt v2.1 (49) (Ribo-Seq: cutadapt - a AGATCGGAAGAG -j 4 –minimum-length 20 -q 20 – trimmed-only; RNA-Seq: cutadapt -a AGATCGGAAGAG $-j$ 4 –minimum-length 20 -q 20). The trimmed sequences were mapped to the mm38 genome assembly (50) with GENCODE M22 genome annotation using STAR v2.7.1a (51) with the default parameters, which also provided gene-level read counts. The arrangement of the samples according to the principal component analysis is shown in Supplementary Figure S1C using the first two principal components. A summary of the sequencing data is presented in Supplementary Table S2.  

The differential gene expression and gene set enrichment analyses were performed in the R environment. Genes passing 1 count-per-million in all samples were used for analysis of the ribosome occupancy with DESeq2 (52). Read counts per gene were normalized with the default DESeq2 relative log expression (RLE) method.  

The ribosome occupancy (RO, Ribo-Seq footprint counts relative to the RNA-Seq read counts) for particular cell types and the differential ribosome occupancy between the cell types for Sugiyama et al. data (18) was estimated using DESeq2. $P$ -values were corrected for multiple testing using the Benjamini-Hochberg procedure (FDR). As there were no replicates in the Ribo-Seq 1 and 2 data sets generated in this study, for these data we performed only basic analysis of the ribosome occupancy and the differential ribosome occupancy by calculating the log ratios upon RLE normalization. We did not consider the two datasets generated in this study as replicates because they significantly differ from each other. This probably happened because the cells were processed at different passages and the ability of the NIH/3T3 eIF4G $2^{(-/-)}$ cells to adapt to the lack of eIF4G2 is documented (31). The data are deposited in NCBI GEO (GSE158136).  

# RESULTS  

# ${\sf5^{\prime}}$ UTRs determine the eIF4G2-dependence of translation  

Elucidating the mechanisms used by eIF4G2 to promote translation on eIF4G2-dependent mRNAs requires experiments with individual model mRNA reporters. To this end, we compared the ribosome profiling data of the cells where the eIF4G2 gene was knocked out as compared to the wild type cells: mES (18) and NIH/3T3 (this study). Several genes were arbitrarily picked for further investigation based on the reduced ribosomal occupancy in the knocked out cells (Supplementary Figure S1). They included Maf1 (a regulator of RNA Pol III), Stard7 (a phosphatidylserine transporter), Pcbp2 (an abundant multifunctional RNAbinding protein) (31), CCNI (a cyclin with unclear functions) and the already reported Map3k3 (18). The $5^{\prime}$ UTRs of these mRNAs vary in length, GC-content and size and number of upstream reading frames (uORFs), i.e., share little in common at first glance (Supplementary Table S1C).  

We cloned the corresponding $5^{\prime}$ UTRs upstream of either Fluc or Nluc coding sequences, prepared the $\mathrm{m^{7}G}$ -capped and polyadenylated mRNAs via in vitro transcription and transfected them into NIH/3T3 $\mathrm{eIF4G}2^{-/-}$ (Supplemental Figure S2A), in 293T (Figure 1A) and Huh7 cells (Figure 1B) where eIF4G2 was depleted by means of RNAimediated knockdown. Indeed, translation of the Maf1, Map3k3, Pcbp2, Stard7 reporters markedly dropped upon both knockout and knockdown. A dozen reporters available in the lab were also tested as control. Among them, the PPFIA4 (a tyrosine phosphatase) $5^{\prime}$ UTR was also found to provide its reporter with eIF4G2-dependence. Interestingly, the human BCL2 (four $5^{\prime}$ UTR variants derived from the P1 and M promoters with or without the alternatively spliced intron) and APAF1 mRNAs that were suggested to be eIF4G2 targets on the basis of artifact-prone DNA transfection or translation in RRL (16,23,27,29) also show the need for eIF4G2 in our mRNA-centered approach (Figure 1A, B and Supplementary Figure S2B). As the BCL2 $5^{\prime}$ UTR supposedly bears a G-quadruplex (53), which could determine eIF4G2-dependence, we mutated it but this did not affect the eIF4G2 knockdown response (Supplementary Figure S2B). Overall, we show that using mRNA reporters bearing the $5^{\prime}$ UTRs of eIF4G2-dependent mRNAs (i.e. APAF1, BCL2, CCNI, Maf1, Map3k3, Pcbp2, PPFIA4 and Stard7) in combination with siRNA-mediated eIF4G2 knockdown is a feasible approach to study eIF4G2. Using knockout cells is slightly more technically difficult and, importantly, the knockout cells may undergo adaptation which could lead to confusing results (31). Then we proceeded to analyse data to find the feature that makes translation of the selected mRNAs dependent on eIF4G2.  

# The eIF4G2-dependent reporter mRNAs employ a cap- and eIF4E-mediated mode of ribosome recruitment  

The vast majority of previously published reports suggest that eIF4G2 drives cap-independent translation. To determine if this is the case for the eIF4G2 targets described above, we first studied the contribution of the $\mathrm{m^{7}G}$ -cap to their translation. Remarkably, in 293T cells, all the reporter mRNAs showed significantly high cap-dependencies (Supplementary Table S3) that considerably exceeded their eIF4G2-dependencies (compare to Figure 1A, B and Supplementary Figure S2A, B). Therefore, these particular mRNAs employ cap-dependent translation that is somehow promoted by eIF4G2. Likewise, eIF4G2 knockdown does not affect the response of the eIF4G2 targets to eIF4E inactivation due to PP242-mediated mTOR inhibition (Figure 2A). In other words, eIF4E inactivation does not increase eIF4G2′s contribution to the translation of its targets (Figure 2B and Supplementary Figure S3). If eIF4G2 only participated in cap-independent translation initiation, then its impact (and, therefore, the effect of knocking down eIF4G2) should have increased when cap-dependent translation was inhibited by PP242. Furthermore, the eIF4G2- dependencies do not correlate with the sensitivities to eIF4E inhibition. Some of the eIF4G2 targets (e.g. Map3k3, Stard7) are as sensitive to the PP242 treatment as the ${\upbeta}$ -globin reporter mRNA, while some others (PPFIA4 and Maf1) respond only slightly, similarly to the previously reported cases of leaderless mRNA (36), APAF1 (34) and the artificial CITE L1JK (38) that tolerate inactivating of eIF4E. These observations show that under normal conditions eIF4G2 most likely participates in the cap- and eIF4Edependent translation downstream of cap recognition. Why the translation of certain mRNAs (e.g. PPFIA4 and Maf1) responds poorly to mTOR inactivation is not known and it can be argued that, on such mRNAs, eIF4G2 participates in cap recognition. Data presented for these particular mRNAs in subsequent sections do not support this hypothesis, nor do they totally exclude it.  

![](images/67f4b0bd69d3bfb95a0ebc5e90dcb8f6be9ed85988c6e93ae2c4530e6f4c06ea.jpg)  
Figure 1. Validation of the eIF4G2 translational targets. (A) Results of in vitro transcribed $\mathfrak{m}^{7}\mathbf{G}.$ -capped and polyadenylated reporters’ transfection into 293T cells treated with either control or anti-eIF4G2 siRNAs. Unlike the other figures, this one incorporates data from all relevant experiments. All reporters were co-transfected with the reference ${\upbeta}$ -globin reporter mRNA coding for Fluc or Nluc, as appropriate. The data are presented as ratios of normalized reporter expression in cells with eIF4G2 knocked down to control cells. For all eIF4G2 targets, ${\upbeta}$ -globin and CDK1 reporters the number of replicates exceeds 20. For the non-targets $n\geq5.$ (B) Western blot analysis of the eIF4G2 knockdown in 293T cells, GAPDH as a loading control. (C) Results of mRNA transfections in Huh7 cells (similar to panel A). (D) Western blot analysis of the eIF4G2 knockdown in Huh7 cells, GAPDH as a loading control.  

Notably, cell treatment with PP242 made the translation of several eIF4G2-independent reporter mRNAs (e.g. eIF4G2, HIV1, HSPA2 and $\mathrm{TGF}\upbeta1\$ ) slightly but reproducibly eIF4G2-dependent $10–20\%$ ,$P<0.05\$ ) (Figure 2B and Supplementary Figure S3). Reciprocally, when cells are depleted of eIF4G2, translation of these three reporters requires eIF4E a little bit stronger (Figure 2A). We cannot tell if, under such conditions, eIF4G2 only promotes scanning or if it also helps or replaces eIF4G1 in mRNA recognition. Strictly speaking, the induction of eIF4G2-dependence may not be attributed to eIF4E inactivation unambiguously, because mTOR governs multiple processes, not eIF4E activity exclusively. Regardless, the data imply that in starved or quiescent cells where mTOR activity is blunted and eIF4E is suppressed, the repertoire of eIF4G2 translational targets may be much broader than we are currently able to estimate.  

The d subunit of eIF3 has also been implicated in cap-binding (54,55) and, specifically, in promoting capdependent translation of eIF4G2 targets (17). To elucidate a role for eIF3d in the translation of the eIF4G2 targets described here, we knocked down eIF3d individually or together with eIF4G2 in 293T cells (Supplementary Figure S4; data for a few mRNAs that were later found to be eIF4G2-dependent are also shown here). The decline in ATF4 reporter translation upon eIF3d knockdown probably reflects eIF3 participation in reinitiation on the ATF4 $5^{\prime}$ UTR (56). Thus, eIF3d is specifically required for ATF4 translation not only under stress (57) but under normal conditions as well (Supplementary Figure S4A). Among other reporter mRNAs, CCNI, PPFIA4 and SMAD1 also showed reduced translation efficiencies upon eIF3d depletion. However, when eIF3d and eIF4G2 were knocked down simultaneously, the effect of the eIF4G2 knockdown clearly dominated over that of eIF3d (Supplementary Figure S4B). No changes in the eIF3d knockdown effect could be observed upon an mTOR inhibition (Supplementary Figure S4A), and, conversely, the eIF3d knockdown did not alter the pattern of translation inhibition upon PP242 treatment (Supplementary Figure S4C). This may be due to the incompleteness of the eIF3d knockdown or, arguably, switching to eIF3d-mediated cap-binding requires (in)activation of other signalling cascades (55), which is not induced by the relatively short-term PP242 treatment (2 h). Under the analysed conditions, eIF3d does not seem to be the protein that brings eIF4G2 into scanning complexes.  

![](images/459d748764a72830b5cf41364557ecae21381a37cc50b913331ceecbb91d66c8.jpg)  
Figure 2. eIF4G2 promotes eIF4E-dependent translation. (A) Control or eIF4G2 knocked down 293T cells were treated with either vehicle (DMSO) or $1\,\upmu\mathrm{M}$ PP242 to activate eIF4E-BPs. Then the cells were transfected with the indicated mRNA reporters along with $\upbeta$ -globin reference mRNA coding for Fluc or Nluc, as an appropriate internal reference $(n\geq5)$ . The data are presented as ratios of normalized reporter expression in cells treated with $1~{\upmu\mathrm{M}}$ PP242 to that in vehicle-treated cells. The dotted line at 1, therefore, corresponds to a behaviour identical to that of the reference $\upbeta$ -globin reporter. The average ${\sim}2.8$ -fold drop in the reference $\upbeta$ -globin mRNA translation is shown by the dashed line, which therefore delineates a totally PP242-unresponsive translation. (B) Data from panel A were replotted to show if the mTOR inactivation alters the response to eIF4G2 knockdown (siRNA #1). The statistical significance is determined by the Mann–Whitney $U$ test. The exact $P_{-}$ -values for HIV1, $\mathrm{TGF}\upbeta1$ and HSPA2 are 0.009, 0.0016 and 0.0012, respectively. (C) Western blotting analysis of control and eIF4G2 knocked down 293T cells treated with either $1\ \upmu\mathrm{M}$ PP242 or vehicle (DMSO). The PP242 treatment resulted in the disappearance of $\mathrm{Thr}37/\mathrm{Thr}46$ phosphorylation of eIF4E-BP1, which is a result of the mTOR inhibition.  

# Presence of an uORF in ${\sf5^{\prime}}$ UTR is an important determinant of eIF4G2-dependence  

Upstream ORF presence is a frequent feature of eIF4G2- dependent $5^{\prime}$ UTRs. Indeed, most of the eIF4G2 targets contain uORFs (the only validated exception among the selected $5^{\prime}$ UTRs is Pcbp2). Therefore, we have checked whether eIF4G2-dependence is associated with these uORFs, especially as they are reckoned to be the main obstacles for scanning 40S ribosomes on the way to the CDS start codon.  

Strikingly, substitution of uAUGs from APAF1, Maf1 and Stard7 $5^{\prime}$ UTRs for the UAG stop codon (schematically shown in Figure 3C) almost completely relaxed the need for eIF4G2 for Maf1 and Stard7, and to a lesser extent for APAF1 (Figures 3A, D and 8E, and Supplementary Figures S5A, D and S8E). The choice of the targets was dictated by a small number of uAUGs (BCL2 has up to 9, depending on the exact $5^{\prime}$ UTR variant, CCNI has 5 and PPFIA4 has 6 uAUGs). Additionally, the mutation of uAUGs increased the translation of Maf1 and Stard7 reporters, but not that of APAF1 (data not shown). This surprising feature of the APAF uORF suggests that either it is short enough (nine amino acids) to permit an efficient translation reinitiation, or that uORF-mediated regulation is more sophisticated than we presume. The apparent lack of any effect could just be a coincidence: like in the case of human iNOS mRNA, deletion of the translated uORF therefrom does not affect the main ORF translation (58).  

Upstream open reading frames are present roughly in half of human or mouse mRNAs (33,59,60) and, thus, it looks unlikely that an uORF presence per se makes the translation dependent on eIF4G2. For example, the EPAS1 and eIF4G2 mRNAs bear a single uORF in their $5^{\prime}$ UTRs, but do not respond to the eIF4G2 knockdown (Figure 1A and B). Even so, we assayed a number of uORFcontaining reporters bearing the AKT2, CFTR1, eIF5, HERC1, IFRD1, MDM2, PKR, SARS-CoV2, SMAD1, TUBA1B or UCP2 $5^{\prime}$ UTRs (35,40,61–63). Among them, only the AKT2 (a kinase that contributes in glucose uptake and skeletal muscle differentiation), SMAD1 (a receptorregulated SMAD) and UCP2 (mitochondrial uncoupling protein 2) reporters were eIF4G2-dependent (Figure 3F).  

![](images/382a0beb8380baf83720913689a63e1cf67a7a05ff4d109aa4f300f0e3215b67.jpg)  
Figure 3. uORFs largely determine the eIF4G2-dependence in $293\mathrm{T}$ cells. The reporter mRNAs were transfected to control cells and those depleted of eIF4G2 along with reference ${\upbeta}$ -globin reporter mRNA coding for Nluc $(n\geq5)$ . The data are presented as ratios of normalized reporter expression in cells with eIF4G2 knocked down to control cells. (A) The uAUG codons of the APAF1, Maf1, Stard7, AKT2, SMAD1 and UCP2 reporters were mutated to the UAG stop codons to eliminate the uORFs (see panel C) and the effect of eIF4G2 knockdown by siRNA #1 was assayed in 293T cells. (B) Upstream AUG codons were introduced into the $5^{\prime}$ UTRs of $\mathrm{TFG}\beta1$ and HSPA2 so that the created uORFs were of the same size and approximately of the same position relative to the main AUG as the Stard7 uORF (see panel C). Then the effect of eIF4G2 knockdown by siRNA #1 was assayed in 293T cells. Statistical significance was determined by Mann–Whitney $U$ test ( $7<0.001$ in both cases). (C) Schematic in-scale representation of the reporters assayed in panels A, B, D and E. The $5^{\prime}$ UTRs and the start of Fluc CDS are depicted. Arrows indicate uORFs, crosses display the positions of the uAUGs that were substituted for the stop codons in the corresponding reporters. Panels D and $\mathbf{E}$ are similar to panels A and B, respectively, except siRNA #2 was used for the eIF4G2 depletion in 293T cells. (F) mRNAs bearing uORF(s), which regulate translation under certain conditions, were assayed for eIF4G2-dependence in 293T cells. Only AKT2, SMAD1 and UCP2 reporter translation was significantly affected by the eIF4G2 knockdown.  

The elimination of the uAUGs relieved the AKT2 (the AKT2 $5^{\prime}$ UTR has two uAUGs and we mutated only one of them as the other has very poor context) and UCP2 reporter translation of eIF4G2-dependence (Figure 3A, D and Supplementary Figure S5A, D). The case of UCP2 is particularly interesting since the uORF therein provides this mRNA with resistance to $\mathrm{eIF}2\upalpha$ phosphorylation (35), and is therefore discussed more closely below. Mutation of the SMAD1 uAUG did not diminish the input from eIF4G2 significantly (Figure 3A and D), thereby expanding the list of cases where requirements for eIF4G2 are determined by factors other than uORF.  

In a complementary approach, we introduced an uAUG codon with the Stard7 uORF context, i.e., GGCAUGA, into the long HSPA2 and $\mathrm{TGF}\upbeta1\,5^{\prime}$ UTRs so that the length of the introduced uORFs (terminated at stop codons naturally present in these $5^{\prime}$ UTRs) and the distance between them and the main ORF were close to what is found in the Stard7 $5^{\prime}$ UTR (schematically shown on Figure 3C). This does render HSPA2 and $\mathrm{TGF}\upbeta1$ mRNAs sensitive to eIF4G2 knockdown ( $\textit{P}<\ 0.001)$ ), albeit not dramatically (Figure 3B, E and Supplementary Figure S5B, E). Ultimately, a single uORF may impart a necessity to use eIF4G2 in translation initiation.  

# Scanning through a translated uORF can make further ribosome movement eIF4G2-dependent  

On mRNAs with an uORF, ribosomes have to either leak through the uAUG or reinitiate scanning after translation of the uORF to reach the main AUG. However, before investigating these possibilities, we studied if eIF4G2 influences the translation of uORFs themselves. To this end, we deleted the sequences linking the upstream and Fluc ORFs in the APAF1, Maf1, Stard7 and UCP2 reporters. Thus, the uORFs became fused directly to the Fluc coding sequence that lacked its AUG start. This excluded any contribution of leaky scanning to the readout. Clearly, the translation initiated from the AUGs of uORFs on Maf1, Stard7 and UCP2 $5^{\prime}$ UTRs occurs in an eIF4G2-independent fashion, while the APAF1 case is again more complex (Figure 4A, D and Supplementary Figure S6A, D). This strongly indicates that the demand for eIF4G2 arises during uORF scanning or after uORF reading. It might thus seem that eIF4G2 is the hypothetical “dedicated rescanning factor” (64), but if this were true, then translation of the ATF4 and ATF5 reporters (which heavily relies on reinitiation), would be compromised upon eIF4G2 depletion. Since this is not the case (Figure 3F and Supplementary Figure S4B), even if eIF4G2 promotes rescanning, it does not participate in each and every reinitiation event.  

It was therefore interesting to check whether eIF4G2 promotes translation under conditions of limited eIF2' when reinitiation plays a crucial role in ATF4 and ATF5 translation. We induced $\mathrm{eIF}2\upalpha$ phosphorylation by a sodium arsenite treatment and examined how eIF4G2 knockdown manifests itself in these circumstances. Along with the eIF4G2 target mRNAs (now including AKT2, SMAD1 and UCP2) we assayed mRNAs known to be resistant to eIF2 inactivation (ATF4, ATF5 and IFRD1). The eIF4G2 targets (except PPFIA4 and UCP2) did not show any resistance to eIF2' phosphorylation, despite the presence of uORFs in their $5^{\prime}$ UTRs (Figure 5A). Translation of the CCNI, PKR and Pcbp2 reporters showed an increased sensitivity to eIF2 inhibition, in accordance with the ribosome footprint profiling data (35,65). In line with the published data, ATF4 translation does not rely on eIF4G2 upon $\mathrm{eIF}2\upalpha$ phosphorylation (57). Notably, all tested eIF4G2 targets became slightly less dependent on eIF4G2 when formation of ternary complex was inhibited (Figure 5B and Supplementary Figure S7). We conclude that eIF4G2 does not contribute to reinitiation-based translation resistance to eIF2 inactivation.  

Finally, to definitively distinguish between leaky scanning and reinitiation, we mutated the stop codons of Stard7, Maf1 and UCP2 uORFs so that these extended uORFs substantially overlapped out-of-frame with the firefly luciferase ORF (schematically shown on Figure 4C). On these reporters, ribosomes can only reach the Fluc AUG codon via leaky scanning, because “backward reinitiation” is extremely inefficient (62,63,66–69). Strikingly, the translation of these mRNAs remained eIF4G2-dependent, albeit to a lesser extent in the case of Maf1. This means that, on the selected mRNAs, eIF4G2 promotes ribosomal scanning inside and/or downstream of the uORFs, rather than reinitiation (Figure 4B, E and Supplementary Figure S6B, E). Moreover, an improvement in the modest context of the Maf1 uORF (GcuAUGUc) to match a strong Kozak sequence (AccAUGAg) inhibited Fluc translation ($11\pm3$ times, $n=7$ ) in $293\mathrm{T}$ cells (data not shown). If reinitiation was the major mechanism of reaching the main AUG, such a dramatic drop would arguably not occur. More importantly, the improvement of the uAUG context did not alter the need for eIF4G2 (see below). This further indicates that a significant fraction of ribosomes that reach the main Maf1 AUG do so via eIF4G2-dependent leaky scanning.  

The idea that eIF4G2 promotes scanning after skipping an uAUG is reminiscent of the mechanism that was proposed to explain UCP2 translation resistance to eIF2 inactivation. It postulates a relief of interference between the ribosomes that are translating the uORF and those that have leaked through the uORF start codon and continued scanning (35,70). eIF4G2 knockdown does not alter the UCP2 reporter response to $\mathrm{eIF}2\upalpha$ phosphorylation (Figure 5A), but only lowers the translation yield. This corroborates the idea that eIF4G2 is a scanning factor that, on some mRNAs with uORFs, usually does not come into play until the scanning ribosome encounters a translated uORF.  

# eIF4G2 requirement depends on sequences both upstream and downstream of uORF, and on uORF properties  

The length of an uORF could affect eIF4G2-dependence on its own. We thus created several variants of the reporters where the corresponding uORFs had been changed to code for 3, 9, 18 and 37 amino acid long peptides. These included the uORFs from Maf1 (naturally codes for a 40 aa peptide), Stard7 (18 aa) and UCP2 (36 aa) $5^{\prime}$ UTRs. The uAUGs were kept in their original sites. These three $5^{\prime}$ UTRs exhibited markedly different responses to variations in uORF size (Figure 6). In the case of Stard7, uORF length has no impact on eIF4G2 involvement. For Maf1, the effect of eIF4G2 knockdown gradually declined with uORF truncation and shortening of the UCP2 uORF from 36 to 18 aa led to complete and immediate eIF4G2-independence. The differences in the uAUG contexts (modest for Maf1 and strong in the cases of Stard7 and UCP2) do not account for the observed difference because improving the Maf1 uAUG context (GcuAUGUcg to AccAUGAgc, the second aa is not altered) or impairing the Stard7 uAUG context (GgcAUGAgg to GgcAUGCgg, the second aa is not altered) failed to change the pattern. The only difference was that the Stard7 reporter became less dependent on eIF4G2, which is not unexpected because less ribosomes initiate on its uAUG.  

In order to investigate whether anything other than uORFs can affect the need for eIF4G2, we replaced the sequence upstream of the Stard7 uORF $\ensuremath{\left(209\mathrm{~nt}\right)}$ with either partial HSPA2 or the entire hnRNPK (v2) $5^{\prime}$ UTR sequences, so that the overall $5^{\prime}$ UTR length remained roughly the same (see Supplementary Table S1C for the exact sequences). In all cases, the involvement of eIF4G2 declined noticeably (Figure 7A). Similarly, substituting the linker between the Stard7 uORF and the main ORF for HSPA2 or hnRNPK sequences of the same length $(\sim\!100\,\mathrm{\nt})$ or the entire hnRNPK (v2) $5^{\prime}$ UTR sequence ( $\sim\!200\ \mathrm{nt})$ also reduced the eIF4G2 contribution (Figure 7A). Arguably, a $5^{\prime}$ -distant uORF that provides eIF4G2-dependence may not do so when placed more closely to the $5^{\prime}$ -end. To test this hypothesis, we simply shortened the $5^{\prime}\mathrm{UTRs}$ of Stard7, Maf1 and UCP2 reporters so that 60-70 nt preceded the corresponding uORFs. This distance is long enough to prevent leaky scanning artificially provoked by a short $5^{\prime}$ UTR (71,72). However, only Stard7 translation showed a markedly reduced eIF4G2-dependence, indicating that the distance from the $5^{\prime}$ -end to uORF is not the major determinant of eIF4G2-dependence (Figure 7B). This clearly demonstrates that various $5^{\prime}$ UTRs are scanned with different efficiencies by the eIF4G1- and eIF4G2-based complexes and that nucleotide sequences both upstream and downstream of an uORF affect the eIF4G2 impact on scanning.  

![](images/f4839ca8f9258f026065a058dcf83de9bf0eccf6011b422adf29c1ec2f54fdbc.jpg)  
Figure 4. eIF4G2 promotes ribosomal scanning inside and/or downstream of the uORFs in 293T cells. The reporter mRNAs were transfected to control cells and those depleted of eIF4G2 along with reference ${\upbeta}$ -globin reporter mRNA coding for Nluc $(n\geq5)$ . The data are presented as ratios of normalized reporter expression in cells with eIF4G2 knocked down to control cells. (A) The uORFs of APAF1, Maf1, Stard7 and UCP2 reporters were fused to the Fluc sequence (see panel C) and the involvement of eIF4G2 in uORF translation was assayed in 293T cells. siRNA #1 was used for the eIF4G2 depletion. (B) The stop codons of Maf1, Stard7 and UCP2 uORFs were mutated so that the extended uORFs became overlapped with the Fluc CDS (see panel C) to eliminate a possibility of translation reinitiation. The eIF4G2-dependence of these reporters was assayed in $293\mathrm{T}$ cells treated either with control or anti-eIF4G2 siRNA #1. (C) Schematic in-scale representation of the reporters assayed in panels A, B, D and E. The $5^{\prime}$ UTRs and the start of Fluc CDS are depicted. Arrows indicate the wild type and extended uORFs in the $5^{\prime}$ UTRs of assayed reporters. Panels D and $\mathbf{E}$ are similar to panels A and B, respectively, except siRNA #2 was used for the eIF4G2 depletion in 293T cells.  

Taken together, these data suggest that yet undiscovered uORF properties, but not its length, contribute to eIF4G2- requirements.  

# eIF4G1 and eIF4G2 can functionally replace each other during scanning  

To better understand the interplay between eIF4G1 and eIF4G2, we performed eIF4G1 knockdown in $293\mathrm{T}$ cells (Figure 8). Since eIF4G1 has a poorly characterized homologue named eIF4G3 (also known as eIF4GII) that could be upregulated upon eIF4G1 knockdown or could be otherwise involved in translation of the eIF4G2 targets, we also assayed the effect of eIF4G1 knockdown in RKO cells where the eIF4G3 gene expression was disrupted using AsCpf1 nuclease (Supplementary Figure S8).  

In $293\mathrm{T}$ cells translation of the majority of tested mRNAs was not affected by eIF4G1 knockdown in any specific way (i.e. relative to ${\upbeta}$ -globin reference), but that of CCNI, Map3k3 and Stard7 diminished (Figure 8D). The data were also plotted to directly show effects of either knockdown under conditions of the depletion of the other protein (Supplementary Figure S9A, B). The responses to eIF4G1 and eIF4E inactivation do not correlate (see also Figure 2A), therefore we tend to think that the observed decline reflects elevated requirements for eIF4G1 during scanning. The effect is also noticeable but less pronounced in the RKO $(\mathrm{eIF4G3^{-/-}})$ cells (Supplementary Figure S8). Simultaneous eIF4G1 and eIF4G2 knockdown in $293\mathrm{T}$ cells markedly inhibited translation of several other reporter mRNAs, i.e., EPAS1, eIF5, PPFIA4, SMAD1 and the uORFless variants of Maf1 and Stard7 in $293\mathrm{T}$ cells (Figure 8B, D and E). In other words, the limited availability of eIF4G1 makes translation of these mRNAs eIF4G2-dependent and vice versa. Again, the effect was not so strong in the RKO ( $\mathrm{eIF4G3^{-/-1}}$ )cells (Supplementary Figures S8 and S9C, D).  

![](images/b27cfe152cbc95b550d2b59d2c412c70b10e3c3064bed7ce01401b3234102d6c.jpg)  
Figure 5. eIF4G2 is not specifically required for translation under conditions of eIF2 inactivation. (A) Control or eIF4G2 knocked down 293T cells were treated with either vehicle (PBS) or $40\ensuremath{~\upmu\mathrm{M}}$ sodium arsenite to induce oxidative stress and then transfected with the indicated mRNA reporters along with the $\upbeta$ -globin mRNA internal reference $(n\geq5)$ . The data are presented as a ratios of normalized reporter expression in the arsenite-treated to the vehicle-treated cells. Please note the log2 scale. (B) Data from panel A were replotted to show if the eIF2 inactivation alters the response to eIF4G2 knockdown (siRNA #1). (C) Western blotting analysis of control and depleted of eIF4G2 (siRNA #1 and #2) 293T cells treated with either $40~\upmu\mathrm{M}$ sodium arsenite or vehicle (PBS), GADPH as a loading control. The sodium arsenite treatment led to phosphorylation of eIF2'.  

This particular observation suggests that eIF4G2 replaces eIF4G1 during scanning, cap-dependent ribosome recruitment, or both. In an attempt to address this, we evaluated possible eIF4G2-dependencies of A-capped mRNAs, which are unable to bind eIF4E. The contribution of eIF4G2 to the translation of its targets remained unchanged compared to their $\mathrm{m^{7}G}$ -capped counterparts (Supplementary Figure S10C, D), which apparently means that the protein does not assist in ribosome loading. However, control $5^{\prime}$ UTRs that show no need for eIF4G2 when $\mathbf{m}^{7}\mathbf{G}.$ -capped (Figure 1), exhibited a length-dependent demand for eIF4G2 in the A-capped forms (Supplementary Figure S10A, B). This effect does not result from the disruption of the closed-loop mRNA structure, because nonpolyadenylated variants of the same $\mathrm{m^{7}G}$ -capped mRNAs did not acquire an eIF4G2-dependence (Supplementary Figure S10E). If we assume that uncapped mRNAs are scanned from their $5^{\prime}$ -ends (38,73,74), then we have to conclude that, on A-capped mRNAs, eIF4G2 operates during scanning, because $5^{\prime}$ UTR length per se is not known to affect modes of ribosome recruitment. This result can explain why mRNAs with longer $5^{\prime}$ UTRs become susceptible to eIF4G2 depletion under conditions of mTOR inhibition.  

At this point, we cannot unequivocally determine if eIF4G2 participates in both scanning and initial capdependent ribosome recruitment, but, in aggregate, data presented here definitely show that scanning requires eIF4G2 when there is an undersupply of eIF4G1.  

# DISCUSSION  

Earlier reports suggested that eIF4G2 acts in capindependent and, specifically, in internal translation initiation. This notion was mostly based on the extensive use of the insufficiently controlled DNA-based bicistronic assay (23,25–29,75), but the fallibility of this approach was not fully recognised then (4,5,76). Also, eIF4G2 knockdown suppressed the translation of monocistronic reporters bearing a stable stem-loop on their $5^{\prime}$ -termini to prevent 40S binding thereto (21). However, the inability of such mRNAs to use the conventional $5^{\prime}$ -end-dependent scanning mechanism was taken for granted and not tested directly by a comparison with bicistronic mRNA reporters. Another line of reasoning arose from the in vitro translation of A-capped monocistronic mRNAs in HeLa extract (24) or RRL (16), but notably, all mRNAs studied this way have relatively long $5^{\prime}$ UTRs. Our data indicate that many mRNAs that naturally do not require eIF4G2 actually acquire the need for the protein after A-capping (Supplementary Figure S10A, S10B). Thus, assaying A-capped mRNAs is inadequate for validating eIF4G2 targets in normal cap-dependent translation. Yet it would be totally incorrect to dismiss all eIF4G2 targets reported in the above-cited papers. Indeed, we show that the translation of the $\mathbf{m}^{7}\mathbf{G}.$ -capped monocistronic APAF1 (Figure 1A, C) and various BCL2 (Supplementary Figure S2B) reporters is eIF4G2- dependent. Altogether, the evidence that internal translation initiation is the realm of eIF4G2 is shaky.  

We show here that eIF4G2 participates in canonical capand eIF4F-dependent translation. Our data strongly suggest that eIF4G2 operates downstream of cap recognition and mRNA accommodation by 43S ribosomal complexes. The eIF4G2-dependence seems to be frequently governed by uORFs (Figure 3A, C and Supplementary Figure S5A, D), but the cases of APAF1, SMAD1 and Pcbp2 $5^{\prime}$ UTRs demonstrate that the mechanics of eIF4G2 engagement are more complex. For the other mRNAs, the translation of uORFs themselves and, therefore, all preceding steps of the initiation including mRNA accommodation are eIF4G2- independent (Figure 4A, D and Supplementary Figure S6A, D). In accordance with this, cap-sepharose does not pull-down eIF4G2 (16,77), thus, eIF4G2 is not a constituent of cap-binding complexes. Our data do not exclude that, on particular mRNAs under conditions of profound eIF4E inactivation, other protein partners bring eIF4G2 to mRNA to promote cap-dependent translation (17,30) or that eI4G2 itself may bind mRNA (in a CITE mode) to attract ribosomes (78). What the presented experiments show is that eIF4G2 is a dedicated scanning factor and, as such, can arguably participate in any scanning-dependent initiation mechanism after ribosome attachment, yet capdependent translation apparently dominates in eukaryotes and thus eIF4G2 de facto promotes it.  

![](images/53a664f218b686972a08201692cc75f1536324d0986e7c4f6b42b62788c08a1e.jpg)  
Figure 6. Alteration of uORF length and context of the uAUG. The uORFs of eIF4G2 target mRNAs were changed to code for 3, 9, 18 and 36 aa-long peptides. The uAUGs were kept in their places. The natural UCP2, Maf1 and Stard7 uORFs encode 36 aa, 40 aa and 18 aa-long peptides, respectively. The uAUG context of studied uORFs was also altered. The uORF of $\surd0^{\circ}$ aa length corresponds to substitution of the uAUGs for the UAG stop codon. The corresponding mRNAs were transfected to control and eIF4G2-depleted $293\mathrm{T}$ cells along with reference ${\upbeta}$ -globin reporter mRNA coding for Nluc ( $n\geq$ 5). The data are presented as ratios of normalized reporter expression in eIF4G2-depleted cells to control cells. (A) The effect of uORF length alteration on eIF4G2-dependence of these reporters (siRNA $\#1$ for the eIF4G2 depletion). (B) The effect of uAUG context alteration on eIF4G2-dependence of the Maf1 reporters with various uORF length (siRNA #1 for the eIF4G2 depletion). The uAUGs had either wild type context (GcuAUGU) or improved (AccAUGA). (C) The effect of uAUG context alteration on eIF4G2-dependence of the Stard7 reporters with various uORF length (siRNA #1 for the eIF4G2 depletion). The uAUGs had either wild type context (GgcAUGA) or worsened one (GgcAUGC). Panels D, E and F are similar to panels A, B and C, respectively, except siRNA #2 was used for the eIF4G2 depletion.  

Our data indicate that in the case of the eIF4G2- dependent mRNAs analysed here (APAF1, Maf1, Stard7, UCP2), leaky scanning plays a principal role in the eIF4G2- driven translation of main ORFs. There are two mutually nonexclusive possibilities as to how the presence of an uORF may lead to the requirement for eIF4G2. The protein may promote uAUG skipping and/or scanning afterwards. The fact that eIF4G2 knockdown does not enhance the Maf1, Stard7 or UCP2 uORFs’ translation (Figure 4A, D and Supplementary Figure S6A, D) is only compatible with the latter explanation.  

The only difference between the uORF-less and the wild type $5^{\prime}$ UTRs is the presence of ribosomes that translate the uORFs. Therefore, the translating ribosomes obstruct the scanning process. Obviously, the scanning and translating ribosomes should interfere with each other. The presented data suggest that eIF4G2 makes a difference when eIF4G1 is lacking. If we combine these two premises, we can hypothesize that ribosome interference leads to a loss of eIF4G1 from scanning complexes. This also suggests that scanning ribosomes that rely on eIF4G2 are not cap-tethered, which is indirectly supported by the apparent lack of eIF4E and eIF4G1 associated with eIF4G2 (17,18). eIF4G1 does not dissociate completely or it is able to re-associate back, because the eIF4G2 targets are somehow translated in the absence of eIF4G2, but most likely eIF4G1 is attracted back to the $\mathrm{m^{7}G}$ -cap to promote the next round of ribosome binding. Dissociation of eIF4G1 from scanning complexes may thus be the hallmark of eIF4G2-dependent translation. Another related scenario considers the possibility of spontaneous eIF4G1 dissociation-association on longer $5^{\prime}$ UTRs. In this case, there are two tangible determinants for reassociation: eIF4G1 availability and steric accessibility. Compared to elongating 80S complexes, which cover around 30 nt of mRNA, scanning ribosomes provide heterogeneous footprints which are significantly longer (up to $80\;\mathrm{nt})$ ) (79–81). These protrusions are most likely formed by initiation factors. Once eIF4G1 dissociates from a queueing ribosome, it might not fit back subsequently (unlike the smaller eIF4G2). The differential response to the uORF shortening (Figure 6) indicate that elongating ribosomes definitely can cause the interference, but can it be caused by initiating or even terminating ribosomes, remains to be established. Either model predicts that mRNAs with short $5^{\prime}$ UTRs where AUG recognition occurs before eIF4G1 has a chance to dissociate, do not require eIF4G2.  

![](images/31b0e16bebf5b44db01bf4ed0c7eb888996f61d87be58815f204eed3384ac219.jpg)  
Figure 7. Requirement for eIF4G2 depends on sequences both upstream and downstream of uORF. The reporter mRNAs were transfected to control 293T cells and those depleted of eIF4G2 along with reference ${\upbeta}$ -globin reporter mRNA coding for Nluc $(n\geq5)$ . The data are presented as ratios of normalized reporter expression in cells with eIF4G2 knocked down to control cells. (A) The sequence upstream of the Stard7 uORF was replaced with either entire hnRNPK (v2) or truncated HSPA2 $5^{\prime}$ UTR so that the overall $5^{\prime}$ UTR length remained roughly the same. Also, the linker between the Stard7 uORF and Fluc CDS was replaced with HSPA2, entire or truncated hnRNPK (v2) $5^{\prime}$ UTR sequences. (B) The Maf1, Stard7 and UCP2 reporters were truncated from their $5^{\prime}$ -termini to leave ${\sim}70\,\mathrm{nt}$ upstream of the uORFs. The effect of eIF4G2 knockdown was assayed in 293T cells (siRNA #1).  

The ability of sequences upstream of the eIF4G2- summoning uORFs to modulate what occurs downstream thereof highlights the interactive nature of uORF-mediated translational control. The Initiation Complexes Interference with Elongating Ribosomes (ICIER) model proposes that ribosomes that translate an uORF impede the movement of scanning ribosomes, thereby explaining how an influx of ribosomes affects the degree of inhibition by an uORF (35,70). Our model suggests that this should also be the case for eIF4G2-dependent mRNAs. Likewise, $5^{\prime}$ UTR length should affect the influx as well. To date, the only established functional result of a $5^{\prime}$ UTR length increase is a hindrance to the closed-loop mediated reinitiation (82). This implies that the stability of interaction between eIF4G1 and a scanning ribosome declines as the latter moves away from the cap and thus cap-distant scanning ribosomes become more vulnerable to miscellaneous impediments. This is especially true when both eIF4E–eIF4G1 and eIF4G1–eIF3 interactions are weakened by mTOR inhibition (83,84), making the link between $\mathrm{m^{7}G}$ -cap and scanning ribosome even more labile. The increased input from eIF4G2 to the scanning of longer $5^{\prime}$ UTRs under conditions of mTOR inhibition (Figure 2B and Supplementary Figure S3) or upon eIF4G1 knockdown (Figure 8 and Supplementary Figures S8, S9) fits this hypothesis nicely.  

Various obstacles may obstruct the scanning process between the cap and the uORF. If scanning complexes lose their eIF4G1 on the way to an uORF due to some secondary structural elements or RNA-binding proteins, then the role of eIF4G2 in rescuing the scanning process increases. This mechanism may play a major role in the high eIF4G2-dependence of long $5^{\prime}$ UTRs lacking uORFs.  

What can be the possible benefits of such complicated machinery? First, the uORF–eIF4G2 cooperation we describe provides cell with another layer of posttranscriptional control of gene expression, which may be crucial for a fine-tuned expression of low abundant proteins, which are often responsible for signaling, regulation and certain important metabolic processes (85). Secondly, eIF4G2 makes it possible for cell to have true bicistronic mRNAs, because the protein can augment a main ORF expression while the uORF is also translated. Third, numerous mRNAs possess two or more functional in-frame start codons and thus their translation results in both fulllength and N-terminally truncated proteoforms (86,87). The expression of the shorter isoforms should be affected by eIF4G2. In terms of evolution, the ability of eIF4G2 to counteract translation inhibition by uORFs can help organisms to overcome the deleterious effect of uORF-creating mutations. For example, in yeasts that lack an eIF4G2 orthologue, uAUGs are found significantly rarer than in higher eukaryotes that contain the eIF4G2 gene (88). Notably, a similar disparity between scanning of cap-proximal and cap-distal $5^{\prime}$ UTR regions exists in yeast cells, but it is Ded1p that participates in cap-distal scanning in that case (89).  

![](images/5721fadb592fffdf0508de205f55a47f6943fd187ce8414c5082553c405941dc.jpg)  
Figure 8. The eIF4G1 and eIF4G2 knockdown in $293\mathrm{T}$ cells. The reporter mRNAs were transfected along with $\upbeta$ -globin reference mRNA into control cells and those depleted of eIF4G1, eIF4G2 (siRNA #1) or both $(n\geq5)$ . The data are presented as ratios of normalized reporter expression in cells with the eIF4G1, eIF4G2 or simultaneous knockdown to control cells. Note that the dotted line at 1, therefore, corresponds to a behaviour identical to that of the reference ${\upbeta}$ -globin reporter. All the transfections were replicated at least five times and the mRNAs are grouped here as follows: (A) reporter mRNAs that are not specifically sensitive to the knockdowns, (B) mRNAs that are sensitive only to the simultaneous eIF4G1 and eIF4G2 knockdown, (C) PTV and leaderless reporters, which translation is stimulated by the eIF4G1 knockdown, (D) eIF4G2 translational targets, (E) eIF4G2 translational targets in which uORF(s) are eliminated. (F) Western blot analysis of control 293T cells and those depleted of eIF4G1, eIF4G2 (siRNA #1) or both, GADPH as a loading control.  

In all likelihood, we have only described the tip of the iceberg. Sufficiently long $5^{\prime}$ UTR and an uORF presence seem to be important but not obligatory and probably not the only determinants of eIF4G2-dependence. Participants and modulators of translation that influence ribosome loading onto mRNA, scanning efficiency and fidelity, translation elongation and termination rates will all affect the balance between scanning and translating ribosomes and thus the exact contribution of eIF4G2 to cell proteome may vary significantly in different physiological conditions. In conclusion, eIF4G2 is a dedicated scanning factor that overcomes the inhibitory effect of uORFs (and probably other obstacles) to support required levels of translation via promoting leaky scanning.  

# DATA AVAILABILITY  

All the raw data and processed files have been deposited in the Gene Expression Omnibus (http://www.ncbi.nlm.nih. gov/geo) under accession number GSE158136.  

# SUPPLEMENTARY DATA  

Supplementary Data are available at NAR Online.  

# ACKNOWLEDGEMENTS  

The authors would like to thank Alan Hinnebusch for commenting on the manuscript, Sergey Dmitriev for helpful discussions, antibodies and plasmids, Kseniya Lashkevich and Andrey Anisenko for plasmids, Ervin Walker for the gift of pTE4396 plasmid, and Irina Eliseeva for antibodies and discussion. We thank the Center for Precision Genome Editing and Genetic Technologies for Biomedicine, EIMB RAS, for providing the computational facilities.  

# FUNDING  

Russian Science Foundation [19-14-00152 to I.N.S.]. Funding for open access charge: Russian Science Foundation. Conflict of interest statement. None declared.  

# REFERENCES  

1. Hinnebusch,A.G. (2014) The scanning mechanism of eukaryotic translation initiation. Annu. Rev. Biochem., 83, 779–812.   
2. Hinnebusch,A.G. (2011) Molecular mechanism of scanning and start codon selection in eukaryotes. Microbiol. Mol. Biol. Rev., 75, 434–467.   
3. Kozak,M. (1980) Evaluation of the ‘scanning model’ for initiation of protein synthesis in eucaryotes. Cell, 22, 7–8.   
4. Shatsky,I.N., Dmitriev,S.E., Terenin,I.M. and Andreev,D.E. (2010) Cap- and IRES-independent scanning mechanism of translation initiation as an alternative to the concept of cellular IRESs. Mol. Cells, 30, 285–293.   
5. Jackson,R.J. (2013) The current status of vertebrate cellular mRNA IRESs. Cold Spring Harb. Perspect. Biol., 5, a011569.   
6. Shirokikh,N.E., Dutikova,Y.S., Staroverova,M.A., Hannan,R.D. and Preiss,T. (2019) Migration of small ribosomal subunits on the $5^{\prime}$ untranslated regions of capped messenger RNA. Int. J. Mol. Sci., 20, 4464.   
7. Pestova,T.V. and Kolupaeva,V.G. (2002) The roles of individual eukaryotic translation initiation factors in ribosomal scanning and initiation codon selection. Genes Dev., 16, 2906–2922.   
8. Terenin,I.M., Dmitriev,S.E., Andreev,D.E., Royall,E., Belsham,G.J., Roberts,L.O. and Shatsky,I.N. (2005) A cross-kingdom internal ribosome entry site reveals a simplified mode of internal ribosome entry. Mol. Cell. Biol., 25, 7879–7888.   
9. Shen,L. and Pelletier,J. (2020) General and target-specific DExD/H RNA helicases in eukaryotic translation initiation. Int. J. Mol. Sci., 21, 4402.   
10. Rogers,G.W., Richter,N.J., Lima,W.F. and Merrick,W.C. (2001) Modulation of the helicase activity of eIF4A by eIF4B, eIF4H, and eIF4F. J. Biol. Chem., 276, 30914–30922.   
11. Garcı´a-Garcı´a,C., Frieda,K.L., Feoktistova,K., Fraser,C.S. and Block,S.M. (2015) RNA BIOCHEMISTRY. Factor-dependent processivity in human eIF4A DEAD-box helicase. Science, 348, 1486–1488.   
12. Imataka,H., Olsen,H.S. and Sonenberg,N. (1997) A new translational regulator with homology to eukaryotic translation initiation factor 4G. EMBO J., 16, 817–825.   
13. Levy-Strumpf,N., Deiss,L.P., Berissi,H. and Kimchi,A. (1997) DAP-5, a novel homolog of eukaryotic translation initiation factor 4G isolated as a putative modulator of gamma interferon-induced programmed cell death. Mol. Cell. Biol., 17, 1615–1625.   
14. Yamanaka,S., Poksay,K.S., Arnold,K.S. and Innerarity,T.L. (1997) A novel translational repressor mRNA is edited extensively in livers containing tumors caused by the transgene expression of the apoB mRNA-editing enzyme. Genes Dev., 11, 321–333.   
15. Shaughnessy,J.D., Jenkins,N.A. and Copeland,N.G. (1997) cDNA cloning, expression analysis, and chromosomal localization of a gene with high homology to wheat eIF-(iso)4F and mammalian eIF-4G. Genomics, 39, 192–197.   
16. Liberman,N., Gandin,V., Svitkin,Y.V., David,M., Virgili,G., Jaramillo,M., Holcı´k,M., Nagar,B., Kimchi,A. and Sonenberg,N. (2015) DAP5 associates with eIF2' and eIF4AI to promote Internal Ribosome Entry Site driven translation. Nucleic Acids Res., 43, 3764–3775.   
17. la Parra,C., Ernlund,A., Alard,A., Ruggles,K., Ueberheide,B. and Schneider,R.J. (2018) A widespread alternate form of cap-dependent mRNA translation initiation. Nat. Commun., 9, 3068.   
18. Sugiyama,H., Takahashi,K., Yamamoto,T., Iwasaki,M., Narita,M., Nakamura,M., Rand,T.A., Nakagawa,M., Watanabe,A. and Yamanaka,S. (2017) Nat1 promotes translation of specific proteins that induce differentiation of mouse embryonic stem cells. Proc. Natl. Acad. Sci. U.S.A., 114, 340–345.   
19. Lee,S.H. and McCormick,F. (2006) p97/DAP5 is a ribosome-associated factor that facilitates protein synthesis and cell proliferation by modulating the synthesis of cell cycle proteins. EMBO J., 25, 4008–4019.   
20. Yamanaka,S., Zhang,X.Y., Maeda,M., Miura,K., Wang,S., Farese,R.V., Iwao,H. and Innerarity,T.L. (2000) Essential role of NAT1/p97/DAP5 in embryonic differentiation and the retinoic acid pathway. EMBO J., 19, 5533–5541.   
21. Yoffe,Y., David,M., Kalaora,R., Povodovski,L., Friedlander,G., Feldmesser,E., Ainbinder,E., Saada,A., Bialik,S. and Kimchi,A. (2016) Cap-independent translation by DAP5 controls cell fate decisions in human embryonic stem cells. Genes Dev., 30, 1991–2004.   
22. Haizel,S.A., Bhardwaj,U., Gonzalez,R.L., Mitra,S. and Goss,D.J. (2020) $5^{\prime}$ -UTR recruitment of the translation initiation factor eIF4GI or DAP5 drives cap-independent translation of a subset of human mRNAs. J. Biol. Chem., 295, 11693–11706.   
23. Henis-Korenblit,S., Shani,G., Sines,T., Marash,L., Shohat,G. and Kimchi,A. (2002) The caspase-cleaved DAP5 protein supports internal ribosome entry site-mediated translation of death proteins. Proc. Natl. Acad. Sci. U.S.A., 99, 5400–5405.   
24. Hundsdoerfer,P., Thoma,C. and Hentze,M.W. (2005) Eukaryotic translation initiation factor 4GI and p97 promote cellular internal ribosome entry sequence-driven translation. Proc. Natl. Acad. Sci. U.S.A., 102, 13421–13426.   
25. Warnakulasuriyarachchi,D., Cerquozzi,S., Cheung,H.H. and Holcı´k,M. (2004) Translational induction of the inhibitor of apoptosis protein HIAP2 during endoplasmic reticulum stress attenuates cell death and is mediated via an inducible internal ribosome entry site element. J. Biol. Chem., 279, 17148–17157.   
26. Weingarten-Gabbay,S., Khan,D., Liberman,N., Yoffe,Y., Bialik,S., Das,S., Oren,M. and Kimchi,A. (2014) The translation initiation factor DAP5 promotes IRES-driven translation of p53 mRNA. Oncogene, 33, 611–618.   
27. Nevins,T.A., Harder,Z.M., Korneluk,R.G. and Holcı´k,M. (2003) Distinct regulation of internal ribosome entry site-mediated translation following cellular stress is mediated by apoptotic fragments of eIF4G translation initiation factor family members eIF4GI and p97/DAP5/NAT1. J. Biol. Chem., 278, 3572–3579.   
28. Lewis,S.M., Cerquozzi,S., Graber,T.E., Ungureanu,N.H., Andrews,M. and Holcı´k,M. (2008) The eIF4G homolog DAP5/p97 supports the translation of select mRNAs during endoplasmic reticulum stress. Nucleic Acids Res., 36, 168–178.   
29. Marash,L., Liberman,N., Henis-Korenblit,S., Sivan,G., Reem,E., Elroy-Stein,O. and Kimchi,A. (2008) DAP5 promotes cap-independent translation of Bcl-2 and CDK1 to facilitate cell survival during mitosis. Mol. Cell, 30, 447–459.   
30. Bukhari,S.I.A., Truesdell,S.S., Lee,S., Kollu,S., Classon,A., Boukhali,M., Jain,E., Mortensen,R.D., Yanagiya,A., Sadreyev,R.I. et al. (2016) A specialized mechanism of translation mediated by FXR1a-associated microRNP in cellular quiescence. Mol. Cell, 61, 760–773.   
31. Smirnova,V.V., Shestakova,E.D., Bikmetov,D.V., Chugunova,A.A., Osterman,I.A., Serebryakova,M.V., Sergeeva,O.V., Zatsepin,T.S., Shatsky,I.N. and Terenin,I.M. (2019) eIF4G2 balances its own mRNA translation via a PCBP2-based feedback loop. RNA, 25, 757–767.   
32. Hinnebusch,A.G., Ivanov,I.P. and Sonenberg,N. (2016) Translational control by $5^{\prime}$ -untranslated regions of eukaryotic mRNAs. Science, 352, 1413–1416.   
33. Johnstone,T.G., Bazzini,A.A. and Giraldez,A.J. (2016) Upstream ORFs are prevalent translational repressors in vertebrates. EMBO J., 35, 706–723.   
34. Andreev,D.E., Dmitriev,S.E., Zinovkin,R., Terenin,I.M. and Shatsky,I.N. (2012) The $5^{\circ}$ untranslated region of Apaf-1 mRNA directs translation under apoptosis conditions via a $5^{\prime}$ end-dependent scanning mechanism. FEBS Lett., 586, 4139–4143.   
35. Andreev,D.E., O’Connor,P.B.F., Fahey,C., Kenny,E.M., Terenin,I.M., Dmitriev,S.E., Cormican,P., Morris,D.W., Shatsky,I.N. and Baranov,P.V. (2015) Translation of $5^{\prime}$ leaders is pervasive in genes resistant to eIF2 repression. Elife, 4, e03971.   
36. Akulich,K.A., Andreev,D.E., Terenin,I.M., Smirnova,V.V., Anisimova,A.S., Makeeva,D.S., Arkhipova,V.I., Stolboushkina,E.A., Garber,M.B., Prokofjeva,M.M. et al. (2016) Four translation initiation pathways employed by the leaderless mRNA in eukaryotes. Sci. Rep., 6, 37905.   
37. Smirnova,V.V., Terenin,I.M., Khutornenko,A.A., Andreev,D.E., Dmitriev,S.E. and Shatsky,I.N. (2016) Does HIV-1 mRNA $5^{\prime}$ -untranslated region bear an internal ribosome entry site? Biochimie, 121, 228–237.   
38. Terenin,I.M., Andreev,D.E., Dmitriev,S.E. and Shatsky,I.N. (2013) A novel mechanism of eukaryotic translation initiation that is neither m7G-cap-, nor IRES-dependent. Nucleic Acids Res., 41, 1807–1816.   
39. Severin,J., Lizio,M., Harshbarger,J., Kawaji,H., Daub,C.O., Hayashizaki,Y. and FANTOM ConsortiumFANTOM Consortium, Bertin,N. and Forrest,A.R.R. (2014) Interactive visualization and analysis of large-scale sequencing datasets using ZENBU. Nat. Biotechnol., 32, 217–219.   
40. Akulich,K.A., Sinitcyn,P.G., Makeeva,D.S., Andreev,D.E., Terenin,I.M., Anisimova,A.S., Shatsky,I.N. and Dmitriev,S.E. (2019) A novel uORF-based regulatory mechanism controls translation of the human MDM2 and eIF2D mRNAs during stress. Biochimie, 157, 92–101.   
41. T´oth,E., Weinhardt,N., Bencsura,P., Husza´r,K., Kulcsa´r,P.I., Ta´las,A., Fodor,E. and Welker,E. (2016) Cpf1 nucleases demonstrate robust activity to induce DNA modification by exploiting homology directed repair pathways in mammalian cells. Biol. Direct, 11, 46–14.   
42. Zeigerer,A., Gilleron,J., Bogorad,R.L., Marsico,G., Nonaka,H., Seifert,S., Epstein-Barash,H., Kuchimanchi,S., Peng,C.G., Ruda,V.M. et al. (2012) Rab5 is necessary for the biogenesis of the endolysosomal system in vivo. Nature, 485, 465–470.   
43. Reynolds,A., Leake,D., Boese,Q., Scaringe,S., Marshall,W.S. and Khvorova,A. (2004) Rational siRNA design for RNA interference. Nat. Biotechnol., 22, 326–330.   
44. Pei,Y. and Tuschl,T. (2006) On the art of identifying effective and specific siRNAs. Nat. Methods, 3, 670–676.   
45. Anderson,E.M., Birmingham,A., Baskerville,S., Reynolds,A., Maksimova,E., Leake,D., Fedorov,Y., Karpilow,J. and Khvorova,A. (2008) Experimental validation of the importance of seed complement frequency to siRNA specificity. RNA, 14, 853–861.   
46. Jackson,A.L., Burchard,J., Leake,D., Reynolds,A., Schelter,J., Guo,J., Johnson,J.M., Lim,L., Karpilow,J., Nichols,K. et al. (2006) Position-specific chemical modification of siRNAs reduces ‘off-target’ transcript silencing. RNA, 12, 1197–1205.   
47. Sergeeva,O., Sergeev,P., Melnikov,P., Prikazchikova,T., Dontsova,O. and Zatsepin,T. (2020) Modification of adenosine196 by Mettl3 methyltransferase in the $5^{\prime}$ -external transcribed spacer of 47S Pre-rRNA affects rRNA maturation. Cells, 9, 1061.   
48. Ingolia,N.T., Lareau,L.F. and Weissman,J.S. (2011) Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes. Cell, 147, 789–802.   
49. Martin,M. (2011) Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17, 10–12.   
50. Meyer,L.R., Zweig,A.S., Hinrichs,A.S., Karolchik,D., Kuhn,R.M., Wong,M., Sloan,C.A., Rosenbloom,K.R., Roe,G., Rhead,B. et al. (2013) The UCSC Genome Browser database: extensions and updates   
2013. Nucleic Acids Res., 41, D64–D69.   
51. Dobin,A., Davis,C.A., Schlesinger,F., Drenkow,J., Zaleski,C., Jha,S., Batut,P., Chaisson,M. and Gingeras,T.R. (2013) STAR: ultrafast universal RNA-seq aligner. Bioinformatics, 29, 15–21.   
52. Love,M.I., Huber,W. and Anders,S. (2014) Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol., 15, 550.   
53. Shahid,R., Bugaut,A. and Balasubramanian,S. (2010) The BCL-2 $!\,5^{\prime}$ untranslated region contains an RNA G-quadruplex-forming motif that modulates protein expression. Biochemistry, 49, 8300–8306.   
54. Lee,A.S., Kranzusch,P.J., Doudna,J.A. and Cate,J.H.D. (2016) eIF3d is an mRNA cap-binding protein that is required for specialized translation initiation. Nature, 536, 96–99.   
55. Lamper,A.M., Fleming,R.H., Ladd,K.M. and Lee,A.S.Y. (2020) A phosphorylation-regulated eIF3d translation switch mediates cellular adaptation to metabolic stress. Science, 370, 853–856.   
56. Hronova´,V., Mohammad,M.P., Wagner,S., Pa´nek,J., Guniˇsova´,S., Zeman,J., Poncova´,K. and Vala´ˇsek,L.S. (2017) Does eIF3 promote reinitiation after translation of short upstream ORFs also in mammalian cells? RNA Biol., 14, 1660–1667.   
57. Guan,B.-J., van Hoef,V., Jobava,R., Elroy-Stein,O., Valasek,L.S., Cargnello,M., Gao,X.-H., Krokowski,D., Merrick,W.C., Kimball,S.R. et al. (2017) A unique ISR program determines cellular responses to chronic stress. Mol. Cell, 68, 885–900.   
58. Gather,F., Schmitz,K., Koch,K., Vogt,L.-M., Pautz,A. and Kleinert,H. (2019) Regulation of human inducible nitric oxide synthase expression by an upstream open reading frame. Nitric Oxide, 88, 50–60.   
59. Calvo,S.E., Pagliarini,D.J. and Mootha,V.K. (2009) Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. U.S.A., 106,   
7507–7512.   
60. Ye,Y., Liang,Y., Yu,Q., Hu,L., Li,H., Zhang,Z. and Xu,X. (2015) Analysis of human upstream open reading frames and impact on gene expression. Hum. Genet., 134, 605–612.   
61. Hurtaud,C., Gelly,C., Bouillaud,F. and L´evi-Meyrueis,C. (2006) Translation control of UCP2 synthesis by the upstream open reading frame. Cell. Mol. Life Sci., 63, 1780–1789.   
62. Vattem,K.M. and Wek,R.C. (2004) Reinitiation involving upstream ORFs regulates ATF4 mRNA translation in mammalian cells. Proc. Natl. Acad. Sci. U.S.A., 101, 11269–11274.   
63. Zhou,D., Palam,L.R., Jiang,L., Narasimhan,J., Staschke,K.A. and Wek,R.C. (2008) Phosphorylation of eIF2 directs ATF5 translational control in response to diverse stress conditions. J. Biol. Chem., 283,   
7064–7073.   
64. Jackson,R.J., Hellen,C.U.T. and Pestova,T.V. (2012) Termination and post-termination events in eukaryotic translation. Adv. Protein Chem. Struct. Biol., 86, 45–93.   
65. Paolini,N.A., Moore,K.S., di Summa,F.M., Fokkema,I.F.A.C., ’t Hoen,P.A.C. and Lindern,M. (2018) Ribosome profiling uncovers selective mRNA translation associated with eIF2 phosphorylation in erythroid progenitors. PLoS ONE, 13, e0193790.   
66. Loughran,G., Sachs,M.S., Atkins,J.F. and Ivanov,I.P. (2012) Stringency of start codon selection modulates autoregulation of translation initiation factor eIF5. Nucleic Acids Res., 40, 2898–2906.   
67. Luukkonen,B.G., Tan,W. and Schwartz,S. (1995) Efficiency of reinitiation of translation on human immunodeficiency virus type 1 mRNAs is determined by the length of the upstream open reading frame and by intercistronic distance. J. Virol., 69, 4086–4094.   
68. Zhang,J. and Maquat,L.E. (1997) Evidence that translation reinitiation abrogates nonsense-mediated mRNA decay in mammalian cells. EMBO J., 16, 826–833.   
69. Kozak,M. (2001) Constraints on reinitiation of translation in mammals. Nucleic Acids Res., 29, 5226–5232.   
70. Andreev,D.E., Arnold,M., Kiniry,S.J., Loughran,G., Michel,A.M., Rachinskii,D. and Baranov,P.V. (2018) TASEP modelling provides a parsimonious explanation for the ability of a single uORF to derepress translation during the integrated stress response. Elife, 7, e03971.   
71. Kozak,M. (1991) A short leader sequence impairs the fidelity of initiation by eukaryotic ribosomes. Gene Expr., 1, 111–115.   
72. Matsuda,D. and Mauro,V.P. (2010) Determinants of initiation codon selection during translation in mammalian cells. PLoS ONE, 5, e15057.   
73. Gunnery,S., Ma¨ivali,U. and Mathews,M.B. (1997) Translation of an uncapped mRNA involves scanning. J. Biol. Chem., 272,   
21642–21646.   
74. De Gregorio,E., Preiss,T. and Hentze,M.W. (1998) Translational activation of uncapped mRNAs by the central part of human eIF4G is $5^{\prime}$ end-dependent. RNA, 4, 828–836.   
75. Henis-Korenblit,S., Strumpf,N.L., Goldstaub,D. and Kimchi,A. (2000) A novel form of DAP5 protein accumulates in apoptotic cells as a result of caspase cleavage and internal ribosome entry site-mediated translation. Mol. Cell. Biol., 20, 496–506.   
76. Terenin,I.M., Smirnova,V.V., Andreev,D.E., Dmitriev,S.E. and Shatsky,I.N. (2017) A researcher’s guide to the galaxy of IRESs. Cell. Mol. Life Sci., 74, 1431–1455.   
77. Tcherkezian,J., Cargnello,M., Romeo,Y., Huttlin,E.L., Lavoie,G., Gygi,S.P. and Roux,P.P. (2014) Proteomic analysis of cap-dependent translation identifies LARP1 as a key regulator of 5′TOP mRNA translation. Genes Dev., 28, 357–371.   
78. Shatsky,I.N., Terenin,I.M., Smirnova,V.V. and Andreev,D.E. (2018) Cap-independent translation: what’s in a name? Trends Biochem. Sci.,   
43, 882–895.   
79. Archer,S.K., Shirokikh,N.E., Beilharz,T.H. and Preiss,T. (2016) Dynamics of ribosome scanning and recycling revealed by translation complex profiling. Nature, 535, 570–574.   
80. Bohlen,J., Fenzl,K., Kramer,G., Bukau,B. and Teleman,A.A. (2020) Selective 40S footprinting reveals Cap-tethered ribosome scanning in human cells. Mol. Cell, 79, 561–574.   
81. Giess,A., Torres Cleuren,Y.N., Tjeldnes,H., Krause,M., Bizuayehu,T.T., Hiensch,S., Okon,A., Wagner,C.R. and Valen,E. (2020) Profiling of small ribosomal subunits reveals modes and regulation of translation initiation. Cell Rep., 31, 107534.   
82. Alekhina,O.M., Terenin,I.M., Dmitriev,S.E. and Vassilenko,K.S. (2020) Functional cyclization of eukaryotic mRNAs. Int. J. Mol. Sci., 21, 1677.   
83. Thoreen,C.C., Chantranupong,L., Keys,H.R., Wang,T., Gray,N.S. and Sabatini,D.M. (2012) A unifying model for mTORC1-mediated regulation of mRNA translation. Nature, 485, 109–113.   
84. Harris,T.E., Chi,A., Shabanowitz,J., Hunt,D.F., Rhoads,R.E. and Lawrence,J.C. (2006) mTOR-dependent stimulation of the association of eIF4G and eIF3 by insulin. EMBO J., 25, 1659–1668.   
85. Beck,M., Schmidt,A., Malmstroem,J., Claassen,M., Ori,A., Szymborska,A., Herzog,F., Rinner,O., Ellenberg,J. and Aebersold,R. (2011) The quantitative proteome of a human cell line. Mol. Syst. Biol., 7, 549.   
86. Menschaert,G., Van Criekinge,W., Notelaers,T., Koch,A., Crapp´e,J., Gevaert,K. and Van Damme,P. (2013) Deep proteome coverage based on ribosome profiling aids mass spectrometry-based protein and peptide discovery and provides evidence of alternative translation products and near-cognate translation initiation events. Mol. Cell. Proteomics, 12, 1780–1790.   
87. Van Damme,P., Gawron,D., Van Criekinge,W. and Menschaert,G. (2014) N-terminal proteomics and ribosome profiling provide a comprehensive view of the alternative translation initiation landscape in mice and men. Mol. Cell. Proteomics, 13, 1245–1261.   
88. Yun,Y., Adesanya,T.M.A. and Mitra,R.D. (2012) A systematic study of gene expression variation at single-nucleotide resolution reveals widespread regulatory roles for uAUGs. Genome Res., 22, 1089–1097.   
89. Sen,N.D., Zhou,F., Ingolia,N.T. and Hinnebusch,A.G. (2015) Genome-wide analysis of translational efficiency reveals distinct but overlapping functions of yeast DEAD-box RNA helicases Ded1 and eIF4A. Genome Res., 25, 1196–1205.  