# Multiplexed Analysis of Human uORF Regulatory Functions During the ISR Using PoLib-Seq  

Gemma E. May1, C. Joel McManus1,2  

1Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA, USA.   
2Computational Biology Department, Carnegie Mellon University, Pittsburgh, PA, USA.  

# Abstract  

Protein synthesis is a highly regulated essential process. As such, it is subjected to substantial regulation in response to stress. One hallmark of the Integrated Stress Response (ISR) is the immediate shutdown of most translation through phosphorylation of the alpha subunit of translation initiation factor eIF2 and activation of eIF4E binding proteins. While these posttranslational modifications largely inhibit cap-dependent translation, many mRNA resist this inhibition by alternative translation mechanisms involving cis-regulatory sequences and structures in $5^{\prime}$ transcript leaders, including upstream Open Reading Frames (uORFs), Internal Ribosome Entry Sites (IRESes), and Cap-Independent Translation Elements (CITEs). Studies of uORF and IRES activity are often performed on a gene-by-gene basis; however, high-throughput methods have recently emerged. Here, we describe a protocol for Polysome Library Sequencing (PoLibSeq; Fig. 1), a multiplexed assay of reporter gene translation that can be used during the ISR. A designer library of reporter RNAs are transfected into tissue-culture cells, and their translation is assayed via sucrose gradient fractionation followed by high-throughput sequencing. As an example, we include PoLib-seq results simultaneously assaying translation of wildtype and uORF mutant human ATF4 reporter RNAs, recapitulating the known function of uORF1 in resisting translational inhibition during the ISR.  

# Keywords  

mRNA translation; Polysome gradient fractionation; uORFs; Massively parallel reporter assay  

# 1 Introduction  

Inhibition of translation is a hallmark of the ISR. Multiple kinases, responding to a variety of stresses, phosphorylate the alpha subunit of eIF2, reducing the availability of ternary complex for translation initiation [1, 2]. While this globally represses mRNA translation, some stress-responsive transcripts continue to be translated through alternative initiation mechanisms controlled by cis-acting sequences and RNA structures located in $5^{\prime}$ transcript leaders [3]. For example, the transcript leader of human ATF4 encodes three uORFs (uORFs 0, 1, and 2; Fig. 1). Previous studies have shown that uORFs 1 and 2 in the homologous mouse ATF4 gene function to activate translation during the ISR [4]. After translation of the first uORF, which is only three amino acids in length, the small ribosomal subunit resumes scanning and quickly reacquires a new ternary complex. This allows subsequent  

reinitiation at uORF2, which overlaps the ATF4 CDS in an alternate reading frame and precludes translation of the ATF4 CDS. During the ISR, reduced levels of ternary complex delay translation reinitiation, allowing ribosomes to bypass uORF 2 and translate ATF4. Many other genes involved in the ISR are regulated via single uORFs, including CHOP, and GADD34 [1]. Thus, uORFs play a prominent role in the translational response to stress.  

Ribosome profiling studies have shown that uORFs are extremely common, being found at ${\sim}15\%$ of yeast and $-50\%$ of human genes, respectively [5, 6]. Although the number of predicted uORFs has increased dramatically, relatively few have been experimentally tested for functions in regulating translation, especially during the ISR. Historically, uORF activity has been tested on a gene-by-gene basis primarily through the use of luciferase reporter assays. Usually, matched wild-type and uORF mutant transcript leaders are cloned upstream of luciferase and separately transfected to compare the amount of functional protein from each reporter, relative to reporter mRNA levels. Although effective, the relatively low throughput of these reporter assays is the chief reason why most uORFs have not been tested.  

Over the past decade, several labs have developed Massively Parallel Reporter Assays (MPRAs) for high-throughput measurements of transcription [7, 8], splicing [9, 10], and translation [11-15]. Some translation MPRAs involve FACS-binning cells based on fluorescent reporter expression, while others separate reporter mRNAs based on their association with ribosomes through polysome sucrose-gradient fractionation. Separation by polysome fractionation has the advantage that it can be used to investigate acute translational responses to stress, as are seen in the ISR. However, this has not been reported previously. Here, we describe our protocol for Polysome Library Sequencing (PoLib-seq) in human tissue culture cells. We include PoLib-seq results using three ATF4 wild-type and uORFmutant reporters as a proof-of-principle, recapitulating the established roles of uORFs 1 and 2 in the ATF4 translational response to stress (Fig. 2). By extension, large designer libraries cloned from pools of synthetic oligos [16] could be used to apply PoLib-seq to thousands of designer reporter RNAs simultaneously.  

# 2 Materials  

1. Pipettes with filter tips (P1000, P200, P20, P2).   
2. Monster Green® Fluorescent Protein phMGFP vector (Promega).   
3. Custom primers (Table 1).   
4. $1.5{\-}\mathrm{ml}$ and 2.0-ml microcentrifuge tubes.   
5. $0.2–\mathrm{ml}$ tubes for PCR.   
6. Q5 Polymerase (New England Biolabs).   
7. 10 mM dNTP mix.   
8. Nuclease-free water.   
9. Thermocycler, any model.   
10. Agarose.   
11. TAE buffer (Tris–acetate EDTA buffer for gel electrophoresis).   
12. SYBR safe DNA gel stain (or equivalent).   
13. DpnI enzyme and CutSmart  buffer (New England Biolabs).   
14. DNA Clean and Concentrator kit (Zymo Research).   
15. NanoDrop  2000 Spectrophotometer or similar.   
16. HiScribe T7 Quick High Yield RNA Synthesis kit (New England Biolabs).   
17. 3 M sodium acetate pH 5.2.   
18. Acid-Phenol:Chloroform pH 4.5 (with IAA, 125:24:1).   
19. Chloroform:Isoamyl alcohol 24:1.   
20. Ethanol, absolute 200 proof (EtOH, molecular biology grade).   
21. $80\%$ Ethanol: $80\%$ absolute ethanol, $20\%$ nuclease-free water.   
22. Refrigerated centrifuge with rotor for 1.5- to $2.0{\-m}1$ microcentrifuge tubes.   
23. Thermomixer with $1.5{\-}\mathrm{ml}$ tube capacity (or similar).   
24. Vaccinia Capping System (New England Biolabs).   
25. 2-propanol (isopropanol).   
26. HEK 293T cell line.   
27. Fetal bovine serum (FBS).   
28. Dulbecco’s Modified Eagle Media (DMEM).   
29. Disposable pipettes (25 ml, 10 ml, 5 ml).   
30. $10{\mathrm{-cm}}$ tissue culture–treated culture dish.   
31. Incubator suitable for tissue culture, supplied with $5\%$ $\mathrm{CO}_{2}$ .  
32. $50{\-}\mathrm{ml}$ conical tube.   
33. Steriflip-GP sterile $50{\-}\mathrm{ml}$ centrifuge tube top filter unit (MilliporeSigma ).   
34. Polysome Lysis Buffer (PLB): $20\,\mathrm{mM}$ Tris–HCl pH 7.5, 250 mM NaCl, 15   
mM $\mathrm{MgCl}_{2}$ ,$0.5\%$ Triton  X-100. Prepare $50\;\mathrm{ml}$ in nuclease-free water. Filter   
sterilize using Steriflip-GP sterile $50{\-}\mathrm{ml}$ centrifuge tube top filter unit according   
to the manufacturer’s instructions. Store at $4\,^{\circ}\mathbf{C}$ .  
35. Opti-MEM Reduced Serum Media.   
36. Lipofectamine  MessengerMAX  transfection reagent (Life Technologies).   
37. $50\,\mathrm{mM}$ sodium arsenite: Mix $0.065\mathrm{~g~}$ sodium arsenite in $10\,\mathrm{ml}$ of nuclease-free   
water. Store at room temperature.  

38. $50\,\mathrm{mg/ml}$ cycloheximide: Mix $50~\mathrm{mg}$ cycloheximide in $1\;\mathrm{ml}$ ethanol. Store at $-20~^{\circ}\mathrm{C}$ .  
39. 1 M DL-dithiothreitol (DTT).   
40. SUPERase-In™ RNase inhibitor, $20\,\mathrm{U/\upmul}$ (Invitrogen™).   
41. TURBO™ DNase, 2 U/ml (Invitrogen™).   
42. Phosphate-buffered saline (PBS).   
43. Cell scraper.   
44. $10{\-}\mathrm{ml}$ syringe with Luer Tip.   
45. 20 gauge needle.   
46. TRIzol™ Reagent.   
47. RNA Clean and Concentrator kit (Zymo Research).   
48. RNA reagents ScreenTape (Agilent Technologies).   
49. TapeStation Instrument (Agilent Technologies, model 4400 or similar).   
50. Sucrose.   
51. $10\%$ Polysome Sucrose-Gradient Solution ( $10\%$ PLB): $5\:\mathrm{g}$ of sucrose, $1\;\mathrm{ml}$ of 1 M Tris-HCl pH 7.5, 3.125 ml of $4\;\mathrm{M}\;\mathrm{NaCl}$ ,$750\,\upmu\mathrm{l}$ of $\mathrm{1}\mathrm{~M~MgCl}_{2}$ ,$250\,\upmu\mathrm{l}$ of Triton $^{\mathrm{{TM}}}\mathrm{{X}}{-}100$ , and nuclease-free water to $50\;\mathrm{ml}$ . Agitate until the sucrose is fully dissolved into solution. Filter sterilize using Steriflip-GP sterile $50{\-}\mathrm{ml}$ centrifuge tube top filter unit according to the manufacturer’s instructions. Store at $4\,^{\circ}\mathbf{C}$ .  
52. $50\%$ Polysome Sucrose-Gradient Solution $50\%$ PLB): $25~\mathrm{g}$ of sucrose, $1\;\mathrm{ml}$ of 1 M Tris–HCl pH 7.5, 3.125 ml of $4\;\mathrm{M}\;\mathrm{NaCl}$ ,$750\,\upmu\mathrm{l}$ of $1\ \mathrm{M}\ \mathrm{MgCl}_{2}$ ,$250\,\upmu\mathrm{l}$ of Triton  X-100, and nuclease-free water to $50\;\mathrm{ml}$ . Agitate until the sucrose is fully dissolved into solution. Filter sterilize using Steriflip-GP sterile $50{\-}\mathrm{ml}$ centrifuge tube top filter unit according to the manufacturer’s instructions. Store at $4\,^{\circ}\mathbf{C}$ .  
53. Polyclear™ open top ultracentrifuge tubes $9/16^{\prime\prime}\times3\,\%^{\prime\prime}$ (Seton Scientific 7031).   
54. Silicone grease.   
55. Stainless steel 304 syringe needle.   
56. Lint-free wipes.   
57. Ultracentrifuge with Beckman Coulter SW 40 Ti swinging bucket rotor (Beckman Coulter).   
58. Gradient station  with gradient forming attachments (Biocomp Instruments).   
59. Gilson Fraction collection system with Triax flow cell and fractionating tube mount (Biocomp Instruments).   
60. Sodium dodecyl sulfate (SDS).   
61. $70\%$ ethanol ( $70\%$ EtOH): $70\%$ ethanol, $30\%\:\mathrm{ddH}_{2}\mathrm{O}$ .  
62. SuperScript  IV Reverse Transcriptase (Thermo Fisher).   
63. Magnetic separation rack for $0.2–\mathrm{ml}$ tubes.   
64. AMPure  XP magnetic PCR purification beads (Beckman Coulter).   
65. $100\,\upmu\mathrm{M}$ forward N primer mix: $10\,\upmu\mathrm{l}$ of each of the following: $100\,\upmu\mathrm{M}$ ATF4_PCR1_N0-F, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N1-F, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N2- F, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N3-F, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N4-F, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N5-F, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N6-F, and $100\,\upmu\mathrm{M}$ ATF4_PCR1_N7- F (Table 1).   
66. $100\,\upmu\mathrm{M}$ reverse N primer mix: $10\,\upmu\mathrm{l}$ of each of the following: $100\,\upmu\mathrm{M}$ ATF4_PCR1_N0_R, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N2_R, $100\,\upmu\mathrm{M}$ ATF4_PCR1_N4_R, and $100\,\upmu\mathrm{M}$ ATF4_PCR1_N6_R (Table 1).   
67. NEBNext  Multiplex Oligos for Illumina (unique dual index primer pairs, New England Biolabs).   
68. D1000 ScreenTape and sample buffer (Agilent).   
69. Qubit  dsDNA HS Assay Kit (Invitrogen).   
70. $0.5{\-}\mathrm{ml}$ optically clear individual PCR tube.  

# 3 Methods  

# 3.1  In Vitro RNA Transcription Template Preparation  

In this section, a template for in vitro transcription is produced by PCR amplification of the phMGFP vector (Fig. 3). The forward primer, phMGFP-T7-F (Table 1), anneals to the T7 polymerase site upstream of the $5^{\prime}\,\mathrm{LS}$ , and the reverse primer, phMGFP-GFP3p-R (Table 1), anneals downstream of the GFP (green fluorescent protein) sequence. The reverse primer also incorporates a 70-nucleotide-long poly A tail sequence into the PCR product. For this step, the vector template can contain either a single $5^{\prime}$ leader sequence or a library of up to thousands of leader sequences.  

1. Dilute the plasmid vector containing the $5^{\prime}$ leader sequence (s) of interest to 1 $\mathrm{ng/\upmul}$ in nuclease-free water. 2. Set up the following reaction in a $0.2–\mathrm{ml}$ PCR tube on ice.  

![](images/bae4c93daa97a2606856282c283fd994dfe44ea983f47d8593b770bb778dd798.jpg)  

![](images/15c0a71b1b561c68006c588e475e1ceb274d3fd24379bee905cfdf8692790ae5.jpg)  

3. Mix the reaction by setting a pipettor to $50\,\upmu\mathrm{l}$ and pipetting up and down several times.   
4. Put the reaction in a thermocycler and run the program: a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .b. 35 cycles of $98\ ^{\circ}\mathbf{C}$ for $30{\mathrm{~s}}$ ,$55\ ^{\circ}\mathrm{C}$ for $30{\mathrm{~s}}$ , and $72\ ^{\circ}\mathbf{C}$ for $45\mathrm{~s~}$ (or 40 s per kb of product).   
5. Run $10\,\upmu\mathrm{l}$ of the PCR product on a $1\%$ agarose gel in $1\times$ TAE to check for successful amplification.   
6. Purify the PCR product using the Zymo DNA clean & concentrator kit (or similar) according to the manufacturer’s instructions. Elute the PCR product in $45\,\upmu\mathrm{l}$ of nuclease-free water.   
7. Set up the following reaction on ice.   
8. Mix the reaction by setting pipettor to $25\,\upmu\mathrm{l}$ and pipetting up and down several times.   
9. Incubate the reaction for $^{1\,\mathrm{h}}$ at $37\ ^{\circ}\mathbf{C}$ in a thermocycler.   
10. Purify the PCR product using the Zymo DNA clean $\&$ concentrator kit according to the manufacturer’s instructions. Elute the PCR product in $10\,\upmu\mathrm{l}$ of nucleasefree water.   
11. Measure the concentration of the PCR product using a Nano-Drop spectrophotometer or something similar.  

![](images/d0e250cd1b38f364229922f259d32efb889a58b1f750e9b05fd5478520cdbafe.jpg)  

# 3.2  In Vitro RNA Transcription  

1. In a $0.2–\mathrm{ml}$ PCR tube, dilute the PCR product from Subheading 3.1, step 12 to 1 $\upmu\mathrm{g}$ in $8\,\upmu\mathrm{l}$ of nuclease-free water.   
2. Add $10\,\upmu\mathrm{l}$ of NTP Buffer mix followed by $2\;\upmu\mathrm{l}$ of T7 RNA polymerase mix from the HiScribe  T7 Quick High Yield RNA Synthesis Kit.   
3. Mix by setting a pipettor to $10\,\upmu\mathrm{l}$ and pipetting up and down several times. The total volume of the reaction should be $20\,\upmu\mathrm{l}$ .  
4. Incubate the reaction for $2\,\mathrm{{h}}$ at $37\ ^{\circ}\mathbf{C}$ in a thermocycler.   
5. Add $30\,\upmu\mathrm{l}$ of water to the reaction, followed by $2\;\upmu\mathrm{l}$ DNase I (supplied in the kit) to the reaction. Incubate for $15\;\mathrm{min}$ at $37\ ^{\circ}\mathbf{C}$ in a thermocycler.   
6. Transfer the reaction to a $1.5{\-}\mathrm{ml}$ tube. Add $150\,\upmu\mathrm{l}$ of nuclease-free water, and 20 $\upmu\mathrm{l}$ of $3\,\mathrm{M}$ sodium acetate, $\mathrm{pH}\,5.2$ . Mix well using a pipettor, or briefly vortex.   
7. Add $250\,\upmu\mathrm{l}$ of acid-phenol:chloroform $\mathrm{pH}\,4.5$ (with IAA, 125: 24:1). Vortex on high for $15\;\mathrm{s}$ .  
8. Centrifuge at $14\mathrm{,}000\times g$ for $2\,\mathrm{min}$ at room temperature. Remove the supernatant to a new $1.5{\-}{\mathrm{ml}}$ tube.   
9. Add $300\,\upmu\mathrm{l}$ of chloroform:isoamyl alcohol 24:1 to the supernatant. Vortex on high for $15\;\mathrm{s}.$ . Repeat step 8.   
10. Add $600\,\upmu\mathrm{l}$ of ethanol to the supernatant. Briefly vortex to mix and precipitate at −20 °C overnight, or for at least several hours.   
11. Centrifuge the RNA at $20{,}000\times g$ for $30\;\mathrm{min}$ at $4\,^{\circ}\mathbf{C}$ .  
12. Carefully remove the supernatant as to not disturb the pellet.   
13. Wash the pellet by adding $1\;\mathrm{ml}$ of $80\%$ ethanol. Briefly vortex and centrifuge the RNA at $20{,}000\times g$ for $10\;\mathrm{min}$ at $4\,^{\circ}\mathbf{C}$ .  
14. Carefully remove the supernatant as to not disturb the pellet. Leave the lid open and let the remaining ethanol evaporate (roughly $2{-}5\;\mathrm{min}$ ). A clear or slightly white pellet should be visible at the bottom of the tube.   
15. Resuspend the pellet in $50\,\upmu\mathrm{l}$ of nuclease-free water.   
16. Measure the concentration of the RNA using a nanodrop. This transcription reaction produces up to about $180\,\upmu\mathrm{g}$ of RNA.   
17. Save the RNA at $-20~^{\circ}\mathrm{C}$ overnight or continue on to Subheading 3.3.  

# 3.3  5′ Capping Reaction  

1. Combine $100\,\upmu\mathrm{g}$ of RNA (step 17) with nuclease-free water in a $1.5{\-}{\mathrm{ml}}$ tube to a final volume of $150\,\upmu\mathrm{l}$ .  
2. Denature the RNA by heating the RNA for $5\;\mathrm{min}$ at $65\;^{\circ}\mathbf{C}.$ . After the incubation is complete, immediately place the tube on ice for $5\;\mathrm{min}$ .  
3. Proceed with the Vaccina Capping System kit. While the RNA is incubating on ice, prepare a $2\;\mathrm{mM}$ solution of SAM by adding $2\;\upmu\mathrm{l}$ of $32\;\mathrm{mM}$ SAM to $30\,\upmu\mathrm{l}$ nuclease-free water. Save the $2\;\mathrm{mM}$ SAM on ice.   
4. Set up the following reaction on ice in a $1.5{\-}\mathrm{ml}$ tube in the order specified.   
5. Mix by setting a pipettor to $50\,\upmu\mathrm{l}$ and pipetting up and down several times.   
6. Aliquot the reaction into five $0.2{\cdot}{\mathrm{ml}}$ PCR tubes with $50\,\upmu\mathrm{l}$ in each tube. Incubate the tubes at $37\ ^{\circ}\mathbf{C}$ in a thermocycler for $30\,\mathrm{min}$ .  
7. Combine the reactions into one $1.5{\-}\mathrm{ml}$ tube. Add $700\,\upmu\mathrm{l}$ of nuclease-free water, and $100\,\upmu\mathrm{l}$ of $3\,\mathrm{M}$ sodium acetate, $\mathrm{pH}\,5.2$ to each tube. Briefly vortex.   
8. Divide the reaction into two $1.5{\-}\mathrm{ml}$ tubes by adding $500\,\upmu\mathrm{l}$ of the reaction to each tube.   
9. To each tube, add $500\,\upmu\mathrm{l}$ of acid-phenol:chloroform $\mathrm{pH}\,4.5$ (with IAA, 125:24:1). Vortex on high for $15\;\mathrm{s}$ .  
10. Centrifuge at $14\mathrm{,}000\times g$ for $2\,\mathrm{min}$ at room temperature. Remove the supernatants to new $1.5{\-}\mathrm{ml}$ tubes.   
11. Add $500\,\upmu\mathrm{l}$ of chloroform:isoamyl alcohol 24:1 to the supernatants. Vortex on high for $15\;\mathrm{s}$ .  
12. Centrifuge at $14\mathrm{,}000\times g$ for $2\,\mathrm{min}$ at room temperature. Remove each supernatant to one $2.0{-}\mathrm{ml}$ tube. The volume of the supernatants is typically about $800{-}900\,\upmu\mathrm{l}$ at this step.   
13. Add one volume of isopropanol. Briefly vortex to mix and precipitate at $-20\,^{\circ}\mathrm{C}$ overnight, or for at least several hours.   
14. Centrifuge the RNA at $20{,}000\times g$ for $30\;\mathrm{min}$ at $4\,^{\circ}\mathbf{C}$ .  
15. Carefully remove the supernatant as to not disturb the pellet.   
16. Wash the pellet by adding $1\;\mathrm{ml}$ of $80\%$ ethanol. Briefly vortex and centrifuge the RNA at $20{,}000\times g$ for $10\;\mathrm{min}$ at $4\,^{\circ}\mathbf{C}$ .  
17. Carefully remove the supernatant as to not disturb the pellet. Leave the lid open and let the remaining ethanol evaporate (roughly $2{-}5\;\mathrm{min}$ ).   
18. Resuspend the pellet in $50\,\upmu\mathrm{l}$ of nuclease-free water.   
19. Measure the concentration of the RNA using a nanodrop. $10\,\upmu\mathrm{g}$ of RNA is required for each transfection. It may be necessary to repeat Subheadings 3.2 and 3.3 to obtain enough capped RNA for multiple transfections.   
20. Save the RNA at $-20~^{\circ}\mathrm{C}$ overnight or continue on to Subheading 3.4.  

![](images/cec690efa379ef4b3adbfb5bc0d139892aaaabec96af3252ce079ecafd533a19.jpg)  

![](images/cf7bc1d0e9942cbb4c79e565e7ce4b40d8cc48463f79222a54398eee95692205.jpg)  

# 3.4  HEK 293 T Cell In Vitro RNA Transfection  

1. For each transfection, seed two million cells in a $10{\mathrm{-cm}}$ tissue culture dish in 10 $\mathrm{ml\;DMEM}+10\%$ FBS. Incubate at $37\ ^{\circ}\mathbf{C}$ in an incubator supplemented with $5\%$ $\mathrm{CO}_{2}$ for $24\,\mathrm{{h}}$ .  
2. Prepare $50\;\mathrm{ml}$ of Polysome Lysis Buffer (PLB, see Subheading 2).   
3. For each transfection, add $30\,\upmu\mathrm{l}$ of Lipofectamine  MessengerMAX transfection reagent to $500\,\upmu\mathrm{l}$ of Opti-MEM Reduced Serum Media in a $1.5{\-}\mathrm{ml}$ tube. Incubate at room temperature for $10\;\mathrm{min}$ .  
4. In a separate $1.5{\-}{\mathrm{ml}}$ tube, add $10\,\upmu\mathrm{g}$ of RNA (Subheading 3.3, step 20) with 500 $\upmu\mathrm{l}$ of Opti-MEM Reduced Serum Media.   
5. Combine the Lipofectamine complexes (step 3) with the RNA (step 4) into one tube. Briefly mix with a pipettor set to $500\,\upmu\mathrm{l}$ . Incubate the transfection mixture at room temperature for $5\;\mathrm{min}$ .  
6. Remove the cells from the $\mathrm{CO}_{2}$ incubator and add the transfection mixture dropwise to the HEK 293T cells while gently swirling the dish. Place the cells back in the incubator. Incubate for $5.5\,\mathrm{h}$ for non-stress conditions, or $^{5\,\mathrm{h}}$ for assays under stressed conditions.   
7. For the stress condition, after $^{5\,\mathrm{h}}$ of incubation, remove the tissue culture dish from the incubator. Add $8\,\upmu\mathrm{l}$ of $50\;\mathrm{mM}$ sodium arsenite to the dish for a final sodium arsenite concentration of $40\,\upmu\mathrm{M}$ . Briefly swirl the media in the dish. Place the tissue culture dish back into the incubator and incubate for $30\;\mathrm{min}$ .  
8. While the cells are incubating, prepare enough PLB for each transfection $(500\,\upmu\mathrm{l}$ per transfection) on ice by adding $1\ \upmu\mathrm{l}$ of $1\,\mathrm{M}$ DTT, $12\,\upmu\mathrm{l}$ of TURBO  DNase, 2 $\upmu\mathrm{l}$ of $50\;\mathrm{mg/ml}$ cycloheximide, and $1\ \upmu\mathrm{l}$ of SUPERase- $\cdot\mathrm{In}^{\mathrm{TM}}$ RNase inhibitor to 1 ml of PLB. Save the PLB on ice.   
9. Prepare one $10{\-}\mathrm{ml}$ syringe with a $20\,\mathrm{G}$ needle attached per transfection.   
10. Prepare $10\;\mathrm{ml}$ of ice-cold PBS per transfection. This can be done by aliquoting $10\,\mathrm{ml}$ of PBS into a $15{\-}\mathrm{ml}$ conical and burying the conical in ice in an ice bucket.  

11. Harvest the cells for each transfection as follows.  

a. Remove the tissue culture dish containing the cells from the incubator into a biosafety cabinet.   
b. Remove $0.5\;\mathrm{ml}$ of the media from the dish into a $1.5{\-}\mathrm{ml}$ tube.   
c. Add $20\,\upmu\mathrm{l}$ of $50\;\mathrm{mg/ml}$ cycloheximide to the $0.5\;\mathrm{ml}$ of media and briefly vortex the tube.   
d. Add the media containing the cycloheximide to the cells in the tissue culture dish. Briefly swirl the media in the dish. Place the cells back into the incubator and incubate for $2\;\mathrm{min}$ .  
e. Remove the cells to a biosafety cabinet. Using a $10{\-}\mathrm{ml}$ pipette, quickly pipette all of the media off of the cells.   
f. Gently wash the cells by adding $10\;\mathrm{ml}$ of ice-cold PBS to the cells by pipetting down the side of the dish, taking care to not disturb the cells.   
g. Gently swirl the dish and place the dish onto ice.   
h. Quickly remove all of the PBS using a $10{\-}\mathrm{ml}$ pipette.   
i. Add $500\,\upmu\mathrm{l}$ of the prepared PLB (step 8) dropwise to the cells.   
j. Tilt the dish to about a $20^{\circ}$ angle while keeping the dish on ice.   
k. Using a cell scraper, scrape all of the cells down to the bottom of the dish that is closest to the ice.   
l. Titrate the cells in the dish by passing the cells through a $20\:\mathrm{G}$ needle about ten times.   
m. Remove the lysed cells to a $1.5{\-}\mathrm{ml}$ tube on ice. Incubate the lysate on ice for $10\,\mathrm{min}$ .  

12. Once all of the lysates are collected, centrifuge the lysates for $10\;\mathrm{min}$ at $20{,}000\times$ $_{g}$ at $4\,^{\circ}\mathbf{C}$ .  

13. The cell debris should be collected at the bottom of the tube. Remove the lysate without disturbing the cell debris to a new tube. The total volume of the lysate at this point is usually about $0.6\;\mathrm{ml}$ .  
14. For the total RNA sample, remove $50\,\upmu\mathrm{l}$ of lysate to a new $1.5{\-}{\mathrm{ml}}$ tube. Proceed to Subheading 3.6.   
15. Freeze and store the lysates at $-80~^{\circ}\mathrm{C}$ . The lysates can be stored overnight or for up to several months.  

# 3.5  Polysome Fractionation  

1. Turn on and cool the ultracentrifuge to $4\,^{\circ}\mathbf{C}$ . Also cool the SW40 rotor to $4\,^{\circ}\mathbf{C}$ (keep in a cold room or refrigerator overnight).   
2. Prepare $6.5\;\mathrm{ml}$ each of $10\%$ and $50\%$ polysome sucrose-gradient solution ( $10\%$ PSG and $50\%$ PSG, see Subheading 2) per gradient by adding the reagents to the solutions as follows. Save the solutions on ice.  

![](images/1e9c1ccf0624c271058b1ce3c2593e61b595d722c0ee6f4e134da5e56cd639b9.jpg)  

3. Place each ultracentrifuge tube into the marker block and mark the tube along the upper marking edge of the block. Lightly grease the inside of the tube with  

silicone grease using the provided bolt and piston tip. Label the ultracentrifuge tube with the sample name.   
4. Prepare two $10{\-}\mathrm{ml}$ syringes with stainless steel syringe needles. To fill one syringe with approximately $7\;\mathrm{ml}$ of $10\%$ PSG, pull the plunger up and down a few times while the syringe is submerged in the $10\%$ PSG. This removes any air bubbles. Gently fill an ultracentrifuge tube with ${\sim}6.5\;\mathrm{ml}$ of $10\%$ PSG, just so the liquid reaches the marked line.   
5. Fill the other syringe with approximately $7\;\mathrm{ml}$ of $50\%$ PSG (again, pull the plunger up and down in the PSG to remove the bubbles) and dry the outside of the tip using a lint-free wipe. Gently place the tip of the metal syringe tip at the bottom of the ultracentrifuge and slowly press the syringe plunger down. Observe the heavier, $50\%$ PSG solution, displace the $10\%$ PSG solution as it moves the $10\%$ solution up the tube. Keep pressing the plunger down until the top of the $50\%$ solution is at the marker line. Carefully remove the syringe as to not disrupt the layers. Repeat steps 4 and 5 for all of the gradients.   
6. Place the rubber lids onto the ultracentrifuge tubes so that the small hole in the side of the lid is the last to be placed into the tube.   
7. Turn on the gradient station and level the platform and tube holder where the gradients will be placed.   
8. Carefully transfer the ultracentrifuge tubes to the gradient station tube holder. Run the $10{-}50\%$ gradient program on the gradient station (see Note 1).   
9. While the gradient station is running, thaw the lysates on ice if they were previously frozen.   
10. Once the gradient station has finished the program, remove the ultracentrifuge tubes from the tube holder and gently place them on ice. Be sure that the tubes are sitting in the ice so that they are upright and not at an angle.   
11. Remove the rubber lids. Using a pipette, remove the equivalent volume of lysate plus an additional $100\,\upmu\mathrm{l}$ . For example, if the volume of the lysate is $600\,\upmu\mathrm{l}$ ,remove $700\,\upmu\mathrm{l}$ of the sucrose gradient from the top of the tube.   
12. Place the gradients into the SW40 buckets. Be sure to record which ultracentrifuge tube was placed into each bucket.   
13. Using a pipettor, gently layer the lysate on top of the gradients by pipetting the sample down the side of the tube. This avoids dropping the sample directly on top of the gradient which could disrupt the layers.   
14. Weigh each bucket with the gradients in them (along with the caps to the buckets) and balance the buckets that will be across from each other in the rotor using the PLB stock solution.   
15. Cap the buckets and place them into the rotor. If there are empty buckets, be sure to load those as well. If there are an odd number of samples, prepare a mock gradient and place it in the bucket opposite of the odd tube.   
16. Load the rotor into the centrifuge. Set the acceleration and de-acceleration to 1 and 7, respectively. Centrifuge the gradients at 35,600 rpm $^{160,000\times g)}$ for 2.5 $_\mathrm{h}$ at $4\,^{\circ}\mathbf{C}$ with the vacuum on.   
17. While the centrifuge is running, clean the metal syringe tips and rubber caps by soaking in $10\%$ bleach for $30\;\mathrm{min}$ , followed by rinsing with distilled water.   
18. Label thirty $1.5{\-}{\mathrm{ml}}$ microcentrifuge tubes per sample and set aside.   
19. See Note 2. Zero the Triax flow cell using $\mathrm{ddH}_{2}\mathrm{O}$ . Leave the fraction collection system on to warm up for at least $30\;\mathrm{min}$ before use.   
20. Remove the rotor from the ultracentrifuge and save the buckets with the gradients on a stable surface as to not tilt the buckets—in an ice bucket, a cold room, or a fridge.   
21. Place thirty $1.5{\-}{\mathrm{ml}}$ labeled tubes with the lids open, in the tube holder for the fractionator in the correct orientation ( $s e e$ fractionator’s user manual).   
22. Put a silicone-greased piston tip onto the piston and load one sample under the piston.   
23. Set up the fractionator to fractionate the sample into 30 fractions using the default settings. Each fraction will be about $400\;\mathrm{ml}$ , with the exception of the last fraction which will be less.   
24. After the fractionator has finished with fractionating one gradient, rinse the system following the prompts, with $\mathrm{ddH}_{2}\mathrm{O}$ . Repeat steps 21–23 for each gradient.   
25. After the last fraction, follow the manufacturer’s suggested protocol for cleaning and storing the instrument. We run mock samples filled with warm $0.5\%$ SDS followed by $\mathrm{ddH}_{2}\mathrm{O}$ . Then, the instrument lines are rinsed with $70\%$ EtOH and air is pumped through to dry.   
26. At this point, the fractions can be saved at $-20~^{\circ}\mathbf{C}.$ , or they can be combined as desired (next steps).   
27. Examine the polysome traces to determine which fractions are representative of the peaks of interest. Combine the fractions as needed so that the total volume of the combined fractions is $400\,\upmu\mathrm{l}$ . For example, if fraction numbers 9 and 10 are where the monosome peak is on the polysome trace, combine $200\,\upmu\mathrm{l}$ of fraction 9 and $200\,\upmu\mathrm{l}$ of fraction 10 into one 1.5-ml ultracentrifuge tube.   
28. Either save the combined fractions at $-20~^{\circ}\mathrm{C}$ or proceed to Subheading 3.6.  

# 3.6  RNA Extraction  

Perform steps 1, 2, and 4 in a fume hood. Dispose of TRIzol  reagent and chloroform as chemical hazardous wastes.  

1. Add $800\,\upmu\mathrm{l}$ of TRIzol  reagent to each of the $400\,\upmu\mathrm{l}$ of combined fractions (or $50\,\upmu\mathrm{l}$ of total unfractionated lysate, Subheading 3.4, step 14) and vortex briefly. Incubate at room temperature for $5\;\mathrm{min}$ .  
2. Add $300\,\upmu\mathrm{l}$ of chloroform:isoamyl alcohol 24:1 to the TRIzol  Reagent/lysate mix and briefly vortex. Incubate at room temperature for $2~\mathrm{min}$ .  
3. Centrifuge at $12{,}000\times g$ for $15\;\mathrm{min}$ at $4\,^{\circ}\mathbf{C}$ .  
4. Carefully remove the supernatant (roughly $900\,\upmu\mathrm{l}$ ) to a new $2.0{\-m}1$ tube.   
5. Add one volume of ethanol and briefly vortex.   
6. Purify the RNA using a Zymo RNA clean and concentrator-5 kit as per the manufacturer’s instructions, and as listed in the subsequent steps. All of the centrifuge spins are at $12{,}000\times g$ at room temperature.   
7. Transfer up to $700\,\upmu\mathrm{l}$ of the sample to the spin column and centrifuge for $30\;\mathrm{s}$ .Discard flow through. Repeat this step as necessary until all of the sample from step 5 has been added to the column.   
8. Add $400\,\upmu\mathrm{l}$ of RNA Prep Buffer and centrifuge for $30\;\mathrm{s}$ . Discard the flow through.   
9. Add $700\,\upmu\mathrm{l}$ of RNA Wash Buffer and centrifuge for $30\;\mathrm{s}$ . Discard the flow through.   
10. Add $400\,\upmu\mathrm{l}$ of RNA Wash Buffer and centrifuge for $30{\mathrm{~s}}.$ . Discard the flow through.   
11. Centrifuge the column for an additional $2\;\mathrm{min}$ to dry the membrane.   
12. Transfer the column to a clean $1.5{\-}\mathrm{ml}$ tube with the cap removed. Add $45\,\upmu\mathrm{l}$ of nuclease-free water to the column membrane.   
13. Elute the RNA by centrifuging the column at room for $1\ \mathrm{min}$ .  
14. Transfer the eluate to a fresh $1.5{\-}\mathrm{ml}$ tube.   
15. For total RNA (unfractionated) samples, check the quality of the lysate preparation by assaying RNA integrity. Run $1\;\upmu\mathrm{l}$ of the total RNA on an RNA ScreenTape using an Agilent TapeStation, following the manufacturer’s instructions. The RNA Integrity Value (RIN) should be at least 9.0 for highquality RNA. The typical range of RIN numbers for this assay ranges between 9.5 and 10.0.   
16. Save the RNA at $-20~^{\circ}\mathrm{C}$ or proceed to Subheading 3.7.  

# 3.7  DNase Treatment  

1. Set up a DNase treatment reaction in a $0.2–\mathrm{ml}$ PCR tube as follows.  

![](images/a33c2d4eac3a19a8842203a44cf7ef1689c125eadadcc05b9eaa027cfda767c7.jpg)  

2. Pipette up and down several times with a P200 pipettor set to $45\,\upmu\mathrm{l}$ to mix.   
3. Incubate the reaction at $37\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{min}$ .  
4. Purify the RNA over an RNA clean and concentrator—5 column following the   
manufacturer’s instructions. Elute the RNA in $15\,\upmu\mathrm{l}$ of nuclease-free water.   
5. Quantify the RNA using a nanodrop.   
6. Either save the RNA at $-20~^{\circ}\mathrm{C}$ or proceed to Subheading 3.8.  

# 3.8  Reverse Transcription  

1. In a $0.2–\mathrm{ml}$ tube, set up the reaction on ice as follows.  

![](images/8688e3566ef75dab0c9968c7b2d3d4822ce44d16617b2826f0e2f447847b385b.jpg)  

2. Incubate the reaction for $5\;\mathrm{min}$ at $65\ ^{\circ}\mathrm{C}$ in a thermocycler. Place the reaction on ice immediately after the incubation is complete.  

3. To the reaction, add the following.  

![](images/ac774e25302dfcc8ba16361639d478eee9faba051f38bce7d47a44cf2d345047.jpg)  

4. In a thermocycler, incubate the reaction for $30\,\mathrm{min}$ at $50\;^{\circ}\mathbf{C}$ . Heat inactivate at 80 $^\circ C$ for $10\,\mathrm{min}$ .  
5. Add $30\,\upmu\mathrm{l}$ of nuclease-free water to the cDNA.   
6. Prepare the bottle of AMPure XP beads by vortexing until the beads are fully resuspended.   
7. Add $50\,\upmu\mathrm{l}$ of AMPure XP of Agencourt beads to the cDNA. Mix well by setting a pipettor to $50\,\upmu\mathrm{l}$ and pipetting up and down.   
8. Incubate the cDNA and beads at room temperature for 5 min.   
9. Place the tube on a magnetic separation rack. Incubate at room temperature for $2\;\mathrm{min}$ . All of the beads should move to one side and the supernatant should be clear.   
10. Pipette off the supernatant and discard.   
11. While the tubes are still on the magnetic rack, wash the beads by adding 180 $\upmu\mathrm{l}$ of $80\%$ EtOH to the side of the tube opposite of the beads. Incubate at room temperature for $30\;\mathrm{s}$ .  
12. Pipette off the $80\%$ EtOH and discard.   
13. While the tube is still on the rack with the lids open, pipette off the ethanol that has been collected at the bottom of the tube and let the remaining ethanol evaporate. The bead pellet should appear to be dry (not shiny) and more matte in appearance.   
14. Resuspend the beads by removing the tube from the magnet and adding $20\,\upmu\mathrm{l}$ of nuclease-free water to the pellet. Pipette up and down several times so that the beads are fully resuspended. Incubate at room temperature for $2\;\mathrm{min}$ .  
15. Place the tube back on the magnet and wait for the supernatant to become clear (approximately 30 s to $1\ \mathrm{min}$ ).   
16. Remove the supernatant which now contains the cDNA to a new $1.5{\-}\mathrm{ml}$ tube.   
17. Either save the cDNA at $-20~^{\circ}\mathrm{C}$ or continue with Subheading 3.9.  

# 3.9  Sequencing Library Preparation  

1. Prepare a $10\,\upmu\mathrm{M}\,\mathrm{F}/\mathrm{R}$ primer mix by adding $10\,\upmu\mathrm{l}$ of $100\,\upmu\mathrm{M}$ forward N primer mix and $10\,\upmu\mathrm{l}$ of $100\,\upmu\mathrm{M}$ reverse $_\mathrm{N}$ primer mix ( $s e e$ Subheading 2), to $80\,\upmu\mathrm{l}$ of nuclease-free water. The primer mix can be stored at $-20~^{\circ}\mathrm{C}$ .  

![](images/01c9d12d3386ffb67f946fd13a639198f15b1e76da55331266df2f7aeeaca09c.jpg)  

3. Put the reaction in a thermocycler and run the program:  

a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .b. 6 cycles of $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ ,$55\ ^{\circ}\mathrm{C}$ for $30\;\mathrm{s}$ , and $72\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  
4. Prepare the bottle of AMPure XP beads by vortexing until the beads are fully resuspended.   
5. Add $50\,\upmu\mathrm{l}$ of AMPure XP of Agencourt beads to the amplification reaction. Mix well by setting a pipettor to $50\,\upmu\mathrm{l}$ and pipetting up and down.   
6. Incubate amplification reaction with the beads at room temperature for $5\;\mathrm{min}$ .  
7. Place the tube on a magnetic separation rack. Incubate at room temperature for $2\;\mathrm{min}$ . All of the beads should move to one side, and the supernatant should be clear.   
8. Pipette off the supernatant and discard.   
9. While the tubes are still on the magnetic rack, wash the beads by adding 180 $\upmu\mathrm{l}$ of $80\%$ EtOH to the side of the tube opposite of the beads. Incubate at room temperature for $30\;\mathrm{s}$ .  
10. Pipette off the $80\%$ EtOH and discard.   
11. While the tube is still on the rack with the lids open, pipette off the ethanol that has been collected at the bottom of the tube and let the remaining ethanol evaporate. The bead pellet should appear to be dry (not shiny) and more matte in appearance.   
12. Resuspend the beads by removing the tube from the magnet and adding $50\,\upmu\mathrm{l}$ of nuclease-free water to the pellet. Pipette up and down several times so that the beads are fully resuspended. Incubate at room temperature for $2\;\mathrm{min}$ .  
13. Place the tube back on the magnet and wait for the supernatant to become clear (approximately 30 s to 1 min).   
14. Remove the supernatant which now contains the library to a new $1.5{\-}{\mathrm{ml}}$ tube.   
15. Either save the library at $-20~^{\circ}\mathrm{C}$ or continue with the subsequent steps.   
16. Dilute the NEBNext  Multiplex Oligos for Illumina to $2\,\upmu\mathrm{M}$ by adding $5\;\upmu\mathrm{l}$ of undiluted primer to $20\,\upmu\mathrm{l}$ of nuclease-free water. This can be done in $0.2–\mathrm{ml}$ PCR  

strip tubes or $1.5{\-}\mathrm{ml}$ tubes. Take care to record which well in the plate in the undiluted primers originated from.  

17. In a $0.2–\mathrm{ml}$ tube, set up the reaction on ice as follows. Be sure to add a different primer pair to each sample, and record which sample received which primer pair.  

![](images/17da35d0ddac97603a3af1feb3c8582e94ab78113ddcae24f9093509d84c1f23.jpg)  

18. Put the reaction in a thermocycler and run the program:  

a. $98\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  

b. 15 cycles of $98\ ^{\circ}\mathbf{C}$ for $10\;\mathrm{s}.$ ,$64\,^{\circ}\!C$ for $10\;\mathrm{s}.$ , and $72\ ^{\circ}\mathbf{C}$ for $30\;\mathrm{s}$ .  

19. Check for amplification and library quality using a DNA ScreenTape (D1000 or similar). Load $1\;\upmu\mathrm{l}$ of the PCR product onto the TapeStation (Fig. 4). If the PCR products are the correct size and the no RT control is negative for a band, proceed to with the next steps.  

20. Add $25\,\upmu\mathrm{l}$ of AMPure XP of Agencourt beads to the amplification reaction. Mix well by setting a pipettor to $50\,\upmu\mathrm{l}$ and pipetting up and down.  

21. Continue with the bead cleanup procedure (steps 6–11 above).  

22. Resuspend the beads by removing the tube from the magnet and adding $15\,\upmu\mathrm{l}$ of nuclease-free water to the pellet. Pipette up and down several times so that the beads are fully resuspended. Incubate at room temperature for $2\;\mathrm{min}$ .  

23. Place the tube back on the magnet and wait for the supernatant to become clear (approximately 30 s to $1\ \mathrm{min}$ ).  

24. Remove the supernatant which now contains the library to a new $1.5{\-}{\mathrm{ml}}$ tube.  

25. Either save the library at $-20~^{\circ}\mathrm{C}$ or continue with the subsequent steps.  

26. Quantify the concentration using the Qubit  dsDNA HS Assay Kit according to the manufacturer’s instructions. Typically, $1{-}2\,\upmu\mathrm{l}$ of library is sufficient for accurate quantification. At this point, the libraries can be pooled and run on an Illumina sequencer.  

# 4 Notes  

1. We use the gradient $:10{-}50\%$ ) program as described in Aboulhouda et al. [17] (described below). Depending on the gradient platform being used, other preset programs for $10{-}50\%$ sucrose gradients can be used.  

a. 05/85/35.   
b. 01/77/0.   
c. 04/86/35.   
d. 03/86.5/35. e. 20/81/14.   
f. 07/86/20.  

Sequence $=$ abcbdbabcbdbef.  

2. The protocol for fractionation may be different depending on the fractionator being used. The steps in this procedure are for the Gilson fraction collection system. Refer to the user’s manual for the Gilson fraction collection system for more detailed instructions or to your particular fraction collector for alternative instructions.  

# References  

1. Ryoo HD, Vasudevan D (2017) Two distinct nodes of translational inhibition in the integrated stress response. BMB Rep 50:539–545 [PubMed: 28803610]   
2. Wek RC (2018) Role of eIF2α kinases in translational control and adaptation to cellular stress. Cold Spring Harb Perspect Biol 10:1–16   
3. Pakos-Zebrucka K, Koryga I, Mnich K et al. (2016) The integrated stress response. EMBO Rep 17:1374–1395 [PubMed: 27629041]   
4. Vattem KM, Wek RC (2004) Reinitiation involving upstream ORFs regulates ATF4 mRNA translation in mammalian cells. Proc Natl Acad Sci U S A 101:11269–11274 [PubMed: 15277680]   
5. Zhang H, Wang Y, Lu J (2019) Function and evolution of upstream ORFs in eukaryotes. Trends Biochem Sci 44:782–794 [PubMed: 31003826]   
6. McGillivray P, Ault R, Pawashe M et al. (2018) A comprehensive catalog of predicted functional upstream open reading frames in humans. Nucleic Acids Res 46:3326–3338 [PubMed: 29562350]   
7. Melnikov A, Murugan A, Zhang X et al. (2012) Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay. Nat Biotechnol 30:271–277 [PubMed: 22371084]   
8. Sharon E, Kalma Y, Sharp A et al. (2012) Inferring gene regulatory logic from high-throughput measurements of thousands of systematically designed promoters. Nat Biotechnol 30:521–530 [PubMed: 22609971]   
9. Soemedi R, Cygan KJ, Rhine CL et al. (2017) Pathogenic variants that alter protein code often disrupt splicing. Nat Genet 49:848–855 [PubMed: 28416821]   
10. Adamson SI, Zhan L, Graveley BR (2018) Vex-seq: high-throughput identification of the impact of genetic variation on pre-mRNA splicing efficiency. Genome Biol 19:1–12 [PubMed: 29301551]   
11. Sample PJ, Wang B, Reid DW et al. (2019) Human $5^{\prime}$ UTR design and variant effect prediction from a massively parallel translation assay. Nat Biotechnol 37:803–809 [PubMed: 31267113]   
12. Dvir S, Velten L, Sharon E et al. (2013) Deciphering the rules by which $5^{\prime}$ -UTR sequences affect protein expression in yeast. Proc Natl Acad Sci U S A 110:E2792–E2801 [PubMed: 23832786]   
13. Lin Y, May GE, Kready H et al. (2019) Impacts of uORF codon identity and position on translation regulation. Nucleic Acids Res 2:1–10   
14. Jia L, Mao Y, Ji Q et al. (2020) Decoding mRNA translatability and stability from the $5^{\prime}$ UTR. Nat Struct Mol Biol 27:814–821 [PubMed: 32719458]   
15. Noderer WL, Flockhart RJ, Bhaduri A et al. (2014) Quantitative analysis of mammalian translation initiation sites by FACS-seq. Mol Syst Biol 10:748 [PubMed: 25170020]   
16. May GE, McManus CJ (2022) High-throughput quantitation of yeast uORF regulatory impacts using FACS-uORF. Methods Mol Biol 2404:331–351. 10.1007/978-1-0716-1851-6_18. [PubMed: 34694618]   
17. Aboulhouda S, Di Santo R, Therizols G, Weinber D (2017) Accurate, Streamlined Analysis of mRNA Translation by Sucrose Gradient Fractionation. Bio Protoc 7(19):e2573. 10.21769/ BioProtoc.2573.  

![](images/ef36fcf08f6ccbff4eabbd1404ead43e70db821387ad670bd7232bc5a9d978c8.jpg)  

# Fig. 1.  

Human ATF4 transcript. The start (green boxes) and stop codons (red boxes) are indicated for the three ATF4 uORFs (transparent gray boxes) and the ATF4 CDS (yellow). The arrows indicate the location and frame of translation for the uORFs, and for ATF4, which is overlapped by the translation of uORF 2  

![](images/27dbf991f796608ab27c73b5769f81098d7a63ba89c79f63efbd12a596576541.jpg)  
F1g.2 The effect on translation of the $5^{\prime}$ leader sequence of ATF4 during stress. HEK 293T cells were simultaneously transfected with wild-type (WT) and start codon mutants of the second (M2) and third (M3) uORFs (red Xs, uORFs 1 and 2) upstream of MGFP (green box). The cells were either subjected to sodium arsenite stress (orange $40\,\upmu\mathrm{M}$ ;$30\;\mathrm{min}$ ) or left untreated (blue). The number of reads for each variant were counted in each fraction and normalized by the number of total reads in each fraction. Data shown are the averages of two biological replicates for three transcript variants assayed. As expected, the wild-type ATF4 transcript leader shifts into the polysome in response to stress (top, compare blue to gold). This translational induction does not occur in transcript leaders mutated for uORF1 (middle). Mutation of the inhibitory uORF2 (bottom) increases translation under both unstressed and stressed conditions (compare to WT, top)  

# 6.Polysome lysate preparation  

![](images/2a05dee5e5cfafed6fa4175a3fd38eb0c7c0fa935c2a8d9dcad13c3cf7f9ce3b.jpg)  
2. In vitro transcription  

![](images/852260582944adc33074619db07ca80d9b9e9bdb0e192f99b2a66071e6fffa26.jpg)  
7. Polysome fractionation  

![](images/e94dd982f2b353c9c0f59e6499e95f415165b7d2d4d5efdc6ae399314fb845c4.jpg)  
3. 5' end capping  

![](images/582fea7c26992b7bb27e9626c86dd323a38e437ff3fc91bce879df0aee7264b9.jpg)  
8. RNA extraction and library preparation  

![](images/915bc0886101b5b85ed8890fa38d1cf85a7b7e7ba38f211595de7ed4341ea187.jpg)  
4.HEK293Tcelltransfection  

![](images/e58a490c41e0f9d3acbfb6280bb3daf0df9dc0b4bd1b277162a30ea93576d26c.jpg)  

# 5. Stress treatment (sodium arsenite)  

![](images/a80b91b4ff9a2e5a7d83274c4a2ff6fa37c65c0016a3701f5ea3efdc2896ea9d.jpg)  

![](images/d15ae714c41313bc59291c83ba43cee80e9891c4afded047b634cb53bb979ff6.jpg)  

# Fig. 3.  

Polib-seq protocol overview. (Step 1) The cloned library is amplified to generate a template for in vitro transcription. A T7 polymerase site and a poly A sequence are incorporated during this step. (Step 2) Uncapped in vitro transcribed RNA is produced using the PCR product as the template. (Step 3) A $5^{\prime}$ cap is enzymatically added to the RNA. (Step 4) HEK 293T cells are transfected for $^{5\,\mathrm{h}}$ with the RNA. (Step 5) The cells are subjected to sodium arsenite for $30\;\mathrm{min}$ to induce the integrated stress response. (Step 6) Cell polysome lysates are prepared and fractionated over a sucrose gradient into 10 fractions (step 7). (Step 8) RNA is extracted from the fractions and sequencing libraries are prepared by targeted RT-PCR  

![](images/10c135504bd1baeb3655c774f68250f9e01345f3b7680df7bd554c3cdc5cbec9.jpg)  
Fig. 4. TapeStation gel image of sequencing library final PCR products (Lanes A1, B1, C1). A standard DNA ladder is shown in lane A0, and molecular size standards are at 1500 bp (purple) and $25\;\mathrm{bp}$ (green). The libraries range from 355 to 368 bp due to the incorporation of up to 13 random bases during the first PCR step. The PCR product will sometimes contain a small amount of non-specific product $({\sim}700\,\mathrm{bp})$ ) which does not affect the quality of the sequencing library. Non-specific products and excess primers less than 100 bp are removed during the subsequent bead clean up step  

![](images/728442063f8074f65032169d56bda7d7777e1691cb44ba6e8f921ea358d7228d.jpg)  

![](images/72a150173cf7dfb7c8c05a47bc7c8a27b4a6dae2f69946d5f7306111a64bd4dd.jpg)  

![](images/b92ef9fb77d9934a89060a17bb02d9c9680a869b2f8b8eeb1cd36bce0a96f1e8.jpg)  

![](images/32370508b7c0ae1f206035327680ce768896703a78038a8398319d56faa74b3d.jpg)  

![](images/1aed7a5501460fd3c9ea7311c47f33c5f54819cdd147d95b153e16a1e3f3c5e5.jpg)  

# Table 1  

Custom primers   

![](images/b1223cb6d29a592f76ca398eee1dbd53e4eb2dca6f791349d50056feb0f66b65.jpg)  