# What, where, and how: Regulation of translation and the translational landscape in plants  

Hsin-Yen Larry Wu $\oplus$ , Joey Jen    and Polly Yingshan Hsu 1, iD  

1 Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI 48824, USA  

\*Author for correspondence: pollyhsu@msu.edu The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (https://academic.oup.com/plcell/pages/General-Instructions) is: Polly Yingshan Hsu (pollyhsu@msu.edu).  

# Abstract  

Translation is a crucial step in gene expression and plays a vital role in regulating various aspects of plant development and environmental responses. It is a dynamic and complex program that involves interactions between mRNAs, transfer RNAs, and the ribosome machinery through both cis- and trans-regulation while integrating internal and external signals. Translational control can act in a global (transcriptome-wide) or mRNA-specific manner. Recent advances in genome-wide techniques, particularly ribosome profiling and proteomics, have led to numerous exciting discoveries in both global and mRNA-specific translation. In this review, we aim to provide a “primer” that introduces readers to this fascinating yet complex cellular process and provide a big picture of how essential components connect within the network. We begin with an overview of mRNA translation, followed by a discussion of the experimental approaches and recent findings in the field, focusing on unannotated translation events and translational control through cis-regulatory elements on mRNAs and trans-acting factors, as well as signaling networks through 3 conserved translational regulators TOR, SnRK1, and GCN2. Finally, we briefly touch on the spatial regulation of mRNAs in translational control. Here, we focus on cytosolic mRNAs; translation in organelles and viruses is not covered in this review.  

# Introduction  

# Basics of mRNA translation  

Eukaryotic gene expression begins with the transcription of genes in the nucleus. The resulting transcripts are conceptu­ ally classified into protein-coding RNAs (or mRNAs) and noncoding RNAs, although translation from some noncoding RNAs has been reported (see section on ‘Unannotated trans­ lation events’). Following transcription, the primary tran­ scripts or pre-mRNAs undergo processing, including the addition of a $5^{\prime}$ cap and a $3^{\prime}$ poly-A tail and splicing. Once the mature mRNAs are transported out of the nucleus, they are generally ready for translation. However, mRNAs may be degraded or stored in some circumstances, prevent­ ing them from being translated.  

The major players in translation include mRNAs, ribo­ somes,  transfer  RNAs  (tRNAs),  and  translation  factors involved in different phases of translation. Below we briefly describe the main steps of translation and core translation factors. For detailed composition of the translation machin­ ery in plants, including ribosomal proteins and translation factors,  we  refer  readers  to  this  comprehensive  review (Browning and Bailey-Serres 2015).  

Eukaryotic mRNAs  typically  have  a $5^{\prime}$ cap (7-methylguanosine) and a $3^{\prime}$ poly-A tail (Fig. 1A: an overview of translation initiation; Fig. 1B: a summary of core translation factors). The cap serves as an anchor for translational initiation. Cap-dependent translation initiation is governed by 2 large protein complexes: the eIF4F complex (consisting of eukaryotic initiation factor 4E (eIF4E) and eIF4G) and the 43S preinitiation complex (PIC) (consisting of the 40S ribosomal small subunit, ternary complex: eIF2/GTP/Met-tRNAi, eIF1, eIF1A, eIF3, and eIF5). First, the $5^{\prime}$ cap is recognized by eIF4E, which  

![](images/d6d12a29ee47c447d87f6d237a66298e68958e8921ec7360938bbe65b1a58dfd.jpg)  

Figure 1. Overview of eukaryotic translation initiation complexes and functions of core translation factors. A) Eukaryotic mRNAs typically have a $5^{\prime}$ m7G cap and a $3^{\prime}$ poly-A tail. Translation initiation is controlled by 2 large protein complexes, including the eIF4F complex and the 43S PIC. eIF4F consists of eIF4E (cap binding) and eIF4G (a scaffold protein). The 43S PIC consists of the 40S ribosomal subunit, the ternary complex (eIF2, GTP, Met-tRNAi), eIF1, eIF1A, eIF3, and eIF5. The eIF4F complex binds to the cap and recruits the 43S PIC, which then scans the $5^{\prime}$ UTR until a start codon is identified. Subsequently, most initiation factors dissociate, and the 60S ribosomal subunit joins. GTP is shown in dark red circles. The figure was adapted from (Castellano and Merchante 2021; Passmore and Coller 2022). B) Summary of core translation factors involved in canonical initiation, elongation, and termination in Arabidopsis. The information was simplified and adapted from Browning and Bailey-Serres (2015) and Raabe et al. (2019).  

forms a complex with eIF4G, a large scaffold protein that inter­ acts with the RNA helicase eIF4A. The 43S PIC is then recruited to the cap through the interaction between multimeric eIF3 and eIF4G. eIF4G also interacts with poly-A binding protein (PABP), which binds to the poly-A tail at the $3^{\prime}$ end of the mRNA. Thus, the $5^{\prime}$ and $3^{\prime}$ ends of the mRNAs are connected, forming a circular structure. The circularization of mRNAs en­ hances their efficient translation, facilitates ribosome recycling, and increases mRNA stability by protecting mRNAs from deg­ radation  via  exonucleases (reviewed  in Browning  and Bailey-Serres 2015; Passmore and Coller 2022).  

Translation can be divided into 3 phases: initiation, elong­ ation, and termination. Translation initiation is considered the key point of regulation and is the most well studied of these 3 phases. Plants, like other eukaryotes, use the scanning model for translation initiation (Fig. 1A). After being re­ cruited to the cap, the 43S PIC scans along the $5^{\prime}$ leader se­ quence (i.e. the region upstream of the main ORF, also commonly called $5^{\prime}$ UTR, although extensive translation may occur in this region; see section on ‘Unannotated trans­ lation events’) until an optimal start codon is identified. Then, the 60S ribosomal large subunit is recruited, with the Met-tRNAi positioned at the peptidyl site (P-site) of the ribo­ some, and the codon on the mRNA and anti-codon on the tRNA base pair. New charged tRNAs corresponding to the next codons then join at the aminoacyl site (A-site). The ami­ no acids at the P-site and A-site form a peptide bond. Subsequently, the amino acid (or the growing peptide) at the P-site is transferred to the amino acid at the A-site. Next, the ribosome translocates by shifting 3 nucleotides (3 nts; 1 codon). As a result, the uncharged tRNA is now located at the exit site, and the tRNA carrying the peptide is at the P-site, which makes the A-site available for the incoming charged tRNAs. This cycle repeats until a stop codon is en­ countered at the A-site, where release factors, rather than a charged tRNA, join the A-site and trigger translation termination. Subsequently, the ribosome subunits dissociate from the mRNA and are recycled for additional rounds of translation (reviewed in Schuller and Green 2018).  

![](images/49fbb410e2ad2d5bdb8b4561b1d70a85ec2fd0ead1812cdaefff2497229ad6ac.jpg)  
Figure 2. Common genomic techniques for studying translation. A) Polysome profiling uses a sucrose gradient to separate mRNAs associated with different numbers of ribosomes. The resulting mRNA populations corresponding to various translational activities can be fractionated and subjected to microarray or RNA-seq for identifying and quantifying the mRNAs. B) Ribo-seq uses ribonuclease digestion to obtain ribosome footprints. Sequencing of ribosome footprints reveals the precise position and quantity of ribosomes on mRNAs globally. High-quality datasets display a strong 3-nt periodicity that corresponds to ribosomes translocating by 3 nt per codon within coding regions. C) TRAP uses antibodies to isolate a sub­ population of ribosomes by expressing an epitope-tagged ribosomal large subunit protein driven by either a constitutive or a tissue-specific pro­ moter. The mRNAs associated with the purified ribosomes can be subjected to microarray or RNA-seq for identifying and quantifying the mRNAs.  

# Unique features of plant mRNA translation  

Although the machinery for plant mRNA translation is large­ ly similar to that of other eukaryotes, several unique charac­ teristics have evolved in plants. In addition to the canonical eIF4F complex, consisting of eIF4E and eIF4G, plants possess eIFiso4F complexes consisting of eIFiso4E and eIFiso4G (en­ coded  by  eIFiso4G1  and  eIFiso4G2  in  Arabidopsis [Arabidopsis  thaliana]).  eIFiso4G  is  deeply  conserved throughout plant lineages, whereas eIFiso4E is present only in flowering plants. The eIFiso4F complex is smaller com­ pared with eIF4F complex ${\sim}80\,\mathsf{k D}$ vs $\sim\!180\,\mathsf{k D}_{\perp}^{\prime}$ ) but more abundant (Patrick and Browning 2012). eIF4G and eIFiso4G have different mRNA sequence preferences and interact dif­ ferently with various PABP isoforms (Gallie and Browning 2001; Gallie 2018); thus, these complexes may favor specific sets of mRNAs and control distinct physiological responses. For example, eIFiso4G has been shown to prefer mRNAs with unstructured sequences, which are characteristics of mRNAs that are preferentially translated under hypoxia (Gallie and Browning 2001; Branco-Price et al. 2005). More recently, eIFiso4G has been reported to control specific mRNAs involved in chloroplast functions, hypoxia, and absci­ sic acid (ABA) signaling (Bi et al. 2019; Cho et al. 2019; Lellis et al. 2019).  

Plant ribosomal proteins have also expanded due to mul­ tiple rounds of genome duplication. In Arabidopsis, there are 80 ribosomal protein (RP) families, and each RP is en­ coded by 2 to 6 genes (Barakat et al. 2001; Browning and Bailey-Serres 2015; Scarpin et al. 2023). The large number of potential combinations of these RPs in ribosomes is pro­ posed to provide heterogeneity to ribosome properties and enable specific mRNA translation in response to various in­ ternal and external cues (Browning and Bailey-Serres 2015).  

# Genome-wide techniques to study translation  

As translation involves mRNAs, which can be amplified and sequenced, much of our knowledge of translational regula­ tion has expanded thanks to the development of nextgeneration  sequencing.  Below  we  discuss  3  common genome-wide techniques for studying translation. For other techniques, we refer readers to a comprehensive review (Mazzoni-Putman and Stepanova 2018).  

The first genomic technique used to study translation is polysome  profiling  (Fig.  2A),  which  involves  separating mRNAs bound by various numbers of ribosomes through a sucrose gradient by ultracentrifugation. The distribution of specific mRNAs in different fractions can be quantified using reverse-transcription PCR, microarrays, or RNA-sequencing (RNA-seq) to estimate their relative translational activity. Although this technique is powerful and has allowed count­ less discoveries (e.g. Coate et al. 2014; Missra et al. 2015; Bai et al. $2017a.$ ; Zhang et al. 2017), polysome profiling cannot re­ veal the exact ribosome occupancy on mRNAs, and the reso­ lution of polysome separation typically decreases as the number of bound ribosomes increases (Fig. 2A).  

The second technique, termed ribosome profiling (a.k.a. Ribo-seq; Fig. 2B), solves the limitation of polysome profiling. Ribo-seq involves ribonuclease digestion of ribosome-bound mRNAs and sequencing of the resulting ribosome-protected mRNA fragments (i.e. ribosome footprints), typically 28 to 30 nts in length (Ingolia et al. 2009; Brar and Weissman 2015). The sequencing of these ribosome footprints reveals the pre­ cise quantity and position of ribosome occupancy on mRNAs throughout the transcriptome. Translation efficiency can be inferred by normalizing the Ribo-seq to RNA-seq read dens­ ity. Moreover, in high-quality Ribo-seq data, ribosome foot­ prints  display  strong  3-nt  periodicity,  consistent  with translating ribosomes moving every 3 nts along coding re­ gions. This periodic property can be used to identify actively translated regions (Ingolia et al. 2009). The first ribosome profiling studies in Arabidopsis were reported a decade ago; one was applied to study translational control of light re­ sponses (Liu et al. 2013), and a variation of the technique was applied to study chloroplast translation (Zoschke et al. 2013). Optimizing the resolution of Ribo-seq led to the discovery of previously unannotated translated ORFs, which dramatically ex­ panded the landscape of the Arabidopsis translatome and im­ proved genome annotation (Hsu et al. 2016). Ribo-seq has been used to study translational responses to many environ­ mental stimuli, such as hypoxia, hormonal responses, pathogen responses, and phosphate deficiency in Arabidopsis (Juntawong et al. 2014; Merchante et al. 2015; Bazin et al. 2017; Xu et al. 2017a) and to characterize the translatomes in crops such as maize (Zea mays), soybean (Glycine max), tomato (Solanum ly­ copersicum), rice (Oryza sativa), sorghum (Sorghum bicolor), and wheat (Triticum aestivum) (Lei et al. 2015; Chotewutmontri and Barkan 2016; Shamimuzzaman and Vodkin 2018; Wu et al. 2019; Li and Liu 2020; Zhu et al. 2021; Sotta et al. 2022; Guo et al. 2023b). However, the short length of ribosome footprints limits their mapping capability. Therefore, it is challenging to imple­ ment Ribo-seq for polyploid genomes where short reads are less  likely  to  be  uniquely  mapped  (Jiang  et  al.  2022). Nevertheless, Ribo-seq has been recently carried out in hexa­ ploid bread wheat (Guo et al. 2023b).  

The third technique, translating ribosome affinity purifica­ tion (TRAP; Fig. 2C), involves the expression of an epitopetagged ribosomal protein (usually a large ribosomal subunit protein to ensure the capture of fully assembled ribosomes), followed by affinity purification of the tagged ribosomes and associated mRNAs using antibodies. The mRNAs associated with translating ribosomes can then be identified and quan­ tified by microarrays or RNA-seq (Inada et al. 2002; Mustroph et al. 2009; Juntawong et al. 2014). Using a constitutive pro­ moter to express the tagged ribosomal protein, TRAP-seq has been applied to study translational responses to different treatments in several species (Juntawong et al. 2014; Bazin et al. 2017; Meteignier et al. 2017; Reynoso et al. 2019; Bonnot and Nagel 2021). When combined with tissuespecific promoters, TRAP enables the study of tissue-specific translatomes (Mustroph et al. 2009; Traubenik et al. 2020; Kajala et al. 2021; Reynoso et al. 2022). It should be noted that TRAP requires generating stable transgenic plants, and the accumulation of tagged ribosomal proteins may alter ribosome properties and physiology (Zhao et al. 2019a). Although TRAP-seq can reveal mRNAs engaged in transla­ tion within specific tissue types, it is difficult to estimate translation  efficiency (average  ribosome  number  per mRNA) because it is technically challenging to quantify mRNA levels from the exact same tissue used to produce the TRAP-seq data. Although, to our knowledge, there are no reports in plants yet, it is possible to combine tissuespecific TRAP with ribosome footprinting (Gonzalez et al. 2014; Chen and Dickman 2017; Doroudgar et al. 2019) to quantify ribosome density and study actively translated ORFs on tissue-specific mRNAs.  

# Unannotated translation events  

The above sequencing techniques, especially Ribo-seq, have transformed our view of translational landscapes. In addition to refining previously annotated ORF models, Ribo-seq has led to 3 major discoveries in the translatomes: (1) the translation of small ORFs (sORFs) in RNAs presumed to be noncoding, (2) the widespread presence of upstream ORFs (uORFs) on proteincoding mRNAs, and (3) the usage of alternative or non-AUG start sites. Unlike canonical protein-coding genes, the act of translating these new ORFs, specifically sORFs and uORFs, may serve a regulatory function, even if the resulting protein products do not play a direct role in regulation (see below).  

# Newly discovered translated sORFs among presumed noncoding RNAs  

To minimize false discoveries, computational genome anno­ tations typically assume that coding sequences are longer than 100 codons. This assumption means that transcripts en­ coding small proteins can be overlooked and annotated as noncoding RNAs (Basrai et al. 1997). Based on the 3-nt peri­ odicity corresponding to ribosomes translocating 3 nt per codon, dozens or hundreds of translated sORFs within anno­ tated noncoding RNAs or previously unannotated tran­ scripts have been identified in Arabidopsis, tomato, maize, and wheat (Hsu et al. 2016; Wu et al. 2019; Kurihara et al. 2020; Liang et al. 2021; Palos et al. 2022; Guo et al. 2023b). Similarly, an alternative approach based on ribosome dissoci­ ation from mRNAs upon translation termination identified over  200  translated  sORFs  in  Arabidopsis  (Bazin  et  al.  

2017). The translation efficiency of sORF mRNAs in tomato is indistinguishable  from  that  of  canonical  protein-coding genes, suggesting that most of these sORF transcripts are likely to be true protein-coding RNAs (Wu et al. 2019). Epitope-tagging followed by western blots and mass spec­ trometry–based proteomic analysis confirmed that at least some of these sORFs produce stable proteins in planta (Hsu et al. 2016; Bazin et al. 2017; Wu et al. 2019).  

Although most of these sORF mRNAs were likely misannotated as noncoding RNAs, surprisingly, some wellcharacterized  noncoding  RNAs  also  contain  translated sORFs. The best documented examples are the primary tran­ scripts of trans-acting small interfering RNAs (tasiRNAs). The biogenesis of tasiRNAs involves microRNA (miRNA) cleavage of primary TAS transcripts, which are then converted to double-stranded RNAs (dsRNAs) and processed into 21- or 22-nt tasiRNAs. Arabidopsis has 4 TAS gene families (TAS1 to TAS4). The primary TAS3 transcript contains a 50-amino acid (aa) sORF upstream of the first miRNA390 binding site; strong 3-nt periodicity of Ribo-seq reads was unambigu­ ously detected in this sORF, supporting that it is actively translated (Hsu et al. 2016; Hsu and Benfey 2018). The ribo­ some occupancy on the TAS3 sORF is also supported by a genome-wide study of $5^{\prime}$ truncated mRNA ends (Hou et al. 2016). By contrast, the primary TAS2 transcript has a 57-aa sORF that contains a miRNA173 binding site. This sORF was shown to be translated in vitro, and introducing a pre­ mature stop codon to this sORF reduces tasiRNA levels. These results suggest that the translation and position of the sORF relative to the miRNA binding site are critical for tasiRNA biogenesis (Yoshikawa et al. 2016). In addition, pri­ mary TAS1A/B/C, TAS2, and TAS3 transcripts are associated with membrane-bound polysomes, suggesting that they are translated by endoplasmic reticulum (ER)-localized ribo­ somes (Li et al. 2016). A recent study further demonstrated that the dsRNA-binding protein SUPPRESSOR OF GENE SILENCING 3 (SGS3) regulates ribosome stalling at the sORFs upstream of the miRNA binding sites on TAS3 and TAS1C by forming a complex with ARGONAUTE (AGO) and the relevant miRNA (Iwakawa et al. 2021).  

Another surprising example of noncoding RNA translation was found in the primary transcripts of miRNAs. The first study reported that pri-miR165a encodes an 18-aa sORF in Arabidopsis  and  pri-miR171b  encodes  a  9-aa  sORF  in Medicago truncatula (Lauressergues et al. 2015). Application of synthetic peptides corresponding to these sORFs led to an increase in pri-miRNA levels (Lauressergues et al. 2015; Sharma et al. 2020). However, the translation levels of these sORFs ap­ pear to be low or condition dependent, as no significant ribo­ some footprints in these sORF regions have been detected in available Ribo-seq datasets (Hsu and Benfey 2018).  

# Prevalent uORFs on mRNAs  

Conventionally, it was assumed that 1 mRNA produces 1 protein from the longest ORF. However, when the 43S PIC scans along the $5^{\prime}$ UTR searching for the optimal start site (usually AUG), it may encounter 1 or multiple smaller ORFs upstream of the annotated ORFs (thus the name up­ stream ORFs). As a result, the ribosome may translate the uORF before reaching the longest/annotated ORF (main ORF; mORF) (Fig. 3). Ribo-seq analysis has revealed numer­ ous  translated  uORFs  in  plants,  for  example,  2,093  in Arabidopsis, 1,329 in tomato, 2,558 in maize, and 1,254 in wheat (Lei et al. 2015; Wu et al. 2019; Wu and Hsu 2022; Guo et al. 2023b). The identification of these uORFs suggests that the translation of their downstream mORFs is likely re­ pressed by the uORFs (see section ‘Translational regulation through mRNA features – uORFs’ for details).  

# Alternative or non-AUG translation start sites  

Like uORFs, the 43S PIC may consider codons different from the annotated start site to initiate translation, which can lead to the production of proteins different from the annotated ORFs through the use of an alternative AUG or a nearcognate start site that differs from AUG by 1 nucleotide. Recently, a specialized Ribo-seq termed translation initiation site sequencing (TI-seq) has been applied in plants to study translation initiation and amino-terminal (N-terminal) pro­ teomics (Willems et al. 2017; Li and Liu 2020). TI-seq uses the translation inhibitor lactimidomycin to capture initiating ribosomes, in contrast to regular Ribo-seq that uses cyclohex­ imide, which captures both initiating and elongating ribo­ somes. Note that even snap freezing in liquid nitrogen during sample harvest can capture many elongating ribosomes (Douka et al. 2022). Thus, a technical challenge with TI-seq is to ensure efficient runoff of elongating ribosomes to avoid false po­ sitives. Therefore, stringent criteria and careful follow-up valid­ ation are necessary (Gao et al. 2015; Li and Liu 2020). Studies in Arabidopsis cultured cells and tomato leaves have revealed that alternative or near-cognate translation start sites are preva­ lent. The most common near-cognate codon is CUG, whereas AAG and AGG are the least used (Willems et al. 2017; Li and Liu 2020). Many alternative translation start sites identified in plants to date are in frame with the corresponding annotated ORF, resulting in changes in the protein N-terminal sequences (Willems  et  al.  2017;  Wu  et  al.  2019;  Li  and  Liu  2020). Follow-up experiments have demonstrated that several alterna­ tive or non-AUG translation start sites lead to different protein subcellular localizations (Li and Liu 2020).  

![](images/df1c989b0cd8cd2602abf6e8c0c044aaa6e941f9cf300f56d6285cdd8c800328.jpg)  
Figure 3. Common cis-regulatory elements or mechanisms contributing to translational control in mRNAs. Cis-regulatory elements or associated mechanisms commonly present in the $5^{\prime}$ UTR, the $\mathsf{m O R F}$ , or $3^{\prime}$ UTR are presented. The figure was adapted from Liu and Qian (2014).  

# Translational regulation through mRNA features  

Translational control can be classified conceptually into cis- and trans-regulation. An mRNA molecule may contain specific cisregulatory elements that contribute to translational control. Many cis-regulatory elements have been found within the $5^{\prime}$ UTR, $3^{\prime}$ UTR, or surrounding the start or stop codons of mRNAs (Fig. 3). Some of these cis-regulatory elements are known to be bound or regulated by specific RNA-binding pro­ teins (i.e. trans-acting factors). In this section, we will discuss gene-specific cis-regulatory elements involved in translational control  and  their  corresponding  RNA-binding  proteins,  if known. In the next section ‘Translational regulation through modulation of translation factors and master translational reg­ ulators’, we will focus on translation factors, mechanisms, or sig­ naling that can affect global translation.  

# Kozak  

During its search for the start codon, the 43S PIC may en­ counter multiple AUG and near-cognate codons present in the $5^{\prime}$ UTR. How does the ribosome decide where to start? An optimal start codon typically possesses a consensus se­ quence flanking the AUG known as the Kozak consensus se­ quence (Kozak 1981). In plants, as well as in other eukaryotes, the Kozak consensus sequence is $5^{\prime}$ -A/GNNAUGG- $3^{\prime}$ , mean­ ing that the upstream third nt $(-3)$ of the AUG is enriched for A or G, and the downstream fourth nt $(+4)$ is enriched for G (Lütcke et al. 1987; Nakagawa et al. 2008). Analysis of actively translated ORFs from Ribo-seq data in Arabidopsis, tomato, maize, and sorghum has confirmed the conserved sequence bias toward $-3/+4$ positions (Liu et al. 2013; Lei et al. 2015; Wu et al. 2019; Sotta et al. 2022). Thus, mRNA se­ quences containing an optimal Kozak consensus sequence are expected to have better translation efficiency.  

# uORF  

# uORF properties  

As $30\%$ to $70\%$ of plant genes contain potential AUG-initiated uORFs (Zhang et al. 2021), uORFs represent a common mechanism for translational control. If a uORF is translated, it may lead to ribosome stalling or dissociation before the ribo­ some reaches the mORF, resulting in repression of mORF trans­ lation (Fig. 4A). Moreover, when a uORF is translated, the transcript will appear to have a long $3^{\prime}$ UTR, which is expected to activate nonsense-mediated decay (NMD), a conserved mechanism to degrade aberrant mRNAs with a premature ter­ mination codon (Kurosaki et al. 2019). Although evidence in an­ imals supports the idea that uORF translation triggers NMD (Hurt et al. 2013; Chew et al. 2016; Johnstone et al. 2016; Baird et al. 2018), whether uORFs trigger NMD in plants is still under debate, particularly with new evidence in Arabidopsis challenging this idea (Wu and Hsu 2022).  

The mORF downstream of a uORF may still get translated via leaky scanning or re-initiation (Fig. 4A). Leaky scanning occurs when the 43S PIC passes a uORF without translating it, and, thus, not all predicted uORFs are translated. In the case of re-initiation, after the $\mathtt{u O R F}$ is translated, the riboso­ mal small subunit remains attached to the mRNA and con­ tinues  scanning  the $5^{\prime}$ UTR,  ultimately  translating  the mORF. Factors that determine whether a uORF is translated or the efficiency of re-initiation remain areas of great interest.  

uORFs are enriched in the transcripts of genes encoding transcription factors and protein kinases (Kim et al. 2007; Wu and Hsu 2022) and are present in the transcripts of nu­ merous metabolic genes (reviewed in Van Der Horst et al. 2020). uORFs are also associated with allelic variation and agricultural traits (Gage et al. 2022; Guo et al. 2022), high­ lighting the importance of uORFs in plant physiology. In some uORF-containing transcripts, such as those for the transcription  factor  genes  ARABIDOPSIS  HOMEOBOX  1 (AtHB1) and BASIC LEUCINE-ZIPPER 11 (bZIP11), overproduc­ tion of their mORFs leads to deleterious growth phenotypes, suggesting that uORF-mediated suppression plays key roles in regulating plant growth (Ribone et al. 2017). Recently, sev­ eral studies have manipulated uORFs of genes of interest to improve specific traits. For example, manipulating the uORF in the bZIP11 transcript increased sugar content in tomato and strawberry (Fragaria $\times$ ananassa; Sagor et al. 2016; Xing et  al.  2020),  and  manipulating  the  uORF  in  the TL1-BINDING FACTOR (TBF1) transcript enhanced pathogen response in Arabidopsis and rice (Xu et al. 2017b).  

Using Ribo-seq with improved coverage, 2,093 actively translated uORFs were identified in Arabidopsis seedlings based on statistically significant 3-nt periodicity (Wu and Hsu 2022). However, this approach may not efficiently iden­ tify short uORFs or uORFs that partially overlap with their as­ sociated mORFs or with other uORFs. Thus, more uORFs likely remain to be identified. Conversely, the number of po­ tential uORFs may be overestimated due to annotation er­ rors. It has been reported that the current Arabidopsis annotation (Araport11) tends to overestimate the length of $5^{\prime}$ UTRs (Parker et al. 2020). If the $5^{\prime}$ UTR length of a tran­ script is overestimated, a uORF may be predicted from the extended $5^{\prime}$ UTR sequence, which does not exist in nature. A common experiment to validate uORF functions is to compare the $5^{\prime}$ UTRs possessing a wild-type uORF or a mu­ tated $\mathtt{u O R F}$ (such as by mutating its start codon) in a report­ er assay. However, researchers should be cautious, as those false-positive uORFs may still affect translation when artifi­ cially expressing non-existing $5^{\prime}$ UTRs. To avoid this issue, re­ searchers can implement $5^{\prime}$ rapid amplification of cDNA ends experiment or exploit available Cap Analysis of Gene Expression-seq data (Takahashi et al. 2012a) to confirm tran­ scription start sites, thereby increasing the confidence in identifying functional uORFs.  

![](images/0c6e04d29b48a28d22b90820bc9cc5d7ed48454a9dc8a9360de5b92c4ab5880c.jpg)  
Figure 4. Overview of uORF regulation and expected mORF translation levels. A) Effects of uORFs on mORF translation may include leaky scanning, ribosome stalling, ribosome dissociation, and re-initiation. B) Mechanisms that regulate the presence of uORFs in mRNA sequences. uORFs may be omitted through alternative splicing or using an alternative transcription start site of mRNAs. uORFs are shown in yellow boxes; mORFs are shown in grey boxes. C) Examples of uORF regulation under different environmental conditions and expected mORF translation levels. mRNA degradation is shown by scissors. Metabolites are shown by green dots. PTI: pattern-triggered immunity.  

The large number of detected uORFs exhibit diverse prop­ erties, such as variation in their length and extent of se­ quence  conservation.  The  smallest  possible  uORFs  are minimum uORFs (AUG-stop; a start codon immediately fol­ lowed by a stop codon). As translation initiation and termin­ ation are the major rate-limiting steps in translation (Shah et al. 2013), translation of minimum uORFs is likely to cause ribosome stalling. The first minimum uORFs reported were identified  in  Arabidopsis  boric  acid  channel  gene NODULIN26-LIKE  INTRINSIC  PROTEIN5;1  and  in  several boron-responsive mRNAs. In response to high boron levels, these minimum uORFs cause strong translational suppres­ sion through ribosome stalling and mRNA degradation of these boron transporter and responsive mRNAs, reducing overall  boron  uptake  (Tanaka  et  al.  2016).  Subsequent Ribo-seq experiments have identified the translation of 153 minimum uORFs in Arabidopsis adult leaves, and 119 of these minimum uORFs have increased Ribo-seq levels under highboron conditions. Interestingly, high boron also leads to global ribosome stalling at the stop codon of mORFs. Therefore, the increase of ribosome footprints at minimum uORFs under highboron conditions may be partially due to the stalling effects of boron on stop codons (Sotta et al. 2021).  

Most identified uORFs lack sequence conservation at the peptide level across plant species, although the presence of uORFs may be conserved between homologous genes. In Arabidopsis, 120 conserved peptide uORFs (CPuORFs) have been identified using bioinformatics approaches (Hayden and Jorgensen 2007; Jorgensen and Dorantes-Acosta 2012; Takahashi et al. 2012b; Vaughn et al. 2012; Van Der Horst et al. 2019; Takahashi et al. 2020). These CPuORFs were fur­ ther grouped into 81 homology groups based on the peptide sequence similarity of the CPuORFs. This sequence conserva­ tion suggests that the CPuORF peptides play important func­ tions. Seven CPuORFs have been reported to interact with specific  metabolites  (reviewed  in  Van  Der  Horst  et  al. 2020), and 5 CPuORF-encoding peptides have been reported to cause ribosomal arrest, possibly through the interaction between the nascent peptide and the ribosome exit channel (Uchiyama-Kadokura  et  al.  2014;  Hayashi  et  al.  2017; Yamashita et al. 2017). Interestingly, the peptide sequence of the $\mathsf{C P u O R F}$ in homology group 30 is similar to CELL DIVISION CYCLE PROTEIN 26 (CDC26) found in fungi and animal species, which is a component of the anaphase pro­ moting complex involved in cell cycle regulation (Takahashi et al. 2012b; Vaughn et al. 2012). A recent study demonstrated that this CPuORF peptide in Arabidopsis indeed acts like CDC26 as a cell cycle regulator, controlling cell division and embryo development (Lorenzo-Orts et al. 2019).  

# Regulation of uORF presence by alternative transcription start sites or alternative splicing  

The presence or absence of uORFs in mRNA sequences may be regulated at transcriptional and posttranscriptional levels by altering the length and sequence of $5^{\prime}$ UTRs (Fig. 4B) (Von Arnim et al. 2014). At the transcriptional level, different tran­ scription start sites (TSSs) can influence whether uORFs are included in the mRNA sequences. Some genes were shown to use different TSSs in response to various light conditions, leading to the gain or loss of uORFs in the mRNA sequences (Kurihara et al. 2018). At the posttranscriptional level, an alternative spli­ cing event at the $5^{\prime}$ UTR can determine whether a uORF is pre­ sent in HOLOCARBOXYLASE SYNTHASE 1 mRNA, which encodes a protein-biotin ligase that catalyzes the biotinylation of several key metabolic enzymes in Arabidopsis (Puyaubert et al. 2008). Similarly, an intron retention event in the $5^{\prime}$ UTR of PHYTOCHROME-INTERACTING FACTOR3 (PIF3) in response to phytochrome B signaling results in the inclusion of a uORF that represses translation of the PIF3 mRNA (Dong et al. 2020). It remains an open question whether uORF presence and regulation are widely controlled by alternative TSSs and/ or alternative splicing.  

# Regulation of uORF repressiveness  

The repressiveness of uORFs can be modulated by stress, phytohormones, nutrients, or metabolites (Fig. 4C).  

uORF regulation by stress. The translational control of uORFs was first studied in yeast (Saccharomyces cerevisiae) and ani­ mals, where the translational repression mediated by certain uORFs is alleviated (i.e. derepression), under stress conditions through 2 mechanisms. First, the stringency for uORF initi­ ation can be increased to reduce uORF translation. Second, after the first uORF is translated, a phenomenon called “de­ layed reinitiation” allows the 40S ribosome to bypass mul­ tiple  repressive  uORFs  and  reinitiate  translation  at  the mORF (Palam et al. 2011; Young and Wek 2016). Under stress, global translation is typically suppressed to save energy, which is accompanied by selective translation of mRNAs critical for survival (reviewed in Liu and Qian 2014). Therefore, derepres­ sion of uORF-containing transcripts provides a way for a rapid and specific increase of certain proteins from the translation of downstream mORFs.  

As sessile organisms, fast responses through derepression of uORF-containing genes are likely critical for plants. Two examples of this phenomenon in stress response have been reported. The first example is involved in hypoxia stress (Juntawong et al. 2014). Using Ribo-seq, it was shown that a high fraction of genes with increased translation efficiency under hypoxia contain long uORFs. Additionally, the number of Ribo-seq reads associated with these uORFs decreases un­ der hypoxia, whereas those reads associated with their mORFs increase (Fig. 4C). These results suggest that transla­ tion of these long uORFs is reduced, implying that the uORFs are frequently skipped during translation under hypoxia. Alternatively, the reduced translation of uORFs may be due to a global reduction of translation, whereas mORF transla­ tion is increased due to enhanced re-initiation.  

The second example of derepression involves TBF1, which en­ codes a heat-shock factor-like transcription factor required for immune response. The TBF1 transcript contains 2 uORFs (Fig. 4C), and the translation of the TBF1 mORF is de-repressed after pathogen challenge (Pajerowska-Mukhtar et al. 2012). Interestingly, the TBF1 transcript is decapped during the im­ mune response, and 2 R-motifs (guanine/adenine-rich motifs) in the $5^{\prime}$ UTR of TBF1 can serve as Internal Ribosome Entry Sites (IRESs) (see section on ‘IRES’ below) to facilitate the translation of TBF1 mORF. Notably, the 2 R motifs in the TBF1 mRNA overlap with or are downstream of the uORFs. Therefore, the IRES sequences can allow ribosomes to avoid the uORFs and directly proceed to the TBF1 mORF (Wang et al. 2022). Whether this IRES-mediated uORF skipping mech­ anism allows the derepression of other uORF-containing tran­ scripts in different stress conditions in plants is an exciting direction for future research.  

uORF regulation by phytohormone(s). The phytohormone auxin can also derepress uORF-containing transcripts by activat­ ing the kinase TARGET OF RAPAMYCIN (TOR) (see section ‘Translational regulation through modulation of translation fac­ tors and master translational regulators – TOR’ for details). In Arabidopsis, TOR promotes the translational re-initiation of uORF-containing mRNAs by directly phosphorylating eIF3h, a factor known to promote re-initiation. Activated TOR and  eIF3h  facilitate  the  polysome  loading  of  several uORF-containing  mRNAs,  including  those  of  AUXIN RESPONSE  FACTORs  (ARFs)  and  bZIP11 (Fig.  4C) (Schepetilnikov  et  al.  2013).  Notably,  TOR  derepresses uORF-containing transcripts to facilitate growth, while other examples  of  derepression  respond  to  stress  conditions. Because TOR is activated or repressed by many environmental cues or phytohormones, such as light, glucose, nutrients, auxin, and ABA (reviewed in Schepetilnikov and Ryabova 2018; McCready et al. 2020), it will be interesting to learn which genes with uORF-containing transcripts are controlled by TOR under different growth conditions.  

uORF regulation by nutrients and metabolites. In addition to boron control of minimum uORFs mentioned above (Tanaka et al. 2016), phosphate deficiency facilitates the derepression of several uORF-containing transcripts, including those in ARF4 and CYTOKININ RESPONSE FACTOR 10, which might promote root growth for phosphate uptake (Bazin et al. 2017).  

It is well documented that the repressiveness of several CPuORFs is modulated by metabolites. For example, sucrose, thermospermine, and galactinol regulate the activities of the CPuORFs on the mRNAs of the transcription factor genes bZIP11, SUPPRESSOR OF ACAULIS 51, and TBF1, respectively (Rook et al. 1998; Wiese et al. 2004; Imai et al. 2006; Rahmani et al. 2009; Cai et al. 2016; Yamashita et al. 2017; Zhu et al. 2018). Both choline and phosphocholine activate the CPuORF of the XIPOTL1 transcript, encoding a phos­ phoethanolamine N-methyltransferase, to repress phosphat­ idylcholine  biosynthesis (Alatorre-Cobos  et  al.  2012). Interestingly, the biosynthesis and degradation of polyamine are modulated by 2 homology groups of CPuORFs in the tran­ scripts  of  S-ADENOSYLMETHIONINE  DECARBOXYLASE (SAMDC) and POLYAMINE OXIDASEs (PAOs), respectively (Hanfrey et al. 2005; Guerrero-González et al. 2014). Finally, un­ der high-ascorbate conditions, a conserved uORF with a nonca­ nonical start site on GDP-L-galactose phosphorylase (GGP) mRNA suppresses downstream mORF translation (Laing et al. 2015; Hsu et al. 2016). GGP encodes a key enzyme for ascorbate biosynthesis. Thus, the suppression of GGP translation by the conserved uORF reduces ascorbate biosynthesis (Laing et al. 2015). The mechanism for uORF-dependent metabolite sensing remains unclear but is possibly due to interactions between the ribosome exit channel, the uORF peptide, and the metabolites (Van Der Horst et al. 2020).  

Most of the CPuORFs mentioned above respond to high me­ tabolite levels to repress the translation of their mORFs and thus negatively regulate the biosynthesis of the metabolites, ex­ cept the CPuORF of PAOs. In the presence of high polyamine levels, the PAO CPuORF is suppressed, promoting mORF trans­ lation for production of PAO for polyamine degradation, where­ as the SAMDC CPuORF suppresses SAMDC mORF translation, preventing the biosynthesis of polyamine. Thus, the 2 groups of CPuORFs act together to balance cellular polyamine levels (Hanfrey et al. 2005; Guerrero-González et al. 2014).  

Intriguingly, in contrast to derepression where uORFs are re­ pressive by default, CPuORF33 in the AtHB1 mRNA is not repres­ sive until activated. AtHB1 encodes a transcription factor that regulates hypocotyl and leaf development. Several observations suggest that the repressiveness of CPuORF33 requires photosyn­ thetic activity: (1) the uORF is not repressive in dark-grown seed­ lings; (2) the uORF is repressive only in the aerial tissues of plants grown under light-dark cycles; and (3) chemicals inhibiting chloroplast function abolish CPuORF33 repressiveness, as shown by a reporter assay (Ribone et al. 2017). Together, this finding provides an interesting example of uORF regulation linking the environment and plant development.  

# Why uORF regulation?  

Regulating gene expression by uORFs is energy inefficient be­ cause these mRNAs are not translated to their full potential.  

An alternative to producing the same amount of proteins is to generate fewer mRNAs but without uORF repression. Is there an advantage to having uORFs repress translation? Advances on this topic suggest that uORF-mediated regula­ tion offers at least 3 benefits: First, uORFs may provide feed­ back  regulation  for  maintaining  homeostasis  of  certain nutrients or metabolites (reviewed by Van Der Horst et al. 2020). Second, regulating gene expression at the translational level allows for quick responses to changing environments (Juntawong et al. 2014; Xu et al. 2017b). Finally, translational suppression could be a general mechanism to reduce gene expression noise (see below).  

Gene expression is inherently stochastic. Each step of the cen­ tral dogma, including the synthesis and decay of mRNAs and proteins, fluctuates. This stochasticity results in variance in pro­ tein numbers among genetically identical cells, known as gene expression noise (Raj and van Oudenaarden 2008). Recently, using both an experimental approach and mathematical mod­ eling,  uORFs  of  a  circadian  regulator,  TIMING  OF  CAB2 EXPRESSION 1 (TOC1), were shown to be critical to suppress the gene expression noise of TOC1 and facilitate the homogen­ eity of the circadian clock in Arabidopsis (Wu et al. 2022). This finding is consistent with prior studies in prokaryotes and other eukaryotes that lower translation can counteract the fluctu­ ation of transcription levels and reduces the variance of protein abundance. Because protein kinases and transcription factors are enriched among uORF-containing genes in plants (Kim et al. 2007; Wu and Hsu 2022), noise reduction for these critical regulators is likely important for stabilizing their regulatory net­ works despite the associated energy cost.  

# Internal ribosome entry site (IRES)  

An IRES allows the 43S PIC to bind to the internal region of the mRNA instead of binding to the $5^{\prime}$ cap first (Fig. 3). Besides re­ cent examples in defense mRNAs (Wang et al. 2022; see section on ‘uORFs’ above), another plant gene reported to possess an IRES is WUSCHEL (WUS; Cui et al. 2015). WUS encodes a tran­ scription factor that maintains the stem cell pool in the shoot apical meristem. The IRES-dependent translation of WUS tran­ scripts is controlled by La1 RNA-binding protein. La1 is mainly located in the nucleus, but it relocates to the cytoplasm in re­ sponse to environmental stress, such as UV and the radiomi­ metic  drug  zeocin.  In  the  cytoplasm,  La1  enhances  the translation of WUS transcripts, which correlates with the prolif­ eration of stem cells. Plants overexpressing La1 continue devel­ oping true leaves in the presence of zeocin, while wild-type plants cannot. Thus, the regulation of WUS by La1 through IRES-mediated translation can control stem cell proliferation in response to environmental stress.  

# RNA secondary structure  

One recent example of translational control through RNA sec­ ondary structures is related to development (Cho et al. 2018). Based on the hypothesis that phloem development is regulated by sucrose, a comparative transcriptome analysis was con­ ducted on poplar (Populus tremula) phloem/cambium-specific genes,  Arabidopsis  herbaceous  phloem/cambium-specific genes, and Arabidopsis sucrose-upregulated genes. One gene that met all 3 criteria encodes an RNA-binding protein named JULGI (“stream” or “plant shoot” in Korean; JUL). A screen using a random single-stranded RNA library revealed that JUL prefer­ entially binds to G-rich sequences and G-quadruplex. A search for phloem-specific mRNAs with potential G-quadruplex se­ quences in their $5^{\prime}$ UTRs identified SUPPRESSOR OF MAX2 1-LIKE4  (SMXL4)  and  SMXL5 as  potential  targets  of  JUL. SMXL4 and SMXL5 are key regulators of phloem differentiation. Follow-up experiments demonstrated that JUL binds to the G-quadruplex region in SMXL4 and SMXL5 mRNAs and re­ presses their translation and thus negatively regulates phloem differentiation.  

Another example of translational control mediated by RNA secondary structures is related to temperature sensing (Chung et al. 2020). A Ribo-seq experiment comparing Arabidopsis seedlings kept at $17~^{\circ}\subset$ or transferred to $27\ ^{\circ}C$ for 15 minutes identified hundreds of transcripts with en­ hanced or reduced translation. A subset of transcripts that were translationally upregulated contained a predicted hair­ pin structure located approximately 30 nt upstream of the start codon. Among these, PIF7 was a promising candidate, as it is homologous to PIF4, a master regulator of the thermo­ sensing pathway. Further experiments revealed that the translation of PIF7 mRNA is mediated by the hairpin struc­ ture responsive to temperature changes in the PIF7 mRNA and that PIF7 protein levels are positively correlated with hypocotyl growth in response to a warmer temperature, like­ ly through upregulation of auxin biosynthesis genes by PIF7.  

# Sequences recognized by RNA-binding proteins  

Two independent studies have described an elegant example of  translational  regulation  acting  on  the $3^{\prime}$ UTR  of EIN3-BINDING F-BOX 1 (EBF1) and EBF2 mRNAs in the ethyl­ ene signaling pathway (Li et al. 2015; Merchante et al. 2015). EBF1 and EBF2 are key repressors of the ethylene signaling pathway by targeting the master transcription factors of ethylene-regulated genes for proteasome-mediated degrad­ ation. It was a mystery that EBF1 and EBF2 mRNAs were up­ regulated in response to ethylene, but no increased protein levels were detected. In 1 of the 2 studies, Ribo-seq revealed that the translation of EBF1 and EBF2 mRNAs is downregu­ lated by ethylene (Merchante et al. 2015). In the second study, it was observed that small RNAs recognizing the EBF1 and $E B F2\;3^{\prime}$ UTRs overly accumulate in a mutant defect­ ive in the $5^{\prime}$ -to- $.3^{\prime}$ decay enzyme EXORIBONUCLEASE4 (XRN4) gene (also named ETHYLENE INSENSITIVE 5). Additionally, overexpression of the $E B F1\ 3^{\prime}$ UTR fragment results in ethyl­ ene insensitivity (Li et al. 2015). Both studies revealed that the translational control of EBF1 and EBF2 by ethylene de­ pends on the $3^{\prime}$ UTR of their mRNAs, particularly poly-U ele­ ments. Furthermore, EIN2, another signaling protein in the ethylene pathway, binds to the $3^{\prime}$ UTR of EBF1 and EBF2 mRNAs  and  brings  them  to  the  processing  bodies (P-bodies,  see  section  ‘Stability  and  spatial  dynamic  of mRNAs – P-bodies and stress granules’) with the help of NMD factors. Thus, the localization to the P-bodies prevents EBF1 and EBF2 mRNAs from being translated (Li et al. 2015; Merchante et al. 2015).  

Another example of translational control through $3^{\prime}$ UTR has been discovered in rice panicle development. FRIZZY PANICLE (FZP) was identified as a quantitative trait locus for grain number per panicle, and it was shown that an in­ crease in $F Z P3^{\prime}$ UTR length was associated with increased grain number, but the underlying mechanism was unknown (Bai et al. 2017b; Fujishiro et al. 2018). Recently, the presence of a conserved CU-rich element (CURE) was identified in the $F Z P3^{\prime}$ UTR. The deletion of the CURE in the $F Z P3^{\prime}$ UTR re­ sulted in an increase in FZP translation levels, without affect­ ing FZP mRNA levels. Similar results were observed by overexpressing the $F Z P3^{\prime}$ UTR fragment, which presumably competes with the endogenous FZP $3^{\prime}$ UTR. Furthermore, the CUREs were shown to be bound by POLYPYRIMIDINE TRACT-BINDING  PROTEIN  1  (PTB1)  and  PTB2. Manipulating PTB1 or PTB2 expression altered FZP transla­ tion and grain number (Chen et al. 2022).  

RNA-binding proteins may also act on elements in the $5^{\prime}$ UTR for translational control. LA AND RELATED PROTEINs (LARPs) are conserved RNA-binding proteins in eukaryotes. Plants possess LARP1 and LARP6 families, each encoded by 3 genes (A to C). LARP6C is specifically expressed in pollen. Recently, using RNA-immunoprecipitation sequencing fol­ lowed by motif analysis, LARP6C was discovered to binds to a B-box motif (either U-rich or UC/UCC-rich) within the $5^{\prime}$ UTRs of its target mRNAs. Interestingly, analysis of a re­ porter construct for its target revealed that LARP6C can switch from a translational activator to a translation repres­ sor when pollen grains transition from dormancy to germin­ ation (Billey et al. 2021).  

Another example of RNA-binding protein acting on a $5^{\prime}$ UTR to regulate translation is related to phytochrome signal­ ing. Phytochromes are photoreceptors that sense red and far-red light and switch between Pr and Pfr forms. When ex­ posed to red light, the Pr form switches to the Pfr form, which relocates to the nucleus, promoting gene expression for photomorphogenesis (reviewed in Cheng et al. 2021). In the  cytosol,  the  Pfr  form  also  interacts  with  the RNA-binding protein PENTA1 (PNT1). PNT1 binds to the $5^{\prime}$ UTR of PROTOCHLOROPHYLLIDE REDUCTASE (PORA) in­ volved in chlorophyll biosynthesis and represses its transla­ tion  in  the  presence  of  Pfr.  The  inhibition  of  PORA translation  is  critical  for  adjusting  chlorophyll  contents when seedlings transition from far-red light to white light conditions (Paik et al. 2012). The specific mRNA features on PORA mRNA that contribute to the translational control remain to be identified.  

# Readthrough  

A special mechanism of translational control is readthrough at the stop codon. It occurs when a near-cognate tRNA, in­ stead of release factors, binds to the stop codon at the A-site of the ribosome and continues translation. The result­ ing C-terminal extensions of the proteins may alter the pro­ tein subcellular localization, function, or stability.  

Interestingly, the first known readthrough example in Arabidopsis is eukaryotic RELEASE FACTOR 1 (eRF1). The eRF1 $3^{\prime}$ UTR possesses sequence features predicted to be tar­ geted by NMD, including a long $3^{\prime}$ UTR and an intron dis­ tantly downstream of the stop codon. It was shown that when eRF1 levels are high, readthrough is unfavored, pro­ moting NMD and reducing eRF1 mRNA levels. By contrast, when eRF1 levels are low, readthrough is favored, preventing the eRF1 $3^{\prime}$ UTR from being targeted by NMD. Thus, the eRF1 expression forms an autoregulatory circuit (Nyikó et al. 2017).  

More recently, mining available Arabidopsis Ribo-seq data re­ vealed that 144 transcripts exhibit stop codon readthrough (Sahoo et al. 2022). Among the 3 possible stop codons, UGA is the preferred stop codon for translational readthrough. In contrast to the consensus readthrough context (stopCARYYA)  defined  by  a  previous  mutational  analysis  in Nicotiana benthamiana (Skuzeski et al. 1991), the nt immediate­ ly upstream and downstream of the stop codons was found to be enriched for A or U in the readthrough cases (Sahoo et al. 2022). Remarkably, among the 144 transcripts with read­ through, 34 encode RNA-binding proteins. Although the bio­ logical significance of these readthrough events remains to be determined, it was demonstrated that the readthrough of CURVATURE THYLAKOID 1B results in the inclusion of a nuclear localization signal peptide in the encoded protein, causing the new protein to localize to the nucleus, in contrast to the original isoform that localizes in the cytoplasm (Sahoo et al. 2022).  

# G-content in the poly(A) tail  

Contrary to the common belief that poly(A) tails are composed exclusively of adenosines, work in human cell lines has revealed the existence of non-A residues in poly(A) tails (Chang et al. 2014). Similarly, $\sim\!16\%$ of Arabidopsis poly(A) tails contain at least 1 non-A residue, and the majority of these are guanines. These G residues break the poly(A) tails into multiple A tracts. Genome-wide analysis showed that the G content in poly(A) tails is negatively correlated with the efficiency of PABP binding and translation, without affecting mRNA stability. Consistent with these observations, transcripts with high PABP binding dis­ play a significant reduction in translation efficiency in pabp mu­ tants (Zhao et al. 2019b).  

# Small interfering RNAs (miRNAs and siRNAs)  

In addition to mRNA cleavage, small interfering RNAs (miRNAs and siRNAs) can target specific mRNA sequences for transla­ tional repression (Brodersen et al. 2008). Consistent with the ex­ pected roles of miRNAs in translational repression, Ribo-seq analysis showed that predicted miRNA targets are associated with lower translation efficiency in Arabidopsis and tomato (Liu et al. 2013; Wu et al. 2019).  

Once miRNAs are produced, they bind to AGO and form the RNA-induced silencing complex (RISC), which can trigger mRNA cleavage or inhibit translation. miRNA-mediated translational inhibition was shown to occur at the ER through interaction between AGO1 and an integral ER mem­ brane protein, ALTERED MERISTEM PROGRAM1 (AMP1; Li et al. 2013).  

More recently, HYPONASTIC LEAVES1 (HYL1), a core component of miRNA processing machinery, was shown to also function as a miRNA-mediated translational inhibitor (Yang  et  al.  2021).  HYL1  is  a  double-stranded  RNA (dsRNA)-binding protein that is dually localized to the nu­ cleus and cytoplasm, but the function of cytosolic HYL1 was unknown. Interestingly, examining the role of cytosolic HYL1  by  introducing  a  construct  encoding  a  nucleuslocalized form of HYL1 in a null hyl1 mutant background revealed that cytosolic HYL1 does not regulate miRNA bio­ genesis or the cleavage of the target mRNAs. Instead, the tar­ get mRNAs accumulate more proteins in the absence of cytosolic HYL1. Further experiments showed that cytosolic HYL1 localized on the ER and associated with AGO1, AMP1, and miRNA-targeted mRNAs. Importantly, HYL1 fa­ cilitates AGO1 loading onto polysomes without affecting the distribution of the target mRNAs in the polysome profile. This result suggests that miRNA-mediated translational in­ hibition occurs after translation initiation.  

Recent work showed that siRNAs, specifically 22-nt siRNAs, play a role in translational regulation (Wu et al. 2020). Plants produce 21-, 22-, and 24-nt siRNAs. Although the functions of the 21- and 24-nt siRNAs are well studied, the function of 22-nt siRNAs was largely unknown due to their low abun­ dance. In double mutants lacking either the $5^{\prime}$ -to- $.3^{\prime}$ decay enzyme XRN4, or the $3^{\prime}$ -to- $.5^{\prime}$ mRNA decay enzyme SUPER KILLER2 (SKI2), along with DICER-LIKE 4 (DCL4), i.e. xrn4 dcl4 and ski2 dcl4, high levels of 22-nt siRNAs are produced, particularly from 2 genes encoding nitrate reductases NIA1 and NIA2. Surprisingly, overaccumulation of 22-nt siRNAs does not lead to a decrease in NIA1 and NIA2 mRNA levels but instead causes a dramatic reduction in their protein le­ vels. Polysome profile revealed global translational repression as well as specific translational repression of NIA1 and NIA2 mRNAs. Consistent with a role for 22-nt siRNAs in transla­ tional repression, a mutant defective in siRNA loading to RISC  (ago1-27)  rescues  the  overaccumulation  of  22-nt siRNAs and restored NIA1 and NIA2 protein levels.  

# Cis-natural antisense RNAs  

Cis-natural antisense transcripts (Cis-NATs) are noncoding RNAs that overlap with protein-coding genes and are tran­ scribed from the opposite DNA strand. Cis-NATs and their mRNA partners are expected to form dsRNA and lead to RNA degradation; however, some cis-NATs were found to in­ crease the translation of their cognate mRNAs (Deforges et al. 2019; Reis et al. 2021). In the case of rice PHOSPHATE1.2 (PHO1.2),  instead  of  long-range  base-pairing  between  the sense-antisense transcripts, both sense (PHO1.2) and antisense transcript $(c i s–N A T_{P H O1.2})$ alone have extensive intramolecular structures.  Upon  interaction  between  the  sense-antisense transcripts, local structural changes occur and reduce inhibition of a high GC region located ${\sim}350$ nt downstream of the start codon, which is associated with translational repression. It was suggested that the interaction between the sense-antisense transcripts leads to increased 80S formation and possibly en­ hanced translation elongation, resulting in increased PHO1.2 translation (Reis et al. 2021).  

# Translational regulation through modulation of translation factors and master translational regulators  

Translation initiation is the rate-limiting step of translation and is critical for controlling the overall translation rates (Shah et al. 2013). Two points of regulation for translation initiation have been the research focus: (1) recognition of the $5^{\prime}$ cap, which involves the eIF4F complexes; and (2) for­ mation of the ternary complex, which is a component of the PIC (Fig. 5). To date, studies have revealed that both points of regulation are controlled through the phosphoryl­ ation of the relevant translation factors. Mass spectrometry and  phosphoproteomics  have  enabled  exciting  break­ throughs in identifying new factors and regulatory mechan­ isms involved in translation initiation. Below, we discuss the approaches and advancements in this area. Several newly dis­ covered players directly phosphorylate or interact with core translation initiation factors, such as eIF(iso)4Es and eIF(iso)4Gs. In addition to defining the interactors, recent studies have also extended the upstream or downstream signaling components of 3 conserved master translational regulators, namely TOR, SNF1 RELATED PROTEIN KINASE 1 (SnRK1), and GENERAL CONTROL NONDEREPRESSIBLE 2 (GCN2; Fig. 5).  

# Regulation of eIF4F complex assembly through 4E-binding proteins  

As eIF4E directly binds to the $5^{\prime}$ cap and recruits eIF4G, which in turn recruits other translation factors, the regulation of eIF4E plays a critical role in translational control, and identi­ fying eIF4E-binding proteins has been of great interest. Most eIF4E-binding proteins in other eukaryotes are translational repressors that compete with eIF4G for binding to eIF4E (Grüner et al. 2018). However, no obvious homologs to yeast or mammalian 4E-binding proteins (4E-BPs) have been iden­ tified in plants. Although Arabidopsis LIPOXYGENASE 2 and BASIC TRANSCRIPTION FACTOR 3 have been shown to bind to eIFiso4E in vitro and by yeast 2-hybrid, respectively (Freire et al. 2000; Freire 2005), they do not possess the evolutionar­ ily conserved 4E-binding site (4E-BS: YXXXXLΦ; X being any amino acid and $\Phi$ being a hydrophobic amino acid), and their roles in translational control remain unclear.  

![](images/f211b0280e15447ae36ac020912e5e2ff41b773ba426e7a55c1c3860b3abc189.jpg)  
Figure 5. Overview of translational regulation through modulation of translation factors and signaling through conserved master translational reg­ ulators. A) Two key points of translation initiation are under intensive study: assembly of the eIF4F complex (highlighted in green) and formation of the ternary complex (highlighted in blue). Proteins interacting with and/or phosphorylating the eIF4F complex are listed. The conserved master translational regulators, including TOR, SnRK1 and GCN2, are highlighted in yellow. B) Summary of activation, target proteins, and effects on trans­ lation of the 3 master translational regulators, TOR, SnRK1, and GCN2. Note that TOR–S6K–eS6 signaling may not have observable effects on trans­ lation as new evidence suggests (Dasgupta et al. 2023); SnRK1 also connects to translation through TOR, but currently no evidence supports that TOR is directly phosphorylated by SnRK1.  

One eIF4E-binding protein, ESSENTIAL FOR POTEX-VIRUS ACCUMULATION (EXA1), was identified from a mutant screen that showed enhanced autoimmunity (Wu et al. 2017). EXA1 binds to eIF4E as shown by split-luciferase com­ plementation and immunoprecipitation in N. benthamiana. EXA1 contains an eIF4E-binding motif as well as a conserved glycine-tyrosine-phenylalanine  domain  that  binds  to proline-rich sequences. Consistent with its potential role as a translational repressor, the exa1 mutant has increased pro­ tein levels for several defense-related nucleotide-binding leucine-rich receptors. However, exa1 does not affect general protein synthesis. How EXA1 specifically regulates a particu­ lar set of mRNAs remains to be determined.  

A  plant-specific  eIF4E-binding  protein,  known  as CONSERVED BINDING OF eIF4E1 (CBE1), was identified by searching for the canonical 4E-BS motif among potential capbinding proteins (Patrick et al. 2018). CBE1 is encoded by a single-copy gene in Arabidopsis and is highly conserved across land plants. CBE1 can form cap-binding complexes in vitro and in vivo. cbe1 mutant plants are viable but exhibit delayed flowering and upregulated expression of mitosisrelated genes. Further investigation is needed to determine the role of CBE1 in translation.  

More recently, an eIF4E-binding protein named CERES (named after the Roman goddess of agriculture) was identi­ fied through yeast 2-hybrid (Toribio et al. 2019). Like CBE1, CERES is encoded by a single copy gene in Arabidopsis, is specif­ ic to plants, and is highly conserved from mosses to angios­ perms.  CERES  interacts  with  eIF4E  via  the  canonical 4E-binding site, which is also present in eIF4G (Miras et al. 2017). Interestingly, CERES appears to replace the role of eIF4G, forming a translation initiation complex with eIF4E, eIF4A, eIF3, and PABP. Furthermore, in vitro analysis revealed that CERES promotes translation, rather than repressing trans­ lation like most 4E-binding proteins in other organisms. CERES also promotes translation of specific mRNAs involved in light response and starch accumulation (Toribio et al. 2019).  

In addition to proteins containing the canonical 4E-BS, other proteins may interact with 4E to regulate translation. An interesting regulation of eIF4E controlling local transla­ tion is the FERONIA (FER) pathway (Zhu et al. 2020). FER is a receptor-like kinase located at the plasma membrane at the tip of root hair cells. Reanalysis of previous yeast 2-hy­ brid and immunoprecipitation-mass spectrometry screens revealed that FER interacts with several translation factors in­ volved in initiation and elongation, as well as ribosomal pro­ teins. The interaction with FER sequesters eIF4E to the plasma membrane of root hair tips. Upon binding to its lig­ and peptide RAPID ALKALINIZATION FACTOR 1 (RALF1), FER undergoes autophosphorylation and triggers eIF4E phos­ phorylation. The phosphorylated eIF4E moves away from the plasma membrane and binds to the $5^{\prime}$ cap and eIF4G to translate specific mRNAs involved in root hair growth, in­ cluding those for ROOT HAIR DEFECTIVE 6-LIKE 4 (RSL4).  

RSL4 is a transcription factor that negatively regulates root hair  growth  and  represses  RALF1  expression.  Thus,  the RALF1–FER–eIF4E–RSL4 signaling axis forms a sophisticated negative feedback loop to control local translation and root hair growth. It is worth further investigation to elucidate how the specific translation of RSL4 mRNA is achieved once eIF4E is released from the plasma membrane. One possibility is that RSL4 mRNA is also specifically localized at the root hair tip.  

Similar to the competition for binding to eIF4E and disrupt­ ing  eIF4F  complex  assembly,  the  RNA-binding  protein SUPPRESSOR OF ABAR-OVEREXPRESSOR 1 (SOAR1) inter­ acts with eIFiso4G1 and eIFiso4G2, preventing the assembly of eIFiso4F complexes and repressing translation initiation (Bi et al. 2019). SOAR1 was initially discovered from a suppres­ sor mutant screen in plants overexpressing the putative ABA receptor ABAR (Mei et al. 2014). In addition to eIFiso4G1 and eIFiso4G2, SOAR1 can bind to the mRNA of a key ABA signal­ ing component, ABA INSENSITIVE5 (ABI5). The binding of ABI5 mRNA releases eIFiso4G1 and eIFiso4G2 from SOAR1 and thus allows the assembly of eIFiso4F complexes and translation ini­ tiation (Bi et al. 2019). The mechanisms by which SOAR1 effi­ ciently sequesters high-abundance eIFiso4G1 and eIFiso4G2 and how ABI5 mRNA binding to SOAR1 releases eIFiso4G1 and eIFiso4G2 remain unclear.  

# TOR acts on multiple aspects of translation  

TOR is a master translational regulator. TOR belongs to the phosphatidylinositol 3-kinase–related kinase family and is conserved throughout eukaryotes. TOR is regularly reviewed (Schepetilnikov and Ryabova 2018; Shi et al. 2018; Pacheco et al. 2021; Scarpin et al. 2022). Here, we summarize critical knowledge for understanding TOR regulation in translational control and discuss recent advances in the field.  

In mammals, TOR regulates at least 3 aspects of translation: (1) translation initiation is regulated by 4E-BPs, which compete with eIF4G for binding to eIF4E. When cells are in a high energy state, 4E-BPs are phosphorylated by TOR, preventing them from binding to eIF4E, which allows the efficient formation of the eIF4F complex and promotes translation initiation. (2) TOR phosphorylates S6 Kinase 1 (S6K1), which in turn phosphory­ lates the ribosomal protein of small subunit 6, RPS6, or eS6 ac­ cording  to  the  updated  ribosomal  protein  nomenclature (Scarpin et al. 2023). Note that the significance of eS6 phosphor­ ylation  in  translation  remains  controversial  (reviewed  in Meyuhas 2015). (3) TOR regulates ribosome biogenesis by con­ trolling the translation of ribosomal protein mRNAs through $5^{\prime}$ terminal oligopyrimidine tract $\mathbf{\nabla}^{5^{\prime}}$ TOP) motifs (reviewed in Burkart and Brandizzi 2021).  

In plants, although no sequence homologous 4E-BPs have been identified, the TOR–S6K–eS6 axis is well conserved, and variations of TOR- $.5^{\prime}$ TOP-ribosomal protein regulation have recently been identified (reviewed in Pacheco et al. 2021; Scarpin et al. 2022). TOR clearly plays an important role in plant translational control, as global polysome levels are re­ duced in tor RNA interference (RNAi) lines or when TOR ac­ tivity is disturbed (Deprost et al. 2007).  

# Organization of the TOR complex  

Most eukaryotes contain 2 TOR complexes (TORC): TORC1 and TORC2. Only TORC1 is conserved throughout eukar­ yotes  (Brunkard  2020).  TORC1  is  composed  of  TOR, REGULATORY-ASSOCIATED PROTEIN OF TOR (RAPTOR), and LETHAL WITH SEC13 PROTEIN 8 (LST8; Scarpin et al. 2020). In Arabidopsis, TOR is encoded by a single gene, whereas RAPTOR and LST8 are both encoded by 2 genes (RAPTOR1 and RAPTOR2; LST8-1 and LST8-2; Menand et al. 2002; Mahfouz et al. 2006; Moreau et al. 2012). RAPTOR as­ sists in the recruitment and phosphorylation of TOR sub­ strates, and LST8 binds to the TOR kinase domain, likely regulating TOR activity. Downstream of TOR, Arabidopsis S6K is also encoded by 2 genes (S6K1 and S6K2). Another conserved TOR substrate that regulates translation is TYPE 2A-PHOSPHATASE-ASSOCIATED  PROTEIN  46  (TAP46), which is encoded by a single gene (Ahn et al. 2011).  

As TOR knockout mutants are embryonic lethal, many stud­ ies of TOR have used inducible RNAi lines and treatment of TOR inhibitors, such as rapamycin, Torin-1, and the secondgeneration inhibitor AZD8055 (Deprost et al. 2007; Xiong and Sheen 2012; Xiong et al. 2013). TOR activity is often monitored through the phosphorylation status of S6Ks or eS6.  

# Upstream regulators, interactors, and downstream targets of TOR  

TOR senses both intracellular and environmental cues, and its activity is regulated by nutrients (nitrogen, nucleotide, and ami­ no acid availability; Cao et al. 2019; Busche et al. 2021; Liu et al. 2021), energy (glucose, light, and the circadian clock; Xiong et al. 2013; Chen et al. 2018; Li et al. 2019), phytohormones (auxin, brassinosteroids, and ABA; Schepetilnikov et al. 2013; Zhang et al. 2016; Wang et al. 2018; Montes et al. 2022), and stress (Meteignier et al. 2017). Under favorable conditions, TOR pro­ motes translation, growth, and anabolism (Pfeiffer et al. 2016; Li et al. 2017) and suppresses catabolism, such as autophagy (Zhang et al. 2016; Pu et al. 2017).  

A phosphoproteomic analysis in response to TOR inhibitors (rapamycin and AZD8055) and a targeted interactome study of TOR complex components (TOR, RAPTOR1, LST8-1) and TOR targets (S6K1, TAP46) revealed an intricate TOR signaling network in plants (Van Leene et al. 2019). In addition to several translation initiation factors, the study determined that CBE1, which interacts with eIF(iso)4E, is regulated and phosphorylated by TOR, suggesting that TOR also mediates the formation of cap-binding complexes in plants (Van Leene et al. 2019). Moreover, parallel profiling of the TOR transcriptome, transla­ tome, proteome, and phosphoproteome placed Arabidopsis LARP1 downstream of TOR in the signaling pathway, which is consistent  with  other  eukaryotes (Scarpin  et  al.  2020). Together these phosphoproteome studies identified nearly 160 TOR-sensitive phosphoproteins; among them, translation factors and ribosomal proteins were enriched, highlighting the critical role of TOR in translational control (Van Leene et al. 2019; Scarpin et al. 2020, 2022).  

Additionally, a suppressor screen performed in the lst8 mu­ tant background identified YET ANOTHER KINASE 1 (YAK1) as a TOR target. YAK1 interacts with RAPTOR and is directly phos­ phorylated by TOR (Forzani et al. 2019), which is also supported by phosphoproteomic analysis (Van Leene et al. 2019). More re­ cently, screening mutants with altered hypocotyl elongation in response to TOR inhibition in the dark discovered that EIN2, a key component of ethylene signaling, is also targeted by TOR (Fu et al. 2021). EIN2 can move between the cytoplasm and the nucleus depending on its phosphorylation status, which is controlled by ethylene (Ju et al. 2012; Qiao et al. 2012). Glucose-activated TOR directly phosphorylates EIN2 through sites distinct from those targeted by the canonical ethylene sig­ naling pathway, and the phosphorylated EIN2 cannot enter the nucleus to orchestrate transcription of ethylene-responsive genes (Fu et al. 2021). Because EIN2 can bind to the $3^{\prime}$ UTR of EBF2 and regulates its translation (Li et al. 2015; Merchante et al. 2015), an interesting question to explore is whether the TOR–EIN2 axis is also involved in translational regulation.  

# Roles of TOR in translational control  

TOR and eIF3h in uORF-containing mRNAs. As in mammalian systems, eIF3h in Arabidopsis can serve as a scaffold for either TOR or S6K1, depending on TOR activity. When TOR is inactive, S6K1 binds to eIF3h; when TOR is activated by auxin, TOR binds to and phosphorylates eIF3h. Furthermore, the TOR–eIF3h complex is bound to polysomes, which is important for the reinitiation of uORF-containing mRNAs, including that of ARFs and bZIP11. By contrast, treatment with Torin-1 leads to depho­ sphorylation of TOR and S6K1, which promotes the formation of the S6K1–eIF3h complex on polysomes. Consistent with this notion, the polysome loading of uORF-containing mRNAs is sig­ nificantly reduced in TOR-RNAi seedlings. These results demon­ strate  the  role  of  TOR  in  the  translational  control  of uORF-containing mRNAs (Schepetilnikov et al. 2013).  

A small GTPase, RHO OF PLANTS 2 (ROP2), is also in­ volved in auxin–TOR signaling. In response to auxin, ROP2 binds to and activates TOR. Plants with high levels of active ROP2 exhibit high levels of active TOR and S6K1. By contrast, TOR activation by auxin is abolished in rop2 rop6 ROP4-RNAi seedlings. ROP2 also determines the localization of TOR in the endosome. It was proposed that the transient binding of ROP2 to TOR in the endosome is associated with TOR ac­ tivation (Li et al. 2017; Schepetilnikov et al. 2017).  

TOR and S6K. Several lines of evidence suggest that the TOR–S6K–eS6 pathway is involved in light-enhanced trans­ lation: (1) the correlation between eS6 phosphorylation and photosynthesis activity during the light period of the di­ urnal cycle; and (2) the association of phosphorylated eS6 with polysomes (Turkina et al. 2011; Boex-Fontvieille et al. 2013; Enganti et al. 2018). When comparing Arabidopsis seedlings kept in the dark with those exposed to light, it was shown that light increases the fraction of mRNAs bound to  polysomes (Liu  et  al.  2012).  This  light-enhanced translation is decreased in the photoreceptor mutant phyA, while high translation is maintained in the negative regulator of light-signaling mutant cop1 in the dark. This finding sug­ gests that light-enhanced translation is under the control of light signaling. In agreement, light-induced eS6 phosphoryl­ ation is abolished in the red/far-red and blue light photorecep­ tor mutants phyA, phyA cry1 cry2, and eS6 phosphorylation remains high in the light-signaling mutant cop1 in darkness and in the light. It was further shown that light-induced eS6 phosphorylation is decreased in TOR-RNAi seedlings. By con­ trast, in the cop1 mutant, TOR signaling remains active in both dark and light conditions and is reduced by a TOR inhibi­ tor or in TOR-RNAi seedlings. Together, these data reveal that TOR–S6K1–eS6 is involved in light-enhanced translation, which is critical for photomorphogenesis (Chen et al. 2018).  

Note that the biochemical and physiological significance of eS6 phosphorylation remains controversial in yeast and mam­ mals (Meyuhas 2015). New evidence in Arabidopsis shows that an eS6 phospho-deficient variant largely does not affect translation, as shown by polysome profiling (Dasgupta et al. 2023). This negative result is consistent with the findings in yeast using Ribo-seq (Yerlikaya et al. 2016). eS6 phosphorylation may still provide physiological advantages, as plants accumulat­ ing the eS6 phospho-deficient isoform showed certain growth defects compared with plants with the eS6 phospho-mimic iso­ form (Dasgupta et al. 2023).  

Additional downstream targets of S6K have been discov­ ered (Lee et al. 2017). Arabidopsis MA3-CONTAINING TRANSLATION REGULATORY FACTORs (MRF1 to MRF4) are homologs of the human tumor suppressor PROGRAMMED CELL DEATH 4, which binds to eIF4A through its MA3 domains, inhibiting eIF4A activity as well as the eIF4A–eIF4G interaction (Loh et al. 2009). Consistent with an expected role in regulating translational initiation, Arabidopsis MRF1 interacts with eIF4A and is associated with ribosomal subunits and monosomes, as revealed by polysome profiling. $^{35}\mathsf{S}$ -methionine labeling and polysome profiling analysis showed that MRF-RNAi plants ex­ hibit reduced translation under low-energy conditions (dark and starvation) compared with the wild type. Interestingly, MRF1 expression, MRF1 phosphorylation, and ribosome associ­ ation are regulated not only by the energy status but also by TOR activity. Furthermore, MRF1 interacts with S6K and is a dir­ ect substrate of S6K (Lee et al. 2017).  

TOR and TAP46. TAP46 is a regulatory subunit of protein phosphatase 2A, which is a downstream target of TOR in yeast and mammals. Arabidopsis TAP46 can also be phos­ phorylated by TOR. TAP46-RNAi plants display similar pheno­ types as TOR silencing, including a reduction in global translation, activation of autophagy, and growth arrest. This observation supports TAP46 as a downstream effector of TOR signaling (Ahn et al. 2011). However, how the TOR– TAP46 axis acts on translational regulation remains unclear.  

TOR–LARP1– $.5^{\prime}$ TOP. Multi-omics profiling of TOR signaling in Arabidopsis revealed that LARP1 acts downstream of TOR (Scarpin et al. 2020). In mammals, TOR–LARP1 signaling reg­ ulates ribosomal protein mRNAs through their $5^{\prime}$ TOP motifs. Analyzing Arabidopsis downstream targets of TOR– LARP1 identified both conserved and plant-specific $5^{\prime}$ TOP mRNAs. Although these $5^{\prime}$ TOP mRNAs do not strictly encode ribosomal proteins, many are related to ribosome biogenesis or translation (Scarpin et al. 2020, 2022).  

As near 160 TOR-sensitive phosphoproteins have been re­ cently identified (Van Leene et al. 2019; Scarpin et al. 2020), further characterization of these proteins will facilitate our understanding of the detailed mechanisms of mRNA transla­ tional regulation by TOR.  

# SnRK1 integrates environmental cues/  

# phytohormones with translational control  

SnRK1 is an evolutionarily conserved protein kinase that func­ tions in energy sensing in plants. It is homologous to mamma­ lian AMP-activated protein kinase (AMPK) and yeast sucrose non-fermenting 1 (SNF1). SnRK1, AMPK, and SNF1 are hetero­ trimeric enzymes that consist of a catalytic subunit (α) with ki­ nase activity and 2 regulatory subunits (β and γ). In Arabidopsis, the α subunit is encoded by 2 genes, SnRK1α1 and SnRK1α2, which have been the focus of much SnRK1 research (reviewed in Broeckx et al. 2016; Peixoto and Baena-González 2022).  

# SnRK1 interacts with translation factors eIF(iso)4E and eIFiso4G1  

Searching for the conserved SnRK1/AMPK/SNF1 binding sites among Arabidopsis translation initiation factors re­ vealed that both eIF4E and eIFiso4E contain 2 SnRK1 consen­ sus binding sites (Bruns et al. 2019). In vitro kinase assays and in vivo interaction studies support the idea that SnRK1α2 dir­ ectly interacts with and phosphorylates eIF4E and eIFiso4E. Furthermore, SnRK1-mediated phosphorylation of eIF(iso) 4E  is  associated  with  global  translational  repression. Consistent with this idea, the reduced polysome formation caused by overexpression of Arabidopsis SnRK1α2 in yeast could be remedied by a nonphosphorylatable variant of eIF(iso)4E but not by a phosphomimic variant of eIF(iso)4E. In addition to interacting with eIF4E, a phosphoproteo­ mic analysis comparing wild-type and snrk1α1 mutant plants under submergence identified eIFiso4G1 as a poten­ tial SnRK1 target (Cho et al. 2016). It was further demon­ strated that SnRK1 phosphorylates eIFiso4G1 in response to submergence, which, in turn, regulates global translation­ al repression and the specific translation of mRNAs import­ ant for hypoxia responses. The eifiso4g1 knockout mutant and variants lacking phosphorylation are more sensitive to submergence  and  affect  the  translation  of  hypoxiaresponsive mRNAs (Cho et al. 2019).  

# SnRK1 integrates ABA signaling and plant growth/stress responses  

Recent studies on SnRK1 provide an elegant example con­ necting the phytohormone ABA, TOR, and plant growth.  

As the snrk1α1 snrk1α2 double mutant is not viable, insights into the role of SnRK1α were gained by studying a sesquimu­ tant $(s n r k1\alpha1^{-/-}\ s n r k1\alpha2^{+/-)}$ or inducible knockdown lines (Belda-Palazón et al. 2020; Henninger et al. 2022). Because the sesquimutant shows reduced ABA-mediated growth re­ pression, the role of TOR, a growth-promoting kinase, was ex­ amined under ABA treatment. It was shown that SnRK1α, but not SnRK2 (a key player in ABA signaling), directly inter­ acts with TOR; this interaction is enhanced in the presence of ABA. Furthermore, SnRK1α is required for TOR repression, as TOR repression in response to ABA is slower in the snrk1α mutant. Under non-ABA treatment conditions, SnRK2 forms a complex with SnRK1 and PROTEIN PHOSPHATASE TYPE 2C (PP2C, a repressor of ABA signaling), promoting growth. By contrast, in the presence of ABA, PP2C binds to ABA and the ABA receptor, thus releasing SnRK1, which represses TOR and reduces growth (Belda-Palazón et al. 2020).  

An independent study has shown that, in the presence of ABA, SnRK2 is activated and phosphorylates RAPTOR, which also inactivates TOR, repressing growth. In nonstress condi­ tions, TOR phosphorylates the ABA receptors, repressing the ABA/stress response (Wang et al. 2018). Thus, SnRK1 and SnRK2 coordinately regulate TOR activity, as well as plant growth and stress responses.  

More recently, it was shown that SnRK1 changes its subcel­ lular localization from the nucleus to the cytoplasm in re­ sponse  to  ABA.  Chemicals  blocking  nuclear  export  or fusing SnRK1α1 to a nucleus localization signal abolished ABA-dependent TOR repression (Belda-Palazón et al. 2022). SnRK1α is also recruited into stress granules in response to heat stress through interaction with the Tudor staphylococ­ cal nuclease (Gutierrez-Beltran et al. 2021). Thus, the subcel­ lular localization of SnRK1α connects environmental input and downstream signaling responses.  

# GCN2 controls ternary complex formation and global translation initiation  

Another important and conserved protein kinase involved in translational control is GCN2, which phosphorylates the $\mathfrak{a}$ subunit of heterotrimeric eIF2. eIF2 plays a unique role in translation initiation as it is part of the ternary complex [eIF2, GTP, Met-tRNA(i)] that carries the initiator tRNA to the 40S ribosome subunit. In yeast and mammals, the forma­ tion of the ternary complex requires efficient GDP/GTP ex­ change on eIF2, with the help of the guanine nucleotide exchange factor eIF2B. Under stress conditions, a highly con­ served serine residue in eIF2α is phosphorylated. The phos­ phorylated  eIF2α  tightly  binds  to  eIF2B,  inhibiting eIF2B-facilitated GDP-GTP exchange. As a result, the avail­ able ternary complex for translation is reduced and leads to global translational repression (reviewed in Browning and Bailey-Serres 2015). Unlike the multiple protein kinases regulating eIF2α phosphorylation in other eukaryotes, only 1 protein kinase, GCN2, is known to phosphorylate eIF2α in plants and yeast (Lageix et al. 2008; Zhang et al. 2008).  

In plants, eIF2α phosphorylation has been associated with numerous abiotic/biotic stressors and phytohormones, in­ cluding reactive oxygen species, UV light, cold, salt, wound­ ing, bacterial infection, ABA, ethylene, salicylic acid, and methyl-jasmonate  (Lageix  et  al.  2008;  Liu  et  al.  2019; Lokdarshi et al. 2020b). Consistent with the role of GCN2 ac­ tivation and eIF2α phosphorylation in translational repres­ sion,  amino  acid  or  purine  starvation,  which  activates GCN2, represses global translation; this repression is reduced in gcn2 mutant plants (Lageix et al. 2008; Wang et al. 2017).  

However, other evidence has challenged whether eIF2α phosphorylation has a strong inhibitory effect on translation. Plant eIF2 is known to exhibit a weaker affinity for GDP than mammalian eIF2, leading to the suggestion that plant eIF2 GDP/GTP exchange does not necessarily need the help of eIF2B (Browning and Bailey-Serres 2015). Moreover, the con­ ditions that activate eIF2α phosphorylation do not always lead to translational repression or depend on GCN2 (Izquierdo et al. 2018; Lokdarshi et al. 2020a; Zhigailov et al. 2020). For a com­ prehensive discussion on this topic, see (Lokdarshi and von Arnim 2022). Below, we highlight recent work related to the role of GCN2 in global translational control.  

First, transcriptome and polysome profiling analyses were performed in wild-type and gcn2 plants under a condition that activates GCN2. These analyses uncovered a set of mRNAs that escape translational repression dependent on GCN2. Surprisingly, unlike the findings in yeast and mam­ mals, plant mRNAs differentially regulated by GCN2 are not  enriched  for  specific  functions  or  possess  distinct mRNA features, such as uORFs (Lokdarshi et al. 2020a). Further investigation is required to understand how GCN2 specifically regulates the translation of these mRNAs.  

A recent study underscores the importance of GCN2 in global translational control in response to submergence (Cho et al. 2022). Upon submergence, ethylene is trapped in the submerged tissues, and it is known that ethylene trig­ gers eIF2α phosphorylation (Lageix et al. 2008), suggesting a role  for  GCN2/eIF2α  phosphorylation  in  submergence. Under submergence, eIF2α phosphorylation is enhanced and global polysome loading of mRNAs is reduced. To char­ acterize the effect of GCN2/eIF2α phosphorylation and ethyl­ ene signaling on ribosome loading of mRNAs, transcriptome and polysome profiling analyses were performed in gcn2 and ein2 (defective in ethylene signaling) compared with wildtype plants under submergence. The results revealed that global translational repression is less effective in gcn2 and ein2. Importantly, GCN2 and EIN2 enhance translation of dis­ tinct sets of mRNAs under submergence. The data suggest that ethylene activates 2 pathways, 1 dependent on GCN2, and the other 1 dependent on EIN2, to regulate the production of hypoxia-responsive proteins for acclimation to submergence. Furthermore, translation of several hypoxia-responsive mRNAs under submergence is affected in an eIF2α nonphosphorylatable variant in which the conserved serine residue was mutated to alanine, highlighting a role for eIF2α phosphorylation in transla­ tional control in response to submergence (Cho et al. 2022).  

# Other previously missed types of regulation tRNA-derived fragments  

In addition to carrying amino acids to translating ribosomes during translation, tRNAs have other regulatory functions. One interesting function is that tRNA-derived fragments (tRFs) can regulate translation. Most tRFs are derived from mature tRNAs and can be grouped into 2 categories based on their size and where the cleavage occurs within the tRNA: long tRFs ranging 30 to 35 nt are cleaved at the anti­ codon region, and short tRFs ranging 19 to 25 nt are cleaved at the D or T arms. Analysis of small RNA-seq data has re­ vealed that plants contain a diverse population of tRFs, and their composition fluctuates depending on developmental stages or environmental conditions (Cognat et al. 2017). Testing 10 synthesized tRFs has shown that 2 of them, derived from tRNAAla(AGC) and tRNAAsn, strongly repress the translation of a reporter mRNA in vitro. Interestingly, this translational re­ pression does not require sequence complementarity between the tRF and the translated mRNA. Furthermore, it was shown that the tRFs are associated with active polysomes but not effi­ ciently with ribosome subunits. Thus, tRFs are unlikely to inter­ fere with the binding of ribosome subunits to the mRNA (Lalande et al. 2020). The mechanism by which tRFs repress translation remains to be resolved.  

# A tRNAHis guanylyltransferase  

Another regulation related to tRNA was discovered serendip­ itously from a genetic screen searching for a thermosensitive mutant in rice, identifying the mutant adaptation to environ­ mental temperature 1 (aet1) (Chen et al. 2019). AET1 encodes a tRNA guanylyltransferase, which modifies pre-tRNAHIS and is required for tRNA maturation and homeostasis. Interestingly, yeast 2-hybrid screening for AET1-interacting partners revealed that AET1 interacts with the ribosomeassociated protein RECEPTOR FOR ACTIVATED C KINASE 1 A and eIF3h. The interaction was confirmed in planta, and it was demonstrated that all 3 proteins localize to the ER. To investigate a potential role for AET1 in mRNA trans­ lation,  RNA-immunoprecipitation  sequencing  was  per­ formed,  and  OsARF  mRNAs  were  highly  enriched.  As discussed above, Arabidopsis ARF mRNAs harbor uORFs and the re-initiation of their mORFs is regulated by eIF3h (Schepetilnikov et al. 2013). Consistent with the potential role of AET1 modulating translation re-initiation through eIF3h interaction, the fraction of OsARF19 and OsARF23 mRNAs in active polysomes is reduced in aet1.  

# eEF2  

During translation elongation, eukaryotic Elongation Factor 2 (eEF2) regulates the transfer of peptidyl tRNA from the ribo­ somal P-site to the A-site. A post-translational modification on a histidine residue of eEF2, termed diphthamide (a target of diphtheria toxin in humans), is conserved in Arabidopsis (Zhang et al. 2022). Arabidopsis mutants of Dph1, which cat­ alyzes the first step of diphthamide biosynthesis, lose the modification on the H700 residue and increase −1 frameshift errors, as shown by a reporter assay. The mutant also displays a decrease in TOR activity, cell proliferation, and plant growth and is accompanied by an increased autophagy. Interestingly, in response to copper and cadmium, wild-type plants accumulate the eEF2 form with unmodified H700 resi­ due, suggesting that an alteration of translational fidelity is involved in acclimating to heavy metal stress.  

# Stability and spatial dynamic of mRNAs  

In the cytosol, mRNAs might be subjected to degradation through various mechanisms or stored in RNA-protein con­ jugates, including P-bodies and stress granules. Both degrad­ ation and storage can prevent mRNAs from being translated.  

# Decay of mRNAs  

Most mRNA degradation is initiated by removing the $3^{\prime}$ poly-A tail, followed by either $5^{\prime}{\cdot}3^{\prime}$ or $3^{\prime}{\cdot}5^{\prime}$ decay mechanisms. The $5^{\prime}{\cdot}3^{\prime}$ decay is governed by the decapping complex and $5^{\prime}{\cdot}3^{\prime}$ exonuclease XRN4, and $3^{\prime}{\cdot}5^{\prime}$ decay is governed by SUPPRESSOR OF VARICOSE or the RNA exosomes (reviewed in Brunkard and Baker 2018; Sieburth and Vincent 2018). Eukaryotic cells also possess 3 conserved translation-dependent RNA quality control mechanisms to remove aberrant RNAs: NMD, no-go decay, and non-stop decay. NMD recognizes mRNAs containing a prema­ ture stop codon, no-go decay targets mRNAs with stalled ribo­ somes, and non-stop decay recognizes mRNAs lacking a stop codon (reviewed in Powers et al. 2020). In plants, NMD is the best-characterized  mechanism  among  the  3  (reviewed  in Ohtani and Wachter 2019), whereas no-go decay and non-stop decay have just started to be characterized (Szádeczky-Kardoss et al. 2018a, 2018b).  

It was long believed that mRNAs were either engaged in trans­ lation, or subjected to degradation or storage. However, preva­ lent co-translational decay was reported in yeast (Hu et al. 2009; Pelechano et al. 2015). Consistent with this idea, heat in­ duces the accumulation of several decapped mRNAs in light polysome fractions in the Arabidopsis xrn4 mutant, supporting the notion that some mRNAs are subjected to degradation while bound by ribosomes (Merret et al. 2015). Furthermore, genomewide analysis of RNA degradation intermediates in wild-type Arabidopsis plants detected a 3-nt periodicity within the coding regions of mRNAs, a signature of actively translating ribosomes, indicating that the transcripts are also occupied by translating ribosomes when degradation occurs (Hou et al. 2016; Yu et al. 2016). Comparative analysis of 10 angiosperm species confirmed that co-translational decay is widespread and conserved across diverse plant lineages (Guo et al. 2023a).  

# P-bodies and stress granules  

P-bodies and stress granules are cytosolic RNA-protein as­ semblies that play an important role in balancing mRNA translation, storage, and degradation. Unlike P-bodies, which are constitutively present in cells, stress granules are induced by stress conditions. These topics are reviewed regularly, for example, (Chantarachot and Bailey-Serres 2018; Kearly et al. 2022). Here, we focus on their functions in translational control.  

P-bodies were thought primarily to be involved in mRNA degradation, as they are associated with decapping enzymes and the $5^{\prime}{\cdot}3^{\prime}$ exonuclease XRN4. However, a recent study has highlighted the storage capacity of P-bodies. As mentioned above, light can enhance the translation of many transcripts (Liu et al. 2012; Chen et al. 2018). Monitoring P-body accu­ mulation in seedlings kept in the dark or transferred to light for $4\,\mathrm{h}$ revealed that light reduces P-body accumulation. Furthermore, transcriptome and translatome analysis com­ paring a mutant defective in P-body formation (dcp5-1, a weak mutant allele of DECAPPING 5) and wild-type seedlings identified over 2,000 mRNAs that remain actively translated in dcp5-1 in the dark without changes in their mRNA abun­ dance. These results suggest that in the dark, plants repress the translation of specific mRNAs involved in the translation machinery, photosynthesis, and apical hook formation by transiently storing these mRNAs in P-bodies (Jang et al. 2019). Similarly, in response to hypoxia, OLIGOURIDYLATE BINDING PROTEIN 1 (UBP1) can sequester thousands of mRNAs into stress granules, preventing them from being translated. Importantly, once the stress is removed, these mRNAs reassociate with polysomes to resume active transla­ tion (Sorenson and Bailey-Serres 2014).  

EBF2 mentioned above is also an elegant example of P-body regulation of mRNA localization. When the $3^{\prime}$ UTR of EBF2 is bound by EIN2-C (the C-terminal fragment of EIN2, following cleavage by a protease in response to ethylene signaling), the mRNA is targeted to P-bodies with the assist­ ance of NMD factors (Li et al. 2015; Merchante et al. 2015).  

A recent exciting study linked P-bodies, NMD, and transla­ tional control (Cairo et al. 2022). THREE-DIVISION MUTANT 1 (TDM1) is specifically expressed during the second meiotic division of pollen development and is required for the ter­ mination of meiosis. TDM1 was shown to be recruited into P-bodies  by  interacting  with  SUPPRESSOR  WITH MORPHOGENETIC EFFECTS ON GENITALIA 7 (SMG7), an NMD factor. A suppressor mutant screen in smg7-6 back­ ground and a yeast 2-hybrid screen for TDM1-interacting pro­ teins led to the identification of canonical eIF4G, as well as the plant-specific eIFiso4G1 and eIFiso4G2, as additional factors in­ volved in meiotic exit. Follow-up experiments revealed that TMD1 recruits eIF(iso)4Gs and 4A into P-bodies, suggesting that translation is temporarily repressed at meiotic exit by moving these translation factors into P-bodies. Consistent with this idea, treating tmd1 mutants with the translation in­ hibitor cycloheximide rescues the pollen meiotic exit defect. Thus, reduced translation of meiotic mRNAs is proposed to be critical for completing pollen meiosis.  

# Conclusions  

Technological advancements have offered unparalleled op­ portunities to discover noncanonical translation events and dissect the detailed mechanisms of translational control in various aspects of plant development and physiology. As our understanding of translational control still lags behind that of transcriptional control, ample exciting opportunities for new discoveries are within reach.  

# Acknowledgments  

We thank Isaiah Kaufman for his critical reading of and thoughtful comments on the manuscript. This work was sup­ ported by a National Science Foundation grant (2051885) to PYH. We apologize to colleagues whose work could not be cited owing to space limitations.  

# Author contributions  

H.L.W. and P.Y.H. conceived the review, wrote the manu­ script, and prepared the figures. J.J. organized the literature. All authors discussed the literature and edited the review.  

Conflict of interest statement: None declared.  

# References  

Ahn CS, Han JA, Lee HS, Lee S, Pai HS. The PP2A regulatory subunit Tap46, a component of the TOR signaling pathway, modulates growth and metabolism in plants. Plant Cell. 2011:23(1):185–209. https://doi.org/10.1105/tpc.110.074005   
Alatorre-Cobos F, Cruz-Ramírez A, Hayden CA, Pérez-Torres CA, Chauvin AL, Ibarra-Laclette E, Alva-Cortés E, Jorgensen RA, Herrera-Estrella  L.  Translational  regulation  of  Arabidopsis XIPOTL1 is modulated by phosphocholine levels via the phylogenet­ ically conserved upstream open Reading frame 30. J Exp Bot. 2012:63(14):5203–5221. https://doi.org/10.1093/jxb/ers180   
Bai B, Peviani A, van der Horst S, Gamm M, Snel B, Bentsink L, Hanson J. Extensive translational regulation during seed germination revealed by polysomal profiling. New Phytol. 2017a:214(1):233–244. https://doi.org/10.1111/nph.14355   
Bai X, Huang Y, Hu Y, Liu H, Zhang B, Smaczniak C, Hu G, Han Z, Xing Y. Duplication of an upstream silencer of FZP increases grain yield in rice. Nat Plants. 2017b:3(11):885–893. https://doi.org/10. 1038/s41477-017-0042-4   
Baird TD, Cheng KCC, Chen YC, Buehler E, Martin SE, Inglese J, Hogg JR. ICE1 promotes the link between splicing and nonsense-mediated mRNA decay. eLife. 2018:7:e33178. https://doi.org/10.7554/eLife.33178   
Barakat A, Szick-Miranda K, Chang IF, Guyot R, Blanc G, Cooke R, Delseny M, Bailey-Serres J. The organization of cytoplasmic riboso­ mal  protein  genes  in  the  Arabidopsis  genome.  Plant  Physiol. 2001:127(2):398–415. https://doi.org/10.1104/pp.010265   
Basrai MA, Hieter P, Boeke JD. Small open Reading frames: beautiful needles in the haystack. Genome Res. 1997:7(8):768–771. https:// doi.org/10.1101/gr.7.8.768   
Bazin J, Baerenfaller K, Gosai SJ, Gregory BD, Crespi M, Bailey-Serres J.  Global  analysis  of  ribosome-associated  noncoding  RNAs unveils new modes of translational regulation. Proc Natl Acad Sci USA. 2017:114:E10018–E10027. https://doi.org/10.1073/pnas.17084 33114   
Belda-Palazón B, Adamo M, Valerio C, Ferreira LJ, Confraria A, Reis-Barata  D,  Rodrigues  A,  Meyer  C,  Rodriguez  PL, Baena-González E. A dual function of SnRK2 kinases in the regula­ tion of SnRK1 and plant growth. Nat Plants. 2020:6(11):1345–1353. https://doi.org/10.1038/s41477-020-00778-w   
Belda-Palazón B, Costa M, Beeckman T, Rolland F, Baena-González E. ABA represses TOR and root meristem activity through nuclear exit of the SnRK1 kinase. Proc Natl Acad Sci USA. 2022:119(28): e2204862119. https://doi.org/10.1073/pnas.2204862119   
Bi C, Ma Y, Jiang SC, Mei C, Wang XF, Zhang DP. Arabidopsis trans­ lation initiation factors eIFiso4G1/2 link repression of mRNA capbinding  complex  eIFiso4F  assembly  with  RNA-binding  protein SOAR1-mediated  ABA  signaling.  New  Phytol.  2019:223(3): 1388–1406. https://doi.org/10.1111/nph.15880   
Billey E, Hafidh S, Cruz-Gallardo I, Litholdo Jr CG, Jean V, Carpentier MC, Picart C, Kumar V, Kulichova K, Maréchal E, et al. LARP6C orchestrates posttranscriptional reprogramming of gene expression during hydration to promote pollen tube guidance. Plant Cell. 2021:33(8):2637–2661. https://doi.org/10.1093/plcell/koab131   
Boex-Fontvieille E, Daventure M, Jossier M, Zivy M, Hodges M, Tcherkez G. Photosynthetic control of Arabidopsis leaf cytoplasmic translation  initiation  by  protein  phosphorylation.  PLoS  ONE. 2013:8(7):e70692. https://doi.org/10.1371/journal.pone.0070692   
Bonnot T, Nagel DH. Time of the day prioritizes the pool of translating mRNAs in response to heat stress. Plant Cell. 2021:33(7):2164–2182. https://doi.org/10.1093/plcell/koab113   
Branco-Price  C,  Kawaguchi  R,  Ferreira  RB,  Bailey-Serres  J. Genome-wide analysis of transcript abundance and translation in Arabidopsis seedlings subjected to oxygen deprivation. Ann Bot. 2005:96(4):647–660. https://doi.org/10.1093/aob/mci217   
Brar GA, Weissman JS. Ribosome profiling reveals the what, when, where  and  how  of  protein  synthesis.  Nat  Rev  Mol  Cell  Biol. 2015:16(11):651–664. https://doi.org/10.1038/nrm4069   
Brodersen P, Sakvarelidze-Achard L, Bruun-Rasmussen M, Dunoyer P, Yamamoto YY, Sieburth L, Voinnet O. Widespread translational inhibition by plant miRNAs and siRNAs. Science. 2008:320(5880): 1185–1190. https://doi.org/10.1126/science.1159151   
Broeckx T, Hulsmans S, Rolland F. The plant energy sensor: evolution­ ary conservation and divergence of SnRK1 structure, regulation, and function. J Exp Bot. 2016:67(22):6215–6252. https://doi.org/10.1093/ jxb/erw416   
Browning KS, Bailey-Serres J. Mechanism of cytoplasmic mRNA trans­ lation. The Arabidopsis Book. 2015:13:e0176. https://doi.org/10.1199/ tab.0176   
Brunkard JO. Exaptive evolution of target of rapamycin signaling in multicellular eukaryotes. Dev Cell. 2020:54(2):142–155. https://doi. org/10.1016/j.devcel.2020.06.022   
Brunkard JO, Baker B. A two-headed monster to avert disaster: hbs1/ski7 is  alternatively  spliced  to  build  eukaryotic  RNA  surveillance complexes. Front Plant Sci. 2018:9:1333. https://doi.org/10.3389/fpls. 2018.01333   
Bruns AN, Li S, Mohannath G, Bisaro DM. Phosphorylation of Arabidopsis eIF4E and eIFiso4E by SnRK1 inhibits translation. FEBS J. 2019:286(19):3778–3796. https://doi.org/10.1111/febs.14935   
Burkart GM, Brandizzi F. A tour of TOR complex signaling in plants. Trends Biochem Sci. 2021:46(5):417–428. https://doi.org/10.1016/j. tibs.2020.11.004   
Busche M, Scarpin MR, Hnasko R, Brunkard JO. TOR coordinates nu­ cleotide availability with ribosome biogenesis in plants. Plant Cell. 2021:33(5):1615–1632. https://doi.org/10.1093/plcell/koab043   
Cai $\pmb{\mathrm{Q}},$ Fukushima H, Yamamoto M, Ishii N, Sakamoto T, Kurata T, Motose H, Takahashi T. The SAC51 family plays a central role in thermospermine  responses  in  Arabidopsis.  Plant  Cell  Physiol. 2016:57(8):1583–1592. https://doi.org/10.1093/pcp/pcw113   
Cairo A, Vargova A, Shukla N, Capitao C, Mikulkova P, Valuchova S, Pecinkova J, Bulankova P, Riha K. Meiotic exit in Arabidopsis is dri­ ven  by  P-body-mediated  inhibition  of  translation.  Science. 2022:377(6606):629–634. https://doi.org/10.1126/science.abo0904   
Cao P, Kim SJ, Xing A, Schenck CA, Liu L, Jiang N, Wang J, Last RL, Brandizzi F. Homeostasis of branched-chain amino acids is critical for the activity of  TOR signaling in  Arabidopsis. eLife.  2019:8: e50747. https://doi.org/10.7554/eLife.50747   
Castellano MM, Merchante C. Peculiarities of the regulation of trans­ lation initiation in plants. Curr Opin Plant Biol. 2021:63:102073. https://doi.org/10.1016/j.pbi.2021.102073   
Chang H, Lim J, Ha M, Kim VN. TAIL-seq: genome-wide determination of poly(A) tail length and $3^{\prime}$ end modifications. Mol Cell. 2014:53(6): 1044–1052. https://doi.org/10.1016/j.molcel.2014.02.007   
Chantarachot T, Bailey-Serres J. Polysomes, stress granules, and pro­ cessing  bodies:  a  dynamic  triumvirate  controlling  cytoplasmic mRNA  fate  and  function.  Plant  Physiol.  2018:176(1):254–269. https://doi.org/10.1104/pp.17.01468   
Chen GH, Liu MJ, Xiong Y, Sheen J, Wu SH. TOR And RPS6 transmit light  signals  to  enhance  protein  translation  in  deetiolating Arabidopsis seedlings. Proc Natl Acad Sci USA. 2018:115(50):12823–­ 12828. https://doi.org/10.1073/pnas.1809526115   
Chen K, Guo T, Li XM, Zhang YM, Yang YB, Ye WW, Dong NQ, Shi CL, Kan Y, Xiang YH, et al. Translational regulation of plant re­ sponse to high temperature by a dual-function tRNAHis guanylyl­ transferase in rice. Mol Plant. 2019:12(8):1123–1142. https://doi. org/10.1016/j.molp.2019.04.012   
Chen $\pmb{\mathrm{Q}},$ Tian F, Cheng T, Jiang J, Zhu G, Gao Z, Lin H, Hu J, Qian $\pmb{\mathrm{Q}},$ Fang X, et al. Translational repression of FZP mediated by CU-rich elem­ ent/OsPTB interactions modulates panicle development in rice. Plant J. 2022:110(5):1319–1331. https://doi.org/10.1111/tpj.15737   
Chen X, Dickman D. Development of a tissue-specific ribosome profil­ ing approach in Drosophila enables genome-wide evaluation of translational  adaptations.  PLoS  Genet.  2017:13(12):e1007117. https://doi.org/10.1371/journal.pgen.1007117   
Cheng M-C, Kathare PK, Paik I, Huq E. Phytochrome signaling net­ works. Annu Rev Plant Biol. 2021:72(1):217–244. https://doi.org/10. 1146/annurev-arplant-080620-024221   
Chew GL, Pauli A, Schier AF. Conservation of uORF repressiveness and sequence features in mouse, human and zebrafish. Nat Commun. 2016:7(1):11663. https://doi.org/10.1038/ncomms11663   
Cho H, Cho HS, Nam H, Jo H, Yoon J, Park C, Dang TV, Kim E, Jeong J, Park S, et al. Translational control of phloem development by RNA G-quadruplex-JULGI determines plant sink strength. Nat Plants. 2018:4(6):376–390. https://doi.org/10.1038/s41477-018-0157-2   
Cho HY, Chou MY, Ho HY, Chen WC, Shih MC. Ethylene modulates translation dynamics in Arabidopsis under submergence via GCN2 and EIN2. Sci Adv. 2022:8(22):7863. https://doi.org/10.1126/sciadv. abm7863   
Cho HY, Lu MYJ, Shih MC. The SnRK1-eIFiso4G1 signaling relay regu­ lates the translation of specific mRNAs in Arabidopsis under submer­ gence. New Phytol. 2019:222(1):366–381. https://doi.org/10.1111/ nph.15589   
Cho HY, Wen TN, Wang YT, Shih MC. Quantitative phosphoproteo­ mics of protein kinase SnRK1 regulated protein phosphorylation in Arabidopsis under submergence. J Exp Bot. 2016:67(9):2745–2760. https://doi.org/10.1093/jxb/erw107   
Chotewutmontri P, Barkan A. Dynamics of chloroplast translation during chloroplast differentiation in maize. PLoS Genet. 2016:12(7): e1006106. https://doi.org/10.1371/journal.pgen.1006106   
Chung BYW, Balcerowicz M, Di Antonio M, Jaeger KE, Geng F, Franaszek K, Marriott P, Brierley I, Firth AE, Wigge PA. An RNA thermoswitch regulates daytime growth in Arabidopsis. Nat Plants. 2020:6(5):522–532. https://doi.org/10.1038/s41477-020-0633-3   
Coate JE, Bar H, Doyle JJ. Extensive translational regulation of gene ex­ pression  in  an  allopolyploid  (Glycine  dolichocarpa).  Plant  Cell. 2014:26(1):136–150. https://doi.org/10.1105/tpc.113.119966   
Cognat V, Morelle G, Megel C, Lalande S, Molinier J, Vincent T,   
Small I, Duchene AM, Marechal-Drouard L. The nuclear and orga­ nellar tRNA-derived RNA fragment population in Arabidopsis thali­ ana is highly dynamic. Nucleic Acids Res. 2017:45(6):3460–3472. https://doi.org/10.1093/nar/gkw1122   
Cui Y, Rao S, Chang B, Wang X, Zhang K, Hou X, Zhu X, Wu H, Tian Z, Zhao Z, et al. Atla1 protein initiates IRES-dependent translation of WUSCHEL mRNA and regulates the stem cell homeostasis of Arabidopsis  in  response  to  environmental  hazards.  Plant  Cell Environ. 2015:38(10):2098–2114. https://doi.org/10.1111/pce.12535   
Dasgupta A, Camacho RAU, Enganti R, Cho SK, Tucker LL, Torreverde JS, Abraham PE, von Arnim AG. A phosphorylation-deficient riboso­ mal protein eS6 is largely functional in Arabidopsis thaliana, rescuing   
mutant defects from global translation and gene expression to photo­ synthesis and growth. bioRxiv: 2023.05.30.542942; 2023.   
Deforges J, Reis RS, Jacquet P, Sheppard S, Gadekar VP, Hart-Smith G, Tanzer A, Hofacker IL, Iseli C, Xenarios I, et al. Control of cog­ nate sense mRNA translation by cis-natural antisense RNAs. Plant Physiol. 2019:180(1):305–322. https://doi.org/10.1104/pp.19.00043   
Deprost D, Yao L, Sormani R, Moreau M, Leterreux G, Nicolaï M, Bedu M, Robaglia C, Meyer C. The Arabidopsis TOR kinase links plant growth, yield, stress resistance and mRNA translation. EMBO Rep. 2007:8(9):864–870. https://doi.org/10.1038/sj.embor.7401043   
Dong J, Chen H, Deng XW, Irish VF, Wei N. Phytochrome B induces in­ tron retention and translational inhibition of phytochrome-interacting factor3. Plant Physiol. 2020:182(1):159–166. https://doi.org/10.1104/pp. 19.00835   
Doroudgar S, Hofmann C, Boileau E, Malone B, Riechert E, Gorska AA, Jakobi T, Sandmann C, Jürgensen L, Kmietczyk V, et al. Monitoring cell-type–specific gene expression using ribosome profil­ ing in vivo during cardiac hemodynamic stress. Circ Res. 2019:125(4): 431–448. https://doi.org/10.1161/CIRCRESAHA.119.314817   
Douka K, Agapiou M, Birds I, Aspden JL. Optimization of ribosome footprinting conditions for Ribo-Seq in human and Drosophila mel­ anogaster  tissue  culture  cells.  Front  Mol  Biosci.  2022:8:791455. https://doi.org/10.3389/fmolb.2021.791455   
Enganti R, Cho SK, Toperzer JD, Urquidi-Camacho RA, Cakir OS, Ray AP, Abraham PE, Hettich RL, von Arnim AG. Phosphorylation of ribo­ somal protein RPS6 integrates light signals and circadian clock signals.   
Front Plant Sci. 2018:8:2210. https://doi.org/10.3389/fpls.2017.02210   
Forzani  C,  Duarte  GT,  Van  Leene  J,  Clément  G,  Huguet  S, Paysant-Le-Roux C, Mercier R, De Jaeger G, Leprince AS, Meyer C. Mutations of the AtYAK1 kinase suppress TOR deficiency in Arabidopsis. Cell Rep. 2019:27(12):3696–3708.e5. https://doi.org/10. 1016/j.celrep.2019.05.074   
Freire MA. Translation initiation factor (iso) 4E interacts with BTF3, the $\upbeta$ subunit of the nascent polypeptide-associated complex. Gene. 2005:345(2):271–277. https://doi.org/10.1016/j.gene.2004.11.030   
Freire MA, Tourneur C, Granier F, Camonis J, El Amrani A, Browning KS, Robaglia C. Plant lipoxygenase 2 is a translation initi­ ation factor-4E-binding protein. Plant Mol Biol. 2000:44(2):129–140. https://doi.org/10.1023/A:1006494628892   
Fu L, Liu Y, Qin G, Wu P, Zi H, Xu Z, Zhao X, Wang Y, Li Y, Yang S, et al. The TOR–EIN2 axis mediates nuclear signaling to modulate plant growth. Nature. 2021:591(7849):288–292. https://doi.org/10. 1038/s41586-021-03310-y   
Fujishiro Y, Agata A, Ota S, Ishihara R, Takeda Y, Kunishima T, Ikeda M, Kyozuka J, Hobo T, Kitano H. Comprehensive panicle phenotyping reveals that qSrn7/FZP influences higher-order branching. Sci Rep. 2018:8(1):12511. https://doi.org/10.1038/s41598-018-30395-9   
Gage JL, Mali S, McLoughlin F, Khaipho-Burch M, Monier B, Bailey-Serres J, Vierstra RD, Buckler ES. Variation in upstream open reading frames contributes to allelic diversity in maize protein abundance. Proc Natl Acad Sci USA. 2022:119(14):e2112516119. https://doi.org/10.1073/pnas.2112516119   
Gallie DR. Plant growth and fertility requires functional interactions between specific PABP and eIF4G gene family members. PLoS ONE. 2018:13(1):e0191474. https://doi.org/10.1371/journal.pone.0191474   
Gallie DR, Browning KS. eIF4G functionally differs from eIFiso4G in promoting  internal  initiation,  cap-independent  translation,  and translation  of  structured  mRNAs.  J  Biol  Chem.  2001:276(40): 36951–36960. https://doi.org/10.1074/jbc.M103869200   
Gao X, Wan J, Liu B, Ma M, Shen B, Qian SB. Quantitative profiling of initiating ribosomes in vivo. Nat Methods. 2015:12(2):147–153. https://doi.org/10.1038/nmeth.3208   
Gonzalez C, Sims JS, Hornstein N, Mela A, Garcia F, Lei L, Gass DA, Amendolara B, Bruce JN, Canoll P, et al. Ribosome profiling reveals a  cell-type-specific  translational  landscape  in  brain  tumors.  J Neurosci.  2014:34(33):10924–10936.  https://doi.org/10.1523/JNEU ROSCI.0084-14.2014   
Grüner S, Weber R, Peter D, Chung MY, Igreja C, Valkov E, Izaurralde E. Structural motifs in eIF4G and 4E-BPs modulate their binding to eIF4E to regulate translation initiation in yeast. Nucleic Acids Res. 2018:46(13): 6893–6908. https://doi.org/10.1093/nar/gky542   
Guerrero-González ML, Rodríguez-Kessler M, Jiménez-Bremont JF. uORF, a regulatory mechanism of the Arabidopsis polyamine oxidase 2.  Mol  Biol  Rep.  2014:41(4):2427–2443.  https://doi.org/10.1007/ s11033-014-3098-5   
Guo R, Yu X, Gregory BD. The identification of conserved sequence features of co-translationally decayed mRNAs and upstream open reading  frames  in  angiosperm  transcriptomes.  Plant  Direct. 2023a:7(1):e479. https://doi.org/10.1002/pld3.479   
Guo Y, Chen Y, Wang Y, Wu X, Zhang X, Mao W, Yu H, Guo K, Xu J, Ma L, et al. The translational landscape of bread wheat during grain development. Plant Cell. 2023b:35(6):1848–1867. https://doi.org/10. 1093/plcell/koad075   
Guo Z, Cao H, Zhao J, Bai S, Peng W, Li J, Sun L, Chen L, Lin Z, Shi C, et al. A natural uORF variant confers phosphorus acquisition diver­ sity  in  soybean.  Nat  Commun.  2022:13:1–14.  https://doi.org/ 10.1038/s41467-022-31555-2   
Gutierrez-Beltran E, Elander PH, Dalman K, Dayhoff GW, Moschou PN, Uversky VN, Crespo JL, Bozhkov PV. Tudor staphylococcal nu­ clease is a docking platform for stress granule components and is es­ sential for SnRK1 activation in Arabidopsis. EMBO J. 2021:40(17): e105043. https://doi.org/10.15252/embj.2020105043   
Hanfrey C, Elliott KA, Franceschetti M, Mayer MJ, Illingworth C, Michael AJ. A dual upstream open reading frame-based autoregulatory circuit  controlling  polyamine-responsive  translation.  J  Biol  Chem. 2005:280(47):39229–39237. https://doi.org/10.1074/jbc.M509340200   
Hayashi N, Sasaki S, Takahashi H, Yamashita Y, Naito S, Onouchi H. Identification of Arabidopsis thaliana upstream open reading frames encoding peptide sequences that cause ribosomal arrest. Nucleic Acids  Res.  2017:45(15):8844–8858.  https://doi.org/10.1093/nar/ gkx528   
Hayden CA, Jorgensen RA. Identification of novel conserved peptide uORF homology groups in Arabidopsis and rice reveals ancient eu­ karyotic origin of select groups and preferential association with transcription  factor-encoding  genes.  BMC  Biol.  2007:5(1):32. https://doi.org/10.1186/1741-7007-5-32   
Henninger M, Pedrotti L, Krischke M, Draken J, Wildenhain T, Fekete A, Rolland F, Müller MJ, Fröschel C, Weiste C, et al. The evolutionarily conserved kinase SnRK1 orchestrates resource mobil­ ization  during  Arabidopsis  seedling  establishment.  Plant  Cell. 2022:34(1):616–632. https://doi.org/10.1093/plcell/koab270   
Hou CY, Lee WC, Chou HC, Chen AP, Chou SJ, Chen HM. Global ana­ lysis of truncated RNA ends reveals new insights into Ribosome Stalling in plants. Plant Cell. 2016:28(10):2398–2416. https://doi. org/10.1105/tpc.16.00295   
Hsu PY, Benfey PN. Small but mighty: functional peptides encoded by small ORFs in plants. Proteomics. 2018:18(10):1700038. https://doi. org/10.1002/pmic.201700038   
Hsu PY, Calviello L, Wu HYL, Li FW, Rothfels CJ, Ohler U, Benfey PN. Super-resolution ribosome profiling reveals unannotated translation events  in  Arabidopsis.  Proc  Natl  Acad  Sci  USA.  2016:113: E7126–E7135. https://doi.org/10.1073/pnas.1614788113   
Hu W, Sweet TJ, Chamnongpol S, Baker KE, Coller J. Co-translational mRNA decay in Saccharomyces cerevisiae. Nature. 2009:461(7261): 225–229. https://doi.org/10.1038/nature08265   
Hurt JA, Robertson AD, Burge CB. Global analyses of UPF1 binding and function reveal expanded scope of nonsense-mediated mRNA decay.  Genome  Res.  2013:23(10):1636–1650.  https://doi.org/10. 1101/gr.157354.113   
Imai A, Hanzawa Y, Komura M, Yamamoto KT, Komeda Y, Takahashi T. The dwarf phenotype of the Arabidopsis ac15 mutant is suppressed by a mutation in an upstream ORF of a bHLH gene. Development.  2006:133(18):3575–3585.  https://doi.org/10.1242/ dev.02535   
Inada T, Winstall E, Tarun SZ, Yates JR, Schieltz D, Sachs AB. One-step affinity purification of the yeast ribosome and its asso­ ciated proteins and mRNAs. RNA. 2002:8(7):948–958. https://doi. org/10.1017/S1355838202026018   
Ingolia  NT,  Ghaemmaghami  S,  Newman  JRS,  Weissman  JS. Genome-wide analysis in vivo of translation with nucleotide reso­ lution using ribosome profiling. Science. 2009:324(5924):218–223. https://doi.org/10.1126/science.1168978   
Iwakawa H, Lam AYW, Mine A, Fujita T, Kiyokawa K, Yoshikawa M, Takeda A, Iwasaki S, Tomari Y. Ribosome stalling caused by the Argonaute-microRNA-SGS3 complex regulates the production of secondary siRNAs in plants. Cell Rep. 2021:35(13):109300. https:// doi.org/10.1016/j.celrep.2021.109300   
Izquierdo Y, Kulasekaran S, Benito P, López B, Marcos R, Cascón T, Hamberg M, Castresana C. Arabidopsis nonresponding to oxylipins locus NOXY7 encodes a yeast GCN1 homolog that mediates nonca­ nonical translation regulation and stress adaptation. Plant Cell Environ. 2018:41(6):1438–1452. https://doi.org/10.1111/pce.13182   
Jang GJ, Yang JY, Hsieh HL, Wu SH. Processing bodies control the se­ lective translation for optimal development of Arabidopsis young seedlings.  Proc  Natl  Acad  Sci  USA.  2019:116(13):6451–6456. https://doi.org/10.1073/pnas.1900084116   
Jiang M, Ning W, Wu S, Wang X, Zhu K, Li A, Li Y, Cheng S, Song B. Three-nucleotide periodicity of nucleotide diversity in a population en­ ables the identification of open reading frames. Brief Bioinformatics. 2022:23(4):1–13. https://doi.org/10.1093/bib/bbac210   
Johnstone TG, Bazzini AA, Giraldez AJ. Upstream ORFs are prevalent translational repressors in vertebrates. EMBO J. 2016:35(7):706–723. https://doi.org/10.15252/embj.201592759   
Jorgensen RA, Dorantes-Acosta AE. Conserved peptide upstream open reading frames are associated with regulatory genes in angios­ perms. Front Plant Sci. 2012:3:191. https://doi.org/10.3389/fpls.2012. 00191   
Ju C, Yoon GM, Shemansky JM, Lin DY, Ying ZI, Chang J, Garrett WM,  Kessenbrock  M,  Groth  G,  Tucker  ML,  et  al.  CTR1 Phosphorylates the central regulator EIN2 to control ethylene hor­ mone  signaling  from  the  ER  membrane  to  the  nucleus  in Arabidopsis. Proc Natl Acad Sci USA. 2012:109(47):19486–19491. https://doi.org/10.1073/pnas.1214848109   
Juntawong P, Girke T, Bazin J, Bailey-Serres J. Translational dynamics revealed  by  genome-wide  profiling  of  ribosome  footprints  in Arabidopsis.  Proc  Natl  Acad  Sci  USA.  2014:111(1):E203–E212. https://doi.org/10.1073/pnas.1317811111   
Kajala K, Gouran M, Shaar-Moshe L, Mason GA, Rodriguez-Medina J, Kawa D, Pauluzzi G, Reynoso M, Canto-Pastor A, Manzano C, et al. Innovation, conservation, and repurposing of gene function in root cell type development. Cell. 2021:184(12):3333–3348.e19. https://doi.org/10.1016/j.cell.2021.04.024   
Kearly A, Nelson ADL, Skirycz A, Chodasiewicz M. Composition and function of stress granules and P-bodies in plants. Semin Cell Dev Biol. 2022:S1084-9521(22)00350-0. Online ahead of print. https:// doi.org/10.1016/j.semcdb.2022.11.008   
Kim BH, Cai X, Vaughn JN, Von Arnim AG. On the functions of the h subunit of eukaryotic initiation factor 3 in late stages of translation initiation. Genome Biol. 2007:8(4):R60. https://doi.org/10.1186/gb2007-8-4-r60   
Kozak M. Possible role of flanking nucleotides in recognition of the aug initiator  codon  by  eukaryotic  ribosomes.  Nucleic  Acids  Res. 1981:9(20):5233–5252. https://doi.org/10.1093/nar/9.20.5233   
Kurihara Y, Makita Y, Kawashima M, Fujita T, Iwasaki S, Matsui M. Transcripts from downstream alternative transcription start sites evade uORF-mediated inhibition of gene expression in Arabidopsis. Proc Natl Acad Sci USA. 2018:115(30):7831–7836. https://doi.org/ 10.1073/pnas.1804971115   
Kurihara Y, Makita Y, Shimohira H, Fujita T, Iwasaki S, Matsui M. Translational landscape of protein-coding and non-protein-coding RNAs  upon  light  exposure  in  Arabidopsis.  Plant  Cell  Physiol. 2020:61(3):536–545. https://doi.org/10.1093/pcp/pcz219   
Kurosaki T, Popp MW, Maquat LE. Quality and quantity control of gene expression by nonsense-mediated mRNA decay. Nat Rev Mol Cell Biol. 2019:20(7):406–420. https://doi.org/10.1038/s41580-019-0126-2   
Lageix S, Lanet E, Pouch-Pélissier MN, Espagnol MC, Robaglia C, Deragon JM, Pélissier T. Arabidopsis eIF2α kinase GCN2 is essential for growth in stress conditions and is activated by wounding. BMC Plant Biol. 2008:8(1):134. https://doi.org/10.1186/1471-2229-8-134   
Laing WA, Martínez-Sánchez M, Wright MA, Bulley SM, Brewster D, Dare AP, Rassam M, Wang D, Storey R, Macknight RC, et al. An upstream open reading frame is essential for feedback regulation of ascorbate biosynthesis in Arabidopsis. Plant Cell. 2015:27(3): 772–786. https://doi.org/10.1105/tpc.114.133777   
Lalande S, Merret R, Salinas-Giegé T, Drouard L. Arabidopsis tRNA-derived fragments as potential modulators of translation. RNA Biol. 2020:17(8): 1137–1148. https://doi.org/10.1080/15476286.2020.1722514   
Lauressergues D, Couzigou JM, San Clemente H, Martinez Y, Dunand  C,  Bécard  G,  Combier  JP.  Primary  transcripts  of microRNAs encode regulatory peptides. Nature. 2015:520(7545): 90–93. https://doi.org/10.1038/nature14346   
Lee DH, Park SJ, Ahn CS, Pai HS. MRF Family genes are involved in translation control, especially under energy-deficient conditions, and their expression and functions are modulated by the TOR signal­ ing pathway. Plant Cell. 2017:29(11):2895–2920. https://doi.org/10. 1105/tpc.17.00563   
Lei L, Shi J, Chen J, Zhang M, Sun S, Xie S, Li X, Zeng B, Peng L, Hauck A, et al. Ribosome profiling reveals dynamic translational landscape in  maize  seedlings  under  drought  stress.  Plant  J.  2015:84(6): 1206–1208. https://doi.org/10.1111/tpj.13073   
Lellis AD, Patrick RM, Mayberry LK, Lorence A, Campbell ZC, Roose JL, Frankel LK, Bricker TM, Hellmann HA, Mayberry RW, et al. EIFiso4g augments the synthesis of specific plant proteins involved in normal chloroplast function. Plant Physiol. 2019:181(1):85–96. https://doi.org/10.1104/pp.19.00557   
Li B, Wang Y, Zhang Y, Tian W, Chong K, Jang JC, Wang L. PRR5, 7 and 9 positively modulate TOR signaling-mediated root cell proliferation by repressing TANDEM ZINC FINGER 1 in Arabidopsis. Nucleic Acids Res. 2019:47(10):5001–5015. https://doi.org/10.1093/nar/gkz191   
Li S, Le B, Ma X, Li S, You C, Yu Y, Zhang B, Liu L, Gao L, Shi T, et al. Biogenesis of phased siRNAs on membrane-bound polysomes in Arabidopsis. eLife. 2016:5:e22750. https://doi.org/10.7554/eLife.22750   
Li S, Liu L, Zhuang X, Yu Y, Liu X, Cui X, Ji L, Pan Z, Cao X, Mo B, et al. MicroRNAs inhibit the translation of target mRNAs on the endoplas­ mic reticulum in Arabidopsis. Cell. 2013:153(3):562–574. https://doi. org/10.1016/j.cell.2013.04.005   
Li W, Ma M, Feng Y, Li H, Wang Y, Ma Y, Li M, An F, Guo H. EIN2-directed  translational  regulation  of  ethylene  signaling  in Arabidopsis.  Cell.  2015:163(3):670–683.  https://doi.org/10.1016/j. cell.2015.09.037   
Li X, Cai W, Liu Y, Li H, Fu L, Liu Z, Xu L, Liu H, Xu T, Xiong Y. Differential TOR activation and cell proliferation in Arabidopsis root and shoot apexes. Proc Natl Acad Sci USA. 2017:114(10): 2765–2770. https://doi.org/10.1073/pnas.1618782114   
Li YR, Liu MJ. Prevalence of alternative AUG and non-AUG translation initiators and their regulatory effects across plants. Genome Res. 2020:30(10):1418–1433. https://doi.org/10.1101/gr.261834.120   
Liang Y, Zhu W, Chen S, Qian J, Li L. Genome-wide identification and characterization of small peptides in maize. Front Plant Sci. 2021:12: 695439. https://doi.org/10.3389/fpls.2021.695439   
Liu B, Qian SB. Translational reprogramming in cellular stress response. Wiley Interdiscipl Rev RNA. 2014:5(3):301–305. https://doi.org/10. 1002/wrna.1212   
Liu M-J, Wu S-H, Chen H-M, Wu S-H. Widespread translational control contributes to the regulation of Arabidopsis photomorphogenesis. Mol Syst Biol. 2012:8(1):566. https://doi.org/10.1038/msb.2011.97   
Liu MJ, Wu SH, Wu JF, Lin WD, Wu YC, Tsai TY, Tsai HL, Wu SH. Translational landscape of photomorphogenic Arabidopsis. Plant Cell. 2013:25(10):3699–3710. https://doi.org/10.1105/tpc.113.114769   
Liu X, Afrin T, Pajerowska-Mukhtar KM. Arabidopsis GCN2 kinase contributes to ABA homeostasis and stomatal immunity. Commun Biol. 2019:2(1):302. https://doi.org/10.1038/s42003-019-0544-x   
Liu Y, Duan X, Zhao X, Ding W, Wang Y, Xiong Y. Diverse nitrogen signals activate convergent ROP2-TOR signaling in Arabidopsis. Dev Cell. 2021:56(9):1283–1295.e5. https://doi.org/10.1016/j.devcel. 2021.03.022   
Loh PG, Yang HS, Walsh MA, Wang Q, Wang X, Cheng Z, Liu D, Song H. Structural basis for translational inhibition by the tumour suppres­ sor Pdcd4. EMBO J. 2009:28(3):274–285. https://doi.org/10.1038/ emboj.2008.278   
Lokdarshi A, von Arnim AG. Review: emerging roles of the signaling network of the protein kinase GCN2 in the plant stress response. Plant Sci. 2022:320:111280. https://doi.org/10.1016/j.plantsci.2022. 111280   
Lokdarshi A, Guan J, Urquidi Camacho RA, Cho SK, Morgan PW, Leonard M, Shimono M, Day B, von Arnim AG. Light activates the translational regulatory kinase GCN2 via reactive oxygen species emanating from the chloroplast. Plant Cell. 2020a:32(4):1161–1178. https://doi.org/10.1105/tpc.19.00751   
Lokdarshi A, Morgan PW, Franks M, Emert Z, Emanuel C, von Arnim AG. Light-dependent activation of the GCN2 kinase under cold and salt stress is mediated by the photosynthetic status of the chloro­ plast.  Front  Plant  Sci.  2020b:11:431.  https://doi.org/10.3389/fpls. 2020.00431   
Lorenzo-Orts L, Witthoeft J, Deforges J, Martinez J, Loubéry S, Placzek  A,  Poirier  Y,  Hothorn  LA,  Jaillais  Y,  Hothorn  M. Concerted expression of a cell cycle regulator and a metabolic en­ zyme from a bicistronic transcript in plants. Nat Plants. 2019:5(2): 184–193. https://doi.org/10.1038/s41477-019-0358-3   
Lütcke HA, Chow KC, Mickel FS, Moss KA, Kern HF, Scheele GA. Selection of AUG initiation codons differs in plants and animals. EMBO J. 1987:6(1):43–48. https://doi.org/10.1002/j.1460-2075.1987. tb04716.x   
Mahfouz MM, Kim S, Delauney AJ, Verma DPS. Arabidopsis TARGET of RAPAMYCIN interacts with RAPTOR, which regulates the activity of  S6  kinase  in  response  to  osmotic  stress  signals.  Plant  Cell. 2006:18(2):477–490. https://doi.org/10.1105/tpc.105.035931   
Mazzoni-Putman SM, Stepanova AN. A plant biologist’s Toolbox to study translation. Front Plant Sci. 2018:9:873. https://doi.org/10. 3389/fpls.2018.00873   
McCready K, Spencer V, Kim M. The importance of TOR kinase in plant development. Front Plant Sci. 2020:11:16. https://doi.org/10. 3389/fpls.2020.00016   
Mei C, Jiang SC, Lu YF, Wu FQ, Yu YT, Liang S, Feng XJ, Comeras SP, Lu K, Wu Z, et al. Arabidopsis pentatricopeptide repeat protein SOAR1 plays a critical role in abscisic acid signalling. J Exp Bot. 2014:65(18):5317–5330. https://doi.org/10.1093/jxb/eru293   
Menand B, Desnos T, Nussaume L, Berger F, Bouchez D, Meyer C, Robaglia C. Expression and disruption of the Arabidopsis TOR (tar­ get of rapamycin) gene. Proc Natl Acad Sci USA. 2002:99(9): 6422–6427. https://doi.org/10.1073/pnas.092141899   
Merchante C, Brumos J, Yun J, Hu Q, Spencer KR, Enríquez P, Binder BM, Heber S, Stepanova AN, Alonso JM. Gene-specific translation regulation mediated by the hormone-signaling molecule EIN2. Cell. 2015:163(3):684–697. https://doi.org/10.1016/j.cell.2015.09.036   
Merret  R,  Nagarajan  VK,  Carpentier  MC,  Park  S,  Favory  JJ, Descombin J, Picart C, Charng YY, Green PJ, Deragon JM, et al. Heat-induced ribosome pausing triggers mRNA co-translational de­ cay  in  Arabidopsis  thaliana.  Nucleic  Acids  Res.  2015:43(8): 4121–4132. https://doi.org/10.1093/nar/gkv234   
Meteignier LV, El-Oirdi M, Cohen M, Barff T, Matteau D, Lucier JF, Rodrigue S, Jacques PE, Yoshioka K, Moffett P. Translatome ana­ lysis of an NB-LRR immune response identifies important contribu­ tors to plant immunity in Arabidopsis. J Exp Bot. 2017:68(9): 2333–2344. https://doi.org/10.1093/jxb/erx078   
Meyuhas O. Ribosomal protein S6 phosphorylation: four decades of re­ search. Int Rev Cell Mol Biol. 2015:320:41–73. https://doi.org/10. 1016/bs.ircmb.2015.07.006   
Miras  M,  Truniger  V,  Silva  C,  Verdaguer  N,  Aranda  MA, Querol-Audí J. Structure of eiF4E in complex with an eiF4G peptide supports a universal bipartite binding mode for protein translation. Plant  Physiol.  2017:174(3):1476–1491.  https://doi.org/10.1104/pp. 17.00193   
Missra A, Ernest B, Lohoff T, Jia $\pmb{\mathrm{Q}},$ Satterlee J, Ke K, von Arnim AG. The circadian clock modulates global daily cycles of mRNA ribosome loading. Plant Cell. 2015:27(9):2582–2599. https://doi.org/10.1105/ tpc.15.00546   
Montes C, Wang P, Liao CY, Nolan TM, Song G, Clark NM, Elmore JM, Guo H, Bassham DC, Yin Y, et al. Integration of multi-omics data  reveals  interplay  between  brassinosteroid  and  Target  of Rapamycin  Complex  signaling  in  Arabidopsis.  New  Phytol. 2022:236(3):893–910. https://doi.org/10.1111/nph.18404   
Moreau M, Azzopardi M, Clé ment G, Dobrenel T, Marchive C, Renne C, Martin-Magniette ML, Taconnat L, Renou JP, Robaglia C, et al. Mutations in the Arabidopsis homolog of LST8/GβL, a part­ ner of the target of Rapamycin kinase, impair plant growth, flowering, and metabolic adaptation to long days. Plant Cell. 2012:24(2): 463–481. https://doi.org/10.1105/tpc.111.091306   
Mustroph A, Zanetti ME, Jang CJH, Holtan HE, Repetti PP, Galbraith DW, Girke T, Bailey-Serres J. Profiling translatomes of discrete cell populations resolves altered cellular priorities during hypoxia in Arabidopsis. Proc Natl Acad Sci USA. 2009:106(44):18843–18848. https://doi.org/10.1073/pnas.0906131106   
Nakagawa S, Niimura Y, Gojobori T, Tanaka H, Miura K. Diversity of preferred nucleotide sequences around the translation initiation co­ don in eukaryote genomes. Nucleic Acids Res. 2008:36(3):861–871. https://doi.org/10.1093/nar/gkm1102   
Nyikó T, Auber A, Szabadkai L, Benkovics A, Auth M, Mérai Z, Kerényi Z, Dinnyés A, Nagy F, Silhavy D. Expression of the eRF1 translation ter­ mination factor is controlled by an autoregulatory circuit involving read­ through and nonsense-mediated decay in plants. Nucleic Acids Res. 2017:45:4174–4188. https://doi.org/10.1093/nar/gkw1303   
Ohtani M, Wachter A. NMD-based gene regulation—a strategy for fit­ ness  enhancement  in  plants?  Plant  Cell  Physiol.  2019:60(9): 1953–1960. https://doi.org/10.1093/pcp/pcz090   
Pacheco JM, Canal MV, Pereyra CM, Welchen E, Martínez-Noël GMA,  Estevez  JM.  The  tip  of the  iceberg:  emerging  roles  of TORC1, and its regulatory functions in plant cells. J Exp Bot. 2021:72(11):4085–4101. https://doi.org/10.1093/jxb/eraa603   
Paik I, Yang S, Choi G. Phytochrome regulates translation of mRNA in the cytosol. Proc Natl Acad Sci USA. 2012:109(4):1335–1340. https:// doi.org/10.1073/pnas.1109683109   
Pajerowska-Mukhtar KM, Wang W, Tada Y, Oka N, Tucker CL, Fonseca JP, Dong X. The HSF-like transcription factor TBF1 is a major molecular switch for plant growth-to-defense transition. Curr Biol. 2012:22(2):103–112. https://doi.org/10.1016/j.cub.2011.12.015   
Palam LR, Baird TD, Wek RC. Phosphorylation of eIF2 facilitates ribo­ somal bypass of an inhibitory upstream ORF to enhance CHOP trans­ lation. J Biol Chem. 2011:286(13):10939–10949. https://doi.org/10. 1074/jbc.M110.216093   
Palos K, Nelson Dittrich AC, Yu L, Brock JR, Railey CE, Wu H-YL,   
Sokolowska  E,  Skirycz  A,  Hsu  PY,  Gregory  BD,  et  al. Identification and functional annotation of long intergenic noncoding RNAs in Brassicaceae. Plant Cell. 2022:34(9):3233–3260. https://doi.org/10.1093/plcell/koac166   
Parker MT, Knop K, Sherwood AV, Schurch NJ, Mackinnon K, Gould PD, Hall AJW, Barton GJ, Simpson GG. Nanopore direct RNA   
sequencing maps the complexity of arabidopsis mRNA processing and  m6A  modification.  eLife.  2020:9:e49658.  https://doi.org/10. 7554/eLife.49658   
Passmore LA, Coller J. Roles of mRNA poly(A) tails in regulation of eu­ karyotic gene expression. Nat Rev Mol Cell Biol. 2022:23(2):93–106. https://doi.org/10.1038/s41580-021-00417-y   
Patrick RM, Browning KS. The eIF4F and eIFiso4F complexes of plants: an evolutionary perspective. Comp Funct Genomics. 2012:2012: 287814. https://doi.org/10.1155/2012/287814   
Patrick RM, Lee JCH, Teetsel JRJ, Yang SH, Choy GS, Browning KS. Discovery and characterization of conserved binding of eIF4E 1 (CBE1), a eukaryotic translation initiation factor 4E-binding plant protein. J Biol Chem. 2018:293(44):17240–17247. https://doi.org/10. 1074/jbc.RA118.003945   
Peixoto B, Baena-González E. Management of plant central metabol­ ism by SnRK1 protein kinases. J Exp Bot. 2022:73(20):7068–7082. https://doi.org/10.1093/jxb/erac261   
Pelechano V, Wei W, Steinmetz LM. Widespread co-translational RNA decay  reveals  ribosome  dynamics.  Cell.  2015:161(6):1400–1412. https://doi.org/10.1016/j.cell.2015.05.008   
Pfeiffer A, Janocha D, Dong Y, Medzihradszky A, Schöne S, Daum G, Suzaki T, Forner J, Langenecker T, Rempel E, et al. Integration of light and metabolic signals for stem cell activation at the shoot apical meristem. eLife. 2016:5:e17023. https://doi.org/10.7554/eLife.17023   
Powers KT, Szeto JYA, Schaffitzel C. New insights into no-go, non-stop and nonsense-mediated mRNA decay complexes. Curr Opin Struct Biol. 2020:65:110–118. https://doi.org/10.1016/j.sbi.2020.06.011   
Pu Y, Luo X, Bassham DC. Tor-dependent and -independent pathways regulate autophagy in Arabidopsis thaliana. Front Plant Sci. 2017:8: 1204. https://doi.org/10.3389/fpls.2017.01204   
Puyaubert J, Denis L, Alban C. Dual targeting of Arabidopsis holocar­ boxylase synthetase1: a small upstream open reading frame regulates translation  initiation  and  protein  targeting.  Plant  Physiol.   
2008:146(2):323–324. https://doi.org/10.1104/pp.107.111534   
Qiao H, Shen Z, Huang SSC, Schmitz RJ, Urich MA, Briggs SP, Ecker   
JR. Processing and subcellular trafficking of ER-tethered EIN2 control response to ethylene gas. Science. 2012:338(6105):390–393. https:// doi.org/10.1126/science.1225974   
Raabe K, Honys D, Michailidis C. The role of eukaryotic initiation fac­ tor 3 in plant translation regulation. Plant Physiol Biochem. 2019:145: 75–83. https://doi.org/10.1016/j.plaphy.2019.10.015   
Rahmani F, Hummel M, Schuurmans J, Wiese-Klinkenberg A, Smeekens S, Hanson J. Sucrose control of translation mediated by an upstream open reading frame-encoded peptide. Plant Physiol.   
2009:150(3):1356–1367. https://doi.org/10.1104/pp.109.136036   
Raj A, van Oudenaarden A. Nature, nurture, or chance: stochastic gene expression and its consequences. Cell. 2008:135(2):216–226. https://doi.org/10.1016/j.cell.2008.09.050   
Reis RS, Deforges J, Schmidt RR, Schippers JHM, Poirier Y. An anti­ sense noncoding RNA enhances translation via localized structural   
rearrangements  of  its  cognate  mRNA.  Plant  Cell.  2021:33(4): 1381–1397. https://doi.org/10.1093/plcell/koab010   
Reynoso MA, Borowsky AT, Pauluzzi GC, Yeung E, Zhang J, Formentin E, Velasco J, Cabanlit S, Duvenjian C, Prior MJ, et al. Gene  regulatory  networks  shape  developmental  plasticity  of root cell types under water extremes in rice. Dev Cell. 2022:57(9): 1177–1192.e6. https://doi.org/10.1016/j.devcel.2022.04.013   
Reynoso MA, Kajala K, Bajic M, West DA, Pauluzzi G, Yao AI, Hatch K, Zumstein K, Woodhouse M, Rodriguez-Medina J, et al. Evolutionary flexibility  in  flooding  response  circuitry in  angiosperms.  Science.   
2019:365(6459):1291–1295. https://doi.org/10.1126/science.aax8862   
Ribone PA, Capella M, Arce AL, Chan RL. A uORF represses the tran­ scription factor AtHB1 in aerial tissues to avoid a deleterious pheno­ type. Plant Physiol. 2017:175(3):1238–1253. https://doi.org/10.1104/   
pp.17.01060  
Rook F, Gerrits N, Kortstee A, Van Kampen M, Borrias M, Weisbeek P, Smeekens S. Sucrose-specific signaling represses translation of the Arabidopsis ATB2 bZIP transcription factor gene. Plant J. 1998:15(2): 253–263. https://doi.org/10.1046/j.1365-313X.1998.00205.x   
Sagor GHM, Berberich T, Tanaka S, Nishiyama M, Kanayama Y,   
Kojima S, Muramoto K, Kusano T. A novel strategy to produce sweeter tomato fruits with high sugar contents by fruit-specific ex­ pression of a single bZIP transcription factor gene. Plant Biotechnol J. 2016:14(4):1116–1126. https://doi.org/10.1111/pbi.12480   
Sahoo S, Singh D, Singh A, Pandit M, Vasu K, Som S, Pullagurla NJ, Laha   
D, Eswarappa SM. Identification and functional characterization of mRNAs that exhibit stop codon readthrough in Arabidopsis thaliana. J Biol Chem. 2022:298(8):102173. https://doi.org/10.1016/j.jbc.2022.102173   
Scarpin MR, Busche M, Martinez RE, Harper LC, Reiser L, Szakonyi D, Merchante C, Lan T, Xiong W, Mo B, et al. An updated nomen­ clature for plant ribosomal protein genes. Plant Cell. 2023:35(2): 640–643. https://doi.org/10.1093/plcell/koac333   
Scarpin MR, Leiboff S, Brunkard JO. Parallel global profiling of plant tor dynamics reveals a conserved role for LARP1 in translation. eLife. 2020:9:1–34. https://doi.org/10.7554/eLife.58795   
Scarpin MR, Simmons CH, Brunkard JO. Translating across kingdoms: target of rapamycin promotes protein synthesis through conserved and  divergent  pathways  in  plants.  J  Exp  Bot.  2022:73(20): 7016–7025. https://doi.org/10.1093/jxb/erac267   
Schepetilnikov M, Dimitrova M, Mancera-Martínez E, Geldreich A, Keller M, Ryabova LA. TOR And S6K1 promote translation reinitia­ tion of uORF-containing mRNAs via phosphorylation of eIF3h. EMBO J. 2013:32(8):1087–1102. https://doi.org/10.1038/emboj.2013.61   
Schepetilnikov M, Makarian J, Srour O, Geldreich A, Yang Z, Chicher J, Hammann P, Ryabova LA. GTP Ase ROP 2 binds and promotes activa­ tion of target of rapamycin, TOR, in response to auxin. EMBO J. 2017:36(7):886–903. https://doi.org/10.15252/embj.201694816   
Schepetilnikov M, Ryabova LA. Recent discoveries on the role of tor (Target  of  rapamycin)  signaling  in  translation  in  plants.  Plant Physiol. 2018:176(2):1095–1105. https://doi.org/10.1104/pp.17.01243   
Schuller AP, Green R. Roadblocks and resolutions in eukaryotic trans­ lation. Nat Rev Mol Cell Biol. 2018:19(8):526–541. https://doi.org/10. 1038/s41580-018-0011-4  
Shah P, Ding Y, Niemczyk M, Kudla G, Plotkin JB. Rate-limiting steps in yeast protein translation. Cell. 2013:153(7):1589–1601. https://doi. org/10.1016/j.cell.2013.05.049   
Shamimuzzaman M, Vodkin L. Ribosome profiling reveals changes in translational status of soybean transcripts during immature cotyle­ don development. PLoS ONE. 2018:13(3):e0194596. https://doi.org/ 10.1371/journal.pone.0194596   
Sharma A, Badola PK, Bhatia C, Sharma D, Trivedi PK. Primary tran­ script of miR858 encodes regulatory peptide and controls flavonoid biosynthesis and development in Arabidopsis. Nat Plants. 2020:6(10): 1262–1274. https://doi.org/10.1038/s41477-020-00769-x   
Shi L, Wu Y, Sheen J. TOR Signaling in plants: conservation and innov­ ation.  Development.  2018:145(13):dev160887.  https://doi.org/10. 1242/dev.160887   
Sieburth LE, Vincent JN. Beyond transcription factors: roles of mrna decay in regulating gene expression in plants [version 1; referees: 3 approved]. F1000Res. 2018:7:F1000 Faculty Rev- 1940. https://doi. org/10.12688/f1000research.16203.1   
Skuzeski JM, Nichols LM, Gesteland RF, Atkins JF. The signal for a lea­ ky UAG stop codon in several plant viruses includes the two down­ stream codons. J Mol Biol. 1991:218(2):365–373. https://doi.org/10. 1016/0022-2836(91)90718-L  
Sorenson  R,  Bailey-Serres  J.  Selective  mRNA  sequestration  by OLIGOURIDYLATEBINDING PROTEIN 1 contributes to translational control during hypoxia in Arabidopsis. Proc Natl Acad Sci USA. 2014:111(6):2373–2378. https://doi.org/10.1073/pnas.1314851111   
Sotta N, Chiba Y, Aoyama H, Takamatsu S, Suzuki T, Miwa K, Yamashita Y, Naito S, Fujiwara T. Translational landscape of a C4Plant, Sorghum bicolor, under normal and sulfur-deficient condi­ tions.  Plant  Cell  Physiol.  2022:63(5):592–604.  https://doi.org/10. 1093/pcp/pcac023   
Sotta N, Chiba Y, Miwa K, Takamatsu S, Tanaka M, Yamashita Y, Naito S, Fujiwara T. Global analysis of boron-induced ribosome stal­ ling reveals its effects on translation termination and unique regula­ tion by AUG-stops in Arabidopsis shoots. Plant J. 2021:106(5): 1455–1467. https://doi.org/10.1111/tpj.15248   
Szádeczky-Kardoss I, Csorba T, Auber A, Schamberger A, Nyikó T, Taller J, Orbán TI, Burgyán J, Silhavy D. The nonstop decay and the RNA silencing systems operate cooperatively in plants. Nucleic Acids  Res.  2018a:46(9):4632–4648.  https://doi.org/10.1093/nar/ gky279   
Szádeczky-Kardoss I, Gál L, Auber A, Taller J, Silhavy D. The No-go decay system degrades plant mRNAs that contain a long A-stretch in the coding region. Plant Sci. 2018b:275:19–27. https://doi.org/10. 1016/j.plantsci.2018.07.008   
Takahashi H, Hayashi N, Hiragori  Y, Sasaki S, Motomura T, Yamashita Y, Naito S, Takahashi A, Fuse K, Satou $\mathbf{K},$ et  al. Comprehensive genome-wide identification of angiosperm upstream ORFs with peptide sequences conserved in various taxonomic ranges using a novel pipeline, ESUCA. BMC Genomics. 2020:21(1):260. https://doi.org/10.1186/s12864-020-6662-5   
Takahashi H, Lassmann T, Murata M, Carninci P. $5^{\prime}$ end-centered ex­ pression profiling using cap-analysis gene expression and nextgeneration sequencing. Nat Protoc. 2012a:7(3):542–561. https://doi. org/10.1038/nprot.2012.005   
Takahashi H, Takahashi A, Naito S, Onouchi H. BAIUCAS: a novel BLAST-based algorithm for the identification of upstream open read­ ing frames with conserved amino acid sequences and its application to the Arabidopsis thaliana genome. Bioinformatics. 2012b:28(17): 2231–2241. https://doi.org/10.1093/bioinformatics/bts303   
Tanaka M, Sotta N, Yamazumi Y, Yamashita Y, Miwa K, Murota K, Chiba Y, Hirai MY, Akiyama T, Onouchi H, et al. The minimum open reading frame, AUG-stop, induces boron-dependent ribosome stalling and mRNA degradation. Plant Cell. 2016:28(11):2830–2849. https://doi.org/10.1105/tpc.16.00481   
Toribio R, Muñoz A, Castro-Sanz AB, Merchante C, Castellano MM. A novel eIF4E-interacting protein that forms non-canonical transla­ tion  initiation  complexes.  Nat  Plants.  2019:5(12):1283–1296. https://doi.org/10.1038/s41477-019-0553-2   
Traubenik S, Reynoso MA, Hobecker K, Lancia M, Hummel M, Rosen B, Town C, Bailey-Serres J, Blanco F, Zanetti ME. Reprogramming of root cells during nitrogen-fixing symbiosis involves dynamic poly­ some  association  of  coding  and  noncoding  RNAs.  Plant  Cell. 2020:32(2):352–373. https://doi.org/10.1105/tpc.19.00647   
Turkina MV, Årstrand H, Vener AV. Differential phosphorylation of ribosomal proteins in Arabidopsis thaliana plants during day and night.  PLoS  ONE.  2011:6(12):e29307.  https://doi.org/10.1371/ journal.pone.0029307   
Uchiyama-Kadokura N, Murakami K, Takemoto M, Koyanagi N, Murota K, Naito S, Onouchi H. Polyamine-responsive ribosomal ar­ rest at the stop codon of an upstream open reading frame of the AdoMetDC1 gene triggers nonsense-mediated mRNA decay in Arabidopsis  thaliana.  Plant  Cell  Physiol.  2014:55(9):1556–1567. https://doi.org/10.1093/pcp/pcu086   
Van Der Horst S, Filipovska T, Hanson J, Smeekens S. Metabolite con­ trol of translation by conserved peptide uORFs: the ribosome as a metabolite  multisensor.  Plant  Physiol.  2020:182(1):110–122. https://doi.org/10.1104/pp.19.00940   
Van Der Horst S, Snel B, Hanson J, Smeekens S. Novel pipeline iden­ tifies new upstream ORFs and non-AUG initiating main ORFs with conserved amino acid sequences in the $5^{\prime}$ leader of mRNAs in Arabidopsis thaliana. RNA. 2019:25(3):292–304. https://doi.org/10. 1261/rna.067983.118   
Van Leene J, Han C, Gadeyne A, Eeckhout D, Matthijs C, Cannoot B, De Winne N, Persiau G, Van De Slijke E, Van de Cotte B, et al. Capturing the phosphorylation and protein interaction landscape of the plant TOR kinase. Nat Plants. 2019:5(3):316–327. https://doi. org/10.1038/s41477-019-0378-z   
Vaughn JN, Ellingson SR, Mignone F, Von Arnim A. Known and novel post-transcriptional regulatory sequences are conserved across plant fam­ ilies. RNA. 2012:18(3):368–384. https://doi.org/10.1261/rna.031179.111   
Von Arnim AG, Jia $\pmb{\mathrm{Q}},$ Vaughn JN. Regulation of plant translation by upstream open reading frames. Plant Sci. 2014:214:1–12. https:// doi.org/10.1016/j.plantsci.2013.09.006   
Wang J, Zhang X, Greene GH, Xu G, Dong X. PABP/purine-rich motif as an initiation module for cap-independent translation in pattern  
triggered immunity. Cell. 2022:185(17):3186–3200.e17. https://doi. org/10.1016/j.cell.2022.06.037   
Wang L, Li H, Zhao C, Li S, Kong L, Wu W, Kong W, Liu Y, Wei Y, Zhu JK, et al. The inhibition of protein translation mediated by AtGCN1 is essential for cold tolerance in Arabidopsis thaliana. Plant Cell Environ. 2017:40(1):56–68. https://doi.org/10.1111/pce.12826   
Wang P, Zhao Y, Li Z, Hsu C-C, Liu X, Fu L, Hou YJ, Du Y, Xie S, Zhang   
C, et al. Reciprocal regulation of the TOR kinase and ABA receptor balances plant growth and stress response. Mol Cell. 2018:69(1): 100–112.e6. https://doi.org/10.1016/j.molcel.2017.12.002   
Wiese A, Elzinga N, Wobbes B, Smeekens S. A conserved upstream open reading frame mediates sucrose-induced repression of translation. Plant Cell. 2004:16(7):1717–1729. https://doi.org/10.1105/tpc.019349   
Willems P, Ndah E, Jonckheere V, Stael S, Sticker A, Martens L, Van   
Breusegem F, Gevaert K, Van Damme P. N-terminal proteomics as­ sisted profiling of the unexplored translation initiation landscape in Arabidopsis thaliana. Mol Cell Proteomics. 2017:16(6):1064–1080. https://doi.org/10.1074/mcp.M116.066662   
Wu H, Li B, Iwakawa H, Pan Y, Tang X, Ling-hu $\pmb{\mathrm{Q}},$ Liu Y, Sheng S,   
Feng L, Zhang H, et al. Plant 22-nt siRNAs mediate translational re­ pression  and  stress  adaptation.  Nature.  2020:581(7806):89–93. https://doi.org/10.1038/s41586-020-2231-y   
Wu HW, Fajiculay E, Wu JF, Yan CCS, Hsu CP, Wu SH. Noise reduc­ tion  by  upstream open  reading  frames. Nat Plants.  2022:8(5): 474–480. https://doi.org/10.1038/s41477-022-01136-8   
Wu H-YL, Hsu PY. Translation of upstream open reading frames gen­ erally does not trigger nonsense-mediated decay in Arabidopsis. bioRxiv: 2021.09.16.460672; 2022.  
Wu HYL, Song G, Walley JW, Hsu PY. The tomato translational landscape revealed by transcriptome assembly and ribosome profiling. Plant Physiol. 2019:181(1):367–380. https://doi.org/10.1104/pp.19.00541   
Wu Z, Huang S, Zhang X, Wu D, Xia S, Li X. Regulation of plant im­ mune receptor accumulation through translational repression by a glycine–tyrosine–phenylalanine  (GYF)  domain  protein.  eLife. 2017:6:e23684. https://doi.org/10.7554/eLife.23684   
Xing S, Chen K, Zhu H, Zhang R, Zhang H, Li B, Gao C. Fine-tuning sugar content in strawberry. Genome Biol. 2020:21(1):230. https:// doi.org/10.1186/s13059-020-02146-5   
Xiong Y, McCormack M, Li L, Hall $\pmb{\mathrm{Q}},$ Xiang C, Sheen J. Glucose-TOR sig­ nalling reprograms the transcriptome and activates meristems. Nature. 2013:496(7444):181–186. https://doi.org/10.1038/nature12030   
Xiong Y, Sheen J. Rapamycin and glucose-target of rapamycin (TOR) protein signaling in plants. J Biol Chem. 2012:287(4):2836–2842. https://doi.org/10.1074/jbc.M111.300749   
Xu G, Greene GH, Yoo H, Liu L, Marqués J, Motley J, Dong X. Global translational reprogramming is a fundamental layer of immune regu­ lation in plants. Nature. 2017a:545(7655):487–490. https://doi.org/ 10.1038/nature22371   
Xu G, Yuan M, Ai C, Liu L, Zhuang E, Karapetyan S, Wang S, Dong X. UORF-mediated translation allows engineered plant disease resist­ ance  without  fitness  costs.  Nature.  2017b:545(7655):491–494. https://doi.org/10.1038/nature22372   
Yamashita Y, Takamatsu $\pmb{5},$ Glasbrenner M, Becker T, Naito S, Beckmann R. Sucrose sensing through nascent peptide-meditated ribo­ some stalling at the stop codon of Arabidopsis bZIP11 uORF2. FEBS Lett. 2017:591(9):1266–1277. https://doi.org/10.1002/1873-3468.12634   
Yang X, Dong W, Ren W, Zhao Q, Wu F, He Y. Cytoplasmic HYL1 modulates miRNA-mediated translational repression. Plant Cell. 2021:33(6):1980–1996. https://doi.org/10.1093/plcell/koab090   
Yerlikaya S, Meusburger M, Kumari R, Huber A, Anrather D, Costanzo M, Boone C, Ammerer G, Baranov PV, Loewith R. TORC1 And TORC2 work together to regulate ribosomal protein S6  phosphorylation in  Saccharomyces cerevisiae.  Mol  Biol Cell. 2016:27(2):397–409. https://doi.org/10.1091/mbc.e15-08-0594   
Yoshikawa M, Iki T, Numa H, Miyashita K, Meshi T, Ishikawa M. A short open reading frame encompassing the microRNA173 target site plays a role in trans-acting small interfering RNA biogenesis. Plant Physiol. 2016:171(1):359–368. https://doi.org/10.1104/pp.16.00148   
Young SK, Wek RC. Upstream open reading frames differentially regu­ late genespecific translation in the integrated stress response. J Biol Chem. 2016:291(33):16927–16935. https://doi.org/10.1074/jbc.R116. 733899   
Yu X, Willmann MR, Anderson SJ, Gregory BD. Genome-wide map­ ping of uncapped and cleaved transcripts reveals a role for the nu­ clear mRNA cap-binding complex in cotranslational RNA decay in Arabidopsis. Plant Cell. 2016:28(10):2385–2397. https://doi.org/10. 1105/tpc.16.00456   
Zhang H, Quintana J, Ütkür K, Adrian L, Hawer H, Mayer K, Gong X, Castanedo L, Schulten A, Janina N, et al. Translational fidelity and growth of Arabidopsis require stress-sensitive diphthamide biosynthesis. Nat Commun. 2022:13(1):4009. https://doi.org/10.1038/s41467-022-31712-7   
Zhang H, Wang Y, Wu X, Tang X, Wu C, Lu J. Determinants of genomewide distribution and evolution of uORFs in eukaryotes. Nat Commun. 2021:12(1):1076. https://doi.org/10.1038/s41467-021-21394-y   
Zhang L, Liu X, Gaikwad K, Kou X, Wang F, Tian X, Xin M, Ni Z, Sun $\pmb{\mathrm{Q}},$ Peng H, et al. Mutations in eIF5b confer thermosensitive and pleiotropic phenotypes via translation defects in Arabidopsis thali­ ana. Plant Cell. 2017:29(8):1952–1969. https://doi.org/10.1105/tpc. 16.00808   
Zhang Y, Wang Y, Kanyuka K, Parry MAJ, Powers SJ, Halford NG. GCN2-dependent Phosphorylation of eukaryotic translation initi­ ation factor-2α in Arabidopsis. J Exp Bot. 2008:59(11):3131–3141. https://doi.org/10.1093/jxb/ern169   
Zhang Z, Zhu JY, Roh J, Marchive C, Kim SK, Meyer C, Sun Y, Wang W, Wang ZY. TOR signaling promotes accumulation of BZR1 to bal­ ance growth with carbon availability in Arabidopsis. Curr Biol. 2016:26(14):1854–1860. https://doi.org/10.1016/j.cub.2016.05.005   
Zhao J, Qin B, Nikolay R, Spahn CMT, Zhang G. Translatomics: the global view of translation. Int J Mol Sci. 2019a:20(1):212. https:// doi.org/10.3390/ijms20010212   
Zhao T, Huan Q, Sun J, Liu C, Hou X, Yu X, Silverman IM, Zhang Y, Gregory BD, Liu CM, et al. Impact of poly(A)-tail G-content on Arabidopsis PAB binding and their role in enhancing translational ef­ ficiency. Genome Biol. 2019b:20(1):1–12. https://doi.org/10.1186/ s13059-019-1799-8   
Zhigailov AV, Alexandrova AM, Nizkorodova AS, Stanbekova GE, Kryldakov  RV,  Karpova  OV,  Polimbetova  NS,  Halford  NG, Iskakov BK. Evidence that phosphorylation of the $\mathfrak{a}$ -subunit of eIF2 does not essentially inhibit mRNA translation in wheat germ cell-free system. Front Plant Sci. 2020:11:936. https://doi.org/10. 3389/fpls.2020.00936   
Zhu S, Estévez JM, Liao H, Zhu Y, Yang T, Li C, Wang Y, Li L, Liu X, Pacheco JM, et al. The RALF1–FERONIA complex phosphorylates eIF4E1 to promote protein synthesis and polar root hair growth. Mol Plant. 2020:13(5):698–716. https://doi.org/10.1016/j.molp.2019.12.014   
Zhu W, Xu J, Chen S, Chen J, Liang Y, Zhang C, Li $\pmb{\mathrm{Q}},$ Lai J, Li L. Large-scale translatome profiling annotates the functional genome and reveals the key role of genic $3^{\prime}$ untranslated regions in transla­ tomic  variation  in  plants.  Plant  Commun.  2021:2(4):100181. https://doi.org/10.1016/j.xplc.2021.100181   
Zhu X, Li Y, Fang W, Kusano T. Galactinol is involved in sequenceconserved upstream open reading frame-mediated repression of Arabidopsis HsfB1 translation. Environ Exp Bot. 2018:156:120–129. https://doi.org/10.1016/j.envexpbot.2018.09.006   
Zoschke R, Watkins KP, Barkan A. A rapid ribosome profiling method elucidates  chloroplast  ribosome  behavior  in  vivo.  Plant  Cell. 2013:25(6):2265–2275. https://doi.org/10.1105/tpc.113.111567  