# Development of nanoRibo-seq enables study of regulated translation by cortical neuron subtypes, showing uORF translation in synaptic-axonal genes  

# Graphical abstract  

# Authors  

![](images/d7923396412ae649f4052f9f3cd21e3fbd4cbb7c116eba3bd6908b9fd6092a81.jpg)  

John E. Froberg, Omer Durak, Jeffrey D. Macklis  

Correspondence jeffreymacklis@fas.harvard.edu  

# In brief  

Froberg et al. develop ‘‘nanoRibo-seq,’’ enabling translational profiling with very small cell numbers. Comparison of translation efficiencies (T.E.s) between interhemispheric callosal and subcerebral cortical projection neuron subtypes shows highly correlated T.E.s, with several dozen differences. nanoRibo-seq shows extensive translation of upstream open reading frames in key synaptic and axonal genes.  

# Highlights  

nanoRibo-seq globally quantifies translation efficiency (T.E.) using as few as 1,000 cells  

nanoRibo-seq enables translatome comparison between distinct neuronal subtypes  

Though T.E. is highly correlated between CPNs and SCPNs, there are ${\sim}40$ differential T.E.s  

uORF translation is especially prominent in mRNAs for synapse and axon development genes  

# Development of nanoRibo-seq enables study of regulated translation by cortical neuron subtypes, showing uORF translation in synaptic-axonal genes  

John E. Froberg,1 Omer Durak,1 and Jeffrey D. Macklis1,2,\*  

1Department of Stem Cell and Regenerative Biology, and Center for Brain Science, Harvard University, Cambridge, MA 02138, USA   
2Lead contact   
\*Correspondence: jeffreymacklis@fas.harvard.edu   
https://doi.org/10.1016/j.celrep.2023.112995  

# SUMMARY  

Investigation of translation in rare cell types or subcellular contexts is challenging due to large input requirements for standard approaches. Here, we present ‘‘nanoRibo-seq’’ an optimized approach using $10^{2}-$ to $10^{3}.$ -fold less input material than bulk approaches. nanoRibo-seq exhibits rigorous quality control features consistent with quantification of ribosome protected fragments with as few as 1,000 cells. We compare translatomes of two closely related cortical neuron subtypes, callosal projection neurons (CPN) and subcerebral projection neurons (SCPN), during their early postnatal development. We find that, while translational effi- ciency is highly correlated between CPN and SCPN, several dozen mRNAs are differentially translated. We further examine upstream open reading frame (uORF) translation and identify that mRNAs involved in synapse organization and axon development are highly enriched for uORF translation in both subtypes. nanoRibo-seq enables investigation of translational regulation of rare cell types in vivo and offers a flexible approach for globally quantifying translation from limited input material.  

# INTRODUCTION  

Regulation of mRNA translation is crucial for establishing and maintaining appropriate protein abundances and sites of production and thus for coordinating complex cellular functions. Dysregulated translation is an important mechanism underlying many human diseases, including ribosomopathies caused by mutations in the core ribosomal machinery1; cancers driven by abnormalities in both signaling pathways regulating global levels of translation, as well as translation initiation and elongation factors themselves2; and neurodevelopmental disorders such as fragile X syndrome caused by mutations in RNA-binding proteins that regulate protein synthesis. Despite the central importance of translational control in development and disease, it has been investigated far less extensively than transcriptional control in the same biological systems due to both the relative simplicity of global transcriptional analyses, in particular RNA sequencing (RNA-seq), compared with more complicated biochemistry and the dramatically larger input material requirements needed to determine translational output. New approaches are needed to enable analyses of translational output across distinct cell types, developmental stages, and genetic manipulations in specific cell populations isolated from appropriate in vivo contexts.  

One such system with immense diversity of cell types, developmental dynamics of function, and critical relevance of genetic manipulations for cell-type-specific disease is cerebral cortex neurogenesis and circuit formation-maintenance, in which translational regulation likely plays critical roles, but with immense gaps in knowledge due to the limited amounts of material available from specific neuronal populations. The mammalian cerebral cortex is organized into six layers, each layer with substantial neuronal diversity. The cortex differentiates from ventricular zone progenitors in an ‘‘inside-out’’ fashion, with deep-layer neurons born and differentiating earlier than superficial-layer neurons and thus with complex subtype-specific developmental dynamics. Cortical ‘‘projection’’ neurons are glutamatergic, excitatory neurons that extend exceptionally long axonal projections from their somata to distant targets $10^{3}{-}105$ cell body diameters away, with diverse subtypes defined by soma location, projection target(s), and gene expression.  

We focus here on two exemplar cortical projection neuron subtypes: callosal projection neurons (CPN) and subcerebral projection neurons (SCPN). CPN project through the corpus callosum (largest axonal tract in the central nervous system) across the midline to homotypic targets in the contralateral hemisphere and provide association and integration of functional areas in the two cortical hemispheres, underlying many cognitive, behavioral, and associative sensorimotor functions.6,7Approximately $80\%$ of CPN are ‘‘late-born’’ superficial-layer (II/III) neurons, with distinct populations of CPN in deeper layers. Beyond their subtype specificity of axonal targeting and function and their highly orchestrated developmental dynamics, numerous human genetic variants and mutations disrupt CPN circuitry in humans and mice, causing distinct, subtype-specific behavioral dysfunction. SCPN are a developmentally closely related type of neuron, with somata mostly located in layer V, and project from the cortex toward nuclei in the brainstem or, for a further subset, to the spinal cord.4,8SCPN are the subtype of cortical projection neurons that specifically degenerate in amyotrophic lateral sclerosis $(\mathsf{A L S})^{9-11}$ and the related frontotemporal dementia (FTD).12–14  

We and others have previously performed extensive transcriptional characterization of CPN,15SCPN,8and other types of cortical projection neurons16,17 across their pre- and postnatal development timelines using non-genetic labeling strategies (in utero electroporation for CPN and retrograde labeling for SCPN) combined with neuronal fluorescence-activated cell sorting (FACS). Though the transcriptomes of these neurons are closely related, a relatively small set of transcription factors are required to specify large elements of specific subtype identity and axonal projection and to repress other subtype identities.  

Posttranscriptional control has the potential to restrict protein expression to proper developmental times, cell/neuron subtypes, and subcellular locations, highlighting the likely complex interplay between transcriptional and translational regulation. For example, cortical progenitors and newborn neurons during early differentiation often transiently co-express mRNAs coding for subtype-identity-specifying transcription factors (TFs) of multiple subtypes19–24 but do not produce these TF proteins.19–22 In addition, shifts in expression of RNA-binding proteins including Pum1/2,20,22 4E-T,20,21 HuR,25 HuD,26 Ire1a,27 Elav4, and Celf128 and of ribosomal proteins Rpl7 and $\mathsf{R p l10^{29}}$ control translation of select mRNAs in progenitors, while neurogenesis shifts from production of deep-layer neurons including SCPN to superficial-layer neurons such as CPN.20–22,25–29 The reported global decrease in ribosome biogenesis and abundance from early to late cortical neurogenesis might also favor translation of select sets of genes at progressive developmental stages.22,29,30 These studies, mostly from bulk neocortex without isolation of particular progenitor classes or distinct neuron subtypes, also suggest an important role for translational regulation in finetuning output of developmental programs that define subtype identity. Importantly, much less is known about translational regulation by distinct cortical subtypes, especially at later developmental stages, as their long-range projections and precise circuitry develop. This is largely due to the extremely limited material that can be obtained from purified subtype-specific neurons.  

To enable cell-type-specific and potentially subcellular analyses of translational regulation across a wide variety of biological systems, we present here an optimized low-input Ribo-seq approach appropriately scaled to facilitate Ribo-seq with nanogram-scale RNA inputs (nanoRibo-seq) (Figure 1) with stringent quality controls (Figure S1). Our approach offers several advantages compared with previous31,32 approaches using microgram-scale inputs, as well as compared with standard Riboseq: we optimize (1) RNase I digestion and (2) library preparation approaches with nanogram-scale input quantities, and (3) the library preparation approach incorporates unique molecular identifiers (UMIs) to enable removal of PCR duplicates for accurate quantification. Importantly, (4) we investigate several strategies for measuring mRNA abundance using randomly fragmented libraries that can be used with the identical library preparation method as for the ribosome-protected fragment (RPF) libraries to measure and directly compare RPF abundance and mRNA abundance, thus eliminating the confounding factor of different library preparation methods employed previously to compare Ribo-seq with RNA-seq. Our approach complements recent similar advances in applying Ribo-seq to small numbers of cells33–35 and also early work measuring translational output in single cells.36,37  

We report high-quality nanoRibo-seq libraries using as few as 1,000 HEK293 cells or several thousand mouse CPN FACS-purified from single cortical hemispheres. We apply nanoRibo-seq to study translational differences between CPN and SCPN and identify that, while translation efficiency (T.E.) for most genes is highly correlated between subtypes, several dozen genes exhibit differential translation, suggesting multilevel regulation of gene output between subtypes. Our data also identify that select sets of short upstream open reading frames (uORFs) are actively translated by both CPN and SCPN. uORFs are initiated within mRNA $5^{\prime}$ UTRs and are often strongly inhibitory for translation of the main coding sequence (CDS), though they are typically poorly translated under constitutive conditions.38 Intriguingly, transcripts coding for genes involved in synapse organization and axon development are enriched among the mRNAs containing translated uORFs in both subtypes. We present experiments and analyses demonstrating that nanoRibo-seq enables study of translational regulation using just a few thousand cells, extending translational investigations to a wide range of cellular, tissue, and developmentally dynamic systems.  

# RESULTS  

# nanoRibo-seq yields rigorous results with as little as approximately 30 ng RNA input  

To investigate the feasibility of nanoRibo-seq in a relatively simple system with low input, we harvested neonatal mouse brains, lysed them in $1\times$ polysome buffer (with cycloheximide to inhibit translational elongation), extracted RNA, measured its concentration, and calculated the approximate volume of brain lysate equivalent to approximately 30 ng measured RNA (likely an underestimate of in vivo RNA due to RNA extraction inefficiencies). We diluted this volume in polysome buffer (to approximate volumes obtained after FACS, with additional dilution to minimize effects of sheath fluid and neuronal disassociation buffer) and digested with RNase I. We selected the concentration of RNase I ‘‘[RNase I]’’ that produced an apparent RPF of approximately 30 nt (Figure 2A). In parallel, we performed ‘‘bulk’’ digestion of undiluted brain lysate using RNase I digestion conditions similar to the original mammalian Ribo-seq method.39 We also digested purified brain lysate total RNA with RNase I for comparison to Riboseq libraries (Figure S2).  

To convert these short RNA fragments into sequencing libraries, we employed the QIA-Seq miRNA library kit, designed for generation of small RNA libraries with as little as 1 ng RNA. The QIA-Seq miRNA library kit introduces a 12-nt UMI  

# Cell Reports Resource  

![](images/47208a24449e30eed291765f86accde699f3733ef2e5f741f4ee4aa5125670c2.jpg)  
A Iterative optimization ofnanoRibo-seq   
nanoRibo-seq fromFAcS-purified cortical neurons   
Figure 1. Schematic of nanoRibo-seq approach to investigate translational regulation by defined neuronal subtypes or subpopulations  

# nanoRibo-seq librarypreparation  

(A) Iterative optimization of nanoRibo-seq. We tested RNase I digestion conditions using ${\sim}30$ ng forebrain lysate, isolated and sequenced RPFs, and tested for the presence of well-known ribosome profiling QC metrics: (1) enrichment over CDSs, (2) distinct length distributions over CDSs, and (3) enrichment in-frame P-sites. We iteratively repeated these nano-scale RNase I digestions until the QC metrics were optimized.  

(C) Workflow for conversion of nanoscale quantities of small RNA into RNA-seq libraries using the QiAseq miRNA library kit, an RNA ligation-based method.  

(B) Generalized workflow for nanoRibo-seq from labeled, FACS-purified cortical neurons.  

![](images/4b606bac1b39f7b8c54561334726e8ac4516bf40819baff555915c4536174eef.jpg)  

(D) Library preparation results and read structure: resulting libraries have a sharp size distribution centered at $\mathord{\sim}180$ bp, consistent with a roughly 30 bp insert between the two sequencing adapters. This results in a read structure in which read 1 sequences the ${\sim}30^{.}$ -nt RPF and read 2 sequences the 12-nt UMI barcode, enabling identification and removal of PCR duplicates before quality control and/or quantitative analyses.   
See also Figure S1  

protection), but RNase I-digested total RNA does not show any specific preferred length (consistent with random digestion) (Figure 2D). The Ribo-seq libraries also show narrower length distributions over CDSs compared to $3^{\prime}$ UTRs and small nucleolar RNAs (snoRNAs) (Figure 2E). This is important since structured RNAs or tightly bound RNAbinding proteins might also protect RNA from digestion but are unlikely to protect RNA in exactly the same way as the ribosome, leading to distinct length distributions between CDSs and ncRNA.40  

![](images/a87c68aaa8b940502f5ccc03a8882398381ee0f75d8f9f6712a02a3d135c6ba5.jpg)  
Sequencing and read structure  

![](images/e688ea615b9d0f3da61ff349612e90e0ad4112cc053894fcd6afe61d691ffef0.jpg)  

sequenced at the start of read 2 (Figure 1D), thus enabling PCR duplicates to be collapsed and removed before all analysis. Importantly, including UMIs eliminates the potential for duplication to confound quantification in low-input libraries. Following duplicate removal, we analyzed the libraries for characteristics that indicate capture of elongating ribosome-protected footprints: (1) higher coverage in CDSs than UTRs, especially $3^{\prime}$ UTRs; (2) distinct read length distributions in CDSs vs. noncoding RNA (ncRNA), and (3) a 3-nt, periodic enrichment of inferred P-sites in frame with the annotated CDS (Figure S1).  

Both the bulk and ${\sim}30$ ng Ribo-seq libraries (Figure 2A), but not the RNase I-digested total RNA control library, exhibit higher coverage over CDSs than both $5^{\prime}$ and $3^{\prime}$ UTRs (Figures 2B and 2C). The Ribo-seq libraries exhibit tight, unimodal fragment length distributions (consistent with ribosome  

Another hallmark of successful Ribo-seq is the distinct enrichment of P-sites in frame with the dominant CDS every 3 nt. We used the RiboWaltz R package to infer P-sites. There was a clear 3-nt, in-frame periodicity to inferred ribosomal P-sites in both the bulk and ${\sim}30$ ng libraries around both the start and stop codons (Figure 2F) and enrichment of in-frame P-sites vs. out-of-frame P-sites (Figure 2G). This enrichment of in-frame P-sites strongly indicates that the nanoRibo-seq approach captures translating ribosomes since ribosomes move codon to codon in 3-nt steps and since few other biological processes can produce such a distinct 3-nt periodicity, in frame, specifically within CDSs. Together, these initial experiments indicate that Ribo-seq is feasible with very small quantities of brain lysate material since both the ${\sim}30$ ng and bulk libraries exhibit the hallmarks of translating ribosomes.  

![](images/1c48ce55faffbedb3102bcd55811df8e39c7f6357ce135187d640cadfa4dcc86.jpg)  
Figure 2. Ribo-seq from neonatal brain lysates with either ${\sim}30$ ng or bulk quantities of material (A) RNase I digestion patterns with brain lysate volume equivalent to either estimated ${\sim}30$ ng R $\mathsf{N A}{+}2.5\,\mathsf{U}/\mathsf{m L}$ RNase I or ‘‘bulk’’ conditions— $120\,\upmu\mathfrak{g}$ RNA $\cdot^{+}$ 1,500 U RNase I.  

# RNase I titration reveals optimal digestion conditions for nanoRibo-seq  

Of note, the ${\sim}30$ ng library displayed a longer, broader length distribution than the bulk library (Figure 2D), slightly longer than the ${\sim}29{-}31$ nt reported in most mammalian Ribo-seq studies.32,39,40 It also displayed a weaker 3-nt periodicity and weaker enrichment for in-frame P-sites compared to the bulk sample (Figures 2F and 2G), which might arise due to decreased accuracy in inferred P-site positions from a longer, broader length distribution.  

We hypothesized that more limited RNA digestion caused the longer and broader length distribution in the ${\sim}30$ ng library, so we rigorously tested multiple conditions to empirically optimize digestion with very-low-input quantities. This both identified optimal conditions for our neuronal samples and measured how digestion depends on [RNase I] in this input range. We tested the same [RNase I] as the original experiment $(2.5~\mathsf{U})$ mL) and a range of increasing [RNase I] (12.5, 62.5, and $312.5\ \mathsf{U/m L})$ (Figure 3A). All four enzyme concentrations yield enrichment of CDSs over $5^{\prime}$ and $3^{\prime}$ UTRs (Figures 3B and 3C), with the strongest enrichment at $12.5\ \mathrm{U/mL}$ . We found that increasing [RNase I] shortens and tightens the fragment length distribution over CDSs, and the higher concentrations yield footprint lengths in line with previous observations $\scriptstyle(\sim29-$ 31 nt) (Figure 3D). All digestion conditions yield distinct length distributions over CDSs compared to $3^{\prime}$ UTRs (Figure 3E). All libraries also displayed an in-frame, 3-nt P-site periodicity (Figure 3F) and enrichment of in-frame $\mathsf{P}$ -sites (Figure 3G). The 12.5 and $62.5\;\mathrm{U/mL}$ conditions displayed the strongest periodicity and the highest percentage of in-frame P-sites. Together, these results demonstrate that a wide range of digestion conditions generate libraries from nanogram-scale inputs that meet the three rigorous and stringent quality control metrics for Ribo-seq.  

# Alkaline fragmentation produces superior low input total RNA control libraries compared with RNase I digestion  

An important comparator for Ribo-seq experiments is a fragmented total RNA library for simultaneous measurement of RPF abundance and mRNA abundance, generated with library methods similar to those employed for the Ribo-seq library to avoid potential biases. We tested three strategies for generating fragmented total RNA control libraries with small quantities of RNA isolated from early postnatal forebrain lysate (Figure S2). We first tested RNase I digestion of purified RNA (Figures S2A and S2B). Since rRNA comprises ${>}90\%$ of total RNA samples, we also tested two rRNA-removal strategies to enrich fragmented control libraries for mRNA. The first was rRNA removal with the NEB human/mouse rRNA Removal Kit after gel purification ( $^{\ast}\mathsf{G}\!\to\!\mathsf{R}$ method’’), but we found that a large amount of short hybridization probe sequences contaminates the libraries (Figure S2C). Thus, we also tested rRNA removal before gel purification to remove the short probes ($^{\ast}\mathsf{R}\!\to\!\mathsf{G}$ method’’) since most run below the expected RPF size. We also found that RNase I produced broad fragment size distributions, and it had a very narrow concentration window between producing the desired ${\sim}20-$ to 60-nt fragment sizes and complete digestion (Figure S2B). Some Ribo-seq approaches instead utilize alkaline fragmentation for the control libraries.39,42 We produced an alkaline fragmented library from 7.5 ng purified RNA—without rRNA removal, both for simplicity and to optimally match the Ribo-seq procedure itself, which also lacks rRNA removal (Figure S2D). We found that alkaline fragmentation produces tighter size distributions, especially within the desired 20- to 60-nt window (Figure S2E).  

We also comparatively examined the mapping characteristics of these methods. Over half of reads in the ‘‘RNase I $\mathsf{G}\!\to\!\mathsf{R}$ method’’ were too short for alignment because they are short probe sequences (Figure S2F). The ‘‘RNase $|\mathsf{\nabla}\mathsf{R}\!\to\!\mathsf{G}$ method’’ had fewer reads too short for alignment but also fewer mRNAaligned reads (Figures S2F and S2G). Though the alkaline fragmentation method generated a high proportion of rRNA-aligning reads (Figure S2F), approximately $5\%$ of reads aligned to mRNA, roughly comparable to the fraction of reads aligning to mRNA in the Ribo-seq libraries themselves (Figure S2G). Importantly, we obtain deep coverage over mRNA despite such high rRNA mapping rates because of deep sequencing (30–50 M reads per library). Given the simplicity of alkaline fragmentation, the resultant tight size distributions, and the fact that the Ribo-seq libraries also omit an rRNA-depletion step, alkaline fragmentation appears to be superior for producing low-input fragmented total RNA control libraries.  

nanoRibo-seq QC metrics remain robust using as few as 1,000 HEK cells and show high correlation with both an alternate low-input Ribo-seq approach, and bulk Ribo-seq  

We tested the sensitivity of nanoRibo-seq to varying input quantities by performing biological replicates $\ h=2$ at each level of input) of nanoRibo-seq with 1,000, 10,000, and 50,000 HEK293T cells using the optimized digestion conditions identified during our tests with brain lysate in Figure 3. We find that all three cell input numbers yield almost identical fragment length distributions (Figure 4A), show strong and consistent enrichment of CDSs over UTRs (especially $3^{\prime}$ UTRs) (Figure 4B), and show clear, reproducible 3-nt periodicity in frame with annotated CDSs (Figures 4C and 4D), thus demonstrating that nanoRibo-seq produces high-quality ribosome footprints from as few as 1,000 cells.  

![](images/b0f31daf8fb014da5e78fd35453be1884b983b2878fc0104b96c0916f707045d.jpg)  
Figure 3. RNase I titration against estimated ${\sim}30$ ng brain lysate (A) RNase I digestion patterns at [RNase $\mathbb{I}=2.5$ , 12.5, 62.5, and $312.5\;\mathrm{U/mL}$ .(B) Read coverage over the Calm1 gene in Ribo-seq libraries (light to dark gray with increasing [RNase I]) vs. in vitro RNase I-digested total RNA (red). (C) RPKM coverage (reads/kilobase)/(million uniquely mapped) over CDSs $(5^{\prime}$ UTR: red, CDS: green, $3^{\prime}$ UTR: blue).  

We examined the correlations between normalized read counts from different replicates and cell inputs. We find $\mathsf{l o g}_{10}(\mathsf{T P M}{+}1)$ values across all replicates, and cell inputs are highly correlated with each other $({\sf R}^{2}=0.93–0.98)$ ) (Figure 4E). We directly compared nanoRibo-seq with the other low-input approach for which HEK data are publicly available (low-input RNase footprinting ) and with bulk Ribo-seq in HEK cells from Li et al.33We find solid correlations between bulk Ribo-seq $(\mathsf{R}^{2}~=~0.83–0.87)$ ) and low-input RNase footprinting (50,000 vs. 50,000: 0.84; 10,000 vs. 50,000: 0.81–0.84; and 1,000 vs. 1,000: 0.80) (Figure 4E). Together, these data indicate and strongly substantiate that CDS counts from nanoRibo-seq are technically reproducible down to 1,000 HEK cells and that they are reliable across a wide range of low-input conditions.  

# nanoRibo-seq QC metrics in sorted neurons remain robust with several thousand cells  

For optimal applicability in complex and rare cell-type systems, we directly tested optimized nanoRibo-seq with more challenging and demanding input material: fluorescently labeled neurons of particular cortical subtypes purified from mouse brain via neuronal FACS. We FACS purified IUE (in utero electroporation)-labeled CPN at postnatal day 4 (P4) from individual labeled cortical hemispheres and performed nanoRibo-seq (RP, short for "ribosome profiling") and alkaline fragmentation (AF), producing a range of 2,500–20,800 sorted neurons between samples. All RP libraries display nearly identical fragment length distributions over CDS (Figure S3A) enrichment for CDSs compared to $3^{\prime}$ UTRs (Figure S3B), and all samples exhibit strong 3-nt periodicity and enrichment in-frame P-sites (Figure S3C). Gene-level raw count distributions reveal that raw counts span nearly four orders of magnitude in the highest-input samples, decreasing to three orders of magnitude in the lower-input samples (Figure S3D). These distributions indicate that many thousands of genes receive 10s–1,000s of reads, even in the lowest-input cases.  

Another consideration is reproducibility between experiments. We examined correlations between the $\mathsf{l o g}_{10}(\mathsf{T P M}{+}1)$ values between samples and found that RP libraries and AF libraries cluster separately from each other (Figure S3E), as expected given the potentially large differences in transcriptional and translational output. Within an experiment type, samples are largely well correlated, particularly for the three samples made from $>\!9{,}000$ cells, though the lowest-input (2,500) RP sample was poorly correlated with the others (Figure S3E). Together, these results demonstrate that nanoRibo-seq can quantify translation in cortical subtypes using just a few thousand neurons as input; it thus enables investigation of translation in rare neuronal subtypes.  

# Quantification and comparison of T.E.s in CPN vs. SCPN reveals dozens of differentially translated mRNAs, with otherwise high T.E. correlation between subtypes for most genes  

A major question in neurobiology is how closely related subtypes of neurons form circuitry distinct from each other. CPN and SCPN are closely related cortical excitatory neurons, differentiating from closely related progenitors in essentially the same cellular environment, but form very distinct long-range axonal projections. Much is known regarding transcriptional programs that specify CPN and SCPN identity, but far less is known regarding potential differential translational programs that might provide a further layer of regulation during development and circuit formation.  

We applied nanoRibo-seq to globally measure translation between CPN and SCPN at P3, early in their postnatal development and circuit formation. We collected 13,000–22,500 CPN for RP and 4,600–7,500 CPN for AF and 13,000–16,000 SCPN for RP and 3,100–5,200 SCPN for AF in $\mathsf{n}=4$ biological replicates of RP and AF libraries. Biological replicates from the same subtype are very highly correlated with each other (most pairs have $\mathsf{R}^{2}=0.99$ , except one SCPN sample with $\mathsf{R}^{2}=0.91\substack{-0.94}$ vs. others) (Figure 5A) and, as expected, exhibit substantial transcriptional differences (Figures 5B–5D). However, changes between RPF abundance in the RP samples were highly correlated with changes in mRNA abundance (Figure 5B), and T.E.s were both much less variable between subtypes than RPF or mRNA abundance (Figures 5E and 5F). Together, these results support the interpretation that most of the distinction between CPNs and SCPNs occurs at the transcriptional level. This strong correlation between T.E. in the two subtypes suggests consistent translational regulatory environment between CPN and SCPN.  

We performed multiple analyses to identify differentially translated genes with optimal rigor. Since there is substantial transcriptional difference between CPN and SCPN (Figures 5B– 5D), and because we were deploying nanoRibo-seq for the first time, we tested several distinct computational methods for identifying differential translation using simulated datasets, enabling estimation of precision and recall and evaluation of these three approaches.  

RNA-seq can be modeled using a negative binomial distribution with two parameters: mean and dispersion. We used DESeq2 to estimate mean and dispersion for all genes above the detection threshold in the SCPN RP and AF datasets separately. We randomly generated RP and AF data in $\mathsf{n}=8$ replicates by sampling from the negative binomial distribution using the estimated mean and dispersion parameters, then grouped them into two arbitrary conditions with $\mathsf{n}=4$ replicates per condition. We next injected either 4,000 (Figures S4A and S4B) or 800 differentially expressed genes (Figures S4E and S4F) by drawing log2 fold changes from a wide gamma distribution and altering the fold change between both RP and AF  

![](images/f10616f0da552a57e901d673d2d9b1c708f6f39cd75d2550eb696cefedb93ca1.jpg)  
Figure 4. nanoRibo-seq in HEK293T cells, and direct comparison with low-input RNase  

footprinting and bulk Ribo-seq   
(A) Read length distributions for $\mathsf{n}=2$ nanoRibo-seq replicates from 1,000, 10,000, and 50,000 HEK293T cells.   
(B) RPKM coverage (reads/kilobase)/(million uniquely mapped) over CDSs(50UTR: red, CDS:green, $3^{\prime}$ UTR: blue).   
(C) Heatmap of P-site signal counts as a function of position from the start (red) or stop codon (blue). (D) Percentage of P-sites mapping to each reading frame (in frame: red, $+1$ : green, $+2:$ : blue).   
(E) Pearson correlation heatmap comparing log10  

$({\mathsf{T P M}}{+}1)$ ) values for genes above detection threshold $(>\!1/4$ of samples with $>\!\!10$ reads) in nanoRibo-seq vs. both bulk Ribo-seq and low-input RNase footprinting (both from Li et al.33).  

ally changed genes with near-perfect recall regardless of the number of confounding differentially expressed genes. In contrast, anota2seq exhibited poor precision, even at low recall. At a false discovery rate (FDR) of 0.1, DESeq2 and xtail have approximately $75\%$ precision and $25\%$ recall and are thus able to accurately and precisely identify small numbers of differentially translated genes despite substantial expression change.  

Following this broad evaluation of possible analytic methods, we applied the DESeq2 interaction model. We thus identify 40 differentially expressed genes (Table S1) between CPN and SCPN using DESeq2 (Figures 5E and 5F). Gene Ontology (GO) enrichment with this relatively small set of genes did not reveal enrichment for any set of terms. Interestingly, the majority (23/40) of these genes conditions across conditions. We then injected translational change for 120 genes by drawing log2 fold changes from a narrow gamma distribution for 40 genes with no transcriptional change (‘‘diffTE-only’’), 40 genes with ‘‘buffering’’ translational change opposing the expression change, and 40 genes with ‘‘intensifying’’ change in the same direction as expression change.  

Using these same randomly generated datasets, we tested three distinct analysis approaches: (1) a DESeq2 interaction model, treating the interaction between experiment type and condition as the change in TE—the same as the ‘‘delta TE,’’ which is an adaptation of DESeq2 for Ribo-seq, also testing two Ribo-seq-specific methods for identifying, quantifying, and evaluating the significance of T.E.; (2) xtail; and (3) anota2- seq.46 We plotted precision vs. recall curves for all three methods with both 4,000 (Figures S4C and S4D) and 800 (Figures S4G and S4H) differentially expressed genes. DESeq2 and xtail performed well, able to detect ${>}50\%$ of all translationare also significantly differentially expressed, with translation either buffering or intensifying expression change between the subtypes (Figure 5G). We provide examples of genes with pure translational change (e.g., Rps21; Figure 5H), translational buffering (translational change in the opposite direction from transcriptional change, e.g., Ergic1; Figure 5I), and intensification (translational change in the same direction as transcriptional change, e.g., Vamp2; Figure 5J). These results support the interpretation that, while most regulatory differences between CPN and SCPN at this stage are largely not translational, a relatively small set of genes have fine-tuned output by additional translational regulation on top of their transcriptional changes.  

# RNA-processing genes have especially high T.E.s in SCPN and CPN, and glycosylation genes have especially low T.E.s  

Given that most T.E.s are highly correlated between subtypes, we next focused on genes that show either especially high or low T.E. within each subtype. We defined such ‘‘skewed T.E.’’ genes as genes with RPF abundance that significantly differs from mRNA abundance, suggesting translational up- or downregulation. We identify thousands of skewed T.E. genes for each subtype (Figure 6A), and, consistent with coordinated translational regulation between subtypes, $80\%$ of skewed T.E. genes that are skewed in one subtype are also skewed in the same direction in the other subtype (Figures 6A–6C). To discover what types of genes have skewed high or skewed low T.E. in CPN and/or SCPN, we performed a GO analysis. RNA processing factors are enriched for high T.E. in both CPN and SCPN (Figures 6D and 6F), while glycosylation/glycoprotein biosynthesis-related terms are enriched among genes with low T.E. in both subtypes (Figures 6E and 6G). These results elucidate broad patterns of seemingly functionally organized translational output by specific cortical subtypes during early postnatal development.  

![](images/8e9370edf27dcdd1f045dae471fab50f0d6d2a3ae8fb2253516e95de7f74d560.jpg)  

# uORF translation by CPN: Transcripts coding for synaptic and neuronal adhesion genes are enriched among mRNAs with translated uORFs  

To rigorously investigate the theoretical possibility that skewed T.E. genes might arise from technical artifacts that might affect some genes more than others, we examined the Ribo-seq quality control (QC) metrics for each class of genes. Read length distributions were virtually identical between high, low, and nonsignificantly skewed genes in both subtypes (Figures S5A and S5B), as were the percentages of inferred P-sites falling into the annotated reading frame (Figures S5C–S5H). Coverage over CDSs was lower for genes with high T.E. vs. low T.E. (expected from the definition of T.E.), and $5^{\prime}$ UTR coverage was higher for genes with low T.E. compared with those with high T.E. (Figures S5I and S5J). This observation of increased coverage in the $5^{\prime}$ UTRs of genes with low T.E. is particularly intriguing since uORFs are short ORFs located in the $5^{\prime}$ UTRs of approximately half of all mammalian mRNAs,47–49 and translation of a uORF is frequently inhibitory for translation of the main $\mathsf{C D S}^{47-49}$ via a variety of mechanisms.38Thus, we investigated uORF translation and its relationship to T.E. in CPN and SCPN.  

To identify all ORFs—novel or annotated—that display evidence of translation by CPN and SCPN, we pooled all de-duplicated reads from each subtype separately and used the RiboCode analysis package to call translated uORFs. We identified approximately 2,000 uORFs and 600 ‘‘overlap_uORFs’’ (which overlap the CDS start codon, out of frame) for each subtype (Figure 7A), of which 1,040 uORFs and 243 overlap_uORFs are shared between subtypes, motivating further analysis (Figure 7B). Interestingly, compared with genes without translated uORFs (mean T.E.: CPN, 0.94; SCPN, 0.92), genes containing either translated uORFs or overlap_uORFs exhibited decreased T.E. The mean T.E. of genes with uORFs are 0.73 for CPN and 0.75 for SCPN, with a p value $<\!2\times10^{-16}$ vs. no uORF for both subtypes. The mean T.E. for genes with overlap uORFs are 0.89 for CPN and 0.88 for SCPN, with a p value $<\!0.1$ for both subtypes (Figures 7C and 7D). These results are consistent with the typically inhibitory role of uORF translation on CDSs observed in many other contexts.  

We next investigated key biological characteristics of genes containing uORFs. We first performed a GO analysis, using genes with uORFs or overlap_uORFs translated by SCPN and CPN as the gene list and genes with annotated protein-coding ORFs translated in both subtypes as background. Intriguingly, we identified strong enrichment of translated uORF-containing transcripts with enrichment for GO terms related to biological processes including synapse organization and assembly and axon development (Figure 7E). This suggests that uORFs in genes involved in neuronal synapse and circuit formation are active during early postnatal CPN circuit development and might function to refine the regulation of, and potentially more tightly control, synthesis of important functional proteins from these synaptic transcripts.  

To test this hypothesis further, we compared the T.E. distributions for genes in the GO terms ‘‘axon development’’ or ‘‘synapse organization’’ with or without uORFs. Interestingly, while ‘‘synapse organization’’ genes with uORFs (mean T.E.: CPN, 0.90; SCPN, 0.91) do not have lower T.E. than ‘‘synapse organization’’ genes without uORFs (mean T.E.: CPN, 0.91; SCPN, 0.92; $\mathsf{p}>0.1$ ), they do have a narrower distribution of T.E. than genes in the same term lacking uORFs (Figures 7F and 7G). In contrast, ‘‘axon development’’ genes with uORFs (mean T.E.: CPN, 0.83; SCPN, 0.84) were significantly less highly translated than those without uORFs (mean T.E.: CPN, 0.95; SCPN, 0.93; $\mathsf{p}<0.1$ for both subtypes), consistent with the typical translational reduction by uORFs (Figures 7H and 7I). These results indicate that, while uORF translation is broadly inhibitory, select uORFs—such as those enriched among axon and synapse development genes—can have more complex relationships between uORF translation and translational output of the gene involved.  

![](images/fc935bda5a35deffbf57e6beea3042caed2e9afffd20577cbc0eaf04235a681f.jpg)  
Figure 6. Genes with high or low T.E. in CPN and/or SCPN share functional characteristics (A) Overlaps between genes with significantly skewed T.E. in SCPN and in CPN. (B and C) Correlation between T.E. in SCPN vs. CPN, with genes with skewed T.E. in CPN (B) or SCPN (C) shown in red or blue. (D–G) Top five enriched GO terms among genes with T.E. skewed higher in CPN (D), T.E. skewed lower in CPN (E), T.E. skewed higher in SCPN (F), and T.E. skewed lower in SCPN (G).  

We identified translated uORFs in many examples of transcripts involved in biological processes highly relevant for CPN identity and with functional roles in axon guidance, synapse formation, and neuronal adhesion needed to form proper cortical circuitry. In particular, we highlight translated uORFs within the $5^{\prime}$ UTRs of multiple semaphorins, neuroligins, and neurexins (Figure S6). The identification of translated uORFs in neurexin mRNAs in vivo is particularly noteworthy since previous work found that multiple conserved, short uORFs within the $\mathsf{N r}\!\times\!\mathsf{n}\!_{2\alpha}$ $5^{\prime}$ UTR cooperatively inhibit translation in vitro.51We also find that there are multiple translated uORFs in the $5^{\prime}$ UTRs of Nrxn1 $\pmb{\alpha}$ and Nrxn1b. Further, we find that a longer Nrxn2a uORF using a non-canonical CUG start codon is also translated in developing CPN and SCPN (Figure S6L). Taken together, our results identifying uORF translation in particular classes of genes involved in circuit development by developing CPN and SCPN suggest that, during early postnatal neuronal development and circuit formation, many genes with translated uORFs function as potential regulators of axon guidance, cell adhesion, and synapse selection/formation, including multiple examples of semaphorins, neuroligins, and neurexin.  

![](images/413658e7a8afa8c19fd35b19cf1db7273e6f2e011f86909722b9be5a6855d9a3.jpg)  
CT.E.by uORF status, CPN   
DT.E. by uORF status, SCPN  

![](images/d048b9e170a8c293f40fc0946bc61dadfe691f6e3d05e1faea427718db260bff.jpg)  

# DISCUSSION  

nanoRibo-seq is a highly effective, quantitative, and optimized approach for ribosomal sequencing that requires $10^{2}–10^{3}$ less input material (demonstrated reproducibility with as few as 1,000 HEK cells, several thousand cortical neurons) than standard approaches for comprehensive investigation of sets of mRNAs undergoing active translation. It offers several important features, enabling high-quality libraries from just a few thousand neurons or other desired cells. These advantageous features include RNase digestion and library preparation, random fragmentation for mRNA abundance quantification, and inclusion of UMIs to eliminate duplicate RNAs. We present nanoRiboseq results and their reliable QC metrics from a range of cellular input and RNase I digestion conditions, using small populations of both HEK cells and FACS-purified cortical neuron subtypes as model systems. We present identification of uORF translation in select genes expressed by either/both CPN and SCPN, highlighting translated uORF enrichment within axon and synapse development genes, with translated uORFs regulating the T.E. of these genes involved in circuit formation.  

We first demonstrate the feasibility and optimized conditions for nanoRibo-seq with as little as ${\sim}30$ ng RNA brain lysate material by testing whether the global read distributions obtained conform to three expected features of successful Ribo-seq datasets: (1) enrichment of CDSs compared to UTRs and ncRNA, (2) distinct length distributions over CDSs and ncRNA, and (3) strong 3-nt, in-frame inferred P-site periodicity. All three QC metrics are stringently and reproducibly achieved. We also present several approaches for generating randomly fragmented control libraries, favoring AF for its simplicity and tight size distributions. We also found excellent correlations between biological replicates generated from between 1,000 and 50,000 HEK cells and strong correlations with both bulk Ribo-seq and with lowinput RNase footprinting,33 an alternate recently developed low-input approach. nanoRibo-seq also yields high-quality libraries from a few thousand FACS-purified cortical projection neurons. nanoRibo-seq offers substantial advantages for globally quantifying translation from rare neuron populations, for which only a few hundred or thousand neurons can be collected from a single brain. We have successfully generated translatomes from as few as 1,000 HEK cells and highly reproducible data with $>\!9{,}000$ labeled neurons isolated from mouse brains.  

We next applied nanoRibo-seq to compare translational output between two closely related cortical projection neuron subtypes (CPN and SCPN) early in their postnatal development. We find that, despite large transcriptional differences between the two subtypes, T.E. is highly correlated between CPN and SCPN, and 40 genes are significantly differentially translated between them. Over half of the differentially translated genes are also differentially expressed, with the translational change either buffering or intensifying the transcriptional change, suggesting that translational regulation might be fine-tuning protein synthesis from these select mRNAs. Our data indicate that, while the bulk of the regulatory differences between CPN and SCPN are transcriptional and not translational, a few dozen select genes have their translation further refined by translational regulation.  

These results indicate relatively consistent translational regulation by both subtypes at this early postnatal stage of development, in contrast to more variation during embryonic cortical development. Cortical projection neuron subtypes such as CPN and SCPN all differentiate from an initially common pool of cycling neural progenitors, and neurogenesis shifts from initially producing deep-layer output subtypes including SCPN to superficial-layer associative subtypes—especially CPN—as embryonic development proceeds. At early stages, newly differentiated neurons frequently co-express mRNAs encoding TFs that individually regulate differentiation of either superficial- or deep-layer neurons,19–24 and translational controls function to sharpen distinctions between subtypes by repressing inappropriately expressed mRNAs for the differentiating subtype while intensifying production of proteins needed for subtype identity.19–22,27,28 There are large changes in expression of key RNA-binding proteins, and in the translation machinery itself, as corticogenesis proceeds and shifts production from deeplayer to superficial-layer subtypes.20–22,25,27–29  

Our data provide multiple biological insights, stemming from the relatively rare and low-input material employed: (1)  

subtype-specific, global translational regulation by specific cortical subtypes vs. bulk cortex or heterogeneous cortical layers and (2) translational regulation at later, postnatal developmental times. With the notable exception of a few dozen genes, our data suggest that large shifts in translational regulatory programs have largely resolved by P3 and that transcriptional differences established earlier in development and continuing to evolve postmitotically are the dominant regulatory mechanisms between subtypes. It remains possible that there are further temporal changes in translation dynamics within each subtype as they form their distinct long-distance circuitry or perhaps even translation of specific sets of mRNAs within functional subcellular compartments, such as the growth cones of developing axons or synapses in more mature circuitry. Our data provide a first ‘‘snapshot’’ of translation and its regulation in subtype-specific neuronal somata at one developmental time. nanoRibo-seq enables increasingly deep investigation of celltype-specific translational regulation and of its dynamics and compartmentalization.  

Intriguingly, nanoRibo-seq data from CPN and SCPN in vivo identify a large number of actively translated uORFs in both subtypes, and these uORFs are enriched among genes involved in key processes for neuronal circuit formation and maturation— axon development and synapse organization. Multiple examples of translated uORFs are present in the $5^{\prime}$ UTRs of key regulators of synapse selection, maintenance, and activity, including semaphorins, neuroligins, and neurexins. The uORFs within neurexins provide important grounding in known biology from in vitro studies and extend importantly from what was previously known—our data indicate in vivo translation of multiple uORFs within $\mathsf{N r}\!\times\!\mathsf{n}\!2\mathsf{\alpha}$ previously shown to be potent regulators of its translation in vitro.51Further, we identify that Nrxn1 $\pmb{\alpha}$ and Nrxn1 $\upbeta$ $5^{\prime}$ UTRs also harbor a complex set of multiple translated uORFs, indicating that uORF translation by cortical projection neurons occurs more broadly, within multiple neurexin isoforms.  

These and other translated uORFs in the $5^{\prime}$ UTRs of multiple key regulators of development and maintenance of neuronal circuitry and synaptic connectivity suggest refinement and potentially tightly regulated translation of these mRNAs by developing CPN and SCPN via uORF translational activity. We find that mRNAs with translated uORFs are generally less efficiently translated than genes lacking uORFs, though there are notable exceptions; we find that synapse organization genes with uORFs are not less efficiently translated. Our results that mRNAs encoding synaptic components do not undergo the more standard uORF inhibition motivates future work into both more complex mechanisms of translational regulation by uORFs and their impact on expression of synaptic machinery. It is possible that uORF translation adds an additional layer of translational control and that it might be a mechanism to dynamically regulate specific sets of mRNAs in response to external cues, across developmental time, or in different subcellular compartments.  

We highlight translation of select uORFs as examples of noncanonical translation identified using low-input nanoRibo-seq because of their well-studied roles in translational regulation of downstream coding ORFs, though we note that this is only one possible role for uORF translation. We also identified over a thousand examples of other non-canonical translated ORFs, and both uORFs and these novel ORFs might produce a variety of translation products with potentially unique and specialized biological functions. Interestingly, and potentially directly related to our results, a recent Ribo-seq analysis of the human neocortex also identified a wide array of translated small ORFs, some of which might be validated as encoding micropeptides by mass spectrometry and many of which are primate or human specific. nanoRibo-seq offers a powerful and potentially especially informative approach for further interrogation of these novel translation events by specific neuronal subtypes, likely at specific developmental stages, at specific neuronal activity and environmental states, and in other specific biological contexts.  

We note with great interest that, during preparation and review of our manuscript, three similar low input RNase footprinting approaches were published: low-input RNase footprinting, LiRibo-seq, and Ribo-lite. These approaches share many of the same advances as nanoRibo-seq, including empirical optimization of RNase digestion, randomly fragmented control libraries, and UMIs for duplicate removal, resulting in high-quality libraries from 1,000 to 50,000 cells. Our approach uses a ligation-based protocol rather than a polyA-tailing and reverse transcription method, and the details of our RNase digestion conditions also differ. We deeply test the reproducibility and replicability of nanoRibo-seq via a direct comparison with LiRibo-seq and bulk Riboseq in HEK293 cells and find close similarity to these other approaches. We also demonstrate the applicability of low-input Ribo-seq in a very different and extremely demanding system: subtype-specific cortical neurons. This is an important advance since many neurodevelopmental and neurodegenerative diseases are caused by mutation and/or dysregulation of RNA-binding proteins that both directly and indirectly regulate translation; future application of nanoRibo-seq to these neurological disorders will likely be particularly informative. Our approaches are complementary, highly comparable, and reinforce each other.  

nanoRibo-seq offers benefits compared with common approaches that examine translational output from specific cell types by identifying ribosome-bound transcripts through Credependent expression of tagged ribosomal proteins.53–55 These approaches rely on specific driver lines, available only in some model systems. nanoRibo-seq does not require genetic labels and is thus amenable to any labeling strategy compatible with FACS or other cellular enrichment or purification methods. Further, as demonstrated here by the identification and analysis of extensive uORF translation in the $5^{\prime}$ UTRs of key synaptic structural and regulatory genes, nanoRibo-seq enables singlenucleotide-level analysis and discovery of novel patterns of translation and/or novel translational regulatory mechanisms. The results presented here predict that nanoRibo-seq will be particularly beneficial for investigation of translational regulation by many specific cell types during their development and maturation, in disease models, in non-model organisms lacking a broad set of genetic tools, in specific subcellular compartments of neurons and other cell types, and from small human patient samples.  

# Limitations of the study  

Although nanoRibo-seq is robust and reproducible, it has limitations. For application to pure populations of subtype-specific neurons, it requires FACS purification of the population of interest, which can limit its application, e.g., to populations of cells that are difficult to isolate by FACS, such as adult neurons. It relies on high sequencing depth to avoid complicated and expensive rRNA-removal steps and to obtain enough mRNA-mapped reads, though rRNA-removal approaches are an option. It also requires a gel purification step, takes several days to complete, and is currently not adapted for automated library preparation. Further, while nanoRibo-seq is well corelated with low-input RNase footprinting and bulk Ribo-seq in HEK293T cells, it is not as well correlated with these two methods as they are with each other when prepared by the same laboratory. This might arise from sample-specific differences, RNase digestion differences, or differences in the library preparation methods. We suspect that the most likely difference arises from the very different library strategies used in the two approaches (A-tailing and template switching vs. direct adapter ligation) since this is the most substantial difference between approaches. nanoRibo-seq produces reproducible libraries using several thousand FACS-purified neurons or 1,000 cultured HEK cells in our hands; future experiments should aim for these amounts of input unless further refinements are made and validated. nanoRibo-seq thus lacks single-cell resolution, and we note the recent publication of single-cell methods for Ribo-seq,36,37 translational profiling by in situ profiling,56,57or targeted base editing of ribosome-bound transcripts.57 nanoRibo-seq complements these approaches since it is a straightforward approach to globally profile translational output from as few as a few thousand purified or cultured cells from defined populations in vivo or in vitro.  

# STAR+METHODS  

Detailed methods are provided in the online version of this paper and include the following:  

d KEY RESOURCES TABLE RESOURCE AVAILABILITY $\bigcirc$ Lead contact $\bigcirc$ Materials availability $\bigcirc$ Data and code availability   
d EXPERIMENTAL MODEL AND STUDY PARTICIPANTS B Mice METHOD DETAILS $\bigcirc$ Neuronal labeling $\bigcirc$ Neuronal FACS B HEK293T culture $\bigcirc$ Generating ribosome protected fragments (RPFs) $\bigcirc$ 30 ng lysate initial experiment $\bigcirc$ Bulk experiment $\bigcirc$ Rnase I titration $\bigcirc$ Sorted CPN and SCPN $\bigcirc$ Sorted CPN from individual cortical hemispheres $\bigcirc$ HEK293T $\bigcirc$ Fragmentation of total RNA for control libraries $\bigcirc$ Oligo length standard sequences $\bigcirc$ Library preparation $\bigcirc$ Sequencing   
d QUANTIFICATION AND STATISTICAL ANALYSIS   
$\bigcirc$ Overview   
$\bigcirc$ Pre-proccessing and alignment pipeline   
$\bigcirc$ Analysis of P-site periodicity with RiboWaltz   
$\bigcirc$ Comparison with other methods in HEK293   
$\bigcirc$ nanoRibo-seq in individual hemispheres   
$\bigcirc$ Comparing translation efficiency between CPN and SCPN   
$\bigcirc$ Simulation and precision/recall analysis   
$\bigcirc$ Identification of translated ORFs with RiboCode   
$\bigcirc$ Analysis of translated uORF-containing mRNAs  

# SUPPLEMENTAL INFORMATION  

Supplemental information can be found online at https://doi.org/10.1016/j.   
celrep.2023.112995.  

# ACKNOWLEDGMENTS  

We thank Dr. Yasuhiro Itoh and Melody Ross for careful reading and discussion of this manuscript. This work was supported by NIH DP1 NS106665, with additional infrastructure support from R01 NS104055, R01 NS045523, and the Max and Anne Wien Professor of Life Sciences fund. J.E.F. was partially supported by T32 AG000222, F32 AG067661, and the DEARS Foundation (to J.D.M.).  

# AUTHOR CONTRIBUTIONS  

J.E.F. and J.D.M. conceived of experiments and wrote the manuscript. J.E.F. performed all Ribo-seq experiments and computational analysis. O.D. performed IUE labeling.  

DECLARATION OF INTERESTS  

The authors declare no competing interests.  

# INCLUSION AND DIVERSITY  

We support inclusive, diverse, and equitable conduct of research.  

Received: September 7, 2022   
Revised: May 26, 2023   
Accepted: August 2, 2023   
Published: August 24, 2023  

# REFERENCES  

1. Mills, E.W., and Green, R. (2017). Ribosomopathies: There’s strength in numbers. Science 358, eaan2755. https://doi.org/10.1126/science. aan2755.   
2. Robichaud, N., Sonenberg, N., Ruggero, D., and Schneider, R.J. (2019). Translational control in cancer. Cold Spring Harbor Perspect. Biol. 11, a032896. https://doi.org/10.1101/cshperspect.a032896.   
3. Siomi, H., Siomi, M.C., Nussbaum, R.L., and Dreyfuss, G. (1993). The protein product of the fragile X gene, FMR1, has characteristics of an RNAbinding protein. Cell 74, 291–298. https://doi.org/10.1016/0092-8674(93)   
90420-U.   
4. Greig, L.C., Woodworth, M.B., Galazo, M.J., Padmanabhan, H., and Macklis, J.D. (2013). Molecular logic of neocortical projection neuron specification, development and diversity. Nat. Rev. Neurosci. 14,   
755–769. https://doi.org/10.1038/nrn3586.   
5. Fame, R.M., MacDonald, J.L., and Macklis, J.D. (2011). Development, specification, and diversity of callosal projection neurons. Trends Neurosci. 34, 41–50. https://doi.org/10.1016/j.tins.2010.10.002.   
6. Aboitiz, F., and Montiel, J. (2003). One hundred million years of interhemispheric communication: The history of the corpus callosum. Braz. J. Med. Biol. Res. 36, 409–420. https://doi.org/10.1590/S0100- 879X2003000400002.   
7. Paul, L.K., Brown, W.S., Adolphs, R., Tyszka, J.M., Richards, L.J., Mukherjee, P., and Sherr, E.H. (2007). Agenesis of the corpus callosum: Genetic, developmental and functional aspects of connectivity. Nat. Rev. Neurosci. 8, 287–299. https://doi.org/10.1038/nrn2107.   
8. Arlotta, P., Molyneaux, B.J., Chen, J., Inoue, J., Kominami, R., and MacKlis, J.D. (2005). Neuronal subtype-specific genes that control corticospinal motor neuron development in vivo. Neuron 45, 207–221. https://doi.org/10.1016/j.neuron.2004.12.036.   
9. ¨O zdinler, P.H., Benn, S., Yamamoto, T.H., G€uzel, M., Brown, R.H., and Macklis, J.D. (2011). Corticospinal Motor Neurons and Related Subcerebral Projection Neurons Undergo Early and Specific Neurodegeneration in hSOD1G93A Transgenic ALS Mice. J. Neurosci. 31, 4166–4177. https://doi.org/10.1523/JNEUROSCI.4184-10.2011.   
10. Taylor, J.P., Brown, R.H., Jr., and Cleveland, D.W. (2016). Decoding ALS: from genes to mechanism. Nature 539, 197–206. https://doi.org/10.1038/ nature20413.   
11. Udaka, F., Kameyama, M., and Tomonaga, M. (1986). Degeneration of betz cells in motor neuron disease. A Golgi study. Acta Neuropathol. 70, 289–295. https://doi.org/10.1007/BF00686086.   
12. Cobos, I., and Seeley, W.W. (2015). Human von economo neurons express transcription factors associated with Layer v subcerebral projection neurons. Cerebr. Cortex 25, 213–220. https://doi.org/10.1093/cercor/ bht219.   
13. Lin, L.C., Nana, A.L., Hepker, M., Hwang, J.H.L., Gaus, S.E., Spina, S., Cosme, C.G., Gan, L., Grinberg, L.T., Geschwind, D.H., et al. (2019). Preferential tau aggregation in von Economo neurons and fork cells in frontotemporal lobar degeneration with specific MAPT variants. Acta Neuropathol. Commun. 7, 159–210. https://doi.org/10.1186/s40478-019-0809-0.   
14. Seeley, W.W., Carlin, D.A., Allman, J.M., Macedo, M.N., Bush, C., Miller, B.L., and DeArmond, S.J. (2006). Early frontotemporal dementia targets neurons unique to apes and humans. Ann. Neurol. 60, 660–667. https:// doi.org/10.1002/ana.21055.   
15. Molyneaux, B.J., Arlotta, P., Fame, R.M., MacDonald, J.L., MacQuarrie, K.L., and Macklis, J.D. (2009). Novel subtype-specific genes identify distinct subpopulations of callosal projection neurons. J. Neurosci. 29, 12343–12354. https://doi.org/10.1523/JNEUROSCI.6108-08.2009.   
16. Galazo, M.J., Emsley, J.G., and Macklis, J.D. (2016). Corticothalamic Projection Neuron Development beyond Subtype Specification: Fog2 and Intersectional Controls Regulate Intraclass Neuronal Diversity. Neuron 91, 90–106. https://doi.org/10.1016/j.neuron.2016.05.024.   
17. Sohur, U.S., Padmanabhan, H.K., Kotchetkov, I.S., Menezes, J.R.L., and Macklis, J.D. (2014). Anatomic and molecular development of corticostriatal projection neurons in mice. Cerebr. Cortex 24, 293–303. https://doi. org/10.1093/cercor/bhs342.   
18. Catapano, L.A., Arnold, M.W., Perez, F.A., and Macklis, J.D. (2001). Specific neurotrophic factors support the survival of cortical projection neurons at distinct stages of development. J. Neurosci. 21, 8863–8872.   
19. Li, Z., Tyler, W.A., Zeldich, E., Santpere Baro´ , G., Okamoto, M., Gao, T., Li, M., Sestan, N., and Haydar, T.F. (2020). Transcriptional priming as a conserved mechanism of lineage diversification in the developing mouse and human neocortex. Sci. Adv. 6, eabd2068-16. https://doi.org/10. 1126/SCIADV.ABD2068.   
20. Zahr, S.K., Yang, G., Kazan, H., Borrett, M.J., Yuzwa, S.A., Voronova, A., Kaplan, D.R., and Miller, F.D. (2018). A Translational Repression Complex in Developing Mammalian Neural Stem Cells that Regulates Neuronal Specification. Neuron 97, 520–537.e6. https://doi.org/10.1016/j.neuron. 2017.12.045.   
21. Yang, G., Smibert, C.A., Kaplan, D.R., and Miller, F.D. (2014). An eIF4E1/ 4E-T complex determines the genesis of neurons from precursors by translationally repressing a proneurogenic transcription program. Neuron 84, 723–739. https://doi.org/10.1016/j.neuron.2014.10.022.   
22. Harnett, D., Ambrozkiewicz, M.C., Zinnall, U., Borisova, E., Dannenberg, R., Imami, K., M€unster-wandowski, A., Fauler, B., Selbach, M., Landthaler, M., et al. (2021). Timed global reorganization of protein synthesis during neocortical neurogenesis at codon resolution.   
23. Telley, L., Agirman, G., Prados, J., Amberg, N., Fie\` vre, S., Oberst, P., Bartolini, G., Vitali, I., Cadilhac, C., Hippenmeyer, S., et al. (2019). Temporal patterning of apical progenitors and their daughter neurons in the developing neocortex. Science 364, eaav2522. https://doi.org/10.1126/science.aav2522.   
24. Azim, E., Shnider, S.J., Cederquist, G.Y., Sohur, U.S., and Macklis, J.D. (2009). Lmo4 and Clim1 progressively delineate cortical projection neuron subtypes during development. Cerebr. Cortex 19, 62–69. https://doi.org/ 10.1093/cercor/bhp030.   
25. Kraushar, M.L., Thompson, K., Wijeratne, H.R.S., Viljetic, B., Sakers, K., Marson, J.W., Kontoyiannis, D.L., Buyske, S., Hart, R.P., and Rasin, M.R. (2014). Temporally defined neocortical translation and polysome assembly are determined by the RNA-binding protein Hu antigen R. Proc. Natl. Acad. Sci. USA 111, E3815–E3824. https://doi.org/10.1073/pnas. 1408305111.   
26. DeBoer, E.M., Azevedo, R., Vega, T.A., Brodkin, J., Akamatsu, W., Okano, H., Wagner, G.C., and Rasin, M.R. (2014). Prenatal deletion of the RNAbinding protein HuD disrupts postnatal cortical circuit maturation and behavior. J. Neurosci. 34, 3674–3686. https://doi.org/10.1523/JNEUROSCI.3703-13.2014.   
27. Ambrozkiewicz, M.C., Borisova, E., Newman, A.G., Kraushar, M.L., Schaub, T., Dannenberg, R., Brockmann, M., Rosa´ rio, M., Turko, P., Jahn, O., et al. (2021). Ire1 a -Regulated Rate of mRNA Translation is Required for Acquisition of Identity and Polarity in Upper Layer Cortical Neurons Ire1 a -Regulated Rate of mRNA Translation is Required for Acquisition of Identity and Polarity in Upper Layer Cortical Neurons, pp. 1–58.   
28. Popovitchenko, T., Park, Y., Page, N.F., Luo, X., Krsnik, Z., Liu, Y., Salamon, I., Stephenson, J.D., Kraushar, M.L., Volk, N.L., et al. (2020). Translational derepression of Elavl4 isoforms at their alternative $5^{\prime}$ UTRs determines neuronal development. Nat. Commun. 11, 1674. https://doi.org/10. 1038/s41467-020-15412-8.   
29. Kraushar, M.L., Viljetic, B., Wijeratne, H.R.S., Thompson, K., Jiao, X., Pike, J.W., Medvedeva, V., Groszer, M., Kiledjian, M., Hart, R.P., and Rasin, M.R. (2015). Thalamic WNT3 secretion spatiotemporally regulates the neocortical ribosome signature and mRNA translation to specify neocortical cell subtypes. J. Neurosci. 35, 10911–10926. https://doi.org/10. 1523/JNEUROSCI.0601-15.2015.   
30. Chau, K.F., Shannon, M.L., Fame, R.M., Fonseca, E., Mullan, H., Johnson, M.B., Sendamarai, A.K., Springel, M.W., Laurent, B., and Lehtinen, M.K. (2018). Downregulation of ribosome biogenesis during early forebrain development. Elife 7, 369988–e37026. https://doi.org/10.7554/eLife. 36998.   
31. Zappulo, A., van den Bruck, D., Ciolli Mattioli, C., Franke, V., Imami, K., McShane, E., Moreno-Estelles, M., Calviello, L., Filipchyk, A., PegueroSanchez, E., et al. (2017). RNA localization is a key determinant of neurite-enriched proteome. Nat. Commun. 8, 583. https://doi.org/10.1038/ s41467-017-00690-6.   
32. Liu, B., Molinaro, G., Shu, H., Stackpole, E.E., Huber, K.M., and Richter, J.D. (2019). Optimization of ribosome profiling using low-input brain tissue from fragile X syndrome model mice. Nucleic Acids Res. 47, e25. https:// doi.org/10.1093/nar/gky1292.   
33. Li, Q., Yang, H., Stroup, E.K., Wang, H., and Ji, Z. (2022). Low-input RNase footprinting for simultaneous quantification of cytosolic and mitochondrial translation. Genome Res. 32, 545–557. https://doi.org/10.1101/gr. 276139.121.   
34. Xiong, Z., Xu, K., Lin, Z., Kong, F., Wang, Q., Quan, Y., Fang, J., Yu, G., Liu, B., Wang, L., et al. (2022). Ultrasensitive Ribo-seq reveals translational landscapes during mammalian oocyte-to-embryo transition and pre-implantation development24. https://doi.org/10.1038/s41556-022-00928-6.   
35. Zhang, C., Wang, M., Li, Y., and Zhang, Y. (2022). Profiling and functional characterization of maternal mRNA translation during mouse maternal-tozygotic transition. Sci. Adv. 8, eabj3967. https://doi.org/10.1126/sciadv. abj3967.   
36. VanInsberghe, M., van den Berg, J., Andersson-Rolf, A., Clevers, H., and van Oudenaarden, A. (2021). Single-cell Ribo-seq reveals cell cycledependent translational pausing. Nature 597, 561–565. https://doi.org/ 10.1038/s41586-021-03887-4.   
37. Ozadam, H., Tonn, T., Han, C.M., Segura, A., Hoskins, I., Rao, S., Ghatpande, V., Tran, D., Catoe, D., Salit, M., and Cenik, C. (2023). Single-cell quantification of ribosome occupancy in early mouse development. Nature 618, 1057–1064. https://doi.org/10.1038/s41586-023-06228-9.   
38. Hinnebusch, A.G., Ivanov, I.P., and Sonenberg, N. (2016). Translational control by $5^{\prime}$ -untranslated regions of eukaryotic mRNAs. Science 352, 1413–1416. https://doi.org/10.1126/science.aad9868.   
39. Ingolia, N.T., Lareau, L.F., and Weissman, J.S. (2011). Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes. Cell 147, 789–802. https://doi.org/10.1016/j.cell. 2011.10.002.   
40. Ingolia, N.T., Brar, G.A., Stern-Ginossar, N., Harris, M.S., Talhouarne, G.J.S., Jackson, S.E., Wills, M.R., and Weissman, J.S. (2014). Ribosome Profiling Reveals Pervasive Translation Outside of Annotated ProteinCoding Genes. Cell Rep. 8, 1365–1379. https://doi.org/10.1016/j.celrep. 2014.07.045.   
41. Lauria, F., Tebaldi, T., Bernabo\` , P., Groen, E.J.N., Gillingwater, T.H., and Viero, G. (2018). riboWaltz: Optimization of ribosome P-site positioning in ribosome profiling data. PLoS Comput. Biol. 14, e1006169. https:// doi.org/10.1371/journal.pcbi.1006169.   
42. Ingolia, N.T., Ghaemmaghami, S., Newman, J.R.S., and Weissman, J.S. (2009). Genome-Wide Analysis in Vivo of Translation with Nucleotide Resolution Using Ribosome Profiling. Science 324, 218–223.   
43. Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550–621. https://doi.org/10.1186/s13059-014-0550-8.   
44. Chothani, S., Adami, E., Ouyang, J.F., Viswanathan, S., Hubner, N., Cook, S.A., Schafer, S., and Rackham, O.J.L. (2019). deltaTE: Detection of Translationally Regulated Genes by Integrative Analysis of Ribo-seq and RNA-seq Data. Curr. Protoc. Mol. Biol. 129, e108. https://doi.org/10. 1002/cpmb.108.   
45. Xiao, Z., Zou, Q., Liu, Y., and Yang, X. (2016). Genome-wide assessment of differential translations with ribosome profiling data. Nat. Commun. 7, 11194. https://doi.org/10.1038/ncomms11194.   
46. Oertlin, C., Lorent, J., Murie, C., Furic, L., Topisirovic, I., and Larsson, O. (2019). Generally applicable transcriptome-wide analysis of translation using anota2seq. Nucleic Acids Res. 47, E70. https://doi.org/10.1093/nar/ gkz223.   
47. Calvo, S.E., Pagliarini, D.J., and Mootha, V.K. (2009). Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. USA 106, 7507– 7512. https://doi.org/10.1073/pnas.0810916106.   
48. Chew, G.L., Pauli, A., and Schier, A.F. (2016). Conservation of uORF repressiveness and sequence features in mouse, human and zebrafish. Nat. Commun. 7, 11663–11710. https://doi.org/10.1038/ncomms11663.   
49. Johnstone, T.G., Bazzini, A.A., and Giraldez, A.J. (2016). Upstream ORF s are prevalent translational repressors in vertebrates. EMBO J. 35, 706–723. https://doi.org/10.15252/embj.201592759.   
50. Xiao, Z., Huang, R., Xing, X., Chen, Y., Deng, H., and Yang, X. (2018). De novo annotation and characterization of the translatome with ribosome profiling data. Nucleic Acids Res. 46, e61. https://doi.org/10.1093/nar/ gky179.   
51. Ding, X., Meng, S., Zhou, J., Yang, J., Li, H., and Zhou, W. (2020). Translational Inhibition of $\propto$ -Neurexin 2. Sci. Rep. 10, 3403. https://doi.org/10. 1038/s41598-020-60289-8.   
52. Duffy, E.E., Finander, B., Choi, G., Carter, A.C., Pritisanac, I., Alam, A., Luria, V., Karger, A., Phu, W., Sherman, M.A., et al. (2021). Developmental Dynamics of RNA Translation in the Human Brain. Preprint at bioRxiv. https://doi.org/10.1001/2021.10.22.465170.   
53. Heiman, M., Kulicke, R., Fenster, R.J., Greengard, P., and Heintz, N. (2014). Cell type-specific mRNA purification by translating ribosome affinity purification (TRAP). Nat. Protoc. 9, 1282–1291. https://doi.org/10.1038/ nprot.2014.085.   
54. Heiman, M., Schaefer, A., Gong, S., Peterson, J.D., Day, M., Ramsey, K.E., Sua´ rez-Farin˜ as, M., Schwarz, C., Stephan, D.A., Surmeier, D.J., et al. (2008). A Translational Profiling Approach for the Molecular Characterization of CNS Cell Types. Cell 135, 738–748. https://doi.org/10.1016/j.cell. 2008.10.028.   
55. Sanz, E., Yang, L., Su, T., Morris, D.R., McKnight, G.S., and Amieux, P.S. (2009). Cell-type-specific isolation of ribosome-associated mRNA from complex tissues. Proc. Natl. Acad. Sci. USA 106, 13939–13944. https:// doi.org/10.1073/pnas.0907143106.   
56. Zeng, H., Huang, J., Ren, J., Wang, C.K., Tang, Z., Zhou, H., Zhou, Y., Shi, H., Aditham, A., Sui, X., et al. (2023). Spatially resolved single-cell translatomics at molecular resolution. Science 380, eadd3067. https://doi.org/ 10.1126/science.add3067.   
57. Brannan, K.W., Chaim, I.A., Marina, R.J., Yee, B.A., Kofman, E.R., Lorenz, D.A., Jagannatha, P., Dong, K.D., Madrigal, A.A., Underwood, J.G., and Yeo, G.W. (2021). Robust single-cell discovery of RNA targets of RNAbinding proteins and ribosomes. Nat. Methods 18, 507–519. https://doi. org/10.1038/s41592-021-01128-0.   
58. Woodworth, M.B., Greig, L.C., Liu, K.X., Ippolito, G.C., Tucker, H.O., Macklis, J.D., Woodworth, M.B., Greig, L.C., Liu, K.X., Ippolito, G.C., and Tucker, H.O. (2016). Ctip1 Regulates the Balance between Specification of Distinct Projection Neuron Subtypes in Deep Article Ctip1 Regulates the Balance between Specification of Distinct Projection Neuron Subtypes in Deep Cortical Layers. Cell Rep. 15, 999–1012. https://doi. org/10.1016/j.celrep.2016.03.064.   
59. Catapano, L.A., Arlotta, P., Cage, T.A., and Macklis, J.D. (2004). Stagespecific and opposing roles of BDNF, NT-3 and bFGF in differentiation of purified callosal projection neurons toward cellular repair of complex circuitry. Eur. J. Neurosci. 19, 2421–2434. https://doi.org/10.1111/j. 0953-816X.2004.03303.x.   
60. O¨ zdinler, P.H., and Macklis, J.D. (2006). IGF-I specifically enhances axon outgrowth of corticospinal motor neurons. Nat. Neurosci. 9, 1371–1381. https://doi.org/10.1038/nn1789.   
61. Liao, Y., Smyth, G.K., and Shi, W. (2013). The Subread aligner: Fast, accurate and scalable read mapping by seed-and-vote. Nucleic Acids Res. 41. https://doi.org/10.1093/nar/gkt214.   
62. Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and Durbin, R. (2009). The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079. https://doi.org/10. 1093/bioinformatics/btp352.   
63. Yu, G., Wang, L.G., Han, Y., and He, Q.Y. (2012). ClusterProfiler: An R package for comparing biological themes among gene clusters. OMICS J. Integr. Biol. 16, 284–287. https://doi.org/10.1089/omi.2011.0118.  

# STAR+METHODS  

KEY RESOURCES TABLE   

![](images/53525ed29ca9cba52b013ff0292a169f17dcfae904c4bf4c60128ffad18f9d97.jpg)  

![](images/1515c002b18f2490aa254b3a0f13d108a456f2fb11f9e061e9cd530f4e951a0b.jpg)  

# RESOURCE AVAILABILITY  

Lead contact  

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jeffrey D.   
Macklis (jeffreymacklis@fas.harvard.edu).  

# Materials availability  

This study did not generate new unique reagents.  

# Data and code availability  

d Data: RNA-seq data have been deposited at GEO and are publicly available as of the date of publication. Data is available at GEO: GSE197060, and this accession number is also listed in the key resources table.   
d Code: All original code has been deposited at Github and is publicly available as of the date of publication. DOIs are listed in the key resources table.   
d Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.  

# EXPERIMENTAL MODEL AND STUDY PARTICIPANTS  

# Mice  

The vertebrate animal experimental protocols were approved by the Harvard University Institutional Animal Care and Use Committee and were performed in accordance with institutional and federal guidelines. For in utero electroporation experiments, we ordered timed pregnant CD-1 dams from Charles River Laboratories, and performed in utero electroporation of fluorescent labeling constructs at $\mathsf{E}14.5$ or E15.5. For SCPN retrograde labeling experiments, we ordered pregnant CD-1 dams from Charles River Laboratories, and performed ultrasound-guided injections into the cerebral peduncles of the P1 litter. We report developmental stages for all experiments in the descriptions of the experiments. All experiments except the individual cortical hemisphere experiments used mixed groups of male and female animals, and in the individual cortical hemisphere experiment we did not record the sex of each individual animal due to the difficulty of both rapidly and accurately determining sex at this developmental stage. We acknowledge that our results are limited in that they do not allow analysis of sex-specific effects.  

# METHOD DETAILS  

# Neuronal labeling  

# CPN  

We performed electroporation of fluorescent constructs using CD-1 mice at E14.5 or E15.5. Briefly, we exposed both uterine horns, and injected a solution of $5\,\upmu\mathrm{g}/\upmu\mathrm{L}$ DNA $+0.1\%$ FastGreen in 1X TE containing 5ug/uL plasmid encoding myristolated-TdTomato into one lateral ventricle in approximately one-half of embryos. We then electroporated each injected embryo with 5 pulses of 500 ms each at $34\;\mathsf{m V}.$ We screened P1 pups for red fluorescence in the cortex under an epifluorescence dissection microscope.  

# SCPN  

We labeled SCPN by ultrasound-guided injection into the cerebral peduncles at P1 CD-1 pups, using injection conditions described previously.16,58 Briefly, we anesthetized the pups hypothermically by placing them on ice for 3 min, then gently stabilizing them on the injection platform. We visualized the cerebral peduncle and placement of the injection micropipette via ultrasound backscatter microscopy. We injected Alexa 555-cholera toxin B (CTB) at four sites in the left cerebral peduncle (dorsal, ventral, medial, and lateral sites) to ensure coverage of the peduncle, with five 5 nL pulses per site.  

# Neuronal FACS  

We performed neuronal FACS using established approaches,15,16,18,58–60 but with the addition of translational inhibitors to prevent ex vivo translational elongation; all buffers contained $100\,\upmu\mathrm{g/mL}$ cycloheximide (CHX). We dissected brains into HBSS buffer $^+$ CHX, visualized the labeled region of cortex using an epifluorescence dissection microscope, and dissected the labeled cortical region into dissociation solution $(\mathsf{D S})+\mathsf{C H X}$ . We washed cortical tissue pieces twice in $\mathsf{D S}+\mathsf{C H X}$ , then enzymatically digested twice by incubation in enzyme solution (DS with cysteine and papain) $+\,\mathsf{C H X}$ for 15 min each time (with inversion every 5 min to keep tissue pieces mixed). We washed twice in wash solution $(\mathsf{W S})+\mathsf{C H X}$ , triturated 15–20 times in ${\sim}1\ m{\sf{L}}$ of $\mathsf{W S}+\mathsf{C H X}$ using fire-polished glass pipettes. We diluted the cell suspension with $4\,\mathsf{m L W S}+\mathsf{C H X},$ , spun down cells for 5 min at $10009$ , triturated again in 1 mL WS $^+$ CHX, and passed the cell suspension through a strainer cap. We added 1:1000 SYTOX Blue to enable screening out dead cells. We then FAC-sorted the red cells, and collected them in a DNA LoBind Eppendorf tube containing 2X polysome buffer $^+$ CHX (25 mM HEPES $\mathsf{p H}=8.0$ , 200 mM KCl, 10 mM $\mathsf{M g C l}_{2}$ ,$4\,\mathsf{m M}\,\mathsf{C a C l}_{2}$ ,$\mid\%$ nonidet P-40, $100\ \upmu\mathrm{g/mL}$ CHX).  

# HEK293T culture  

We obtained a fresh vial of HEK293T cells from ATCC, and cultured in 1X DMEM $^+$ GlutaMAX supplemented with $10\%$ Fetal Bovine Serum and $10{\cup}/{\sf m}{\sf L}$ Penicillin and $10\ensuremath{\,\upmu\mathrm{g/mL}}$ Streptomycin. Once cultures reached $80\%$ confluency, we harvested for Ribo-seq. We added $100\,\upmu\mathrm{g/mL}$ CHX to the culture media for 2 min at $37^{\circ}\mathrm{C}$ , quickly aspirated media, washed once in 1X PBS supplemented with $100\ \upmu\mathfrak{g}/\mathfrak{m L}$ CHX, and harvested by trypsinization (with CHX added to trypsin at a final concentration of $100\ \upmu\mathrm{g/mL})$ ). We quenched trypsin with 7 volumes of media (supplemented with CHX), pelleted at $2009$ for 5 min at $4\mathrm{{}^{\circ}C}$ , resuspended in $1\times\mathsf{P B S+C H X}$ and pelleted again. We then resuspended the pellet in 1X PBS+CHX, counted cells using a BioRad TC20 Automated Cell Counter, and proceeded to Ribo-seq.  

# Generating ribosome protected fragments (RPFs)  

To perform RNase I digestion, we diluted brain lysate or FACS-purified neurons into 1X polysome digestion buffer (2X polysome buffer diluted 1:1 with dilution buffer, which is $0.75\times$ PBS, $80~\mathsf{m M}$ sucrose, $100~\upmu{}{\mathfrak{g}}/{\mathfrak{m L}}$ CHX, 1/500X RNasin), added the specific amount of Rnase I for each experimental variation (Ambion brand, ThermoFisher #2294), and placed the mixture on a rocker for 45 min at room temperature. To stop digestion, and concentrate the very dilute RNA for purification, we added $10~\upmu\mathrm{L}$ Superasin (200 units, Ambion brand, ThermoFisher #2694), poured the reaction into an Amicon Ultra-15 Centrifugal Filter ( $30\mathsf{k D}$ NMWL filter, EMD Millipore UFC903024), and centrifuged at $5000\;\mathrm{g}$ at $4\mathrm{{}^{\circ}C}$ for approximately 25–60 min, until ${<}400~\upmu\mathrm{L}$ of retenate remained. To purify the RNA, we used a slight modification of the Zymo Quick RNA Microprep (Zymo Research #1051) extraction kit procedure to purify small RNAs out of large volumes. We transferred the retained RNA to a $15\,\mathrm{mL}$ Falcon tube, added $800\,\upmu\mathrm{L}$ lysis buffer (mixing vigorously by pipette), then added 4 mL $100\%$ EtOH (mixing by pipette or inversion at least 7 times). We then repeatedly centrifuged the solution onto a single Zymo spin column, spinning at full speed for 15 s in a microcentrifuge until all solution from one sample had been passed over that single column. We then followed the remainder of the Zymo RNA preparation protocol; in brief: we spun the column with $400~\upmu\mathrm{L}$ prep buffer, then $700\,\upmu\mathrm{L}$ wash buffer, $400~\upmu\mathrm{L}$ wash buffer, and spun the column empty to remove residual wash buffer. We then transferred the column to a labeled DNA LoBind Eppendorf $1.5\,\mathsf{m L}$ tube (VWR # 022431021), added $20\,\upmu\mathrm{L}$ nucleasefree $\mathsf{H}_{2}\mathsf{O}$ , allowed the column to sit for 5–10 min at room temperature, then spun at full speed on a microcentrifuge. The eluted RNA is able to be frozen at $-80$ for long-term storage.  

To enrich RNAs in the RPF size range, we gel purified the RNA fragments. First, we added $20\,\upmu\mathrm{L}$ 2X formamide denaturing solution $(20\%$ Glycerol, $80\%$ Formamide (Promega #H5051), $60\,\mathrm{mg/mL}$ Bromophenol Blue) to the RNAs, heated at $70^{\circ}{\mathrm{C}}$ for 5 min, placed on ice until ready, then loaded onto a $15\%$ TBE-UREA acrylamide gel, and electrophoresed in 1X TBE running buffer at 200V for approximately $45\;\mathrm{min}$ (until the bromophenol blue marker reached the bottom of the gel). To ensure proper sizing, we ran 5 ng 10/60 Oligo Length Standards 10/60 Ladder (IDT # 51-05-15-01) and 1–3 ng of synthetic 27 and 36 nt RNA oligos. We stained the RNA in gels with SYBRGold (Thermo S11494) for $10\;\mathsf{m i n}$ in 1X TBE, followed by a 5 min wash with 1X TBE, and two rinses with 1X TBE to remove background. We then imaged RNA in the gels using a standard gel documentation instrument with UV transillumination, exposing for $_{20\;\mathsf{s}.}$ We excised the RNA running between the 27–36 nt markers. To purify the RNA, we placed the gel slice in $400~\upmu\mathrm{L}$ Crush and Soak buffer (300 mM KOAc, 1 mM EDTA, $\mid\%$ SDS) in a LoBind Eppendorf Tube, and rotated on a shaker at room temperature overnight. We transferred the liquid to a $15\,\mathrm{mL}$ Falcon tube, added $400\,\upmu\mathrm{L}$ Zymo Lysis Buffer and 4 mL $100\%$ EtOH, repeatedly centrifuged one sample onto one spin column, then purified RNA using the standard Zymo kit protocol from the $400~\upmu\mathrm{L}$ prep buffer step onwards after binding the RNA to the column as described above.  

Since the digestion conditions varied between experiments, some specific conditions varied across experiments.  

# 30 ng lysate initial experiment  

To prepare the homogenate for the approximately ${\sim}30$ ng initial trial, as well as for the bulk trial, we harvested three P4 CD-1 forebrains (olfactory bulbs removed; midbrain, cerebellum, and brainstem removed via vertical cut at caudal end of cortex) into ice-cold 1X polysome buffer $+\mathsf{C H X}$ , homogenized in a Dounce homogenizer with 11 strokes on ice, transferred to a $15\;\mathrm{mL}$ Falcon tube, and centrifuged at $1660\,\mathfrak{g}$ at $4^{\circ}\mathrm{C}$ for 15 min to pellet nuclei. We then extracted RNA from $60\,\upmu\mathrm{L}$ of homogenate using the standard Zymo Quick RNA protocol, and measured RNA concentration via a Nanodrop (Nanodrop 1000, Thermo). From this estimate of RNA concentration, we calculated the volume of homogenate equivalent to ${\sim}30$ ng of RNA. We added this volume of homogenate to 6 mL of 1X polysome digestion buffer, added Rnase I to a final concentration of $2.5\,\mathrm{U/mL}$ , and allowed digestion to proceed for 45 min on a rocking platform covered in aluminum foil to prevent photodegradation of cycloheximide. We then added $10~\upmu\mathrm{L}$ Superasin, and purified footprints as described above.  

# Bulk experiment  

We used the same lysate as used for the ${\sim}30$ ng experiment, and digested ${\sim}120\,\upmu\g$ RNA equivalent $(640\,\upmu\mathrm{L})$ with 1500 U Rnase I in $640\,\upmu\mathrm{L}$ for 45 min at RT on a rocker covered in foil. We added Superasin, then split the lysate across 6 separate columns, purified RNA using the standard Zymo Quick RNA Microprep protocol, and combined elutions into one tube. We measured the RNA concentration of this bulk footprint elution using a Nanodrop, and ran the equivalent volume of $1~{\upmu\gimel}$ on PAGE gel to purify footprints.  

# Rnase I titration  

We homogenized P6 forebrains in 10X volume of 1X polysome buffer $\mathrel{\+}\mathrm{CHX}$ , as described in the ${\sim}30$ ng and bulk initial trials. We extracted RNA from $60~\upmu\mathrm{L}$ homogenate, measured the RNA concentration, and calculated the volume of lysate equivalent to ${\sim}30$ ng RNA. We added this volume to four separate reactions consisting of 6 mL 1X polysome digestion buffer, and either 2.5, 12.5, 62.5 or $312.5\,\mathsf{U}/\mathsf{m L}$ Rnase I. We allowed digestions to proceed for 45 min at room temperature on a rocking platform covered in foil, added Superasin, and extracted footprints as described above.  

# Sorted CPN and SCPN  

We harvested IUE (CPN) or peduncle injected (SCPN) P3 brains into ice-cold HBSS $+100\,\upmu\mathrm{g/mL}$ cyclohexmide, dissected the labeled cortical hemispherea, and dissociated it for neuronal FACS using our established protocol except with $100\ \upmu\mathrm{g/mL}$ cycloheximide added to all solutions to keep translational elongation inhibited. We sorted fluorescent neurons into 2X polysome digestion buffer $\mathsf{+C H X}$ and diluted the sorted cells to a concentration of 12,500 cells/mL in 1X polysome digestion buffer following sorting. We set aside $\mathtt{1/5}$ of the sorted cells from each sample for RNA extraction, and preparation of the alkaline fragmented control libraries described below. We digested all samples with $12.5\,\mathsf{U}/\mathsf{m L}$ Rnase I for 45 min at room temperature on a foil-covered rocking platform. We then added Superasin, and extracted footprints as described above.  

# Sorted CPN from individual cortical hemispheres  

We labeled CPN via in utero electroporation of myristoylated-TdTomato plasmids into E15.5 embryos. We dissected P4 brains into ice-cold HBSS $+100\,\upmu\mathrm{g/mL}$ cyclohexmide, keeping each individual brain as a separate sample, dissected each labeled cortical hemisphere, and dissociated it for neuronal FACS using our established protocol except with $100\ \upmu\mathrm{g/mL}$ cycloheximide added to all solutions to keep translational elongation inhibited. We sorted red fluorescent neurons into 2X polysome digestion buffe $\mathsf{+C H X}$ , and diluted the sorted cells to a concentration of 12,500 cells/mL in 1X polysome digestion buffer following sorting. We set aside 1/5 of the sorted cells from each sample for RNA extraction, and preparation of the alkaline fragmented control libraries described below. We digested all samples with $12.5\,\mathsf{U}/\mathsf{m L}$ Rnase I for $45\,\mathsf{m i n}$ at room temperature on a foil-covered rocking platform. We then added Superasin, and extracted footprints as described above.  

# HEK293T  

We prepared biological replicates from independent HEK293T cultures. We diluted 1K, 10K, 50K HEK cells to 12,500 cell/mL into 1X polysome digestion buffer, and added $12.5\,\mathsf{U}/\mathsf{m L}$ Rnase I. We digested for $45\,\mathsf{m i n}$ at room temperature, and then added Superasin. For the 10K and 50K samples, we extracted footprints as above, with concentration via centrifugal filtration followed by Zymo RNA purification. The 1K sample is already in a small volume, so we simply omitted centrifugal filtration, and directly entered RNA purification.  

# Fragmentation of total RNA for control libraries  

# Rnase I digestion  

$G->R$ method. We digested ${\sim}30$ ng purified forebrain RNA in $60\,\upmu\mathrm{L}$ 1X polysome buffer with $1~\upmu\mathrm{L}$ RNaseIn and 0.16 U RNase I for 45 min at room temperature, then purified the RNA by first adding $90~\upmu\mathrm{L}$ lysis buffer $+650~\upmu\mathrm{L}$ $100\%$ EtOH, then following the Zymo Quick RNA Microprep protocol. We then gel purified the fragments between 20 and 60 nt as described for the ribosome footprints. We then carried out rRNA depletion with the NEBNext rRNA Depletion Kit (Human/Mouse/Rat NEB #E6310S) using the manufacturer’s instructions, however, we purified RNA at the end of the procedure using a Zymo Quick RNA Microprep cleanup (adding $90~\upmu\mathrm{L}$ Lysis buffe $\cdot{+}650~\upmu\mathsf{L}~100\%$ EtOH to the final reaction, then following the standard protocol thereafter) to purify the small RNAs at the last step of the NEB protocol instead of the suggested bead-based RNA cleanup protocol listed in the protocol. $R->G$ method. We digested and purified ${\sim}30$ ng forebrain lysate RNA as described in the ‘ $\mathsf{\Delta G}->\mathsf{R}^{\ast}$ method. After digestion, we then carried out rRNA depletion with the NEBNext rRNA Depletion Kit as described above. We ran the purified RNAs after rRNA depletion out on a $15\%$ TBE-Urea PAGE gel, and purified fragments from 27- to 60-nt using the previously-described gel purification procedure used to isolate ribosome footprints.  

# Alkaline fragmentation  

We performed the alkaline fragmentation time titration experiments using RNA extracted from whole dissociated P3 cortex from CD-1 mice. For one reaction, we mixed 7.5 ng RNA in $10~\upmu\mathrm{L}$ nuclease-free water with $10~\upmu\mathrm{L}$ bicarbonate fragmentation buffer (12 mM $\mathsf{N a}_{2}\mathsf{C O}_{3}$ ,$88\,\mathsf{m M N a H C O}_{3}$ ,$\mathsf{p H}\!=\!9.2\!;$ ) in a PCR strip, heated at $95\mathrm{^\circC}$ for either 0, 5, 10, 15, 20, 25, 30, 35, or 40 min, and stopped the reaction by placing on ice, and adding $20\,\upmu\mathrm{L}\,2\mathrm{X}$ formamide denaturing solution, and directly loading onto a $15\%$ TBE-UREA PAGE gel immediately. We found that timepoints between 35 and 40 min produce the size distribution with the most RNA between ${\sim}20.$ - to  

60-nt, in line with the input size range for the QiA-Seq miRNA Library Kit. We repeated the fragmentation for 37 min using 7.5 ng P3 cortex RNA, then gel purified fragments between ${\sim}27-$ to 60-nt for library preparation.  

To generate the alkaline fragmented libraries paired with Ribo-seq libraries from single cortical hemispheres, we set aside 1/5 of the material after sorting, extracted RNA using the Zymo Quick RNA Microprep protocol, and fragmented the eluted RNA by adding one volume of bicarbonate fragmentation buffer, heating for 37 min at $95\mathrm{^\circC}$ , then gel purifying the fragmented RNA as described above.  

# Oligo length standard sequences  

27 nt standard: AGUCACUUAGCGAUGUACACUGACUGA/3Phos/ 36 nt standard: AUGUACACGGAGUCGAGCACCCGCAACGCGACUGUA/3Phos/  

# Library preparation  

To ensure proper $5^{\prime}\mathsf{P}$ and $3^{\prime}01$ end moieties on the RNA fragments, we treated the RNA with polynucleotide kinase (PNK). We added $5~\upmu\mathrm{L}$ 1X T4 Ligase Buffer (NEB #B0202S), $0.5~\upmu\mathrm{L}$ T4 PNK (NEB #M0201S), $0.2~\upmu\mathrm{L}$ RNaseIn and $25~\upmu\mathrm{L}$ Nuclease-Free $\mathsf{H}_{2}\mathsf{O}$ to RNA samples, and incubated at $37^{\circ}\mathrm{C}$ for $30\,\mathsf{m i n}$ . We then added $80\,\upmu\mathrm{L}$ Zymo Lysis Buffer, $650\,\upmu\mathrm{L}$ $100\%$ EtOH, and purified the RNA using the standard Zymo Quick RNA Microprep protocol, eluting in $14\,\upmu\mathrm{L}$ . To estimate [RNA] and confirm purification of small RNAs, we ran the RNA on an RNA Pico Bioanalyzer Chip. We then used a SpeedVac to remove excess volume until RNA samples were under $5\,\upmu\mathrm{L}$ .We prepared sequencing libraries using the QiA-seq miRNA Library Kit protocol as described, and we used the following dilutions of adapters and RT primers: $_{3^{\prime}}$ adapter was diluted 1:5, $5^{\prime}$ adapter was diluted 1:2.5, RT primer was diluted 1:5. We amplified all libraries for 21 PCR cycles, and estimated library concentrations and size distributions with either the High-Sensitivity DNA or DNA Tapestation kits (we initially used the High-Sensitivity kit, but found that the library concentrations were often above the linear range of that kit).  

# Sequencing  

We pooled samples based on the library concentration estimates from the Tapestation runs. The Harvard Bauer Core performed final library quantification using KAPA-qPCR, and sequenced the libraries using either NextSeq or NovaSeq Illumina chemistry.  

# QUANTIFICATION AND STATISTICAL ANALYSIS  

# Overview  

Computational analyses were developed and performed to evaluate the following three Ribo-seq quality control metrics: 1) enrichment for CDS; 2) distinct length distributions over CDS compared to non-coding RNA; 3) enrichment for P-sites in-frame with annotated ORFs (Figure S1), as well as to 4) calculate translation efficiency; 5) identify genes displaying significantly skewed T.E. (Figure 6); and 6) investigate a potentially broad set of both annotated and novel ORFs (Figure 7). Importantly, we leveraged the low input library preparation approach using the QiA-seq miRNA kit to identify and remove PCR duplicates prior to analysis.  

To accomplish these goals, we began by extracting the random barcode, which comprises the first 12-nt of Read 2 using umi-tools. We then aligned reads to rRNA using STAR, and filtered out the rRNA-aligned reads, as well as reads too short for unique alignment (<21-nt). We aligned the remainder to the mm10 genome and transcriptome with STAR. We next calculated CDS enrichment, and length distributions over CDS and other non-coding RNA, using custom shell scripts. We analyzed P-site periodicity and P-site frame enrichment with the R-package RiboWaltz.41We used RiboCode50to identify all translated ORFs, including novel, unannotated ORFs, such as uORFs,50 and we performed a gene ontology analysis to find gene sets enriched among uORF containing genes using the R-package TopGO. We calculated T.E.s by summing up read counts over CDS in RP datasets, and over entire genes for AF datasets for each gene using subRead:featureCounts, then performed filtering, normalization, and analysis of differential expression (between AF and RP datasets) with DESeq2.  

We have created a Github accession (https://github.com/6LayeredCortex/nanoRibo-Seq) containing all pipelines used for this study. We performed all read processing, duplicate removal, rRNA and mm10 genome and transcriptomes alignment, length distributions, raw counting, RiboCode translated ORF identification using a Snakemake pipeline run as a batch job using Harvard University’s Cannon cluster. We performed RiboWaltz P-site periodicity analysis, TPM calculations, sample clustering, comparison, and sensitivity analysis, as well as uORF TopGo analysis using an Rmarkdown pipeline also run as a batch job (and as the final step in the Snakemake pipeline) on Harvard’s Cannon cluster. We include these Snakemake and Rmarkdown files, all needed inputs such as GTF files and Fasta files, and all custom UNIX shell scripts in the Github accession as full documentation of the computational methods used in this study.  

# Pre-proccessing and alignment pipeline  

To enable duplicate removal from these variable-length Ribo-seq sequences, we took advantage of the fact that we used paired-end sequencing, and that the first 12-nt of Read 2 of QiA-seq miRNA libraries is a 12-nt random barcode. We filtered for all reads with Read $2>12$ reads (the vast majority), and extracted this random barcode using umi-tools:  

umi_tools extract –extract-method $=$ string –bc-pattern $=$ NNNNNNNNNNNN.  

To remove the QiASeq miRNA adapters from the $3^{\prime}$ ends of Read 1, we ran Cutadapt v2.8.: cutadapt -a AACTGTAGGCACCATCAAT.  

We aligned Read 1 to the reference mouse rRNA sequences using STAR v2.6:  

STAR –runMode alignReads –runThreadN 8 –outSAMtype BAM Unsorted –readFilesCommand gunzip -c –outReadsUnmapped Fastx –outSAMmultNmax 1 –outFilterScoreMinOverLread 0 –outFilterMatchNminOverLread 0 –outFilterMatchNmin 21.  

We then used the \*final.out files produced by STAR for accounting the number and fraction of reads aligned to rRNA. We aligned the reads that failed to align to rRNA to the mm10 genome, and to the mm10 transcriptome for analysis of footprint length distribution, coverage over CDS, and producing gene-level raw counts. We aligned to the mm10 genome (GRCm38 build) using STAR $\times2.6$ and the following parameters:  

STAR –sjdbGTFfile Mus_musculus.GRCm38.95_chrNamed_headFix.gtf –runMode alignReads –runThreadN 8 –outSAMtype BAM SortedByCoordinate –quantMode TranscriptomeSAM –readFilesCommand gunzip -c –outSAMmultNmax 1 –outFilterScore MinOverLread 0 –outFilterMatchNminOverLread 0 –outFilterMatchNmin 21).  

We completed duplicate removal following mm10 alignment to collapse all reads with the same alignment coordinates and exactly matching barcode sequences using umi-tools: umi_tools dedup –method $=$ unique.  

We used the duplicate-removed bamFiles produced by this alignment for visualization, and for all further analysis. We then obtained raw, gene level counts over UTRs, CDS, and non-CDS using subRead:featureCounts, according to the annotations in Mus_musculus.GRCm38.95.gtf. We calculated coverage over $5^{\prime}$ ,$3^{\prime}$ UTRs and CDS as reads per kilobase per million mapped. We used custom shell scripts (makeLengthDistrosOverFeatures.sh, makeLengthDistrosOverNonTrans.sh) to compute the length distributions of the aligned reads from the lengths of aligned sequences.  

# Analysis of P-site periodicity with RiboWaltz  

We used the R-package RiboWaltz41to infer P-sites by first inferring the optimal offsets from the $5^{\prime}$ and $3^{\prime}$ ends of reads to the start codon, then using these offset estimates to define P-site coordinates for all reads mapping to coding transcripts, then testing whether these P-site coordinates show 3-nt periodicity and/or preference for the predominant reading frame. RiboWaltz requires alignments to the transcriptome as input, so we used STAR $\times2.6$ to align the non-ribosomal Read 1 reads to the transcripts annotated in Mus_musculus.GRCm38.95.gtf:  

STAR –runMode alignReads –runThreadN 8 –outSAMtype BAM SortedByCoordinate –genomeDir \$index –readFilesIn \$fastq1–sjdbGTFfile \$gtf –readFilesCommand gunzip –c –outFileNamePrefix \$prefix –outSAMmultNmax 1 –quantMode TranscriptomeSAM –outFilterScoreMinOverLread 0 –outFilterMatchNminOverLread 0 –outFilterMatchNmin 21.  

We next used the RiboWaltz package to perform P-site analysis. We loaded the transcriptomes’ alignment and annotation using the RiboWaltz functions ‘‘bamtolist()’’ and ‘‘create_annotation()’’. We then filtered for reads between 27:33 using the RiboWaltz ‘‘length_filter()’’ function. Next, we inferred P-site offsets for each sample using the ‘‘psite()’’ function, and estimated P-site coordinates of each read using the ‘‘psite_info()’’ function. To plot heatmaps of P-site signal, we ran the ‘‘reads_psite_list()’’ function, extracted estimates of P-site density at each position, and normalized the signals for depth by dividing the total number of CDS-aligned reads. To plot the percentage of P-sites in each frame, we first tabulated the numbers of reads in each frame with the ‘‘frame_psite()’’ function, then plotted the percentages.  

# Comparison with other methods in HEK293  

We calculated nanoRibo-seq QC metrics and P-site periodicity using our preprocessing, alignment, duplicate removal, and read counting pipeline, and P-site periodicity analysis with RiboWaltz. We counted reads per gene in our HEK293T nanoRibo-seq datasets using subread:featureCounts. We downloaded count matrices from Low Input RNase Footprinting and bulk Ribo-seq datasets from GEO. We filtered for genes with ${>}10$ reads in at least 3 of the 12 datasets in the combined set, and calculated TPM values $\mathsf{n}=10250$ genes). We then plotted heatmaps where each element is the correlation between $\mathsf{l o g}_{10}(\mathsf{T P M}{+}1)$ ) values for all genes between the pair of samples.  

# nanoRibo-seq in individual hemispheres  

We generated nanoRibo-seq RP and AF datasets from CPN sorted from 5 individual IUE labeled P4 cortical hemispheres (2.5–20.8K cells). We calculated nanoRibo-seq QC metrics and P-site periodicity using our preprocessing, alignment, duplicate removal, and read counting pipeline, and P-site periodicity analysis with RiboWaltz. We plot the distribution of raw read counts for all genes for all samples. We then filtered for genes with ${>}10$ reads in $8/10$ samples, and generated $\mathsf{l o g}_{10}(\mathsf{T P M}{+}1)$ correlation heatmaps with hierarchical clustering.  

# Comparing translation efficiency between CPN and SCPN  

We counted reads per gene in our CPN and SCPN nanoRibo-seq AF and RP datasets using subread:featureCounts. We then filtered for genes with ${>}10$ reads in at least 4 of 16 samples, calculated TPM values, and plotted heatmaps where each element is the correlation between $\mathsf{l o g}_{10}(\mathsf{T P M}{+}1)$ values for all genes between the pair of samples $\mathrm{~n~}=9057$ genes). We then identified differentially expressed genes, genes with ‘‘skewed’’ T.E. (i.e., significantly higher or lower RPF counts than mRNA abundance), and differentially translated genes (genes with significantly altered T.E. between CPN and SCPN) using a DESeq2 interaction model: 'exp $^{1+}$ subtype $^+$ exp:subtype. In this interaction model, the significance of the ‘‘exp’’ term tests T.E. skew, the significance of the ‘‘subtype’’ term tests differential expression between subtypes, and the significance of the ‘‘exp:subtype’’ interaction term tests differential translation (adjusted p value cutoff for all comparisons: 0.1).  

Given the focused set of differentially translated genes and the high correlation between T.E. in CPN and SCPN, we focused on the sets of genes that show skewed T.E. We performed a gene ontology analysis for skewed T.E. genes in each subtype using the enrichGO function in the clusterProfiler Bioconductor package, and plot p values, and gene counts for the top-5 terms (all ontologies) enriched among either T.E. higher or lower genes in each subtype.  

# Simulation and precision/recall analysis  

RNA-seq data can be well-approximated, and thus simulated, using a negative binomial distribution. We measured, for each gene in the CPN and SCPN datasets, its mean and dispersion from the parameters of the DESeq2 model, using the observed values for AF and RP datasets separately from each other. We then passed the parameters for each gene to the negative binomial distribution, and randomly drew 8 AF and 8 RP values, and randomly assigned 4 of the values to two arbitrary conditions, n1 or n2, for each experiment type. We next injected transcriptional change into a subset of either 800 or 4000 genes. We drew fold changes from a wide gamma distribution (shape $=2$ , rate $=1.5$ , minimum change $=0.25$ ), which approximates the range of expression fold changes observed in our datasets), and multiplied the counts in one condition for AF and RP experiments by the selected fold change (so that the ratios of counts between conditions n1 and n2 is now equal to the selected fold change). We next drew fold changes for differential translation from a narrower gamma distribution (shape $=1$ , rate $=1.5$ , minimum change $=0.25$ ), and multiplied only RP values by the selected T.E. fold change (so that the ‘‘ratio of ratios’’ corresponding to RP/AF counts in n2 vs. RP/AF counts in n1, equivalent to differential T.E. between conditions, is now equal to the selected fold change). Importantly, we selected differentially translated genes to either have ‘‘pure’’ translation change (no injected transcriptional change), ‘‘buffering’’ translational change (differential translation opposing differential expression), or ‘‘intensifying’’ translational change (differential translation in the same direction as differential selection). We selected 40 differentially translated genes in each of these three categories.  

We then tested precision and recall using three approaches 1) a DESeq2 interaction model 2) anota2seq 3) xtail. For DESeq2, we used an 'exp+condition $^+$ exp:condition interaction model for DESeq2, and default parameters. For xtail, we used parameters: bins $=$ 10000, $\mathsf{C}\mathsf{i}=0.90$ . For anota2seq, we used parameters: "minSlopeTranslation $"=-5$ , "maxSlopeTranslation" $=5$ , "minSlopeBuffering" $=$ -5, "maxSlopeBuffering" $=5$ ,$"m a x P A d j"=0.1$ . We then compared the lists of differentially translated genes identified by DESeq2, anota2seq, and xtail with the lists of ‘‘true positives’’ (genes with injected fold change in the simulations), and generated precision vs. recall curves for each method.  

# Identification of translated ORFs with RiboCode  

We extracted the transcripts encoded by the nuclear genome in Mus_musculus.GRCm38.95.gtf (creating GTF_nucGenome.gtf), and assigned the sequence names of the mm10_genome to match those of the nuclear chromosomes (creating mm10_genome.fasta) in the nuclear gtf file using custom unix scripts. We next prepared the transcripts for RiboCode analysis by running:  

prepare_transcripts $^{-9}$ GTF_nucGenome.gtf -f mm10_genome.fasta -o RiboCode_annot.gtf.  

We used RiboCode to produce ‘‘metaplots’’, which are used to infer the optimal P-site offsets for reads of a given length based on the distribution of $5^{\prime}$ and $3^{\prime}$ end distances from start and stop codons (very similar to how RiboWaltz infers P-sites). From this point on, we performed all steps for CPN and SCPN separately. We ran (ex: for the CPN datasets):  

metaplots -m 27 -M 36 –a RiboCode_annot.gtf -i CPN.dedup.bams  

to produce metaplots_pre_config.txt (this file contains the pairing of BAM file, read length, and optimal P-site offsets). We pooled together all CPN or SCPN datasets and ran RiboCode to identify translated ORFs:  

RiboCode –a RiboCode_annot.gtf –c metaplots_pre_config.txt -A CTG,GTG,TTG -l yes -b –min-AA-length 6 -o "RiboCode_ORFs".  

This produces calls for translated ORFs both within each sample, and within a ‘‘collapsed’’ set of translated ORFs, which is the union of all translated ORFs across all samples. We took this ‘‘collapsed’’ dataset, and used it to analyze ORF length and start codon distributions using a custom R-script. To plot ORF density across a transcript, we used RiboCode to make the plot:  

plot_orf_density –a RiboCode_annot.gtf -c metaplots_pre_config.txt -t \$txID -s \$ORFstart -e \$ORFend -o \$outName  

in which $\mathbb{S}{\sf t x}|\sf D\$ is the transcript ID, $\mathfrak{s}$ ORFstart and \$ORFend are the start and end coordinates of the predicted ORF (with respect to the transcript), and $\mathfrak{S}$ outName is the output file.  

# Analysis of translated uORF-containing mRNAs  

We focus on the set of uORFs and Overlap_uORFs identified as translated by RiboCode in both CPN and SCPN. We classified all genes based on whether they contain a translated uORF, Overlap_uORF, or no uORF, and compared T.E. distributions for these three sets in CPN and SCPN, and evaluated the significance of shifts in the T.E. distribution using the Wilcoxon ranksum test (significance threshold: $p<0.1$ ). We then performed a gene ontology analysis using the enrichGO function within the clusterProfiler Bioconductor package to identify terms overrepresented among genes with both uORFs and Overlap_uORFs in their $5^{\prime}$ UTRs, and we plot the top-5 enriched terms (all ontologies). We further focused on the impact of uORF translation on genes annotated within the terms ‘‘Axon Development’’ and ‘‘Synapse Organization’’. We plot the T.E. distributions in CPN and SCPN for genes in these terms with, and without, translated uORFs, and evaluate the significance of shifts in T.E. distribution between genes with and without uORFs within these terms using the Wilcoxon rank-sum test (significance threshold: $\mathsf{p}<0.1$ ). Finally, we provide several examples of translated uORFs in key genes. We plot RP, AF counts for the gene of interest in CPN and SCPN; the slope of the lines represents the RP/AF ratio, equivalent to T.E. for that gene in each subtype. We also plot read counts in each frame in each subtype for the uORFs in the $5^{\prime}$ UTRs of the genes of interest (frame 0 represents the reading frame of the uORF).  