Opinion  

# Natural uORF variation in plants  

Jiangen Wang,1,3 Juhong Liu,2,3 and Zilong Guo 1,\* Taking advantage of natural variation promotes our understanding of phenotypic diversity and trait evolution, ultimately accelerating plant breeding, in which the identification of causal variations is critical. To date, sequence variations in the coding region and transcription level polymorphisms caused by variations in the promoter have been prioritized. An upstream open reading frame (uORF) in the $5^{\prime}$ untranslated region ( $"5"$ UTR) regulates gene expression at the posttranscription or translation level. In recent years, studies have demonstrated that natural uORF variations shape phenotypic diversity. This opinion article highlights recent researches and speculates on future directions for natural uORF variation in plants.  

# Natural variation in plants  

Natural variation in a plant species is derived from spontaneous mutations in its wild progenitors, usually harboring signatures generated by long-term natural and artificial selection [1]. Studying natural variation provides insights into the genetic basis of trait variation and evolution, thereby accelerating knowledge-driven plant breeding [2]. Determination of causal variations of cloned genes in mapped genetic loci underlying qualitative and quantitative traits enables an understanding of natural variation at nucleotide resolution [3]. To date, sequence variations in the coding region and transcription level polymorphisms caused by variations in the promoter have been prioritized in the search for candidate genes and causal variants [2,4–6]. However, a large number of genetic mapping hits in plants are enriched in UTRs, which have elements affecting the gene translation efficiency or stability of messenger RNAs (mRNAs) [7]. In addition, the translation of mRNA into protein is an essential process for gene function according to the central dogma (see Glossary) of biology; only low or moderate correlations between mRNA levels and protein abundance and the modulation of gene expression via translational regulation are observed in plants [8,9]. These findings suggest an overlooked genetic source of phenotypic diversity: translational or post-transcriptional polymorphisms caused by UTR variations.  

# Introduction to uORFs and their regulatory activity  

A primary regulatory element of gene expression at the translation or post-transcription level is the uORF, which lies upstream of the main open reading frame (mORF) in $18–57\%$ of plant mRNAs [10]. The prevalence of uORFs might be underestimated because of the existence of non-AUG uORFs and incomplete annotation of $5^{\prime}$ UTRs. Recent advances in translatomic and proteomic profiling techniques, such as ribosome profiling and mass spectrometry, support the translation of uORF at the genome-wide level [11]. Experimental studies using mutagenesis analyses [12–14], luciferase-based reporter assays [8,14–16], and CRISPR-based editing [17,18] have demonstrated the functions of uORFs and their biological relevance. uORFs function either (i) by affecting (usually repressing) translation initiation of the mORF and/or inducing nonsense-mediated decay (NMD) in a cis-acting manner [8,19] or (ii) by encoding bioactive peptides with distinct biological functions beyond regulating mORF translation [20], of which (i) is the main means [21]. Although only a few uORFs enhance mORF translation [12–14], they broaden the scope of uORF activities (Box 1). According to the location of a uORF’s stop  

# Highlights  

Upstream open reading frames (uORFs) regulate gene expression by modulating (mostly repressing) main open reading frame (mORF) translation.  

The regulatory activity of uORFs is affected by their sequence features and varying environments.  

Polymorphisms at the translation level caused by uORF variations can underlie phenotypic diversity in plants.  

uORFs are selected during evolution in eukaryotes and might contribute to local adaptation, domestication, and improvement in plants.  

Evolutionary conservation analyses, development of Ribo-seq technologies, utilization of pan-genome or superpangenome resources, improvement of bioinformatics tools and databases to predict functional uORFs and the effects of uORF variants, phenomicsand tranlatomics-based genetic mapping, and CRISPR-based uORF engineering will collectively accelerate the application of natural uORF variations in plant breeding.  

codon, uORFs are classified into three types: (i) nonoverlapping uORFs, (ii) out-of-frame overlapping uORFs, and (iii) in-frame overlapping uORFs [10] (Figure 1A). The type 1-uORF is the most prevalent, accounting for $90.79\%$ and $87.63\%$ in arabidopsis (Arabidopsis thaliana) and rice, respectively [10] and shows three possible regulatory scenarios: leaky scanning, re-initiation, and stalling (Figure 1B); the type 2-uORF exerts the most repressive effects on mORF translation, and the type 3-uORF results in N-terminal extension of the mORF-encoded protein. In addition to regulating the translation efficiency of an mORF, a uORF could mediate the translation of an internal overlapping small ORF (smORF) instead of an mORF through delayed assembly of preinitiation complex (PIC) caused by the uORF, producing a smORF protein with a distinct function [22] (Figure 1C).  

Alteration of uORF features and varying environments/tissues could affect a uORF’s regulatory activities [23]. Here, we summarize nine important uORF features (Figure 1D), including the distance between the $5^{\prime}$ Cap and the uORF start codon, the RNA structure around the uORF, the Kozak context, the presence of rare triplet codons, the peptide charge, the strength of the stop codon, the length of the uORF length, the number of uORFs, and the distance between the uORF and the mORF [14,21,24–30]. uORF features have been investigated mostly in yeast and vertebrate species, but rarely in plants. Systematic studies of uORF natural variation followed by gene-by-gene uORF investigations might accelerate the dissection of uORF features in plants.  

# Box 1. Recent issues of uORF activity  

# Enhancer uORFs  

In addition to the repressive effects of uORFs, enhancer uORFs, which promote mORF translation, have also been discovered [12–14]. For instance, GmPHF1-uORF is required for the downstream GmPHF1 translation in soybean, such that a truncated uORF caused by a naturally occurring SNP results in lower GmPHF1 abundance, and artificial disruption of the uORF start codon led to undetectable GmPHF1 abundance [12]. We summarize four possible mechanisms of enhancer uORFs [14,89,90], which require further investigation and confirmation (Figure I).  

# uORF activity under the Brownian ratchet scanning model  

In addition to the strict 5′-3′ unidirectional scanning model of the pre-initiation complex (PIC), the Brownian ratchet model is supported by several studies [91,92]. In the model, the PIC scans an mRNA in both 5′-3′ and 3′-5′ directions locally and moves along the mRNA in the 5′-3′ direction globally [93]. For two AUG start codons of an mRNA, the downstream one could not affect the upstream one in the 5′-3′ unidirectional scanning model, whereas mutual competition of the two start codons would occur in the Brownian ratchet model, especially for two proximal $(<17$ nt) and out-of-frame start codons [93]. We speculated that the repressive effects on mORF translation exerted by out-of-frame overlapping uORFs could be compromised when two start codons are proximal.  

# Mechanisms of environment-responsive uORF activities  

It is reported that varying environments/tissues could alter a uORF’s regulatory activities (Figure II). Firstly, stress conditions promote the phosphorylation of the α subunit of eIF2 (eIF2α-P), which inhibits translation factor eIF2B (converting eIF2- GDP to eIF2-GTP) and thus decreases the abundance of eIF2-GTP and ternary complex (TC, eIF2–GTP–Met-tRNAi ) [23,94]. Low TC availability attenuates translation initiation efficiency of uORFs. The mechanism is exemplified by TBF1- mediated growth-to-defense in arabidopsis [32]. While stress conditions attenuate eIF2-dependent translation initiation of canonical uORFs mentioned previously, eIF2A-mediated translation initiation of a non-AUG uORF (with CUG/UUG start codons) is upregulated under stress conditions [38]. Besides, varying tRNA levels in different tissues/cell types alter the codon translation speed of a uORF, especially for rare codons [14]. In addition, to maintain metabolic homeostasis, uORF activity can be modulated at the metabolic levels through interactions between metabolites and the uORF-encoded peptide [39] or translation factor (such as eIF5A) [40]. In addition, uORF activity can be affected by stress-induced alteration at transcription level, such as stress-induced alternative transcription start site usage (aTSS) and alternative splicing (AS). These two mechanisms might result in the presence/absence of uORFs, as exemplified by blue light-triggered aTSS of HY5 [34] and light-activated AS of PIF3 in arabidopsis [33]. Genetic screens using a uORF-FLUC reporter identified two translational regulators (CDC123 and HEM1) in arabidopsis, which interact or condense with the translation machinery during effector-triggered immunity (ETI) [15,16]. These translational regulators might modulate the activity of stress-responsive uORFs [15].  

# Glossary  

Alternative splicing (AS): a process that creates multiple mRNA isoforms derived from a precursor mRNA through different splicing events.  

Alternative transcription start site usage (aTSS): a process that selects different TSS to initiate RNA synthesis and produces different mRNA isoforms.  

Central dogma: a theory showing the flow of genetic information and the processes of gene expression, at least including transcription from DNA to RNA, and translation from RNA to protein.  

Core genome: a pan-genome component that is shared among all accessions of a species.  

Dispensable genome: a pan-genome component that is specifically harbored by some accessions of a species.  

Genome-wide association study (GWAS): a genetic mapping approach mostly in natural diversity panels which links genome-wide DNA sequence variants to phenotypic or molecular diversity.  

Kozak context: sequence similarity to the Kozak consensus sequence, which can be recognized for translation initiation.  

Main open reading frame (mORF): coding sequence of an mRNA which is annotated to encode gene product in reference genome, also named as primary ORF or annotated ORF.  

Multiparent intercross populations: an advanced mapping population created by extensively intercrossing multiple accessions.  

non-AUG uORFs: a uORF of which the start codon is not AUG.  

Nonsense-mediated mRNA decay (NMD): a translation-coupled mRNA decay mechanism by which mRNA is degraded when confronted with premature translation termination.  

#  

Open reading frame (ORF): a sequence of triplet codons which harbors a start codon and ends with a stop codon and encodes a peptide when recognized by translation machinery.  

Positive selection: the selection force to promote the spread of beneficial alleles during evolution.  

Purifying selection: the selection force to remove deleterious alleles during evolution.  

Quantitative trait locus (QTL): a genomic region controlling a quantitative trait with continuous variance, which can  

![](images/020c908d9ca096f1e222892e432db0a3ee28b4784ce0fa2a8c549c6c11c190dd.jpg)  

Figure I. Possible mechanisms of enhancer uORFs. (A) uORF’s translation could unwind the RNA structure in the 5′ UTR to facilitate the pre-initiation complex (PIC) approaching the mORF start codon [14]. (B) uORF translation could optimize the ribosome density along the mORF to avoid ribosome collisions and No-Go decay (a mechanism that degrades mRNAs with stacks of stalled ribosomes) [14]. (C) uORF translation might attract the translation machinery to the mRNA and in turn enhance mORF translation [90]. (D) For mRNA containing multiple uORFs, especially under stress conditions, the first uORF acts as an insulator, as in GCN4-uORF1 [89], promotes 40S small ribosome subunits to bypass the downstream more repressive uORFs because of the low re-initiation efficiency of downstream uORFs under stress conditions, and thus enhances mORF translation. This mechanism might explain the phenomenon that more than half of out-of-frame overlapping uORFs are accompanied by nonoverlapping uORFs in plants [10]. The figure was created using Biorender.com. Abbreviations: mORF, main open reading frame; uORF, upstream open reading frame.  

![](images/a881066be82d164d142f5c1a45829a3e970f8569e088f4f720a6eb2443507ed5.jpg)  

Figure II. Mechanisms of the activity of environment-responsive uORFs. The underlying mechanisms include alternative transcription start site usage (aTSS) [34], phosphorylation of eIF2α (eIF2α-P) [32,94,95], elevated eIF2A abundance (to initiate non-AUG uORF translation and in turn enhance mORF translation) [38], tRNA level changes [14], alternative splicing (AS) [33], and interactions between metabolites and uORF-encoded peptides or translation factors [37,39–41], which can be triggered by external or internal signals. The figure was created using Biorender.com. Abbreviations: mORF, main open reading frame; uORF, upstream open reading frame.  

be identified by linkage analysis or association mapping.  

Ribosome profiling: a powerful tool of quantifying genome-wide translation levels using deep sequencing of ribosome-protected mRNA fragments, also named as Ribo-seq.  

40S small ribosome subunit: the small subunit of a eukaryotic ribosome with 40S sedimentation coefficient during centrifugation, which participates in start codon scanning and ORF translation.  

Structure variation: variation that alters genomic region of approximately 1 kb and larger in size, such as presence/absence variation (PAV) and copy number variation (CNV).  

Upstream open reading frame (uORF): an ORF of which the start codon lies upstream of the mORF in an mRNA.  

![](images/d047dc9ec6865d9503a12a720fac9262071be8bd41462dc273ce30054d324942.jpg)  

Figure 1. uORF regulatory activity. (A) According to positional relationship between the uORF stop codon and the mORF, uORFs are classified into three types: $\mathrm{(\ddot{l})}$ nonoverlapping uORFs; (ii) out-of-frame overlapping uORFs, which partially overlap the mORF and are out of frame with the mORF; and (iii) in-frame overlapping uORFs, which share the mORF’s stop codon. $\mathtt{(B)}$ For the nonoverlapping uORFs, the most prevalent uORF type, three major scenarios occur: $(\mathrm{j})$ leaky scanning, in which uORF evades being scanned by the pre-initiation complex (PIC), and thus, mORF translation is not affected; 40S small ribosome subunits acquire the ternary complex (TC, eIF2–GTP–Met-tRNAi ) to form PIC, and eIF2-GDP is converted to eIF2-GTP by eIF2B; (ii) translation re-initiation, in which after uORF translation, mRNA-bound 40S small ribosome subunits reacquire the TC to assemble the PIC to scan the mORF start codon, and mORF translation would be initiated but attenuated; and (iii) ribosome stalling, in which the ribosome stalls at the uORF stop codon or a specific motif and no mORF translation occurs. $(\mathrm{C})$ A uORF could mediate the translation of an internal overlapping small ORF (smORF) instead of an mORF through delayed assembly of PIC caused by the uORF, producing a smORF protein with a distinct function [22]. (D) $+,-,\,\?$ in parenthesis indicates positive, negative, and uncertain correlations between a uORF feature and uORF regulatory strength, respectively. Favorable Kozak context [27], rare triplet codon [14], and a strong stop codon [24] are beneficial for a uORF’s repressive activity. A less stable RNA structure around a uORF start codon [28], a more negatively charged uORF-encoded peptide (with unknown mechanisms) [26], and a shorter distance between the uORF and the mORF enhance the activity of the uORF [26,28,30]; a less stable RNA secondary structure facilitates uORF translation initiation [28]; a shorter distance between the uORF and the mORF might result in less time for mRNA-bound 40S small ribosomal subunits to reacquire the TC for PIC assembly and repress mORF translation [26]. For the distance between the $5^{\prime}$ Cap and the uORF start codon [23,26,30], the uORF length [26,28,30], and the number of uORFs [9,29,89], discordant results are shown under varying conditions or in different studies. The figure was created using Biorender.com. Abbreviations: mORF, main open reading frame; uORF, upstream open reading frame.  

Research has identified stress-responsive relief of uORFs' repressive effects and tissue/cell typespecific uORF activity [30,31]. In plants, studies have demonstrated uORF-mediated growth-todefense transition upon pathogen invasion [32], uORF-mediated plant adaptation under varying light conditions [33–35], alleviation of uORF repressive effects upon low nutrient stress [36], and alteration of uORF activity under varying metabolite levels [37]. We summarize several underlying mechanisms: environment-induced phosphorylation of the α subunit of eIF2 (eIF2α-P) [32], elevated eIF2A abundance [38], changes in tRNA levels [14], the interaction between metabolite and uORF-encoded peptides or translation factors [37,39–41], alternative transcription start site usage (aTSS) [34], and alternative splicing (AS) [33] $(\mathrm{BO}\times\,1\,]$ ). To date, studies have demonstrated that uORF-based gene regulation plays important role in plant metabolism [37,42–48], development [18,31,49–51], disease resistance [32,52], and nutrient uptake [12,13,19,53].  

uORF-based genetic engineering for trait improvement has been performed in plants in recent years, which involves uORF-cassette transgenes and CRISPR-based endogenous uORF editing. Taking advantage of the pathogen-responsive activities of uORFs, uORFTBF1 (a cassette consisting of the immune-inducible promoter and two pathogen-responsive uORFs of the TBF1 gene [32])-like cassettes are used to enhance plants' resistance to pathogen attack with minimum adverse effects on plant growth and development [54,55]. With the rapid development of CRISPR-based editing technologies, the editing of repressive uORFs has been successfully performed to produce transgene-free materials with enhanced gene expression and improved performance in plants [18,56].  

# Examples of natural uORF variation shaping phenotypic diversity  

Although uORFs are fine-tuning elements in plant metabolism and abiotic/biotic stress resistance, natural uORF variation and its contribution to phenotypic diversity remain elusive in plants. In 2020, Reis et al. identified an out-of-frame overlapping uORF in the $5^{\prime}$ UTR of PHO1 (PHOSPHATE1, encoding a root-to-shoot phosphate exporter) in arabidopsis, and demonstrated the uORF’s repressive effect on $P H O\uparrow$ translation and shoot $\mathsf{P i}$ accumulation [19]. To explore the natural variation of the uORF, a natural arabidopsis population (comprising 854 accessions) was screened, and $10\%$ (86/854) of the natural accessions harbored PHO1-uORF variants, of which eight accessions lacked the out-of-frame overlapping uORF (Figure 2A). To demonstrate the effect of uORF variation on phenotype, an $\mathsf{F}_{2}$ segregating population was produced by crossing a uORF-present accession and a uORF-absent accession, in which the co-segregation between uORF-presence/absence and low/high shoot Pi content was observed.  

Also in 2020, a candidate gene HvDEP1 (encoding a γ-subunit of the heterotrimeric G-protein complex) and its uORF (a nonoverlapping uORF) were found in a barley quantitative trait locus (QTL) controlling grain length [57]. A 9-bp insertion/deletion variation (INDEL) in the HvDEP1 uORF was associated with grain length in a double-haploid population and a diversity panel (Figure 2B). Although the association has not been verified, it inspired geneticists to propose that a natural uORF variant could underlie a QTL.  

A systematic investigation of proteomic data in a natural maize population provided statistical evidence for the contribution of natural uORF variants to protein abundance diversity [27]. Considering that rare alleles generally have larger effect sizes and more disruptive effects in general, the study focused on the rare variants. The study observed that rare repressive alleles (which create or strengthen the uORF’s Kozak context) tended to decrease the protein abundance of the downstream mORF, whereas the rare derepressive alleles (which disrupt or weaken the Kozak context of pre-existing uORFs) had the opposite effect (Figure 2C). Besides, the allelic effects on protein abundance were true for common uORF variants, and these common variants were enriched in genome-wide association study (GWAS) hits of metabolic and physiological traits. The study provided no empirical evidence; however, the ‘uORF variation - protein translation - phenotype diversity’ thread became clear and just awaits proof of concept.  

![](images/124cf21d79b6cf150c14f9054eb75d4cc740fa841cfa1b0526acc8131c35468f.jpg)  
Figure 2. Natural uORF variation in plants. (A) INDEL-mediated uORF-type alteration (type $^{2\rightarrow}$ type 3) for PHO1, which is experimentally supported for its contribution to shoot Pi content diversity in arabidopsis (Arabidopsis thaliana) [19]. (B) INDELmediated change in the uORF length for HvDEP1, which might contribute to grain length diversity in barley [57]. (C) SNPmediated Kozak context change, which is statistically supported for its contribution to protein abundance diversity in maize [27]. (D) SNP-mediated uORF premature termination for $G m P H F7$ , which is experimentally supported for its contribution to phosphorus acquisition efficiency diversity in soybean [12]. (E) SNP-mediated creation of an upstream start codon and an out-of-frame overlapping uORF for PELAN, which is experimentally supported for its contribution to flower color diversity in monkeyflower [58]. (F) Transposon-mediated creation of a uORF for ZmSWEET4c, which is experimentally supported for its contribution to kernel size diversity in maize [59]. (G) Presence/absence variation of a uORF caused by alternative splicing (AS), which is implied in plants [10,60] and experimentally supported in humans [61]. (H) SNP-mediated stop codon disruption/premature termination of a uORF leading to uORF-type alteration (type $^{1\leftrightarrow}$ type 2 or type 3), which is implied in plants [10]. The figure was created using Biorender.com. Abbreviations: mORF, main open reading frame; uORF, upstream open reading frame.  

In addition, GWAS of phosphorus acquisition in a soybean diversity panel identified a candidate gene, GmPHF1, which encodes phosphate transporter traffic facilitator 1, a SEC12-like protein that facilitates the exit of phosphate transporters from the endoplasmic reticulum [12]. In search of causal variants of GmPHF1, neither amino acid changes nor transcription level polymorphisms were found. The study fully verified the gene's function in phosphorus acquisition using genetic transformations; however, GmPHF1 could not be determined as the causal gene provided that its causal variants failed to be discovered. Further investigation of GmPHF1-based association mapping revealed two significant single-nucleotide polymorphisms (SNPs) in the $5^{\prime}$ UTR. A nonoverlapping uORF was identified in the $5^{\prime}$ UTR and one significant SNP introduced premature stop codon (TCA $\rightarrow$ TAA) that truncated the uORF from 39 to 27 amino acids (Figure 2D). Using molecular biotechnology and genetic transformation, the causal SNP was demonstrated to alter protein translation of the downstream mORF and the mRNA steady state in a uORFdependent manner and to underlie the observed phenotypic diversity.  

A recent genetic study of monkeyflower provided another solid support [58]. PELAN (PETAL LOBE ANTHOCYANIN), encoding an anthocyanin-activating R2R3 MYB, was identified in a QTL for flower color in monkeyflower species using fine scale genetic mapping in $\mathsf{F}_{2}$ and backcross populations and was supported by transgenic experiments. Neither the transcription level nor protein function of PELAN was responsible for the QTL. However, a SNP in the $5^{\prime}$ UTR was observed to create a start codon (AGG $\longrightarrow$ ATG) and thus produced an out-of-frame overlapping uORF (Figure 2E). The uORF represses PELAN translation, resulting in lower anthocyanin levels, and transforms the flower color from dark pink to pale pink, which would change pollinator preference from hummingbird pollination to self-pollination in phenotypic evolution.  

In 2023, a study in maize showed that a natural allele of ZmSWEET4c harbors a uORF introduced by the insertion of a transposon [59] (Figure 2F). The gene encodes a glucose and fructose transporter and was mapped using bulk segregant RNA sequencing (BSR-seq) of a large $\mathsf{F}_{2}$ population. The transposon insertion represses ZmSWEET4c translation partially by introducing a uORF, resulting in a small kernel size. The study revealed transposon-derived uORF and a new uORF variation type in plants. Taken together, the aforementioned studies suggest a new genetic source of phenotypic diversity in nature and the coming age of natural uORF variation in plants.  

To date, only a few experimental studies of natural uORF variation have been reported in plants; however, diverse natural uORF variations have been implied in plant uORF databases and population-scale studies [10]. Here, to provide a comprehensive viewpoint of natural uORF variation in plants, we analyzed the demonstrated and implied uORF variation types in plants, which were assigned into three categories: uORF presence/absence, uORF-type alteration, and uORF feature alteration.  

For uORF presence/absence variation (PAV), the creation or ablation of a uORF start codon caused by SNP-mediated nucleotide substitution or by transposon insertion polymorphisms has been demonstrated in plants, as mentioned previously (Figure 2F). Although not demonstrated, uORF presence/absence caused by AS is implied in plants (Figure 2G). Natural variation in AS among different individuals is prevalent, and its underlying QTLs have been identified in plants [60]. uORF PAV among different splicing models of a gene has been discovered in plants [10]. Furthermore, it has been demonstrated that a SNP at the splicing donor site mediates AS variation and thus causes uORF PAV in human disease-related genes [61].  

As mentioned previously, three uORF types are determined based on the position relationship between the uORF stop codon and mORF. Mutual conversions between nonoverlapping uORFs (type 1) and overlapping uORFs (types 2 and 3), which are caused by SNP/INDELmediated stop codon disruption or premature termination of an existing uORF, have been discovered in arabidopsis and rice [10] (Figure 2H). In addition, conversion between type 2 and type 3 caused by INDEL-mediated frameshift has been demonstrated in plants (Figure 2A).  

The features of uORFs affect their regulatory activity. In principle, any natural variant that alters the key uORF features could affect mORF translation and contribute to phenotypic diversity. The associations between SNP-mediated uORF Kozak sequence variation and the protein abundance from the mORF have been supported at the genome scale in a maize population [27] (Figure 2C). SNP-mediated alteration of stop codon strength (stronger to weaker: $\mathsf{U A A}>\mathsf{U A G}>\mathsf{U G A})$ of uORFs has not been demonstrated or implied in plants; however, it merits investigation given that its role has been demonstrated in human studies [24]. Alteration of the distance between a uORF and an mORF (without altering the uORF type), caused by SNP-mediated premature termination or stop codon disruption, has been demonstrated experimentally in soybean [12] (Figure 2D), and an altered uORF length caused by an in-frame INDEL has been discovered in barely [57] (Figure 2B). A change to the sequence of the uORF-encoded peptide caused by a SNP or an out-of-frame INDEL (without altering the uORF type) has been observed in plants; however, its genetic effect remains to be investigated [10].  

While most attention has been paid to natural variation in the coding sequence or promoter region, the aforementioned natural variation of uORFs should be considered in search for candidate genes and causal variants.  

# Evolution of uORFs  

Studying natural uORF variation will not only reveal the genetic mechanism of trait diversity mentioned previously but also provide insights into uORF evolution [62]. While uORF evolution is rarely studied in plants, the principles have been systematically studied based on genome sequences of a variety of species [63] or population-scale genotypic data in humans [25]. While uORFs are generally deleterious, and newly formed uORFs tend to be under purifying selection, a number of uORFs could exist that are favored and fixed by positive selection [30,63]. Upon fixation, uORFs are maintained by selection and show high conservation levels of their start codon AUG (uAUG) and stop codon [63]. uORF coding regions are generally under neutral evolution and show less conservation than uAUG and uORF stop codon, which is consistent with the notion that uORFs function mainly by regulating mORF translation rather than encoding functional peptides [24]. However, a small fraction of uORFs harbor conserved sequences among different species, suggesting their potential functional importance [64]. The conservation of amino acid sequences might be used as a feature to explore functional uORFs, especially for non-AUG uORFs [63]. In addition, codon optimality to ensure translation elongation of uORFs might constrain variation in uORF coding regions and thus coding region is conferred modest conservation [24]. The conservation patterns of uORF coding regions could be leveraged to identify critical codons for uORF repressive activities. As expected, uORF variants that create or disrupt uAUG, introduce new stop codons, or increase stop codon strength for existing uORFs, are under strong purifying selection and are potentially associated with diseases in human populations [24,25].  

Considering the conservation of uORF activities and sequence features in different species [28], we propose that these findings from nonplant species can provide valuable implications for the plant community. As mentioned previously, the uAUG variant of PELAN is responsible for the evolution of flower coloration in monkeyflower species, a critical determinant of pollinator preference in the natural habitat, suggesting that uORFs contribute to trait evolution in plants [58]. We speculate that if an mORF-encoded protein regulates a trait that is the target of local adaptation, domestication, and improvement, the corresponding uORF might underlie the evolutionary processes and might undergo divergent selection under varying environments or different human needs in plants. In the near future, this speculation could be tested by genome-wide scanning of selection signals using wild progenitors, landraces, and cultivars of a species, and genetic mapping of the traits that have undergone selection, based on uORF variants.  

# Current approaches and future research  

Identifying functional uORFs will pave the way for an in-depth understanding of uORF variants. Although non-AUG uORFs have limited repressive activities, they are prevalent and might have distinct functions [63]. For example, the eIF2A-dependent translation of non-AUG uORFs induces mORF translation under stress conditions [65]. However, it is a challenge to identify functional non-AUG uORFs partially because of the lack of definitive characteristics of start codons. Combining Ribo-seq and evolutionary conservation analyses might accelerate the exploration of these hidden, but critical, elements. As mentioned previously, selection force to maintain sequence conservation of uORFs is weak [24]. Despite being rare, a subset of conserved peptide uORFs (CPuORFs) exist, including non-AUG CPuORFs [66]. CPuORFs might have conserved functions and their roles have been partially demonstrated in stress-responsive translational regulation in plants [64]. It is speculated that variations in CPuORF coding regions could largely alter their activity and might affect biological processes in plants. A variety of bioinformatic tools and databases to search for CPuORFs have been reported [10,67], which are useful to identify potential functional uORFs and predict causal variants.  

Although Ribo-seq is an efficient tool to identify translated uORFs [29], it is insufficient to provide the complete landscape of functional uORFs based on a few Ribo-seq resources, because uORFs are translated and function in a tissue-, condition-, and developmental stage-specific manner [23,30]. With the increasing availability of Ribo-seq data in plants, combining resources from multiple experiments under different conditions in different plant species after correcting for batch and sampling effects will facilitate the identification of functional uORFs at spatial and temporal resolution, the exploration of the housekeeping uORFs, and an understanding of uORF’s function. Furthermore, single-cell Ribo-seq technologies have been developed and applied in mouse [68,69], of which the latest spatially resolved single-cell translatomics technology, RIBOmap [70], paves the way for identifying uORFs of cell type and spatial specificity in plants.  

Until now, almost all the uORF studies have relied on a single reference genome for a given species, which could result in missing uORFs that are not present in the reference genome and the lack of uORF variations caused by structural variations. Species-level pan-genomes harboring the relatively complete genetic repertoire of a species have been successfully established in plants [71–75], and super-pangenomes integrating multiple pan-genomes of different species and their wild relatives for a given genus have been developed in a few plant genera [76–78]. It is speculated that the establishment of pan-genome or super-pangenome will accelerate the identification of core uORFs (from the core genome) and dispensable uORFs (which are from the dispensable genome, and might be related to stress resistance, and could have been lost during domestication or improvement processes [79]); the exploration of uORF variations; and the comparison of uORFs among different species of a genus or different genera.  

Natural plant populations contain a number of variants in the $5^{\prime}$ UTRs of genes; however, bioinformatics tools to annotate the effects of $5^{\prime}$ UTR variants are limited because most tools focus on coding variants. Among the existing $5^{\prime}$ UTR variant annotation tools, UTRannotator is a relatively comprehensive tool [80], which predicts the creation/disruption of uORF start and stop codons and frameshifts of a canonical uORF. In the future, a predictor of the effects of uORF variants should consider variation of non-AUG uORFs, more types of uORF variation (such as uORF-type alteration), and the combined effects of multiple variants (Figure 3).  

Given that GWAS is a promising approach to link genomic variants to phenotypic diversity [2] and that plant phenomic platforms have developed rapidly in recent years [81,82], GWAS between predicted uORF-perturbing variants and plant phenomic data will accelerate our understanding of the contribution of natural uORF variation to phenotypic diversity and candidate gene identifi- cation (Figure 3). Furthermore, comparative and parallel GWAS in multiple plant species is expected to reveal uORF-mediated conservation of the genetic architecture of important traits [83] (Figure 3). In addition to GWAS in natural populations to detect common uORF variants, it is necessary to perform GWAS in multiparent intercross populations [such as multiparent advanced generation intercross (MAGIC) [84] and complete-diallel plus unbalanced breedingFigure 3. Future studies for natural uORF variation in plants. In the future, pan-genomes or super-pangenomes can be leveraged to identify the core and dispensable uORFs, to explore more uORF variations, and to compare uORFs among different species or genera. A predictor to annotate the effects of uORF variants with high accuracy is needed. These variants could be incorporated to perform GWAS of plant phenomic data (outer track of the Circos plot) or translatome profiling (inner track of the Circos plot) in a diversity panel (for common variants) or in a multiparent intercross mapping population (for rare variants), to systematically reveal the contribution of natural uORF variation to phenotypic diversity and/or translational polymorphisms in plants. Comparative and parallel studies across multiple species are expected to reveal uORF-mediated conserved genetic architecture of important traits and convergent selection in trait evolution. A multi-species uORF variation database integrating multiple layers of population-scale data is needed. CRISPR-based uORF engineering can be guided by natural uORF variation and in turn will broaden our understanding of natural uORF variation. In the outer track of the Circos plot, different shapes and colors indicate different trait ontologies and traits from plant phenomics, respectively. The figure was created using Biorender.com. Abbreviations: GWAS, genome-wide association study; uORF, upstream open reading frame.  

![](images/672b126df961f90491a442ac3eba24a46d630e3506dbe5fcfd3e0a5b646570f9.jpg)  

derived intercross (CUBIC) populations [85]] to detect rare uORF variants (Figure 3). Following GWAS, fluorescent-based report assays, CRISPR-based editing, and genetic transformation are needed to confirm the direct effect of uORF variants on plant traits.  

GWAS of population-scale RNA-seq data can reveal promoter-mediated transcription level polymorphisms [86], and similarly, GWAS of translatome profiling data is expected to reveal uORFmediated translation efficiency polymorphisms (Figure 3). Among translatome profiling techniques, ribosome profiling (Ribo-seq) has been widely adopted in plants because of its high accuracy and resolution; however, its high cost limits its application in population-scale studies. Recently, a newly developed technique, $3^{\prime}$ ribosome-profiling sequencing (3′Ribo-seq), has shown similar profiling efficiency to conventional Ribo-seq but at a lower cost; indeed, GWAS of 3′Ribo-seq data has been performed in a maize diversity panel [87]. We propose that GWAS of high-throughput translatome profiling data and phenome data will provide translational insights into the genetic architecture of important traits.  

As the first plant uORF variation database, uORFlight provides information on natural uORF variation and its associated phenotypic traits based on public population-scale genetic data in arabidopsis and rice [10]. The database also shows uORF variation among different splicing models for ten plant species. In the future, a multi-species uORF variation database integrating multiple layers of population-scale data is required to facilitate comparative and parallel studies of natural uORF variation in a range of species (Figure 3).  

CRISPR-based editing has been successfully applied to uORF engineering in plants. In the future, not only can uORF engineering be guided by natural uORF variation to improve traits, but also novel, non-natural uORF variation could be created, which would broaden our understanding of uORF variation and generate diverse alleles to improve complex quantitative traits [17] (Figure 3). As a proof of concept, a suite of genome-edited rice plants with graded phenotypic levels of plant heights and tiller numbers was developed by using CRISPR-based design of uORF variants with varying regulatory activities [88], indicating that uORF engineering is a promising approach for precise breeding in the near future.  

Are uORFs enriched in the core or the dispensable genome for a given species?  

Outstanding questions What are the characteristics and mechanisms of enhancer uORFs?  

Can we find convincing evidences that uORFs undergo selection during local adaptation, domestication, and improvement in plants?  

How can we reduce the cost of Riboseq for population-scale studies?  

How can we correct batch and sampling effects when combining Ribo-seq resources from multiple experiments?  

How can we predict functional nonAUG uORFs and their causal variants with high accuracy?  

# Concluding remarks  

To completely decode the genetic basis of natural variation in plants, attention should be paid to the long-neglected uORF variation. Several questions remain to be answered in the future (see Outstanding questions). Understanding of uORFs’ regulatory activities, the exploration of natural uORF variation, and CRISPR-based uORF engineering will mutually reinforce and collectively accelerate the dissection of the uORF-mediated genetic basis of phenotypic diversity and trait evolution, ultimately promoting the use of uORFs in plant breeding (Figure 3).  

# Acknowledgments  

This work is supported by the National Natural Science Foundation of China (grant no. 31902098 to Z.G.) and the Program for Outstanding Youth Scientific Research of Fujian Agriculture and Forestry University (no. 201914 to Z.G.).  

# Declaration of interests  

No interests are declared.  

# References  

1. Huang, $\times.$ and Han, B. (2014) Natural variations and genomewide association studies in crop plants. Annu. Rev. Plant Biol. 65, 531–551   
2. Liang, Y. et al. (2021) Natural variation in crops: realized understanding, continuing promise. Annu. Rev. Plant Biol. 72, 357–385   
3. Wei, X. et al. (2021) A quantitative genomics map of rice provides genetic insights and guides breeding. Nat. Genet. 53, 243–253   
4. Yano, K. et al. (2016) Genome-wide association study using whole-genome sequencing rapidly identifies new genes influencing agronomic traits in rice. Nat. Genet. 48, 927–934 5. Gaut, B.S. et al. (2018) Demography and its effects on genomic variation in crop domestication. Nat. Plants 4, 512–520   
6. Chen, Q. et al. (2021) Harnessing knowledge from maize and rice domestication for new crop breeding. Mol. Plant 14, 9–26   
7. Wallace, J.G. et al. (2014) Association mapping across numerous traits reveals patterns of functional variation in maize. PLoS Genet. 10, e1004845   
8. Wu, H.-W. et al. (2022) Noise reduction by upstream open reading frames. Nat. Plants 8, 474–480   
9. Xiong, Q. et al. (2020) Ribosome profiling reveals the effects of nitrogen application translational regulation of yield recovery after abrupt drought-flood alternation in rice. Plant Physiol. Biochem. 155, 42–58   
10. Niu, R. et al. (2020) uORFlight: a vehicle toward uORF-mediated translational regulation mechanisms in eukaryotes. Database 2020, baaa007   
11. Chen, Y. et al. (2020) PsORF: a database of small ORFs in plants. Plant Biotechnol. J. 18, 2158   
12. Guo, Z. et al. (2022) A natural uORF variant confers phosphorus acquisition diversity in soybean. Nat. Commun. 13, 3796   
13. Yang, S.-Y. et al. (2020) Upstream open reading frame and phosphate-regulated expression of rice OsNLA1 controls phosphate transport and reproduction. Plant Physiol. 182, 393–407   
14. Lin, Y. et al. (2019) Impacts of uORF codon identity and position on translation regulation. Nucleic Acids Res. 47, 9358–9367   
15. Zhou, Y. et al. (2023) Plant HEM1 specifies a condensation domain to control immune gene translation. Nat. Plants 1–13   
16. Chen, T. et al. (2023) Global translational induction during NLRmediated immunity in plants is dynamically regulated by CDC123, an ATP-sensitive protein. Cell Host Microbe 31, 334–342.e5   
17. Xing, S. et al. (2020) Fine-tuning sugar content in strawberry. Genome Biol. 21, 1–14   
18. Zhang, H. et al. (2018) Genome editing of upstream open reading frames enables translational control in plants. Nat. Biotechnol. 36, 894–898   
19. Reis, R.S. et al. (2020) Modulation of shoot phosphate level and growth by PHOSPHATE1 upstream open reading frame. Plant Physiol. 183, 1145–1156   
20. Jayaram, D.R. et al. (2021) Unraveling the hidden role of a uORFencoded peptide as a kinase inhibitor of PKCs. Proc. Natl. Acad. Sci. 118, e2018899118   
21. Zhang, T. et al. (2020) uORFs: important cis-regulatory elements in plants. Int. J. Mol. Sci. 21, 6238   
22. Inchingolo, M.A. et al. (2023) TP53BP1, a dual-coding gene, uses promoter switching and translational reinitiation to express a smORF protein. iScience 26, 106757   
23. Zhang, H. et al. (2019) Function and evolution of upstream ORFs in eukaryotes. Trends Biochem. Sci. 44, 782–794   
24. Lee, D.S. et al. (2021) Disrupting upstream translation in mRNAs is associated with human disease. Nat. Commun. 12, 1515   
25. Whiffin, N. et al. (2020) Characterising the loss-of-function impact of 5’untranslated region variants in 15,708 individuals. Nat. Commun. 11, 2523   
26. May, G. et al. (2023) Unraveling the influences of sequence and position on yeast uORF activity using massively parallel reporter systems and machine learning. eLife 12, e69611   
27. Gage, J.L. et al. (2022) Variation in upstream open reading frames contributes to allelic diversity in maize protein abundance. Proc. Natl. Acad. Sci. 119, e2112516119   
28. Chew, G.-L. et al. (2016) Conservation of uORF repressiveness and sequence features in mouse, human and zebrafish. Nat. Commun. 7, 11663   
29. Zhu, X.-T. et al. (2022) Ribosome profiling reveals the translational landscape and allele-specific translational efficiency in rice. Plant Commun. 4, 100457   
30. Zhang, H. et al. (2018) Genome-wide maps of ribosomal occupancy provide insights into adaptive evolution and regulatory roles of uORFs during Drosophila development. PLoS Biol. 16, e2003903   
31. Ribone, P.A. et al. (2017) A uORF represses the transcription factor AtHB1 in aerial tissues to avoid a deleterious phenotype. Plant Physiol. 175, 1238–1253   
32. Pajerowska-Mukhtar, K.M. et al. (2012) The HSF-like transcription factor TBF1 is a major molecular switch for plant growthto-defense transition. Curr. Biol. 22, 103–112   
33. Dong, J. et al. (2019) Phytochrome B induces intron retention and translational inhibition of PHYTOCHROME-INTERACTING FACTOR 3. Plant Physiol. 182, 159–166   
34. Kurihara, Y. et al. (2018) Transcripts from downstream alternative transcription start sites evade uORF-mediated inhibition of gene expression in Arabidopsis. Proc. Natl. Acad. Sci. 115, 7831–7836   
35. Gregory, B.D. (2018) Shedding some blue light on alternative promoter usage in plants. Proc. Natl. Acad. Sci. 115, 7654–7656   
36. Tanaka, M. et al. (2016) The minimum open reading frame, AUGstop, induces boron-dependent ribosome stalling and mRNA degradation. Plant Cell 28, 2830–2849   
37. Laing, W.A. et al. (2015) An upstream open reading frame is essential for feedback regulation of ascorbate biosynthesis in Arabidopsis. Plant Cell 27, 772–786   
38. Kearse, M.G. and Wilusz, J.E. (2017) Non-AUG translation: a new start for protein synthesis in eukaryotes. Genes Dev. 31, 1717–1731   
39. Wu, C. et al. (2012) Arginine changes the conformation of the arginine attenuator peptide relative to the ribosome tunnel. J. Mol. Biol. 416, 518–533   
40. Ivanov, I.P. et al. (2018) Polyamine control of translation elongation regulates start site selection on antizyme inhibitor mRNA via ribosome queuing. Mol. Cell 70, 254–264.e256   
41. Salazar-Díaz, K. et al. (2021) TOR senses and regulates spermidine metabolism during seedling establishment and growth in maize and Arabidopsis. iScience 24, 103260   
42. Alatorre-Cobos, F. et al. (2012) Translational regulation of Arabidopsis XIPOTL1 is modulated by phosphocholine levels via the phylogenetically conserved upstream open reading frame 30. J. Exp. Bot. 63, 5203–5221   
43. Cruz-Ramiérez, A. et al. (2004) The xipotl mutant of Arabidopsis reveals a critical role for phospholipid metabolism in root system development and epidermal cell integrity. Plant Cell 16, 2020–2034   
44. Salazar-Díaz, $\mathsf{K}.$ et al. (2021) TOR senses and regulates spermidine metabolism during seedling establishment and growth in maize and Arabidopsis. iScience 24, 103260   
45. Guerrero-González, M.L. et al. (2014) uORF, a regulatory mechanism of the Arabidopsis polyamine oxidase 2. Mol. Biol. Rep. 41, 2427–2443   
46. de la Luz Guerrero-González, M. et al. (2016) Arabidopsis polyamine oxidase-2 uORF is required for downstream translational regulation. Plant Physiol. Biochem. 108, 381–390   
47. Hanfrey, C. et al. (2005) A dual upstream open reading framebased autoregulatory circuit controlling polyamine-responsive translation. J. Biol. Chem. 280, 39229–39237   
48. Thalor, S.K. et al. (2012) Deregulation of sucrose-controlled translation of a bZIP-type transcription factor results in sucrose accumulation in leaves. PLoS One 7, e33111   
49. Dong, J. et al. (2020) Phytochrome B induces intron retention and translational inhibition of PHYTOCHROME-INTERACTING FACTOR3. Plant Physiol. 182, 159–166   
50. Rosado, A. et al. (2012) Arabidopsis ribosomal proteins control developmental programs through translational regulation of auxin response factors. Proc. Natl. Acad. Sci. 109, 19537–19544   
51. Nishimura, T. et al. (2005) The Arabidopsis STV1 protein, responsible for translation reinitiation, is required for auxinmediated gynoecium patterning. Plant Cell 17, 2940–2953   
52. Ai, G. et al. (2022) Making use of plant uORFs to control transgene translation in response to pathogen attack. BioDesign Res. 2022, 2022   
53. Tanaka, M. et al. (2011) Boron-dependent degradation of NIP5; 1 mRNA for acclimation to excess boron conditions in Arabidopsis. Plant Cell 23, 3547–3559   
54. Xu, G. et al. (2017) uORF-mediated translation allows engineered plant disease resistance without fitness costs. Nature 545, 491–494   
55. Kim, J.H. et al. (2022) Increasing the resilience of plant immunity to a warming climate. Nature 607, 339–344   
56. Si, X. et al. (2020) Manipulating gene translation in plants by CRISPR–Cas9-mediated genome editing of upstream open reading frames. Nat. Protoc. 15, 338–363   
57. Watt, C. et al. (2020) A novel polymorphism in the $5^{\prime}$ UTR of HvDEP1 is associated with grain length and 1000-grain weight in barley (Hordeum vulgare). Crop Pasture Sci. 71, 752–759   
58. Liang, M. et al. (2022) Lost in translation: molecular basis of reduced flower coloration in a self-pollinated monkeyflower (Mimulus) species. Sci. Adv. 8, eabo1113   
59. Chen, G. et al. (2023) A novel active transposon creates allelic variation through altered translation rate to influence protein abundance. Nucleic Acids Res. 51, 595–609   
60. Chen, Q. et al. (2018) Genome-wide association analyses reveal the importance of alternative splicing in diversifying gene function and regulating phenotypic variation in maize. Plant Cell 30, 1404–1423   
61. Wiestner, A. et al. (1998) An activating splice donor mutation in the thrombopoietin gene causes hereditary thrombocythaemia. Nat. Genet. 18, 49–52   
62. Mitchell-Olds, T. and Schmitt, J. (2006) Genetic mechanisms and evolutionary significance of natural variation in Arabidopsis. Nature 441, 947–952   
63. Zhang, H. et al. (2021) Determinants of genome-wide distribution and evolution of uORFs in eukaryotes. Nat. Commun. 12, 1076   
64. Causier, B. et al. (2022) Plants utilise ancient conserved peptide upstream open reading frames in stress-responsive translational regulation. Plant Cell Environ. 45, 1229–1241   
65. Sendoel, A. et al. (2017) Translation from unconventional 5′ start sites drives tumour initiation. Nature 541, 494–499   
66. Spealman, P. et al. (2018) Conserved non-AUG uORFs revealed by a novel regression analysis of ribosome profiling data. Genome Res. 28, 214–222   
67. Egorov, A.A. and Atkinson, G.C. (2023) uORF4u: a tool for annotation of conserved upstream open reading frames. Bioinformatics 39, btad323   
68. VanInsberghe, M. et al. (2021) Single-cell Ribo-seq reveals cell cycle-dependent translational pausing. Nature 597, 561–565   
69. Ozadam, H. et al. (2023) Single-cell quantification of ribosome occupancy in early mouse development. Nature 618, 1057–1064   
70. Zeng, H. et al. (2022) Spatially resolved single-cell translatomics at molecular resolution. Science 380, eadd3067   
71. Jayakodi, M. et al. (2020) The barley pan-genome reveals the hidden legacy of mutation breeding. Nature 588, 284–289   
72. He, Q. et al. (2023) A graph-based genome and pan-genome variation of the model plant Setaria. Nat. Genet. 55, 1232–1242   
73. Gao, L. et al. (2019) The tomato pan-genome uncovers new genes and a rare allele regulating fruit flavor. Nat. Genet. 51, 1044–1051   
74. Qin, P. et al. (2021) Pan-genome analysis of 33 genetically diverse rice accessions reveals hidden genomic variations. Cell 184, 3542–3558.e3516   
75. Liu, Y. et al. (2020) Pan-genome of wild and cultivated soybeans. Cell 182, 162–176.e113   
76. Li, N. et al. (2023) Super-pangenome analyses highlight genomic diversity and structural variation across wild and cultivated tomato species. Nat. Genet. 55, 852–860   
77. Leonard, A.S. et al. (2023) Graph construction method impacts variation representation and analyses in a bovine superpangenome. Genome Biol. 24, 124   
78. Shang, L. et al. (2022) A super pan-genomic landscape of rice. Cell Res. 32, 878–896   
79. Khan, A.W. et al. (2020) Super-pangenome by integrating the wild side of a species for accelerated crop improvement. Trends Plant Sci. 25, 148–158   
80. Zhang, X. et al. (2021) Annotating high-impact 5′ untranslated region variants with the UTRannotator. Bioinformatics 37, 1171–1173   
81. Yang, W. et al. (2014) Combining high-throughput phenotyping and genome-wide association studies to reveal natural genetic variation in rice. Nat. Commun. 5, 5087   
82. Guo, Z. et al. (2018) Genome-wide association studies of image traits reveal genetic architecture of drought resistance in rice. Mol. Plant 11, 789–805   
83. Chen, W. et al. (2016) Comparative and parallel genome-wide association studies for metabolic and agronomic traits in cereals. Nat. Commun. 7, 12767   
84. Dell’Acqua, M. et al. (2015) Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays. Genome Biol. 16, 1–23   
85. Liu, H.-J. et al. (2020) CUBIC: an atlas of genetic architecture promises directed maize improvement. Genome Biol. 21, 1–17   
86. Liu, S. et al. (2020) Mapping regulatory variants controlling gene expression in drought response and tolerance in maize. Genome Biol. 21, 1–22   
87. Zhu, W. et al. (2021) Large-scale translatome profiling annotates the functional genome and reveals the key role of genic $3^{\prime}$ untranslated regions in translatomic variation in plants. Plant Commun. 2, 100181   
88. Xue, C. et al. (2023) Tuning plant phenotypes by precise, graded downregulation of gene expression. Nat. Biotechnol. Published online March 9, 2023. https://doi.org/10.1038/s41587-023- 01707-w   
89. Hinnebusch, A.G. (2005) Translational regulation of GCN4 and the general amino acid control of yeast. Annu. Rev. Microbiol. 59, 407–450   
90. Waern, K. and Snyder, M. (2013) Extensive transcript diversity and novel upstream open reading frame regulation in yeast. $G3$ Genes Genomes Genet. 3, 343–352   
91. Alekhina, O.M. and Vassilenko, K.S. (2012) Translation initiation in eukaryotes: versatility of the scanning model. Biochem. Mosc. 77, 1465–1477   
92. Spirin, A.S. (2009) How does a scanning ribosomal particle move along the 5′-untranslated region of eukaryotic mRNA? Brownian Ratchet model. Biochemistry 48, 10688–10692   
93. Li, K. et al. (2022) Distance-dependent inhibition of translation initiation by downstream out-of-frame AUGs is consistent with a Brownian ratchet process of ribosome scanning. Genome Biol. 23, 1–37   
94. Young, S.K. and Wek, R.C. (2016) Upstream open reading frames differentially regulate gene-specific translation in the integrated stress response. J. Biol. Chem. 291, 16927–16935   
95. Baird, T.D. et al. (2014) Selective mRNA translation during eIF2 phosphorylation induces expression of IBTKα. Mol. Biol. Cell 25, 1686–1697  