# Neuronal activity rapidly reprograms dendritic translation via eIF4G2:uORF binding  

Received: 3 August 2023  

Accepted: 5 March 2024  

Published online: 8 April 2024  

Check for updates  

# Ezgi Hacisuleyman     , Caryn R. Hale , Natalie Noble, Ji-dung Luo    , John J. Fak, Misa Saito, Jin Chen , Jonathan S. Weissman        & Robert B. Darnell  

Learning and memory require activity-induced changes in dendritic translation, but which mRNAs are involved and how they are regulated are unclear. In this study, to monitor how depolarization impacts local dendritic biology, we employed a dendritically targeted proximity labeling approach followed by crosslinking immunoprecipitation, ribosome profiling and mass spectrometry. Depolarization of primary cortical neurons with KCl or the glutamate agonist DHPG caused rapid reprogramming of dendritic protein expression, where changes in dendritic mRNAs and proteins are weakly correlated. For a subset of pre-localized messages, depolarization increased the translation of upstream open reading frames (uORFs) and their downstream coding sequences, enabling localized production of proteins involved in long-term potentiation, cell signaling and energy metabolism. This activity-dependent translation was accompanied by the phosphorylation and recruitment of the non-canonical translation initiation factor eIF4G2, and the translated uORFs were sufficient to confer depolarization-induced, eIF4G2-dependent translational control. These studies uncovered an unanticipated mechanism by which activity-dependent uORF translational control by eIF4G2 couples activity to local dendritic remodeling.  

The complex and highly elongated structure of neurons renders subcellular regions subject to local demands. mRNA localization in neurites is thought to play a critical role in neuronal homeostasis and synaptic plasticity , and activity-dependent changes in translation are required to drive synaptic plasticity, learning and memory . However, with current molecular tools, the dynamics of synaptic metabolism and molecular plasticity are not fully understood.  

Imaging approaches established the groundwork for the discovery of mRNAs in neurites7,8. The subcellular localization and translation of these RNAs have been studied using mechanical separation methods9,10. Enzymatic tagging has recently enabled the cell-type-specific analysis of protein composition of subcellular compartments11–13. Studies in resting neurons revealed the unique biology of localized transcripts, their isoform specificity and $5^{\prime}$ and $3^{\prime}$ untranslated region (UTR) regulation by RNA-binding proteins (RBPs). Dendritically localized RNAs, for example, have longer $5^{\prime}\,{\mathrm{UTRS}}^{14}$ , which can enable complex translational programs by forming structural motifs and interacting with RBPs. Association of these RBPs with $5^{\prime}$ UTRs correlates with the translation dynamics of neuronal transcripts15. Sequence elements within the $5^{\prime}$ UTRs impact ribosome engagement16,17 and translation of the downstream coding sequence (CDS) in response to different stimuli and cellular states18–20. However, it is not known if and how $5^{\prime}$ UTRs or other mechanisms modulate CDS translation of dendritically localized messages.  

A longstanding goal in neuroscience is to understand how the localization of dendritic RNAs leads to protein synthesis-dependent synaptic plasticity. In the present study, we developed proximity-based labeling methods to simultaneously isolate dendritic RNAs and their bound regulatory proteins, along with dendritic ribosomes and proteins, to investigate how neuronal depolarization impacts molecular events in synapses. We localized a biotin ligase to postsynapses and analyzed local molecular changes within 20–30 min after depolarization with either glutamate receptor agonist DHPG or KCl. Surprisingly, depolarization induced the translation of many short upstream open reading frames (uORFs) and downstream CDSs in transcripts that were dormant in resting dendrites. Motif analysis and crosslinking immunoprecipitation (CLIP) revealed that the $5^{\prime}$ UTRs harboring these uORFs were bound by eIF4G2, and mutation studies uncovered that its binding was necessary and sufficient for the downstream CDS translation. Taken together, with these molecular tools, we were able to reveal previously unanticipated aspects of localized activity-dependent translational control with spatial and temporal resolution.  

# Results Dendritic TurboID identifies postsynaptic RNAs and proteins  

We used a bio-orthogonal and kinetically enhanced biotin ligase, TurboID21, to specifically biotinylate proteins within a $10\mathrm{-}50\;\mathrm{nm}$ radius. We engineered a tagged TurboID to confer dendritic localization by linking it to the elements of the transcript encoding the postsynaptic protein PSD95 (TurboID-PSD95) (Fig. 1a), expressed from a doxycycline-inducible lentiviral construct. First, we confirmed that our culture was devoid of contaminating non-neuronal cell types (Extended Data Fig. 1a). The palmitoylation signal and UTRs of PSD95 were necessary for the proper and punctate dendritic localization of TurboID-PSD95, consistent with previous studies22 (Extended Data Fig. 1b–d). The qualitative and quantitative immunofluorescence (IF) data confirmed that TurboID-PSD95 is localized to dendritic puncta despite some variation in the intensity of expression across neurons. In resting neurons, exogenous addition of biotin induced robust biotinylation and local labeling of nearby proteins in $30\,\mathrm{min}$ , as visualized by streptavidin pulldowns and IF (Fig. 1b,c and Extended Data Fig. 1e).  

To demonstrate the localization and specificity of TurboID-PSD95, we compared it to TurboID without a localization signal (Pan-TurboID; Extended Data Fig. 1e,f), which is similarly expressed and records information from the whole neuron. After addition of biotin, both Pan-TurboID and TurboID-PSD95 efficiently labeled nearby proteins, albeit with more background labeling with Pan-TurboID even in the absence of additional biotin (Fig. 1c and Extended Data Fig. 1e,g). Streptavidin pulldowns followed by western blots showed that TurboID-PSD95 specifically biotinylated and allowed the recovery of known dendritic binding partners of PSD95 but not non-localized proteins (Extended Data Fig. 1h) . Comparison of the enrichment of dendrite-localized and nuclear proteins isolated by TurboID-PSD95 revealed up to 30-fold purification of dendritic proteins compared to Pan-TurboID (Extended Data Fig. 1i). Taken together, these data demonstrate that proximity labeling with TurboID-PSD95 was highly efficient in identifying dendritic proteins in resting neurons.  

Activity-driven RNA localization and translation in dendrites is crucial for tuning the synaptic proteome and inducing long-lasting modifications of neuronal circuits; however, little is known about the dynamics of postsynaptic RNA regulation after neuronal activation. To bridge this gap, we used $\mathsf{K C l}^{24}$ (referred to as ‘dep’) or the specific glutamate agonist DHPG25 to acutely depolarize neurons (Fig. 1d,e). We optimized parameters based on the increase in the phosphorylation of known translational control and signal transduction pathway components, transcription of immediate early genes (IEGs) and dendritic spine lengthening while also allowing time for ample TurboID biotinylation (Extended Data Fig. 1j–l). Neuronal depolarization was assessed by monitoring increased intracellular calcium concentration, demonstrating activation of the majority $(-80\%)$ of neurons (Fig. 1f).  

Depolarization by KCl and DHPG resulted in similar biotinylation, phosphorylation, IEG expression and calcium influx (Fig. 1e and Extended Data Fig. 1j,k,m). We compared these activation paradigms with sodium arsenite, which specifically induces stress responses, and found that depolarization led to distinct changes (IEGs and P-EEF2) that distinguished it from a classical stress response (activation of IRE1, CHOP and ATF4) (Extended Data Fig. 1j,k). Based on these data, our strategy provides a means to measure specific molecular changes in dendrites in response to depolarization and to probe the unusual nature of these responses.  

To isolate local dendritic mRNAs enriched before and after depolarization, we extended previous CLIP studies14,26,27 by integrating TurboID proximity ligation (PL) and termed this strategy PL-CLIP. Biotin-labeled UV-crosslinked RNA–protein complexes in neurons expressing TurboID constructs were purified using streptavidin, and RNAs directly bound to the biotinylated proteome were purified. Crosslinked RNAs isolated by Pan-TurboID or TurboID-PSD95 were sequenced across four biological replicates in resting and KCl-depolarized neurons, which were deprived of contaminating non-neuronal cell types as assessed by IF and transcriptomic analyses (Extended Data Figs. 1a and 2a). Principal component analysis (PCA) revealed that depolarization state was the primary differentiator across replicates both for inputs and for pulldowns; however, TurboID (localization) was important only for pulldowns, indicating compartment-specific and activation-specific isolation (Extended Data Fig. 2b).  

The dendritic enrichment for each transcript was calculated by comparing RNAs crosslinked to TurboID-PSD95-labeled versus Pan-labeled proteins. The results revealed 2,788 and 3,727 transcripts enriched in resting and depolarized dendrites, respectively (Supplementary Table 1). PL-CLIP data from resting dendrites corresponded well with previously published datasets of neuropil transcripts (Fig. 2a)1,9,10,14,28–35. Moreover, previously established axonal transcripts36–39 were depleted or not significantly enriched in our dendritic transcriptome, indicating that PL-CLIP could distinguish subcellular transcriptomes with high specificity (Extended Data Fig. 2c). Comparing PL-CLIP to in vivo dendritic RNAs and FMRP RNA targets from mouse hippocampal CA1 neurons previously identified by our laboratory  revealed them to be significantly enriched in the dendritic transcriptome (Extended Data Fig. 2d). Although FMRP binds RNAs across the whole neuron14,40,41, we found only its postsynaptic targets to be enriched in our data, further supporting the overrepresentation of FMRP targets in dendritic RNAs and the ability of PL-CLIP to distinguish subcellular transcriptomes.  

To explore the roles encoded by dendritically enriched transcripts, we performed gene set enrichment analysis (GSEA) in resting dendrites. Gene sets associated with postsynaptic membrane, dendritic development and cytoskeleton were enriched, consistent with previous reports (Fig. 2b) . Several transcripts known to be localized in resting dendrites were enriched, including Shank1, Rpl24, Map1b and Syn1, along with previously known (Tamm41) and undiscovered (Uqcrc1) mitochondrial RNAs9,42 and others (Frmd6) with yet to be identified dendritic functions (Fig. 2c). Some of these new RNAs encoded for proteins associated with Golgi, protein complex localization and chromatin binding (Extended Data Fig. 2e). Together, these data demonstrate that PL-CLIP specifically enriches for dendrite-specific RNAs, including previously known and unknown transcripts.  

![](images/5c674db653af4216a397b12e87c34ec39a1de8adfb1b8ec892fe27791abc2922.jpg)  
Fig. 1 | TurboID is a robust methodology to isolate the molecular composition  levels in culture; then, activated with KCl (dep) for 1 h or by DGPH for $10\,\mathrm{min}$ ;of the postsynaptic compartment in primary cortical neurons. a, The and, finally, incubated with the same silencers and biotin to induce biotinylation implementation of TurboID in primary cortical neurons. b, Doxycycline (Dox) and allow for recovery $\mathtt{30\,m i n}$ for KCl and $20\,\mathrm{min}$ for DHPG). Colors: salmon induction of TurboID-PSD95 (Flag) and its biotinylation pattern (streptavidin) (resting); burgundy (depolarized by KCl); cyan (DHPG). e, Distribution of in the presence and absence of biotin shown by western blots. c, IF to detect streptavidin signal (cyan) in Pan-TurboID-transduced or TurboID-PSD95- TurboID expression and biotinylation in primary cortical neurons transduced transduced activated neurons. DAPI (blue) marker for nuclei. Magnification, with TurboID-PSD95 after $30\,\mathrm{min}$ of biotin incubation. DAPI (blue) marker for $\times20.\mathbf{f},$ , Fura-2 AM staining (left) and quantification (right) in resting and KClnuclei, Flag (red) stain for TurboID and streptavidin (cyan) to demarcate the depolarized neurons. Each circle represents information from one field (data biotinylated proteins. Magnification, $\times40$ . d, Diagram of neuronal activation are mean ± s.d., 15 fields total from three biological replicates). Significance workflow with TurboID labeling: neurons are first silenced with TTX (sodium was derived from the biological replicates and calculated using the two-tailed, channel blocker) and DL-AP5 (NMDA receptor antagonist) to standardize activity  unpaired Student’s t-test. Scale bars, $50\,\upmu\mathrm{m}$ .  

We next used PL-CLIP to identify dendritic RNAs after neuronal depolarization. RNAs encoding proteins involved in RNA processing, $3^{\prime}$ end RNA regulation, RNA transport and RNP granules were enriched in depolarized dendrites (Fig. 2d). The latter set was particularly intriguing because P-bodies and RNP granules disassemble upon synaptic stimulation . In contrast, we found that stress marker transcripts were decreased or unchanged in dendrites upon depolarization (Extended Data Fig. 2f), supporting the conclusion that response to depolarization was distinct from a classical stress response. Moreover, we examined the distribution of dendritic RNAs upon depolarization. Shank1, Rpl24, Map1b, Syn1, Tamm41, Uqcrc1 and Frmd6 showed changes in dendritic enrichment in response to depolarization. For example, Rpl24 and Uqcrc1 became de-enriched; Frmd6, however, became more localized in dendrites (Fig. 2c). FRMD6 regulates actin dynamics ; hence, its preferential localization in dendrites suggests that cytoskeletal processes are modified in response to neuronal activation.  

Our comparison of resting and depolarized transcriptomes revealed the properties of these dendritic transcripts. Overall, dendritic transcripts were longer in their CDSs and 5′ UTRs than the average lengths of all expressed transcripts, consistent with previous studies14 (Extended Data Fig. 3a). Although it was previously shown that transcripts enriched in neurites have longer $3^{\prime}\mathrm{UTRS}^{14}$ , we did not find this to be the case for PL-CLIP-enriched mRNAs in resting dendrites. This discrepancy might be explained by the specificity of our analysis, which is confined to interactions proximal to PSD95 in dendritic spines and not neurites. However, longer transcripts were enriched in dendrites after depolarization, and this difference in the length between resting and depolarized conditions was primarily driven by the $3^{\prime}$ UTRs (Extended Data Fig. 3a). Additionally, dendritic RNAs were richer in GC content in resting neurons, and this property switched upon activity (Extended Data Fig. 3b), potentially driven by significantly longer $3^{\prime}$ UTRs, which have lower GC content45. The presence of activity-related longer dendritic $3^{\prime}$ UTRs may reflect rapid recruitment of such transcripts to the PSD95 local subcompartment of the dendrite and allow synapses to decode genomic information by isoform selectivity. These $3^{\prime}$ UTRs might play diverse roles, ranging from regulation of local translation to recruitment of RBPs and generation of alternative protein-coding isoforms46–49.  

To independently validate PL-CLIP in resting and activated neurons, we used RNAscope fluorescence in situ hybridization (FISH). We verified several dendritically enriched (Kmt2d, Map2, Dlg4 and Ppp1r9b) and de-enriched (Snca and Rapgef4) RNAs that were identified in resting neurons in previous studies (Extended Data Fig. 3c,d). Upon activation, Ppp1r9b transcript levels decreased and were evenly distributed between soma and dendrites, whereas Rapgef4 levels increased but became predominantly dendritic (Extended Data Fig. 3d,e).  

To understand the extent to which the dendritically localized RNAs may determine local protein content, we performed mass spectrometry (MS) on the biotinylated proteome in resting and KCl-depolarized neurons. Because TurboID is a promiscuous biotin ligase, we used a ‘minus biotin’ condition for each sample, in which equivalent TurboID was expressed but no biotin was added, to subtract proteins labeled without temporal specificity (Fig. 2e). We defined dendritic localization of proteins by calculating their enrichment in TurboID-PSD95 over Pan-TurboID pulldowns after the ‘minus biotin’ counterpart of each was subtracted, and we termed this proximity ligation mass spectrometry (PL-MS) (Supplementary Table 2). All five replicates were consistent, with higher differential between the resting and depolarized conditions in the PSD95-enriched proteome than in the Pan-enriched proteome (Extended Data Fig. 4a).  

We compared our resting proteomic results with a similar high-throughput approach employing a kinetically less efficient version of TurboID, BirAR118G, to isolate dendritic proteins in the mouse brain13. We specifically detected the reported PSD95-BirA proteome to be enriched and the inhibitory (Gephyrin-BirA) proteome to be depleted in our PL-MS (Extended Data Fig. 4b). We also looked at another dataset that used synaptosome preparation50 and found that this postsynaptic proteome had the strongest agreement with our PL-MS-enriched proteome (Extended Data Fig. 4c).  

We then compared our PL-CLIP and PL-MS data and found that detected RNAs and proteins were highly correlated in resting dendrites (Extended Data Fig. 4d,e), but, surprisingly, this was not evident in depolarized conditions (Extended Data Fig. 4f).  

# Dendritic TurboID identifies postsynaptic translatome  

We were intrigued by the lack of correlation of PL-CLIP and PL-MS data in depolarized conditions and wanted to understand if rapid changes in local translation could shape the local proteome after depolarization. TurboID-PSD95 transduction of neurons resulted in biotinylation of dendritic proteins and ribosomes, as shown by western blots after streptavidin pulldowns (Fig. 3a), and IF of RPL10A, a component of the 60S ribosomal subunit, which co-localized with streptavidin (Fig. 3b). To monitor dendritic translation in the postsynaptic compartment, we combined TurboID labeling with ribosome profiling (deep sequencing of ribosome-protected mRNA fragments) . Specifically, we purified biotinylated ribosomes from Pan-TurboID-expressing and TurboID-PSD95-expressing cells and used sequencing to quantitate ribosome-protected mRNA footprints in the input and pulldown fractions (Fig. 3a and Extended Data Fig. 5a). We calculated the dendritic translation of mRNAs as the ratio of ribosome protected footprints mapping to those in the PSD95 pulldown compared to input samples, and we termed this approach proximity ligation ribosome sequencing (PL-Ribo-seq) (Supplementary Table 3). The ribosome profiling protocol was optimized to accommodate the low input nature of our samples, and we applied riboWaltz  and Plastid53 to assess quality control (Extended Data Fig. 5b). Three replicates were consistent across TurboID constructs and conditions (Extended Data Fig. 5c).  

![](images/a702b27762fa0c65b185e9f1ae2368143a00eb59f41a8508d8923b089012c9b8.jpg)  

![](images/106e78e280adfdaf7609595293660ecf359a23bdaf0a03308f71a5df7984d90b.jpg)  

![](images/8aa6590b37ccb95f9a521d1d6df79a673e6ad085c719df3c3e7e52f26ee686e6.jpg)  

![](images/5933f9dd7d79ef4aea14fafee6f4efff5a6edd79d261bf67b597d7508c06649b.jpg)  

To determine whether ribosomes were labeled in a compartmentspecific manner, the ribosome-protected fragments in pulldowns and inputs for Pan-TurboID and TurboID-PSD95 were analyzed. Although Pan-TurboID provides information from the whole neuron, TurboID-PSD95 targeted only dendritic ribosomes (Fig. 3c). PL-Ribo-seq agreed with previously published datasets on resting dendrite-specific but not axon-enriched translatomes (Extended Data Fig. 5d), allowing a comprehensive picture of translation specifically in the postsynaptic space.  

In resting dendrites, PL-Ribo-seq identified gene sets that were also found in PL-CLIP, with roles in postsynapse, glutamatergic synapse, microtubule cytoskeleton and endosome, and was depleted in ones encoding nucleosome, DNA packaging, mitochondrial translation and translational regulation-related proteins (Fig. 3d). Additionally, although immunity-related transcripts were present only in resting dendrites, they were substantially enriched in the resting dendritic translatome (Fig. 3d), which was largely unanticipated despite prior evidence of the importance of class I major histocompatibility complex (MHC) molecules in remodeling synaptic connections .  

KCl-depolarized dendrites showed altered translation of specific mRNAs with diverse functions. Sphk1, Rgs14 and Ghrl, with roles in G protein-coupled receptor signaling and postsynaptic chemical transmission; Apobec1 and Igf2bp1, with roles in regulating binding at $3^{\prime}$ UTRs; Pml and Eif1ad3, associated with translation initiation; and P2rx7 and Timm23, involved in ATP-coupled respiration and mitochondrial protein import, showed increased translation in dendrites in response to depolarization (Fig. 3e). These mRNAs were present in gene sets that were significantly overrepresented in the depolarized dendritic translatome (Fig. 3f).  

To further understand activity-dependent local translational control, we analyzed global changes in translation upon depolarization. We observed a substantial decrease in the number of ribosomes found on the CDS of dendritic transcripts (Extended Data Fig. 5e) as well as of pan-neuronal transcripts (detected by Pan-TurboID, in agreement with previous reports)56,57. We extended these observations by finding reduced puromycin incorporation, a means of measuring active translation, in depolarized neurons (Extended Data Fig. 5f). Translational downregulation during neuronal depolarization might allow the recovery of energy and resources and/or reversion to a re-polarized state.  

Fig. 3 | TurboID-mediated ribosome profiling reveals activity-dependent increase in ribosome occupancy and uORF usage in 5′ UTRs of dendritic mRNAs. a, Schematic for PL-Ribo-seq. TurboID-PSD95 labels dendritic ribosomes in primary cortical neurons. RPL10A western blot showing the input and streptavidin pulldown fractions from TurboID-PSD95-expressing neurons that are incubated with (+biotin) and without (−biotin) biotin for $30\,\mathrm{min}$ . Percentages refer to the volumetric percentage loaded on the gel. b, In TurboID-PSD95-transduced neurons, biotinylation (streptavidin, cyan) can be detected only in dendrites, even though RPL10A (red) stains across the whole neuron, as detected by IF. DAPI (blue) marker for nuclei. Magnification, $\times20,$ . Scale bar, $50\,\upmu\mathrm{m}$ . c, Streptavidin pulldown and input RPKM values $(\log_{2})$ for each gene are plotted for Pan-TurboID (left) and TurboID-PSD95 (right) PL-Ribo-seq. Pan-TurboID pulldown and input correlation indicates that Pan-TurboID represents information from the whole neuron. d, GSEA on dendritically translated RNAs by resting PL-Ribo-seq $\mathrm{FDR}<0.05)$ . Multiple testing correction was performed using the Benjamini–Hochberg method. e, Examples of RNAs that are translationally upregulated in dendrites in response to depolarization by PL-Ribo-seq ${}\langle n\!=\!3$ biologically independent samples, values from PL-Ribo-seq). Dendritic log2 enrichment of translation for each RNA shown in resting and depolarized conditions. Significance was calculated using the two-tailed, unpaired Student’s t-test. Data are presented as mean $\pm\,\mathsf{s.d}$ .P values: NS (not significant) >0.05; \* <0.05; \* $^{\ast}{<}0.01$ ;$^{***}\!<\!0.001$ ; \*\*\*\* ${<}0.0001$ .P value $\therefore S p k h I=0.00046,R g s I\mathbfit{4}=0.01$ ,$G h r l\!=\!7.12\times\!10^{-8}$ , Apobec1 $=0.0075$ ,$I g f2b p I=0.00053,P m l=0.0023,E i f l a d3=0.0022,P2r x7=0.024$ and  

Comparing PL-Ribo-seq and PL-MS data revealed that PL-Ribo-seq correlated with the equivalent dendritic proteomics datasets in all conditions, in contrast to PL-CLIP (Extended Data Fig. 5g–j). These findings indicated that there is a discrepancy between the RNA levels and protein levels in dendrites, suggesting that ribosomal complexes are acutely rearranged after depolarization. Moreover, these findings indicate that localized translation is a more precise predictor of the local proteomic composition than simply the presence of RNAs, particularly in response to neuronal activity.  

# Dendritic uORFs regulate a translational switch at downstream CDSs  

Although our data and previous reports show that overall translation decreases upon neuronal depolarization , they do not address if and how specific transcripts become translationally upregulated to meet synaptic needs. One of the most surprising findings from our data was that there was an increase in ribosome occupancy in the 5′ UTRs of dendritically localized mRNAs upon depolarization (Fig. 3g). This was specific to the dendritic compartment, because ribosome profiling of the total ribosome pool did not reveal any global changes in 5′ UTR ribosome occupancy (Fig. 3g). Interestingly, we did not observe any correlation between changes in translation of the $5^{\prime}$ UTR and CDS (Extended Data Fig. 6a).  

To better understand how increased ribosome occupancy in the 5′ UTR impacts translation, we sought to identify the full set of translated uORFs in $5^{\prime}$ UTRs in primary cortical neurons using ORF-RATER , an algorithm that identifies translated ORFs based on features of ribosome profiling data. This approach detected 946 translated uORFs that were largely unknown (Extended Data Fig. 6b and Supplementary Table 4). Most of these $(78\%)$ used the canonical ATG start codon, although a prominent subset used near-cognate start sites (Extended Data Fig. 6b).  

Integration of the set of translated uORFs with PL-Ribo-seq revealed that neuronal activation, by both KCl depolarization and DHPG, led to an overall increase in uORF translation in dendrites (Fig. 3h and Extended Data Fig. 6c), whereas translation of their corresponding CDS varied among individual mRNAs. For example, translation of a previously identified uORF present in the Gria2 5′ UTR increased with depolarization, but this was not accompanied by a substantial change in the expression of the downstream CDS (Fig. 3i). Homer3 and Atf3, by contrast, displayed increased uORF translation upon depolarization $T i m m23\,{=}\,0.0052.{\bf f}$ , Examples of gene sets (n indicates number of genes detected in each gene set) that are translationally upregulated in dendrites with neuronal depolarization and that correspond to examples of RNAs from e. Significance for box plots was determined by the two-sided Wilcoxon signed-rank test. g, Ribosome occupancy in $5^{\prime}$ UTRs of all detected RNAs $^{\leftmoon}_{\leftmoon}=14,684$ in rest; $n\!=\!14{,}770$ in dep) in the whole neuron (input) and in dendrites (pulldown) detected by PL-Ribo-seq from TurboID-PSD95-transduced neurons. RPKM changes for each region between resting and depolarized conditions were tested using the permutation t-test. The P values were then adjusted with Bonferroni correction. h, Dendritic translation of 5′ UTRs with uORFs identified by ORF-RATER in resting and depolarized PL-Ribo-seq. Rest: $n$ of all $5^{\prime}\mathrm{UTRs}=10.818$ ;uORFs $=967.$ Dep: n of all $\begin{array}{r}{5^{\prime}\mathrm{UTRs}\!=\!10,\!7\!18;\!\mathrm{uORFs}\!=\!859.}\end{array}$ Significance for box plots was determined by the one-sided Wilcoxon signed-rank test. i, Examples of RNAs with uORFs in their $5^{\prime}$ UTRs. PL-Ribo-seq reads $(\log_{2})$ in 5′ UTRs and CDS from the pulldowns (pd) of TurboID-PSD95-transduced neurons are shown for resting (top) and depolarized (bottom) dendrites. j, GSEA performed on translationally upregulated and downregulated dendritic RNAs with increased $5^{\prime}$ UTR translation in response to depolarization $(P<0.01)$ . P values were derived using the ‘fgsea’ package and the adaptive multi-level split Monte Carlo method. Colors: salmon (resting); burgundy (depolarized). Box plots show lower and upper hinges corresponding to the first and third quartiles (representing the 25th and 75th percentiles, respectively). Whiskers extend from the hinge to the $1.5\times$ interquartile range. The center line indicates the median. FC, fold change; NES, normalized enrichment score.  

that was accompanied by a concomitant increase in CDS translation in dendrites (Fig. 3i). The $5^{\prime}$ UTR and CDS translation of Apaf1 were both decreased by depolarization, and Mphosph9 was an example where translation increased in the $5^{\prime}$ UTR but decreased in the CDS (Fig. 3i).  

Interestingly, for most dendritic mRNAs with increased ribosome occupancy in their $5^{\prime}$ UTRs after depolarization, there was a corresponding increase in CDS translation without a significant change in transcript levels. These mRNAs encoded proteins involved in mitochondrial biology, long-term potentiation and cell signaling (Fig. 3j and Supplementary Table 5). A distinct set of these transcripts with increased $5^{\prime}$ UTR ribosome occupancy had decreased CDS translation without a significant decrease in mRNA levels. These encoded proteins function in the immune system, microtubule processes and kinase binding, indicating specific but differential regulation of functionally unique groups of dendritic mRNAs (Fig. 3j). These data suggest that depolarization caused an upregulation of ribosome occupancy in dendritic uORFs ( ${\sim}40\%$ and $34\%$ of 946 for KCl and DHPG, respectively), which leads to complex regulation of the downstream CDS translation. Several studies established that uORFs can hamper or promote downstream CDS translation in different cell types, particularly under Fig. 4 | uORFs in the 5′ UTRs of dendritic mRNAs regulate downstream CDS translation upon neuronal activation. a, Dendritic GFP reporter with myristoylation (myr) and LDLR-C-terminal (LDLRct) sequences and its localization by Flag (green) IF. DAPI (blue) marker for nuclei; MAP2 (red) marker for dendrites. Magnification, $\times20.$ . Scale bar, $50\,\upmu\mathrm{m}$ . b, Flag western blot for input (in) and pulldown (pd) from TurboID-PSD95-transduced neurons. c, Coverage of PL-Ribo-seq pulldown $(\log_{2})$ for Kcnj9 uORF and main ORF from resting and depolarized (dep) TurboID-PSD95-transduced neurons. d, Myristoylated, myristoylated with Kcnj9-5′ UTR, myristoylated with LDLRct and myristoylated with LDLRct and Kcnj9- $5^{\prime}$ UTR reporters in resting and depolarized neurons. Myristoylation and LDLRct sequences are in the CDS. GFP fold changes (dep/rest) from Flag (GFP) and ${\upbeta}$ -Actin western blots are calculated as: $\left[\boldsymbol{\mathsf{F l a g}}^{\mathrm{Dep}}\right]$ /${\upbeta}$ -ActinDep] / [FlagRest / β-ActinRest]. All are compared to myrGFP negative control $\left(n\!=\!3\right)$ . The center line is at mean. e,f, GFP fold changes of dendritic reporters with $5^{\prime}$ UTRs housing uORFs with increased translation upon depolarization $S^{\prime}$ UTR-up) and translationally upregulated (CDS-up) (e) or downregulated (CDS-down) (f) CDS. Negative controls: myrGFP, scrambled-Efcab-, -Rspo3-, -Immp1l- and -Spink10-5′ UTRs. All are compared to myrGFP $(n\,{=}\,4)$ . P values: Efcab1-scr $=0.16$ ; Rspo3-scr $=0.59$ ;$C m c4\!=\!0.013$ ;  

![](images/43260b010f1c19578ee80a287cd44ffb4a011044cf056076e78faecef5978ea5.jpg)  

![](images/84cd35af2c67d57b4bd20658cc8618373110421d0e0452e21bf96eb7a5f54f35.jpg)  
Efcab $!=\!0.0067$ ; Katnb1l $\mathbf{\chi}=0.021$ ;$L r r c5I\!=\!0.0038$ ;$N s u n3\!=\!0.000$ 11; $P o l g\!=\!0.89$ ;$R s p o3\!=0.0086;S l c25a l9\!=0.011;Z f\!p2\!86\!=0.0089;I n$ mmp1l-scr $=0.089$ ; Spink10- $\mathsf{s c r}=0.063;C n e p l r l=0.0072;C p t p=0.0021;l k z f\mathsf{S}=0.00029;$ Immp1 $\mathit{\Theta}^{'}\,{=}\,0.00041$ $M p h o s p h9=0.00067;M t i f2\!=\!0.00077;S p i n k l O\!=\!0.0032;$ and $V\!r\!k3\!=\!0.00031$ .g,h, Western blots of Mphosph9 and Kcnj9 CDS translation with and without their $5^{\prime}$ UTRs $({\bf g})$ are quantified by normalizing Flag expression in each condition to ${\upbeta}$ -Actin levels (h). In the same plot, also shown are the qPCRs of reporters, normalized as described for the western blots $(n\!=\!3)$ . Protein changes of CDS with $5^{\prime}$ UTRs are indicated by red asterisks. $P$ values: Mphosph9 (protein $=0.13$ ;$\mathtt{R N A}\!=\!0.0017$ ), $5^{\prime}\mathrm{UTR}+$ Mphosph (protein $=0.0014$ ;$\mathsf{R N A}\!=\!0.066$ ), Kcnj9 (protein $=0.49$ ;$\mathsf{R N A}\!=\!0.091$ ), $5^{\prime}\mathrm{UTR}+$ Kcnj9 (protein ${=}0.00031$ ;$\mathsf{R N A}\!=\!0.34\!;$ .i,j, IF images (i) and quantifications (j) of MPHOSPH9 (downregulated) and KCNJ9 (upregulated) with MAP2 (dendrites) and DAPI (nuclei) in resting and depolarized neurons $\left(n\!=\!3\right)$ . Magnification, $\times20.$ Scale bars, $25\,\upmu\mathrm{m}$ . All data are presented as mean $\pm$ s.d. Significance was calculated by two-tailed paired $\left(\mathbf{d}{-}\mathbf{f}\right)$ and unpaired (h,j) Student’s t-test. $P$ values: NS (not significant) $>\!0.05$ ;$^{*}{<}0.05;^{**}{<}0.01$ ; \*\*\* $<\!0.001$ ; \*\*\*\* ${<}0.0001.1$ $n$ indicates the number of biologically independent samples.  

stress conditions; however, their roles in localized translational control or how they achieve this regulation have not been fully explored59–62. Previous studies of localized RNAs in dendrites paved the way for the discovery of a wide range of dendritic localization signals, mainly in $3^{\prime}$ UTRs of transcripts, termed zipcodes63–69. We used zipcodes to direct the localization of reporter mRNAs and, thereby, explore the impact of neuronal activity on dendritic translation. We tested the effects of three zipcodes—Camk2a- $.3^{\prime}$ UTR, BC1 and myristoylated-LDLR-C-terminal (myr-LDLRct) sequences—on the localization and dendritic protein accumulation of a reporter mRNA that encodes for Flag-tagged $\mathbf{GFP}^{70}$ in cortical neurons. We analyzed reporter mRNA localization by FISH and protein localization and dendritic translation via IF and TurboID-PSD95-mediated biotinylation. We found that myr-LDLRct most accurately localized the majority of reporter mRNA and protein to dendrites and allowed the reporter protein to be biotinylated by TurboID-PSD95 (Fig. 4a,b and Extended Data Fig. 6d,e), enabling us to study the effects of uORFs and other features of $5^{\prime}$ UTRs on activity-dependent postsynaptic CDS translation.  

Using this reporter in cortical neurons, we tested the Kcnj9-5′ UTR, which harbors a uORF identified by ORF-RATER and showed increased $5^{\prime}$ UTR and CDS translation upon depolarization in dendrites by PL-Ribo-seq (Fig. 4c and Extended Data Fig. 6f). We detected a 2.5-fold increase in reporter protein levels after depolarization (Fig. 4d). This effect was strictly dependent on the dendritic localization and the $5^{\prime}$ UTR of the reporter mRNA (Fig. 4d), demonstrating that $5^{\prime}$ UTRs with uORFs are sufficient to confer activity-dependent translational control.  

We next expanded the set of reporters using a selection of 17 $5^{\prime}$ UTRs with uORFs based on the criteria that (1) they had increased ribosome occupancy in dendrites in response to depolarization; (2) they coincided with the greatest effects on levels of increased (Fig. 4e and Extended Data Fig. 6g) or decreased (Fig. 4f and Extended Data Fig. 6g) dendritic CDS translation without a significant change in their transcript levels (Supplementary Table 1); and (3) they were mostly uninvestigated (with the exception of Polg and Immp1l (refs. 59,71)). Strikingly, the downstream effects of 16 of 17 $5^{\prime}$ UTRs tested agreed with the findings from PL-Ribo-seq on the endogenous genes that they were derived from. We further confirmed the activity-dependent increase in translation using DHPG activation for three $5^{\prime}$ UTR reporter constructs (Cmc4, Lrrc51 and Nsun3) (Extended Data Fig. 6h). By contrast, scrambled versions of the same length $5^{\prime}$ UTRs failed to regulate translation, suggesting that specific motifs within these $5^{\prime}$ UTRs dictate downstream CDS translational regulation (Fig. 4e,f and Extended Data Fig. 6g).  

To test whether the translational control observed with the GFP reporter accurately recapitulated control of the endogenous genes, we examined mRNAs in their natural contexts by replacing GFP with the Mphosph9 or Kcnj9 CDS, with and without their $5^{\prime}$ UTRs. Mphosph9 and Kcnj9 were examples of uORF-containing mRNAs with increased $5^{\prime}$ UTR translation in dendrites upon depolarization. The Mphosph9 CDS was translationally downregulated (Fig. 3i), whereas Kcnj9 was upregulated (Fig. 4c), by PL-Ribo-seq. Using the dendritic reporter, we found that the translation of Mphosph9 decreased, and Kcnj9 increased, only in the presence of their $5^{\prime}$ UTRs and, in both cases, did so independently of their mRNA levels in each condition (Fig. 4g,h). We were able to visualize the changes in the endogenous levels of these proteins in dendrites after depolarization by IF (Fig. 4i,j and Extended Data Fig. 6i), highlighting the robustness of our findings. These data further strengthened the observation that $5^{\prime}$ UTRs with uORFs are the necessary and independent units that drive changes in dendritic CDS translation after neuronal depolarization.  

eIF4G2 binds to 5′ UTRs and regulates local mRNA translation Next, we sought to dissect mechanistically how uORFs might regulate CDS translation in an activity-dependent and localization-dependent Fig. 5 | eIF4G2 is required to upregulate dendritic mRNA translation upon activation. a, Dendritic mRNAs with increased ribosome occupancy in their $5^{\prime}$ UTRs upon depolarization are divided into two groups: translationally upregulated (CDS-up) and downregulated (CDS-down) in their CDS. Enriched mRNA binding protein (RBP) sites in the 5′ UTRs of these groups are shown in the heatmap using motifs from RBPmap72. P values were determined using hypergeometric testing. b, GSEA performed on differential (depolarized minus resting) $\log_{2}$ -ranked eIF4G2 CLIP target mRNAs bound in their $5^{\prime}$ UTRs. P values were derived using the ‘fgsea’ package and the adaptive multi-level split Monte Carlo method. NES, normalized enrichment score. c, mRNAs with increased eIF4G2 binding in their $5^{\prime}$ UTRs upon depolarization are determined by eIF4G2 CLIP and referred to as eIF4G2-bound. CDS translation of all detected mRNAs in the differential PL-Ribo-seq (depolarized (dep) minus resting (rest)) (all, $n\!=\!16{,}759_{.}$ ) and the eIF4G2-bound group $_{\left.\left.n=1,420\right\rangle}$ ) is compared. d, Nsun3-5′ UTR reporters: wild-type (Wt) harbors eIF4G2 binding sites (vertical bars); eIF $4{\bf G}2^{++}$ , two more eIF4G2 binding sites added; eIF4G2−, endogenous eIF4G2 binding sites scrambled. uORF mutant Nsun3-5′ UTR reporters: Start mut (start codon mutated); Elong (stop codon inserted after start); Stop (stop codon mutated). Fold changes are quantified from western blots of Flag and ${\upbeta}$ -Actin as in Fig. 4d, and significance was calculated by comparing to Wt $\left(n\right)=3,$ ). P values: Start mut $=0.0022$ , eIF $\mathbf{:}4{\bf G}2\mathbf{+}\mathbf{+}=0.59$ , eI $\mathbf{\dot{\delta}}4\mathbf{G}2-=0.0023$ , Elong $=0.0010$ and $\mathsf{S t o p}=0.0034.$ e, Western blot quantifications of Flag and ${\upbeta}$ -Actin of Nsun3-5′ manner. We hypothesized that RBPs might mediate the effects of uORFs on downstream translation. To that end, we examined RBP binding sites enriched in the $5^{\prime}$ UTRs of dendritically translated messages using RBPmap . Comparing resting, KCl-depolarized and differential (depolarized minus resting) ribosome occupancy in the $5^{\prime}$ UTRs of all dendritically translated mRNAs that we discovered by PL-Ribo-seq, we found binding sites for IGF2BP1, HNRNPK/L, eIF2α and eIF4G2 to be enriched in the $5^{\prime}$ UTRs of all mRNAs that were translationally regulated after depolarization (Extended Data Fig. 7a). We further divided the group of dendritic mRNAs with enhanced $5^{\prime}$ UTR translation according to their downstream CDS translation properties (CDS-up versus CDS-down) to assess whether specific RBPs were associated with differential effects. HNRNPK and TIA1 binding sites were enriched in the $5^{\prime}$ UTRs of the CDS-down group (Fig. 5a), in agreement with their known roles in translational repression by binding at the $5^{\prime}\,{\cup}\,{\sf T R S}^{73,74}$ . In contrast, eIF4G2 and RBFOX2 binding sites were enriched only in the $5^{\prime}$ UTRs of the CDS-up group in dendrites (Fig. 5a).  

Out of the four predicted binding motifs for eIF4G2, ‘CGCGGC’ (eIF4G2(4)) was the most enriched motif in the $5^{\prime}$ UTRs of translationally upregulated dendritic mRNAs (Fig. 5a and Extended Data Fig. 7b). Although the presence of translated uORFs canonically leads to decreased translation of downstream CDS, there are a few examples where they also enhance it60,61,75. The non-canonical translation initiation factor eIF4G2 is known to play roles in cap-dependent and cap-independent translation and to upregulate translation of RNAs with structured $5^{\prime}$ UTRs or uORFs in non-neuronal systems76–79. Previously known roles of eIF4G2 in translation initiation80 led us to focus on eIF4G2 as a potential activity-dependent translational regulator in dendrites.  

We experimentally determined how neuronal activity might impact eIF4G2 binding by performing eIF4G2 CLIP in resting and KCl-depolarized primary cortical neurons. The ‘CGCGGC’ motif was the most crosslinked motif in our eIF4G2-CLIP $75\%$ of all $5^{\prime}$ UTR peaks; Extended Data Fig. 7c), consistent with our analysis of transcripts undergoing translational control in dendrites (Fig. 5a). For the resting eIF4G2 CLIP, we found examples of bound RNAs previously identified in human cell lines81 (Extended Data Fig. 7d). Overall, approximately $50\%$ of our eIF4G2 CLIP binding was localized in $5^{\prime}$ UTRs (Extended Data Fig. 7e), consistent with its purported function in translation initiation. Together, these data strongly implicated eIF4G2 binding in translational control after depolarization.  

UTR reporters in non-targeting $\left(-\right)$ or eIF4G2 siRNA-treated $\left(+\right)$ conditions in resting and depolarized neurons are quantified as in Fig. 4d $(n\,{=}\,4)$ . Box plot whiskers extend to minimum and maximum, with the center line at median. Significance was calculated using the two-tailed, unpaired Student’s t-test. f, Subset of eIF4G2-bound mRNAs that are translationally upregulated $.n=321\$ )are shown in Wt (all, $n\!=\!16{,}759^{\circ},$ and eIF4G2 knockdown (all, $n\!=\!17{,}547\!,$ PL-Ribo-seq data, in both conditions compared to all detected mRNAs in the corresponding dep-rest PL-Ribo-seq. g, Dendritic translational regulation of the Wt Nsun3- $5^{\prime}$ UTR reporter is tested in resting and depolarized Wt and eIF4G2 knockdown (Kd) neurons, in which eIF4G2 levels are rescued by a dendritically localized Wt or phospho-mutant version of eIF4G2. Fold changes from western blots of Flag and ${\upbeta}$ -Actin are quantified as in Fig. 4d. Significance was calculated by comparing to Wt $(n\,{=}\,3)$ . P values: $\mathsf{K d}\,{=}\,0.0090$ ,${\sf K d}\!+\!{\sf W t}=0.39$ ,$\mathsf{K d}\!+\!\mathsf{M u t1}\!=\!0.0005\mathsf{\xi}$ 5, $\mathsf{K d}\!+\!\mathsf{M u t}2\!=\!0.66$ ,$\mathsf{K d}\!+\!\mathsf{M u t}3\!=\!0.52$ and $\mathsf{K d}\!+\!\mathsf{M u t}4=0.041$ .c,f, Significance was calculated using the two-sided Kolmogorov–Smirnov test. Box plots show lower and upper hinges corresponding to 25th and 75th percentiles. Whiskers extend from the hinge to the $1.5\times$ interquartile range. The center line indicates the median. d,g, Significance was calculated using the two-tailed, paired Student’s t-test. All data are presented as mean $\pm\,\mathbf{s}.\mathbf{d}.\,P$ values: NS (not significant) ${>}0.05;{^*}{<}0.05$ ;$^{\times*}{<}0.01$ ; \*\*\* $<\!0.001$ ; \*\*\*\* ${<}0.0001.$ n indicates the number of biologically independent samples.  

Using our CLIP data, we identified a set of mRNAs with increased eIF4G2 binding in their $5^{\prime}$ UTRs in response to depolarization (termed ‘eIF4G2-bound’; Supplementary Table 6). The eIF4G2-bound targets were particularly enriched in mitochondria-related mRNAs by GSEA, along with additional targets encoding signaling, receptor clustering and mRNA processing-related messages (Fig. 5b). We then examined the dendritic translation and localization of this group of mRNAs using our PL-Ribo-seq and PL-CLIP datasets. Transcripts that had more eIF4G2 binding in their $5^{\prime}$ UTRs were translationally upregulated in depolarized dendrites, despite the observation that their transcript levels were decreased (Fig. 5c and Extended Data Fig. 7f). Furthermore, eIF4G2-bound targets with increased $5^{\prime}$ UTR translation upon depolarization showed higher levels of CDS translation compared to all mRNAs with enhanced $5^{\prime}$ UTR translation in dendrites (Extended Data Fig. $7\mathbf{g},$ ). Overall, these analyses revealed that eIF4G2 binding at the $5^{\prime}$ UTRs is associated with enhanced translation independent of mRNA levels in response to neuronal activity in dendrites.  

![](images/0f7863acd6fc00d2e29e61d6f9eeef76387908f48ea32635096adcf742a15a31.jpg)  
The efect of eIF4G2 phosphorylation on uORF-mediated dendritic translational control  

![](images/7f588579319d0d6290077e56ec652e7585ea2311329092bee24b1730312c80ca.jpg)  

![](images/e170ef75d882159a5343e45313f8783eb6d50aa0a6cd32200921194def2a9ea1.jpg)  
Fig. 6 | Model for activity-dependent eIF4G2:uORF-mediated translational  downstream CDSs. Translation of uORFs from these 5′ UTRs is enhanced even control in dendrites. A subset of dendritically enriched mRNAs, including those  though overall dendritic translation is suppressed (Fig. 3h and Extended Data with roles in signaling and mitochondrial functions, is translationally silent Figs. 5e and 6c). eIF4G2 binding then allows efficient scanning of ribosomes into in resting postsynaptic sites. We demonstrate that, after neuronal activation the downstream CDSs, enabling the rapid production of dendritic proteins that and calcium influx, eIF4G2 is phosphorylated at threonine (T507), enabling are needed for synaptic plasticity and energy homeostasis. direct binding to select $5^{\prime}$ UTRs (Fig. $5\mathrm{g})$ to upregulate the translation of the  

We next studied the regulation of Nsun3 as its transcript harbored an activity-dependent uORF that had the largest effect on downstream CDS translation (\~3.3-fold increase; Fig. 4e) and had eIF4G2 binding sites in its 5′ UTR. Moreover, Nsun3 plays a role in mitochondrial translation, consistent with our observation that mitochondrial regulation was the most enriched category of dendritic mRNAs upon depolarization (Figs. 3j and 5b). The absolute level of Nsun3 CDS translation increased in depolarized dendrites by PL-Ribo-seq despite the fact that the local transcript levels were decreased (as measured by PL-CLIP; Extended Data Fig. 8a). Nsun3- $.5^{\prime}$ UTR has two endogenous eIF4G2 binding sites, and we generated reporters with a variety of mutations to study if uORF translation and/or eIF4G2 binding were necessary to enhance CDS translation (Fig. 5d). Wild-type Nsun3 uORF and Nsun3 uORF with two additional eIF4G2 binding sites $\mathsf{e I F4G2++}^{\dagger}$ ) performed similarly and increased the downstream GFP translation approximately 3.3-fold without impacting mRNA levels (Fig. 5d and Extended Data Fig. 8b). However, translational increase was ablated by mutants that prevented uORF translation, despite an increase in mRNA levels. Moreover, eliminating all eIF4G2 binding sites (eIF4G2−) entirely abrogated depolarization-dependent upregulation of CDS translation, indicating that uORF translation along with eIF4G2 binding are both necessary for the downstream CDS upregulation.  

Using a proximity ligation assay (PLA) that incorporates puromycin to visualize nascent translation and IF to assess total protein levels, we were able to confirm endogenous Nsun3 regulation in resting and depolarized neurons. In agreement with our PL-Ribo-seq, we saw a significant increase in newly translated Nsun3 in dendrites with depolarization along with an increase in total protein levels (Extended Data Fig. 8c,d). Notably, this activity-dependent increase in translation is eIF4G2 dependent, because siRNA-mediated knockdown of eIF4G2 or mutation of its binding sites (eIF4G2−) inhibited translational upregulation (Fig. 5e and Extended Data Fig. 8e).  

We observed additional transcripts that were translationally upregulated in dendrites through activity-induced uORF translation without a change in transcript levels (Extended Data Fig. 9a–i). Only uORFs with eIF4G2 binding sites, similar to that of Nsun3, were dependent on eIF4G2 to upregulate their CDS (for example, Mtf1; Extended Data Fig. 9j,k). In addition, eIF4G2-bound dendritic mRNAs that were translationally upregulated and harbored uORFs showed a significant decrease in activity-dependent local translation levels upon eIF4G2 knockdown (Fig. 5f and Extended Data Fig. 10a). These findings further establish that dendritic eIF4G2 binding within the 5′ UTRs is necessary for the activity-dependent increase in the translation of the main open reading frame by uORFs.  

To assess how eIF4G2 might mediate its activity-dependent translational control, we measured total eIF4G2 protein in resting and KCl-depolarized dendrites. Although there was a slight increase in total eIF4G2, it was not significant (Extended Data Fig. 10b). We, thus, hypothesized that eIF4G2 phosphorylation might be critical for its activity-dependent role because another eIF4G protein, eIF4G3, has been shown to be a target of calcium-dependent phosphorylation82, and we found calcium influx upon depolarization (Fig. 1e and Extended Data Fig. 1m) to be critical for the translational changes that we observed (Extended Data Fig. 10c). To that end, we engineered dendritically localized wild-type and phospho-mutant versions of eIF4G2 using our dendritic localization signals (Figs. 4a and 5f and Extended Data Fig. 10d) and rescued eIF4G2 levels with these variants where eIF4G2 was depleted across the whole neuron. Although all eIF4G2 mutants accumulated at similar levels as the wild-type protein (Extended Data Fig. 10d), only T507V mutant completely failed to rescue activity-dependent translational control, suggesting that this site might be critical for calcium-mediated regulation of dendritic protein synthesis by eIF4G2.  

# Discussion  

Activity-dependent alterations in RNA localization and localized translation in synapses underlie learning and memory3,6; however, to date, our ability to globally analyze the mechanisms and dynamics of their regulation has been limited. Here we describe a new proximity-based labeling platform engineered to monitor activity-dependent changes in postsynaptic mRNA, translation and protein levels. This approach revealed a new mechanism of translational control in which depolarization leads to calcium influx, eIF4G2 phosphorylation and de novo binding to target transcripts and increased uORF translation in the 5′ UTRs of those transcripts, resulting in enhanced downstream protein synthesis. Our studies provide an unprecedented view into both the nature and mechanisms by which neuronal activity sculpts the dendritic proteome.  

KCl treatment was initially chosen for triggering neuronal depolarization because it is well established, has been widely used to study synaptic biology and enables the detection of robust changes in dendrites. We recognize that prolonged KCl activation does not mimic physiologic neuronal stimulation. To demonstrate the rigor of our experimental findings, we shortened the KCl activation time, relative to previous prolonged depolarizations83–85, and, notably, replicated our key findings with the glutamate agonist DHPG. Although dephosphorylation followed by increased de novo translation has been observed in behavioral studies86,87, our finding that depolarization induces phosphorylation of translation control factors and global translational downregulation agrees with several other analyses88–92. The disparity among studies could be attributed to different activation paradigms and experimental strategies or could, in fact, highlight the spatial and temporal complexity of activity-dependent synaptic translational control.  

Our results indicate that, although multiple mechanisms contribute to activity-dependent changes in dendritic proteomes, translational control is crucial in regulating the rapid upregulation of a functionally coherent subset of messages in dendrites. This finding helps elucidate one of the longstanding conundrums in the field: why do dendrites localize a large number of mRNAs when many of these messages are poorly engaged by the translation machinery?  

To address this unknown, we compared RNA, translation and protein levels in resting and activated dendrites treated with short pulses of either KCl or DHPG, accompanied by calcium influx. The lack of correlation in RNA and protein levels upon depolarization led us to examine the translation of pre-existing pools of localized RNAs and to uncover a uORF-mediated mechanism of protein synthesis. These pre-emptively localized RNAs contain necessary information in their $5^{\prime}$ UTRs to allow for activity-dependent uORF translation and recruitment of phosphorylated eIF4G2 that, together, upregulate downstream ribosome binding and CDS translation (Fig. 6). Additionally, we noted the differences in the resting basal protein levels when the $5^{\prime}$ UTRs cannot be translated or bound by eIF4G2, which might be driven by changes in RNA levels (Extended Data Fig. 8b,e). We postulate that this involves additional cis-acting signals on these transcripts. Taken together, this mechanism brings a new understanding as to why these RNAs are enriched at postsynaptic sites and how eIF4G2:uORF targeting can rapidly change ribosome scanning to generate a coherent set of proteins upon synaptic needs.  

Neuronal survival and synaptic and metabolic plasticity depend critically on ATP production and calcium buffering by mitochondria . The changes in protein synthesis impacted by depolarization point to a central role for the upregulation of mitochondrial proteins and metabolic activity (Figs. 3f,j, 5b and 6) in response to dendritic activation. In agreement with our work, Li et al.  showed that depolarization enhances mitochondrial localization at dendritic spines in a calcium influx-dependent manner.  

Taken together, our studies reveal an essential mechanism of activity-dependent uORF:eIF4G2-mediated translational control present in many dendritic transcripts. It remains to be determined if the translated uORFs encode functional micropeptides and how the de novo translation of downstream proteins affects neuronal physiology. This regulation is poised to help maintain neuronal integrity and dendritic function during calcium influx, providing insights into the specialized nature of mechanisms underlying protein synthesis-dependent synaptic plasticity.  

# Online content  

Any methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41593-024-01615-5.  

# References  

1. Zappulo, A. et al. RNA localization is a key determinant of neurite-enriched proteome. Nat. Commun. 8, 583 (2017).   
2. Martin, K. C. & Ephrussi, A. mRNA localization: gene expression in the spatial dimension. Cell 136, 719–730 (2009).   
3. Huber, K. M., Kayser, M. S. & Bear, M. F. Role for rapid dendritic protein synthesis in hippocampal mGluR-dependent long-term depression. Science 288, 1254–1257 (2000).   
4.	 Fonkeu, Y. et al. How mRNA localization and protein synthesis sites influence dendritic protein distribution and dynamics. Neuron 103, 1109–1122 (2019).   
5.	 Ju, W. et al. Activity-dependent regulation of dendritic synthesis and trafficking of AMPA receptors. Nat. Neurosci. 7, 244–253 (2004).   
6. Frey, U., Krug, M., Reymann, K. G. & Matthies, H. Anisomycin, an inhibitor of protein synthesis, blocks late phases of LTP phenomena in the hippocampal CA1 region in vitro. Brain Res. 452, 57–65 (1988).   
7. Garner, C. C., Tucker, R. P. & Matus, A. Selective localization of messenger RNA for cytoskeletal protein MAP2 in dendrites. Nature 336, 674–677 (1988).   
8. Wallace, C. S., Lyford, G. L., Worley, P. F. & Steward, O. Differential intracellular sorting of immediate early gene mRNAs depends on signals in the mRNA sequence. J. Neurosci. 18, 26–35 (1998).   
9.	 Cajigas, I. J. et al. The local transcriptome in the synaptic neuropil revealed by deep sequencing and high-resolution imaging. Neuron 74, 453–466 (2012).   
10.	 Middleton, S. A., Eberwine, J. & Kim, J. Comprehensive catalog of dendritically localized mRNA isoforms from sub-cellular sequencing of single mouse neurons. BMC Biol. 17, 5 (2019).   
11.	 Dumrongprechachan, V. et al. Cell-type and subcellular compartment-specific APEX2 proximity labeling reveals activity-dependent nuclear proteome dynamics in the striatum. Nat. Commun. 12, 4855 (2021).   
12.	 Loh, K. H. et al. Proteomic analysis of unbounded cellular compartments: synaptic clefts. Cell 166, 1295–1307 (2016).   
13.	 Uezu, A. et al. Identification of an elaborate complex mediating postsynaptic inhibition. Science 353, 1123–1129 (2016).   
14.	 Hale, C. R. et al. FMRP regulates mRNAs encoding distinct functions in the cell body and dendrites of CA1 pyramidal neurons. eLife 10, e71892 (2021).   
15.	 Popovitchenko, T. et al. Translational derepression of Elavl4 isoforms at their alternative $5^{\prime}$ UTRs determines neuronal development. Nat. Commun. 11, 1674 (2020).   
16.	 Xue, S. et al. RNA regulons in Hox 5′ UTRs confer ribosome specificity to gene regulation. Nature 517, 33–38 (2015).   
17.	 Beaudoin, J.-D. & Perreault, J.-P. $5^{\prime}$ -UTR G-quadruplex structures acting as translational repressors. Nucleic Acids Res. 38, 7022–7036 (2010).   
18.	 Lohse, I., Reilly, P. & Zaugg, K. The CPT1C 5′UTR contains a repressing upstream open reading frame that is regulated by cellular energy availability and AMPK. PLoS ONE 6, e21486 (2011).   
19.	 Gerashchenko, M. V., Lobanov, A. V. & Gladyshev, V. N. Genome-wide ribosome profiling reveals complex translational regulation in response to oxidative stress. Proc. Natl Acad. Sci. USA 109, 17394–17399 (2012).   
20.	 Young, S. K. & Wek, R. C. Upstream open reading frames differentially regulate gene-specific translation in the integrated stress response. J. Biol. Chem. 291, 16927–16935 (2016).   
21.	 Branon, T. C. et al. Efficient proximity labeling in living cells and organisms with TurboID. Nat. Biotechnol. 36, 880–887 (2018).   
22.	 Ifrim, M. F., Williams, K. R. & Bassell, G. J. Single-molecule imaging of PSD-95 mRNA translation in dendrites and its dysregulation in a mouse model of fragile X syndrome. J. Neurosci. 35, 7116–7130 (2015).   
23.	 Shin, S. M. et al. GKAP orchestrates activity-dependent postsynaptic protein remodeling and homeostatic scaling. Nat. Neurosci. 15, 1655–1666 (2012).   
24.	 Malik, A. N. et al. Genome-wide identification and characterization of functional neuronal activity-dependent enhancers. Nat. Neurosci. 17, 1330–1339 (2014).   
25.	 Palmer, M. J., Irving, A. J., Seabrook, G. R., Jane, D. E. & Collingridge, G. L. The group I mGlu receptor agonist DHPG induces a novel form of LTD in the CA1 region of the hippocampus. Neuropharmacology 36, 1517–1532 (1997).   
26.	 Ule, J. et al. CLIP identifies Nova-regulated RNA networks in the brain. Science 302, 1212–1215 (2003).   
27.	 Licatalosi, D. D. et al. HITS-CLIP yields genome-wide insights into brain alternative RNA processing. Nature 456, 464–469 (2008).   
28.	 Poon, M. M., Choi, S.-H., Jamieson, C. A. M., Geschwind, D. H. & Martin, K. C. Identification of process-localized mRNAs from cultured rodent hippocampal neurons. J. Neurosci. 26, 13390–13399 (2006).   
29.	 Zhong, J., Zhang, T. & Bloch, L. M. Dendritic mRNAs encode diversified functionalities in hippocampal pyramidal neurons. BMC Neurosci. 7, 17 (2006).   
30.	 Lein, E. S. et al. Genome-wide atlas of gene expression in the adult mouse brain. Nature 445, 168–176 (2007).   
31.	 Taliaferro, J. M. et al. Distal alternative last exons localize mRNAs to neural projections. Mol. Cell 61, 821–833 (2016).   
32.	 Ainsley, J. A., Drane, L., Jacobs, J., Kittelberger, K. A. & Reijmers, L. G. Functionally diverse dendritic mRNAs rapidly associate with ribosomes following a novel experience. Nat. Commun. 5, 4510 (2014).   
33.	 Tushev, G. et al. Alternative $3^{\prime}$ UTRs modify the localization, regulatory potential, stability, and plasticity of mRNAs in neuronal compartments. Neuron 98, 495–511 (2018).   
34.	 Glock, C. et al. The translatome of neuronal cell bodies, dendrites, and axons. Proc. Natl Acad. Sci. USA 118, e2113929118 (2021).   
35.	 Stickels, R. R. et al. Highly sensitive spatial transcriptomics at near-cellular resolution with Slide-seqV2. Nat. Biotechnol. 39, 313–319 (2021).   
36.	 Shigeoka, T. et al. Dynamic axonal translation in developing and mature visual circuits. Cell 166, 181–192 (2016).   
37.	 Hafner, A.-S., Donlin-Asp, P. G., Leitch, B., Herzog, E. & Schuman, E. M. Local protein synthesis is a ubiquitous feature of neuronal pre- and postsynaptic compartments. Science 364, eaau3644 (2019).   
38.	 Zivraj, K. H. et al. Subcellular profiling reveals distinct and developmentally regulated repertoire of growth cone mRNAs. J. Neurosci. 30, 15464–15478 (2010).   
39.	 Taylor, A. M. et al. Axonal mRNA in uninjured and regenerating cortical mammalian axons. J. Neurosci. 29, 4697–4707 (2009).   
40.	 Dictenberg, J. B., Swanger, S. A., Antar, L. N., Singer, R. H. & Bassell, G. J. A direct role for FMRP in activity-dependent dendritic mRNA transport links filopodial-spine morphogenesis to fragile X syndrome. Dev. Cell 14, 926–939 (2008).   
41.	 Monday, H. R., Kharod, S. C., Yoon, Y. J., Singer, R. H. & Castillo, P. E. Presynaptic FMRP and local protein synthesis support structural and functional plasticity of glutamatergic axon terminals. Neuron 110, 2588–2606 (2022).   
42.	 Farris, S. et al. Hippocampal subregions express distinct dendritic transcriptomes that reveal differences in mitochondrial function in CA2. Cell Rep. 29, 522–539 (2019).   
43.	 Zeitelhofer, M. et al. Dynamic interaction between P-bodies and transport ribonucleoprotein particles in dendrites of mature hippocampal neurons. J. Neurosci. 28, 7555–7562 (2008).   
44.	 Chen, D., Yu, W., Aitken, L. & Gunn-Moore, F. Willin/FRMD6: a multi-functional neuronal protein associated with Alzheimer’s disease. Cells 10, 3024 (2021).   
45.	 Palazzo, A. F. & Kang, Y. M. GC-content biases in protein-coding genes act as an ‘mRNA identity’ feature for nuclear export. Bioessays 43, e2000197 (2021).   
46.	 Bae, B. & Miura, P. Emerging roles for $3^{\prime}$ UTRs in neurons. Int. J. Mol. Sci. 21, 3413 (2020).   
47.	 Hwang, H.-W. et al. cTag-PAPERCLIP reveals alternative polyadenylation promotes cell-type specific protein diversity and shifts Araf isoforms with microglia activation. Neuron 95, 1334–1349 (2017).   
48.	 Jereb, S. et al. Differential 3′ processing of specific transcripts expands regulatory and protein diversity across neuronal cell types. eLife 7, e34042 (2018).   
49.	 Brigidi, G. S. et al. Genomic decoding of neuronal depolarization by stimulus-specific NPAS4 heterodimers. Cell 179, 373–391 (2019).   
50.	 Distler, U. et al. In-depth protein profiling of the postsynaptic density from mouse hippocampus using data-independent acquisition proteomics. Proteomics 14, 2607–2613 (2014).   
51.	 Ingolia, N. T., Brar, G. A., Rouskin, S., McGeachy, A. M. & Weissman, J. S. The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments. Nat. Protoc. 7, 1534–1550 (2012).   
52.	 Lauria, F. et al. riboWaltz: optimization of ribosome P-site positioning in ribosome profiling data. PLoS Comput. Biol. 14, e1006169 (2018).   
53.	 Dunn, J. G., Foo, C. K., Belletier, N. G., Gavis, E. R. & Weissman, J. S. Ribosome profiling reveals pervasive and regulated stop codon readthrough in Drosophila melanogaster. eLife 2, e01179 (2013).   
54.	 Huh, G. S. et al. Functional requirement for class I MHC in CNS development and plasticity. Science 290, 2155–2159 (2000).   
55.	 Corriveau, R. A., Huh, G. S. & Shatz, C. J. Regulation of class I MHC gene expression in the developing and mature CNS by neural activity. Neuron 21, 505–520 (1998).   
56.	 Dalal, J. S. et al. Quantitative nucleotide level analysis of regulation of translation in response to depolarization of cultured neural cells. Front. Mol. Neurosci. 10, 9 (2017).   
57.	 Krichevsky, A. M. & Kosik, K. S. Neuronal RNA granules: a link between RNA localization and stimulation-dependent translation. Neuron 32, 683–696 (2001).   
58.	 Fields, A. P. et al. A regression-based analysis of ribosome-profiling data reveals a conserved complexity to mammalian translation. Mol. Cell 60, 816–827 (2015).   
59.	 Calvo, S. E., Pagliarini, D. J. & Mootha, V. K. Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl Acad. Sci. USA 106, 7507–7512 (2009).   
60.	 Starck, S. R. et al. Translation from the $5^{\prime}$ untranslated region shapes the integrated stress response. Science 351, aad3867 (2016).   
61.	 Andreev, D. E. et al. Translation of 5′ leaders is pervasive in genes resistant to eIF2 repression. eLife 4, e03971 (2015).   
62.	 Wright, B. W., Yi, Z., Weissman, J. S. & Chen, J. The dark proteome: translation from noncanonical open reading frames. Trends Cell Biol. 32, 243–258 (2022).   
63.	 Kobayashi, H., Yamamoto, S., Maruo, T. & Murakami, F. Identification of a cis-acting element required for dendritic targeting of activity-regulated cytoskeleton-associated protein mRNA. Eur. J. Neurosci. 22, 2977–2984 (2005).   
64.	 Meer, E. J. et al. Identification of a cis-acting element that localizes mRNA to synapses. Proc. Natl Acad. Sci. USA 109, 4639–4644 (2012).   
65.	 Patel, V. L. et al. Spatial arrangement of an RNA zipcode identifies mRNAs under post-transcriptional control. Genes Dev. 26, 43–53 (2012).   
66.	 Ciolli Mattioli, C. et al. Alternative 3′ UTRs direct localization of functionally diverse protein isoforms in neuronal compartments. Nucleic Acids Res. 47, 2560–2573 (2019).   
67.	 Robeck, T., Skryabin, B. V., Rozhdestvensky, T. S., Skryabin, A. B. & Brosius, J. BC1 RNA motifs required for dendritic transport in vivo. Sci. Rep. 6, 28300 (2016).   
68.	 Mori, Y., Imaizumi, K., Katayama, T., Yoneda, T. & Tohyama, M. Two cis-acting elements in the 3′ untranslated region of $\upalpha$ -CaMKII regulate its dendritic targeting. Nat. Neurosci. 3, 1079–1084 (2000).   
69.	 Kameda, H. et al. Targeting green fluorescent protein to dendritic membrane in central neurons. Neurosci. Res. 61, 79–91 (2008).   
70.	 Pédelacq, J.-D., Cabantous, S., Tran, T., Terwilliger, T. C. & Waldo, G. S. Engineering and characterization of a superfolder green fluorescent protein. Nat. Biotechnol. 24, 79–88 (2006).   
71.	 Loughran, G. et al. Unusually efficient CUG initiation of an overlapping reading frame in POLG mRNA yields novel protein POLGARF. Proc. Natl Acad. Sci. USA 117, 24936–24946 (2020).   
72.	 Paz, I., Kosti, I., Ares, M. Jr., Cline, M. & Mandel-Gutfreund, Y. RBPmap: a web server for mapping binding sites of RNA-binding proteins. Nucleic Acids Res. 42, W361–W367 (2014).   
73.	 Mukhopadhyay, N. K. et al. Heterogeneous nuclear ribonucleoprotein K is a novel regulator of androgen receptor translation. Cancer Res. 69, 2210–2218 (2009).   
74.	 Damgaard, C. K. & Lykke-Andersen, J. Translational coregulation of 5′TOP mRNAs by TIA-1 and TIAR. Genes Dev. 25, 2057–2068 (2011).   
75.	 Rodriguez, C. M., Chun, S. Y., Mills, R. E. & Todd, P. K. Translation of upstream open reading frames in a model of neuronal differentiation. BMC Genomics 20, 391 (2019).   
76.	 Smirnova, V. V. et al. Ribosomal leaky scanning through a translated uORF requires eIF4G2. Nucleic Acids Res. 50, 1111–1127 (2022).   
77.	 Lewis, S. M. et al. The eIF4G homolog DAP5/p97 supports the translation of select mRNAs during endoplasmic reticulum stress. Nucleic Acids Res. 36, 168–178 (2008).   
78.	 Haizel, S. A., Bhardwaj, U., Gonzalez, R. L. Jr, Mitra, S. & Goss, D. J. 5′-UTR recruitment of the translation initiation factor eIF4GI or DAP5 drives cap-independent translation of a subset of human mRNAs. J. Biol. Chem. 295, 11693–11706 (2020).   
79.	 de la Parra, C. et al. A widespread alternate form of cap-dependent mRNA translation initiation. Nat. Commun. 9, 3068 (2018).   
80.	 Imataka, H., Olsen, H. S. & Sonenberg, N. A new translational regulator with homology to eukaryotic translation initiation factor 4G. EMBO J. 16, 817–825 (1997).   
81.	 Van Nostrand, E. L. et al. A large-scale binding and functional map of human RNA-binding proteins. Nature 583, 711–719 (2020).   
82.	 Qin, H., Raught, B., Sonenberg, N., Goldstein, E. G. & Edelman, A. M. Phosphorylation screening identifies translational initiation factor 4GII as an intracellular target of Ca /calmodulin-dependent protein kinase I. J. Biol. Chem. 278, 48570–48579 (2003).   
83.	 Tyssowski, K. M. et al. Different neuronal activity patterns induce different gene expression programs. Neuron 98, 530–546 (2018).   
84.	 Mendonsa, S. et al. Massively parallel identification of mRNA localization elements in primary cortical neurons. Nat. Neurosci. 26, 394–405 (2023).   
85.	 Li, B. et al. Neuronal inactivity co-opts LTP machinery to drive potassium channel splicing and homeostatic spike widening. Cell 181, 1547–1565 (2020).   
86.	 Im, H.-I. et al. Post-training dephosphorylation of eEF-2 promotes protein synthesis for memory consolidation. PLoS ONE 4, e7424 (2009).   
87.	 Costa-Mattioli, M. et al. Translational control of hippocampal synaptic plasticity and memory by the eIF2α kinase GCN2. Nature 436, 1166–1173 (2005).   
88.	 Scheetz, A. J., Nairn, A. C. & Constantine-Paton, M. NMDA receptor-mediated control of protein synthesis at developing synapses. Nat. Neurosci. 3, 211–216 (2000).   
89.	 Yoon, S. H., Song, W. S., Oh, S. P., Kim, Y. S. & Kim, M.-H. The phosphorylation status of eukaryotic elongation factor-2 indicates neural activity in the brain. Mol. Brain 14, 142 (2021).   
90.	 Heise, C. et al. Elongation factor-2 phosphorylation in dendrites and the regulation of dendritic mRNA translation in neurons. Front. Cell. Neurosci. 8, 35 (2014).   
91.	 Verpelli, C. et al. Synaptic activity controls dendritic spine morphology by modulating eEF2-dependent BDNF synthesis. J. Neurosci. 30, 5830–5842 (2010).   
92.	 Di Prisco, G. V. et al. Translational control of mGluR-dependent long-term depression and object-place learning by eIF2α. Nat. Neurosci. 17, 1073–1082 (2014).   
93.	 Delaunay, S. et al. Mitochondrial RNA modifications shape metabolic plasticity in metastasis. Nature 607, 593–603 (2022).   
94.	 Li, Z., Okamoto, K.-I., Hayashi, Y. & Sheng, M. The importance of dendritic mitochondria in the morphogenesis and plasticity of spines and synapses. Cell 119, 873–887 (2004).  

Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.  

Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons. org/licenses/by/4.0/.  

$\circledcirc$ The Author(s) 2024  

# Methods Plasmid and reporter cloning  

Constructs were cloned into the doxycycline-inducible Lenti-X Tet-ON 3G expression system (Takara Bio, 631187) using the $P_{\mathrm{TRE3GS}}$ promoter. TurboID was generously provided by A. Y. Ting at Stanford University. PSD95 was amplified from rat PSD95, and the 5′ UTR and 3′ UTR regions of PSD95 were amplified from the pCMV-5U-Venus-PSD-95-3U construct (Addgene, 102949). Superfolder GFP (sfGFP) was included between TurboID and PSD95 to visualize TurboID in live cells, and Flag tag was included at the $5^{\prime}$ end of TurboID to use in immunostaining. In the final 5′ UTR-TurboID-sfGFP-PSD95- $_{3^{\prime}}$ UTR construct, TurboID-sfGFP-PSD95 were separated by GS and NSRV linkers, respectively. All viral constructs were cloned and propagated at $30\,^{\circ}\mathbf{C}.$ Control experiments to test the leakiness and doxycycline sensitivity of the Tet-ON 3G system were done using the luciferase reporter that was included in the Takara kit. Luciferase activity was measured using the Varioskan LUX plate reader and the SkanIt RE 5.0 program. All constructs were expressed at $300\,\mathsf{n g\,m l}^{-1}$ doxycycline concentration.  

Myristoylation (Met-Gly-Thr-Val-Leu-Ser-Leu-Ser-Pro-Ser-Tyr) and LDLRct sequences were cloned at the $5^{\prime}$ and $3^{\prime}$ end of sfGFP, respectively, using the Gibson strategy (New England Biolabs (NEB), E5510S). Myristoylation, LDLRct, BC1 and Camk2a-3UTR were amplified from neuronal cDNA. $5^{\prime}$ UTRs were either amplified from neuronal cDNA or generated via gene blocks at Integrated DNA Technologies (IDT). All constructs were cloned and propagated at $30\,^{\circ}{\bf C}.5^{\prime}$ UTR and uORF mutations and eIF4G2 phosphorylation mutations were performed using a Q5 Site-Directed Mutagenesis Kit (NEB, E0554S). All clones were confirmed with forward and reverse sequencing primers using GENEWIZ Sanger sequencing services.  

# Lentivirus preparation and lentiviral transduction  

Human embryonic kidney (HEK) 293T cells (American Type Culture Collection, CRL-11268) were cultured in DMEM (Thermo Fisher Scientific, 11965092) supplied with $10\%$ FBS (Thermo Fisher Scientific, SH3007103), 2 mM L-glutamine (Thermo Fisher Scientific, 25030081), 1 mM sodium pyruvate (Thermo Fisher Scientific, 11360070) and $_{1\times}$ non-essential amino acids (Thermo Fisher Scientific, 11140076) at $37\,^{\circ}\mathrm{C}$ under $5\%\,{\bf C O}_{2}$ . The cells were passaged at $80\mathrm{-}90\%$ confluence by trypsinization fewer than 20 times to make the lentivirus and were seeded at 85,000 cells per $\mathsf{c m}^{2}$ confluence in 10-cm dishes 1 d before transfection. High-titer lentivirus mixes were prepared using Lenti-X Packaging Single Shots (Takara Bio, 631275) according to the manufacturer’s guidelines. HEK293T cells were incubated with the transfection media for $4^{-12}\,\mathrm{h}$ , after which $6\,\mathrm{ml}$ of fresh media was added onto the plates. After $48\,\mathfrak{h}$ , the supernatant was collected carefully from the dishes, centrifuged at $500g$ for 10 min and subsequently filtered through a $0.45\mathrm{{-}\upmu m}$ syringe filter. Viral soups were either flash frozen with liquid nitrogen and stored at $-80\,^{\circ}\mathbf{C}$ or used immediately. Lentivirus aliquots, which were used for the sequencing experiments, were not freeze–thawed more than once. Optimal expression of each fusion construct and the titer were determined experimentally for Pan-TurboID and TurboID-PSD95.  

# Primary cortical cultures  

Pregnant mice were purchased from Charles River Laboratories (albino, CD-1, strain 022) and treated according to Institutional Animal Care and Use Committee guidelines at The Rockefeller University. Embryonic day (E) 14.5 embryos were sacrificed in $\mathbf{1}\!\times\!\mathbf{H}\mathbf{B}\!\mathbf{S}\!\mathbf{S}$ . Dissected embryonic cortical tissues were gently dissociated and digested for 30 min at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ with a combination of Papain and DNaseI according to the manufacturer’s instructions (Worthington Biochemical, LK003150). Digested tissues were filtered through a $100\mathrm{-}\upmu\mathrm{m}$ mesh and centrifuged at 1,000 r.p.m. for 4 min at room temperature. The cells were then seeded and grown in Neurobasal Medium (Thermo Fisher Scientific, 21103049) supplemented with $2\%$ (v/v) B-27 (Thermo Fisher Scientific,  

17504044) and $1\%$ (v/v) GlutaMAX (Thermo Fisher Scientific, 35050061) at approximately 25,000 cells per $\mathbf{cm}^{2}$ on pre-coated dishes. Dishes were pre-incubated with $0.01\%$ poly-l-ornithine at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ (or room temperature) overnight, washed three times with sterile distilled water and dried before plating the cells. Additional neurons that were not going to be treated with virus were plated in separate dishes to provide conditioned media for the experimental cells. Virus was added the next day, and the media was replaced with half fresh media and half conditioned media supplied with $300\,\mathsf{n g\,m l}^{-1}$ doxycycline after $24\,\mathfrak{h}$ . Primary neuronal cells were grown in vitro for 12 d, replacing one-third of the media with half fresh and half conditioned media (with doxycycline) every 3 d.  

# Neuronal activation by KCl depolarization  

Before KCl depolarization on the 12th day, neurons were silenced with $1\,\upmu\mathrm{M}$ sodium channel blocker tetrodotoxin (TTX) (Abcam, ab120054) and $100\,\upmu\mathrm{M}$ NMDA receptor antagonist DL-2-amino-5- phosphopentanoic acid (DL-AP5) (Abcam, b120004) for 2 h at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ . TTX was dissolved in pH 4.8 citrate buffer to 1 mM, and DL-AP5 was dissolved in water to $10\,\mathsf{m M}$ as stock concentrations. Subsequently, neurons were activated for 1 h by adding warm KCl depolarization buffer ( $170\,\mathsf{m M}$ $\mathrm{KCl},2\,\mathrm{mM\,CaCl}_{2},1\,\mathrm{mM\,MgCl}_{2}$ and $10\,\mathsf{m M}$ HEPES in Neurobasal Medium) to a final concentration of $33\%$ of the total culture medium in the plate. Whole medium was then replaced with fresh neurobasal medium for $30\,\mathrm{min}$ , including biotin, TTX and DL-AP5 (biotin was included for all of the labeling experiments).  

# Neuronal activation by DHPG  

Before depolarization on the 12th day, neurons were silenced as described in the ‘Neuronal activation by KCl depolarization’ subsection. After silencing, neurons were washed with regular media twice and incubated in regular media for $15\,\mathrm{{min}}$ before they were activated with $100\,\upmu\mathrm{M}$ DHPG ((R.S.)-3,5-dihydroxyphenylglycine) (Tocris, 0805) for 10 min. DHPG medium was then replaced with fresh Neurobasal Medium for $20\,\mathrm{min}$ , during which biotinylation was performed.  

# Biotin labeling with TurboID in primary mouse cortical neurons  

Cells were incubated with $100\,\upmu\mathrm{g}\,\mathsf{m}\Gamma^{-1}$ cycloheximide (Chx) for $2\,\mathrm{min}$ before adding $100\,\upmu\mathrm{M}$ biotin for $30\,\mathrm{min}$ at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ . Inducible and controlled expression of TurboID was critical for site-specific biotinylation in neurons, consistent with the observation that constitutive expression of TurboID leads to promiscuous biotin ligase activity . For the minus biotin samples, cells were incubated with Chx only for the same amount of time as the plus biotin samples, and all were harvested at the same time. For activated neurons, biotinylation was induced after replacing the KCl media with fresh media, which included TTX and DL-AP5 that were used to silence neurons before depolarization.  

# Reporter transfection  

After 10 d in culture, primary cortical neurons were transfected using Lipofectamine LTX with PLUS reagent (Thermo Fisher Scientific, 15338030). For one well of a 12-well plate, TET $(0.6\,\upmu\mathrm{g})$ was co-transfected along with the reporter constructs $(0.6\,\upmu\mathrm{g})$ using $1.6\,\upmu\mathrm{l}$ of LTX and $1.2\,\upmu\mathrm{l}$ of PLUS reagents. Media was changed with fresh media including doxycycline $(300\,\mathsf{n g\,m l^{-1}})$ after $4^{-6}\,\mathrm{h}$ , and cells were harvested within $12^{-14}\,\mathrm{h}$ .  

# Fura-2 AM and Fluo-4 AM staining and fluorescence measurement  

Fura-2 AM (Abcam, ab120873) stock solution was prepared in DMSO at $10\,\mathsf{m M}$ . Resting cells were loaded with the Fura-2 AM dye simultaneously with the depolarized cells at a final concentration of $2\,\upmu\mathrm{M}$ . The dye was added along with the KCl solution to the depolarized cells, and the cells were incubated for $45\,\mathrm{min}$ at $37\,^{\circ}\mathbf{C}$ and $5\%\,{\bf C O}_{2}$ . Then, cells were washed with neurobasal media three times and were kept in neurobasal media without Fura-2 AM at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ for another $30\,\mathrm{min.}$ . The fluorescence was then imaged using a Keyence microscope at the excitation wavelength 340/380. Fluorescent nuclei were counted using ImageJ software.  

Fluo-4 AM (Thermo Fisher Scientific, F14201) was resuspended in DMSO to 1 mM. Similar to the Fura-2 AM strategy, resting and depolarizing cells were loaded with the Fluo-4 AM at $2\,\upmu\mathrm{M}$ simultaneously at the beginning of depolarization for $45\,\mathrm{min}$ at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ . Pluronic F-127 (Thermo Fisher Scientific, P6866) was added at $0.02\%$ to help disperse the dye in the media. Cells were then washed with regular neurobasal media three times and were kept in neurobasal media for another 30 min at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ . The fluorescence was then imaged using the Keyence microscope at the excitation wavelength 494/506. Fluorescent nuclei were counted using ImageJ.  

In addition to imaging, fluorescence by Fura-2 AM or Fluo-4 AM was measured using the Varioskan LUX plate reader and the SkanIt RE 5.0 program at excitation/emission at $340/380\,\mathrm{nm}$ and $494/506\,\mathrm{nm}$ ,respectively. Each biological replicate was calculated as the average of three wells (technical replicates) in the 96-well plate plated from the same batch of neurons.  

# PL-CLIP lysate preparation  

For the UV-crosslinking experiments, neurons were washed twice with $\mathbf{1}\!\times\!\mathbf{P}\mathbf{B}\mathbf{S}$ supplemented with $100\,\upmu\mathrm{g}\,\mathsf{m}\Gamma^{-1}$ Chx and crosslinked on $150{\cdot}\mathsf{m m}$ plates on ice in $_{1\times}$ PBS with Chx with one pulse of $400\,\mathsf{m}\mathsf{I c m}^{-2}$ and one pulse of $200\,\mathsf{m}\mathsf{I c m}^{-2}$ . As described previously by Kaewsapsak et al.95 and Hendrickson et al.96, cells were lysed in $0.5\,\mathrm{ml}$ of ice-cold RIPA buffer $50\,\mathrm{mM}$ Tris (pH 8), 150 mM KCl, $0.1\%$ SDS, $1\%$ Triton $\mathsf{X}\mathrm{{-}}100,5\,\mathsf{m M}$ EDTA, $0.5\%$ sodium deoxycholate, freshly supplemented with $0.5\,\mathsf{m M}$ dithiothreitol (DTT), $^{1\times}$ EDTA-free protease inhibitor cocktail (Thermo Fisher Scientific, 87785) and $100\,\mathsf{U}\,\mathsf{m}\mathsf{I}^{-1}$ RNaseOUT (Life Technologies, 10-777–019)) for 10 min on ice and clarified by centrifugation at 15, $.000g$ for 10 min at $4\,^{\circ}\mathbf{C}.$ Samples were diluted by adding $0.5\,\mathrm{ml}$ of Native Lysis Buffer (NLB) (150 mM KCl, $25\,\mathsf{m M}$ Tris pH 7.5, 5 mM EDTA, $0.5\%{\times}{\sf P}{\cdot}40$ ,0.5 mM DTT, $_{1\times}$ protease inhibitor and $100\,\mathsf{U}\,\mathsf{m}\mathsf{I}^{-1}$ RNaseOUT). Lysates were subsequently flash frozen or used for streptavidin pulldowns.  

# PL-Ribo-seq lysate preparation and monosome fractionation  

For the ORF-RATER experiments, three $150{\cdot}\mathrm{mm}$ cell culture dishes were combined per sample per replicate. Neurons were treated with nothing (WT), Chx $(100\,\upmu\mathrm{g}\,\mathsf{m}\mathsf{I}^{-1})$ for 2 min or harringtonine (Harr) $(2\,\upmu\mathrm{g}\,\mathsf{m}\Gamma^{-1})$ for 2 min, followed by a Chx pulse. For PL-Ribo-seq experiments, four $150{\cdot}\mathrm{mm}$ cell culture dishes were combined per sample per replicate, one-fourth of which was spared for proteomics. The PL-Ribo-seq samples were treated with Chx $(100\,\upmu\mathrm{g}\,\mathsf{m}\mathsf{I}^{-1})$ for $2\,\mathrm{min}$ , followed by a 30-min biotin pulse. Both for the ORF-RATER and PL-Ribo-seq experiments, cells were quickly rinsed in ice-cold polysome gradient buffer $.20\,\mathsf{m M}$ Tris pH 7.5, 150 mM NaCl, $5\,\mathrm{{mM}\,\mathrm{{MgCl}_{2}}}$ , freshly supplemented with 1 mM DTT and $100\,\upmu\mathrm{g}\,\mathsf{m}\vert^{-1}\,\mathsf{C h x},$ ) and then scraped and lysed on plates on ice in $^{1\,\mathsf{m l}}$ of ice-cold polysome lysis buffer $.20\,\mathsf{m M}$ Tris pH 7.5, 150 mM NaCl, $5\,\mathsf{m M}\,\mathsf{M g C l}_{2},1$ $1\%$ Triton $x{-}100$ (Thermo Fisher Scientific, 85111), freshly supplemented with $20\,\mathsf{U}\,\mathsf{m}\mathsf{I}^{-1}$ SUPERase-In RNase inhibitor (Thermo Fisher Scientific, AM2694), $24\,{\mathrm{U}}\,{\mathrm{m}}{\mathrm{I}}^{-1}$ Turbo DNase (Thermo Fisher Scientific, AM2239), 1 mM DTT, $100\,\upmu\mathrm{g}\,\mathsf{m}\vert^{-1}\mathsf{C h}\mathsf{x}$ and $_{1\times}$ EDTA-free protease inhibitor (MilliporeSigma, 11836170001)). Lysates were clarified by centrifugation at $20{,}000g$ for 2 min at $4\,^{\circ}\mathbf{C}.$ ,after which the supernatant was immediately loaded onto the cold 5-ml 7-kDa molecular weight cutoff (MWCO) Zeba desalting column (Thermo Fisher Scientific, 89892) that was previously equilibrated with the ice-cold polysome gradient buffer. This step was tested to be necessary to minimize post-lysis biotinylation and was omitted for the ORF-RATER samples. Lysates were subsequently flash frozen or used for monosome fractionation. For monosome fractionation, lysates were added $5\,\mathsf{m M C a C l}_{2}$ , incubated with micrococcal nuclease $(3\,{\mathsf{U}}\,{\mathsf{H}}{\mathsf{g}}^{-1}$ of RNA) for $45\,\mathrm{min}$ at room temperature, quenched with $6.25\,\mathrm{mM}$ EGTA and loaded on sucrose gradients as previously described . Samples were then centrifuged for 2 h at 41,000 r.p.m. in an SW-41 rotor (Beckman Coulter), and monosomes were collected using BioComp Gilson fraction collection (Gilson FC203B collector) and Triax software (BioComp Instruments).  

# PL-MS lysate preparation  

One-fourth of the lysate from the ribosome profiling experiments before spin was spared for the MS experiments. To this fraction of lysate, SDS and sodium deoxycholate were added at final concentrations of $0.1\%$ and $0.5\%$ , respectively, to increase lysis efficiency and release of membrane proteins. The additional detergent-included lysate was then incubated on ice for $10\,\mathsf{m i n}$ and clarified by centrifugation at 15,000g for 5 min at $4\,^{\circ}\mathbf{C}.$ The lysate was then desalted with the Zeba desalting column and either flash frozen or subsequently used for streptavidin pulldowns. Minus biotin samples were prepared the same way but without a biotin incubation step before harvesting the cells.  

# Streptavidin pulldowns  

For TurboID-CLIP RNA sequencing experiments, $15\%$ of the lysate was taken for input, and $50\,\upmu\mathrm{l}$ of RIPA:NLB equilibrated C1 magnetic beads (Thermo Fisher Scientific, 65-001) was added to the remaining lysate. The pulldown was allowed to proceed overnight at $4\,^{\circ}\!C.$ . Beads were then washed briefly at $4\,^{\circ}\mathbf{C}$ with ice-cold (1) RIPA buffer twice; (2) high-salt buffer (1 M KCl, $50\,\mathsf{m M}$ Tris, pH 8, 5 mM EDTA); (3) urea buffer (2 M urea, $50\,\mathsf{m M}$ Tris, pH 8, 5 mM EDTA); (4) RIPA buffer; (5) 1:1 RIPA:NLB; (6) NLB; and (7) TE ( $\mathrm{10\,mM}$ Tris, pH 7.5, 1 mM EDTA). All buffers were supplemented with $100\,\mathsf{U}\,\mathsf{m}\mathsf{I}^{-1}$ RNaseOU. The inputs and beads were then treated with sarkosyl and proteinase K $2\%$ N-lauryl sarkosyl, 10 mM EDTA, 5 mM DTT, in $1\times$ PBS, supplemented with $200\,\upmu\mathrm{g}$ of proteinase K (Roche) and 4 U of RNaseOUT) at $42\,^{\circ}\mathrm{C}$ for 1 h, followed by $55\,^{\circ}\mathrm{C}$ for 1 h to digest the biotinylated proteins and free the bound RNAs. RNA isolation was then performed using TRIzol (Thermo Fisher Scientific, 15596026) according to the manufacturer’s instructions.  

For the ribosome profiling experiments, $10\%$ of the monosome fraction was kept for input. Triton X-100 was added to the remaining monosomes to a final concentration of $0.05\%$ . The biotinylated monosomes were isolated using $50\,\upmu\mathrm{l}$ of MyOne streptavidin C1 magnetic Dynabeads that were prepared according to the manufacturer’s guidelines and washed twice with Buffer A (100 mM NaOH, $50\,\mathsf{m M}$ NaCl), twice with Buffer B $(100\,\mathsf{m M}\,\mathsf{N a C l})$ and once with polysome gradient buffer (with DTT and Chx). The monosome and bead mix was mutated at $4\,^{\circ}\mathbf{C}$ overnight. The supernatant was removed the next day, and the beads were moved into a new tube with low-salt wash buffer $^{20}\,\mathrm{mM}$ Tris pH 8, $150\,\mathsf{m M}$ NaCl, $5\,\mathsf{m M\,M g C l}_{2},$ 1 mM DTT, $100\,\upmu\mathrm{g}\,\mathsf{m}\Gamma^{-1}$ Chx and $0.1\%$ Triton X-100) and consequently washed three times for 10 min at $4\,^{\circ}\mathbf{C}$ with high-salt wash buffer ( $20\,\mathsf{m M}$ Tris pH 8, $500\,\mathrm{mM}$ NaCl, $5\,\mathrm{mM}\,\mathrm{MgCl}_{2}$ , 1 mM DTT, $100\,\upmu\mathrm{g}\,\mathsf{m}\Gamma^{-1}$ Chx and $0.1\%$ Triton X-100). The beads were then moved into a new tube with the low-salt binding buffer, and RNA extraction was performed on the pulldowns and inputs using TRIzol.  

Pulldowns for the MS and control pulldown experiments were performed on whole cell lysates described in the ‘PL-MS lysate preparation’ subsection. The lysates were diluted five-fold with $^{1\times}$ PBS. Then, $50\,\upmu\mathrm{l}$ of C1 magnetic Dynabeads was washed three times with 1× PBS and incubated with the lysates for $30\,\mathrm{min}$ at room temperature, followed by at $4\,^{\circ}\mathbf{C}$ overnight. The next day, beads were moved into a new tube with the low-salt wash buffer and washed with the high-salt wash buffer twice for 5 min at $4\,^{\circ}\mathbf{C}$ . After the high-salt washes, beads were washed once with $0.5\%$ Tween 20 (in $10\,\mathsf{m M}$ Tris pH 8) for 5 min at $4\,^{\circ}\mathbf{C}$ , once with 2 M urea (i $\phantom{-}110\,\mathrm{mM}$ Tris pH 8) for 3 min at room temperature and three times with $10\,\mathsf{m M}$ Tris pH 8 for 3 min each at room temperature, changing tubes each time for the last three washes. Changing the tubes notably reduced the detergent contamination in the samples.  

# Library preparation  

For PL-CLIP, total RNA (input and pulldown, ${\bf-100\,n g}$ ) was used as the starting material and first treated with DNAse on columns. TruSeq stranded RNA sample protocol was followed using RiboZero to get rid of ribosomal RNAs (Illumina, 20020594). For unique adapters, IDT Illumina TruSeq RNA UD Indexes (Illumina, 20022371) were used. The samples were then pooled and sequenced on an Illumina NovaSeq 6000 sequencer.  

For ribosome profiling libraries, input and pulldown samples were prepared with barcoded linkers as described previously97,98. Pooled libraries were then sequenced on the Illumina NovaSeq 6000 sequencer.  

# MS  

Proteins coupled to streptavidin-coated magnetic beads were reduced with $10\,\mathsf{m M}$ DTT and alkylated with $25\,\mathsf{m M}$ iodoacetamide (Sigma-Aldrich). Proteins were eluted from beads using partial on-bead trypsinization with $0.5\,\upmu\mathrm{g}$ of trypsin (Promega), followed by a second digestion with a 1:1 solution of $0.5\,\upmu\mathrm{g}$ of trypsin (Promega) and $0.5\,\upmu\mathrm{g}$ of LysC (Wako). Samples were then solid-phase extracted using C18 micro-purification tips constructed in-house and analyzed by liquid chromatography with tandem mass spectrometry (LC–MS/MS). For LC–MS/MS, samples were separated by reversed-phase chromatography using an analytical gradient $(98\%\,\mathsf{A},2\%$ B and $62\%$ A, $38\%$ B where A is $0.1\%$ formic acid and B is $80\%$ acetonitrile in $0.1\%$ formic acid) for 50 min in 12-cm built-in-emitter columns. Spectra were collected using an Orbitrap Fusion LUMOS (Thermo Fisher Scientific). Then, spectra were queried against a Mouse UniProt proteome FASTA database (March 2020) using Proteome Discoverer 1.4 (Thermo Fisher Scientific) and Mascot 2.4 (Matrix Science). Perseus version 1.6.10.50 was used for data analysis. Matched proteins were filtered for possible contaminants. Proteome Discoverer 1.4–calculated protein abundances (average of the three most abundant peptides for a matched protein) were $\log_{2}$ transformed. Matched proteins were filtered, keeping only proteins with signals in a minimum two-thirds of the replicates for at least one condition. Missing signals were imputed (width, 0.3; downshift, 1.8), followed by a quartile-based width adjustment normalization as described in the Perseus software documentation. Overall, 4,418 proteins were detected, and 2,776 unique ones that passed all the quality controls were used for the downstream analyses. Resting and depolarized enriched proteomes were determined with significance cutoff of $\mathsf{l o g}_{2}\!>\!0$ and $P<0.05$ using the two-tailed, paired Student’s t-test on Pan-TurboID versus TurboID-PSD95 replicates after the minus biotin counterpart was subtracted from each sample. Differential (depolarized minus resting) enriched proteome was determined with significance cutoff of $\mathsf{l o g}_{2}\!>\!0$ and $P\!<\!0.05$ using the two-tailed, unpaired Student’s $t$ -test on resting versus depolarized replicates (Supplementary Table 2).  

# IF  

In brief, neurons were grown in two-well or four-well chambered Nunc Lab-Tek II coverglasses that were previously coated with poly-l-ornithine as described in the ‘Primary cortical cultures’ subsection. Cells were rinsed with $_{1\times}$ PBS twice and fixed with $4\%$ paraformaldehyde (PFA) at room temperature for 10 min. After fixation, cells were washed with $_{1\times}$ PBS three times and permeabilized in $0.2\%$ Triton X-100 (in $_{1\times}$ PBS) for 10–15 min on ice. Cells were washed again three times with $\mathbf{1}\times\mathbf{P}\mathbf{B}\mathbf{S}$ and blocked in $3\%$ donkey or goat serum (depending on the secondary antibody) diluted in $_{1\times}$ PBS for 1 h at room temperature. Primary antibody diluted in $3\%$ serum in $_{1\times}$ PBS was added for overnight incubation at $4\,^{\circ}\mathbf{C}$ , followed by three $\mathbf{1}\times\mathbf{P}\mathbf{B}\mathbf{S}$ washes the next day and secondary antibody (diluted in $3\%$ serum in $_{1\times}$ PBS) incubation for $^{2\,\mathrm{h}}$ at room temperature. Three $\mathbf{1}\times\mathbf{P}\mathbf{B}\mathbf{S}$ washes were performed after the secondary antibody incubation, and DAPI was added during the second wash. The cells were kept in $\mathbf{1}\times\mathbf{P}\mathbf{B}\mathbf{S}$ at $4\,^{\circ}\mathbf{C}$ in dark until imaging.  

A Keyence Bz-9000e fluorescence microscope was used to image IF samples. IF images were quantified in ImageJ using the same threshold parameters across resting and depolarized conditions. To calculate the dendritic signal specifically, MAP2 co-localization was used as a proxy, and the signal surrounding the soma was subtracted. ROI Manager was used to select dendrites and measure fluorescence intensity, and background was subtracted for each image.  

# RNA FISH  

RNA FISH was performed using the RNAscope assay according to the manufacturer’s instructions. In brief, for primary cortical neurons, fixation was performed with $4\%$ PFA for $30\,\mathrm{min}$ at room temperature. The cells were then washed with $\mathbf{1}\!\times\!\mathbf{P}\mathbf{B}\mathbf{S}$ twice and dehydrated with $50\%$ EtOH, $70\%$ EtOH and $100\%$ EtOH with each incubation for $5\,\mathrm{{min}}$ , performed twice. The cells were left in $100\%$ EtOH at $-20\,^{\circ}\mathbf{C}$ at least for half an hour before proceeding with the rest of the FISH protocol. Before protease treatment, the cells were rehydrated with $70\%$ EtOH and $50\%$ EtOH for 2 min at room temperature and washed with 1× PBS. ProteaseIII from the RNAscope kit was diluted 15-fold in RNAse-free water, and the diluted protease was applied to cells for 10 min at room temperature. The rest of the incubations for the ACD probe (pTHSSe-sfGFP-C2, ACD Bio, 844781-C2) and washes were performed according to the manufacturer’s instructions. When combining FISH with IF, one of the channels was spared for IF, and the amplification step for that channel was omitted. Instead, cells were blocked with $3\%$ serum in 1× PBS for 1 h at room temperrature and then incubated with the primary antibody in $3\%$ serum overnight at $4\,^{\circ}\!C.$ . The rest of the IF protocol was followed as described in the ‘IF’ subsection. Imaging slides were kept in ProLong Gold Antifade (Thermo Fisher Scientific, P10144) with a coverslip at $4\,^{\circ}\mathrm{C}$ for a few weeks or at $-20\,^{\circ}\mathbf{C}$ for longer periods. Samples were imaged using the Keyence Bz-9000e fluorescence microscope. FISH signals were quantified using the ImageJ ‘threshold’ and ‘analyze particles’ features. Signals within DAPI and in MAP2 were counted separately to differentiate the soma and dendrites.  

# PLA  

PLAs were performed to detect nascent protein production in neurons. To detect new protein production, puromycin (Thermo Fisher Scientific, A1113803) was incorporated at $2\,\upmu\mathrm{M}$ for 10 min at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ . Cells were then washed with pre-warmed PBS-MC $\left(1\times\mathsf{P B S\,p H}\right)$ 7.4, 1 mM $\mathsf{M g C l}_{2}$ ,$0.1\,\mathsf{m M\,C a C l}_{2}$ ), fixed in PBS-MC supplemented with $4\%$ PFA and $4\%$ sucrose, washed with PBS-MC again and permeabilized in $0.5\,\%$ Triton X-100 in $^{1\times}$ PBS for 15 min as described previously99. Anti-puromycin and protein-specific antibodies were used in combination for each protein of interest. Blocking, PLA probe application, ligation and amplifications were performed using the Duolink kit (Sigma-Aldrich, DUO92101) according to the manufacturer’s recommendations. Samples were imaged and quantified as described in the ‘RNA FISH’ subsection.  

# Western blots  

Lysates were boiled at $95\,^{\circ}\mathrm{C}$ for 5 min with NuPAGE LDS Sample Buffer (NP0007) and NuPAGE Sample Reducing Agent (NP0009) and run in $4–12\%$ Bis-Tris gels. Protein transfers were performed using the iBlot2 nitrocellulose dry transfer system (at $_{20\,\upnu}$ for 6 min for proteins $^{<20}$ kDa and $25\,\upnu$ for 7 min for $>\!20\,\mathsf{k D a}_{.}$ ). The membranes were blocked in Intercept TBS Blocking Buffer for 1 h at room temperature (LI-COR Biosciences, 927-60001). TBS buffer performed better in minimizing the background signal, especially for streptavidin blots. Primary antibodies were diluted according to their specifications in the Intercept buffer and incubated with the membranes on orbital shakers overnight at $4\,^{\circ}\!C.$ . The next day, membranes were washed in TBST ( $\mathrm{1\times}$ TBS with $0.1\%$ Tween 20) for 5 min at room temperature three times and incubated with the secondary antibodies diluted in TBST for $^{2\,\mathrm{h}}$ at room temperature. If the membranes were going to be blotted for biotin, they were incubated with the streptavidin antibody diluted in TBST for 7–10 min at room temperature after the secondary antibody incubation. The membranes were washed again in TBST for 5 min at room temperature three times. Two more PBS washes were added if the membranes were blotted for streptavidin.  

The samples for reporter constructs were divided into two, one half for the western and the other for RNA extraction for each biological replicate. If sequential antibody incubation was performed, the western blots were stripped at room temperature or at $37\,^{\circ}\mathrm{C}$ (if high-affinity antibody was used) using the Restore PLUS Stripping Buffer (Thermo Fisher Scientific, 46430) for 10–15 min and subsequently blocked in the Intercept TBS Blocking Buffer before the next antibody incubation.  

# Quantitative polymerase chain reaction  

Total RNA was prepared using TRIzol after the lysates were treated with RQ1 DNase (Promega, M6101) according to the manufacturer’s instructions. cDNA was generated using iScript reverse transcription mix (Bio-Rad, 1708891), and quantitative polymerase chain reaction (qPCR) was performed using the FastStart SYBR Green Master (Roche, 04673492001). The forward and reverse primers used were: TACCGTTAGCCCCTATGCCATC and CTCGGTTGCCCATCCTCACC for Arc; ATGCTCCCCGGGCTGTATTC and GATCTTCTCCATGTCGTCCCAG for $\beta$ -Actin; AACACACAGGACTTTTGCGC and GCTCTGGTCTGCGATGGG for Fos; ATGACTGCAAAGATGGAAACG and CAGGTTCAAGGTCATGCTCT for Jun; CAGACAACCATTACCTGTCGAC and CTCTGTGGTCTTCTGGTAGACT for the GFP reporter; GCACTAAGCCGAATGCCTTCT and CTCTGTGGTCTTCTGGTAGACT for the Mphosph9 reporter; and CTCGATGCCCATCTCTACTGGT and CTCTGTGGTCTTCTGGTAGACT for the Kcnj9 reporter.  

# Antibodies and fluorescent dyes  

Fluorescent streptavidin conjugate. Streptavidin (1:5,000 for western blots and 1:10,000 for IF; Thermo Fisher Scientific, S32358).  

Primary antibodies. Puromycin (1:3,000, mouse, Kerafast, EQ0001, RRID: AB_2620162), Flag (1:3,000, mouse, Sigma-Aldrich, F1804, RRID: AB_262044), ${\upbeta}$ -Actin antibody (1:2,500, mouse, Sigma-Aldrich, A1978, RRID: AB_476692), RPL10A (1:1,000, rabbit, Abcam, ab174318), MAP2 (1:2,500, guinea pig, Synaptic Systems, 188004, RRID: AB_2138181), GFAP (1:500, rabbit, Abcam, ab7260, RRID: AB_305808), OLIG2 (1:500,rabbit, Proteintech, 13999-1-AP, RRID: AB_2157541), PSD95 (1:500, mouse, Millipore, MABN68, RRID: AB_10807979), Synaptophysin (1:300, mouse, Abcam, ab8049, RRID: AB_2198854), SHANK3 (1:500,mouse, Novus, NBP1-47610, RRID: AB_10010567), GKAP (1:500, rabbit, Novus, NBP1-76911, RRID: AB_11017331), NLGN1 (1:200, mouse, Novus, NBP2-42192), HOMER1 (1:1,000, rabbit, Proteintech, 12433- 1-AP, RRID: AB_2295573), GAPDH (1:5,000, mouse, Thermo Fisher Scientific, AM4300, RRID: AB_2536381), BAIAP2 (1:500, rabbit, Proteintech, 11087-2-AP, RRID: AB_2063075), DLGAP3 (1:500, rabbit, Proteintech, 55056-1-AP, RRID: AB_10858793), TBR1 (1:500, rabbit, Proteintech, 20932-1-AP, RRID: AB_10695502), H4 (1:1,000, mouse, Abcam, ab31830, RRID: AB_1209246), H2A.X (1:1,000, rabbit, Proteintech, 10856-1-AP, RRID: AB_2114985), EEF2 (1:1,000, rabbit, Cell Signaling Technology, 2332, RRID:AB_10693546), P-EEF2 (1:1,000, rabbit, Cell Signaling Technology, 2331, RRID: AB_10015204), eIF2α (1:1,000, rabbit, Cell Signaling Technology, 9722, RRID: AB_2230924), P-eIF2α (1:1,000, rabbit, Cell Signaling Technology, 3398, RRID: AB_2096481), p42/44 MAPK (1:1,000, rabbit, Cell Signaling Technology, 4695, RRID: AB_390779), P-p42/44 MAPK (1:1,000, rabbit, Cell Signaling Technology, 9101, RRID: AB_331646), P-IRE1 (1:500, rabbit, Novus, NB100- 2323SS, RRID: AB_10145203), IRE1 (1:500, rabbit, Novus, NB100-2324SS, RRID: AB_10000972), CHOP (1:1,000, mouse, Cell Signaling, 2895T, RRID: AB_2089254), ATF4 (1:1,000, rabbit, Cell Signaling Technology, 11815S, RRID: AB_2616025), MPHOSPH (1:300, rabbit, Biorbyt, orb100446), KCNJ9 (1:300, rabbit, LSBio, LS-C352416), KCNJ9 (1:300, mouse, Antibodies Incorporated, 75-445, RRID: AB_2686912), eIF4G2 (1:1,000, rabbit, Cell Signaling Technology, RRID: AB_10622189 and rabbit, Cell Signaling Technology, RRID: AB_2261993), NSUN3 (1:250, rabbit, LSBio, LS-C163024), MTF1 (1:300, rabbit, Novus, NBP1-86380, RRID: AB_11011361), ZFP64 (1:300, rabbit, Proteintech, 17187-1-AP, RRID: AB_2218826) and KATNBL1 (1:250, rabbit, Proteintech, 24795- 1-AP, RRID: AB_2879730).  

Secondary antibodies. The dilution used for the secondary antibodies was 1:1,000. Donkey anti-rabbit 800 (LI-COR Biosciences, 926- 32213), donkey anti-rabbit 680 (LI-COR Biosciences, 926-68073) and goat anti-mouse 800 (LI-COR, 926-32210) for western blots. Donkey anti-mouse Alexa Fluor 488 (Thermo Fisher Scientific, R37114, RRID: AB_2556542), donkey anti-guinea pig Alexa Fluor 488 (Jackson ImmunoResearch, 706-545-148, RRID: AB_2340472), donkey anti-rabbit Alexa Fluor 647 (Jackson ImmunoResearch, 711-605-152, RRID: AB_2492288) and donkey anti-mouse Alexa Fluor 555 (Thermo Fisher Scientific, A-31570, RRID: AB_2536180).  

# siRNA knockdowns  

Knockdown of eIF4G2 was performed using Lipofectamine LTX with PLUS reagent (Thermo Fisher Scientific, 15338030) at greater than $85\%$ confluency at days 8–9 in vitro for 48–72 h. The siRNA for eIF4G2 (Santa Cruz Biotechnology, sc-35170) was optimal at $20\,\mathsf{n M}$ final. A non-targeting siRNA control was included for both resting and depolarized conditions.  

For the eIF4G2 rescue experiments, knockdown of eIF4G2 was performed using 15 nM siRNA and $1.4\,\upmu\mathrm{l}$ of LTX and $\cdot\,\upmu\mathrm{l}$ of PLUS reagents to keep the neurons healthier for the next round of transfections, and neurons were transfected with the dendritic eIF4G2 variants $48\,\mathrm{h}$ after siRNA knockdown as described in the ‘Reporter transfection’ subsection. The eIF4G2 variants were conjugated to myristoylation and LDLR-C-terminal sequences to localize them in dendrites (similar to our dendritic reporters), which rendered the size of eIF4G2 to be higher and allowed us to see the levels of endogenous eIF4G2 knockdown (Extended Data Fig. 10d).  

# Calcium deprivation by EGTA  

Resting or silenced neurons were treated with 10 mM EGTA for 10 min at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ to test if the KCl-mediated effects on translation were dependent on calcium influx. For the depolarized cells, KCl was added after the EGTA treatment.  

# Induction of stress  

Neurons were treated with $0.5\,\mathsf{m M}$ sodium arsenite $(\mathsf{N a A s O}_{2})$ (Sigma-Aldrich, S7400) at $37\,^{\circ}\mathrm{C}$ and $5\%\,{\bf C O}_{2}$ for $1.5\,\mathrm{h}$ .  

# CLIP  

Neurons were washed twice with $\mathbf{1}\times\mathbf{P}\mathbf{B}\mathbf{S}$ with $100\,\upmu\mathrm{g}\,\mathsf{m}\Gamma^{-1}$ Chx and UV crosslinked on ice in the same wash buffer with one pulse of $400\,\mathsf{m}\mathsf{I c m}^{-2}$ and one pulse of $200\,\mathsf{m}\mathsf{I c m}^{-2}$ . Cells were then immediately scraped in fresh $_{1\times}$ ice-cold PBS with Chx and centrifuged at $5{,}300g$ for 5 min at $4\,^{\circ}\!C.$ . The pellets were flash frozen or processed as described previously14,100 with modifications. Five biological replicates were used, each replicate prepared from three 15-cm cell culture dishes. Pellets were resuspended in $0.5\,\mathrm{ml}$ of lysis buffer ( $\mathbf{1}\times$ PBS, $0.1\%$ SDS, $0.5\%$ NaDOC, $0.5\%{\tt N P-40}$ with freshly added protease inhibitor), and immunoprecipitations were performed overnight at $4\,^{\circ}\mathbf{C}$ using the rabbit monoclonal antibody against eIF4G2. Resting and depolarized samples were pooled after barcoding at the reverse transcription step to increase yield.  

# Bioinformatics  

PL-CLIP analysis. Transcript expression was quantified from RNA sequencing (RNA-seq) reads using salmon and mm10 UCSC knownGene gene models using the ‘TxDb.Mmusculus.UCSC.mm10.knownGene’ Bioconductor package. The longest transcript for each gene was used for all analyses. Lowly expressed genes were filtered with edgeR’s ‘filterByExpr’ command101. limma’s ‘voomWithQualityWeights’102,103 was used for differential gene expression analysis by grouping pulldown samples according to condition (rest versus dep) and bait (Pan versus PSD95). To determine transcripts that are localized in a given condition, limma was used to compare the TurboID-PSD95 pulldown to the Pan-TurboID pulldown (PSD95 − Pan). For depolarized versus resting comparisons, these contrasts were compared ((Dep-PSD95 − Dep-Pa n) − (Rest-PSD95 − Rest-Pan)). The design model included group and batch (for the four replicates). Multiple test correction was performed using the ‘decideTests’ function, using the Benjamini–Hochberg method. Localized transcripts were defined by $t$ -statistic $^{\,\,>1}$ in resting and depolarized and ${>}1.25$ for depolarized versus resting comparisons (Supplementary Table 1).  

PCA. PCA was calculated by taking the top 500 variable genes into account and using the ‘prcomp’ and ‘pcascree’ functions in RStudio. For visualization, the ‘fviz_pca_ind’ function from the ‘factoextra’ package was used.  

GSEA and GSEA-based analysis to compare RNA-seq, Ribo-seq and proteomics datasets. To identify the gene sets enriched in the relevant dendritic datasets, GSEA was performed using the ‘fgsea’ package from Bioconductor (https://www.biorxiv.org/content/10.1101/060012v3), using false discovery rate (FDR) $<0.05$ and default settings unless otherwise stated.  

For the dataset comparisons, instead of using pathways (Gene Ontology terms) in the ‘fgsea’ package, actual gene names from the list that was being compared were fed to the ‘fgsea’ function. For Fig. 2a and Extended Data Figs. 2c and 4e, genes were ranked according to dendritic enrichment in resting PL-CLIP; for Extended Data Fig. 5d,h, genes were ranked by dendritic enrichment in resting PL-Ribo-seq; for Extended Data Fig. 5j, genes were ranked by dendritic enrichment in differential PL-Ribo-seq; and for Extended Data Fig. 4b,c, genes were ranked by dendritic enrichment in resting PL-MS.  

PL-Ribo-seq data analysis. FASTQ files were processed using the Bioconductor ‘Rfastp’ package. Adapters were trimmed, and low-quality reads were removed using the same package. Demultiplexing was performed using ‘fastq-multx’. Six-nucleotide unique molecular identifiers were removed from the $5^{\prime}$ end. Finally, reads longer than 20 nucleotides were used for the following analyses. The processed FASTQ files were mapped to mm10 (UCSC) with ‘Rsubread’104. The counts of each transcript in each sample were calculated by Plastid53,58 with the annotation GTF file generated from the ‘TxDb.Mmusculus.  

UCSC.mm10.knownGene’ package of Bioconductor. Raw counts and reads per kilobase per million mapped reads (RPKM) per transcript were generated, and the longest transcript per gene was selected. The RPKMs of each sample were merged and normalized with quantile normalization.  

riboWaltz  was applied to assess ribosome profiling quality control. In brief, the transcript-aligned BAM files were generated by STAR. Then, the quality control plots, including trinucleotide periodicity and P-site phasing, were generated according to the instructions in riboWaltz.  

CDS dendritic translation for each sample was determined by normalizing TurboID-PSD95 pulldown to the average of Pan-TurboID and TurboID-PSD95 inputs. Dendritic translation in resting and depolarized neurons was calculated by taking the $\log_{2}$ transforms of normalized RPKM values of CDSs in each condition. To test the significance of dendritic translation in resting and depolarized data, we used the likelihood ratio test (LRT) in edgeR. In brief, the counting matrix of transcripts in each replicate was modeled with a genewise negative binomial generalized linear model. Then, dendritic translation and $P$ values were tested with the LRT with basemean $^{\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}$ . Differential CDS dendritic translation (depolarized minus resting) was calculated by taking the difference of the depolarized and resting $\log_{2}$ RPKM of TurboID-PSD95 pulldown and average of TurboID-PSD95 and Pan-TurboID inputs for the longest transcript per gene. To test the significance of differential expression, the $\log_{2}$ fold change of each transcript was $z$ -transformed. The P value of each transcript was calculated based on the normal distribution of z-score (cutoff: $\pm1.96$ ,$2.5\%$ top/bottom). If the basemean value for a transcript was $^{<1}$ in the differential but $^{\,\,>1}$ in the corresponding resting and depolarized sets, then the value of the highest expressed transcript was used for the differential. For each transcript, the normalized counts for each replicate were calculated. For significance cutoff, $\log_{2}$ fold change ${>}0$ with $P<0.05$ was used unless otherwise stated (Supplementary Table 3).  

To determine the ribosome occupancy in the $5^{\prime}$ UTRs, RPKMs, segregated by Ensembl transcript ID, of $5^{\prime}$ UTRs in each sample were calculated with Plastid53,58. Then, the RPKM changes for each region between resting and depolarized conditions were tested using the permutation t-test. The $P$ values were then adjusted with Bonferroni correction.  

$5^{\prime}$ UTR dendritic translation in resting and depolarized ribosome profiling data was calculated by taking the difference of the $\log_{2}$ RPKM of TurboID-PSD95 pulldown and average of TurboID-PSD95 and Pan-TurboID inputs for the longest transcript per gene with basemean cutoff of 3. Translation of all the detected uORFs was calculated by taking into account transcript levels using PL-CLIP data to establish that the observed effects are independent of RNA level changes in dendrites (t-stat for both conditions $^{<1}$ out of all the detected transcripts) and mediated by translational control (Fig. 3h). To establish more stringent, dendritically enriched translated $5^{\prime}$ UTRs, the difference between the $\log_{2}$ RPKM of TurboID-PSD95 and Pan-TurboID enrichments was considered for the longest transcript per gene in the differential data (increased with depolarization, depolarized minus resting). If the basemean value for a transcript was $^{<1}$ in the differential but >1 in the corresponding resting and depolarized sets, then the value of the highest expressed transcript was used for the differential. To determine the significance, the $\log_{2}$ fold change of each transcript was z-transformed. The P value of each transcript was calculated based on the normal distribution of z-score (cutoff: $\pm1.96,2.5\%$ top/bottom) (Supplementary Table 5).  

Dendritic targets with increased $5^{\prime}$ UTR and CDS translation were determined by ${>}0$ of $\log_{2}$ and $P<0.05$ cutoff, taking into account transcript levels determined by TurboID-PSD95 RNA-seq data (differential PL-CLIP t-stat $^{<2}.$ ). Dendritic targets with increased $5^{\prime}$ UTR but decreased CDS translation were determined as ${>}0$ of $\log_{2}$ and $P<0.05$ cutoff, taking into account transcript levels determined by TurboID-PSD95 RNA-seq data (differential PL-CLIP $t$ -stat $_{>-2}$ ) (Supplementary Table 5).  

RBP motif analysis. Motifs (in the form of position weight matrices) for RBPs were obtained from RBPmap (http://rbpmap.technion.ac.il/) for mouse72. A background list was made based on all expressed transcripts in primary cortical neurons determined by our PL-CLIP data using the longest transcript per gene. Motif search was performed on $5^{\prime}$ UTR sequences for transcripts of interest using the ‘countPWM’ function in the ‘Biostrings’ package using a minimum score of $95\%$ . Hypergeometric testing was performed to test for enrichment of motifs among dendritically translated $5^{\prime}$ UTRs (when compared to all expressed $5^{\prime}$ UTRs) with $\mathrm{FDR}<0.1.$ . The ‘seqLogo’ and ‘pheatmap’ packages were used to visualize the binding motifs and generate the heatmaps, respectively.  

RBP sites in $5^{\prime}$ UTRs were determined by defining the dendritically increased $5^{\prime}$ UTR translation as ${>}0$ of $\log_{2}$ ,$P\!<\!0.05$ in $5^{\prime}$ UTR PL-Ribo-seq and dendritically increased CDS as ${>}0$ of $\log_{2}$ ,$P<0.05$ and dendritically decreased CDS as ${<}0$ of $\log_{2}$ ,$P\!<\!0.05$ (Fig. 5a) in CDS PL-Ribo-seq, taking into account transcript levels determined by TurboID-PSD95 RNA-seq data (differential PL-CLIP $t$ -stat $^{<2}$ for increased and t-stat $_{>-2}$ for decreased CDS targets). For comparison of RBP binding sites in dendritic $5^{\prime}$ UTRs enriched in resting, depolarized and differential (depolarized minus resting) PL-Ribo-seq (Extended Data Fig. 7a), the top approximtely 800 significant genes in each category were considered based on the basemean cutoffs mentioned in the ‘PL-Ribo-seq data analysis’ section $5^{\prime}$ UTR $P\!<\!0.01\!\$ ). For this analysis, transcript levels were taken into consideration, adjusting the lists according to PL-CLIP for each condition (resting and depolarized: $t\cdot$ -stat $^{<1}$ and differential: t-stat <1.3).  

ORF-RATER analysis. For processing of ribosome profiling data, linker sequences were removed from sequencing reads, and samples were de-multiplexed using FASTX-clipper and FASTX-barcode splitter (FASTX-Toolkit). Unique molecular identifiers and sample barcodes were then removed from reads using a custom Python script. Bowtie version 1.1.2 was used to filter out reads aligning to rRNAs and contaminants, and all surviving reads were aligned to the mouse transcriptome with TopHat version 2.1.1 using –b2-very-sensitive– transcriptome-only–no-novel-juncs–max-multihits $=64$ flags. These alignments were assigned a specific P-site nucleotide using a 12-nucleotide offset from the $3^{\prime}$ end of reads. The ORF-RATER pipeline (https://github.com/alexfields/ORF-RATER) was run starting with the BAM files as previously described58,105. All uORFs longer than nine nucleotides including the stop codon were considered with orfrater score of $>\!0.7.$ For uORF comparisons with eIF4G2 CLIP and PL-Ribo-seq datasets, orfrater score $>\!0.6$ was used.  

CLIP analysis. CLIP libraries were sequenced on an Illumina MiSeq to obtain 75-nucleotide, single-end reads. CLIP reads were processed as described previously106,107 using CLIP Tool Kit software to filter for quality, demultiplex, remove $5^{\prime}$ and 3′ linker sequences and collapse exact duplicates. The resulting reads were mapped to the mm10 genome using the ‘align’ function from the ‘Rsubread’ package104, allowing a maximum of five mismatches and a minimum fragment length of 20. $5^{\prime}$ UTR sequences were extracted from the ‘TxDb.Mmusculus.UCSC. mm10.knownGene’ R package, and ‘summarizeOverlaps’ from the ‘GenomicAlignments’ package108 was used to count eIF4G2-CLIP reads over each $5^{\prime}$ UTR. The longest transcript was used for each gene. The number of reads mapping to each $5^{\prime}$ UTR was normalized for library depth, and $\log_{2}$ (fold change, depolarized versus resting) values were calculated using a pseudocount of 0.1. Binomial tests were performed using the normalized CLIP tag values (Supplementary Table 6). To test the dendritic translation and localization of eIF4G2-bound targets in response to depolarization, $\log_{2}$ fold change ${>}0$ and $P\!<\!0.2$ were used for eIF4G2 CLIP. To intersect eIF4G2-bound targets with the set of RNAs with enhanced $5^{\prime}$ UTR or CDS translation in dendrites, fold change >1 was used for the eIF4G2 CLIP and PL-Ribo-seq (with basemean $^{\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}{}^{\,\,\,\,\,\,\,\,\,}$ and differential PL-CLIP $t$ -stat $^{<2}$ ) lists.  

# Statistics and reproducibility  

Representative images in Figs. 1b,c,e, 3a,b and 4a,b and Extended Data Figs. 1a,c–e,g and 6d,e,g,i were replicated independently at least three times. Biological replicates were processed (for reporter western blots, imaging and qPCRs) by E.H. and N.N. independently and were replicated independently. The number of biological replicates for each corresponding experiment is reported in the figure legends. No data exclusion was performed.  

No sample size calculation was performed, but extensive work in sequencing and MS, particularly performed in neurons, informed our choices of minimal number of sample sizes to provide the required statistical power1,13,14,100. We increased the number of biological replicates to four and five for our sequencing and MS experiments, respectively, to improve the statistical power to be able to identify the differences between resting and depolarized conditions. Four biological replicates were chosen for PL-CLIP, three for PL-Ribo-seq, five for PL-MS and five for eIF4G2 CLIP. For all the other streptavidin immunoprecipitation and reporter experiments, at least three biological replicates were chosen. For the imaging studies, at least two biological replicates were chosen with multiple fields from each biological replicate that would represent the whole slide, and the number of images and fields studied for each figure is reported in the corresponding figure legends.  

Data distribution was assumed to be normal, but this was not formally tested. The details of all the statistical tests used are reported in the respective figure legends.  

The primary cortical neuron cell culture plates were randomized when deciding resting versus activated conditions or Pan-TurboID versus TurboID-PSD95 virus addition. All the resting and activated neuronal samples from the same biological replicate were processed simultaneously for all the experiments.  

Processing of the reporter samples and imaging of resting versus activated neurons were blinded. Further blinding was not possible during the preparation of samples for RNA-seq, ribosome profiling, MS and CLIP because different conditions needed to be identified for downstream processing.  

# Reporting summary  

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.  

# Data availability  

All sequencing data generated in this study were deposited in the Gene Expression Omnibus with accession number GSE213083. Proteomics data were deposited in the ProteomeXchange Consortium via the PRIDE79 partner repository with dataset identifier PXD050222. The databases used in this study include the Mouse UniProt proteome FASTA database (March 2020), the mm10 UCSC mouse genome and RBPmap version 1.1 (http://rbpmap.technion.ac.il/). We have the rights to publish BioRender figures, and Figs. 1a,d, 3a, 4a and 6 were created with BioRender. All other data used in this study are available or are described in the paper or supplementary materials. Source data are provided with this paper.  

# Code availability  

All packages and software described in the Methods section were used according to standard protocols. Custom code was not generated in this study.  

# References  

95.	 Kaewsapsak, P., Shechner, D. M., Mallard, W., Rinn, J. L. & Ting, A. Y. Live-cell mapping of organelle-associated RNAs via proximity biotinylation combined with protein-RNA crosslinking. eLife 6, e29224 (2017).   
96.	 Hendrickson, G. D., Kelley, D. R., Tenen, D., Bernstein, B. & Rinn, J. L. Widespread RNA binding by chromatin-associated proteins. Genome Biol. 17, 28 (2016).   
97.	 Jan, C. H., Williams, C. C. & Weissman, J. S. Principles of ER cotranslational translocation revealed by proximity-specific ribosome profiling. Science 346, 1257521 (2014).   
98.	 McGlincy, N. J. & Ingolia, N. T. Transcriptome-wide measurement of translation by ribosome profiling. Methods 126, 112–129 (2017).   
99.	 Dieck, S. T. et al. Direct visualization of newly synthesized target proteins in situ. Nat. Methods 12, 411–414 (2015).   
100.	Sawicka, K. et al. FMRP has a cell-type-specific role in CA1 pyramidal neurons to regulate autism-related transcripts and circadian memory. eLife 8, e46919 (2019).   
101.	 Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010).   
102.	Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47 (2015).   
103.	Liu, R. et al. Why weight? Modelling sample and observational level variability improves power in RNA-seq analyses. Nucleic Acids Res. 43, e97 (2015).   
104.	Liao, Y., Smyth, G. K. & Shi, W. The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads. Nucleic Acids Res. 47, e47 (2019).   
105.	Chen, J. et al. Pervasive functional translation of noncanonical human open reading frames. Science 367, 1140–1146 (2020).   
106.	Moore, M. J. et al. Mapping Argonaute and conventional RNA-binding protein interactions with RNA at single-nucleotide resolution using HITS-CLIP and CIMS analysis. Nat. Protoc. 9, 263–293 (2014).   
107.	 Shah, A., Qian, Y., Weyn-Vanhentenryck, S. M. & Zhang, C. CLIP Tool Kit (CTK): a flexible and robust pipeline to analyze CLIP sequencing data. Bioinformatics 33, 566–567 (2017).   
108.	Lawrence, M. et al. Software for computing and annotating genomic ranges. PLoS Comput. Biol. 9, e1003118 (2013).  

# Acknowledgements  

We thank C. Zhao and C. Lai from the Genomics Core and H. Molina from the Proteomics Core for their insights, advice and support and members of the Darnell laboratory at The Rockefeller University for thoughtful discussions and review of the paper. We also thank A. Y. Ting and previous members of the Weissman laboratory, C. H. Jan and E. A. Costa, for their extensive help and input on the initial discussions about TurboID. This work was supported by Helen Hay Whitney and Kavli Neuroscience Institute postdoctoral fellowships awarded to E.H. and National Institutes of Health grants NS081706 and R35NS097404 awarded to R.B.D. R.B.D. and J.S.W. are investigators of the Howard Hughes Medical Institute.  

# Author contributions  

E.H., J.S.W. and R.B.D. conceived and designed the study. E.H. administered the project, conducted all the experiments and performed all the analyses. C.R.H. contributed to the RNA-seq, CLIP and RBP analyses. N.N. contributed to making primary cortical cultures, imaging and performing the reporter and eIF4G2 experiments. J.L. and J.C. contributed to ribosome profiling analyses. J.J.F. prepared the RNA-seq libraries. M.S. performed the mouse dissections for primary cortical cultures. E.H., J.S.W. and R.B.D. obtained funding. E.H., J.S.W. and R.B.D. wrote and reviewed the paper. All authors edited the paper.  

# Competing interests  

The authors declare no competing interests.  

# Additional information  

Extended data is available for this paper at https://doi.org/10.1038/s41593-024-01615-5.  

Supplementary information The online version contains supplementary material available at https://doi.org/10.1038/s41593-024-01615-5.  

Correspondence and requests for materials should be addressed to Ezgi Hacisuleyman, Jonathan S. Weissman or Robert B. Darnell.  

Peer review information Nature Neuroscience thanks Nahum Sonenberg and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.  

Reprints and permissions information is available at www.nature.com/reprints.  

![](images/eec97999b8c7ded7dd3fa96a71c86c6e6ad9602240a0f49eead0da2a488e4542.jpg)  

Extended Data Fig. 1 | See next page for caption.  

Extended Data Fig. 1 | Labeling by TurboID and activation by KCl or DHPG are used to study resting and activated dendritic molecular profiles in primary cortical neurons. a, Immunofluorescence (IF) images of primary cortical neurons immunostained for glial fibrillary acidic protein (GFAP) and oligodendrocyte transcription factor 2 (OLIG2) simultaneously with PSD95 to show that the cultures are devoid of glial cells or oligodendrocytes, respectively. DAPI for nuclei; PSD95 for excitatory neurons. Magnification, $\times40.$ . Scale bars, $50\,\upmu\mathrm{m}$ . b, TurboID-PSD95 was cloned without (top row) and with (bottom row) its $5^{\prime}$ and $3^{\prime}$ UTRs and lentivirally expressed in primary cortical neurons. White dashed boxes are zoomed in areas in black&white images. DAPI for nuclei; MAP2 for dendrites; Flag for each TurboID. $\%$ dendritically localized TurboIDPSD95 is quantified by co-localization with MAP2 signal in ImageJ. 3 different areas of images per replicate $(n\!=\!3)$ . Magnification, $\times20,$ Scale bars, $50\,\upmu\mathrm{m}$ .Significance was derived from biological replicates, showing the center line at mean. c, IF images of TurboID-PSD95-transduced neurons immunostained for DAPI (blue, for nuclei), PSD95 (red, for endogenous PSD95) and TurboIDPSD95 (cyan, detected by Flag). Magnification, $\times60$ . Scale bar, $50\,\upmu\mathrm{m}$ . d, IF images show the expression of a presynaptic marker, Synaptophysin (cyan), and TurboID-PSD95 (red, detected by Flag antibody) in primary cortical neurons transduced with TurboID-PSD95. DAPI (blue) marker for nuclei. Three zoomed in regions are marked by the white boxes. Magnification, $\times60.$ Scale bar, $10\,\upmu\mathrm{m}$ .e, IF images show TurboID expression and biotinylation in primary cortical neurons transduced with TurboID-PSD95 or Pan-TurboID after 30 minutes of biotin incubation. DAPI (blue, nuclei); MAP2 (green, dendrites); Flag (red, TurboID); and Streptavidin (cyan, biotinylated proteins). Magnification, $\times20$ .Scale bars, $50\,\upmu\mathrm{m}$ . f, Western blots stained for Flag and ${\upbeta}$ -Actin from Pan-TurboID and TurboID-PSD95-transduced neurons in the absence (−) or presence $\left(+\right)$ of exogenous biotin shown to indicate the relative expression levels of TurboID proteins. Quantifications of TurboID protein levels normalized to ${\upbeta}$ -Actin are shown on the right $\left(n\!=\!3\right)$ ; relative levels are not significant by two-tailed, paired Student’s t-test. g, Western blots stained for streptavidin signal in inputs (‘in’) and streptavidin pulldowns (‘pd’) from Pan-TurboID or TurboID-PSD95-transduced neurons in the absence $(-)$ or presence (+) of exogenous biotin. h, Streptavidin pulldowns shown for dendritic (SHANK3, GKAP, NLGN1 and HOMER1) and negative control (GAPDH) proteins from TurboID-PSD95-transduced neurons in the absence $(-)$ or presence $\left(+\right)$ of exogenous biotin. Flag signal indicates self-biotinylation of each construct. Loaded on the gel are $10\%$ (by volume) of input and $50\%$ (by volume) of pulldowns. Percent isolated by TurboID-PSD95 in each condition is calculated by dividing the signal in the pulldown lane by that of the input lane, after each is adjusted to total, and quantifications are shown as bar graphs $(n\!=\!3)$ . P values: Flag $=0.58$ ,$\mathsf{S H A N K}3\!=\!0.0061$ ,$\mathsf{G K A P}=0.018$ ,$\mathbf{NLGN1}\!=\!0.00052$ , HOMER1 $=0.021$ ,${\mathsf{G A P D H}}\!=\!0.42.$ i, Streptavidin pulldowns  

shown for dendritic (BAIAP2 and DLGAP3) and nuclear (TBR1, H4 and H2AX) proteins from Pan-TurboID and TurboID-PSD95-transduced neurons in the presence (+) of exogenous biotin. Loaded on the gel are $10\%$ (by volume) of input and $50\%$ (by volume) of pulldowns. Percent isolated by each TurboID is calculated as in (h) $\left(n\!=\!3\right)$ . P values: BAIAP2 $:=0.0052$ , DLGAP3 $\mathsf{i}=0.0035$ , TBR $1\,{=}\,0.0063$ ,$\mathsf{H}4\,{=}\,0.018,\mathsf{H}2\mathsf{A}\mathsf{X}\,{=}\,0.0037.\mathbf{j}$ , Phosphorylation of EEF2, eIF2α, ERK1/2 and IRE1 and total levels of ATF4 and CHOP are shown in resting (rest), activated (DHPG, Dep) and stressed (Sodium arsenite $(\mathtt{N a A s O}_{2})$ ) cells by using phospho-specific and total antibodies. The amount of phosphorylated or total protein is shown in the bar graphs, calculated by dividing the phosphorylated signal to total and ${\upbeta}$ -Actin for the phosphorylated proteins and by dividing the total to ${\upbeta}$ -Actin for ATF4 and CHOP $(n\!=\!3)$ . Significance was calculated with respect to rest. $P$ values: P-EEF2 $\mathbf{DHPG}=0.0088$ , Dep $=0.0023$ ,$\mathrm{NaAsO}_{2}\,{=}\,0.039)$ ), P-eIF2α $\mathbf{\nabla}(\mathbf{D}\mathsf{H}\mathbf{P}\mathbf{G}\!=\!0.018$ ,D$\mathtt{o e p}\!=\!0.0034$ ,$\mathsf{N a A s O}_{2}\!=\!0.028)$ , P-ERK1/2 $\mathbf{\nabla}\mathbf{D}\mathsf{H}\mathbf{P}\mathbf{G}\!=\!0.015$ , De ${\sf7}\!=\!0.0067,$ ,$\mathsf{N a A s O}_{2}\!=\!0.00084_{\cdot}$ ), P-IRE1 $\mathsf{P G}\,{=}\,0.06,\mathsf{D e p}\,{=}\,0.37,\mathsf{N a A s O}_{2}\,{=}\,0.0027)$ , ATF4 $(\mathrm{DHPG}=0.038,\mathrm{Dep}=0.42,\mathrm{NaAsO}_{2}=0.016)$ , CHOP (DHPG = 0.044, Dep $=0.18$ ,$\mathsf{N a A s O}_{2}\!=\!0.024)$ . k, Quantitative PCR (qPCR) results shown for immediate early genes, Arc, Fos and Jun. The fold changes for each gene are calculated by first normalizing to the house-keeping gene $\beta$ -Actin in each condition, then dividing the value of each condition by that of the resting state $(n\!=\!3)$ . l, Dendritic spine size in resting and KCl-depolarized neurons are measured using the Keyence microscope. Red squares are examples of spines that are counted $^{\prime}\!n\!=\!3,12$ spines from each biological replicate are counted as technical replicates). Significance was derived from the biological replicates using the two-tailed, unpaired Student’s t-test. Box plots show the min and max, with the center line at median. Magnification, $\times100$ . Scale bars, $5\,\upmu\mathsf{m}.\mathsf{m}$ , Fluo-4-AM staining in resting, KCldepolarized and DHPG-depolarized cells. Fluo4-AM was loaded in resting cells and measurements were taken at indicated time points after Fluo4-AM removal. In depolarized cells, the dye was loaded during silencing. After silencing, fluorescence was measured during stimulus at 10, 30 and 60-minute time points for the KCl treatment and at 10-minute for the DHPG-induced activation. Fluorescence was also measured 60 minutes after the stimulus removal (60′post KCl and 60′post DHPG). Circles represent data from 2 biological and 3 technical replicates. Below: Examples of Fluo4-AM fluorescence are shown in resting, 10-minute KCl-treated and 10-minute DHPG-treated neurons. Fluo4-AM loading (45 minutes) was performed during the last 45 minutes of the silencing step prior to stimulus addition for the KCl and DHPG treatment and simultaneously for the resting neurons. Imaging was performed 10 minutes after the stimulus was added. Scale bars, $50\,\upmu\mathrm{m}.$ .$(\mathbf{b},\mathbf{f},\mathbf{h}{\cdot}\mathbf{k},\mathbf{m})$ Data are mean $\pm\,\mathbf{s.d}$ Significance was calculated using the two-tailed, paired Student’s t-test. P values: ns (not significant) $>\!0.05$ ;$^{*}{<}0.05$ ;$^{**}\!<\!0.01$ ; \*\*\* $<\!0.001$ ; \*\*\*\* $<\!0.0001.n$ indicates the number of biologically independent samples.  

![](images/cd8909e2e553e688c206b7ca70b261f53f857effd4e6920d83a8ac4f17f4d06a.jpg)  
Extended Data Fig. 2 | See next page for caption.  

Extended Data Fig. 2 | PL-CLIP identifies dendritic RNAs. a, The expression of neuronal (mouse CA1 pyramidal and developing forebrain) and non-neuronal (oligodendrocytes, astrocytes, glial cells, microglial cells, endothelial cells and mural cells) transcripts are assessed among RNAs detected by PL-CLIP. Significance was determined by the two-sided Wilcoxon signed-rank test. $P$ values: Neuron and CA1 pyramida $<2.2\mathrm{e}^{-16}$ , Oligodendrocyte $=\!0.00013$ ,Astrocyte $\scriptstyle=1.71\mathrm{e}^{-8}$ , Glia $<2.2\mathrm{e}^{-16}$ , Microglia $=2.77\mathrm{e}^{-12}$ , Endothelia $=0.21$ , Mural $=\!0.00043.$ b, Principal component analysis (PCA) shown for Pan-TurboID and TurboID-PSD95 inputs (left) and streptavidin pulldowns (right) in resting and depolarized PL-CLIP $.n\,{=}\,4$ biologically independent samples). Inputs separate only by neuronal state but not by TurboID; whereas, pulldowns are differentiated by state and TurboID (localization). c, Comparison of PL-CLIP-enriched RNAs with previously published dendritic29 and axonal36–39 RNAs ( $\mathsf{F D R}<0.05)$ . Multiple testing correction was performed using the Benjamini-Hochberg method. d, Cumulative distribution function (CDF) plot of PL-CLIP RNAs, dendriteenriched/present RNAs (left) and FMRP CA1 targets (right) from . FC: fold change. Significance was calculated by the two-sided Wilcoxon signed-rank test.  

$P$ values: Dendrite present and enriched $<\!2.2\mathrm{e}^{-16}$ , CA1 FMRP targets $=\!0.00059$ ,Dendritic FMRP targets ${\it\Omega}=3.69{\bf e^{-10}}$ , FMRP cell body targets $<\!2.2\mathrm{e}^{-16}$ , FMRP synaptic targets $=2.23{\bf e}^{-13}$ . e, Examples of dendritically-enriched RNAs that encode chromatin, Golgi and protein complex localization-related proteins ${n=4}$ , values from PL-CLIP). $P$ values: $C b x\!=\!0.049,F a n c a\!=\!0.026,F o x o3\!=\!0.012,$ $R a d S O=0.0022,P d e9a=0.019,A b c a l=0.032,H a p I=0.035,N p t x r=0.022.$ Significance was calculated using the two-tailed, paired Student’s t-test. f, Left: Dendritic enrichment of Chop, Ire1 and $A t f4$ levels in resting and depolarized neurons. Dendritic enrichment $(\log_{2})$ is calculated by normalizing the transcript levels in TurboID-PSD95 to those in Pan-TurboID. $P$ values: Chop $\phantom{-}1\,{=}\,0.038$ ,$I r e l\!=\!0.47,A t f\!4\!=\!0.99$ . Right: Total Chop, Ire1 and Atf4 expression levels $(\mathsf{l o g}_{2}\mathsf{c p m})$ are shown in the inputs from resting and depolarized Pan-TurboID-transduced neurons ${}\langle n\!=\!4$ , values from PL-CLIP). $P$ values: $C h o p\!=\!0.14,I r e l\!=\!0.95,$ $A t f\!4=\!0.052$ . Significance was calculated using the two-tailed, unpaired Student’s t-test. $P$ values: ns (not significant) $>\!0.05;^{*}\!<\!0.05;^{**}\!<\!0.01;^{***}\!<\!0.001;$ \*\*\* $^{\circ}{<}0.0001.r$ indicates the number of biologically independent samples.  

Extended Data Fig. 3 | PL-CLIP identifies properties of dendritic RNAs in resting and depolarized neurons. a, Length and (b) GC content of resting and depolarized PL-CLIP-enriched dendritic RNAs and all detected RNAs in cortical neurons (‘All’, $n\!=\!17{,}654^{"}\!$ ) are compared. Colors: salmon (resting, $n\,{=}\,2{,}788\$ ); burgundy (depolarized, $n\,{=}\,3,727)$ . Significance was calculated by the two-sided Wilcoxon signed-rank test. Box plots show lower and upper hinges corresponding to the first and third quartiles (representing 25th and 75th percentile, respectively). Whiskers extend from the hinge to the 1.5x interquartile range. The center line indicates the median. P values: all significance values reported are $<\!2.2\mathrm{e}$ -16 except for Transcript length (Rest vs. Dep $=4.18\mathrm{e}{-15}$ ), CDS (Rest vs. Dep $=0.081$ ), 5′UTR (All vs. Rest $=2.10\mathrm{e}{-8}$ ; All vs. $\mathsf{D e p}\!=\!4.22\mathrm{e}\!\cdot\!10$ ;Rest vs. Dep $=0.93$ ), 3′UTR (All vs. Rest $=0.15$ ). c, RNA fluorescence in situ  

![](images/033ab43d4ee32bdf9a2ff85016669cf9fe5947c3c5b3686f6db6c055e54537c9.jpg)  
hybridization (RNA-FISH) combined with MAP2 IF to show dendrites and dendritically de-enriched (Snca) and enriched (Kmt2d, Map2 and Dlg4) RNAs by resting PL-CLIP. Images are quantified on the right $(n\,{=}\,3)$ . Significance was derived from biological replicates, showing the center line at median and using the two-tailed, paired Student’s t-test. d, RNA-FISH combined with IF showing dendritic (MAP2-green) localization of Rapgef4 (red) and Ppp1r9b (white) in resting and depolarized neurons, (e) quantified for soma and dendrites. 4 fields of images per replicate for $n\!=\!2$ presented as mean $\pm s.\mathbf{d}.$ All images are taken at $\times40$ magnification, except for Dlg4, which was taken at $\times60.P$ values: ns (not significant) $\vdash0.05;^{*}<0.05;^{**}<0.01;^{***}<0.001;^{****}<0.0001$ Scale bars, $100\,\upmu\mathrm{m}$ .$n$ indicates the number of biologically independent samples.  

![](images/d9fc15daff6bf104313bf838864214979a5b33f2e6e3f09277ecbfec931ce945.jpg)  
Extended Data Fig. 4 | PL-MS identifies the dendritic proteome and its relation to local RNA levels. a, PCA plot showing five replicates of PL-MS from Pan-TurboID and TurboID-PSD95-transduced resting and depolarized neurons. Minus biotin counterpart of each sample is subtracted for each dot on the plot. b, Comparison of PL-MS-enriched proteome with previously published dataset $(\mathsf{F D R}<0.05)$ ). c, TurboID-PSD95-enriched proteome vs. Distler et al. data $(\mathsf{F D R}<0.05)$ ). Multiple testing correction was performed using the BenjaminiHochberg method. d, Pearson correlation of PL-MS and PL-CLIP on dendritic   
RNAs in resting neurons. e, Shown are the distributions of PL-MS-enriched dendritic proteome from resting (rest), depolarized (dep) and differential (dep minus rest) data in ranked resting PL-CLIP using a GSEA-based strategy $(\mathsf{F D R}<0.05)$ (Methods). Multiple testing correction was performed using the Benjamini-Hochberg method in the fgsea package. P adjusted: Rest $=0.0026$ ,Dep $\it1=0.18$ , Dep-Rest $=\!0.0027.P$ values: $^{**}{<}0.01$ . f, Pearson correlation of PL-MS and PL-CLIP on dendritic RNAs in depolarized neurons. (d,f) The reported $P$ values are two-sided.  

![](images/06e4d03d2a2b5127d8c665156a9655c059ac8f77e1f180679ada975bbb0f2826.jpg)  
Extended Data Fig. 5 | See next page for caption.  

Extended Data Fig. 5 | PL-Ribo-seq accurately identifies locally translated RNAs and their relation to local protein levels in dendrites. a, Percent RPL10A recovered from pulldown samples from Pan-TurboID or TurboIDPSD95-transduced neurons was calculated by dividing the pulldown band $(100\%$ volume loaded) value by that of the input band $10\%$ volume loaded) $\left(n=3\right)$ . The center line is at median. Significance was calculated using the two-tailed, paired Student’s t-test. b, Quality metrics of ribosome profiling data on the streptavidin pulldown fraction of TurboID-PSD95 in resting neurons shown on the top row. The coverage in the CDS and UTRs are shown. Similar metrics are shown for the input fraction in the bottom two rows: the read length distribution, percentage of P-sites in the CDS and UTRs along with the length of each region, the P-site signal in each reading frame and the P-site coverage from the start and stop codons. c, PCA shown for Pan-TurboID and TurboID-PSD95 input (left) and streptavidin pulldown (right) fractions in resting and depolarized neurons for three replicates of PL-Ribo-seq. d, Comparison of TurboID-PSD95-enriched translated RNAs in resting neurons with previously published sets of dendritic and axonal translatomes using a GSEA-based strategy. $\mathbf{\left^{\lceilFDR<0.05\right\rceil}}$ ). e, Ribosome profiling RPKM read distribution in the coding sequences (CDS) of all detected RNAs (n of Res $_{:=18,855}$ , n of Dep $_{1}=18{,}887;$ ) in inputs and streptavidin pulldowns from PanTurboID and TurboID-PSD95-transduced resting and depolarized neurons. RPKM changes for each region between resting and depolarized conditions were tested using the permutation t-test. The $P$ values were then adjusted with Bonferroni  

correction. Box plots show lower and upper hinges corresponding to the first and third quartiles (representing 25th and 75th percentile, respectively). Whiskers extend from the hinge to the 1.5x interquartile range. The center line indicates the median. f, Puromycin incorporation in resting and depolarized neurons shown by IF using a puromycin-specific antibody. DAPI (blue) marker for nuclei, red for puromycin. Magnification, $\times20,$ . Corrected fluorescence per cell was calculated by counting 45 cells $\left(n\!=\!3\right)$ . Significance was derived from the biological replicate averages and calculated using the two-tailed, unpaired Student’s t-test. The center line is at median. Scale bars, $50\,\upmu\mathrm{m}$ . g, Pearson correlation of PL-MS and PL-Ribo-seq on dendritic RNAs in resting neurons. h, Shown are the distributions of PL-MS-enriched dendritic proteome from resting (rest), depolarized (dep) and differential (dep minus rest) data in ranked resting PL-Ribo-seq using a GSEA-based strategy $\mathrm{FDR}<0.05)$ .$P$ adjusted: Rest $=3.70{\bf e^{-9}}$ ,$\mathsf{D e p}\!=\!0.0019_{\cdot}$ ,Dep-Rest $=0.69.$ i, Pearson correlation of PL-MS and PL-Ribo-seq on dendritic RNAs in depolarized neurons. j, Shown are the distributions of PL-MS-enriched dendritic proteome from resting, depolarized and differential data in ranked differential PL-Ribo-seq using a GSEA-based strategy $(\mathsf{F D R}<0.05)$ ). $P$ adjusted $=$ $\mathtt{R e s t\!=\!0.00083}$ , D $\mathbf{ep}\!=\!1.60\mathbf{e}^{-13}$ , Dep-Rest $=6.95\mathrm{e}^{-16}$ .$P$ values: \*\* $<\!0.01$ ; \*\* $^{*}\!<\!0.001$ ;$*****<0.0001.n$ indicates the number of biologically independent samples. (d,h,j) Multiple testing correction was performed using the Benjamini-Hochberg method. $({\bf g},{\bf i})$ The reported $P$ values are two-sided.  

![](images/822ad47ed29be7c8bcf0c83d03efb3428819b6bd58f06d40ef0b2041c7ad84f0.jpg)  
Extended Data Fig. 6 | See next page for caption.  

Extended Data Fig. 6 | Dendritically localized reporters reveal the effects of ${\bf\delta}^{\prime}$ UTRs on the downstream translation in an activity-dependent manner. a, Shown is the Pearson correlation of differential (depolarized minus resting) PL-Ribo-seq counts $(\mathsf{l o g}_{2}\,\mathsf{R P K M})$ from 5′UTR and CDS across all transcripts detected by dendritic ribosome profiling. Correlation coefficient, R, and the two-sided P value are shown in the upper right corner. b, Results of ORF-RATER analysis on all detected ORFs (top) and start codon usage (bottom). c, Dendritic translation of uORFs that are identified by ORF-RATER in DHPG-activated PL-Ribo-seq (n of all 5′UTRs $=11$ 0,313; $\mathtt{u O R F S=881}_{,}$ ). Significance was calculated by the one-sided Wilcoxon signed-rank test. Box plots show lower and upper hinges corresponding to the first and third quartiles (representing 25th and 75th percentile, respectively). Whiskers extend from the hinge to the 1.5x interquartile range. The center line indicates the median. d, Different dendritic localization signals, Camk2a-3′UTR and BC1, are tested by streptavidin pulldowns from TurboID-PSD95 expressing cortical neurons. e, FISH and IF performed on superfolder GFP localized with BC1, Camk2a-3′UTR and myr-LDRct localization signals. DAPI shown for nuclei. For FISH and IF, GFP RNA and protein (by Flag antibody) are targeted, respectively. Magnification, $\times20,$ Scale bars, $25\,\upmu\mathrm{m}.$ f, ORF-RATER ribosome coverage in the 5′UTR and CDS of Kcnj9, with a non-cognate start codon in its uORF, GTG. Harr: harringtonine-treated; Chx: cycloheximide-treated; Unt: untreated neurons. g, Shown are western blots that are quantified in Fig. 4e and f. GFP protein detected by Flag, and ${\upbeta}$ -Actin used as loading control. h, GFP fold changes of dendritic reporters with Cmc4, Lrrc51 and Nsun3 5′UTRs are quantified by Flag and ${\upbeta}$ -Actin western blots and qPCRs upon neuronal activation by DHPG ( $\ln=3$ biologically independent samples). All data are presented as mean ±s.d. Significance was calculated using the two-tailed, unpaired Student’s t-test. $P$ values: Cmc4 (protein $=\!0.00016$ ;$\mathsf{R N A}\!=\!0.64$ ), Lrrc51 (protein $=0.0010$ ; RNA $=0.98$ ), Nsun3 (protein $=\!0.0027$ ;$\mathsf{R N A}\!=\!0.42,$ ). i, Larger fields of IFs for MPHOSPH9 (downregulated) and KCNJ9 (upregulated) in resting and depolarized (dep) neurons in support of Fig. 4i. Magnification, $\times20$ .Scale bars, $25\,\upmu\mathrm{m}.P$ values: ns (not significant) $>\!0.05;^{\ast}\!<\!0.05;^{\ast\ast}\!<\!0.01;^{\ast\ast\ast}\!<\!0.001$ ;\*\*\*\* <0.0001.  

![](images/42996493bec6927067c2ce58082999fff8fd3881ffd31b301dc0cce330fd82a3.jpg)  

Extended Data Fig. 7 | eIF4G2 binding increases in the 5′ UTRs of dendritic mRNAs upon depolarization and is associated with their increased translation. a, RBP sites in the 5′ UTRs of mRNAs enriched in resting (rest) and depolarized (dep) dendrites are shown in the heatmap using motifs from RBPmap . P values were determined using hypergeometric testing. The RBP sites enriched more upon depolarization are shown in the delta (dep minus rest) column. b, The most enriched binding motif of eIF4G2 in $5^{\prime}$ UTRs that are increasingly translated in dendrites upon depolarization by PL-Ribo-seq. eIF4G2 has 4 predicted motifs in RBPmap. c, The distribution of eIF4G2 CLIP peak counts in 3′ UTRs, 5′ UTRs and CDS for all 4 eIF4G2 binding motifs. d, Examples of eIF4G2 target mRNAs identified in resting eIF4G2 CLIP (peak heights, $\log_{2})$ ; peaks in similar genomic locations were identified in human cell lines by eCLIP . Pcdh and Pafah1b1 harbor uORFs in their 5′ UTRs. e, The distribution of eIF4G2 CLIP peaks is calculated by combining the eIF4G2 peaks from resting and depolarized primary cortical neurons. f, mRNAs with $5^{\prime}$ UTRs that are increasingly bound  

by eIF4G2 upon depolarization as determined by eIF4G2 CLIP are referred to as eIF4G2-bound. All detected mRNAs in the differential (depolarized (dep) minus resting (rest)) PL-CLIP (All, $n\!=\!17{,}654\rangle$ are compared to the eIF4G2-bound group ($_{n=1,413}$ ). g, Differential dendritic CDS translation of all mRNAs with increased ribosomes in their $5^{\prime}$ UTRs (All 5′UTR up, $n\!=\!1,\!920\!\mathrm{,}$ ) and differential dendritic CDS translation of mRNAs with increased eIF4G2 binding and increased ribosome occupancy in their $5^{\prime}$ UTRs (5′UTR up $^+$ eIF4G2-bound, $n\,{=}\,238$ ) are compared. The latter group is translated more in dendrites upon depolarization compared to the average of all mRNAs with increased ribosome occupancy in their 5′UTRs in dendrites. (f,g) Significance was calculated using the two-sided KolmogorovSmirnov test. Box plots show lower and upper hinges corresponding to the first and third quartiles (representing 25th and 75th percentile, respectively). Whiskers extend from the hinge to the 1.5x interquartile range. The center line indicates the median. P values: ns (not significant) $>\!0.05;\!^{\circ}\!<\!0.05;\!^{\circ*}\!<\!0.01$ ;\*\*\* $<\!0.001$ ; \*\*\*\* $\mathrm{{<}}0.0001$ .  

![](images/d378aeefcca36d18cbbe10ee5ca6853dc7794c0d8727972e1f7921aff10777f3.jpg)  

Extended Data Fig. 8 | Nsun3 is translationally upregulated in dendrites upon depolarization. a, Shown on the left is the dendritic translation by PLRibo-seq $(\log_{2})$ in the $5^{\prime}$ UTR, CDS and 3′ UTR of Nsun3 in resting and depolarized neurons. Shown on the right are the PL-CLIP expression values $(\log_{2})$ of Nsun3 in the Pan-TurboID and TurboID-PSD95 pulldowns from resting and depolarized neurons $\scriptstyle{n=3}$ , values from PL-Ribo-seq). P values: PL-Ribo-seq $\mathrm{^75^{\prime}U T R}=0.019$ ;$\mathsf{C D S}\,{=}\,0.0035$ ; 3′UTR $=0.64$ ), PL-CLIP (Pan $=0.88$ ,$\mathsf{P S D95}\,{=}\,0.045\$ ). b, Protein and RNA analyses of Nsun3-5′UTR reporters in resting (Dep-) and depolarized $({\mathsf{D e p}}+)$ ) neurons. Wild type (Wt) Nsun3-5′UTR harbors eIF4G2 binding sites; eIF $\mathbf{4G2++}$ , additional eIF4G2 binding sites; eIF4G2-, endogenous eIF4G2 binding sites scrambled. uORF mutants: Start mut (start codon mutated); Elong (stop codon inserted after start); Stop (stop codon mutated). Shown on the left are the Flag and ${\upbeta}$ -Actin western blots that are quantified in Fig. 5d. Shown on the right are the qPCR quantifications (quantified as in Fig. 4d) for each Nsun3  

reporter $(n\!=\!3)$ . Significance was calculated by comparing to the Wt and using the two-tailed, paired Student’s t-test. P values: Start mut $=\!0.0098$ , eIF $:4{\bf G}2++$ $=0.26$ , eIF4G2- $=0.014$ , Elong $=0.048$ , Stop $=0.0026.$ c, Nascent translation (by proximity ligation assay, PLA) and (d) total protein (by IF) levels of NSUN3 are shown $(n\!=\!2)$ plotted as floating box plots from max to min with center lines at mean). Magnification, $\times20.$ Scale bars, $25\,\upmu\mathrm{m}$ e, Western blots of Flag and ${\upbeta}$ -Actin of Nsun3-5′UTR reporters in non-targeting $\left(-\right)$ or eIF4G2 siRNA-treated (+) conditions in resting and depolarized neurons for Fig. 5e. Western blot of eIF4G2 and the quantifications of the knockdowns are shown $\left(n\!=\!3\right)$ . (a,e) Significance was calculated using the two-tailed, unpaired Student’s t-test. (a,b,e) All data are presented as mean $\pm s.\mathbf{d}.P$ values: ns (not significant) ${>}0.05;{^*}{<}0.05$ ;$^{**}{<}0.01$ ;\*\*\* ${}^{\prime}{<}0.001$ ; \*\*\*\* ${<}0.0001.n$ indicates the number of biologically independent samples.  

![](images/8509b737c412b11f2f1fe1a11efe076441ec847905678e23f5121e77732d65c5.jpg)  

Extended Data Fig. 9 | Activity-dependent eIF4G2 regulation by uORFs is specific to dendritic mRNAs with eIF4G2 binding sites in their 5′ UTRs. a, Shown on the left is the dendritic translation by PL-Ribo-seq $(\log_{2})$ in the $5^{\prime}$ UTR, CDS and 3′ UTR of Mtf1 in resting and depolarized neurons $(n\,{=}\,3)$ . Shown on the right are the PL-CLIP expression values $(\log_{2})$ of Mtf1 in the Pan-TurboID and TurboID-PSD95 pulldowns from resting and depolarized neurons $(n\,{=}\,4)$ .$P$ values: PL-Ribo-seq $\mathrm{(5^{\prime}U T R=0.0079;C D S=0.00014;3^{\prime}U T R=0.39)}$ , PL-CLIP (Pan $=0.74$ ;$\mathsf{P S D95}\,{=}\,0.87\$ ). b, Reporter constructs with wild type (Wt) Mtf1 5′UTR, which have eIF4G2 binding sites (black vertical bars), and with Mtf1 5′UTR with mutations in its uORF. ATG, uORF start codon mutated to ATG; Start, uORF start codon mutated to GAG; Scr, uORF sequence scrambled with start and stop codons unchanged; eIF4G2-, eIF4G2 binding sites scrambled; Insert, distance between the uORF and GFP increased by randomized 15 nucleotide insertion; Elong, stop codon inserted after the start codon in the uORF; Stop, stop codon in the uORF is mutated to prevent uORF termination. The quantifications of the GFP protein and mRNA fold changes by western blots and qPCRs, respectively, are calculated and shown as in Fig. 4d $\langle n\!=\!3\rangle$ ). $P$ values: protein $\mathbf{Atg}\!=\!0.0091$ ;Start $=\!0.00028$ ; Sc $=0.0026$ ; eIF4G2- $=0.0012$ ; Inser $\mathrel{\mathop:}=0.029$ ; Elong $=\!0.00073$ ;Stop $=0.00046\rangle$ ), RNA $\mathbf{\left.Atg=0.19\right.}$ ; Start $=\!0.0092$ ; Scr $=0.044$ ; eIF4G2- $=0.43$ ;Insert $=0.49$ ; Elong $=0.065$ ; Stop $=0.37$ ). (c, d) Similar to (a) and (b), PL-Ribo-seq and PL-CLIP data for Zfp64 and the constructs with Zfp64 5′UTR $(n\!=\!3)$ are shown. P values: PL-Ribo-seq (5′UTR $=0.011$ ;${\mathsf{C D S}}=0.040$ ; 3′UTR = 0.66), PL-CLIP (Pan $=0.18$ ;$\mathsf{P S D95}=0.0069$ ). Zfp64 5′UTR does not have eIF4G2 binding sites. For Zfp64, a variant version of the uORF, where it is duplicated, is included (2x) as well as Start and Insert mutants designed as in (b). P values: protein $(2\mathbf{x}\,{=}\,0.047$ Start $=\!0.00039$ ; Insert $=0.0049_{\cdot}$ ), RNA $2\mathbf{x}\,{=}\,0.016$ ; Start $=0.043$ ; Insert $=\!0.019$ ). (e, f) Similar to (a) and (b), PL-Ribo-seq and PL-CLIP data for Katnbl1 and the constructs with Katnbl1 5′UTR $(n\!=\!3)$ are shown. $P$ values: PL-Ribo-seq (5′UTR $=0.013$ ;$\mathsf{C D S}\,{=}\,0.0037$ ; 3′UTR $=0.38$ ), PL-CLIP (Pan $=0.86$ ;$\mathsf{P S D95}\!=\!0.68_{\cdot}$ ). Since Katnbl1 uORF has two overlapping ORFs, separate start codon mutations (St 1 and St 2) as well as a double mutant $(1+2)$ are included. Deletion (Del) mutant: the distance between the uORF and GFP start codon is decreased by 30 nucleotides. Katnbl1 5′UTR does not have eIF4G2 binding sites. P values: protein $l=0.0029;\mathrm{}5{\mathrm{t}}\,2=0.0057;1+2=0.0062;\mathrm{Del}=0.0061)$ , RNA ($\mathbf{\nabla}\mathbf{S}\mathbf{t}\mathbf{1}\!=\!0.35$ ; S ${\bf\dot{t}}\,2\,{=}\,0.13;1\,{+}\,2\,{=}\,0.085$ ; Del $=0.022$ ). (g-i) Nascent translation and total protein levels using PLA and IF, respectively, shown for G) MTF1, H) ZFP64 and I) KATNBL1. Magnification, $\times20$ . Quantifications are shown on the right $(n\!=\!2)$ plotted as floating box plots from max to min with center lines at mean. j, Western blots showing GFP translation (Flag) from constructs with wild type and eIF4G2- Mtf1 5′UTR, Zfp64 5′UTR and Katnbl1 5′UTR with (siRNA+) and without eIF4G2 (siRNA-) knockdown. ${\upbeta}$ -Actin used as loading control; eIF4G2 shown to indicate knockdowns. k, Flag and ${\upbeta}$ -Actin western blots from ( j) are quantified as in Fig. 4d. Significance was calculated between siRNA- and siRNA+ conditions $(n\!=\!3)$ . The center lines for box plots are at mean. P values: $M t f I\,{=}\,0.0013$ , eIF4G2- Mtf $=0.55$ , Zfp64 = 0.72, Katnbl1 $=0.65$ . (a,c,e,k) Significance was calculated using the two-tailed, unpaired Student’s t-test. (b,d,f) All data are presented as mean ±s.d. Significance was calculated with respect to Wt, using the two-tailed, paired Student’s t-test. P values: ns (not significant) ${>}0.05;{^*}{<}0.05$ ; \*\* $<\!0.01$ ; \*\*\* $<\!0.001$ ; \*\*\*\* ${<}0.0001$ .$n$ indicates the number of biologically independent samples.  

![](images/d4b462158047a68ce17343527dd2bd7b3791c0a8470bf6fe6aa8cfbe62fcb50d.jpg)  
Extended Data Fig. 10 | Translational upregulation of dendritic downstream ORFs by eIF4G2 is mediated by its local calcium-influx-induced  

phosphorylation. a, CDF plots of dendritic translation of eIF4G2-bound transcripts that harbor uORFs in wild type and eIF4G2 knockdown conditions. Significance was calculated by the two-sided Wilcoxon signed-rank test. b, Dendritic enrichment $(\log_{2})$ of eIF4G2 is shown using the resting and depolarized PL-MS data $\scriptstyle n\,=\,5,$ values from PL-MS data). Significance was calculated using the two-tailed, unpaired Student’s t-test. Box plot whiskers extend from min to max, with the center line at median. c, Nascent translation of Nsun3 and Slc25a19, examples of nuclear-encoded mitochondria-related RNAs, is shown by PLA (red) in the presence (EGTA-) and upon depletion $\mathrm{(EGTA+)}$ ) of calcium in resting and KCl-depolarized neurons $(n\!=\!2)$ . DAPI (blue, marker for nuclei). d, Shown are the Flag and ${\upbeta}$ -Actin western blots that are quantified in Fig. 5g. eIF4G2 western blot shows endogenous eIF4G2 (lower bands), which is reduced by siRNA knockdowns (Kd), as well as the localized wild type and phospho-mutant eIF4G2 variants (upper bands, higher molecular weight due to the inclusion of dendritic localization signals). Fold changes are shown as in Fig. 5g but here significance was calculated with respect to the knockdown, using the two-tailed, paired Student’s t-test $\left(n=3\right)$ . (c,d) All data are presented as mean ±s.d. n indicates the number of biologically independent samples.  

# nature portfolio  

# Reporting Summary  

Nature Portfolio wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Portfolio policies, see our Editorial Policies and the Editorial Policy Checklist.  

# Statistics  

For all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section.  

The exact sample size (n) for each experimental group/condition, given as a discrete number and unit of measurement  

A statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly  

![](images/b5700a60fef2b0673ce583b17ec882aaccb8f0c76d942b09ee4b2780ae92c1f4.jpg)  

A description of all covariates tested  

A description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons  

The statistical test(s) used AND whether they are one- or two-sided Only common tests should be described solely by name; describe more complex techniques in the Methods section.  

A full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals)  

For null hypothesis testing, the test statistic (e.g. $F,\,t,\,r)$ with confidence intervals, effect sizes, degrees of freedom and $P$ value noted Give P values as exact values whenever suitable.  

For Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings  

For hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes  

Estimates of effect sizes (e.g. Cohen's d, Pearson's r), indicating how they were calculated  

Our web collection on statistics for biologists contains articles on many of the points above.  

# Software and code  

Policy information about availability of computer code  

Data collection the SkanIt Re 5.0 program ; Odyssey Clx Imager LI-COR Bioscience, Keyence BZ-X800 Series Fluorescence Microscope alysis All packages and software described in the methods section were used according to the standard protocols. voomWithQualityWeights (Limma), RStudio $(2022.07.0)+\mathbb{R}$ (4.2.1), prcomp and pcascree (PCA), Rfastp, Plastid, RiboWaltz, Biostrings (2.64.1), STAR,  ORF-RATER, TxDb.Mmusculus.UCSC.mm10.knownGene (3.10.0), fgsea (1.22.0), Rsubread (2.10.5), countPWM summarizeOverlaps, annotationTools (1.70.0), tidyverse (2.0.0),  biomaRt (2.52.0), ImageJ $1.52\mathfrak{q},$ ggrepel (0.9.4), Pyhton, GenomicsAlignments (1.32.1), seqLogo (1.62.0), pheatmap (1.0.12), Prism v9, Adobe Illustrator v28.2.  

For manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors and reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Portfolio guidelines for submitting code & software for further information.  

Policy information about availability of data  

All manuscripts must include a data availability statement. This statement should provide the following information, where applicable: - Accession codes, unique identifiers, or web links for publicly available datasets - A description of any restrictions on data availability - For clinical datasets or third party data, please ensure that the statement adheres to our policy  

All sequencing data generated in this study have been deposited in Gene Expression Omnibus with the accession number GSE213083. Proteomics data have been deposited in the ProteomeXchange Consortium via the PRIDE79 partner repository with the data set identifier PXD050222. The databases that are used in this study are Mouse Uniprot proteome FASTA database (March 2020), mm10 UCSC mouse genome, and RBPmap v1.1 (http://rbpmap.technion.ac.il/). We have the rights to publish BioRender figures, and Figures 1a, d, 3a, 4a, and 6 have been created with BioRender.com.  

# Research involving human participants, their data, or biological material  

Policy information about studies with human participants or human data. See also policy information about sex, gender (identity/presentation), and sexual orientation and race, ethnicity and racism.  

![](images/dfeb637521eba7fb6ae0e729b587230ffbffce1c0476b33d36c826420f0acead.jpg)  

Note that full information on the approval of the study protocol must also be provided in the manuscript.  

# Field-specific reporting  

Please select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection.  

Life sciences Behavioural & social sciences Ecological, evolutionary & environmental sciences For a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf  

# Life sciences study design  

All studies must disclose on these points even when the disclosure is negative.  

![](images/45ae8908af04ecc9524b5de5f29526592062984995d00bbcb8ae5eb2634ca9ef.jpg)  

# Behavioural & social sciences study design  

All studies must disclose on these points even when the disclosure is negative.  

![](images/04133d2dc5e24e2d673e4331ebbd1510f8c6f4cae3e4baa62dd95f3d7862ea2c.jpg)  

# Ecological, evolutionary & environmental sciences study design  

All studies must disclose on these points even when the disclosure is negative.  

![](images/29e31217945c0b668840b3fc168857a6b4319a3dfc0f0bc028cc3145c5dbaa47.jpg)  

![](images/6a8793d1b87ad9fbad7d61fd0a3800e71dc6fb6249dcc0a97873789d95a5f620.jpg)  

# Reporting for specific materials, systems and methods  

We require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.  

# Materials & experimental systems  

# Methods  

n/a Involved in the study ChIP-seq Flow cytometry   
XMRI-based neuroimaging   
n/a Involved in the study Antibodies XEukaryotic cell lines   
区Palaeontology and archaeology Animals and other organisms   
区Clinical data   
区Dual use research of concern Plants  

# Antibodies  

Antibodies used  

Primary antibodies: Streptavidin (Thermo Scientific, S32358), Puromycin (mouse, Kerafast, EQ0001,  RRID: AB_2620162), Flag (mouse, Sigma, F1804, RRID: AB_262044), $\upbeta{\cdot}$ -Actin antibody (mouse, Sigma, A1978, RRID:AB_476692), RPL10A (rabbit, Abcam, ab174318), MAP2 (guinea pig, Synaptic Systems, 188004, RRID:AB_2138181), GFAP (rabbit, Abcam, ab7260, RRID:AB_305808),OLIG2 (rabbit, Proteintech, 13999-1-AP, RRID:AB_2157541), PSD95 (mouse, Millipore, MABN68, RRID:AB_10807979), SYNAPTOPHYSIN (mouse, Abcam, ab8049, RRID:AB_2198854), SHANK3 (mouse, Novus, NBP1-47610, RRID:AB_10010567), GKAP(rabbit, Novus, NBP1-76911, RRID:AB_11017331), Nlgn1 (mouse, Novus, NBP2-42192), Homer1 (rabbit, Proteintech, 12433-1-AP, RRID:AB_2295573), GAPDH (mouse, Thermo Fisher Scientific, AM4300, RRID:AB_2536381), BAIAP2 (rabbit, Proteintech, 11087-2-AP, RRID:AB_2063075), DLGAP3 (rabbit, Proteintech, 55056-1-AP, RRID:AB_10858793), TBR1 (rabbit, Proteintech, 20932-1-AP, RRID:AB_10695502), H4 (mouse, Abcam, ab31830, RRID:AB_1209246), H2A.X (rabbit, Proteintech, 10856-1-AP, RRID:AB_2114985),EEF2 (rabbit, Cell Signaling Technology, 2332, RRID:AB_10693546), P-EEF2 (rabbit, Cell Signaling Technology, 2331, RRID:AB_10015204), eIF2α (rabbit, Cell Signaling Technology, 9722, RRID:AB_2230924), P-eIF2α (rabbit, Cell Signaling Technology, 3398, RRID:AB_2096481), p42/44 MAPK (rabbit, Cell Signaling Technology, 4695, RRID:AB_390779), P-p42/44 MAPK (rabbit, Cell Signaling Technology, 9101, RRID:AB_331646), P-IRE1 (rabbit, Novus, NB100-2323SS, RRID:AB_10145203), IRE1 (rabbit, Novus, NB100-2324SS, RRID:AB_10000972), CHOP (mouse, Cell Signaling, 2895T, RRID:AB_2089254), ATF4 (rabbit, Cell Signaling, 11815S, RRID:AB_2616025), MPHOSPH (rabbit, Biorbyt, orb100446), KCNJ9 (rabbit, LSBio, LS-C352416), KCNJ9 (mouse, Antibodies Incorporated, 75-445, RRID: AB_2686912), eIF4G2 (rabbit, Cell Signaling, RRID: AB_10622189 and rabbit, Cell Signaling, RRID: AB_2261993), Puromycin (mouse, Kerafast, EQ0001,  RRID: AB_2620162), NSUN3 (rabbit, LSBio, LS-C163024), MTF1 (rabbit, Novus, NBP1-86380, RRID:AB_11011361), ZFP64 (rabbit, Proteintech, 17187-1-AP, RRID:AB_2218826), KATNBL1 (rabbit, Proteintech, 24795-1-AP, RRID:AB_2879730).  

Secondary antibodies: Donkey anti-rabbit 800 (LI-COR Biosciences, 926-32213), donkey anti-rabbit 680 (LI-COR Biosciences, 926-68073), goat anti-mouse 800 (LI-COR, 926-32210) for Western blots. Donkey anti-mouse Alexa Fluor 488 (Thermo Fisher Scientific, R37114, RRID: AB_2556542), Donkey anti-guinea pig Alexa Fluor 488 (Jackson ImmunoResearch Labs, 706-545-148, RRID:AB_2340472), Donkey anti-rabbit Alexa Fluor 647 (Jackson ImmunoResearch Labs, 711-605-152, RRID:AB_2492288), Donkey anti-mouse Alexa Fluor 555 (Thermo Fisher Scientific, A-31570, RRID:AB_2536180).  

Validation  

Antibodies were chosen from antibodypedia, and the ones that were validated by the manufacturer and reviewers were used. For the IF and PLA experiments, negative controls such as exclusion of primary antibody and knockdowns of target proteins and inhibition of protein synthesis were performed (relevant for puromycin, NSUN3, MTF1, ZFP64, and KATNBL1). eIF4G2 antibodies were also tested for specificity in neurons where it was knocked down by eIF4G2-specific siRNAs. Flag: "detects a specific single band on a Western blot from mammalian lysates"  and "specific for WB and IFs" as stated on the  

manufacturer's website.  

${\upbeta}$ -Actin: Stated on the manufacturer's website: "Monoclonal mouse anti-actin antibody was used as a loading control for western blot analysis of immunoprecipitated proteins from rat dorsal root ganglion cocultures. Monoclonal mouse anti-actin was used as a loading control for western blot analysis of rat liver protein lysates."  

Rpl10a: The use of this antibody for Western blots and immunocytochemistry has been demonstrated on the manufacturer's website.  

Map2: 172 references cite this antibody (on the manufacturer's wesbite) for use in ICC and IHC.   
GFAP and OLIG2: tested and referenced for their use in IF on the manufacturers' websites.   
PSD95: validated and referenced for its use in IH, IC on the manufacturer's website.   
SYNAPTOPHYSIN: validated, tested, and referenced for IF on the manufacturer's website.   
SHANK3, GKAP, Nlgn1, Homer1, GAPDH, BAIAP2, DLGAP3, TBR1, H4, H2.AX, EEF2, p-EEF2, eIF2α, p-eIF2α,  p42/44 MAPK, p-p42/44 MAPK, P-IRE1, IRE1, CHOP, and ATF4: validated, tested, and referenced for WB on the manufacturers' websites.   
MPHOSPH and KCNJ9: tested and validated by knockout for their use in IF.  

# Eukaryotic cell lines  

Policy information about cell lines and Sex and Gender in Research  

Cell line source(s) Human embryonic kidney (HEK) 293T cells (ATCC, CRL-11268)   
Authentication Was tested for mycoplasma after receival but no further authentication was performed after. Mycoplasma contamination Not detected   
Commonly misidentified lines N/A   
(See ICLAC register)  

# Animals and other research organisms  

Policy information about studies involving animals; ARRIVE guidelines recommended for reporting animal research, and Sex and Gender in Research  

![](images/2f10a543f511909f37b8795fdcecb3c9a50eb23bab28582f7980f10a1447603b.jpg)  

Note that full information on the approval of the study protocol must also be provided in the manuscript.  

# Clinical data  

Policy information about clinical studies All manuscripts should comply with the ICMJE guidelines for publication of clinical research and a completed CONSORT checklist must be included with all submissions.  

Clinical trial registration  

Study protocol  

Data collection  

Outcomes  

Describe how you pre-defined primary and secondary outcome measures and how you assessed these measures.  

Policy information about dual use research of concern  

# Hazards  

Could the accidental, deliberate or reckless misuse of agents or technologies generated in the work, or the application of information presented in the manuscript, pose a threat to:  

No Yes Public health National security Crops and/or livestock Ecosystems   
?Any other significant area  

Experiments of concern  

Does the work involve any of these experiments of concern:  

No Yes Demonstrate how to render a vaccine ineffective Confer resistance to therapeutically useful antibiotics or antiviral agents   
区Enhance the virulence of a pathogen or render a nonpathogen virulent   
区Increase transmissibility of a pathogen Alter the host range of a pathogen   
区Enable evasion of diagnostic/detection modalities   
区Enable the weaponization of a biological agent or toxin Any other potentially harmful combination of experiments and agents  

# Plants  

Seed stocks Novel plant genotypes  

Report on the source of all seed stocks or other plant material used. If applicable, state the seed stock centre and catalogue number. If plant specimens were collected from the field, describe the collection location, date and sampling procedures.   
Describe the methods by which all novel plant genotypes were produced. This includes those generated by transgenic approaches, gene editing, chemical/radiation-based mutagenesis and hybridization. For transgenic lines, describe the transformation method, the number of independent lines analyzed and the generation upon which experiments were performed. For gene-edited lines, describe the editor used, the endogenous sequence targeted for editing, the targeting guide RNA sequence (if applicable) and how the editor was applied.   
Describe any authentication procedures for each seed stock used or novel genotype generated. Describe any experiments used to assess the effect of a mutation and, where applicable, how potential secondary effects (e.g. second site T-DNA insertions, mosiacism, off-target gene editing) were examined.  

Authentication  

# ChIP-seq  

# Data deposition  

Confirm that both raw and final processed data have been deposited in a public database such as GEO.  

Confirm that you have deposited or provided access to graph files (e.g. BED files) for the called peaks.  

Data access links May remain private before publication.  

Files in database submission  

For "Initial submission" or "Revised version" documents, provide reviewer access links.  For your "Final submission" document, provide a link to the deposited data.   
Provide a list of all files available in the database submission.   
Provide a link to an anonymized genome browser session for "Initial submission" and "Revised version" documents only, to enable peer review.  Write "no longer applicable" for "Final submission" documents.  

Genome browser session (e.g. UCSC)  

Replicates Sequencing depth Antibodies  

Describe the experimental replicates, specifying number, type and replicate agreement.   
Describe the sequencing depth for each experiment, providing the total number of reads, uniquely mapped reads, length of reads and whether they were paired- or single-end.   
Describe the antibodies used for the ChIP-seq experiments; as applicable, provide supplier name, catalog number, clone name, and lot number.   
Specify the command line program and parameters used for read mapping and peak calling, including the ChIP, control and index files used.   
Describe the methods used to ensure data quality in full detail, including how many peaks are at FDR 5% and above 5-fold enrichment. Describe the software used to collect and analyze the ChIP-seq data. For custom code that has been deposited into a community repository, provide accession details.  

Peak calling parameters Data quality Software  

# Flow Cytometry  

# Plots  

Confirm that:  

The axis labels state the marker and fluorochrome used (e.g. CD4-FITC).   
The axis scales are clearly visible. Include numbers along axes only for bottom left plot of group (a 'group' is an analysis of identical markers).   
All plots are contour plots with outliers or pseudocolor plots.  

A numerical value for number of cells or percentage (with statistics) is provided.  

# Methodology  

Sample preparation  

Instrument  

Software  

Cell population abundance  

Gating strategy  

![](images/e4a3dd8ef245f3b34405efd82499f4e4e5daecdc8e005d6702ef5ecc8739b97b.jpg)  

Tick this box to confirm that a figure exemplifying the gating strategy is provided in the Supplementary Information.  

# Magnetic resonance imaging  

# Experimental design  

Design type Design specifications  

Indicate task or resting state; event-related or block design.   
Specify the number of blocks, trials or experimental units per session and/or subject, and specify the length of each trial or block (if trials are blocked) and interval between trials.   
State number and/or type of variables recorded (e.g. correct button press, response time) and what statistics were used to establish that the subjects were performing the task as expected (e.g. mean, range, and/or standard deviation across subjects).  

Behavioral performance measures  

Field strength  

Sequence & imaging parameters  

Specify: functional, structural, diffusion, perfusion.   
Specify in Tesla   
Specify the pulse sequence type (gradient echo, spin echo, etc.), imaging type (EPI, spiral, etc.), field of view, matrix size, slice thickness, orientation and TE/TR/flip angle.   
State whether a whole brain scan was used OR define the area of acquisition, describing how the region was determined.  

Area of acquisition  

Diffusion MRI Used Not used  

# Preprocessing  

Preprocessing software Normalization Normalization template  

Provide detail on software version and revision number and on specific parameters (model/functions, brain extraction, segmentation, smoothing kernel size, etc.).   
If data were normalized/standardized, describe the approach(es): specify linear or non-linear and define image types used for transformation OR indicate that data were not normalized and explain rationale for lack of normalization.   
Describe the template used for normalization/transformation, specifying subject space or group standardized space (e.g. original Talairach, MNI305, ICBM152) OR indicate that the data were not normalized.   
Describe your procedure(s) for artifact and structured noise removal, specifying motion parameters, tissue signals and physiological signals (heart rate, respiration).   
Define your software and/or method and criteria for volume censoring, and state the extent of such censoring.  

Noise and artifact removal Volume censoring  

# Statistical modeling & inference  

Model type and settings Effect(s) tested  

Specify type (mass univariate, multivariate, RSA, predictive, etc.) and describe essential details of the model at the first and second levels (e.g. fixed, random or mixed effects; drift or auto-correlation).   
Define precise effect in terms of the task or stimulus conditions instead of psychological concepts and indicate whether ANOVA or factorial designs were used.  

Specify type of analysis:  

# Whole brain ROI-based Both  

Specify voxel-wise or cluster-wise and report all relevant parameters for cluster-wise methods.  

Statistic type for inference (See Eklund et al. 2016)  

Correction  

Describe the type of correction and how it is obtained for multiple comparisons (e.g. FWE, FDR, permutation or Monte Carlo).  

# Models & analysis  

Involved in the study Functional and/or effective connectivity Graph analysis Multivariate modeling or predictive analysis Functional and/or effective connectivity  

![](images/42b28b4e10830b4388d0d4ad64cab97e953a5c1600f15e62db55a9c586d4bd80.jpg)  

Graph analysis  

Report the measures of dependence used and the model details (e.g. Pearson correlation, partial correlation, mutual information).   
Report the dependent variable and connectivity measure, specifying weighted graph or binarized graph, subject- or group-level, and the global and/or node summaries used (e.g. clustering coefficient, efficiency, etc.).   
Specify independent variables, features extraction and dimension reduction, model, training and evaluation metrics.  

Multivariate modeling and predictive analysis  