# Editing and genome-wide analysis upstream open reading frames contributes to enhancing salt tolerance in tomato  

Chunping Jia1,2,† , Juan Wang1,†, Bin Guo1,3, Tao Yang1, Haitao Yang1, Baike Wang1,\* and Qinghui $\mathsf{Y}\mathsf{u}^{1,2,\star}$  

1Key Laboratory of Genome Research and Genetic Improvement of Xinjiang Characteristic Fruits and Vegetables, Institute of Horticultural Crops, Xinjiang Academy of   
Agricultural Sciences, Urumqi, China   
2College of Life Science and Technology, Xinjiang University, Urumqi, China   
3College of Computer and Information Engineering, Xinjiang Agricultural University, Urumqi, China Received 28 January 2024;   
revised 9 July 2024;   
accepted 31 July 2024.   
\*Correspondence (Tel $+86$ 13699956638; fax $+86$ 0991 4590390; email $46367247\textcircled{\textcircled{2}}$ qq.com (BW) and Tel $+86$ 13999176606; fax $+86$ 0991 4590390; email yuginghui98@ sina.com (QY))   
†These authors have contributed equally to this work.  

# Summary  

The salinization of soil constitutes a substantial hindrance to the advancement of sustainable agriculture. Our research seeks to elucidate the role of a Rab GTPase-activating protein (RabGAP) family member, SlRabGAP22, in salt tolerance and its translational regulation under salt stress in tomatoes, employing gene-editing techniques and ribosome profiling methodologies. Findings demonstrate that SlRabGAP22 acts as a positive regulator of tomato salt tolerance, with four predicted upstream open reading frames (uORFs) classified into three categories. Functional uORFs were found to be negative regulation. Editing these uORFs along with altering their classifications and characteristics mitigated the inhibitory effects on primary ORFs and fine-tuned gene expression. Enhanced tomato salt tolerance was attributed to improved scavenging of reactive oxygen species, reduced toxicity $\mathsf{N a}^{+}.$ , and diminished osmotic stress effects. Furthermore, we conducted genome-wide analysis of ORFs to lay the foundation for further research on uORFs in tomatoes. In summary, our findings offer novel perspectives and important data for the enhancement of genetic traits via uORF-based strategies and translational regulation against the backdrop of salt stress.  

Keywords: uORFs, gene editing, Riboseq, translational regulation, salt stress.  

# Introduction  

Salt stress, a significant abiotic stressor, dramatically impedes plant germination, growth, development and biomass accumulation. Moreover, soil salinization represents a key obstacle to crop productivity improvement, with related crop losses presenting a growing challenge to contemporary agriculture (Xiao and Zhou, 2022). To combat salt stress, plants have evolved a series of physiological, biochemical and molecular regulatory mechanisms (Rosca et al., 2023).  

Rab proteins enhance plant defence against salt stress by facilitating ion endocytosis, vesicular transportation, and vacuolar compartmentalization (Agarwal et al., 2009; Ganotra et al., 2023). For instance, overexpression (OE) AtRabG3e (AtRab7) in Arabidopsis thaliana (Mazel et al., 2004) and PjRab7 and PgRab7 in tobacco (Agarwal et al., 2008; George and Parida, 2011) yielded transgenic plants with improved salt tolerance relative to wild type (WT). The Rab GTPase-activating proteins (RabGAPs), key in regulating Rabs between two forms of interchange, is present across myriad eukaryotes. Notably, research has revealed 24, 25, and 24 RabGAPs in A. thaliana, rice, and tomato, respectively (Jiang and Ramachandran, 2006; Madrid et al., 2019). RabGAPs typically possess a conserved Tre-2/Bub2/Cdc16 (TBC) domain housing six essential motifs crucial for catalytic function and structural stability. Additionally, an exposed arginine residue critically determines $\mathsf{G A P S^{\prime}}$ catalytic activity, both in vitro and in vivo. Furthermore, its hydrolytic function, governed by arginine and glutamine, operates via the dual finger mechanism characteristic of GTP-bound Rab GTPases (Ganotra et al., 2023; Jiang and Ramachandran, 2006; Mishra and Lambright, 2016).  

Similarly, RabGAPs serve as Rabs regulators under salt stress and as effector or scaffold proteins crucial for membrane transport and signalling pathway coordination. In rice, for instance, OsGAP1, a Rab-specific RabGAP, enhances vesicle transport to the plasma membrane (PM) or central vacuole by activating OsRab11’s GTPase activity and promoting OsRab11 recycling (Heo et al., 2005). Salt stress triggers elevated transcript levels of OsGAP1, OsVHA-a1 and OsRab11, which work in concert with the vesicular transport of OsVHA-a1 to the prevacuolar compartment or central vesicle in high-salt conditions (Son et al., 2013). OsGAP1 can bind to OsYchF1, a negative regulator of plant defence and abiotic stress response, to mitigate its function, which in turn positively influences plant stress resilience (Cheung et al., 2010, 2020). Transgenic A. thaliana with heterologous OE of OsGAP1 displayed stronger salt stress resilience, demonstrated by decreased leaf chlorosis, lower reactive oxygen species (ROS) production and ion leakage, and elevated expression of salt-responsive genes, relative to WT. Bidirectional validation of AtGAP1 OE and knockdown in A. thaliana yielded comparable outcomes. Additionally, $O S G A P1$ exerts a crucial effect in combating oxidative stress due to salt stress. Transgenic A. thaliana with heterologous $O S G A P\mathcal{T}$ OE exhibited increased antioxidant enzyme activities and reduced lipid peroxidation and ROS accumulation compared to WT (Cheung et al., 2013).  

Upstream open reading frames (uORFs) are a class of mRNA elements located in the $5^{\prime}$ untranslated regions ( $\langle5^{\prime}$ UTR) that can precisely regulate protein translation. These frames mostly comprise a canonical start codon, which is translated into short peptides during mRNA translation. This translation process can potentially cause ribosome stalling and dissociation. uORFs generally regulate primary open reading frames (pORFs) by hindering translation initiation and modulating the rate of translation (Um et al., 2021). uORFs-bearing stress-responsive transcripts are typically induced under certain environmental conditions, and sequence mutations in the uORFs diminish their negative regulatory efficacy. The clustered regularly interspaced palindromic repeats (CRISPR)-CRISPR-associated protein 9 (CRISPR-Cas9) system offers a novel approach for efficient regulation of endogenous mRNA translation through gene editing (GE) of plant uORFs. Enhancing translation efficiency (TE) can increase the protein levels produced by the targeted gene pORFs at the translational stage (Zhang et al., 2018). Advantages of this method include evasion of gene silencing, minimal impact on transcription patterns, and straightforward prediction and validation (Si et al., 2020). Recent studies have successfully demonstrated the use of CRISPR-Cas9 in editing uORFs to control pORF translation in species such as A. thaliana, lettuce (Zhang et al., 2018), rice (Liu et al., 2021) and strawberry (Xing et al., 2020), showing promising results. However, reports on uORF editing for abiotic stress tolerance are still scant.  

Ribosome profiling sequencing (Ribo-seq) offers precise monitoring of translation processes by quantifying the translated ORFs through analysis of ribosome footprints on mRNAs. The crucial role of translational regulation in plant adaptation to abiotic stresses such as drought in maize (Lei et al., 2015), salinity in Medicago truncatula (An et al., 2023), drought or heat in potato (Jian et al., 2023), and cold in tea plants (Wang et al., 2021) has become increasingly evident.  

The tomato, as a key crop and model plant, is extensively cultivated and studied in genetics, fruit development and stress tolerance (Rothan et al., 2019). Tomatoes exhibit moderate sensitivity to salinity during their growth and development. Developing salt-tolerant tomato varieties necessitates comprehensive research into the physiological, metabolic and genomic responses to salt stress (Ma et al., 2022). Studies employing transcriptome, proteome and integrative transcriptome-metabolome analyses have shed light on the mechanisms of salt tolerance in tomatoes, yielding valuable findings (Jia et al., 2022). While studies have described the translational landscape in tomato roots under normal conditions (Wu et al., 2019), the translational regulation in tomato leaves under salt stress remains unexplored.  

This study systematically investigates the expression regulation of SlRabGAP22, a salt stress-responsive gene within the plantspecific RabGAP family subfamily II harbouring diverse uORFs. By utilizing uORF GE lines, along with corresponding pORF GE lines, OE lines, and WT as controls, we assessed its role in salt tolerance. Moreover, we conducted genome-wide analysis of ORFs to lay the foundation for further research on uORFs in tomatoes. This provides a robust theoretical and practical framework for understanding the molecular mechanisms underpinning salt tolerance and developing resistant cultivars.  

# Results  

# Structure and predictive functional analysis of the SlRabGAP22  

The study isolated SlRabGAP22 from tomato variety Ailsa Craig (AC), which encodes a protein of 518 amino acids (AA) from a 1557 bp cDNA sequence. Analysis of physicochemical properties showed that it is an unstable and hydrophilic protein (Table S1). Sequence comparisons revealed a conserved TBC structural domain and exposed arginine residues in SlRabGAP22, similar to other RabGAP subfamily II members, suggesting analogous catalytic functions (Figure S1). In the phylogenetic tree constructed from the protein sequences of the unique RabGAP subfamily II in tomato, A. thaliana and rice, it was observed that SlRabGAP22 exhibited the greatest homology to StRabGAP12, while displaying the least homology with OsRabGAP15 (Figure S2A). Gene structure analysis of SlRabGAP22 revealed five exons and four introns, mirroring the composition found in most of the other members, with both $5^{\prime}$ and $3^{\prime}$ UTRs present (Figure S2B). All members, SlRabGAP22 included, feature the conserved TBC domain at the C-terminal end (Figure S2C). Additionally, seven motifs (1, 2, 3, 5, 6, 8 and 16) identified in SlRabGAP22 are conserved across the subfamily, indicating a potential functional significance (Figure S2D). In addition, the predictive analysis of cis-acting elements indicated that SlRabGAP22 encompasses cis-acting elements like MYB, MYC, MBS, STRE, ARE, ERE, TGACG-motif and CGTCA-motif, indicating a regulatory effect across abiotic stress and phytohormone responses.  

# Expression pattern analysis and subcellular localization of SlRabGAP22  

Notably, SlRabGAP22 transcripts were more abundant and consistently up-regulated under salt stress compared to other conditions (Figure 1a). The results of transcriptional profiling also revealed a broad expression of SlRabGAP22 in various tomato tissues (Figure 1b). Post-NaCl treatment, SlRabGAP22 expression in roots and leaves significantly increased. The substantial induction of SlRabGAP22 by NaCl implies its critical role in the tomato’s response to salt stress (Figure 1c). Subcellular localization analysis revealed a ubiquitous distribution of the empty GFP fluorescent signal, used for reference throughout the experiment, in the nucleus, cytoplasm and cell membrane. Conversely, the SlRabGAP22-GFP fusion protein exhibited strong fluorescence solely at the cell membrane, suggesting SlRabGAP22’s significant role in membrane signalling and transport, mediated by its regulation of Rabs, i.e., members of the largest small GTPase family (Figure 1d).  

![](images/a9c00c18843e623dfd72245cce16a9dc9863bff36ada81b2535a40759b0296a9.jpg)  

![](images/8b1687860f21d7a38d1eb2170c5e73e47488189e39ad06b0728f7fd74e0214e9.jpg)  

Figure 2 Prediction, functional characterization, and GE of SlRabGAP22 uORFs. (a) Schematic representation of CRISPR-Cas9 targeting uORFs to manipulate gene translation. Small green ovals depict the 40S, while large ones represent the 60S ribosomal subunits. A jagged line linked to the 60S subunit indicates the emerging polypeptide from uORF or pORF. Green dots signify proteins synthesized by pORF. (b) A table detailing the positions, types, and lengths of original uORF sequences. Information table predicted by the uORFlight website. ICCu: sequence matching $^{-3}$ to $+4$ of the Kozak sequence in uORF, denoting the start codon context. ICCm: sequence aligning with pORF’s Kozak sequence from $^{-3}$ to $+4$ , representing the start codon context. (c) Utilization of the dual luciferase reporter system and (d) qRT-PCR technique to verify the presence of negatively regulated functional uORFs in the uORF original sequences and their changes at the LUC/REN mRNA level. To the left, we see diagrams of various dual luciferase vectors; to the right, bar graphs display the corresponding LUC/REN activities. Horizontal axes indicate uORF mutation types while vertical axes show LUC/REN mRNA levels. ‘ns’ denotes no significance, while ‘\*’ and ‘\*\*’ mark statistical significance at $P<0.05$ and $P<0.01$ when compared to control, respectively. (e) Schematic of the original gene structure, design targets, and uORF editing of SlRabGAP22. PAM sites are marked in red, sgRNAs with thin blue lines, and base deletions by short green lines. (f) Sanger sequencing peak maps. PAM sites feature red boxes, sgRNAs are delineated by thin black lines, and base deletions are highlighted with inverted red triangles. (g) Table of information on the position, type, and length of the edited uORF sequences. (h) Schematic displaying the SlRabGAP22 gene structure post-uORF editing. (i) Application of the dual luciferase reporter system and (j) qRT-PCR to verify the presence of negatively regulated functional uORFs in the edited uORF sequences and their changes at the LUC/REN mRNA level.  

# Screening and validation analysis of SlRabGAP22 uORFs  

uORFs are prevalent in eukaryotic mRNAs. They act as a translational control element for precisely tuning the expression of the downstream pORF. The schematic diagram illustrates that the translation of uORFs typically impedes pORF translation. CRISPR-Cas9-mediated disruption of the uORFs decreased its inhibitory effect, thereby enhancing pORF translation (Figure 2a). Based on the uORF finder tool in the uORFlight database constructed by predecessors (Niu et al., 2020), we found that the SlRabGAP22 $5^{\prime}$ UTR contains four uORFs, classified into three distinct types. uORF1 through uORF4 exhibited variation in position, length, and GC content (Figure 2b). Referring to the relevant technical processes described by previous authors (Si et al., 2020), we hypothesized that the four uORFs of SlRabGAP22 have different functions, among which there may be negatively regulated functional uORFs. Functional characterization of SlRabGAP22’s multiple uORFs, tested individually and in combination via a dual luciferase reporter system, revealed that mutations in the start codons (ATG-AAA) of uORF1 and uORF3 led to a notable increase in firefly (LUC) / renilla luciferase (REN) activity. This suggests a diminished inhibitory effect and identifies them as negatively regulated uORFs (Figure 2c). Additionally, the non-significant difference in LUC/REN mRNA levels suggests that uORFs’ inhibitory effects happen translationally (Figure 2d).  

# Generation, expression changes, and selection of GE and OE lines  

The CRISPR-Cas9 system was utilized to engineer two high-scoring sgRNA targets: one near uORF1 (where uORF1 is too short for a suitable PAM site nearby) and another at the overlapping uORF2-4 sequence, to create a uORF editing line (Figure 2e,f). We selected a homozygous line exhibiting 8 bp and 1 bp deletions at the respective target sites, leading to two modified uORF types. The alterations included uORF2 transitioning from type 2 to type 1 and uORF3 from type 3 to type 2, along with changes in position, length and GC content of uORFs 2–4 (Figure 2g,h). Functional characterization of the edited uORFs, both singly and in combination, revealed that mutated uORF1 and uORF2 function as negatively regulated uORFs (Figure 2i). Selecting the uORFs-edited homozygous line with increased LUC/REN activity and stable LUC/REN mRNA levels (Figure 2j) implies a reduced translational inhibition of SlRabGAP22, offering a basis for future studies.  

To investigate SlRabGAP22’s potential role, we created its OE lines. Additionally, we created its pORFs edited line utilizing the CRISPR-Cas9 system. The edited homozygous lines exhibited a 1 bp insertion at the sgRNA2 target site, leading to premature protein termination and complete disruption of SlRabGAP22’s conserved TBC domain (311–449 AA) (Figure 3a,b). qRT-PCR and Western blot analyses revealed significant differences in gene and protein expression levels between the WT and the pORFs edited (porf ), uORFs edited (uorf ) and OE lines 14 (OE 14) and 15 (OE 15), which were chosen for further study (Figure 3c–e).  

# Comparative analysis of salt tolerance performance in WT, OE and GE lines  

To elucidate SlRabGAP22’s role in salt stress response, we treated WT, OE and GE lines ( $\mathrm{T}_{2}$ generation) with NaCl and conducted phenotyping and photography. Prior to NaCl treatment, growth characteristics were consistent across all lines. Following 21 days of $200\ m_{\mathsf{M}}\ N a C|$ (salinity levels verified as excessively damaging effects by preliminary experiments) exposure, porf and WT displayed markedly more leaf and stem damage than uorf, OE 14, and OE 15 lines. Under control conditions, all lines grew well with no notable differences (Figure 3f). The growth inhibition in porf lines was significantly greater than in WT, whereas it was significantly less in uorf, OE 14, and OE 15 lines (Figure 3g). Photographic and statistical calculations of PSII maximum quantum efficiency $(F_{v}/F_{m})$ ratios showed that uorf, OE 14, and OE 15 lines reduced the damage caused to the tomato photosynthesis system under salt stress compared to WT (Figure $3\mathrm{h},$ i). In summary, uORFs edited and/or OE of SlRabGAP22 reduced growth inhibition and enhanced photosynthetic effi- ciency, thereby increasing tomato salt tolerance through elevated protein and/or gene expression while preserving normal growth. Furthermore, we examined the stomatal aperture in leaf samples from various lines under control conditions, abscisic acid (ABA), and NaCl treatments (Figure S3A). Our findings reveal no significant variation in stomatal opening or closing across the lines under any of the three treatments (Figure S3B). Consequently, suggesting their salt stress tolerance does not stem from stomatal dynamics.  

Salt stress causes excessive accumulation of ROS in plant cells. Using 3,3-Diaminobenzidine (DAB), nitroblue tetrazolium chloride (NBT) staining, and various kits, we assessed the enzyme activities and physiological indices of ROS scavenging systems across different tomato lines to methodically evaluate the effects of SlRabGAP22 on ROS scavenging in tomato under salt stress. The results indicated that uorf, OE 14, and OE 15 lines, under salt stress, showed decreased accumulation of hydrogen peroxide $(\mathsf{H}_{2}\mathsf{O}_{2})$ and superoxide anion $({\ O_{2}}^{-})$ , lower ROS levels, elevated antioxidant enzyme activity, and enhanced osmolyte concentrations compared to WT (Figure 4a–e). Additionally, we employed $2^{\prime}$ ,$7^{\prime}$ -Dichlorodihydrofluorescein diacetate $({\mathsf{H}}_{2}{\mathsf{D}}{\mathsf{C}}{\mathsf{F}}{\mathsf{D}}{\mathsf{A}})$ , a reduced fluorescein derivative, as a cellular ROS indicator (Figure S4A). Fluorescence assays on tomato roots revealed that salt stress markedly increased ROS in porf lines while lowering it in uorf, OE 14, and OE 15 lines compared to WT, indicating reduced oxidative stress (Figure S4B). SlRabGAP22 exerts an effect in modulating the ROS pathway, reducing oxidative damage via ROS scavenging, and thereby enhancing tomato salt tolerance.  

![](images/9a23913f2516a433dc4a1b937e7bfa631649560a21ee0f740f0f537291606cca.jpg)  

Figure 3 Ge of SlRabGAP22 pORF, screening of different lines, phenotyping, growth inhibition degree, and $F_{v}/F_{m}$ ratio analysis. (a) CRISPR-Cas9-mediated generation of SlRabGAP22 pORF edited lines. Green squares illustrate UTRs, orange ones denote exons, and the intervening slender grey lines indicate introns. PAM sites are marked in red, sgRNAs with thin blue lines beneath, and base deletions with short green strokes. (b) Peak plot analysis of sgRNA2 in WT and porf lines via Sanger sequencing. PAM sites are indicated by black boxes, sgRNA by thin black lines, and base deletions by inverted red triangles. (c) Differential expression analysis of SlRabGAP22 across lines. The horizontal coordinates represent the names of different lines and the vertical coordinates represent the relative gene expression. (d) Western blot analysis across different lines. Actin (plant) served as the loading control. (e) Quantitative assessment of SlRabGAP22 protein levels in different lines. The horizontal coordinates represent the names of different lines, and the vertical coordinates represent the relative expression of proteins calculated as grey values. (f) Phenotypic characterization of plants. The scale bar represents $5~\mathsf{C m}$ . (g) Analysis of shoot and root growth inhibition. The horizontal coordinates in the bar chart indicate the shoot and root, the vertical coordinates indicate the growth inhibition degree, and the bars of different colours represent different lines. (h) Chlorophyll fluorescence imaging of leaves. A gradient from blue to red depicts the range of $F_{v}/F_{m}$ ratios, from low to high, and the scale bar is 1 cm. (i) Statistical analysis of $F_{v}/F_{m}$ ratios represented by violin plots. Horizontal axes correspond to various treatments, vertical axes to $F_{v}/F_{m}$ ratios, and distinct colour patterns to different experimental lines. ‘ns’ denotes no significance, while ‘\*’ and $\prime\ast\ast\prime$ mark statistical significance at $P<0.05$ and $P<0.01$ when compared to WT, respectively.  

Enhanced endocytosis during vesicular transport is linked to plant salt stress tolerance (Salinas-Cornejo et al., 2021). Given the potential involvement of SlRabGAP22 in vesicular transport, we posited an active role for this protein in endocytosis. We employed the FM4-64 probe for vesicle dynamic analysis, tracing membrane internalization and vesicular transport in live cells. Fluorescence assays of tomato roots indicated that under control conditions, uorf, OE 14, and OE 15 lines exhibited increased internalization rates relative to WT (Figure 5a), which were significantly different only post $30~\mathsf{m i n}$ of treatment (Figure 5c). NaCl treatment significantly hastened endocytosis in root cells (Figure 5b), with uorf, OE 14, and OE 15 lines displaying a notably higher FM4-64 internalization ratio than WT (Figure 5d). We also used Sodium Green (consisting of two $2^{\prime}.7^{\prime}.$ -dichlorofluorescein dyes), which is selective for $\mathsf{N a}^{+}.$ , as a fluorescent indicator to detect $\left\mathsf{N a}^{+}$ levels. Fluorescence assays revealed that salt stress increased the signal intensity in uorf, OE 14, and OE 15 lines compared to WT, with a marked $\left\mathsf{N a}^{+}$ accumulation in vesicles over the epidermal and cortical areas (Figure S5A). Statistical analysis demonstrated a significantly higher Sodium Green  intensity ratio within the vesicles of uorf, OE 14, and OE 15 lines relative to WT under salt stress (Figure S5B). These results suggest that SlRabGAP22 may regulate vesicular transport processes and perform an essential effect on tomato salt tolerance by increasing the rate of endocytosis.  

To ascertain if SlRabGAP22 modulates tomato salt tolerance through $\mathsf{N a}^{+}/\mathsf{K}^{+}$ homeostasis, we quantified and compared the ${\mathsf{N a}}^{+}$ and $K^{+}$ contents and their corresponding ratios in shoots and roots across the different lines. Results illustrated no significant differences in ${\mathsf{N a}}^{+}$ and $K^{+}$ concentrations and their ratios in shoots and roots of the various lines under control conditions compared to WT (Figure S6A–F). Salt stress up-regulated $\mathsf{N a}^{+}$ concentrations and $\mathsf{N a}^{+}/\mathsf{K}^{+}$ ratios in both shoots and roots across lines (Figure S6A,C,D,F), while $K^{+}$ levels were down-regulated (Figure S6B,E). Relative to WT, uorf, OE 14, and OE 15 lines exhibited a significant decrease in shoot $\mathsf{N a}^{+}$ and an increase in $K^{+}$ levels (Figure S6A,B). The $\mathsf{N a}^{+}/\mathsf{K}^{+}$ ratio significantly declined due to the reduced $\left\mathsf{N a}^{+}$ accumulation and $K^{+}$ efflux (Figure S6C). In summary, $S I R a b G A P22$ enhanced endocytosis dynamics, preferentially compartmentalizing $\left\mathsf{N a}^{+}$ in root vesicles, thereby diminishing ${\sf N a}^{+}$ translocation to shoots, lowering the shoot $\mathsf{N a}^{+}/\mathsf{K}^{+}$ ratio, and bolstering tomato salt tolerance by alleviating $\left\mathsf{N a}^{+}$ toxicity and osmotic stress.  

# Genome-wide analysis of ORFs in tomato  

Previous studies have characterized the translational landscape in tomatoes, primarily focusing on roots in normal conditions (Wu et al., 2019). Sequencing data were analysed with RiboCode (Xiao et al., 2018) software, we have for the first time identified and categorized ORFs in leaves from various lines during different salt stress periods at a genome-wide level (Figure 6a). These include 17 270 annotated ORFs (aORFs), 761 uORFs, 254 downstream ORFs, and 825 internal ORFs within the annotated CDS but in different reading frames. Small ORFs (sORFs) are pivotal for plant growth, development, and acclimation and are markedly shorter than aORFs (Figure 6b).  

Given the stringent criteria of RiboCode software, which necessitates high sequence depth for identifying statistically significant 3 nt periodicity, and its restricted range of transcript analysis, some less frequent and overlapping uORFs might be overlooked, despite being reliable, are considerably rarer (Xiao et al., 2018; Zhu et al., 2023). To comprehensively characterize the translational regulation of uORFs, we re-predicted them within the $5^{\prime}$ UTR using Python, eliminated redundancies based on genomic positioning, and executed quantitative analysis. We identified 13 746 uORFs. When setting a threshold of $\mathsf{F P K M}>1$ ,6006 $(43.69\%$ ) uORFs were recognized as translated (Figure 6c).  

Subcellular localization analysis, predicated on gene TE data, revealed that $88.52\%$ of translated uORFs $(\mathsf{u O R F+})$ predominantly localize to the nucleus, chloroplasts, cytoplasm and PM, respectively, while a lesser fraction of $1\,1.48\%$ localize to the mitochondria, vacuoles and extracell etc, respectively (Figure 6d). mRNA sequence characteristics could significantly influence translational regulation and modulate RNA translation (Lei et al., 2015). Our findings indicated that $\mathsf{u O R F+}$ exhibited lower normalized minimum free energy (NMFE) and higher GC content (Figure 6e,f). Additionally, the lengths of $\mathsf{u O R F+}$ and their respective $5^{\prime}$ UTR sequences were significantly shorter than those of untranslated uORFs (uORF ) (Figure 6g). This aligns with earlier research that demonstrated how sequence length, GC content, and NMFE influence mRNA translation (An et al., 2023; Lei et al., 2015). Regarding Kozak consensus sequences, the base frequency near the start codon varied between translated and untranslated forms of uORFs. Guanine was more probable at the $+4$ position in $\mathsf{U O R F+}$ than in $\mathsf{U O R F-}$ (Figure 6h).  

# Discussion  

Soil salinization impacts approximately $20\%$ of globally irrigated land, posing a significant challenge to food security (Xiao and Zhou, 2022; Yang and Guo, 2018). The tomato plant, being moderately salt-sensitive, experiences hindered growth and development under high salinity conditions, potentially leading to drastic yield losses or extinction.  

![](images/b030dbfee207b0cf4fa9bb76518060ebcc741ea34c3ea2d0819460e0e539f701.jpg)  
Figure 4 Leaf staining and determination of physiological and biochemical indices. (a–e) Analysis of various lines’ leaves via (a), DAB staining (b), Determination of $\mathsf{H}_{2}\mathsf{O}_{2}$ contents (c), NBT staining (d), Determination of ${{\sf O}_{2}}^{-}$ contents, and (e) Determination of antioxidant enzymes, oxidative damage parameters, and osmoregulatory substance content/activity. Scale bar represents 1 cm. Bars of different colours represent different treatments/lines. ‘ns’ denotes not significance, while $\prime\ast\prime$ and $\prime\ast\ast\prime$ mark statistical significance at $P<0.05$ and $P<0.01$ when compared to WT, respectively.  

![](images/b2966fe709a7479e3c98fe31922b7c2aa764ba2cb9f3b8132516d81bec99d60f.jpg)  
Figure 5 Assessment of endocytosis rates in tomato roots using FM4-64 Indicator. (a, b) Fluorescence imaging of different lines (a) under control and (b) under $200\ m{\mathsf{M}}\ N a{\mathsf{C}}{\mathsf{I}}$ treatment. The scale bar is $20~\upmu\mathrm{m}$ . (c–d) Bar graph displaying (c) endocytosis ratio (intracellular/PM) under control and (d) endocytosis ratio (line/WT) under $200~\mathsf{m M}$ NaCl treatment. In the bar graphs, colour variations distinguish between different lines. ‘ns’ denotes no significance, while ‘\*’ marks statistical significance at $P<0.05$ when compared to WT.  

Specific RabGAPs can fine-tune Rabs signalling, thereby regulating particular cellular functions. Relative to yeast and animals, the understanding of structural characteristics, regulatory mechanisms and roles of plant RabGAP proteins in stress tolerance is underdeveloped, warranting further study into their functions (Jiang and Ramachandran, 2006). Recent studies have revealed the significant roles of plant Rab and/or RabGAP proteins in mitigating salt stress in A. thaliana (Mazel et al., 2004), tobacco (Agarwal et al., 2008; George and Parida, 2011) and rice (Cheung et al., 2013).  

Meanwhile, regulation through uORFs is crucial for plant development, disease resistance and nutrient uptake (Wang et al., 2024). The application of uORFs in plant genetic engineering and CRISPR-Cas9-mediated GE underscores their potential in molecular crop breeding (Si et al., 2020; Um et al., 2021). Prior research has demonstrated the identification of various uORFs in the target genes of A. thaliana, lettuce (Zhang et al., 2018), strawberry (Xing et al., 2020) and rice (Liu et al., 2021), leading to enhanced traits including development, oxidative stress tolerance, ascorbic acid level, sugar concentration and heading date through functional validation and GE of uORFs.  

Our findings indicate that SlRabGAP22, a member of plantspecific RabGAP subfamily II, responds positively to salt stress with a $5^{\prime}$ UTR comprises four uORFs, classified into three categories: non-overlapping type 1, out-of-frame overlapping type 2, and in-frame overlapping type 3. Typically, type 1 uORFs are most prevalent, type 2 uORFs most strongly inhibit pORFs translation, and type 3 uORFs result in N-terminal extensions of the proteins they encode (Wang et al., 2024). We identified uORF1 and uORF3 of SlRabGAP22 as types 1 and 3, respectively, both exhibiting negative regulatory functions. uORF2 and uORF4, both type 2, did not exhibit negative regulatory functions. This could be attributed to the insulating role of type 1 uORFs in mRNAs with multiple uORFs, particularly under stress conditions, where they facilitate a mechanism allowing the 40S ribosomal subunit to bypass the inhibitory type 2 uORF downstream. This mechanism may account for the observation that over $50\%$ of type 2 uORFs in plants are associated with upstream type 1 uORFs (Wang et al., 2024).  

![](images/f094b6eb684ffdfcef4d0c58fbbc4687ba1033da7a7e4562e4098ca339ee7c42.jpg)  
Figure 6 Genome-wide identification and characterization of ORFs and sORFs. (a) Schematic overview of identified ORFs and sORFs. The schematic includes five types with their respective locations and quantities on the genome distinctly labelled. (b) Size distribution of identified ORFs and sORFs. Lengths were normalized. (c) Counts of translated and untranslated members in re-predicted uORFs. Purple bars indicate translated members and green bars indicate untranslated ones, with an FPKM threshold of ${>}1$ for inclusion. (d) Predicted subcellular localization for coding sequences of $\mathsf{U O R F+}$ genes. Using the gene TE dataset, the proportions of different organelles are denoted by various coloured circles. (e) NMFE analysis for the uORFs. (f) GC content analysis for uORFs. Thin red lines denote $\mathsf{U O R F+}$ , and thin blue lines denote $\mathsf{U O R F-}$ . (g) Length of uORFs and their respective $5^{\prime}$ UTR sequence. The yellow boxes in the box plots represent $\mathsf{u O R F+},$ , while the green boxes denote $\mathsf{U O R F-}$ . P values were independently validated using Student’s t-test. (h) Kozak consensus sequences for uORFs. The height of each capital letter correlates with the base’s frequency of occurrence at that position. Capital letters are sized in descending order of their frequency, with the most frequent represented by the largest size.  

The dynamic modification of translation is an intricate process in plant salt tolerance, encompassing diverse molecular regulatory mechanisms (Son and Park, 2023). This includes, for instance, the modification and selective employment of transfer RNAs to encourage or inhibit the translation of certain genes. Post-transcriptional RNA splicing and modification can influence translation through the regulation of mRNA structures and their interactions with translation-related entities. Furthermore, dynamic alterations in translation are intimately connected to the modulation of cellular signalling pathways. Certain signalling pathways can impact the translation of specific genes by controlling the activity of translation initiation factors or the expression levels of translation suppressors (Son and Park, 2023; Van Ruyskensvelde et al., 2018; Wang et al., 2021). Under salt stress, uORFs swiftly modulate the translation efficiency of certain genes by controlling their presence and initiation of translation, consequently influencing associated metabolic or signalling pathways and enhancing plant adaptability to salt stress (Kurihara, 2020; Um et al., 2021; Wang et al., 2024).  

Commonly, type 1 uORFs modify the translation process by altering the assembly and/or initial location of ribosomes, whereas type 2 and/or type 3 uORFs might compete with pORFs to control the translation of downstream genes, through the selection of distinct ribosomes, thereby impacting translation efficiency (Kurihara, 2020; Wang et al., 2024). In aggregate, translational regulation facilitates the preferential translation of particular mRNAs in plant stress response, thereby enabling quick proteomic adaptations to adverse environments. In the context of plant stress response, the role of uORFs in translational regulation is of significant importance. Thus, a profound exploration of the types and characteristics of uORFs, along with their role in translational regulation in plant stress response, is crucial to unravel the molecular regulatory mechanisms underlying plant stress response (Um et al., 2021).  

Changes in the characteristics and environments of uORFs can impact their regulatory functions. Essentially, mutations in critical uORFs features can influence pORFs translation, potentially leading to phenotypic diversity (Wang et al., 2024). The repressive activity of uORFs is influenced by several factors: a favourable Kozak background, infrequent triplet codons, strong stop codons, an unstable RNA structure near the start codon, and proximity to pORFs (Wang et al., 2024; Xiang et al., 2023). Utilizing the CRISPR-Cas9 system, we edited the uORFs of SlRabGAP22, modifying their type, position, distance, length and GC content. Subsequent mutations led to two mutated forms, uORF1 and uORF2, now type 1 with negative regulatory functions, and two unchanged forms, uORF3 and uORF4, type 2 without such functions. Ultimately, we selected edited homozygous uORFs lines with markedly increased LUC/REN activity and unchanged LUC/REN mRNA levels, signifying diminished translational repression of SlRabGAP22. Further investigation is warranted into the limitations of this approach, including its applicability solely to mRNAs with functional uORFs and the modest strength of functional uORF repression; typically, editing leads to only modest increases in gene expression levels (Si et al., 2020). Consequently, this study could not edit uORF1 because of its short sequence length and the absence of nearby suitable PAM sites. The limitations inherent to the CRISPR-Cas9 system merit consideration, and a single-base editing system could potentially alleviate the suppressive impact of uORFs on this target gene more effectively. Recent studies have provided robust support for this perspective, demonstrating that engineered base and prime editing within the $5^{\prime}$ UTR can down-regulate endogenous gene expression by introducing, expanding, or adding new uORFs (Xue et al., 2023). Significantly, traditional gene expression enhancement methods may lead to transgene silencing in descendants and unpredictable transfer DNA insertions that can disrupt surrounding genomic sequences. In contrast, editing uORFs as an alternative method offers distinct benefits, such as preventing gene silencing, negligible effects on growth or development, easy prediction and validation, and no biosafety issues since it does not involve foreign DNA integration (Si et al., 2020).  

Our study focused on editing functional uORFs with negative regulation to produce transgene-free tomato lines with enhanced expression and improved performance of endogenous target gene. We thoroughly examined the expression regulation of SlRabGAP22 and assessed its salt-tolerance capabilities by utilizing WT, GE, and OE lines as multiple references, thus hastening gene function analysis and crop trait enhancement. Our findings indicate a tight association between SlRabGAP22 expression regulation and salt stress tolerance, with SlRabGAP22 serving as a positive regulator of salt tolerance in tomato. Relative to the WT, the salt tolerance functionality was amplified in OE and GE tomato lines. This enhanced tolerance can be ascribed to a more efficient and improved gene expression regulation mechanism in SlRabGAP22, which diminishes growth inhibition and augments photosynthetic efficiency. Consequently, it improves tomato salt tolerance by mitigating oxidative stress-induced damage via ROS scavenging, alleviating ${\mathsf{N a}}^{+}$ toxicity, and diminishing the osmotic impact of salt stress.  

Computational predictions, which typically presume protein-coding sequences exceed 100 AA and translate only the longest ORFs, frequently omit sORFs from annotations. Through genome-wide Ribo-seq analysis, we systematically identified and characterized ORFs and sORFs without bias. The sORFs including uORFs, those acting as signalling peptides critical to plant development and stress response, hold substantial potential for future research. Studies on uORF features have predominantly focused on yeast and vertebrate species, with plants receiving less attention. A review consolidated nine key uORF features, alterations to which, along with environmental and organizational shifts, can impact their regulatory functions (Wang et al., 2024).  

In our study, we compared multiple sequence attributes of $\mathsf{U O R F+/-}$ , and identified significant differences that may be intrinsically linked to uORFs expression levels. This aligns with prior research indicating that sequence length, GC content, and NMFE influence mRNA translation (An et al., 2023; Lei et al., 2015). The efficiency of start codon recognition is pivotal in the translational repression mechanism of uORFs, determining the effectiveness of their regulatory impact. Kozak sequences play a crucial role in start codon recognition and the initiation of translation, modulating the translational process through various mechanisms. In sorghum, cytosine is more abundant in the sequences flanking ATGs of $\mathsf{U O R F+}$ compared to uORF  (Sotta et al., 2022). We discovered variations in the frequency of bases surrounding the ATGs of $\mathsf{U O R F+}$ and uORF', with $\mathsf{U O R F+}$ ATG flanking sequences being less conserved than Kozak sequences. In contrast, uORF  sequences displayed the lowest conservation, potentially leading to either ignored or recognized (Um et al., 2021). This obliquely substantiates the weak selection power for uORF sequence conservation (Wang et al., 2024). Nonetheless, the limited number of conserved peptide uORFs (CPuORFs) with potential conserved roles, some of which can conditionally regulate translation by causing ribosomal stalling, should not be disregarded, as their part in plant stress response through translational regulation has been partially uncovered. It is hypothesized that variations within the CDS of CPuORFs could significantly alter their activity, thereby influencing plant biological processes (Wang et al., 2024).  

Recent studies indicate that uORF natural variations can enhance crop phenotypic diversity. Recent advancements in uORF-based genetic engineering have targeted plant trait enhancement, including propositions of uORF-box transgenics (Kim et al., 2022; Xu et al., 2017) and the CRISPR editing of endogenous uORFs, aiming to alleviate repressive uORF effects for increased gene expression and better plant performance (Liu et al., 2021; Si et al., 2020; Xing et al., 2020; Zhang et al., 2018). The current study’s findings on uORF GE improving tomato salt tolerance suggest that the identified uORF datasets, key pathways, and genes could offer novel insights into genetic breeding and molecular adaptations for salt tolerance in tomatoes via GE or traditional allelic selection.  

# Experimental procedures  

# Prediction of uORFs and functional characterization based on a dual luciferase reporter system  

The uORFs in the SlRabGAP22 $5^{\prime}$ UTR were predicted and classified using the uORFlight database (http://www.rnairport. com:443/). The SlRabGAP22 uORFs were functionally analysed using a dual luciferase reporter system. Briefly, mutant sequences of the all start codon (ATG-AAA) and individual uORFs (uORF1- uORF4) in both WT and gene-edited $5^{\prime}$ UTR variants were inserted into the pGreenII 0800-LUC vector through full gene synthesis and seamless cloning. Following the preparation and transformation of A. thaliana protoplasts, fluorescence signals were detected with a Dual-Luciferase Reporter System Kit (DL101-01; Vazyme) using a GloMax  20/20 Luminometer fluorescence detector (Promega, Wisconsin, USA), and expressed as LUC/REN ratios with three biological replicates per assay. Concurrently, changes in dual luciferase mRNA levels were quantified with qRTPCR, using specifically designed primers and three biological replicates (Table S2).  

# Transformation and generation of tomato GE and OE plants  

The tomato variety AC SlRabGAP22 edited lines were generated by the CRISPR-Cas9 system. sgRNA1 and sgRNA2, targeting uORF and pORF sequences respectively, were identified utilizing the CRISPR-P tool (http://cbi.hzau.edu.cn/crispr/). Following the system’s protocol, these sgRNAs were ligated into the pYLCRISPR/- Cas9 Pubi-H vector for knockout construct assembly (Ma et al., 2015). The recombinant vector was transformed into Agrobacterium tumefaciens (GV3101) utilizing the freeze–thaw method. The tomato genetic transformation method was used to obtain ${\sf T}_{0}$ generation lines with knockouts in SlRabGAP22 uORF and pORF sequences. Mutation types were identified with Sanger sequencing and Hi-TOM platforms, enabling the acquisition of ${\sf T}_{2}$ generation homozygous (Cas9-free) lines (Liu et al., 2019). The recombinant plasmid for the OE vector was also introduced into A. tumefaciens (GV3101) using the freeze–thaw method, yielding ${\sf T}_{0}$ generation SlRabGAP22 OE lines through tomato genetic transformation. After two generations of self-crossing and PCR confirmation of the hygromycin resistance gene, ${\sf T}_{2}$ generation lines were established. Specific primers were used to quantify SlRabGAP22 expression in edited and OE lines via qRT-PCR. Lines with notably low and high expression were selected for subsequent analysis. All primer sequences are available in Table S2.  

# Determination of $F_{v}/F_{m}$ ratio and leaf staining  

The $F_{v}/F_{m}$ ratio of tomato leaves was measured with the CF Imager chlorophyll fluorescence imaging system (Technologica, Colchester, UK). Leaves were dark-adapted for 30 min before $F_{v}/F_{m}$ analysis using FluorImager V2.305 software, then analysed and imaged using a non-invasive chlorophyll a fluorometric assay. Fresh leaves were washed with distilled water and stained by immersing in brown containers with DAB (1 mg/mL, pH 38) (SL1805; Coolaber, Beijing, China) and NBT $(0.1\%$ ,$\mathsf{p H}\,\,7.8\rangle$ (SL18061; Coolaber) solutions respectively. The leaves were incubated in the dark at room temperature for $24~\mathrm{h}$ and $12\ h.$ for DAB and NBT treatment respectively, after which they were transferred to containers with $95\%$ ethanol and heated in a water bath until all chlorophyll was removed.  

# Identification of salt tolerance phenotypes and calculation of the growth inhibition degree  

WT, OE, porf, and uorf lines were cultivated in controlled environment chambers in $100.0\,\pm\,1.0\,\mathfrak{g}$ pots (3:1 nutrient soil to vermiculite) at $24\pm2~^{\circ}\mathrm{C}$ ,$60\pm\:10\%$ relative humidity, 100– $150\ \upmu\mathrm{mol/m}^{2}/\mathrm{s}$ light intensity, and a $16/8\mathrm{~h~}$ photoperiod for 4 weeks. Once soil moisture was essentially reduced to the same degree (to prevent dilution), the soil was saturated with $\mathsf{a}\,200\mathsf{\ m u}$ NaCl solution. The excess solution was drained after 1 day, and plants were then watered with the salt solution every 3 days, repeating this cycle four times. After 21 days of treatment, salt tolerance was assessed and photographed in each line, using well-watered plants as controls. Shoots and roots from five plant lines were separately collected, washed under running water, and rinsed thrice with deionized water to remove contaminants. Fresh weight samples were initially dried at $105~^{\circ}\mathrm{C}$ for $20\ m i n$ and subsequently at $75\,^{\circ}\mathsf{C}$ until a constant weight was achieved to determine the dry weights of the shoots and roots for each line. Growth inhibition $(\%)$ was calculated using the formula: (control dry weight – salt-treated dry weight)/control dry weight $\times~100$ .Each experimental set was replicated three times biologically.  

# qRT-PCR analysis  

Synthesis of first-strand cDNA was conducted following the reverse transcription of pre-extracted total RNA, utilizing the $5\times\cdot$ All-In-One RT Master Mix kit (G490; ABM, Vancouver, Canada) as prescribed by the supplier. Primers were designed using PrimerBLAST, the NCBI’s online tool, with Actin (Solyc03g078400.2) serving as the internal control. The qRT-PCR was executed using a LightCycler real-time fluorescent quantitative PCR system (Roche, Basel, Switzerland) and ChamQ Universal SYBR qPCR Master Mix kit (Q711; Vazyme, Nanjing, China), applying primers detailed in Table S2. Each sample had three technical replicates to ascertain the average cycle threshold value, with relative expression determined by the $2^{-\Delta\Delta\Delta\subset\mathfrak{t}}$ method (Livak and Schmittgen, 2001).  

# Quantification of fluorescent labelling  

We manually quantified FM4-64 fluorescence (excitation at $515~\mathsf{n m})$ using ImageJ for both intracellular and PM compartments. The FM4-64 internalization rate was calculated by comparing intracellular to PM signal ratios, in line with previously described protocols (Baral et al., 2015; Salinas-Cornejo et al., 2021, 2023). Signals from $\mathsf{H}_{2}\mathsf{D}\mathsf{C}\mathsf{F}\mathsf{D}\mathsf{A}$ and Sodium GreenTM were individually quantified (emission wavelengths $517{-}561\ \mathrm{nm})$ using the same software. Images were intensity colour-coded and the average fluorescence per 100 pixels was calculated for predetermined regions of interest. Fluorescence intensity was quantified using consistent laser, gain, and pinhole settings on the confocal microscope for all treatments. Experiments included three technical replicates, analysing a minimum of 25 cells (relative units) from the roots of five plants per line.  

# Statistical analysis  

Data analysis was conducted with three independent biological replicates per experiment, utilizing GraphPad Prism (version 9.0.0; GraphPad Software, San Diego, CA, USA). Statistical significance between samples was assessed using one-way or two-way ANOVA, followed by Dunnett’s test for multiple comparisons. ‘ns’ indicates non-significance, while \*, $^{2}\ast\ast\ast\ast\ast\ast\ast$ , and $^{\ast}\ast\ast\ast\ast\ast$ denote significance levels at $P<0.05$ , 0.01, 0.001, and 0.0001, respectively.  

# Acknowledgements  

This work was supported by the Project of Fund for Stable Support to Agricultural Sci-Tech Renovation (xjnkywdzc2024003-01, xjnkywdzc-2023001-8, xjnkywdzc-2023001-pt1), the Key Research and Development Task Special Project of Xinjiang Uygur Autonomous (2022B02002), the China Agriculture System of MOF and MARA (CARS-23-G24), and the Earmarked Fund for XJARS (XJARS-07).  

# Conflict of interest  

The authors declare that they have no conflicts of interest.  

# Author contributions  

Q.Y. and B.W. conceived the experiment and methodology. C.J. and J.W. performed the majority of the experimental work. B.G. carried out the data analysis. T.Y. and H.Y. completed plant material treatment. Q.Y., B.W., C.J., and J.W. wrote the manuscript. All authors involved in experimental design and data analysis, commented on the manuscript, and gave final approval for publication.  

# Data availability statement  

All data is available in the manuscript or in the supplementary information. RNA sequencing and Ribo-seq data sets are available at the NCBI Sequencing Read Archive, BioProject ID PRJNA1063032.  

# References  

Agarwal, P., Reddy, M.K., Sopory, S.K. and Agarwal, P.K. (2009) Plant Rabs: characterization, functional diversity, and role in stress tolerance. Plant Mol. Biol. Report. 27, 417–430.   
Agarwal, P.K., Agarwal, P., Jain, P., Jha, B., Reddy, M.K. and Sopory, S.K. (2008) Constitutive overexpression of a stress-inducible small GTP-binding protein PgRab7 from Pennisetum glaucum enhances abiotic stress tolerance in transgenic tobacco. Plant Cell Rep. 27, 105–115.   
An, Y., Wang, Z., Liu, B., Cao, Y. and Chen, L. (2023) Translational landscape of Medicago truncatula seedlings under salt stress. J. Agric. Food Chem. 71, 16657–16668.   
Baral, A., Irani, N.G., Fujimoto, M., Nakano, A., Mayor, S. and Mathew, M.K. (2015) Salt-induced remodeling of spatially restricted clathrin-independent endocytic pathways in Arabidopsis root. Plant Cell 27, 1297–1315.   
Cheung, M.Y., Li, M.W., Yung, Y.L., Wen, C.Q. and Lam, H.M. (2013) The unconventional P-loop NTPase OsYchF1 and its regulator OsGAP1 play opposite roles in salinity stress tolerance. Plant Cell Environ. 36, 2008– 2020.   
Cheung, M.Y., Ngo, J.C., Chen, Z., Jia, Q., Li, T., Gou, Y., Wang, Y. et al. (2020) A structure model explaining the binding between a ubiquitous unconventional G-protein (OsYchF1) and a plant-specific C2-domain protein (OsGAP1) from rice. Biochem. J. 477, 3935–3949.   
Cheung, M.Y., Xue, Y., Zhou, L., Li, M.W., Sun, S.S. and Lam, H.M. (2010) An ancient P-loop GTPase in rice is regulated by a higher plant-specific regulatory protein. J. Biol. Chem. 285, 37359–37369.   
Ganotra, J., Sharma, B., Biswal, B., Bhardwaj, D. and Tuteja, N. (2023) Emerging role of small GTPases and their interactome in plants to combat abiotic and biotic stress. Protoplasma 260, 1007–1029.   
George, S. and Parida, A. (2011) Over-expression of a Rab family GTPase from phreatophyte Prosopis juliflora confers tolerance to salt stress on transgenic tobacco. Mol. Biol. Rep. 38, 1669–1674.   
Heo,J.B., Rho,H.S., Kim, S.W.,Hwang, S.M.,Kwon,H.J., Nahm,M.Y., Bang, W.Y. et al. (2005) OsGAP1 functions as a positive regulator of OsRab11-mediated TGN to PM or vacuole trafficking. Plant Cell Physiol. 46, 2005–2018.   
Jia, C., Guo, B., Wang, B., Li, X., Yang, T., Li, N., Wang, J. et al. (2022) Integrated metabolomic and transcriptomic analysis reveals the role of phenylpropanoid biosynthesis pathway in tomato roots during salt stress. Front. Plant Sci. 13, 1023696.   
Jian, H., Wen, S., Liu, R., Zhang, W., Li, Z., Chen, W., Zhou, Y. et al. (2023) Dynamic translational landscape revealed by genome-wide ribosome profiling under drought and heat stress in potato. Plan. Theory 12, 2232.   
Jiang, S.Y. and Ramachandran, S. (2006) Comparative and evolutionary analysis of genes encoding small GTPases and their activating proteins in eukaryotic   
genomes. Physiol. Genomics 24, 235–251.   
Kim, J.H., Castroverde, C.D.M., Huang, S., Li, C., Hilleary, R., Seroka, A., Sohrabi, R. et al. (2022) Increasing the resilience of plant immunity to a warming climate. Nature 607, 339–344.   
Kurihara, Y. (2020) uORF shuffling fine-tunes gene expression at a deep level of the process. Plan. Theory 9, 608.   
Lei, L., Shi, J., Chen, J., Zhang, M., Sun, S., Xie, S., Li, X. et al. (2015) Ribosome profiling reveals dynamic translational landscape in maize seedlings under drought stress. Plant J. 84, 1206–1218.   
Liu, Q., Wang, C., Jiao, X., Zhang, H., Song, L., Li, Y., Gao, C. et al. (2019) HiTOM: a platform for high-throughput tracking of mutations induced by CRISPR/Cas systems. Sci. China Life Sci. 62, 1–7.   
Liu, X., Liu, H., Zhang, Y., He, M., Li, R., Meng, W., Wang, Z. et al. (2021) Finetuning flowering time via genome editing of upstream open reading frames of heading date 2 in rice. Rice 14, 59.   
Livak, K.J. and Schmittgen, T.D. (2001) Analysis of relative gene expression data using real-time quantitative PCR and the $2^{-\Delta\Delta\subset T}$ method. Methods 25, 402– 408.   
Ma, L., Liu, X., Lv, W. and Yang, Y. (2022) Molecular mechanisms of plant responses to salt stress. Front. Plant Sci. 13, 934877.   
Ma, X., Zhang, Q., Zhu, Q., Liu, W., Chen, Y., Qiu, R., Wang, B. et al. (2015) A robust CRISPR/Cas9 system for convenient, high-efficiency multiplex genome editing in monocot and dicot plants. Mol. Plant 8, 1274–1284.   
Madrid, E., Salinas, C. and Ruiz, L. (2019) The RabGAP gene family in tomato (Solanum lycopersicum) and wild relatives: identification, interaction networks, and transcriptional analysis during plant development and in response to salt stress. Gen 10, 638.   
Mazel, A., Leshem, Y., Tiwari, B.S. and Levine, A. (2004) Induction of salt and osmotic stress tolerance by overexpression of an intracellular vesicle trafficking protein AtRab7 (AtRabG3e). Plant Physiol. 134, 118–128.   
Mishra, A.K. and Lambright, D.G. (2016) Small GTPases and their GAPs. Biopolymers 105, 431–448.   
Niu, R., Zhou, Y., Zhang, Y., Mou, R., Tang, Z., Wang, Z., Zhou, G. et al. (2020) uORFlight: a vehicle toward uORF-mediated translational regulation mechanisms in eukaryotes. Database 2020, 1–10.   
Rosca, M., Mihalache, G. and Stoleru, V. (2023) Tomato responses to salinity stress: from morphological traits to genetic changes. Front. Plant Sci. 14, 1118383.   
Rothan, C., Diouf, I. and Causse, M. (2019) Trait discovery and editing in   
tomato. Plant J. 97, 73–90.   
Salinas-Cornejo, J., Madrid-Espinoza, J., Verdugo, I., Norambuena, L. and RuizLara, S. (2023) A SNARE-like protein from Solanum lycopersicum increases salt tolerance by modulating vesicular trafficking in tomato. Front. Plant Sci. 14, 1212806.   
Salinas-Cornejo, J., Madrid-Espinoza, J., Verdugo, I., Perez-Dıaz, J., MartınDavison, A.S., Norambuena, L. and Ruiz-Lara, S. (2021) The exocytosis associated SNAP25-type protein, SlSNAP33, increases salt stress tolerance by modulating endocytosis in tomato. Plan. Theory 10, 1322.   
Si, X., Zhang, H., Wang, Y., Chen, K. and Gao, C. (2020) Manipulating gene translation in plants by CRISPR-Cas9-mediated genome editing of upstream open reading frames. Nat. Protoc. 15, 338–363.   
Son, S. and Park, S.R. (2023) Plant translational reprogramming for stress resilience. Front. Plant Sci. 14, 1151587.   
Son, Y.S., Im, C.H., Kim, D.W. and Bahk, J.D. (2013) OsRab11 and OsGAP1 are essential for the vesicle trafficking of the vacuolar $\mathsf{H}^{+}$ -ATPase OsVHA-a1 under high salinity conditions. Plant Sci. 198, 58–71.   
Sotta, N., Chiba, Y., Aoyama, H., Takamatsu, S., Suzuki, T., Miwa, K., Yamashita, Y. et al. (2022) Translational landscape of a C4 plant, Sorghum bicolor, under normal and sulfur-deficient conditions. Plant Cell Physiol. 63, 592–604.   
Um, T., Park, T., Shim, J.S., Kim, Y.S., Lee, G.S., Choi, I.Y., Kim, J.K. et al. (2021) Application of upstream open reading frames (uORFs) editing for the development of stress-tolerant crops. Int. J. Mol. Sci. 22, 3743.   
Van Ruyskensvelde, V., Van Breusegem, F. and Van Der Kelen, K. (2018) Posttranscriptional regulation of the oxidative stress response in plants. Free Radic. Biol. Med. 122, 181–192.   
Wang, J., Liu, J. and Guo, Z. (2024) Natural uORF variation in plants. Trends Plant Sci. 29, 290–302.   
Wang, P., Jin, S., Chen, X., Wu, L., Zheng, Y., Yue, C., Guo, Y. et al. (2021) Chromatin accessibility and translational landscapes of tea plants under chilling stress. Hortic. Res. 8, 96.   
Wu, H.L., Song, G., Walley, J.W. and Hsu, P.Y. (2019) The tomato translational landscape revealed by transcriptome assembly and ribosome profiling. Plant Physiol. 181, 367–380.   
Xiang, Y., Huang, W., Tan, L., Chen, T., He, Y., Irving, P.S., Weeks, K.M. et al. (2023) Pervasive downstream RNA hairpins dynamically dictate start-codon selection. Nature 621, 423–430.   
Xiao, F. and Zhou, H. (2022) Plant salt response: perception, signaling, and tolerance. Front. Plant Sci. 13, 1053699.   
Xiao, Z., Huang, R., Xing, X., Chen, Y., Deng, H. and Yang, X. (2018) De novo annotation and characterization of the translatome with ribosome profiling data. Nucleic Acids Res. 46, e61.   
Xing, S., Chen, K., Zhu, H., Zhang, R., Zhang, H., Li, B. and Gao, C. (2020) Finetuning sugar content in strawberry. Genome Biol. 21, 230.   
Xu, G., Yuan, M., Ai, C., Liu, L., Zhuang, E., Karapetyan, S., Wang, S. et al. (2017) uORF-mediated translation allows engineered plant disease resistance without fitness costs. Nature 545, 491–494.   
Xue, C., Qiu, F., Wang, Y., Li, B., Zhao, K.T., Chen, K. and Gao, C. (2023) Tuning plant phenotypes by precise, graded downregulation of gene expression. Nat. Biotechnol. 41, 1758–1764.   
Yang, Y. and Guo, Y. (2018) Unraveling salt stress signaling in plants. J. Integr. Plant Biol. 60, 796–804.   
Zhang, H., Si, X., Ji, X., Fan, R., Liu, J., Chen, K., Wang, D. et al. (2018) Genome editing of upstream open reading frames enables translational control in plants. Nat. Biotechnol. 36, 894–898.  

Zhu, X.T., Zhou, R., Che, J., Zheng, Y.Y., Tahir ul Qamar, M., Feng, J.W., Zhang, J. et al. (2023) Ribosome profiling reveals the translational landscape and allele-specific translational efficiency in rice. Plant Comm. 4, 100457.  

# Supporting information  

Additional supporting information may be found online in the Supporting Information section at the end of the article.  

Figure S1 Amino acid sequence alignment of RabGAP subfamily II members.   
Figure S2 Phylogeny, gene structure, conserved structural domains, motifs, and cis-acting elements of RabGAP subfamily II members.   
Figure S3 Imaging and statistical analysis of stomatal aperture characteristics.   
Figure S4 Fluorescence-based detection of the ROS indicator $\mathsf{H}_{2}\mathsf{D}\mathsf{C}\mathsf{F}\mathsf{D}\mathsf{A}$ in tomato roots under salt stress.   
Figure S5 Detection of ${\mathsf{N a}}^{+}$ level by Sodium GreenTM indicator in tomato roots under salt stress.   
Figure S6 Shoot and root ${\mathsf{N a}}^{+}$ and $K^{+}$ contents and their ratios across different lines.   
Methods S1 Bioinformatics and transcriptional profiling.   
Methods S2 Gene cloning, subcellular localization, and polyclonal antibody preparation.   
Methods S3 Physiological and biochemical indices as well as determination of $\left\mathsf{N a}^{+}$ and $K^{+}$ contents.   
Methods S4 Analysis of ROS accumulation, PM internalization rate, and $\left\mathsf{N a}^{+}$ accumulation.   
Methods S5 Measurement of stomatal opening.   
Methods S6 Plant growth, treatment, and samples collection. Methods S7 Library construction and sequencing.   
Methods S8 Identification and sequence characterization of ORFs and sORFs.   
Table S1 Identification and characterization information of RabGAP subfamily II members.   
Table S2 The primer sequences used in this study.  