# Improving rice grain shape through upstream ORF editing-mediated translation regulation  

Qingqing Yang, $\begin{array}{r}{1,2,3\oplus}\end{array}$ Wenjie Zhu,1 Xu Tang,3,4,5 Yuechao Wu, $^{1}\oplus$ Guanqing Liu, $^{1}\oplus$ Dongsheng Zhao,1 Qiaoquan Liu,1,2,\* Yong Zhang,3,4,\*  

Tao Zhang1,2,\*  

![](images/16b9133caa2223fda94702eed5f79953c3f7621fed0bc57f6ad43ab288457adb.jpg)  

Jiangsu Key Laboratory of Crop Genomics and Molecular Breeding/Zhongshan Biological Breeding Laboratory/Key Laboratory of Plant Functional Genomics of the   
Ministry of Education, Agricultural College of Yangzhou University, Yangzhou 225009, China   
2Jiangsu Co-Innovation Center for Modern Production Technology of Grain Crops/Jiangsu Key Laboratory of Crop Genetics and Physiology, Yangzhou University,   
Yangzhou 225009, China   
3Department of Biotechnology, School of Life Sciences and Technology, Center for Informational Biology, University of Electronic Science and Technology of China,   
Chengdu 610054, China   
Integrative Science Center of Germplasm Creation in Western China (Chongqing) Science City, Chongqing Key Laboratory of Tree Germplasm Innovation and   
Utilization, School of Life Sciences, Southwest University, Chongqing 400715, China   
5State Key Lab of Rice Biology, China National Rice Research Institute, Hangzhou 311400, Zhejiang, China  

The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (https://academic.oup.com/plphys/pages/General-Instructions) is: Tao Zhang (zhangtao@yzu.edu.cn).\*Author for correspondence: zhangtao@yzu.edu.cn (T.Z.), zhangyong916@swu.edu.cn (Y.Z.), qqliu@yzu.edu.cn (Q.L.).  

Dear Editor, Rice (Oryza sativa) is vital for global food security and human health, influenced by its yield and quality. Grain shape is primarily determined by 3 key morphometric parameters, namely grain length, width, and thickness, which has a direct and important impact on both the yield and quality of rice, rendering it a critical consideration in rice breeding programs. Varieties with longer and narrower grains typically exhibit superior appearance quality and higher commercial value (Zhao et al. 2022).  

To date, numerous genes/quantitative trait locus (QTLs) affect­ ing rice grain shape have been identified, and gene-editing tech­ nologies also greatly promote the improvement of rice grain traits (Zhao et al. 2022). Despite this progress, many elite genes re­ main underutilized in breeding programs. The positive regulation of key genes like grain length and weight on chromosome 7 (GLW7) and grain width on chromosome 7 (GW7) is beneficial in breeding due to their minimal impact on other desirable agronomic traits. However, genetic knockout editing may not directly enhance grain shape or yield traits.  

Combining upstream ORF (uORF) identification and geneediting technology provides a promising strategy for positively regulating gene expression (Zhang et al. 2018; Si et al. 2020; Tang and Zhang 2023). The uORF is a primary regulatory element within the $5^{\prime}$ -UTR, which potentially affects main ORF (mORF) translation, and lies upstream of the mORF in $18\%$ to $57\%$ of plant mRNAs (Zhong et al. 2023). Recent studies have demonstrated uORFs contribute to phenotypic diversity, and uORF-based genet­ ic engineering for trait improvement has been performed in plants (Wang et al. 2024). Editing repressive uORFs has been successfully produced transgene-free materials with enhanced gene expres­ sion and improved performance in crops. Nonetheless, natural variation of uORF related to rice grain shape has not been found and identified, thus limiting the application of genes with breed­ ing value in improving rice appearance quality.  

In this study, considering the important impact of grain shape on rice appearance quality and yield traits, we focused on GLW7 and GW7 as regulatory targets, uORFs of which have not been identified (Wang et al. 2015, 2024; Si et al. 2016). Our goal was to develop a strategy for improving rice grain shape and yield through gene edit­ ing in molecular breeding (Fig. A). To explore this, we integrated bio­ informatic analysis of Ribo-Seq data from a public database, which revealed 1 potential uORF in the 5′-UTR of GLW7 and 2 uORFs at the $5^{\prime}$ -UTR of GW7 (Fig. B and F). Preliminary assessment results showed that altering ATG to AAA in GLW7-uorf1 reduced luciferase (LUC)/Renilla  luciferase  (REN)  activity  ratios  compared  with wild-type uORFs in the dual-LUC reporter system (Fig. D). Similar re­ sults were found for GW7-uorf1, whereas GW7-uORF2 had no effect on LUC/REN activity level (Fig. h). Therefore, 2 gRNAs targeted the sequences on both sides of GLW7-uORF1 (Fig. C; Supplementary Table S2), and another 2 gRNAs targeted the ATG start codon sites of GW7-uORF1 (Fig. G; Supplementary Table S2). Cas9-based uORF-KO constructs (Tang et al. 2019) were constructed and transformed into Nipponbare (O. sativa ssp. japonica; NIP) via Agrobacterium tumefaciens-mediated transformation (Zheng et al. 2023). Ten transgenic plants were produced, and after continuous screening from $\mathrm{{T_{0}}}$ to $\mathrm{T}_{4}$ , 3 homozygous uorf mutants from independ­ ent transformants were selected for further analysis.  

To determine the genome-editing result, sequencing analysis revealed that all uorf mutants contained base insertions or dele­ tions at the ATG site (Fig. C and G; Supplementary Fig. S1). To in­ vestigate the effects of uORFs on target gene expression and translation levels, we conducted quantitative PCR (qPCR) and western blot experiments. qPCR assays revealed no substantial difference in GLW7 and GW7 mRNA levels among various uORF mutants and NIP (Fig. E and I; Supplementary Fig. S2). Western blot assay showed that GLW7 levels of GLW7-uorf $\#30$ (55-bp dele­ tions), GLW7-uorf #34 (2- and 4-bp deletions), and GLW7-uorf $\#19$ (14-bp deletions) increased by $86\%$ ,$63\%$ , and $53\%$ , respectively (Fig. J). Based on the transient assay, qPCR, western blot experi­ ments, and previous research, we concluded that uORF mutations in GLW7 or GW7 upregulate translation level, rather than tran­ scription level.  

![](images/dcb57f4d3fa39a947bf7a095a33f51f1f7ca8f96e1f041c997d908d60447e6b6.jpg)  
Figure. Editing uORFs of GLW7 and GW7 improves rice grain appearance quality. A) Schematic representation of improving rice grain shape by manipulating uORFs through gene editing. B, F) Schematic diagram of predicted uORFs of GLW7 and GW7 via Ribo-seq (http://www.ncbi.nlm.nih.gov/; PRJNA637713 and PRJNA725700) and genome sequence analysis (http://rice.plantbiology.msu.edu/pub/data/). C, G) Nucleotide variation analysis of the target sites in mutants. The target sequence is the sequence within 20-bp from the PAM (i.e. NGG) site. The ATG bases is in red. PAM is underlined. “-” means deletion. Lowercase bases means insertion. D, H) The effect of ATG mutation to AAA in the $5^{\prime}$ lead sequence on translation of the mORF in the dual-LUC reporter system. E, I) Expressions of GLW7 and GW7 at the RNA level in young panicles of wild type and uorf mutants. J) GLW7 level was analyzed with western blot. Rice HSP82 was used as the internal control. The gray ratios were measured using ImageJ software. K) Grain and panicle traits in GLW7-uorf mutants and wild type. Bar: 5 mm. L to O) Grain length $(\mathtt{C m})$ , grain width (cm), ratio of grain length to width, and 1,000-grain weight $(\mathrm{g})$ of GLW7-uorf mutants and wild type. P to U) Analysis of main agronomic traits of both GLW7-uorf mutant and wild type. V) Polished rice of NIP and GLW7-uorf mutants. Bar: 5 cm. W to Y) Comparisons of apparent amylose content, gel consistency, and rapid viscosity analyzer spectra of rice flours of GLW7-uorf mutants and wild type. Z) Grain shape in GW7-uorf mutants and wild type. Bar: $5\;\mathrm{mm}$ . A1 to D1) Grain length (cm), grain width $(\mathtt{C m})$ , ratio of grain length to width, and 1,000-grain weight of GW7-uorf mutants and wild type. Five independent plants were used for analysis of field agronomic traits $(n\!=\!5)$ , while the remaining data from 3 biological replicates $(n\!=\!3)$ . All data are presented as mean $\pm\,S\mathbb{D}$ . \* and \*\* indicate significant differences from wild-type plants ( $\mathrm{^*P<0.05}$ and $^{**}\!\mathrm{P}<$ 0.01) by using 2-tailed Student’s t-tests.  

Further functional analysis showed that all GLW7-uorf mutants positively regulated grain length and the ratio of length to width without grain width compared to the wild type (Fig. K to N). The 1,000-grain weight values of GLW7-uorf $\#30$ , GLW7-uorf #34, and GLW7-uorf $\#19$ lines were significantly increased by $19.41\%$ ,$12.14\%$ , and $11.07\%$ , respectively, relative to NIP (Fig. O). The results from different generations showed the stability of the mutant grain shape traits (Supplementary Table S1). Milled rice from the mutant line was longer than that of NIP, with no obvious differences ob­ served in the chalkiness percentage and degree of the milled rice be­ tween GLW7-uorf mutants and NIP (Fig. V). GLW7 has an important role in panicle development (Si et al. 2016), and all mutant lines ex­ hibited consistent performance with respect to the effects on panicle traits, and panicle length and number of primary branches were significantly higher in the mutant plants than in the wild-type plants (Fig. R to S). The number of grains per panicle increased by $10.49\%$ to $26.40\%$ in mutants relative to NIP (Fig. T). Notably, no significant differences in plant height, seed setting rate, and ef­ fective tiller number were found among the mutants and NIP (Fig. P, $\mathrm{Q},$ and U).  

Interestingly, the mutant rice has lower amylose content, high­ er gel consistency, and softer taste than NIP (Fig. W to Y). The ap­ parent amylose contents of GLW7-uorf #30, GLW7-uorf #34, and GLW7-uorf #19 lines were $3.23\%$ to $5.82\%$ lower, whereas gel con­ sistencies increased by $1.67\%$ to $2.51\%$ , respectively (Fig. W and X). The viscosity profiles also changed (Fig. Y), suggesting a softer taste mutant rice compared to NIP. These results support that ma­ nipulation of GLW7 uORF can effectively improve grain shape, yield, and quality traits.  

We also examined the grain shape in GW7-uorf mutants, which exhibited increased grain length and length/width ratio, decreased grain width, and increased 1,000-grain weight compared with NIP (Fig. Z to D1). No significant differences were observed in the main agronomic traits (Supplementary Fig. S3). Moreover, both GW7-uorf mutants had lower chalkiness degrees than NIP, with no significant changes in apparent amylose, gel consistency, and rapid viscosity analyzer profile compared to NIP (Supplementary Fig. S4). These findings demonstrate that editing the uORF(s) of GW7 can im­ prove grain shape and weight without changing the eating quality. The above results also highlight the effectiveness of the method for mining potential uORFs in rapid rice breeding.  

In summary, we introduce an efficient and convenient strategy for improve grain traits of rice by disturbing or destroying uORFs of target genes using bioinformatics and CRISPR gene editing. This provides a practical approach for crop breeding, utilizing pos­ itively regulated elite genes without modifying their coding se­ quence, and creating valuable genetic resources for improving appearance quality in production applications. It is possible to create rice accessions with excellent appearance, superior qual­ ity, and sustained yield under the background of good tasting quality via CRISPR/Cas9-based uORF mutant.  

# Author contributions  

Y.Z., T.Z., and Q.L. contributed to the original concept of the proj­ ect and designed the experiments. Q.Y. and W.Z. performed the experiments. X.T. conducted the genome engineering constructs. Y.W. and G.L. analyzed the Ribo-seq data. D.Z. planted and man­ aged the field materials and investigated the major agronomic traits. T.Z., Y.Z., and Q.Y. analyzed the data and wrote the manu­ script. All authors participated in the discussion and revision of the manuscript.  

# Supplementary data  

The following materials are available in the online version of this  

article. Supplementary Figure S1. Sequencing chromatogram analysis   
of target sites in the wild type and mutants. Supplementary Figure S2. Actin, GLW7, and GW7 transcript   
levels analyzed using semiquantitative RT-PCR in young panicles   
from mutant and wild-type plants. Supplementary Figure S3. Analysis of main agronomic traits of   
both GW7-uorf mutants and wild type. Supplementary Figure S4. Comparisons of polished rice ap­   
pearance, apparent amylose content, gel consistency, and rapid   
viscosity analyzer spectra of rice flours of GW7-uorf mutants   
and wild type. Supplementary Table S1. Analysis of grain shape traits of dif­   
ferent generations between uorf mutant and wild-type plants. Supplementary Table S2. Primers used in this study. Supplementary Materials and Methods.  

# Funding  

This work was supported by the Program of Jiangsu Province Government  (BE2023331),  the  National  Natural  Science Foundation of China (32270586), the China Postdoctoral Science Foundation (2021M690552), the Jiangsu Qinglan Project, and the High Talent Supporting Program of Yangzhou University to Q.Y.; the Key R&D Program of Jiangsu Province (Modern Agriculture) (BE2022335) and the Project of Zhongshan Biological Breeding Laboratory (BM2022008-02) to T.Z.; and the Jiangsu Science Association Young Science and Technology Talent Support Project (JSTJ-2023-037) and the PAPD Program from Jiangsu Government to Q.Y. and T.Z.  

Conflict of interest statement. None declared.  

# Data availability  

The datasets used and/or analyzed during the current study are available from the corresponding author upon request.  

# References  

Si L, Chen J, Huang X, Gong H, Luo J, Hou $\mathrm{Q},$ Zhou T, Lu T, Zhu J, Shangguan Y, et al. OsSPL13 controls grain size in cultivated rice. Nat Genet. 2016:48(4):447–456. https://doi.org/10.1038/ng.3518   
Si X, Zhang H, Wang Y, Chen K, Gao C. Manipulating gene translation in plants by CRISPR-Cas9-mediated genome editing of upstream open reading frames. Nat Protoc. 2020:15(2):338–363. https://doi. org/10.1038/s41596-019-0238-3   
Tang X, Ren Q, Yang L, Bao Y, Zhong Z, He Y, Liu S, Qi C, Liu B, Wang Y, et al. Single transcript unit CRISPR 2.0 systems for robust Cas9 and Cas12a mediated plant genome editing. Plant Biotechnol J. 2019:17(7):1431–1445. https://doi.org/10.1111/pbi.13068   
Tang X, Zhang Y. Beyond knockouts: fine-tuning regulation of gene expression in plants with CRISPR-Cas-based promoter editing. New  Phytol.  2023:239(3):868–874.  https://doi.org/10.1111/nph. 19020 Wang J, Liu J, Guo Z. Natural uORF variation in plants. Trends Plant Sci.   
2024:29(3):290–302. https://doi.org/10.1016/j.tplants.2023.07.005 Wang S, Li S, Liu $\mathrm{Q},$ Wu K, Zhang J, Wang S, Wang Y, Chen X, Zhang Y, Gao C, et al. The OsSPL16-GW7 regulatory module determines grain shape and simultaneously improves rice yield and grain quality. Nat Genet. 2015:47(8):949–954. https://doi.org/10.1038/ng.3352 Zhang H, Si X, Ji X, Fan R, Liu J, Chen K, Wang D, Gao C. Genome edit­ ing of upstream open reading frames enables translational con­ trol in plants. Nat Biotechnol. 2018:36(9):894–898. https://doi.org/   
10.1038/nbt.4202 Zhao DS, Zhang C, Li $\mathrm{Q},$ Liu Q. Genetic control of grain appearance quality in rice. Biotechnol Adv. 2022:60:108014. https://doi.org/10.   
1016/j.biotechadv.2022.108014 Zheng X, Zhang S, Liang Y, Zhang R, Liu L, Qin P, Zhang Z, Wang Y, Zhou J, Tang X, et al. Loss-function mutants of OsCKX gene family based on CRISPR-Cas systems revealed their diversified roles in rice. Plant Genome. 2023:16:e20283. https://doi.org/10.1002/tpg2.   
20283 Zhong W, Archibald BN, Brophy JAN. Transcriptional and posttranscriptional controls for tuning gene expression in plants. Curr Opin Plant Biol. 2023:71:102315. https://doi.org/10.1016/j.pbi.2022.102315  