# CRISPR‑Based Modulation of uORFs in DEP1 and GIF1 for Enhanced Rice Yield Traits  

Qingqing Yang1,3,4†, Xu Tang1,2,5†, Yuechao Wu3,4, Wenjie Zhu3,4, Tao Zhang3,4\* and Yong Zhang2\*  

Dear Editor,  

Rice is a staple food for more than half of the world’s population. However, with decreasing arable land and a growing global population, increasing crop yield has become an increasingly pressing challenge. Rice plant architecture, which includes tillering, stem structure, panicle morphology, and leaf shape, represents key agronomic traits closely linked to rice yield (Li et  al 2021). The DEP1 gene is a major quantitative trait locus (QTL) associated with panicle architecture, grain shape, and nitrogen-use efficiency. Overexpression of DEP1 in transgenic rice has been shown to increase grain length and yield traits without affecting overall plant stature (Sun et  al 2018). Similarly, GRF-INTERACTING FACTOR 1 (GIF1) is a positive regulator of leaf proliferation, which can also enhance panicle and grain traits (Shimano et al 2018). Therefore, high expression of DEP1 and GIF1 holds potential for yield improvement.  

Genome editing has emerged as a relevant, versatile, and preferred tool in rice molecular breeding, significantly expanding the potential for rice improvement (Li et  al 2024). Traditionally, crop breeding research has focused on variations in coding region sequences to knock out protein-coding genes, with more recent efforts beginning to explore transcriptional cis-elements for regulating gene expression levels (Tang and Zhang 2023; Zhong et  al 2023b). However, many genetic mapping studies in plants have revealed that untranslated regions (UTRs) are enriched with elements that influence initial translation efficiency or mRNA stability, ultimately regulating protein levels post-transcriptionally (Wang et  al 2024). Among these regulatory elements, the upstream open reading frame $(\mathrm{uORF)}$ within the $5^{\prime}$ untranslated region $(5^{\prime}$ UTR) is known to play a crucial role by interfering with the translation of the downstream main ORF (mORF) (Wang et  al 2024), thereby regulating gene expression at the post-transcriptional or translational level (Zhong et  al 2023a). Some studies have demonstrated that uORF-based gene regulation is integral to plant metabolism, disease resistance, development, and nutrient uptake (Zhang et  al 2018; Zhong et  al $2023\mathrm{a}$ ;Wang et al 2024). In recent years, CRISPR editing technologies have been increasingly applied to target uORFs, offering a promising approach for enhancing crop agronomic traits (Zhang et al 2018).  

In this study, we employed gene editing strategies to target the uORFs of the DEP1 (LOC_Os09g26999) and GIF1 (LOC_Os03g52320) genes, aiming to upregulate gene expression at the translational level and improve plant, panicle, and grain morphology in rice (Fig.  1A). Initially, we integrated Ribo-seq data from public databases with nucleotide sequence analysis and bioinformatics tools, identifying two potential uORFs in the $5^{\prime}$ UTR of both the DEP1 and GIF1 genes (Fig. 1A–B, D). Among them, uORF1 and uORF2 are located at position -58 of -267 of $D E P I\,\,\,\mathrm{mORF},$ respectively; uORF1 and uORF2 are located at position -44 of -144 of GIF1 mORF. A dual-luciferase reporter assay demonstrated that altering the start codons (ATG to AAA) of GIF1- uorf1 and GIF1-uorf2 led to reduced LUC/REN activity ratios (Fig. 1A, C). Similarly, mutating the start codon of DEP1-uorf1 to AAA also resulted in a reduction of LUC/ REN activity, whereas DEP1-uorf2 showed no significant effect (Fig. 1A, E). Based on these findings, we designed two gRNAs to target the ATG sites of uORF1 and uORF2 in both the DEP1 and GIF1 genes (Fig. 1B, D). We then constructed  a  CRISPR/Cas9-based  uORF-KO  vector and introduced it into Nipponbare (Oryza sativa L.spp.japonica) via Agrobacterium-mediated transformation (Zheng et al 2023). From this, ten transgenic plants were produced, and four homozygous edited plants with different mutation types were selected for yield trait analysis after continuous screening from the $\mathrm{{T_{0}}}$ to $\mathrm{T}_{4}$ generation (Fig. 1A–B, D).  

Sequence alignment of the GIF1-uorf mutants revealed a 5-bp deletion in uORF1 of the GIF1-uorf1 line, a 6-bp deletion in uORF2 of the GIF1-uorf2 line, and 166-bp and 157-bp deletions, including the ATG sites of both uORFs, in the GIF1-uorf1&2 #1 and GIF1-uorf1&2 $\#2$ lines (Fig.  1B, Supplementary Fig.  1). In the DEP1-uorf mutants, three plants exhibited base insertions or deletions that disrupted the start codon of DEP1-uorf1, while another three plants had a single base insertion that formed a termination codon after the ATG, leading to the premature termination of DEP1-uorf2 (Fig. 1D, Supplementary Fig. 1). qRT-PCR analysis indicated that the mRNA expression levels of the DEP1 and GIF1 genes were not affected by uORF editing (Fig.  1F, P, Supplementary Fig. 2). Simultaneously, the DEP1-uorf mutants showed a significant increase in DEP1 protein abundance in young panicles compared to wild type (WT) (Fig. 1Q). To further assess the potential of uORF editing in DEP1 and GIF1 genes for trait improvement, we individually analyzed the effects of uorf mutations in these two genes. The GIF1-uorf mutant plants exhibited longer and wider leaves compared to WT (Fig. 1F), along with longer panicles, increased plant height and effective tiller number (Fig.  1G–H, 1K–L; Supplementary Fig.  3A). However, traits such as effective tiller number, seed setting rate, and grain number per panicle remained unchanged (Supplementary  Fig.  3B–D).  Additionally,  the  grains of these mutants were significantly longer and wider than those of WT, resulting in increased grain length, grain width, and 1000-grain weight (Fig. 1I, M; Supplementary Fig. 4A–C). Notably, the GIF1-uorf1&2 double mutants showed greater effects on yield traits than the individual uorf mutations, suggesting that both uORF1 and uORF2, either alone or in combination, have potential for yield improvement. For the DEP1-uorf mutants, only the DEP1-uorf1 mutant showed significant differences compared to WT, with increased grain length and length–width ratio by $5.15{-}6.50\%$ , but unchanged grain width (Fig.  1R; Supplementary Fig.  4D–F), and formed slender grain which is a popular quality in the consumer market (Custodio et al 2019). The DEP1-uorf1&2 double mutant exhibited a phenotype similar to DEP1-uorf1, with increased 1000-grain weight (Fig. 1W), while other agronomic traits such as panicle length, primary branch number, seed setting rate, and grain number per panicle remained similar to WT (Fig. 1S–V; Supplementary Fig. 3E–G). Chalkiness was either absent or only slightly increased in the mutants (Fig.  1J, W; Supplementary Fig. 5A–B). Additionally, the agronomic and grain traits of different generations ­( $\mathrm{{T}}_{3}$ generation) mutants were investigated, and the results were generally consistent  

(See figure on next page.)   
Fig. 1  Rational improvement of rice yield traits through editing the uORFs of the DEP1 and GIF1. A, Schematic representation of enhancing rice yield traits by manipulating uORFs through gene editing. B,D, Schematic diagram of predicted uORFs of GIF1 and DEP1 via Ribo-Seq (https://​ www.​ncbi.​nlm./; PRJNA637713 and PRJNA725700) and genome sequence analysis (http://​rice.​plant​biolo​gy.​msu.​edu/​pub/​data/). Sequence analysis of the target sites in mutants. The editing target sequence is in blue. PAM site is underlined. The ATG sites of predicted uORF is in red. The blue columns above the sequence represents the predicted uORF sequence. “-” means base deletion and green base means insertion. C, E, The LUC activity detected in tobacco leaves using wild type uORF and mutant uorf (ATG to AAA). REN, Renilla luciferase; LUC, firefly luciferase. F, GIF1 gene expression level. G-H, Seedling and panicle morphology of $G/F7$ -uorf1&2 and WT. Bar, $5\;\mathrm{{cm}}$ . I-J, Mature grain and polished rice of GIF1-uorf and WT. Bar, $5\:\mathrm{mm}$ . K–M, Field agronomic traits of GIF1-uorf and WT. N–O, Comparisons of apparent amylose content and rapid viscosity analyzer spectra of GIF1-uorf mutants and WT. P, Transcript abundance analysis of $D E P1.\mathsf{Q},$ DEP1 protein level was analyzed using western blot. Rice HSP82 was used as the internal control. The gray ratios were measured using ImageJ software. R-S, Grain and panicle morphology of DEP1-uorf mutants and WT. T-W, Field agronomic traits of $D E P7$ -uorf mutant and WT. Effective tiller number (S), grain number per panicle (T), seed setting rate $(\%)$ (U), 1000-grain weight (g) (V). ${\sf X},$ Polished rice of $D E P7$ -uorf and WT. Bar, $5\:\mathrm{mm}$ . Five independent plants were used for analysis of field agronomic traits (i.e. $\cap=5;$ , while the remaining data from three biological replicates (i.e. $\mathsf{n}\!=\!3.$ ). All data are presented as mean ± SD. \* and \*\* indicate significant differences from wild-type plants (\*, $P\!<\!0.05$ and \*\*, $P\!<\!0.01$ ) by using two-tailed Student’s t-tests  

![](images/b5e97d088cadaea34017e43913a793bc28004f5f1b75aa27390523f37285cb0f.jpg)  
Fig. 1  (See legend on previous page.)  

with the above conclusions (Supplementary Fig.  6A–F and Supplementary Fig.  7A–N), suggested the stability of these main agronomic traits in uorf mutant plants. These results indicate that only uORF1 of the DEP1 gene has a regulatory effect on DEP1, thereby influencing yield traits. In summary, both uORFs of the GIF1 gene improved yield traits, with the combined effect of uorf1 and uorf2 mutations being greater than individual mutations, while only uORF1 of DEP1 had a significant impact on yield traits. These findings suggest that uORFs have distinct features, and appropriate gene editing strategies should be adopted, particularly when editing multiple uORFs.  

Considering the typical negative correlation between cooking and eating quality and yield traits, we further examined several important quality indicators, including gel consistency (GC), apparent amylose content (AAC), and viscosity. The GIF1-uorf1&2 lines exhibited significantly lower AAC and viscosity values compared to WT, while no significant difference was observed in GC (Fig.  1N–O; Supplementary Fig.  5C). This suggests that the GIF1-uorf1&2 mutants have a slightly softer texture and more transparent appearance than WT. In contrast, no significant differences were found in AAC, GC, and viscosity in the DEP1-uorf mutants (Supplementary Fig. 5D–F). Although different countries around the world prefer different levels of amylose in their rice (Custodio et al 2019), softer texture and more transparent appearance rice is preferred by many countries where rice is the staple food, as China is. These results indicate that editing the uORFs of GIF1 and DEP1 can improve yield traits while maintaining or enhancing eating quality. Moreover, these findings imply that uORF editing regulates gene expression in a milder manner than traditional overexpression, offering a promising approach to addressing the tradeoffs in rice breeding. We believe that using precision genome editing to edit uORFs of coding genes represents an kindly and widely applicable approach to generating diverse alleles and improving target traits in crop breeding.  

In conclusion, we present an efficient and convenient method for upregulating gene expression by disrupting or deleting uORFs of target genes through gene editing, thereby manipulating protein levels to improve desired traits. Compared to traditional overexpression techniques, our strategy effectively enhances target gene expression without leaving genetically modified traces, making a significant contribution to the breeding and diversification of elite germplasms for high yield.  

# Abbreviations  

uORF Upstream open reading frame QTL Quantitative trait locus GIF1GRF-INTERACTING FACTOR 1UTRs Untranslated regions 5′UTR​ 5′Untranslated region mORF Main ORF NIP Nipponbare  

# Supplementary Information  

The online version contains supplementary material available at https://​doi.​ org/​10.​1186/​s12284-​024-​00743-7.  

Additional file1.  

# Author contributions  

Y.Z. and T.Z. contributed to the original concept of the project and designed the experiments. Q.Y. and W.Z. performed the experiments. X.T. designed sgRNAs and constructed all the vectors. Y.W. analyzed the Ribo-seq data and searched for uORFs. Y.Z., T.Z., Q.Y., and X.T. analyzed the data and wrote the manuscript. All authors participated in discussion and revision of the manuscript.  

# Funding  

This work was supported by the Biological Breeding-Major Projects (2023ZD04076) to X.T., the Program of Jiangsu Province Government (BE2023331) and the National Natural Science Foundation of China (32270586) to Q.Y., the Key R&D Program of Jiangsu Province (Modern Agriculture) (BE2022335) and the Project of Zhongshan Biological Breeding Laboratory (BM2022008-02) to T.Z., Jiangsu Science Association Young Science and Technology Talent Support Project (JSTJ-2023–037) and the PAPD Program from Jiangsu Government to Q.Y. and T.Z., Jiangsu Qinglan Project and High Talent Supporting Program of Yangzhou University to Q.Y.  

# Data Availability  

All data generated or analysed during this study are included in this article and its supplementary information files.  

# Declarations  

Ethics Approval and Consent to Participate Not applicable.  

# Consent for Publication  

Written informed consent for publication was obtained from all participants.  

# Competing Interests  

The authors declare no competing interests.  

Received: 27 August 2024   Accepted: 8 October 2024   
Published online: 25 October 2024  

# References  

Custodio MC, Cuevas RP, Ynion J, Laborte AG, Velasco ML, Demont M (2019) Rice quality: how is it defined by consumers, industry, food scientists, and geneticists? Trends Food Sci Technol 92:122–137   
Li GM, Tang JY, Zheng JK, Chu CC (2021) Exploration of rice yield potential: Decoding agronomic and physiological traits. Crop J 9(3):577–589   
Li B, Sun C, Li J, Gao C (2024) Targeted genome-modification tools and their advanced applications in crop breeding. Nat Rev Genet 25(9):603–622   
Shimano S, Hibara KI, Furuya T, Arimura SI, Tsukaya H, Itoh JI (2018) Conserved functional control, but distinct regulation, of cell proliferation in rice and Arabidopsis leaves revealed by comparative analysis of GRF-INTERACTING FACTOR 1 orthologs. Development 145(7):159624. https://​doi.​org/​10.​ 1242/​dev.​159624   
Sun S, Wang L, Mao H, Shao L, Li X, Xiao J, Ouyang Y, Zhang Q (2018) A G-protein pathway determines grain size in rice. Nat Commun 9(1):851   
Tang X, Zhang Y (2023) Beyond knockouts: fine-tuning regulation of gene expression in plants with CRISPR-Cas-based promoter editing. New Phytol 239(3):868–874   
Wang JE, Liu JH, Guo ZL (2024) Natural uORF variation in plants. Trends Plant Sci 29(3):290–302   
Zhang H, Si X, Ji X, Fan R, Liu J, Chen K, Wang D, Gao C (2018) Genome editing of upstream open reading frames enables translational control in plants. Nat Biotechnol 36(9):894–898. https://​doi.​org/​10.​1038/​nbt.​4202   
Zheng X, Zhang S, Liang Y, Zhang R, Liu L, Qin P, Zhang Z, Wang Y, Zhou J, Tang X, Zhang Y (2023) Loss-function mutants of OsCKX gene family based on CRISPR-Cas systems revealed their diversified roles in rice. Plant Genome 16(2):e20283  
Zhong VV, Archibald BN, Brophy JAN (2023a) Transcriptional and post-transcriptional controls for tuning gene expression in plants. Curr Opin Plant Biol 71:102315   
Zhong Z, Liu G, Tang Z, Xiang S, Yang L, Huang L, He Y, Fan T, Liu S, Zheng X, Zhang T, Qi Y, Huang J, Zhang Y (2023b) Efficient plant genome engineering using a probiotic sourced CRISPR-Cas9 system. Nat Commun 14(1):6102  

# Publisher’s Note  

Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.  