Review  

# Engineering crop performance with upstream open reading frames  

Rui Mou1,4, Ruixia Niu1,4, Ruoying Yang1, and Guoyong Xu $\oplus$ 1,2,3,\* Plants intricately regulate the expression of protein-coding genes at multiple stages – including mRNA transcription, translation, decay, and protein degradation – to control growth, development, and responses to environmental challenges. Recent research highlights the importance of translational reprogramming as a pivotal mechanism in regulating gene expression across diverse physiological scenarios. This regulatory mechanism bears practical implications, particularly in bolstering crop productivity by manipulating RNA regulatory elements (RREs) to modulate heterologous gene expression through transgene and endogenous gene expression through gene editing. Here, we elucidate the potential of upstream open reading frames (uORFs), a prominent and stringent class of RREs, in optimizing crop performance, exemplifying the efficacy of translational control in enhancing agricultural yields.  

# Highlights  

New breeding strategies are needed to accelerate crop improvement to address the challenges posed by frequent climate extremes and a growing global population.  

Translational control is becoming crucial in plant growth, development, and stress responses, with upstream open reading frames (uORFs) playing a key role in modulating translation precisely.  

Technological advances, such as ribosome profiling, have significantly improved our understanding of uORFs and their regulatory mechanisms.  

# Genetic engineering in modern agriculture  

Modern agriculture faces many pressing challenges today. These include a rapidly growing global population, decreasing arable land, unpredictable climatic stresses, new and re-emerging diseases and environmental pollution [1,2]. Concurrently, the agricultural sector grapples with a labor shortage while striving to meet the increasing demand for crops in human food, animal feed, and industrial applications. Moreover, consumers worldwide increasingly seek highquality, nutrient-rich produce with distinctive nutritional profiles and potential health benefits, necessitating a shift in agricultural practices [3]. Compared to traditional breeding strategies, genetic engineering is a potent solution to swiftly address these challenges with transgene technology [4] and gene editing techniques [5]. Both methodologies require precise spatial and temporal modulation of gene expression. Additionally, they must control expression levels to achieve desirable agronomic traits while avoiding unintended effects [6]. With respect to controlling expression levels, uORFs (see Glossary) play a central role by regulating gene expression at the translation level.  

For pleiotropic genes, especially stressresilient ones with antagonistic pleiotropy, stringent control of expression levels is essential to balance stress tolerance and growth. uORFs are critical for this balance.  

Recent advancements in transgene and gene editing technologies enable precise uORF engineering, allowing fine-tuned control of transgene and endogenous gene expression.  

# What is a uORF?  

Plants dynamically reprogram gene expression to accommodate growth, development, and stress responses, utilizing proteins, RNAs, or metabolites as agents for cellular activities. In the context of protein-coding genes, intricate processes involve regulating messenger RNA (mRNA) expression at transcription and translation levels (Figure 1A). Consequently, the promoters for transcriptional control [7–16] and RREs for translational control [17–29] are plausible cis-elements that could be managed to tailor heterogeneous or endogenous gene expression levels. Emerging evidence underscores the essential role of translational control in governing gene expression, both during static states and transitions in plants [30–38]. uORFs are common RREs that impose stringent translational control over the downstream major ORF (mORF). In Arabidopsis thaliana Col-0, which has a wellannotated transcriptome, it is estimated that approximately $50\%$ of genes contain at least one uORF [39,40]. Recent advances in using uORFs for modulating gene expression through transgene and  

1State Key Laboratory of Hybrid Rice, Institute for Advanced Studies (IAS), Wuhan University, Wuhan, Hubei   
430072, China   
2Hubei Hongshan Laboratory, Wuhan, Hubei 430070, China   
3RNA Institute, Wuhan University,   
Wuhan, Hubei 430072, China   
4These authors contributed equally to this work.  

![](images/1cd5d304b36b413b4c8fc1744645fe6a02975c904ff60fc794a8c7288e3e11a4.jpg)  

$\mathsf{\Sigma}^{\prime}$ -leader: also known as the $5^{\prime}$ -untranslated region (5′-UTR), this is a sequence of nucleotides located at the beginning of an mRNA molecule, upstream of the initiation codon, that initiates translation of the protein-coding sequence.  

# Glossary  

$\mathtt{c}_{+}$ strategy: a method for conditionally upregulating transgene expression,   
spatially and temporally, to desired   
levels using specific promoters and   
translational control   
modules.  

Cell division cycle 123 (CDC123): an ATP-grasp protein that regulates the assembly of the eIF2 complex.  

CRISPR-aTrE-uORF: a gene-editing strategy that utilizes clustered regularly interspaced short palindromic repeats (CRISPR)-mediated base editing or INDEL techniques to alter uORF types across the genome – from none to some, some to none, or different variants – to modulate translation efficiency.  

Hematopoietic protein 1 (HEM1): a key component of the WAVE regulatory complex (WRC) in actin cytoskeleton regulation and a global translational regulator.  

Major open reading frame (mORF): also known as the primary ORF (pORF) the main open reading frame within an mRNA that encodes the primary protein product. The same mRNA may also contain upstream ORFs (uORFs), internal ORFs (iORFs), and downstream ORFs (dORFs).  

Representative gene model: a reference transcript isoform chosen from among various isoforms produced. This model is selected based on specific criteria, such as being the longest isoform, the most abundant and frequently expressed, or encoding the most conserved protein.  

(Figure legend continued at the bottom of the next page.)  

TTrreennddss iinn PPllaanntt SScciieennccee  

Figure 1. Upstream open reading frame (uORF)-mediated translational control. (A) uORFs within the gene structure. A representative transcript isoform is used to annotate uORFs in the $5^{\prime}$ -leader. Attributes influencing uORFmediated translational control are shown. uORFs are classified by their stop codons, and mRNAs are categorized as uORF-free, type1 uORF-only, or type2 uORF-containing. Further classification is based on the number of uORFs, single or multiple, with various organization patterns. Traits such as RNA modifications, RNA structures, linear motifs, initiation codon context (ICC; ICCu, uORF ICC; ICCm, mORF ICC), $5^{\prime}$ -space, and $3^{\prime}$ -space may influence uORF functions. (B) Mechanisms to overcome suppressive uORFs. Type1 uORFs allow mORF translation through leaky scanning and reinitiation, while type2 uORFs enable translation only via leaky scanning. These mechanisms result in different levels of translational suppression compared to 5′-leaders without suppressive uORFs. (C) Criteria for defining a translatable uORF.  

Responsive uORF: a uORF in the 5′-leader of mRNA that regulates the mORF translation in response to specific physiological or environmental signals, such as nutrient availability, stress conditions, or hormonal  

# signals.  

RNA regulatory elements (RREs): including linear motifs, unique structures, and nucleotide modifications in the 5′-leader, coding sequence, and 3′-UTR.  

Suppressive uORF: a uORF in the $5^{\prime}$ -leader of mRNA that suppresses the mORF translation by affecting ribosome scanning.  

The advent of ribosome profiling, initially developed in yeast, has significantly enhanced our understanding of uORF-mediated translational control [41]. By enabling high-resolution measurement of ribosome occupancy at the nucleotide level, this technique captures ribosome engagement on both uORFs and their corresponding mORFs under static conditions and during dynamic state transitions. Ribosome binding signals reveal whether a uORF is actively translated, while the relative binding intensity between uORFs and mORFs indicates whether the uORF exerts a suppressive effect on mORF translation. Furthermore, changes in the relative binding intensities in response to developmental or environmental stimuli suggest how uORF-mediated translational control adapts to these conditions [42–45]. These genome-wide studies, complemented by targeted case studies (deposited in the uORFlight database [39]), have unveiled three key properties that define the uORF function. We discuss how uORF attributes afford the three uORF properties as the rationale in genetic engineering.  

gene editing have significantly enhanced crop performance. Our exploration begins with defining uORFs and delving into their different attributes (Figure 1A and Box 1).  

uORF is closer to the $5^{\prime}$ -cap of mRNA than the mORF, prioritizing the translation of uORF and inhibiting the translation of mORF. According to the classical translation initiation model, a 43S preinitiation complex loads onto the mRNA by the cap-associated eIF4F complex and starts to scan the optimal initiation codon from the $5^{\prime}$ -end [36,46,47]. uORFs contain an earlier initiation codon. When this initiation codon is in an optimal context, the 80S ribosome preferentially assembles at the uORF initiation site. This reduces ribosome accessibility to the mORF, resulting in lower protein yield from the mORF. Consequently, on the one hand, most uORFs impose a translational block on mORF translation. On the other hand, the extremely high translation fidelity makes suppressive uORFs challenging to overcome. The sophisticated translation machinery ensures stringent and faithful selection of the initiation codon, preventing the translation from incorrect initiation codons to produce junk peptides. In eukaryotes, high levels of eIF5 decrease the stringency of initiation codon selection, while high levels of eIF1 increase it. To mitigate disturbances from eIF1 and eIF5, cells employ a regulatory feedback loop for both factors, resulting in low efficiency of leaky scanning [48]. Thus, leaky scanning appears to have low efficiency. According to the classical translation model, most ribosomes disassemble and release after encountering uORF stop codons. Only a small fraction of post-termination ribosomes remain on mRNAs and can reinitiate at the mORF. This reinitiation likely occurs either by retaining initiation factors due to the shorter length of uORFs or by recruiting initiation factors such as eIF4G, eIF3, and DENR–MCT-1 [49–63]. Current knowledge suggests that reaching the downstream mORF initiation codon can occur only through leaky scanning or reinitiation after translating uORFs [64]. Theoretically, type1 uORFs whose stop codons are located before the mORF initiation codon could allow mORF translation through leaky scanning and reinitiation. Type2 uORFs whose stop codons are located after the mORF initiation codon could solely offer mORF translation through leaky scanning. Thus, the stringent  

Transcript isoform: a variant of mRNA produced from the same gene through processes of alternative splicing, alternative transcription start sites, and alternative polyadenylation sites.  

# The first property: suppressive uORFs  

Transcriptional control: a mechanism for regulating gene expression by controlling when, where, and how often a gene is transcribed into mRNA, mainly involving the regulon between transcription factors and DNA sequences.  

# Why engineering with uORFs?  

Translatable uORF: a uORF in the 5′- leader of mRNA can be translated into a peptide, which may act in trans or cis to modulate mORF translation.  

Translational control: a mechanism for regulating gene expression by influencing the translation of mRNA into protein, mainly involving the regulon between RNA-binding proteins and RREs.  

Translation efficiency (TrE): the rate at which mRNA is converted into its corresponding protein product by the ribosome.  

Translation factors (TrFs): including translation initiation, elongation, and termination factors.  

uORF: an ORF with its initiation codon in the $5^{\prime}$ -leader of mRNA; it can regulate the translation of the mORF by affecting ribosome scanning.  

# Box 1. uORF attributes  

A uORF is a small ORF whose initiation codon is located in the 5′-leader region upstream of the mORF. Here, we focus on AUG as the initiation codon, as it is the most well-understood. The uORF types depend on the location of their stop codons, which can vary: type1 has a stop codon within the 5′-leader, type2 has a stop codon within the mORF, and type3 shares a stop codon with the mORF. Type3 uORFs are very rare and are not further discussed here [39]. Based on uORFtypes, mRNAs can be categorized into three types: uORF-free, type1-only, or type2-containing if they contain at least one type2 uORF. Additionally, uORF-containing mRNAs can be further divided based on the number of uORFs they contain: single uORF or multiple uORFs with different organization patterns. These attributes, along with others – including 5′- space, initiation codon context, codon usage, stop codon context and $3^{\prime}$ -space – may influence how much a specific uORF can affect mORF translation.  

Moreover, other regulatory RREs, including surrounding RNA modifications, unique structures, and linear motifs that dynamically exist or couple with the actions of trans-acting factors, may also contribute to uORF functions. These attributes collectively form the basis of uORF functionality. It is important to note that our understanding of these attributes and the dynamic regulators is primarily based on a limited number of case studies in eukaryotes, given its prevalent existence in eukaryotic mRNAs [39,100,101]. Consequently, the engineering rationale may not be universally applicable to all genes.  

mechanisms of translation make both processes inefficient, making suppressive uORFs difficult to overcome, particularly for type2 uORFs (Figure 1B).  

# The second property: translatable uORFs  

Another interest in using uORFs in engineering is their potential to encode functional peptides (Figure 1C). Some uORF peptides can cause ribosomal arrest during translational elongation or termination, preventing ribosomes from accessing mORFs [65]. For example, the arrest peptide encoded by a uORF in Arabidopsis S-adenosylmethionine decarboxylase mRNA inhibits the translation of mORF, the product of which is an enzyme crucial for polyamine biosynthesis. Increased cellular polyamine levels trigger a feedback mechanism, enhancing translational inhibition and reducing the enzyme's protein level [66]. This type of feedback regulation is common in many uORF-containing genes, particularly those involved in metabolism across different species. Since uORF-peptide-dependent cis-regulation mechanisms often respond to metabolites and environmental stresses [65,67,68], engineering uORFs with this function must account for their impact on signal sensing and feedback regulation.  

In addition to their cis-regulatory functions, some uORF peptides may also act in trans. For example, in animals, these uORF peptides can form stable complexes with mORF-encoded proteins or act as a major histocompatibility complex class I ligand during immune responses [69–71]. The growing interest in peptide biology has driven the continuous identification of translatable uORFs [72]. Consequently, determining how many mRNAs contain translatable uORFs remains a key area of interest for understanding both cis- and trans-regulatory mechanisms and, of course, for engineering purposes $(\mathsf{B O}\times2)$ ). Engineering uORFs with this function must consider their potential effects on trans-acting roles.  

# Box 2. uORF translatability  

Searching for AUG-initiated uORFs indicates the prevalence of uORFs among plant mRNAs [39]. These sequence-based translatable uORFs will then be fed into algorithms to identify translatable uORFs using data generated by ribosome footprinting technology [102]. This gold-standard technique determines translatability by mapping ribosome occupancy on mRNAs [41]. Translatable uORFs are identified by the presence of ribosome binding signals (ribosome-binding-based translatable uORFs) or, more precisely, by a discernible 3-nt periodicity (3-nt-based translatable uORFs; see Figure 1C in main text). However, deciphering translational activity becomes intricate when mRNAs harbor multiple uORFs, especially when their organizations overlap or with mORFs (see Figure 1C in main text), posing challenges in the unambiguous assignment of ribosome footprints. Such situations can lead to the underestimation of translatable uORFs. Although progress has been made in identifying translatable uORFs in plants [103], further studies are needed to understand their functions in a cis or trans manner. If they function in cis, it remains interesting to find out whether they are responsive to certain metabolites or stressors, which are crucial for future engineering endeavors. Reported cases in yeasts, plants, and animals may provide insights into how to uncover their potential roles.  

# The third property: responsive uORFs  

Akin to cis-elements on DNA responsive to growth, development, and stresses through binding with trans-acting factors, uORFs, as essential RREs acting on mRNAs, also respond to these signals through different strategies (Figure 1D). The well-known responsive uORFs were initially identified through studies on integrated stress responses in yeasts and animals. Under stress conditions, global translation is typically inhibited by the limited availability of the eIF2 complex due to increased phosphorylation of eIF2α by eIF2α kinases. However, the decreased availability of this complex can enhance the reinitiation of adaptive mORFs with uORFs, producing stressresolving products, such as yeast GCN4 and animal ATF4 [37,39]. It is noted that phosphorylation of eIF2α in plants may not be sufficient to inhibit global translation. The evidence comes from studies on the Arabidopsis mutant deficient in General Control Non-derepressible 2 kinase (GCN2), the only known eIF2α kinase in plants. Despite the lack of eIF2α phosphorylation in this mutant, the response of uORFs in TBF1 mRNA during immune activation remains comparable to that of wild-type plants. However, it is still unclear whether GCN2-mediated eIF2α phosphorylation serves as a regulatory mechanism for other uORFs under unknown conditions.  

Beyond eIF2α phosphorylation, plants can utilize cell division cycle 123 (CDC123) to enhance the availability of the eIF2 complex, thereby promoting the translation of immune genes [73]. Unlike in animals and yeasts, where eIF2α phosphorylation typically inhibits global translation, plants employ an alternative strategy: the condensation of hematopoietic protein 1 (HEM1) with translation factors (TrFs). This mechanism limits TrF availability for translation, thereby inhibiting global protein synthesis. As a result, this process prevents the excessive translation of immune genes, which could otherwise lead to tissue damage [74,75]. Moreover, a recent study in Arabidopsis revealed that phosphorylation of the carboxyl-terminal eIF2α by SPA kinases increases eIF2 complex assembly and boosts translation efficiency (TrE) during photomorphogenesis, whereas unphosphorylated eIF2α results in reduced TrE [76]. In addition, the activation of target-of-rapamycin (TOR) followed by the phosphorylation of S6 kinase 1 (S6K1) and eIF3h is critical for the translation reinitiation of auxin-responsive factors that contain multiple uORFs [55,62]. If uORF responsiveness is conserved in response to global translational perturbations, the roles of GCN2, CDC123, HEM1, and SPA kinases in regulating uORFs during these disturbances remain to be elucidated. Furthermore, it is still unknown whether and how uORFs exhibit specific responsiveness to these signals.  

Several studies demonstrate how uORFs respond specifically to various signals. Many of these responsive uORFs employ a ribosomal arrest strategy that involves uORF-encoded peptides. In the presence of specific signals, these uORFs can enhance the ribosomal arrest, allowing for reduced translation of the mORF. Changes in their uORF codon usage can affect their responsiveness. For example, responsive uORFs are found in Arabidopsis S-adenosylmethionine decarboxylase, bZIP11, and TBF1, which respond to changes in endogenous polyamine, sucrose, and phenylalanine levels, respectively [67,68]. Some uORFs function through this responsive strategy, although the specific signals they respond to have not yet been identified [65].  

A responsive uORF in Arabidopsis NIP5;1, a boric acid diffusion facilitator, consists only of the AUG initiation codon and UAA stop codon but confers ribosomal arrest after sensing the increase of environmental boric acid [77]. This indicates the existence of a ribosomal arrest strategy independent of uORF peptides. The information about most of these known responsive uORFs in plants, including crops, has been deposited in the uORFlight database [39]. Recent studies in animals and plants have suggested using RNA secondary structures to control uORFs or their responsiveness [45,78,79]. Other responsive strategies, such as dynamic m6A modification for human ATF4 uORF in response to starvation and linear purine-rich motifs for Arabidopsis TBF1 uORF in response to immune activation, have also been reported [42,80,81]. These newly emerged strategies may also effectively regulate other responsive uORFs, suggesting that uORFs can respond to specific signals through both the specific peptides they encode (peptide-dependent manner) and peptide-independent mechanisms. The latter may involve trans-acting factors that interact with other specific RREs functionally associated with uORFs.  

# How to engineer with uORFs?  

uORFs have been successfully engineered to improve crop performance through transgene and gene editing approaches. Identifying uORFs for transgene applications typically requires uORFs that suppress basal translation of the gene of interest but can respond to specific signals. These uORFs are often translatable by hijacking ribosomes, but their regulatory mechanisms may act in a peptide-dependent or -independent manner. In gene-editing strategies, the suppressive nature of uORFs is primarily used to fine-tune their repressive effects on the mORF translation. This modulation can be achieved by generating, removing, or altering uORF traits, such as initiation codon context. If a uORF is translatable and functions in a peptide-dependent manner, modifications to its length or coding sequence can influence its regulatory activity. Conversely, if a uORF operates in a peptide-independent manner and is regulated through associated RREs, editing these RREs can similarly adjust its suppressive and responsive capacities. The following section will illustrate these two approaches and discuss the uORF properties utilized in each case (Table 1). Further considerations are also discussed to expand their applications in agriculture.  

Table 1. Engineering uORFs using transgene and gene-editing technologies   

![](images/55f88983ce0a3d7a2a583a4ecdf0b3e4a6c05248ba8705ad8bcf1bd40fe2c454.jpg)  
Abbreviations: Ls, Lactuca sativa; Os, Oryza sativa; Sl, Solanum lycopersicum.  

# Transgene with uORFs  

A single gene can influence multiple traits. When the expression of such a pleiotropic gene results in competing effects – one trait beneficial and the other detrimental to an organism's fitness or crop productivity – the gene exhibits antagonistic pleiotropy [82]. This phenomenon is common among stress-resilient genes, which help plants to endure biotic and abiotic stresses but often hinder plant growth when overexpressed [83,84]. Consequently, using constitutive expression strategies, like constitutive promoters or translational enhancers, is not feasible for transgene breeding with these genes. Instead, transgene breeding should aim to restrict gene expression under normal conditions while allowing for conditional upregulation in response to stress, a strategy termed conditional upregulation $^{\tt c+}$ strategy). Given the limited understanding of whether uORF-mediated translational control is spatiotemporal, the current $\mathrm{C}+$ strategy involves coupling uORFs-contained-5′-leader with known spatiotemporal promoters to regulate gene expression [20,22].  

An example of this approach is the expression control cassette $\mathsf{P r o_{T B F1}\!-\!u O R F s_{T B F1}}$ from the Arabidopsis TBF1 gene, which has been used successfully to engineer disease-resistant plants in both Arabidopsis and rice (Figure 2A). This cassette integrates the $\mathsf{P r o}_{\mathsf{T B F}^{\prime}1}$ transcriptional control module and the $\mathsf{U O R F S}_{\mathsf{T B F1}}$ translational control module, stringently regulating the autoimmune gene snc1, whose weak expression can retard plant growth. This method increases the success rate of obtaining transgenic lines that maintain normal growth while resisting virulent bacterial and oomycete pathogens, outperforming constitutive strategies or separate transcriptional or translational controls. In rice, this cassette similarly ensures stringent control of the heterologous expression of the master immune regulator NPR1, resulting in transgenic lines with broad-spectrum disease resistance to both bacterial and fungal pathogens.  

We propose a working model that integrates the three properties of uORFs to help understand transgene expression. Without immune activation, the ProTBF1 transcriptional control module allows for low transcription of transgene mRNA. Additionally, the presence of suppressive $\mathsf{U O R F S}_{\mathsf{T B F1}}$ in the $5^{\prime}$ -leader of the pre-existing transgene mRNA further restricts transgene translation, resulting in very low levels of transgene protein that stay below the harmful threshold for plant growth. Upon pathogen infection, the responsive uORFsTBF1 in the pre-existing transgene mRNAs may react to the signal through integrated effects, utilizing strategies such as uORF peptide [68], RNA secondary structure [45], or linear purine-rich motifs [42,80]. These mechanisms enhance the translation of the transgene. Moreover, the $\mathsf{P r o}_{\mathsf{T B F}+}$ transcriptional control module will be activated to supply additional transgene mRNAs for more protein yield [20]. However, whether translational and transcriptional inductions are coupled or sequentially processed remains unknown. In some cases, the uORFsTBF1 translational control module alone is sufficient to restrict immune gene expression. For example, transgene expression of the Arabidopsis immune gene $C B P6O g$ with uORFsTBF1, even when driven by the CaMV 35S constitutive promoter, can produce transgenic lines with normal plant growth while preserving resistance to pathogens [23].  

The capacity for modular assembly will likely enable the control of transgenes from various sources, such as incorporating metabolic enzymes, even from prokaryotes, into eukaryotic crops. Additionally, this flexibility allows for the seamless combination of promoters and translational control modules to achieve diverse $\mathrm{C}+$ expression goals. For instance, the inducible promoter from the Nicotiana benthamiana PR1 can be transplanted with the uORFsACD11 module from the Arabidopsis ACD11 gene [22] (Figure 2A). These findings highlight the promising future of exploring and combining different modules to control transgene expression in tailored and precise ways, allowing for the full modulation of gene expression both spatially and temporally, as well as the control of expression levels.  

![](images/49402d3049498b3f4cec45c1adc35b1a7b38242d411626c8259ebdf17ad3ee00.jpg)  

Figure 2. Examples to illustrate upstream open reading frame (uORF) engineering in crops. (A) Schematic of the uORF-containing expression control cassette for engineering uORFs through transgene. The 5′-leader with uORFs from the Arabidopsis TBF1 gene is fused with either the CaMV 35S promoter (35S) or the TBF1 promoter $(\mathsf{P r o}_{\mathsf{T B F1}})$ , integrating transcriptional and translational control modules. The ProPR1-uORFsACD11 cassette features the inducible Nicotiana benthamiana PR1 promoter and the $\mathsf{U O R F S}_{\mathsf{A C D1}\,1}$ translational control module from Arabidopsis ACD11. These cassettes can limit target gene expression without pathogen infection to avoid negative effects on plant growth while enabling expression upon infection to boost disease resistance. (B) Gene editing to remove a known suppressive uORF or modify ICCu for enhancing mORF translation to increase lettuce vitamin C content. $G G P2$ , GDP-L-galactose phosphorylase homologs in lettuce for vitamin C biosynthesis. (C) Clustered regularly interspaced short palindromic repeats (CRISPR)/ CRISPR-associated protein 9 (Cas9) (CRISPR/Cas9) used to insert a custom-designed uATG-containing element to generate a type2 uORF for improving rice grain quality. Waxy, whose reduced expression leads to grains with lower amylose content. Abbreviations: CRISPR-INDEL, CRISPR-insertion and deletion; WT, wild type. The positions of edited uORFs are marked by a triangle with different colors matching the edited lines.  

# Gene editing of endogenous uORFs  

Protein abundance is mainly determined by the rates of mRNA transcription, translation and decay, and protein degradation. uORFs are prevalent and efficient regulatory elements that influence TrE. Genomic sequence variations, ranging from single nucleotides to insertions and deletions, can lead to phenotypic trait variation by affecting gene expression. When such variations occur in uORFs or associated attributes, they are likely to change mRNA TrE and control the levels of proteins encoded by mORFs. Many natural uORF variations have been associated with plant phenotypic trait variations [39,85–90] (reviewed in [91]). uORF variants can also be generated by nonspecific DNA mutation approaches, such as ethyl methanesulfonate (EMS) treatment. For example, EMS treatment has been reported to generate a type2 uORF on a negative regulator of the brassinosteroid signal transduction pathway, grain size and number 1 (GLN1), leading to increased rice grain size and 1000-grain weight [92]. However, variations in regions that control protein translation have been largely neglected in genetic studies compared to those that directly influence protein-coding sequences. These studies highlight the critical importance of focusing on these variations, as they can significantly impact plant traits and agricultural productivity.  

These natural variations can result in differences in uORF types, lengths, initiation codon context, and even the presence or absence of the uORF itself [91]. Such natural events inspire the artificial design of uORFs to offer different control levels on the target mORFs (Table 1). Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein (CRISPR-Cas) technology provides a targeted approach for introducing artificial variants, ranging from single nucleotide changes to insertions and deletions, through methods like CRISPR-insertion and deletion (CRISPR-INDEL), CRISPR-base editing (CRISPR-BE) and CRISPR-prime editing (CRISPRPE) [5]. This technology has been successfully used in uORF engineering. CRISPR-INDEL with a single gRNA targeting the initiation codon to remove a known suppressive uORF or modifying its initiation codon context can enhance mORF translation. This approach has been applied to enhance the translation of mORFs encoding homologs of GDP-L-galactose phosphorylase (GGP) in lettuce and tomato, thereby increasing vitamin C biosynthesis in these crops [18,19] (Figure 2B). It has also been used to enhance the translation of Heading Date 2 (HD2), a negative regulator of flowering in rice, thereby delaying rice heading [17], and to increase the translation of Rab GTPase-activating protein 22 (RabGAP22), a positive regulator of salt tolerance in tomato, resulting in improved salt tolerance [29]. Another method, CRISPR-BE, can edit the stop codon of a suppressive uORF and extend its length to enhance its suppression function. Additionally, CRISPR-PE can generate a suppressive uORF by introducing initiation codons in the $5^{\prime}$ -leader. These methods have been used to obtain graded decreased expression levels of OsDLT, which encodes a member of the GRAS family involved in the brassinosteroid transduction pathway, resulting in a series of rice plants with varied plant height and tiller numbers [27].  

A recent strategy involves using CRISPR-INDEL and CRISPR-BE technologies to generate and screen variants of uORF types at a genome-wide scale. This method, called CRISPR-aTrEuORF, can change uORF types from none to some, some to none, or to different ones across the genome to generate alternative TrE (aTrE). Screening these uORF variants in rice has identified several disease-resistant genes. For instance, switching a type1 uORF to a type2 uORF by CRISPR-BE for the plastid glutamine synthetase 2 (GS2) can further suppress the translation of this enzyme, providing broad-spectrum disease resistance in rice with minimal fitness costs [21]. Generating a type2 uORF is a promising strategy for achieving in-locus knockdown of endogenous gene expression. This has been demonstrated using CRISPR/Cas9 coupled with chemically modified donor DNA to insert a tailor-designed uATG-containing element, creating a type2 uORF. This strategy has been successfully tested for several rice genes, including Waxy, whose reduced expression leads to grains with lower amylose content [26] (Figure 2C). These examples suggest that it is plausible to generate or remove uORFs and modify uORF attributes to control mORF translation, with significant applications in agricultural domains.  

An immediate question is how to apply these strategies to other genes. In cases where enhancing mORF translation is desired, removing a suppressive uORF, particularly a type2 uORF, is more promising. Optimal results may be achieved by changing mRNA types in the following order: best from type2 to uORF-free, next best from type2 to type1-only, and least from type1-only to uORF-free, which have been accomplished using CRISPR-BE or CRISPR-INDEL [18,21]. Conversely, generating a suppressive uORF, particularly a type2 uORF, is more promising for suppressing mORF translation. Optimal results may be achieved by changing mRNA types in the following order: best from uORF-free to type2, next best from type1-only to type2, and least from uORF-free to type1-only, which can be achieved using CRISPR-BE, CRISPR-INDEL, or CRISPRPE [21,26,27]. Other strategies, such as modifying initiation codon context or altering uORF codons and lengths, can also be used to modify uORF suppression effects. A pilot experiment in a transient expression system may be necessary before generating transgenic plants [26].  

Since uORF functions may be influenced by unknown RREs, generating as many potential variants as possible can help to achieve finely tuned and gradient expression levels. Although transforming and screening all possible edited transgenic plants may be resource-intensive, it can be worthwhile when the process is not costly or time-consuming – especially for crops or varieties that can readily produce large transgenic pools. After generating the transgenic plants with desirable uORF variants, selective ribosome footprinting technology can be employed to assess the TrE of the modified mORF specifically. This approach is particularly useful in cases where antibodies are unavailable or mRNA levels are low, limiting the efficacy of genome-wide ribosome footprinting in capturing sufficient footprints for evaluation [21].  

# The advancement of engineering crop performance with uORFs  

For genes that need to be regulated using the $\mathrm{C}+$ transgene breeding strategy, transcriptional control with a spatiotemporal promoter may suffice. However, genes where minor expression changes can trigger significant cellular reprogramming – such as core regulators of signal transduction, transcription, translation, or metabolism, or those exhibiting cellular toxicity – require a more stringent regulatory module to prevent unwanted effects. Although only a few uORFs have been extensively studied, cross-species analyses suggest that uORFs have the potential to meet these stringent expression requirements. For example, coupling uORFsTBF1-mediated translational control to $\mathsf{P r o}_{\mathsf{T B F}^{+}1}$ -mediated transcriptional control has been shown to improve the likelihood of obtaining normal-growth plants [20]. As the role of translational regulation in gene expression gains recognition, further investigations are likely to identify additional RREs that can meet these stringent requirements. Such precise translational control permits basal mRNA expression while restricting translation, presumably enabling pre-existing mRNAs to respond rapidly to incoming signals and adapt to changing conditions.  

For gene editing approaches aimed at modulating endogenous gene expression, editing promoter regions and RREs has shown promise in enhancing crop performance. However, promoters are often long, and their core regulatory regions can be uncertain or distant from the gene loci. Similarly, other RREs require precise mapping of their locations, as well as a thorough understanding of their trans-acting factors and regulatory consequences. By contrast, uORF locations and types are precisely defined within mRNAs. uORF-mediated suppression of mORF translation arises from the early recognition of uORFs when the 43S scanning ribosome searches for the initiation codon – a mechanism highly conserved in eukaryotes [21]. At a minimum, editing uORFs can modulate their suppressive strengths, although further investigation is needed to determine if such edits will impact their responsiveness to specific signals.  

# Concluding remarks and future perspectives  

The accurate assembly of mRNA transcript isoform is crucial for understanding gene expression and regulation. Although about $10–75\%$ of protein-coding genes are estimated to contain at least one uORF in plants [39,93], these figures may be either overestimated or underestimated due to various $5^{\prime}$ -leader isoforms from alternative splicing and transcription start sites [40,94]. Current mRNA assembly and annotation lag behind advancements in DNA sequencing. Tools like RNAirport help to assemble and identify the most abundant and frequent splicing isoforms  

# Outstanding questions  

How can we achieve near-complete assembly of all transcript isoforms, including those resulting from alternative splicing, transcription start sites and polyadenylation sites, across various tissues, developmental stages, and environmental conditions? Additionally, what criteria should be used to define a representative isoform for simplifying the annotation of different RREs, such as uORFs?  

How can we identify suppressive, translatable, or responsive uORFcontaining genes genome-wide, understand their molecular mechanisms, investigate their specificity to various stimuli, elucidate their physiological roles, and apply findings to genetic engineering?  

How can we identify additional promoters that effectively control the spatial and temporal expression of transgenes when combined with uORF-mediated translational control?  

How can the latest CRISPR toolkits be utilized to engineer uORFs in crops effectively? What are the best practices for applying these advanced CRISPR technologies to precisely modify uORFs and optimize their impact on gene regulation and crop traits?  

Are there general guidelines for engineering various uORFs for transgene and gene editing applications? Can uORF engineering be applied to other agronomic traits beyond those currently explored? How can different uORFs be engineered in metabolic genes to achieve optimal yields or enhance the production of medicinal products?  

and transcription start sites, using these to create representative gene models. However, the complete mRNA assembly for most crop plants, especially polyploids, has not been achieved. This makes it challenging to identify uORF-containing genes accurately. Precise mRNA assembly, in turn, will improve our understanding of uORF-mediated translational control mechanisms, which are currently limited to a few known genes. Thus, enhancing mRNA assembly and annotation of the associated RREs is key to advancing research in engineering uORFs following the current strategies.  

In addition to using uORFs for the transgenic expression of plant defense genes, many abiotic stress-related genes also require the $\mathrm{C}+$ transgene breeding strategy [84]. However, current engineering efforts have not identified suitable uORFs for these applications. The discovery of uORFsTBF1 may pave the way for identifying additional uORFs that can enhance various agronomic traits. Given that genetic variations in uORFs can cause severe human diseases, gene editing of DNA or RNA using the strategies discussed here in plants may also hold promise for precision medicine, in addition to its applications in precision agriculture [95–99] (see Outstanding questions).  

# Acknowledgments  

This study was supported by grants from the National Natural Science Foundation of China (32470282), the Biological Breeding-National Science and Technology Major Project (2023ZD04073), the Major Project of Hubei Hongshan Laboratory (2022hszd016), the Key Research and Development Program of Hubei Province (2022BFE003), and the Fundamental Research Funds for the Central Universities to G.X. We apologize to colleagues whose excellent work was not cited in this review due to space limits.  

# Declaration of interests  

The authors declare no competing interests.  

# References  

1. Bailey-Serres, J. et al. (2019) Genetic strategies for improving crop yields. Nature 575, 109–118   
2. Hickey, L.T. et al. (2019) Breeding crops to feed 10 billion. Nat. Biotechnol. 37, 744–754   
3. Zhang, Q. (2021) Purple tomatoes, black rice and food security. Nat. Rev. Genet. 22, 414   
4. Kumar, K. et al. (2020) Genetically modified crops: current status and future prospects. Planta 251, 91   
5. Liu, G. et al. (2022) The CRISPR-Cas toolbox and gene editing technologies. Mol. Cell 82, 333–347   
6. Tian, J. et al. (2020) Towards engineering broad-spectrum disease-resistant crops. Trends Plant Sci. 25, 424–427   
7. Xu, Z. et al. (2019) Engineering broad-spectrum bacterial blight resistance by simultaneously disrupting variable TALE-binding elements of multiple susceptibility genes in rice. Mol. Plant 12, 1434–1446   
8. Oliva, R. et al. (2019) Broad-spectrum resistance to bacterial blight in rice using genome editing. Nat. Biotechnol. 37, 1344–1350   
9. Eom, J.-S. et al. (2019) Diagnostic kit for rice blight resistance. Nat. Biotechnol. 37, 1372–1379   
10. Rodriguez-Leal, D. et al. (2017) Engineering quantitative trait variation for crop improvement by genome editing. Cell 171, 470–480.e8   
11. Lloyd, J.P.B. et al. (2022) Synthetic memory circuits for stable cell reprogramming in plants. Nat. Biotechnol. 40, 1862–1872   
12. Amack, S.C. and Antunes, M.S. (2020) CaMV35S promoter – a plant biology and biotechnology workhorse in the era of synthetic biology. Curr. Plant Biol. 24, 100179   
13. Ochoa-Fernandez, R. et al. (2020) Optogenetic control of gene expression in plants in the presence of ambient white light. Nat. Methods 17, 717–725   
14. Zhou, Y. et al. (2021) Optogenetic control of plant growth by a microbial rhodopsin. Nat. Plants 7, 144–151   
15. Yasmeen, E. et al. (2023) Designing artificial synthetic promoters for accurate, smart, and versatile gene expression in plants. Plant Commun. 4, 100558   
16. Fernandez-Moreno, J.-P. et al. (2024) A rapid and scalable approach to build synthetic repetitive hormone-responsive promoters. Plant Biotechnol. J. 22, 1942–1956   
17. Liu, $\times.$ et al. (2021) Fine-tuning flowering time via genome editing of upstream open reading frames of heading date 2 in rice. Rice (N Y) 14, 59   
18. Zhang, H. et al. (2018) Genome editing of upstream open reading frames enables translational control in plants. Nat. Biotechnol. 36, 894–898   
19. Li, T. et al. (2018) Domestication of wild tomato is accelerated by genome editing. Nat. Biotechnol. 36, 1160–1163   
20. Xu, G. et al. (2017) uORF-mediated translation allows engineered plant disease resistance without fitness costs. Nature 545, 491–494   
21. Tian, J. et al. (2024) Engineering disease-resistant plants with alternative translation efficiency by switching uORF types through CRISPR. Sci. China Life Sci. 67, 1715–1726   
22. Ai, G. et al. (2022) Making use of plant uORFs to control transgene translation in response to pathogen attack. Biodes. Res. 2022, 9820540   
23. Kim, J.H. et al. (2022) Increasing the resilience of plant immunity to a warming climate. Nature 607, 339–344   
24. Sagor, G.H.M. et al. (2016) A novel strategy to produce sweeter tomato fruits with high sugar contents by fruit-specific expression of a single bZIP transcription factor gene. Plant Biotechnol. J. 14, 1116–1126   
25. Wang, H. et al. (2024) Genome editing of 3′ UTR-embedded inhibitory region enables generation of gene knock-up alleles in plants. Plant Commun. 5, 100745   
26. Shen, R. et al. (2024) In-locus gene silencing in plants using genome editing. New Phytol. 243, 2501–2511   
27. Xue, C. et al. (2023) Tuning plant phenotypes by precise, graded downregulation of gene expression. Nat. Biotechnol. 41, 1758–1764   
28. Xiong, X. et al. (2022) A cytosine base editor toolkit with varying activity windows and target scopes for versatile gene manipulation in plants. Nucleic Acids Res. 50, 3565–3580   
29. Jia, C. et al. (2024) Editing and genome-wide analysis upstream open reading frames contributes to enhancing salt tolerance in tomato. Plant Biotechnol. J., Published online August 20, 2024. https://doi.org/10.1111/pbi.14450   
30. Wu, H.-Y.L. et al. (2023) What, where, and how: regulation of translation and the translational landscape in plants. Plant $C e//$ 36, 1540–1564   
31. Farache, D. et al. (2022) Moonlighting translation factors: multifunctionality drives diverse gene regulation. Trends Cell Biol. 32, 762–772   
32. Castellano, M.M. and Merchante, C. (2021) Peculiarities of the regulation of translation initiation in plants. Curr. Opin. Plant Biol. 63, 102073   
33. Srivastava, A.K. et al. (2018) UTR-dependent control of gene expression in plants. Trends Plant Sci. 23, 248–259   
34. Mazzoni-Putman, S.M. and Stepanova, A.N. (2018) A plant biologist’s toolbox to study translation. Front. Plant Sci. 9, 873   
35. Merchante, C. et al. (2017) Translation regulation in plants: an interesting past, an exciting present and a promising future. Plant J. 90, 628–653   
36. Browning, K.S. and Bailey-Serres, J. (2015) Mechanism of cytoplasmic mRNA translation. Arabidopsis Book 13, e0176   
37. Yuan, S. et al. (2023) Translation machinery: the basis of translational control. J. Genet. Genomics 51, 367–378   
38. Wu, H.-W. et al. (2022) Noise reduction by upstream open reading frames. Nat. Plants 8, 474–480   
39. Niu, R. et al. (2020) uORFlight: a vehicle toward uORF-mediated translational regulation mechanisms in eukaryotes. Database 2020, baaa007   
40. Zhu, S. et al. (2024) RNAirport: a deep neural network-based database characterizing representative gene models in plants. J. Genet. Genomics 51, 652–664   
41. Ingolia, N.T. et al. (2009) Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. Science 324, 218–223   
42. Xu, G. et al. (2017) Global translational reprogramming is a fundamental layer of immune regulation in plants. Nature 545, 487–490   
43. Liu, M.J. et al. (2013) Translational landscape of photomorphogenic Arabidopsis. Plant Cell 25, 3699–3710   
44. Juntawong, P. et al. (2014) Translational dynamics revealed by genome-wide profiling of ribosome footprints in Arabidopsis. Proc. Natl. Acad. Sci. U. S. A. 111, E203–E212   
45. Xiang, Y. et al. (2023) Pervasive downstream RNA hairpins dynamically dictate start-codon selection. Nature 621, 423–430   
46. Aitken, C.E. and Lorsch, J.R. (2012) A mechanistic overview of translation initiation in eukaryotes. Nat. Struct. Mol. Biol. 19, 568–576   
47. Hinnebusch, A.G. (2014) The scanning mechanism of eukaryotic translation initiation. Annu. Rev. Biochem. 83, 779–812   
48. Loughran, G. et al. (2012) Stringency of start codon selection modulates autoregulation of translation initiation factor eIF5. Nucleic Acids Res. 40, 2898–2906   
49. Wagner, S. et al. (2020) Selective translation complex profiling reveals staged initiation and co-translational assembly of initiation factor complexes. Mol. Cell 79, 546–560.e7   
50. Bohlen, J. et al. (2020) Selective 40S footprinting reveals cap-tethered ribosome scanning in human cells. Mol. Cell 79, 561–574.e5   
51. Mohammad, M.P. et al. (2017) In vivo evidence that eIF3 stays bound to ribosomes elongating and terminating on short upstream ORFs to promote reinitiation. Nucleic Acids Res. 45, 2658–2674   
52. Mohammad, M.P. et al. (2021) eIF4G is retained on ribosomes elongating and terminating on short upstream ORFs to control reinitiation in yeast. Nucleic Acids Res. 49, 8743–8756   
53. Hronová, V. et al. (2017) Does eIF3 promote reinitiation after translation of short upstream ORFs also in mammalian cells? RNA Biol. 14, 1660–1667   
54. Roy, B. et al. (2010) The h subunit of eIF3 promotes reinitiation competence during translation of mRNAs harboring upstream open reading frames. RNA 16, 748–761   
55. Schepetilnikov, M. et al. (2013) TOR and S6K1 promote translation reinitiation of uORF-containing mRNAs via phosphorylation of eIF3h. EMBO J. 32, 1087–1102   
56. Schepetilnikov, M. et al. (2011) Viral factor TAV recruits TOR/ S6K1 signalling to activate reinitiation after long ORF translation. EMBO J. 30, 1343–1356   
57. Pöyry, T.A.A. et al. (2004) What determines whether mammalian ribosomes resume scanning after translation of a short upstream open reading frame? Genes Dev. 18, 62–75   
58. Skabkin, M.A. et al. (2013) Reinitiation and other unconventional posttermination events during eukaryotic translation. Mol. $C e//$ 51, 249–264   
59. Watanabe, R. et al. (2010) The eukaryotic initiation factor (eIF) 4G HEAT domain promotes translation re-initiation in yeast both dependent on and independent of eIF4A mRNA helicase. J. Biol. Chem. 285, 21922–21933   
60. Schleich, S. et al. (2014) DENR-MCT-1 promotes translation reinitiation downstream of uORFs to control tissue growth. Nature 512, 208–212   
61. Skabkin, M.A. et al. (2010) Activities of Ligatin and MCT-1/ DENR in eukaryotic translation initiation and ribosomal recycling. Genes Dev. 24, 1787–1801   
62. Zhou, F. et al. (2010) Translation reinitiation and development are compromised in similar ways by mutations in translation initiation factor eIF3h and the ribosomal protein RPL24. BMC Plant Biol. 10, 193   
63. Kim, T.H. et al. (2004) Translational regulation via 5' mRNA leader sequences revealed by mutational analysis of the Arabidopsis translation initiation factor subunit eIF3h. Plant Cell 16, 3341–3356   
64. Hinnebusch, A.G. et al. (2016) Translational control by 5'- untranslated regions of eukaryotic mRNAs. Science 352, 1413–1416   
65. Hayashi, N. et al. (2017) Identification of Arabidopsis thaliana upstream open reading frames encoding peptide sequences that cause ribosomal arrest. Nucleic Acids Res. 45, 8844–8858   
66. Hanfrey, C. et al. (2005) A dual upstream open reading framebased autoregulatory circuit controlling polyamine-responsive translation. J. Biol. Chem. 280, 39229–39237   
67. Yamashita, Y. et al. (2017) Sucrose sensing through nascent peptide-meditated ribosome stalling at the stop codon of Arabidopsis bZIP11 uORF2. FEBS Lett. 591, 1266–1277   
68. Pajerowska-Mukhtar, K.M. et al. (2012) The HSF-like transcription factor TBF1 is a major molecular switch for plant growth-todefense transition. Curr. Biol. 22, 103–112   
69. Chen, J. et al. (2020) Pervasive functional translation of noncanonical human open reading frames. Nature 367, 1140–1146   
70. Wright, B.W. et al. (2021) The dark proteome: translation from noncanonical open reading frames. Trends Cell Biol. 32, 243–258   
71. Starck, S.R. et al. (2016) Translation from the 5' untranslated region shapes the integrated stress response. Science 351, aad3867   
72. Tavormina, P. et al. (2015) The plant peptidome: an expanding repertoire of structural features and biological functions. Plant Cell 27, 2095–2118   
73. Chen, T. et al. (2023) Global translational induction during NLRmediated immunity in plants is dynamically regulated by CDC123, an ATP-sensitive protein. Cell Host Microbe 31, 334–342.e5   
74. Zhou, Y. et al. (2023) Plant HEM1 specifies a condensation domain to control immune gene translation. Nat. Plants 9, 289–301   
75. Tang, Z. et al. (2024) Alleviating protein-condensationassociated damage at the endoplasmic reticulum enhances plant disease tolerance. Cell Host Microbe 32, 1552–1565.e8   
76. Chang, H.-H. et al. (2024) The phosphorylation of carboxylterminal eIF2α by SPA kinases contributes to enhanced translation efficiency during photomorphogenesis. Nat. Commun. 15, 3467   
77. Tanaka, M. et al. (2024) Boric acid intercepts 80S ribosome migration from AUG-stop by stabilizing eRF1. Nat. Chem. Biol. 20, 605–614  
78. Hedaya, O.M. et al. (2023) Secondary structures that regulate mRNA translation provide insights for ASO-mediated modulation of cardiac hypertrophy. Nat. Commun. 14, 6166   
79. Smirnova, A.M. et al. (2024) Stem-loop-induced ribosome queuing in the uORF2/ATF4 overlap fine-tunes stress-induced human ATF4 translational control. Cell Rep. 43, 113976   
80. Wang, J. et al. (2022) PABP/purine-rich motif as an initiation module for cap-independent translation in pattern-triggered immunity. Cell 185, 3186–3200.e17   
81. Zhou, J. et al. (2018) $N^{6}$ -methyladenosine guides mRNA alternative translation during integrated stress response. Mol. Cell 69, 636–647.e7   
82. Stearns, F.W. (2010) One hundred years of pleiotropy: a retrospective. Genetics 186, 767–773   
83. He, Z. et al. (2022) Growth-defense trade-offs in plants. Curr. Biol. 32, R634–R639   
84. Zhang, H. et al. (2020) Thriving under stress: how plants balance growth and the stress response. Dev. Cell 55, 529–543   
85. Guo, Z. et al. (2022) A natural uORF variant confers phosphorus acquisition diversity in soybean. Nat. Commun. 13, 3796   
86. Chen, G. et al. (2023) A novel active transposon creates allelic variation through altered translation rate to influence protein abundance. Nucleic Acids Res. 51, 595–609   
87. Liang, M. et al. (2022) Lost in translation: molecular basis of reduced flower coloration in a self-pollinated monkeyflower (Mimulus) species. Sci. Adv. 8, eabo1113   
88. Chen, Y. et al. (2023) Two open reading frames of Rht-B1b acting as brake and throttle contributed to wheat Green Revolution. Plant Physiol. 194, 1290–1293   
89. Li, W. et al. (2012) Molecular diversity of restriction enzyme sites, indels and upstream open reading frames (uORFs) of 5′ untranslated regions (UTRs) of Waxy genes in Triticum L. and Aegilops L. species. Genet. Resour. Crop. Evol. 59, 1625–1647   
90. Gage, J.L. et al. (2022) Variation in upstream open reading frames contributes to allelic diversity in maize protein abundance. Proc. Natl. Acad. Sci. U. S. A. 119, e2112516119   
91. Wang, J. et al. (2023) Natural uORF variation in plants. Trends Plant Sci. 29, 290–302   
92. Zhang, X. et al. (2019) A single nucleotide substitution at 5′-UTR of GSN1 represses its translation and leads to an increase of grain length in rice. J. Genet. Genomics 46, 105–108   
93. Zhang, H. et al. (2021) Determinants of genome-wide distribution and evolution of uORFs in eukaryotes. Nat. Commun. 12, 1076   
94. Parker, M.T. et al. (2020) Nanopore direct RNA sequencing maps the complexity of Arabidopsis mRNA processing and m6A modification. eLife 9, e49658   
95. Somers, J. et al. (2013) A perspective on mammalian upstream open reading frame function. Int. J. Biochem. Cell Biol. 45, 1690–1700   
96. Barbosa, C. et al. (2013) Gene expression regulation by upstream open reading frames and human disease. PLoS Genet. 9, e1003529   
97. Calvo, S.E. et al. (2009) Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. U. S. A. 106, 7507–7512   
98. Whiffin, N. et al. (2020) Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals. Nat. Commun. 11, 2523   
99. Liang, X.H. et al. (2016) Translation efficiency of mRNAs is increased by antisense oligonucleotides targeting upstream open reading frames. Nat. Biotechnol. 34, 875–880   
100. Liu, Q. et al. (2022) Ribo-uORF: a comprehensive data resource of upstream open reading frames (uORFs) based on ribosome profiling. Nucleic Acids Res. 51, D248–D261   
101. Manske, F. et al. (2022) The new uORFdb: integrating literature, sequence, and variation data in a central hub for uORF research. Nucleic Acids Res. 51, D328–D336   
102. Johnstone, T.G. et al. (2016) Upstream ORFs are prevalent translational repressors in vertebrates. EMBO J. 35, 706–723   
103. Wu, H.-Y.L. et al. (2023) Improved super-resolution ribosome profiling reveals prevalent translation of upstream ORFs and small ORFs in Arabidopsis. Plant Cell 36, 510–539  