# Trends in Genetics  

Special issue: Microproteins Review  

# Cis to trans: small ORF functions emerging through evolution  

Casimiro Baena-Angulo $\circledcirc$ 1 ,2 , Ana Isabel Platero $\oplus$ 1,2 , and Juan Pablo Couso $\circledcirc$ 1, \* Hundreds of thousands of small open reading frames (smORFs) of less than 100 codons exist in every genome, especially in long noncoding RNAs (lncRNAs) and in the 5′ leaders of mRNAs. smORFs are often discarded as nonfunctional, but ribosomal profiling (RiboSeq) reveals that thousands are translated, while characterised smORF functions have risen from anecdotal to identifiable trends: smORFs can either have a cis-noncoding regulatory function (involving low translation of nonfunctional  peptides)  or  full  coding  function  mediated  by  robustly  translated peptides, often having cellular and physiological roles as membrane-associated regulators  of  canonical  proteins.  The  evolutionary  context  reveals  that  many smORFs represent new genes emerging de novo from noncoding sequences. We suggest a mechanism for this process, where cis-noncoding smORF functions provide niches for the subsequent evolution of full peptide functions.  

# The world of short coding sequences  

smORFs (see Glossary) are DNA and RNA stretches that can be translated into peptides or short proteins of less than 100 amino acids (aa) [1]. These sequences commence with a start codon (usually ATG/AUG; but see Cao et al. [2] for  a review on non-AUG ORFs) and  end  in a stop codon, and thus these directly synthesised short peptides differ from those arising from the proteolytic cleavage of longer precursor proteins (i.e., insulin). In eukaryotes, smORFs are found in genomic sequences in numbers ranging from hundreds of thousands in fruit flies (Drosophila melanogaster) to millions in humans [3–6], with thousands mapping to putative noncoding RNAs [7–10]. smORFs had been deemed noncoding, and thus remained nonannotated, on the basis of their high numbers, small size, and limited experimental study of their functionality, but there is a growing consensus that hundreds if not thousands of smORFs are translated [9,11–13] and that smORF peptides can have essential functions and be conserved across metazoans or even eukaryotes [14–16]. Thus, transcribed smORFs sit at the interface between coding and noncoding RNA, challenging our understanding of our genomes’ coding potential. However, the full repertoire of smORF functional peptides is not known, nor are the genomic and evolutionary roles of smORF sequences. It is as if we have a genome within our genome, a new genome about which we know very little.  

smORFs in genomes have an average size of around 25 codons, having distributions of size and number fitting with an origin at random [6]. According to genomic features such as transcript structure and encoded aa sequence, smORFs belong to distinct classes that correlate with their attributed function (Figure 1), from nontranscribed and likely nonfunctional DNA sequences (intergenic ORFs) to sequences producing peptides that can regulate canonical proteins [6,17]. Some $10\%$ of smORFs (tens of thousands) appear in RNA transcripts: either upstream and downstream ORFs (uORFs and dORFs, respectively) within the $5^{\prime}$ and $3^{\prime}$ untranslated region of mRNAs or smORFs  

# Highlights  

Small,  or  short,  open  reading  frames (smORFs) of less than 100 codons exist in the hundreds of thousands in every transcriptome  but  until  recently  have been discarded as nonfunctional.  

Ribosomal profiling reveals translation of at least hundreds of smORFs, often at low levels undetected by proteomics.  

This translation can produce no biologically detectable peptide but still have a cis effect on the synthesis or stability of the encoding RNA.  

Robustly translated smORF peptides often  regulate  canonical  proteins associated with cellular membranes and organelles.  

These regulatory inputs can modulate deeply conserved cellular and physiological functions.  

The functional and translational profiles of smORFs suggest an evolutionary process producing new peptides from noncoding sequences.  

present in putatively noncoding RNAs [long noncoding ORFs (lncORFs)], often in complex polycistronic arrangements. Around $1\,\%$ (\~1000) are translated into peptides and short proteins of around 80 aa (short coding sequences, or sCDSs) [9,11,17], in most cases as the sole coding ORF in a shorter but otherwise canonical mRNA. Some of these sCDSs arise from short transcript isoforms of canonical proteins [18,19] (Figure 1). In this review, we aim to integrate recent and existing data to reveal a double emergence of smORF functions: their artificial emergence into our knowledge and their natural emergence through the evolutionary process.  

smORF function has been difficult to ascertain: The encoded peptides’ shortness often precludes experimental detection, be it by proteomics [11] or by random genetic screens. This shortness also defeats standard computational methods for homology detection, our main weapon to identify functionality [4,20]. There have been pioneering efforts to functionally characterise smORFs. The ‘smORF problem’ was first noted in yeast, in parallel with its genome annotation introducing an arbitrary cutoff point of 100 aa (in bacteria, the cutoffs were around 50 aa, and thus smORFs have long been part of mainstream research in prokaryotes). A pioneering screen [3] identified function for many sCDSs, and the in-depth study of the GCN4 gene has long provided a model for uORFs functioning as translational cis regulators, acting on a canonical protein located downstream in the same mRNA [21–24] (Figure 2A). However, lncRNAs were not noted in yeast until recently [25], and the discovery of thousands of potential lncRNAs in metazoans paradoxically reinforced the unconscious bias against the coding potential of smORFs. In Drosophila, our group characterised the 11-aa peptide encoded by tarsal-less (tal) [14], a polycistronic gene annotated as an lncRNA that was, in fact, coding, essential for development, and conserved across arthropods [14]. This was followed by the characterisation of the sarcolamban smORF family [15], which also includes four other vertebrate ex-lncRNAs [26–29], and the shortest smORF characterised in humans (sarcolipin, 30 aa) [30]. These peptides form single transmembrane domains that regulate sarcoplasmic/endoplasmic reticulum ${\mathrm{C}}\mathfrak{a}^{2+}$ -ATPase (SERCA) to control ${\mathrm{C}}\mathfrak{a}^{2+}$ -mediated heart rhythmicity from Drosophila to humans, revealing an astonishingly deep functional and structural conservation of smORF peptides (\~500 million years ago). Together with other smORFs, described not only in Drosophila and humans [8,16,18,22] but also in other systems [31–33] (see later), these cases established a proof of principle for function and conservation of even very short lncORFs, but one could not extrapolate from them the functionality of all other smORF sequences. However, it was noted that characterised smORF peptides showed a tendency to act as regulators of canonical proteins, probably due to their small size not being suitable for structural roles. Noted regulatory hotspots were protein synthesis, mitochondria [34], and membrane-associated functions [6,35].  

Recently, an exciting overlap has occurred with another fast-developing field: de novo gene evolution. For a long time, it has been noted that up to $2\%$ of coding genes in every species seem to have no homologues in the phylogeny [36–38]. It is as if these genes have appeared suddenly, de novo, and not from mutation or duplication of existing genes. If genomes can sprout de novo genes, this would offer an exciting path for evolution. However, there is no consensus on the mechanisms involved in de novo gene creation. It has been proposed that de novo genes arise from protogenes, sequences with features somehow intermediate between inert DNA and full coding canonical proteins [39], but these intermediate features were not defined, nor were the mechanisms involved in the transition to a de novo gene. More recently, it has been proposed that lncRNAs include young coding sequences [6,10] Going one step further, we envision that, actually, smORFs include both noncoding protogenes and young coding genes, each with distinctive genetic and molecular features [6], and that studying changes in these features across phylogeny will reveal the mechanisms for novel coding gene evolution. Here we propose a ‘cis-to-trans path’ to coding function that can explain the transition from a noncoding smORF to a coding one (see below).  

# Glossary  

# Canonical: genes that  

produce mRNAs and encode proteins longer than 100 aa.  

De novo genes: a gene  sequence  that was not transcribed in its immediate evolutionary ancestors but has acquired transcription in its current species.  

dORF: stands for ‘downstream ORF.’; smORFs located in the $3^{\prime}$ untranslated region of an mRNA.  

Framing: trinucleotide periodicity shown by the alignment of RiboSeq reads over translated sequences. This periodicity corresponds with the codons in the translated ORF and reflects the stepwise movement of the translated RNA through the ribosome.  

Intergenic ORFs: these ORFs appear throughout the genomic sequence outside of any annotated coding or noncoding genes. They are generally smORFs and seem to have lengths that correspond statistically with being generated at random.  

lncORFs: smORFs located within an lncRNA gene.  

Microprotein: protein whose length ranges from 50 to 100 aa.  

ORF: open reading frame; a stretch of nucleotides in a DNA or RNA sequence that in principle could be translated into a protein. An ORF begins with a start codon (usually AUG) and ends with a stop codon (such as UAA, UAG, or UGA). It is a crucial element in gene prediction and annotation processes. Peptide: protein whose length is less than 50 aa. RiboSeq: high-throughput sequencing of ribosome-protected RNA fragments (footprints) to identify translated RNA sequences.  

Ribosome-protected fragments (RPFs): RNA fragments that are bound and thus protected by ribosomes during an RNase digestion. These fragments are then subjected to high-throughput sequencing to measure their expression. Sometimes referred as ‘footprints.’  

SEPs: ‘smORF encoded peptides’; they are peptides and microproteins encoded by any class of smORF.  

sCDSs: ‘short coding sequences’; they are  ORFs that  encode  proteins less than 100 aa in length. Although more conserved than other smORFs, they are usually less conserved in phylogeny than canonical proteins, and their function seems to involve regulating larger proteins.  

# RiboSeq: a silver bullet for smORF detection  

Given that ORFs appear at random in nucleotide sequences, the mere presence of an ORF does not guarantee its functionality. To clarify how many of these tens of thousands of transcribed smORFs are functional, we first have to identify the translated ones. smORF translation used to be proved by single-gene experimental methods such as genetic analyses and reporter assays (Box  1).  These  methods  still  constitute  the  final  test  for  smORF  translation,  but  currently smORF translation is detected by high-throughput genomic methods. Some smORF encoded peptides (SEPs) have been identified by proteomic approaches [40]. Most of these SEPs have been detected in mammals [41], mainly in humans [41–46] and some in Drosophila [11,47]. However, analysis of these results and comparison with RiboSeq (see later and $\mathsf{B O X}\;^{\mathrm{~1~}}$ 1) shows that only smORFs with high translation levels are detected by proteomics [11,48].  

RiboSeq provides a tool to detect translated sequences at a genomic scale [49,50]. Applying RiboSeq to detect smORFs revealed the translation of hundreds of smORFs in Drosophila [9] and vertebrates [48,49,51]. Recent advances include single-cell RiboSeq, which has been used successfully in mammalian cell cultures [52,53], and RIBOmap, able to perform in situ amplification sequencing in cells and tissue samples. These techniques can detect translation at the cellular level in a gene-specific manner [54], overcoming a weakness in RiboSeq and proteomics to date, since their need for micrograms of sample forces their application to either cell lines, whole organs, or entire embryos. Despite the great potential of the upgraded techniques, their major flaws concern their single-gene focus (RIBOmap) and their low sensitivity because of the small amount of genomic material obtained for sequencing (single-cell RiboSeq) [53]. These limitations hinder their application to genome-wide detection of translated smORFs, which are often obscured by the more highly translated canonical genes.  

RiboSeq has shown that smORF translation occurs in two modes: sCDSs get translated often and with high translation efficiency (TE) similar to canonical proteins, whereas lncORFs and uORFs are translated rarely and with much lower TE. Patraquim et al. [55] revealed that around $30\%$ of fly lncRNAs contain smORFs engaged by ribosomes, leading to reproducible translation of at least 100 peptides of around 22 aa. This translation appears regulated in time and space and, notably, depends on specific lncRNA features such as cistronicity, Kozak sequences, and conservation. In both yeast and vertebrate cell lines, similar smORF-tailored efforts have revealed translation of hundreds of lncORFs and uORFs with similar characteristics [48,49,51] (see later). But is such low-level translation functional? Interestingly, reproducible lncORF translation correlates with conservation of aa sequence across species, a standard indicator of protein and peptide function [55], whereas translation of more recently arisen lncORFs (species-specific), or nonproductive ribosomal association, does not [55,56]. While many lncORFs may be translated at lower or random levels, some lncORFs encoding beneficial peptides are evolutionarily selected for robust translation. Hence, the evolutionary context would appear essential to understanding smORF function, which is not revealed by translatomic techniques. However, a number of screens and single-gene studies have also been used to elucidate smORF function, and we discuss them first.  

# Characterising smORF function at the genomic level  

Recent functional studies, supported by high-throughput sequencing and functional genomics, have attempted a functional characterisation of annotated and nonannotated smORFs in a genome-wide manner.  

Vakirlis et al. [57] carried out a systematic functional screen in the Saccharomyces genome to identify the functionality of recently emerged genes identified as having homologies restricted to the genera Saccharomyces. Previous works by these authors and others in de novo gene and  

Short isoforms: ORFs that come from spliced transcripts of a canonical gene. Thus, a short isoform always has a longer, canonical isoform expressed from the same gene.  

smORF: small open reading frame; ORFs with 100 codons or less, potentially able to be translated into proteins shorter than 100 aa before post-translational modifications (such as proteolytic cleavage).  

Translation efficiency (TE): RiboSeq metric that divides the RPKM of footprints (RPKM-FP) by the RPKM of RNA sequencing from the same sample. uORF: ‘upstream ORFs’; smORFs found in the 5′ leader of an mRNA.  

Young coding genes:  a gene that  has acquired translation recently in evolutionary terms, often within its genera.  

![](images/c3cebe8ac271190d9d5114c26f8b0edbcc694d23913722b49e0540ce23abf94f.jpg)  

Figure 1. Classification of small open reading frames (smORFs). smORFs (red blocks) can be classified in six groups according with their transcript type and structure. This structure seems to influence smORF function, whether coding or noncoding. Canonical ORFs (black squares) are included for clarity. Intergenic ORFs are the most numerous, appearing by chance outside any annotated gene transcript and without reported functions [6]. Long noncoding ORFs (lncORFs) are located in lncRNA exons often in polycistronic arrangements, and although hundreds have been reported to be translated into peptides, only a fraction of these show conservation in other species [8,14,26–28,71–73,77,88–91]. Upstream ORFs (uORFs) and downstream ORFs (dORFs) have been shown to act as cis-regulators of translation by noncoding mechanisms (i.e., ribosome stalling at uORFs) or coding ones (involving the uORF-encoded peptides). They are relatively abundant, because most mRNAs contain these sequences in their $5^{\prime}$ and $3^{\prime}$ untranslated regions, and they are poorly conserved because their amino acid (aa) sequence may not be essential for their regulatory function [21–23,92,93]. Short coding sequences (sCDSs) are canonical-like ORFs less than 100 codons long, conserved at the class level on average [6], which produce peptides and microproteins whose primary function is the regulation of larger proteins. Most characterised smORFs belong to this class [9,11,17]. Short isoforms are the fewest in number, appearing as spliced isoforms of canonical genes, producing truncations of larger proteins. Such truncations are a source of negative regulators, working as ‘dominant-negative’ versions of the canonical protein [18,19]. ‘AA median size’ and ‘numbers in metazoans’ were obtained from the mean among the species Drosophila melanogaster, Danio rerio, Mus musculus, and Homo sapiens in Couso and Patraquim [6], except for dORFs that were computed here (see the supplemental information online). Numbers in metazoans are rounded up and presented as a range from Drosophila melanogaster (left values) to Homo sapiens (right values), so that Drosophila and human values were recalculated here according to FlyBase (genome assembly: dmel_r6.51_FB2023_02) and NCBI (genome assembly: GRCh38.p14), respectively. smORFs were found using EMBOSS getORF and further curated by redundant ORFs according to their coordinates. Abbreviation: AA, amino acid.  

smORF studies have established a high overlap between de novo coding genes and smORFs. In other words, most de novo coding genes are smORFs [39,58,59]. Vakirlis et al.analysed Saccharomyces young ORFs by conservation of aa sequence and then by genetic analysis of those with variable coding sequence (239 genes), using both induced mutations and natural variations. Most $(92\%)$ of emerging (short) ORFs were found to have no function, understood as lack of fitness effects (i.e., effects on yeast growth and proliferation). This agrees with the expectations in de novo gene studies and the framework of protogenes, which posits that ORFs emerge in noncoding DNA as quasigenic sequences lacking some of the attributes of a coding gene (RNA or protein expression, coding protein function). However, Vakirlis et al.reasoned that emerging genes may not have achieved high, canonical expression levels and hence that overexpressing young coding genes could improve fitness. Interestingly, 14 out of 285 ORFs provide such benefits, increasing Figure 2. Cis and trans small open reading frame (smORF) functions, showing the possible emergence of smORF coding function through evolution. (A) RNA sequences randomly contain smORFs, which can attract ribosomes as part of RNA processing, degradation, or translation mechanisms. Hence, random smORFs exert an influence on their encoding RNAs, either as long noncoding ORFs (lncORFs) regulating the processing and degradation of their miRNA-containing lncRNAs (top) or as upstream ORFs (uORFs) and downstream ORFs (dORFs) by regulating positively [11,93] or negatively [65] the translation of canonical, longer ORFs in their mRNA (bottom). This influence can be modulated by evolution by enhancing or reducing the smORF translation. (B) Beyond a certain threshold of translation, the smORFexpressed peptide (SEP) would be biologically detectable. This peptide could also exert a kind of ‘cis’ function, either on its encoding lncRNA (top) or on the canonical protein (red) cis-encoded in the same mRNA (bottom). By acting on close cis-molecules, the SEP does not need to be stable, which might explain the lack of detection of smORF peptides by proteomics; in fact, SEPs may remain unstable to minimise unwanted trans effects. $(\mathrm{C})$ If SEP expression and stability are increased, the newly abundant SEP can diffuse across the cell and regulate other trans-protein partners (top), either from proteins paralogous to the earlier cis-partners or simply proteins containing a similar domain (purple). In time, it may be advantageous to dissociate the smORF from other cis-RNA functions, and thus a truncated or modified version of the mRNA may appear with the peptide-producing smORF as the sole ORF that is a new coding gene or short coding sequence (sCDS). Thus, a ‘cis-to-trans path to function’ might provide a niche for smORF translation, and this translation could then allow the evolution of a beneficial and stable peptide.  

![](images/33cd88b0103a059e5577182af5e1a17e7e8712ea459c8eed794ce9f2f10e4dcf.jpg)  

to 28 $10\%)$ in challenging environments (whereas only $3\%$ of canonical gene overexpression is beneficial). They computationally predicted the 28 protein structures, focusing on the capacity to be membrane proteins. They suggested a ‘transmembrane domains-first’ (TM-first) evolutionary model in which intergenic sequences with high thymine are potential starting points for translation,  hence  giving  rise  to  new  functional  and  adaptive  membrane  proteins,  including hydrophobic and nonpolar aa with Thy-rich codons [60,61]. This model is strikingly validated by a detailed analysis of the ORF YBR196C-A, including phylostratigraphy and ancestral reconstruction. This model is applicable to other organisms with AT-rich noncoding sequences, where an association between translated smORF peptides and membrane-related functions had been noted [6,9,35].  

In Drosophila, a consortium of two large laboratories attempted a large-scale functional characterisation of smORFs [62]. They selected 242 sCDSs annotated or observed to be conserved from flies to humans. They attempted somatic CRISPR mutagenesis and CRISPR-mediated overexpression for most of these genes and generation of whole knockout $(\mathsf{K O})$ animals in a fraction of them. However, the results are surprising in that only two smORFs show an absolute requirement  for  viability,  and  only  14  show  a  mutant  phenotype  (morphological,  viability, fecundity) in whole KOs. The two essential smORFs act in the mitochondrial electron transport, reinforcing the importance of the mitochondria as a niche for smORF peptide function, including those arising in the mitochondrial genome [34,63].  

This low rate of functional requirement for sCDSs is in contrast to the high rate observed in yeast [3] and is surprising, given the deep conservation of these smORFs. This could be explained by technical reasons (the somatic CRISPR produces mosaic animals whose nonmutant cells can overcome the loss of function in the mutant ones) or by the possibility that these deeply conserved smORFs might have cellular or physiological functions undetectable by the observations carried out. In other studies, fly sCDSs such as sarcolamban [15] and Hemotin [16] do affect fly behaviour and longevity, but their main requirement is for cellular physiology, and a mutant phenotype was only revealed by using specific cellular sensors. In both cases, transcript tissue expression and peptide subcellular localisation were guides to look for such cellular phenotypes, but these types of studies cannot be carried out at high throughput.  

In humans, Chen et al. [64] used RiboSeq, mass spectrometry (MS), and microscopy, together with CRISPR screening, to discover noncanonical sequences in several cell lines. RiboSeq analysis determined $22\%$ ORFs (3455) ‘distinct’ from the canonical sequences, approximately $23\%$ of which were new lncORFs, $67\%$ were uORFs, and only some were sCDSs. To mutate lncORFs, they used CRISPR, observing cellular phenotypes in seven, followed by subcellular localisation through tagging and coimmunoprecipitation assays to identify the proteins interacting with the lncORF peptides. They carried out the same tests for uORFs. On the basis of the literature on uORFs  acting  as  cis-regulators  of  canonical  protein  translation  [11,21–23,65] (Figure  2A, bottom), they studied this issue first and identified several uORFs not affecting the translation of  

# Box 1. A workflow for characterising translated smORFs functionality  

Bioinformatic approaches and transcriptomics might suggest that studying smORF translation and characterisation is straightforward, with coding sequence and functional domain predictors helping to demonstrate translation and establish functions. However, proving smORF translation requires an experimental approach combining predictors, transcriptomics, proteomics, and molecular biology corroborations.  

Most genome-wide analyses to determine translated smORFs start with RiboSeq [84], providing a snapshot of ribosome positions on RNA with single-nucleotide resolution. The three-nucleotide periodicity analysis of ribosome-protected fragments (RPFs) allows researchers to quickly identify which RNAs are involved in actual translation through their framing signal. RiboSeq is effective where proteomics-based screens often fail due to the limitations of proteomic techniques and the biological characteristics of peptides. Although mass spectrometry (MS) is a popular method to confirm translation, it has high stringency thresholds that challenge peptide detection. Short peptides typically do not produce identifiable tryptic fragments [85] and are often short-lived and expressed in specific biological contexts. From fruit fly to humans, only $4\%$ to $22\%$ of sCDSs are detected by proteomics, whereas around $50\%$ of microproteins are ORF predictions, and less than $10\%$ are able to be verified by homology (Figure I). Then, manual curation of MS data is required using RiboSeq evidence to find this ‘beautiful needle in the haystack’ [1] and still be capable of detecting an SEP at least twice, either as different MS spectra or in replicates from different biological contexts [17].  

Bioinformatic analysis supports transcriptomic or proteomic evidence by determining underlying biological functions based on sequence conservation or selective pressure among species. Tools such as PhyloCSF [86] score evolutionary signatures of coding potential across species to identify regions likely encoding functional proteins. Researchers also use the dN/dS ratio to analyse evolutionary pressure [55,87], providing insights into the selective forces acting on a protein-coding gene and indicating whether a gene is under positive selection. However, the short sequences of many smORFs are not able to produce a clear conservation signal.  

Despite these approaches, experimental corroboration is essential to confirm smORF translation and function. Techniques include small peptide tagging (hemagglutinin tag or Flag tag) [55,73,75], directed mutagenesis to generate KO mutants, altering the ATG start codon, or inducing frameshifts to prevent translation [71], alongside ectopic expression to understand the associated phenotype. Ideally, a well-characterised smORF should meet several of these experimental criteria. Altogether, characterising smORFs is often more challenging than canonical proteins because of their subtle genomic nature, often masked by factors such as low or time-/space-specific expression, shortness, and possible noncoding functions.  

![](images/a3e65a463dc2393685567061a6f2b673d01d635533ac44e09539fa32a8201c0f.jpg)  
Figure I. Percentage of annotated proteins in canonical and short coding sequence (sCDS) genes. Annotation status was classified with the UniProt ‘protein existence’ classification: evidence at protein level, evidence at transcript level, inferred from homology, or simply predicted by an open reading frame (ORF) searching programme (orange). Each box represents the average percentages from Homo sapiens (UPID: UP000005640), Mus musculus (UPID: UP000000589), Danio rerio (UPID: UP000000437), and Drosophila melanogaster (UPID: UP000000803) proteomes,  extracted  from  UniProt  database.  Canonical  and  sCDS  genes  were  sorted  by  their  peptide  length [canonical $\geq\uparrow00$ amino acid (aa); sCDS $<\!100$ aa] (see supplemental information online). Note that some $50\%$ of small open reading frames (smORFs) remain as unverified ORF predictions (versus $20\%$ of canonical proteins), with a lower percentage of homology verification, illustrating the difficulties in smORF annotation.  

their main ORFs. Of these, two (TBPL1 and ARL5A) produced peptides with wholly trans-acting function (i.e., fully independent of the main ORF). This joins examples proposed before [22,23] and reinforces a trend for trans-coding uORF function, in effect part of a dicistronic mRNA. Furthermore, they noticed that at least five uORF peptides did interact physically and functionally with the canonical protein encoded by their main ORF, acting as protein regulators (MIEF1, DDIT3, FBXO9, HMGA2, and HAUS6) [64]. Thus, in practice, these uORFs and their associated main ORFs behave as a dicistronic operon, a discovery with all sorts of potential ramifications, not least for smORF biology and evolution (see later).  

# Single smORF characterisation  

Single-gene in-depth studies are generally a better guide to smORF function than high-throughput genome-wide studies. However, they are difficult to undertake since smORF genes are a very small mutational target, and their encoded peptides are smaller and less stable in biochemical assays. Nonetheless, here we revise some recent studies that seem to cement postulated trends in smORF function and suggest new ones.  

# Membrane-related sCDS peptides  

Bouncer  is  an  80-aa  microprotein  belonging  to  the  Ly6/uPAR  superfamily,  identified in zebrafish through studies on predicted protein-coding genes [33]. This microprotein is crucial for fertilisation because it facilitates the entry of sperm into the egg. Herberg et al. [33] found that Bouncer has homology with the SPACA4 protein in tetrapods, suggesting evolutionary conservation across species. Bouncer is located on the surface of the egg and interacts with sperm in a species-specific manner, acting as a ‘gatekeeper’ to ensure that only sperm from the same species can successfully fertilise the egg, thereby contributing to reproductive isolation.  

Kuduk encodes a 78-aa transmembrane protein that belongs to a member of the UPF0197 proteins conserved from flies to humans and is located at the outer nuclear membrane [66]. A cellular function for Kuduk linking the nuclear envelope with the cytoskeleton was described in fly ovaries using mosaic animals, since Kuduk mutants die as larvae [67]. This suggested function of Kuduk involving the cytoskeleton is based on its interaction with the LINC complex. The LINC complex is a conserved group of proteins that physically connect the nucleus and cytoskeleton through two main domains (KASH and SUN) that link the inner and outer nuclear membranes. Through protein tagging and coimmunoprecipitation experiments, it has been demonstrated that Kuduk interacts with LINC complex through Klarsicht, a protein that has a KASH domain [66]. Such evidence suggests that Kuduk modulates the LINC complex [66,67].  

Pegasus (peg) encodes an 80-aa secreted peptide required to extend the diffusion range of the signalling growth factor Wingless/Wnt1 [18]. Mutants mimic a subtle loss of wingless phenotype, only visible at the microscope, and have reduced viability. The Peg peptide probably works by binding Wingless/Wnt1 and repressing the interaction with peptidoglycans of the extracellular matrix, an interaction that otherwise reduces Wnt1 diffusion. Interestingly, Peg contains an incomplete Kazal domain, looking like a truncated version on an ancestral canonical long protein, thus similar to a small interfering peptide [19] (Figure 1).  

Pueyo et al. [68] showed that the Drosophila smORF gene purriato (CG9034) is  translated into a 77-aa microprotein with a single transmembrane alpha-helix domain. Their results show localisation in membrane organelles involved in membrane synthesis and traffic. Purriato is required for protein homeostasis under mechanical stress mostly in the muscles, likely due to their higher reliance on macroprotein assemblies to exert mechanical forces. These results are based on RNAi techniques and hence also subject to technical caveats (partial or offtarget lack of function effects), but, unlike in Bosch et al., in principle a morphological and cellular phenotype is apparent.  

Interestingly, all these recently characterised sCDSs (Bouncer, Kuduk, Pegasus, and  Purriato) seem to act as regulators in membrane-related functions, altogether supporting a trend in sCDS, and possibly smORF, peptide function as membrane-related regulators of canonical proteins [6,9,57,69] (Figure 2C) [69].  

# Peptides encoded by lncRNAs  

Linc-miprep and linc-wrb are two lncORFs that, in zebrafish, encode microproteins of 87 and 93 aa,  respectively  [70].  Both  genes  are  highly  conserved  from  zebrafish  to  the  human  gene HMGN1. Linc-miprep and Linc-wrb are nuclear-localised proteins that seem to work together in cerebellar cells and oligodendrocyte progenitor cells (OPCs), impacting locomotor activity and behaviour in a dose-dependent manner. It seems that changes in their expression alter chromatin accessibility, affecting $N\!\cdot$ -methyl-D-aspartate receptor signalling in OPCs. From an evolutionary point of view, these  two smORFs  first appear in jawed vertebrates, and mutations in both genes seem to more severely affect the set of nerve cells that emerge from this evolutionary juncture. These findings suggest that the emergence and regulation of these genes coevolved with the development of these cell lineages [70] and raise the interesting possibility that smORF peptides and microproteins may work together, perhaps as a kind of ‘cleaved multidomain protein,’ and thus overcome limitations imposed by their small size.  

pTUNAR [71], also known as BNLN [72], is a human lncRNA encoding lncORFs. The TUNAR RNA is highly expressed in the central nervous system and pancreas, among other organs [72]. RiboSeq and proteomics show that this lncORF produces 48- to 63-aa-long peptides well conserved amongst vertebrates [72]. pTUNAR localises in the endoplasmic reticulum (ER) and is involved in intracellular ${\mathrm{C}}\mathfrak{a}^{2+}$ coordination [71,72]. In mouse embryonic stem cells (mESCs), pTUNAR overexpression increases ${\mathrm{Ca}}^{2+}$ levels in the ER and reuptake efficiency from the cytoplasm, impairing neural differentiation and reducing neurite formation [71]. However, pTUNAR seems to lower ER ${\mathrm{Ca}}^{2+}$ levels in pancreatic $\upbeta$ -cells [72], turning on glucosestimulated induced secretion (GSIS) [72]. Thus, TUNAR function seems to be yet another smORF interactor of the SERCA family, regulating ${\mathrm{Ca}}^{2+}$ -ATPase activity, albeit in a cell context dependent manner.  

Another SEP encoded by an lncORF is pTINCR, synthesised from the TINCR lncRNA, highly expressed in human and mouse skin [73]. TINCR produces conserved 87- to 140-aa small proteins across vertebrates, as evidenced by in vitro translation, MS, Western blotting, and immunostaining. pTINCR is a CDC42 activator involved in keratinocyte proliferation, promoting epithelial differentiation in human and mouse cell cultures [74]. In vivo pTINCR overexpression reduces teratoma malignancy in mice and promotes differentiation in HaCaT cells and other human epithelial tumour cell cultures. Molecularly, pTINCR resembles a ubiquitin-like protein with two conserved SUMO interacting motifs at the C-terminus, interacting with SUMO, and allowing CDC42 activation that triggers epithelial differentiation or tumour suppression [73].  

A third recently characterised lncRNA peptide is PACMP, encoded by the human lncRNA CTD2256P15.2. The cytoplasmic 44-aa PACMP peptide has a dual function in maintaining CtIP protein stability and promoting PARylation signalling in DNA repair, thus enhancing chemoresistance. Depletion of this lncRNA inhibits tumour growth but also confers sensitivity to PARP/ATR/CDK4/ 6 inhibitors and multiple DNA damage-inducing reagents [75].  

These works fit a previously identified trend for lncRNA peptide function in cancer according to Ye and collaborators [76]. In that review, the authors revised ten lncORF-produced peptides of 53 to 130 aa. In seven cases, these peptides affect cell proliferation and migration in a variety of systems, apparently mediated by different molecular mechanisms. However, it is not clear whether this apparent functional clustering may be a sampling bias, as the research focused from the start on oncogenesis and cancer. An effect on cell proliferation and migration might betray a generic cellular requirement, the molecular bases of which may differ from the peptides’ impact under oncogenic conditions. A further interesting case may be Nobody, a 7 kDa peptide possibly involved in RNA stability and degradation [77], a process where other lncORFs have been implicated (see later). In principle, these lncORF examples offer a new window for anticancer treatment, either as peptidic drugs themselves or as new targets for existing drugs.  

# Cis function – a noncoding path to peptide function  

Dozier et al. [78] characterised several translated lncORFs translated from miRNA-producing RNAs in Drosophila and humans (Figure 3). Translation of these smORFs helps the function of the cis-encoded miRNA, but, surprisingly, this cooperation is based not on peptide function but on the act of smORF translation. Experiments that block translation reduce miRNA function, whereas changing the entire ORF aa sequence had no effect. The molecular basis of this effect is not clear but seems to be based on regulating the synthesis of the miRNA from the original lncRNA transcript. This is similar to what had been observed in plants [79]. It is possible that elements of the translation machinery acting on these lncORFs provide a link to RNA processing and surveillance [80]. These miRNA-related smORFs can be understood as a new type of cisregulatory tool to enhance the expression and function of the miRNA produced by their transcripts. Cis-function is not usually studied as part of smORF and lncORF function, but it would be interesting to revisit how many putative lncORF peptide functions are actually mediated by their encoding lncRNA.  

![](images/5c125e9379834edc61034b32380502fceb1d6d8234cc7756af3bb4bbfd22b564.jpg)  
Figure 3. Cis-roles of long noncoding RNA (lncRNA) small open reading frames (smORFs). Translation of smORFs from lncRNA helps the function of the cis-encoded miRNA. This help could be due to the translation of the smORF itself (as Dozier et al. proposed [78]) or by the function of the peptide encoded by the smORF. In plants, it has been proposed that smORF-expressed peptides (SEPs) may be involved in the regulation of the transcription of miRNA genes [79]. Translation of the lncORFs may act either on the pri-miRNA, which is cleaved by the enzyme DROSHA, forming the  pre-miRNA,  or  on  the DICER processing the pre-miRNA to generate a mature miRNA. The  ARGONAUTE protein binds to the mature miRNA, forming the RNA-induced silencing (RISC) complex, which binds to the target mRNA to degrade it or inhibit its translation [94]. However, cis-smORF translation, whether pioneering or productive, involves ribosomes; thus, it is possible that elements of the translation machinery provide an unsuspected link to RNA processing and surveillance mechanisms [78,79,95]. Solid arrows: corroborated interactions; dashed arrows: putative interactions.  

Altogether, the results reported by Chen et al.[64] and Dozier et al.[78] reinforce previous evidence for cis-related smORF functions, whereby smORFs modulate other functions present in their transcript. This cis-regulation can be mediated by the smORF peptide or the translation machinery, but evolution-wise it provides a cis-to-trans path for smORFs to achieve coding function (Figure 2). Random smORFs can exert an influence on their encoding RNAs, either by regulating the processing and degradation of miRNA-containing lncRNAs or by regulating positively [11]  or negatively [65] the translation of canonical, longer ORFs in an mRNA (Figure 2A). This influence can be modulated by evolution by enhancing or reducing the smORF translation. Once past a certain threshold of TE, the produced peptide would be biologically detectable and might start to interfere with its physically close relatives: the encoding RNA and any other protein translated from it or any protein interacting with either such RNA or cis-protein (Figure 2B). Most of these interactions would be detrimental, but even a ‘toxic’ effect may be selected as a form of negative regulation. Any beneficial interaction will be improved by evolution acting on the new peptide expression, stability, and aa sequence. If peptide expression and stability are increased, even though its original function may have been cis-regulatory, there is nothing to stop the newly abundant peptide from diffusing across the cell and interacting with other protein partners. Beneficial  trans-interactions  may  require  expression  changes,  making  it  advantageous  to dissociate  the  smORF  from  other  cis-RNA  functions,  and  thus  a  truncated  or  modified version of the RNA may be selected containing the peptide-producing smORF as the sole ORF (i.e., a new sCDS) (Figure 2C).  

This ‘cis-to-trans path to function’ model provides a parsimonious route from noncoding ORFs to fully coding sCDS. The limited likelihood of each step accounts for the great disparity between smORF numbers, from tens of thousands of lncORFs and uORFs to hundreds of sCDSs per genome. Two further factors can be integrated into this view. First is evidence for de novo smORF genes. Although this is still controversial and no clear mechanism has been accepted, speciesspecific genes are regularly annotated in multiple species, from unicellular eukaryotes to metazoans, and plants. Furthermore, Vakirlis et al. provide compelling examples for the emergence of coding function  from  ancestral noncoding sequences  [57]. The second factor  is that young coding genes seem to quickly acquire a highly tuned function, usually as membrane-related regulators of larger proteins. There is a debate about whether there is a mysterious nucleotide or structural bias in noncoding RNA sequences that would favour encoding certain codon compositions (and hence protein domains when they acquire translation) [60,61,81] or whether we are simply missing the evolutionary short-lived intermediate aa forms predating the current ones. Fixed young genes appear not to encode unstructured peptides (as could be expected from random sequences) but usually contain a structured domain, typically transmembrane alpha-helixes, whereas speciesspecific uORF and lncORF peptides, despite their aa sequence bias [6,39,57], are much less structured, if at all. Thus, a ‘cis-path to function’ might provide a niche for the adaptive value of smORF translation and later for the evolution of the encoded peptide.  

# Concluding remarks and future perspectives  

In summary, recent studies conclude that (i) smORFs may only be translated in a minority of cases; and (ii) that, among those translated, only a minority provide a function, be it coding or noncoding,  cis  or  trans.  This  raises  a  series  of  questions  that  remain  unresolved (see Outstanding questions). In the case of sCDSs, their ORFs occupy a larger fraction of their mRNA, leaving little space for cis- or trans-interacting noncoding elements, and their strongly expressed peptides and microproteins would seem difficult to restrain to their cis environment. In  the case of  uORFs  or lncORFs with coding functions, the same genetic analysis data that indicate such coding function $(\mathrm{Bo}\times\,1)$ )  could  also  probe  possible  cis-interactions  with  their encoding lncRNA.  

Subtle phenotypes fit with the expectation of smORFs as new genes, emerging with minor adaptive effects. Furthermore, it is possible that translation of smORFs with little or no function, or even mildly deleterious effects, may be evolutionarily tolerated as they might provide future adaptive value. In this view, even nonfunctional or mildly detrimental smORF translation might be tolerated as a source of new genes. Recent evidence reinforces that membrane association is a fertile niche for smORF peptides, whether old and widely conserved or new and emergent. A fascinating possibility is that our genome is inevitably primed to constantly produce new membrane-associated peptides out of previously noncoding sequences, guided by biases in noncoding DNA and RNA sequences. It remains to fully prove this hypothesis, for which more incisive functional genomics methods and more in-depth gene studies will be needed. But particularly, we would need to understand the reasons, contingent or deterministic, for this noncoding sequence bias (whether DNA- or RNA-based or both). More speculatively, we might ask its possible relationship with the emergence of coding genes during the early episodes of life on Earth [82,83] and even its relevance to the genomic organisation of possible extraterrestrial life.  

# Outstanding questions  

Are current ribosomal profiling protocols enough to reveal all smORF translation, or  do  we  need  more  sensitive techniques?  

At  which  threshold  does  smORF translation becomes biologically relevant and not just noise?  

Do all uORFs and lncORFs have a cisregulatory  effect  on  their  encoding RNAs? Is there a threshold level, or a qualitative characteristic, differentiating cis-regulatory translation from transrelevant peptide production?  

Why are so many smORF peptides not detected  by  proteomics? Is this a consequence  of  low  peptide  levels and stability, or can peptidomics be improved?  

Are  smORFs  really  de  novo  genes? Which  ones?  At  which  point  does absence  of  evidence  (no  homologies detected) become evidence of absence (there are no gene relatives)?  

Do biases in noncoding RNA sequences dictate the amino acid sequence of peptides evolved from them? What are the reasons for these biases?  

What do we call ‘coding,’ ‘potentially coding,’ and ‘noncoding’? Do such distinctions make sense any longer? If  there  is  a  current  evolutionary process generating new genes with specific  characteristics,  how  does  it impinge  on  our  view  of  the  origin  of life on Earth and elsewhere?  

# Acknowledgments  

We thank Rosario Morrugares and Sarah A. Bishop for comments on the manuscript. This work was funded by grants from the Spanish Ministerio de Ciencia to J.P.C. (refs. PID2022-141831NB-I00/AEI/10.13039/501100011033/FEDER, UE and EUR2022-134063),  the  project  CEX2020-001088-M,  funded  by  the  Programme  Grant  MCIN/AEI/10.13039/ 501100011033O and an FPI fellowship (PRE2019-091601) associated with Programme Grant MDM-2016-0687 to C.B.A.  

# Declaration of interests  

The authors declare that they have no competing interests.  

# Supplemental information  

Supplemental information associated with this article can be found online at https://doi.org/10.1016/j.tig.2024.10.012.  

# References  

1.  Basrai, M.A. et al. (1997) Small open reading frames: beautiful needles in the haystack. Genome Res. 7, 768–771   
2.  Cao, $\mathsf{X}.$ and  Slavoff,  S.A.  (2020)  Non-AUG  start  codons: expanding and regulating the small and alternative ORFeome. Exp. Cell Res. 391, 111973   
3.  Kastenmayer, J.P. et al. (2006) Functional genomics of genes with small open reading frames (sORFs) in S. cerevisiae. Genome Res. 16, 365–373   
4.  Ladoukakis, E. et al. (2011) Hundreds of putatively functional small open reading frames in Drosophila. Genome Biol. 12, R118   
5.  Frith, M.C. et al. (2006) The abundance of short proteins in the mammalian proteome. PLoS Genet. 2, 515–528   
6.  Couso, J.P. and Patraquim, P. (2017) Classification and function of small open reading frames. Nat. Rev. Mol. Cell Biol. 18, 575–589   
7.  Kondo, T. et al. (2010) Small peptides switch the transcriptional activity of Shavenbaby during Drosophila embryogenesis. Science 329, 336–339   
8.  Pauli, A. et al. (2014) Toddler: an embryonic signal that promotes cell movement via Apelin receptors. Science 343, 1248636   
9.  Aspden, J.L. et al. (2014) Extensive translation of small open reading frames revealed by poly-ribo-seq. Elife 3, e03528   
10.  Ruiz-Orera, J. et al. (2014) Long non-coding RNAs as a source of new peptides. Elife 3, e03523   
11.  Patraquim, P. et al. (2020) Developmental regulation of canonical and small ORF translation from mRNAs. Genome Biol. 21, 128   
12.  Olexiouk, V. et al. (2015) sORFs.org: a repository of small ORFs identified  by  ribosome  profiling.  Nucleic  Acids  Res.  44, D324–D329   
13.  Hao,  Y.  et  al.  (2018)  SmProt:  a  database  of  small  proteins encoded by annotated coding and non-coding RNA loci. Brief. Bioinform. 19, 636–643   
14.  Galindo, M.I. et al. (2007) Peptides encoded by short ORFs control development and define a new eukaryotic gene family. PLoS Biol. 5, 1052–1062   
15.  Magny, E.G. et al. (2013) Conserved regulation of cardiac calcium uptake by peptides encoded in small open reading frames. Science 341, 1116–1120   
16.  Pueyo, J.I. et al. (2016) Hemotin, a regulator of phagocytosis encoded by a small ORF and conserved across metazoans. PLoS Biol. 14, e1002395   
17.  Mudge, J.M. et al. (2022) Standardized annotation of translated open reading frames. Nat. Biotechnol. 40, 994–999   
18.  Magny, E.G. et al. (2021) Pegasus, a small extracellular peptide enhancing short-range diffusion of Wingless. Nat. Commun. 12, 5660   
19.  Seo, P.J. et al. (2011) Competitive inhibition of transcription factors by small interfering peptides. Trends Plant Sci. 16, 541–549   
20.  Weisman, C. et al. (2020) Many, but not all, lineage-specific genes can be explained by homology detection failure. PLoS Biol. 18, e3000862   
21.  Kozak, M. (2005) Regulation of translation via mRNA structure in prokaryotes and eukaryotes. Gene 361, 13–37   
22.  Andrews, S.J. and Rothnagel, J.A. (2014) Emerging evidence for functional peptides encoded by short open reading frames. Nat. Rev. Genet. 15, 193–204   
23.  Crowe, M.L. et al. (2006) Evidence for conservation and selection of upstream open reading frames suggests probable encoding of bioactive peptides. BMC Genomics 7, 16   
24.  Mueller, P.P. and Hinnebusch, A.G. (1986) Multiple upstream AUG codons mediate translational control of GCN4. Cell 45, 201–207   
25.  Niederer, R.O. et al. (2017) Long noncoding RNAs in the yeast S. cerevisiae. Adv. Exp. Med. Biol. 1008, 119–132   
26.  Anderson, D.M. et al. (2015) A micropeptide encoded by a putative long noncoding RNA regulates muscle performance. Cell 160, 595–606  
27.  Anderson, D.M. et al. (2016) Widespread control of calcium signaling by a family of SERCA-inhibiting micropeptides. Sci. Signal. 9, ra119   
28.  Nelson, B.R. et al. (2016) A peptide encoded by a transcript annotated as long noncoding RNA enhances SERCA activity in muscle. Science 351, 271–275   
29.  Makarewich, C.A. and Olson, E.N. (2017) Mining for micropeptides. Trends Cell Biol. 27, 685–696  
30.  Odermatt, A. et al. (1997) Characterization of the gene encoding human sarcolipin (SLN), a proteolipid associated with SERCA1: absence of structural mutations in five patients with Brody disease. Genomics 45, 541–553   
31.  Bi, P. et al. (2017) Control of muscle formation by the fusogenic micropeptide myomixer. Science 356, 323–327   
32.  Sunjaya, A.P. et al. (2019) Apela/Elabela/Toddler: new perspectives in molecular mechanism of heart failure. Global Cardiol. Sci. Pract. 2019, e201915   
33.  Herberg, S. et al. (2018) The Ly6/uPAR protein Bouncer is necessary and sufficient for species-specific fertilization.  Science 361, 1029–1033   
34.  Saghatelian, A. and Couso, J.P. (2015) Discovery and characterization of smORF-encoded bioactive polypeptides. Nat. Chem. Biol. 11, 909–916   
35.  Hemm, M.R. et al. (2008) Small membrane proteins found by comparative genomics and ribosome binding site models. Mol. Microbiol. 70, 1487–1501   
36.  Begun, D.J. et al. (2006) Recently evolved genes identified from Drosophila yakuba and D. erecta accessory gland expressed sequence tags. Genetics 172, 1675–1681   
37.  Zhao, L. et al. (2014) Origin and spread of de novo genes in Drosophila melanogaster populations. Science 343, 769–772   
38.  Baalsrud, H.T. et al. (2017) De novo gene evolution of antifreeze glycoproteins in codfishes revealed by whole genome sequence data. Mol. Biol. Evol. 35, 593–606   
39.  Carvunis, A.R. et al. (2012) Proto-genes and de novo gene birth. Nature 487, 370–374   
40.  Mackowiak, S.D. et al. (2015) Extensive identification and analysis of conserved small ORFs in animals. Genome Biol. 16, 179   
41.  Prabakaran, S. et al. (2014) Quantitative profiling of peptides from RNAs classified as noncoding. Nat. Commun. 5, 5429   
42.  Slavoff, S.A. et al. (2013) Peptidomic discovery of short open reading frame-encoded peptides in human cells. Nat. Chem. Biol. 9, 59–64   
43.  Vanderperre, B. et al. (2013) Direct detection of alternative open reading frames translation products in human significantly expands the proteome. PLoS One 8, e70698   
44.  Ma, J. et al. (2014) Discovery of human sORF-encoded polypeptides (SEPs) in cell lines and tissue. J. Proteome Res. 13, 1757–1765   
45.  Wilhelm, M. et al. (2014) Mass-spectrometry-based draft of the human proteome. Nature 509, 582–587   
46.  Kim, M.S. et al. (2014) A draft map of the human proteome. Nature 509, 575–581   
47.  Casas-Vila, N. et al. (2017) The developmental proteome of Drosophila melanogaster. Genome Res. 27, 1273–1285   
48.  Bazzini, A.A. et al. (2014) Identification of small ORFs in vertebrates using ribosome footprinting and evolutionary conservation. EMBO J. 33, 981–993   
49.  Ingolia, N.T. et al. (2011) Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes. Cell 147, 789–802   
50.  Mumtaz, M.A. and Couso, J.P. (2015) Ribosomal profiling adds new coding sequences to the proteome. Biochem. Soc. Trans. 43, 1271–1276   
51.  Chew, G.L. et al. (2013) Ribosome profiling reveals resemblance between long non-coding RNAs and 5′ leaders of coding RNAs. Development 140, 2828–2834   
52.  VanInsberghe, M. et al. (2021) Single-cell Ribo-seq reveals cell cycle-dependent translational pausing. Nature 597, 561–565   
53.  Su, D. et al. (2024) Ribosome profiling: a powerful tool in oncological research. Biomark. Res. 12, 11   
54.  Dong, X. et al. (2023) Small open reading frame-encoded micropeptides: an emerging protein world. Int. J. Mol. Sci. 24, 10562   
55.  Patraquim, P. et al. (2022) Translation and natural selection of micropeptides from long non-canonical RNAs. Nat. Commun. 13, 6515   
56.  Ruiz-Orera, J. et al. (2018) Translation of neutrally evolving peptides provides a basis for de novo gene evolution. Nat. Ecol. Evol. 2, 890–896   
57.  Vakirlis, N. et al. (2020) De novo emergence of adaptive membrane  proteins  from  thymine-rich  genomic  sequences.  Nat. Commun. 11, 781   
58.  Neme, R. and Tautz, D. (2013) Phylogenetic patterns of emergence of new genes support a model of frequent de novo evolution. BMC Genomics 14, 117   
59.  Tautz, D.D.-L. and T. (2011) The evolutionary origin of orphan genes. Nat. Rev. Genet. 12, 692–702   
60.  D’Onofrio, G. et al. (1999) The correlation of protein hydropathy with the base composition of coding sequences. Gene 238, 3–14   
61.  D’Onofrio, G. et al. (1991) Correlations between the compositional properties of human genes, codon usage, and amino acid composition of proteins. J. Mol. Evol. 32, 504–510   
62.  Bosch, J.A. et al. (2023) Molecular and functional characterization of the Drosophila melanogaster conserved smORFome. Cell Rep. 42, 113311   
63.  Miller, B. et al. (2020) Peptides derived from small mitochondrial open reading frames: genomic, biological, and therapeutic implications. Exp. Cell Res. 393, 112056   
64.  Chen, J. et al. (2020) Pervasive functional translation of noncanonical human open reading frames. Science 367, 1140–1146   
65.  Johnstone, T.G. et al. (2016) Upstream ORFs are prevalent translational repressors in vertebrates. EMBO J. 35, 706–723   
66.  Ding, Z.Y. et al. (2017) Outer nuclear membrane protein Kuduk modulates the LINC complex and nuclear envelope architecture. J. Cell Biol. 216, 2827–2841   
67.  Villa-Fombuena, G. et al. (2021) Live imaging of the Drosophila ovarian niche shows spectrosome and centrosome dynamics during asymmetric germline stem cell division. Development 148, dev199716   
68.  Pueyo, J.I. et al. (2023) Purriato is a conserved small open reading frame gene that interacts with the CASA pathway to regulate muscle homeostasis and epithelial tissue growth in Drosophila. Front. Cell Dev. Biol. 11, 1117454   
69.  Makarewich,  C.A.  (2020)  The  hidden  world  of  membrane microproteins. Exp. Cell Res. 388, 111853   
70.  Tornini,  V.A.  et  al.  (2023)  linc-mipep  and  linc-wrb  encode micropeptides that regulate chromatin accessibility in vertebratespecific neural cells. Elife 12, e82249   
71.  Senís,  E.  et  al.  (2021)  TUNAR  lncRNA  encodes  a microprotein that regulates neural differentiation and neurite formation by modulating calcium dynamics. Front. Cell Dev. Biol. 9, 747667   
72.  Li, M. et al. (2021) A putative long noncoding RNA-encoded micropeptide maintains cellular homeostasis in pancreatic β cells. Mol. Ther. Nucleic Acids 26, 307–320   
73.  Boix, O. et al. (2022) pTINCR microprotein promotes epithelial differentiation and suppresses tumor growth through CDC42 SUMOylation and activation. Nat. Commun. 13, 6840   
74.  Nita, A. et al. (2021) A ubiquitin-like protein encoded by the ‘noncoding’ RNA TINCR promotes keratinocyte proliferation and wound healing. PLoS Genet. 17, e1009686   
75.  Zhang, C. et al. (2022) Micropeptide PACMP inhibition elicits synthetic lethal effects by decreasing CtIP and poly(ADP-ribosyl) ation. Mol. Cell 82, 1297–1312.e8   
76.  Ye,  M.  et  al.  (2020)  Emerging  role  of  long  noncoding  RNAencoded micropeptides in cancer. Cancer Cell 20, 506   
77.  D’Lima, N.G. et al. (2017) A human microprotein that interacts with  the  mRNA  decapping  complex.  Nat.  Chem.  Biol.  13, 174–180   
78.  Dozier, C. et al. (2022) Small ORFs as new regulators of primiRNAs and miRNAs expression in human and Drosophila. Int. J. Mol. Sci. 23, 5764   
79.  Ren, Y. et al. (2021) Coding of Non-coding RNA: insights into the regulatory  functions  of  pri-microRNA-encoded  peptides  in plants. Front. Plant Sci. 12, 641351   
80.  Bazzini, A.A. et al. (2012) Ribosome profiling shows that miR-430 reduces translation before causing mRNA decay in zebrafish. Science 336, 233–237   
81.  Wu, Q. and Bazzini, A.A. (2023) Translation and mRNA stability control. Annu. Rev. Biochem. 92, 227–245   
82.  Kirschning, A. (2022) On the evolutionary history of the twenty encoded amino acids. Chemistry 28, e202201419   
83.  Youssef, N. et al. (2021) Shifts in amino acid preferences as proteins evolve: a synthesis of experimental and theoretical work. Protein Sci. Publ. Protein Soc. 30, 2009–2028   
84.  Ingolia, N.T. et al. (2009) Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. Science 324, 218–223   
85.  Omenn, G.S. et al. (2017) Progress on the HUPO draft human proteome: 2017 metrics of the Human Proteome Project. J. Proteome Res. 16, 4281–4287   
86.  Lin,  M.F.  et  al.  (2011)  PhyloCSF:  a  comparative  genomics method to distinguish protein coding and non-coding regions. Bioinformatics 27, i275–i282   
87.  Yang, D. et al. (2000) Evidence that processed small dsRNAs may  mediate  sequence-specific  mRNA  degradation  during RNAi in Drosophila embryos. Curr. Biol. 10, 1191–1200   
88.  Cai, B. et al. (2017) LncRNA-Six1 encodes a micropeptide to activate Six1 in cis and is involved in cell proliferation and muscle growth. Front. Physiol. 8, 230   
89.  Matsumoto, A. et al. (2017) mTORC1 and muscle regeneration are regulated by the LINC00961-encoded SPAR polypeptide. Nature 541, 228–232   
90.  Stein,  C.S.  et  al.  (2018)  Mitoregulin:  a  lncRNA-encoded microprotein that supports mitochondrial supercomplexes and respiratory efficiency. Cell Rep. 23, 3710–3720.e8   
91.  Makarewich,  C.A.  et  al.  (2018)  MOXI  is  a  mitochondrial micropeptide that enhances fatty acid $\upbeta$ -oxidation. Cell Rep. 23, 3701–3709   
92.  Szamecz, B. et al. (2008) eIF3a cooperates with sequences 5′ of uORF1 to promote resumption of scanning by post-termination ribosomes  for  reinitiation on GCN4 mRNA. Genes  Dev.  22, 2414–2425   
93.  Wu, Q. et al. (2020) Translation of small downstream ORFs enhances translation of canonical main open reading frames. EMBO J. 39, e104763   
94.  Meng, X. et al. (2021) Interplay between miRNAs and lncRNAs: mode  of  action  and  biological  roles  in  plant  development and  stress  adaptation.  Comput.  Struct.  Biotechnol.  J.  19, 2567–2574   
95.  Andjus, S. et al. (2024) Pervasive translation of Xrn1-sensitive unstable long noncoding RNAs in yeast. RNA 30, 662–679  